{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "This is a simple walkthrough of how the code works.\n",
    "\n",
    "First, we look at the main components of the pipeline, the `context_shortener` and the `form_filler`\n",
    "The `context_shortener` is responsible for the retrieval part (reducing the length of the context, from the full document to the chunks.)\n",
    "The `form_filler` takes the context and a form, and fills it."
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## context shortening"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/sondre/miniconda3/envs/upcast/lib/python3.10/site-packages/tqdm/auto.py:21: TqdmWarning: IProgress not found. Please update jupyter and ipywidgets. See https://ipywidgets.readthedocs.io/en/stable/user_install.html\n",
      "  from .autonotebook import tqdm as notebook_tqdm\n"
     ]
    }
   ],
   "source": [
    "import context_shortening"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "First we chunk some text"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "0 : Melanoma is the most dangerous type of skin\n",
      "1 : cancer; it develops from the melanin-producing\n",
      "2 : cells known as melanocytes.[1] It typically\n",
      "3 : occurs in the skin, but may rarely occur in the\n",
      "4 : mouth, intestines, or eye (uveal melanoma).[1][2]\n"
     ]
    }
   ],
   "source": [
    "# some sample text from wikipedia\n",
    "document = \"\"\"Melanoma is the most dangerous type of skin cancer; it develops from the melanin-producing cells known as melanocytes.[1] It typically occurs in the skin, but may rarely occur in the mouth, intestines, or eye (uveal melanoma).[1][2]\"\"\"\n",
    "chunks = context_shortening.chunk_by_headeres_and_clean(document, 50, 0, False)\n",
    "for i, chunk in enumerate(chunks):\n",
    "    print(i,\":\",chunk.text)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Now we need a pydantic model in order for the context_shortener to know what to look for."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [],
   "source": [
    "import pydantic\n",
    "from typing import Literal\n",
    "class Disease(pydantic.BaseModel):\n",
    "\n",
    "    Disease_name: str = pydantic.Field(description=\"Name of disease\")\n",
    "    Body_part: Literal[\"Leg\", \"Arm\", \"Skin\"] = pydantic.Field(description=\"Affected body part\")\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Now lets try out some context shorteners."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "mouth, intestines, or eye (uveal melanoma).[1][2]\n",
      "...\n",
      "occurs in the skin, but may rarely occur in the\n",
      "...\n",
      "Melanoma is the most dangerous type of skin\n"
     ]
    }
   ],
   "source": [
    "description_based_retriever = context_shortening.Retrieval(\n",
    "    chunk_info_to_compare=\"direct\", # this means we use the chunk directly, as opposed to \"keybert\" which generates n keywords per chunk.\n",
    "    field_info_to_compare=\"description\",\n",
    "    include_choice_every=1,\n",
    "    embedding_model_id=\"all-MiniLM-L6-v2\",\n",
    "    n_keywords=1,\n",
    "    top_k=3, # find k most relevant chunks, and put into one string\n",
    "    chunk_size=50,\n",
    "    chunk_overlap=0,\n",
    "    pydantic_form=Disease,\n",
    ")\n",
    "description_based_retriever.set_document(document)\n",
    "disease_name_context = description_based_retriever(answer_field_name = \"Disease_name\")\n",
    "print(disease_name_context)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "occurs in the skin, but may rarely occur in the\n",
      "...\n",
      "mouth, intestines, or eye (uveal melanoma).[1][2]\n",
      "...\n",
      "Melanoma is the most dangerous type of skin\n"
     ]
    }
   ],
   "source": [
    "body_part_context = description_based_retriever(answer_field_name = \"Body_part\")\n",
    "print(body_part_context)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "The choices or choise-list alternative require only literal values in the pydantic_form"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Melanoma is the most dangerous type of skin\n",
      "...\n",
      "occurs in the skin, but may rarely occur in the\n"
     ]
    }
   ],
   "source": [
    "import pydantic\n",
    "from typing import Literal\n",
    "class LiteralDisease(pydantic.BaseModel):\n",
    "\n",
    "    Disease_name: Literal[\"Carcinoma\", \"'Melanoma'\", \"Medulloblastoma\"] = pydantic.Field(description=\"Name of disease\")\n",
    "    Body_part: Literal[\"Leg\", \"Arm\", \"Skin\"] = pydantic.Field(description=\"Affected body part\")\n",
    "\n",
    "\n",
    "\n",
    "\n",
    "choices_based_retriever = context_shortening.Retrieval(\n",
    "    chunk_info_to_compare=\"direct\", # this means we use the chunk directly, as opposed to \"keybert\" which generates n keywords per chunk.\n",
    "    field_info_to_compare=\"choices\",\n",
    "    include_choice_every=1,\n",
    "    embedding_model_id=\"all-MiniLM-L6-v2\",\n",
    "    n_keywords=1,\n",
    "    top_k=2,\n",
    "    chunk_size=50,\n",
    "    chunk_overlap=0,\n",
    "    pydantic_form=LiteralDisease,\n",
    ")\n",
    "\n",
    "choices_based_retriever.set_document(document)\n",
    "\n",
    "context = choices_based_retriever(answer_field_name = \"Body_part\")\n",
    "print(context)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Form Filling\n",
    "\n",
    "\n",
    "First we load an llm - llama3.1 8b instruct, gptq-int4 quantization."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "CUDA extension not installed.\n",
      "CUDA extension not installed.\n",
      "Loading checkpoint shards: 100%|██████████| 2/2 [00:02<00:00,  1.18s/it]\n",
      "Some weights of the model checkpoint at hugging-quants/Meta-Llama-3.1-8B-Instruct-GPTQ-INT4 were not used when initializing LlamaForCausalLM: ['model.layers.0.mlp.down_proj.bias', 'model.layers.0.mlp.gate_proj.bias', 'model.layers.0.mlp.up_proj.bias', 'model.layers.0.self_attn.k_proj.bias', 'model.layers.0.self_attn.o_proj.bias', 'model.layers.0.self_attn.q_proj.bias', 'model.layers.0.self_attn.v_proj.bias', 'model.layers.1.mlp.down_proj.bias', 'model.layers.1.mlp.gate_proj.bias', 'model.layers.1.mlp.up_proj.bias', 'model.layers.1.self_attn.k_proj.bias', 'model.layers.1.self_attn.o_proj.bias', 'model.layers.1.self_attn.q_proj.bias', 'model.layers.1.self_attn.v_proj.bias', 'model.layers.10.mlp.down_proj.bias', 'model.layers.10.mlp.gate_proj.bias', 'model.layers.10.mlp.up_proj.bias', 'model.layers.10.self_attn.k_proj.bias', 'model.layers.10.self_attn.o_proj.bias', 'model.layers.10.self_attn.q_proj.bias', 'model.layers.10.self_attn.v_proj.bias', 'model.layers.11.mlp.down_proj.bias', 'model.layers.11.mlp.gate_proj.bias', 'model.layers.11.mlp.up_proj.bias', 'model.layers.11.self_attn.k_proj.bias', 'model.layers.11.self_attn.o_proj.bias', 'model.layers.11.self_attn.q_proj.bias', 'model.layers.11.self_attn.v_proj.bias', 'model.layers.12.mlp.down_proj.bias', 'model.layers.12.mlp.gate_proj.bias', 'model.layers.12.mlp.up_proj.bias', 'model.layers.12.self_attn.k_proj.bias', 'model.layers.12.self_attn.o_proj.bias', 'model.layers.12.self_attn.q_proj.bias', 'model.layers.12.self_attn.v_proj.bias', 'model.layers.13.mlp.down_proj.bias', 'model.layers.13.mlp.gate_proj.bias', 'model.layers.13.mlp.up_proj.bias', 'model.layers.13.self_attn.k_proj.bias', 'model.layers.13.self_attn.o_proj.bias', 'model.layers.13.self_attn.q_proj.bias', 'model.layers.13.self_attn.v_proj.bias', 'model.layers.14.mlp.down_proj.bias', 'model.layers.14.mlp.gate_proj.bias', 'model.layers.14.mlp.up_proj.bias', 'model.layers.14.self_attn.k_proj.bias', 'model.layers.14.self_attn.o_proj.bias', 'model.layers.14.self_attn.q_proj.bias', 'model.layers.14.self_attn.v_proj.bias', 'model.layers.15.mlp.down_proj.bias', 'model.layers.15.mlp.gate_proj.bias', 'model.layers.15.mlp.up_proj.bias', 'model.layers.15.self_attn.k_proj.bias', 'model.layers.15.self_attn.o_proj.bias', 'model.layers.15.self_attn.q_proj.bias', 'model.layers.15.self_attn.v_proj.bias', 'model.layers.16.mlp.down_proj.bias', 'model.layers.16.mlp.gate_proj.bias', 'model.layers.16.mlp.up_proj.bias', 'model.layers.16.self_attn.k_proj.bias', 'model.layers.16.self_attn.o_proj.bias', 'model.layers.16.self_attn.q_proj.bias', 'model.layers.16.self_attn.v_proj.bias', 'model.layers.17.mlp.down_proj.bias', 'model.layers.17.mlp.gate_proj.bias', 'model.layers.17.mlp.up_proj.bias', 'model.layers.17.self_attn.k_proj.bias', 'model.layers.17.self_attn.o_proj.bias', 'model.layers.17.self_attn.q_proj.bias', 'model.layers.17.self_attn.v_proj.bias', 'model.layers.18.mlp.down_proj.bias', 'model.layers.18.mlp.gate_proj.bias', 'model.layers.18.mlp.up_proj.bias', 'model.layers.18.self_attn.k_proj.bias', 'model.layers.18.self_attn.o_proj.bias', 'model.layers.18.self_attn.q_proj.bias', 'model.layers.18.self_attn.v_proj.bias', 'model.layers.19.mlp.down_proj.bias', 'model.layers.19.mlp.gate_proj.bias', 'model.layers.19.mlp.up_proj.bias', 'model.layers.19.self_attn.k_proj.bias', 'model.layers.19.self_attn.o_proj.bias', 'model.layers.19.self_attn.q_proj.bias', 'model.layers.19.self_attn.v_proj.bias', 'model.layers.2.mlp.down_proj.bias', 'model.layers.2.mlp.gate_proj.bias', 'model.layers.2.mlp.up_proj.bias', 'model.layers.2.self_attn.k_proj.bias', 'model.layers.2.self_attn.o_proj.bias', 'model.layers.2.self_attn.q_proj.bias', 'model.layers.2.self_attn.v_proj.bias', 'model.layers.20.mlp.down_proj.bias', 'model.layers.20.mlp.gate_proj.bias', 'model.layers.20.mlp.up_proj.bias', 'model.layers.20.self_attn.k_proj.bias', 'model.layers.20.self_attn.o_proj.bias', 'model.layers.20.self_attn.q_proj.bias', 'model.layers.20.self_attn.v_proj.bias', 'model.layers.21.mlp.down_proj.bias', 'model.layers.21.mlp.gate_proj.bias', 'model.layers.21.mlp.up_proj.bias', 'model.layers.21.self_attn.k_proj.bias', 'model.layers.21.self_attn.o_proj.bias', 'model.layers.21.self_attn.q_proj.bias', 'model.layers.21.self_attn.v_proj.bias', 'model.layers.22.mlp.down_proj.bias', 'model.layers.22.mlp.gate_proj.bias', 'model.layers.22.mlp.up_proj.bias', 'model.layers.22.self_attn.k_proj.bias', 'model.layers.22.self_attn.o_proj.bias', 'model.layers.22.self_attn.q_proj.bias', 'model.layers.22.self_attn.v_proj.bias', 'model.layers.23.mlp.down_proj.bias', 'model.layers.23.mlp.gate_proj.bias', 'model.layers.23.mlp.up_proj.bias', 'model.layers.23.self_attn.k_proj.bias', 'model.layers.23.self_attn.o_proj.bias', 'model.layers.23.self_attn.q_proj.bias', 'model.layers.23.self_attn.v_proj.bias', 'model.layers.24.mlp.down_proj.bias', 'model.layers.24.mlp.gate_proj.bias', 'model.layers.24.mlp.up_proj.bias', 'model.layers.24.self_attn.k_proj.bias', 'model.layers.24.self_attn.o_proj.bias', 'model.layers.24.self_attn.q_proj.bias', 'model.layers.24.self_attn.v_proj.bias', 'model.layers.25.mlp.down_proj.bias', 'model.layers.25.mlp.gate_proj.bias', 'model.layers.25.mlp.up_proj.bias', 'model.layers.25.self_attn.k_proj.bias', 'model.layers.25.self_attn.o_proj.bias', 'model.layers.25.self_attn.q_proj.bias', 'model.layers.25.self_attn.v_proj.bias', 'model.layers.26.mlp.down_proj.bias', 'model.layers.26.mlp.gate_proj.bias', 'model.layers.26.mlp.up_proj.bias', 'model.layers.26.self_attn.k_proj.bias', 'model.layers.26.self_attn.o_proj.bias', 'model.layers.26.self_attn.q_proj.bias', 'model.layers.26.self_attn.v_proj.bias', 'model.layers.27.mlp.down_proj.bias', 'model.layers.27.mlp.gate_proj.bias', 'model.layers.27.mlp.up_proj.bias', 'model.layers.27.self_attn.k_proj.bias', 'model.layers.27.self_attn.o_proj.bias', 'model.layers.27.self_attn.q_proj.bias', 'model.layers.27.self_attn.v_proj.bias', 'model.layers.28.mlp.down_proj.bias', 'model.layers.28.mlp.gate_proj.bias', 'model.layers.28.mlp.up_proj.bias', 'model.layers.28.self_attn.k_proj.bias', 'model.layers.28.self_attn.o_proj.bias', 'model.layers.28.self_attn.q_proj.bias', 'model.layers.28.self_attn.v_proj.bias', 'model.layers.29.mlp.down_proj.bias', 'model.layers.29.mlp.gate_proj.bias', 'model.layers.29.mlp.up_proj.bias', 'model.layers.29.self_attn.k_proj.bias', 'model.layers.29.self_attn.o_proj.bias', 'model.layers.29.self_attn.q_proj.bias', 'model.layers.29.self_attn.v_proj.bias', 'model.layers.3.mlp.down_proj.bias', 'model.layers.3.mlp.gate_proj.bias', 'model.layers.3.mlp.up_proj.bias', 'model.layers.3.self_attn.k_proj.bias', 'model.layers.3.self_attn.o_proj.bias', 'model.layers.3.self_attn.q_proj.bias', 'model.layers.3.self_attn.v_proj.bias', 'model.layers.30.mlp.down_proj.bias', 'model.layers.30.mlp.gate_proj.bias', 'model.layers.30.mlp.up_proj.bias', 'model.layers.30.self_attn.k_proj.bias', 'model.layers.30.self_attn.o_proj.bias', 'model.layers.30.self_attn.q_proj.bias', 'model.layers.30.self_attn.v_proj.bias', 'model.layers.31.mlp.down_proj.bias', 'model.layers.31.mlp.gate_proj.bias', 'model.layers.31.mlp.up_proj.bias', 'model.layers.31.self_attn.k_proj.bias', 'model.layers.31.self_attn.o_proj.bias', 'model.layers.31.self_attn.q_proj.bias', 'model.layers.31.self_attn.v_proj.bias', 'model.layers.4.mlp.down_proj.bias', 'model.layers.4.mlp.gate_proj.bias', 'model.layers.4.mlp.up_proj.bias', 'model.layers.4.self_attn.k_proj.bias', 'model.layers.4.self_attn.o_proj.bias', 'model.layers.4.self_attn.q_proj.bias', 'model.layers.4.self_attn.v_proj.bias', 'model.layers.5.mlp.down_proj.bias', 'model.layers.5.mlp.gate_proj.bias', 'model.layers.5.mlp.up_proj.bias', 'model.layers.5.self_attn.k_proj.bias', 'model.layers.5.self_attn.o_proj.bias', 'model.layers.5.self_attn.q_proj.bias', 'model.layers.5.self_attn.v_proj.bias', 'model.layers.6.mlp.down_proj.bias', 'model.layers.6.mlp.gate_proj.bias', 'model.layers.6.mlp.up_proj.bias', 'model.layers.6.self_attn.k_proj.bias', 'model.layers.6.self_attn.o_proj.bias', 'model.layers.6.self_attn.q_proj.bias', 'model.layers.6.self_attn.v_proj.bias', 'model.layers.7.mlp.down_proj.bias', 'model.layers.7.mlp.gate_proj.bias', 'model.layers.7.mlp.up_proj.bias', 'model.layers.7.self_attn.k_proj.bias', 'model.layers.7.self_attn.o_proj.bias', 'model.layers.7.self_attn.q_proj.bias', 'model.layers.7.self_attn.v_proj.bias', 'model.layers.8.mlp.down_proj.bias', 'model.layers.8.mlp.gate_proj.bias', 'model.layers.8.mlp.up_proj.bias', 'model.layers.8.self_attn.k_proj.bias', 'model.layers.8.self_attn.o_proj.bias', 'model.layers.8.self_attn.q_proj.bias', 'model.layers.8.self_attn.v_proj.bias', 'model.layers.9.mlp.down_proj.bias', 'model.layers.9.mlp.gate_proj.bias', 'model.layers.9.mlp.up_proj.bias', 'model.layers.9.self_attn.k_proj.bias', 'model.layers.9.self_attn.o_proj.bias', 'model.layers.9.self_attn.q_proj.bias', 'model.layers.9.self_attn.v_proj.bias']\n",
      "- This IS expected if you are initializing LlamaForCausalLM from the checkpoint of a model trained on another task or with another architecture (e.g. initializing a BertForSequenceClassification model from a BertForPreTraining model).\n",
      "- This IS NOT expected if you are initializing LlamaForCausalLM from the checkpoint of a model that you expect to be exactly identical (initializing a BertForSequenceClassification model from a BertForSequenceClassification model).\n"
     ]
    }
   ],
   "source": [
    "import dspy\n",
    "import outlines\n",
    "model_id = \"hugging-quants/Meta-Llama-3.1-8B-Instruct-GPTQ-INT4\"\n",
    "dspy_model = dspy.HFModel(model = model_id, hf_device_map = \"cuda:0\")\n",
    "\n",
    "hf_model = dspy_model.model\n",
    "hf_tokenizer = dspy_model.tokenizer\n",
    "\n",
    "# set some dspy model options\n",
    "#dspy_model.kwargs[\"max_tokens\"]=args.max_tokens\n",
    "dspy_model.drop_prompt_from_output = True\n",
    "\n",
    "# define outlines llm and sampler\n",
    "outlines_llm = outlines.models.Transformers(model=hf_model, tokenizer=hf_tokenizer)\n",
    "outlines_sampler = outlines.samplers.GreedySampler()\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Compiling FSM index for all state transitions: 100%|██████████| 3/3 [00:00<00:00,  5.15it/s]\n"
     ]
    }
   ],
   "source": [
    "import form_filling\n",
    "\n",
    "form_filler = form_filling.SequentialFormFiller(\n",
    "    outlines_llm=outlines_llm,\n",
    "    outlines_sampler=outlines_sampler,\n",
    "    pydantic_form=Disease,\n",
    ")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "We detected that you are passing `past_key_values` as a tuple and this is deprecated and will be removed in v4.43. Please use an appropriate `Cache` class (https://huggingface.co/docs/transformers/v4.41.3/en/internal/generation_utils#transformers.Cache)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Disease_name='Malignant melanoma' Body_part='Skin'\n"
     ]
    }
   ],
   "source": [
    "filled_form = form_filler.forward(description_based_retriever)\n",
    "\n",
    "print(filled_form)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Evaluation\n",
    "\n",
    "Disease_name is str (not literal) so its evaluated with similarity.\n",
    "Body_part is Literal, so 1 if its correct else 0."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "{'Disease_name': 0.8594872219577656, 'Body_part': 1.0}\n"
     ]
    }
   ],
   "source": [
    "labels = {\"Disease_name\":[\"melanoma\"], \"Body_part\":[\"Skin\"]}\n",
    "\n",
    "import evaluation\n",
    "scores = evaluation.score_general_prediction(labels, filled_form)\n",
    "print(scores)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "To compare, we make up some other labels;"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 25,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "{'Disease_name': 0.6777031254235824, 'Body_part': 0.0}\n"
     ]
    }
   ],
   "source": [
    "labels = {\"Disease_name\":[\"Cancer\"], \"Body_part\":[\"Leg\"]}\n",
    "\n",
    "import evaluation\n",
    "scores = evaluation.score_general_prediction(labels, filled_form)\n",
    "print(scores)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## dataset loader"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {},
   "outputs": [],
   "source": [
    "import dataset_loader\n",
    "documents, labels = dataset_loader.load_arxpr_data(max_amount=8, version=\"2_25\", mode=\"train\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'26925227': 'BioC-API\\ncollection.key\\nCC BY\\n[version 2; referees: 2 approved]\\nRNA-seq quantification gene expression transcriptomics\\nThis is an open access article distributed under the terms of the Creative Commons Attribution Licence, which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.\\nsurname:Soneson;given-names:Charlotte\\nsurname:Love;given-names:Michael I.\\nsurname:Robinson;given-names:Mark D.\\nIn version 2 of the manuscript, we have reworded and improved the clarity of the text in several places, to better convey the differences between the contrasted approaches and clarify the questions that are addressed by each analysis. We have also expanded the supplementary material with additional analyses of accuracy of abundance estimates among paralogous genes, and added more detailed descriptions of aspects such as the calculation of each abundance measure and the generation of the incomplete annotation files, information that was previously only available in the respective Datasets. The Discussion section has been extended, mainly to include a discussion of differential expression analysis methods incorporating variance estimates.\\nTITLE\\nAmendments from Version 1\\nfront\\nDifferential analyses for RNA-seq: transcript-level estimates improve gene-level inferences\\nABSTRACT\\nabstract\\nHigh-throughput sequencing of cDNA (RNA-seq) is used extensively to characterize the transcriptome of cells. Many transcriptomic studies aim at comparing either abundance levels or the transcriptome composition between given conditions, and as a first step, the sequencing reads must be used as the basis for abundance quantification of transcriptomic features of interest, such as genes or transcripts. Various quantification approaches have been proposed, ranging from simple counting of reads that overlap given genomic regions to more complex estimation of underlying transcript abundances. In this paper, we show that gene-level abundance estimates and statistical inference offer advantages over transcript-level analyses, in terms of performance and interpretability. We also illustrate that the presence of differential isoform usage can lead to inflated false discovery rates in differential gene expression analyses on simple count matrices but that this can be addressed by incorporating offsets derived from transcript-level abundance estimates. We also show that the problem is relatively minor in several real data sets. Finally, we provide an R package ( tximport) to help users integrate transcript-level abundance estimates from common quantification pipelines into count-based statistical inference engines.\\nINTRO\\ntitle_1\\nIntroduction\\nINTRO\\nparagraph\\nQuantification and comparison of isoform- or gene-level expression based on high throughput sequencing reads from cDNA (RNA-seq) are arguably among the most common tasks in modern computational molecular biology. Currently, one of the most widely used approaches amounts to defining the genomic locations of a set of non-overlapping targets (typically, genes) and using the number of aligned reads overlapping a target as a measure of its abundance, or expression level. Several software packages have been developed for performing such \"simple\" counting (e.g., featureCounts  and HTSeq-count ). More recently, the field has seen a surge in methods aimed at quantifying the abundances of individual transcripts (e.g., Cufflinks , RSEM , BitSeq , kallisto  and Salmon ). These methods provide higher resolution than simple counting, and by circumventing the computationally costly read alignment step, some (notably, kallisto and Salmon) are also considerably faster. However, isoform quantification is more complex than the simple counting, due to the high degree of overlap among transcripts. Currently, there is no consensus regarding the optimal resolution or method for quantification and downstream analysis of transcriptomic output.\\nINTRO\\nparagraph\\nAnother point of debate is the unit in which abundances are given. The traditional R/FPKM ,  (reads/fragments per kilobase per million reads) have been largely superseded by the TPM  (transcripts per million), since the latter is more consistent across libraries. Regardless, all these units attempt to \"correct for\" sequencing depth and feature length and thus do not reflect the influence of these on quantification uncertainty. In order to account for these aspects, most statistical tools for analysis of RNA-seq data operate instead on the count scale. Most of these tools were designed to be applied to simple read counts, and the degree to which their performance is affected by using fractional estimated counts resulting from portioning reads aligning to multiple transcripts is still an open question. The fact that the most common sequencing protocols provide reads that are much shorter than the average transcript implies that the observed read counts depend on a transcript\\'s length as well as its abundance; thus, simple counts are arguably less accurate measures than TPMs of the true abundance of RNA molecules from given genes. The use of gene counts as input to statistical tools typically assumes that the length of the expressed part of a gene does not change across samples and thus its impact can be ignored for differential analysis.\\nINTRO\\nparagraph\\nIn the analysis of transcriptomic data, as for any other application, it is of utmost importance that the question of interest is precisely defined before a computational approach is selected. Often, the interest lies in comparing the transcriptional output between different conditions, and most RNA-seq studies can be classified as either: 1) differential gene expression (DGE) studies, where the overall transcriptional output of each gene is compared between conditions; 2) differential transcript/exon usage (DTU/DEU) studies, where the composition of a gene\\'s isoform abundance spectrum is compared between conditions, or 3) differential transcript expression (DTE) studies, where the interest lies in whether individual transcripts show differential expression between conditions. DTE analysis results can be represented on the individual transcript level, or aggregated to the gene level, e.g., by evaluating whether at least one of the isoforms shows evidence of differential abundance.\\nINTRO\\nparagraph\\nIn this report, we make and give evidence for three claims: 1) gene-level estimation is considerably more accurate than transcript-level; 2) regardless of the level at which abundance estimation is done, inferences at the gene level are appealing in terms of robustness, statistical performance and interpretation; 3) taking advantage of transcript-level abundance estimates when defining or analyzing gene-level abundances leads to improved DGE results compared to simple counting for genes exhibiting DTU. The magnitude of the effect in a given data set thus depends on the extent of DTU, and the global impact is relatively small in several real data sets analyzed in this study.\\nINTRO\\nparagraph\\nTo facilitate a broad range of analysis choices, depending on the biological question of interest, we provide an R/Bioconductor package, tximport, to import transcript lengths and abundance estimates from several popular quantification packages and export (estimated) count matrices and, optionally, average transcript length correction terms (i.e., offsets) that can be used as inputs to common statistical engines, such as DESeq2 , edgeR  and limma .\\nMETHODS\\ntitle_1\\nData and methods\\nMETHODS\\nparagraph\\nThroughout this manuscript, we utilize two simulated data sets and four experimental data sets (Bottomly  [ Data set 3], GSE64570  [ Data set 4], GSE69244  [ Data set 5], GSE72165  [ Data set 6], see Supplementary File 1 for further details) for illustration. Details on the data generation and full records of the analyses are provided in the data sets and Supplementary File 1. The first simulated data set (sim1; Data set 1) is the synthetic human data set from Soneson et al. , comprising 20,410 genes and 145,342 transcripts and is available from ArrayExpress (accession E-MTAB-3766). This data set consists of three biological replicates from each of two simulated conditions, and differential isoform usage was introduced for 1,000 genes by swapping the relative expression levels of the two most dominant isoforms between conditions. For each gene in this data set, the total transcriptional output is the same in the two conditions (i.e., no overall DGE); it is worth noting that this is an extreme situation, but provides a useful test set for contrasting DGE, DTU and DTE. The second simulated data set (sim2; Data set 2) is a synthetic data set comprising the 3,858 genes and 15,677 transcripts from the human chromosome 1. It is available from ArrayExpress with accession E-MTAB-4119. Also here, we simulated two conditions with three biological replicates each. For this data set, we simulated both overall DGE, where all transcripts of the affected gene showed the same fold change between the conditions (420 genes), differential transcript usage (DTU), where the total transcriptional output was kept constant but the relative contribution from the transcripts changed (420 genes) and differential transcript expression (DTE), where the expression of 10% of the transcripts of each affected gene was modified (422 genes, 528 transcripts). The three sets of modified genes were disjoint. Again, this synthetic data set represents an extreme situation compared to most real data sets, but provides a useful test case to identify underlying causes of differences between results from various analysis pipelines.\\nMETHODS\\nparagraph\\nIn addition to Data set 1- Data set 6, which contain all code for reproducing our analyses, further method descriptions are given in Supplementary File 1.\\nMETHODS\\ntitle_1\\nGene abundance estimates are more accurate than transcript abundance estimates\\nMETHODS\\nparagraph\\nTo evaluate the accuracy of abundance estimation with transcript and gene resolution, we used the quasi-mapping mode of Salmon  (v0.5.1) to estimate the TPM for each transcript in each of the data sets. Gene-level TPM estimates, representing the overall transcriptional output of each gene, were obtained by summing the corresponding transcript-level TPM estimates. For the two simulated data sets, the true underlying TPM of each feature was known and we could thus evaluate the accuracy of the estimates. Unsurprisingly, gene-level estimates were more accurate than transcript-level estimates ( Figure 1A, Supplementary Figures 4,5). We also derived TPM estimates from simple gene-level counts obtained from traditional alignment of the reads to the genome using STAR followed by counting with featureCounts, by dividing the read count for each gene with a reasonable measure of the length of the gene (the length of the union of its exons) and the total number of mapped reads, and scaling the estimates to sum to 1 million. The simple count estimates showed a lower correlation with the true TPMs than the Salmon estimates, in line with previous observations . It is worth noting that we are comparing entire (typical) workflows, and that differences may also occur if the set of reads that STAR is able to align to the genome is not identical to the set of reads that are contributing to the abundance estimation of Salmon. However, due to the large fraction of aligned reads and the high mapping rate with Salmon (both exceeding 99.8%, more than 95% of the reads were subsequently unambiguously assigned to genes by featureCounts), we do not expect this to have a major impact on the results shown in Figure 1A.\\nf1\\nFIG\\nfig_caption\\n\\nA: Accuracy of gene- and transcript-level TPM estimates from\\nSalmon and scaled FPKM estimates derived from simple counts from\\nfeatureCounts, in one of the simulated samples (sampleA1). Spearman correlations are indicated in the respective panels. Top row: using the complete annotation. Bottom row: using an incomplete annotation, with 20% of the transcripts randomly removed. Gene-level estimates are more accurate than transcript-level estimates. Gene-level estimates from\\nSalmon are more accurate than those from\\nfeatureCounts.\\nB: Distribution of the coefficients of variation of gene- and transcript-level abundance estimates from\\nSalmon, calculated across 30 bootstrap samples of one of the simulated samples (sampleA1). Gene-level estimates are less variable than transcript-level estimates.\\nC: An example of unidentifiable transcript-level estimates, as uneven coverage does not cover the critical regions that would determine the amount that each transcript is expressed, while gene-level estimation is still possible.\\nMETHODS\\nparagraph\\nGene-level estimates derived from both simple counts and Salmon tended to show a high degree of robustness against incompleteness of the annotation catalog, as evidenced from estimation errors after first removing (at random) 20% of the transcripts ( Figure 1A, see also Supplementary File 1); in contrast, Salmon\\'s transcript estimate accuracies deteriorated. To further compare the merits of genome alignment-based vs alignment-free quantification, especially in their handling of multi-mapping reads, we investigated the accuracy of the abundance estimates within sets of paralogous genes ( Supplementary Figures 1-3). Also here, Salmon provided more consistently accurate estimates than STAR+ featureCounts. From the bootstrap estimates generated by Salmon, we also estimated the coefficient of variation of the abundance estimates. The gene-level estimates showed considerably lower variability than the transcript-level estimates in both simulated and experimental data ( Figure 1B, Supplementary Figures 6,7). Taken together, these observations suggest that the gene-level estimates are more accurate than transcript-level estimates and therefore potentially allow a more accurate and stable statistical analysis. A further argument in favor of gene-level analysis is the unidentifiability of transcript expression that can result from uneven coverage caused by underlying technical biases. While some extent of coverage variability might be alleviated by corrections for sequence- or position-specific biases , there remain cases where transcript expression cannot be inferred from data ( Figure 1C). Intermediate approaches, grouping together \"indistinguishable\" features are also conceiveable , but not yet standard practice.\\nMETHODS\\ntitle_1\\nDTE is more powerful and easier to interpret on gene level than for individual transcripts\\nMETHODS\\nparagraph\\nDTE is concerned with inference of changes in abundance at transcript resolution, and thus invokes a statistical test for each transcript. We argue that this can lead to several complications: the first is conceptual, since the rows (transcripts) in the result table will in many cases not be interpreted independently, since the researcher is often interested in comparing the results for transcripts from the same gene locus, and the second one is more technical, since the number of transcripts is considerably larger than the number of genes, which could lead to lower power due to the portioning of the total set of reads across a larger number of features and a potentially higher multiple testing penalty. We tested for DTE on the simulated data by applying edgeR  to the transcript counts obtained from Salmon (the application of count models to estimated counts is discussed in the next Section), and represented the results as transcript-level p-values or aggregated these to the gene level by using the perGeneQValue function from the DEXSeq  R package. Note that the transcript-level DTE test assesses the null hypothesis that individual transcripts do not change their expression, whereas the gene-level DTE test assesses the null hypothesis that all transcripts from a given gene exhibit no change in expression. Framing the DTE question at the gene level results in higher power, without sacrificing false discovery rate control ( Figure 2A). This is not surprising given the different null hypotheses, and, in fact, for many of the genes detected as true positives with the gene-level test, only a subset of the truly changing transcripts were detected ( Supplementary Figure 8). We note that this type of gene-level aggregation may favor genes in which one transcript shows strong changes, and that other approaches to increase power against specific alternatives are conceivable, e.g., capitalizing on the rich collection of methods for gene set analysis.\\nf2\\nFIG\\nfig_caption\\n\\nA: DTE detection performance on transcript- and gene-level, using\\nedgeR applied to transcript-level estimated counts from\\nSalmon. The statistical analysis was performed on transcript level and aggregated for each gene using the\\nperGeneQValue function from the\\nDEXSeq R package; aggregated results show higher detection power. The curves trace out the observed FDR and TPR for each significance cutoff value. The three circles mark the performance at adjusted p-value cutoffs of 0.01, 0.05 and 0.1.\\nB: Schematic illustration of different ways in which differential transcript expression (DTE) can arise, in terms of absence or presence of differential gene expression (DGE) and differential transcript usage (DTU).\\nMETHODS\\nparagraph\\nWhile DTE analysis is more suitable than DGE analysis for detecting genes with changes in absolute or relative isoform expression but no or only minor change in overall output ( Supplementary Figure 9), we argue that even gene-level DTE results may suffer from lack of interpretability. DTE can manifest in several different ways, as an overall differential expression of the gene or differential relative usage of its transcripts, or a combination of the two ( Figure 2B). We argue that the biological question of interest is in many cases more readily interpretable as a combination of DGE and DTU, rather than DTE. It has been our experience that results reported at the transcript level are still often cast to the gene level (i.e., given a differentially expressed transcript, researchers want to know whether also other isoforms of the gene are changing), suggesting that asking two specific gene-level questions (Is the overall abundance changing? Are the isoform abundances changing proportionally?) trumps the interpretability of one broad question addressing the transcript abundances (Are there changes in any of the isoform expression levels?), despite the increased need for multiple testing correction associated with performing two tests for each gene rather than one. There are of course also situations when a transcript-centric approach provides superior interpretability, for example in targeted experiments where specific isoforms are expected to change due to an administered treatment.\\nRESULTS\\ntitle_1\\nIncorporating transcript-level estimates leads to more accurate DGE results\\nRESULTS\\nparagraph\\nDGE (i.e., testing for changes in the overall transcriptional output of a gene) is typically performed by applying a count-based inference method from statistical packages such as edgeR  or DESeq2  to gene counts obtained by read counting software such as featureCounts , HTSeq-count  or functions from the GenomicAlignments  R package. A lot has been written about how simple counting approaches are prone to give erroneous results for genes with changes in relative isoform usage, due to the direct dependence of the observed read count on the transcript length , and alternatives, such as Cuffdiff , which utilizes estimated transcript abundances, have been proposed. However, the extent of the problem in real data has not been thoroughly investigated. Here, we show that taking advantage of transcript-resolution estimates (e.g., obtained by Salmon) in count-based inference methods can lead to improved DGE results. We propose two alternative ways of integrating transcript abundance estimates into the DGE pipeline: to define an \"artificial\" count matrix, or to calculate offsets that can be used in the statistical modeling of the observed gene counts from, e.g., featureCounts. Both approaches are implemented in the accompanying tximport R package (available from http://bioconductor.org/packages/tximport).\\nRESULTS\\nparagraph\\nFor the DGE analyses, we defined three different gene-level count matrices for each data set (see also Supplementary File 1): 1) using featureCounts from the Rsubread  R package (denoted featureCounts below), 2) summing the estimated transcript counts from Salmon within genes ( simplesum), 3) summing the estimated transcript TPMs from Salmon within genes, and multiplying with the total library size in millions ( scaledTPM). We note that the scaledTPM values are artificial values, transforming underlying abundance measures to the \"count scale\" to incorporate the information provided by the sequencing depth. We further used the effective transcript lengths and estimated TPMs from Salmon to define average transcript lengths for each gene and each sample (normalization factors) as described in the Supplementary material, to be used as offsets for edgeR and DESeq2 when analyzing the featureCounts and simplesum count matrices ( featureCounts_avetxl and simplesum_avetxl).\\nRESULTS\\nparagraph\\nOverall, the counts obtained by all methods were highly correlated ( Supplementary Figures 10-12), which is not surprising since any differences are likely to affect a relatively small subset of the genes. In general, the simplesum and featureCounts matrices led to similar conclusions in all considered data sets, even though there are differences between the two approaches in terms of how multi-mapping reads and reads partly overlapping intronic regions are handled . Previous studies have also shown that some loss of sensitivity for certain genes may be encountered from discarding multi-mapping fragments, which may be recovered through the use of transcript abundance estimators such as Salmon . The concordance between simplesum and featureCounts results also suggests that statistical methods based on the Negative Binomial assumption are applicable also to summarized, gene-level estimated counts, which is further supported by the similarity between the p-value histograms as well as the mean-variance relationships observed with the three types of count matrices ( Supplementary Figures 13-18).\\nRESULTS\\nparagraph\\nAccounting for the potentially varying average transcript length across samples when performing DGE, either in the definition of the count matrix (scaledTPM) or by defining offsets (featureCounts_avetxl, simplesum_avetxl), led to considerably improved false discovery rate (FDR) control compared to using the observed featureCounts or aggregated Salmon counts directly ( Figure 3A, Table 1). It is important to note that this improvement is entirely attributable to an improved handling of genes with changes in isoform composition between the conditions ( Figure 3B, Supplementary Figure 19), that we purposely introduced strong signals in the simulated data set in order to pinpoint these underlying causes, and that the overall effect in a real data set will depend on the extent to which considerable DTU is present. Experiments on various real data sets ( Supplementary Figure 20) show only small differences in the collections of significant genes found with the simplesum and simplesum_avetxl approaches, suggesting that the extent of the problem in many real data sets is limited, and that most findings obtained with simple counting are not induced by counting artifacts. Further support for this conclusion is shown in Figure 4 (see also Supplementary Figures 21-23 and Supplementary Table 1), where log-fold change estimates from edgeR, based on the simplesum and scaledTPM matrices, are contrasted. For the genes with induced DTU in the sim2 data set, log-fold changes based on the simplesum matrix are overestimated, as expected. However, this effect is almost absent in all the real data sets, again highlighting the extreme nature of our simulated data and suggesting that the effect of using different count matrices is considerably smaller for many real data sets. Table 1 further suggests that the lack of error control for simplesum and featureCounts matrices is more pronounced when there is a large difference in length between the differentially used isoforms. In the group with smallest length difference, where the longer differentially used isoform is less than 34% longer than the shorter one, all approaches controlled the type I error satisfactorily. It is worth noting that among all human transcript pairs in which both transcripts belong to the same gene, the median length ratio is 1.85, and for one third of such pairs the longer isoform is less than 38% longer than the shorter one (see Data set 1).\\nf3\\nFIG\\nfig_caption\\n\\nA: DGE detection performance of\\nedgeR applied to three different count matrices (simplesum, scaledTPM, featureCounts), with or without including an offset representing the average transcript length (for simplesum and featureCounts, avetxl indicates that such offsets were used). Including the offset or using the scaledTPM count matrix leads to improved FDR control compared to using simplesum or featureCounts matrices without offset. The curves trace out the observed FDR and TPR for each significance cutoff value. The three circles mark the performance at adjusted p-value cutoffs of 0.01, 0.05 and 0.1.\\nB: stratification of the results in\\nA by the presence of differential isoform usage. The improvement in FDR control seen in\\nA results from an improved treatment of genes with differential isoform usage, while all methods perform similarly for genes without differential isoform usage.\\nT1.xml\\nT1\\nTABLE\\ntable_title_caption\\nObserved false positive rates from a differential gene expression analysis using\\nedgeR applied to various count matrices (with a nominal p-value cutoff at 0.05), limited to genes with true underlying differential isoform usage (recall that no genes are truly differentially expressed in this data set).\\nT1.xml\\nT1\\nTABLE\\ntable_caption\\nThe results are stratified by \"effect size\" (the difference in relative abundance between the two differentially used isoforms) and the length ratio between the longer and the shorter of the differentially used isoforms. FPRs below the nominal p-value threshold (0.05) are marked in bold. For more details, see\\nData set 1.\\nT1.xml\\nT1\\nTABLE\\ntable\\n<?xml version=\"1.0\" encoding=\"UTF-8\"?>\\n<table frame=\"hsides\" rules=\"groups\" content-type=\"article-table\"><thead><tr><td rowspan=\"1\" colspan=\"1\"/><th align=\"center\" rowspan=\"1\" colspan=\"1\">simplesum</th><th align=\"center\" rowspan=\"1\" colspan=\"1\">featureCounts</th><th align=\"center\" rowspan=\"1\" colspan=\"1\">simplesum_avetxl</th><th align=\"center\" rowspan=\"1\" colspan=\"1\">featureCounts_avetxl</th><th align=\"center\" rowspan=\"1\" colspan=\"1\">scaledTPM</th></tr></thead><tbody><tr><td rowspan=\"1\" colspan=\"1\">[0,0.33), [1,1.34)</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">\\n<bold>0.019</bold>\\n</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">\\n<bold>0.019</bold>\\n</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">\\n<bold>0.023</bold>\\n</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">\\n<bold>0.023</bold>\\n</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">\\n<bold>0.023</bold>\\n</td></tr><tr><td rowspan=\"1\" colspan=\"1\">[0.33,0.67), [1,1.34)</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">0.059</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">0.059</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">0.059</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">0.059</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">0.059</td></tr><tr><td rowspan=\"1\" colspan=\"1\">[0.67,1), [1,1.34)</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">\\n<bold>0.000</bold>\\n</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">0.053</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">0.053</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">0.053</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">0.053</td></tr><tr><td rowspan=\"1\" colspan=\"1\">[0,0.33), [1.34,2.57)</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">0.075</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">0.070</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">0.070</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">0.065</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">0.065</td></tr><tr><td rowspan=\"1\" colspan=\"1\">[0.33,0.67), [1.34,2.57)</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">0.240</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">0.220</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">\\n<bold>0.050</bold>\\n</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">\\n<bold>0.033</bold>\\n</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">0.066</td></tr><tr><td rowspan=\"1\" colspan=\"1\">[0.67,1), [1.34,2.57)</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">0.420</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">0.540</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">\\n<bold>0.038</bold>\\n</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">0.077</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">\\n<bold>0.038</bold>\\n</td></tr><tr><td rowspan=\"1\" colspan=\"1\">[0,0.33), [2.57,35.4]</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">0.150</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">0.140</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">\\n<bold>0.037</bold>\\n</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">\\n<bold>0.043</bold>\\n</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">\\n<bold>0.037</bold>\\n</td></tr><tr><td rowspan=\"1\" colspan=\"1\">[0.33,0.67), [2.57,35.4]</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">0.650</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">0.650</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">0.060</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">0.060</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">\\n<bold>0.034</bold>\\n</td></tr><tr><td rowspan=\"1\" colspan=\"1\">[0.67,1), [2.57,35.4]</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">0.970</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">0.970</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">\\n<bold>0.034</bold>\\n</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">\\n<bold>0.034</bold>\\n</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">\\n<bold>0.034</bold>\\n</td></tr></tbody></table>\\n\\nf4\\nFIG\\nfig_title_caption\\nComparison of log-fold change estimates from\\nedgeR, based on\\nsimplesum and\\nscaledTPM count matrices, in four different data sets.\\nf4\\nFIG\\nfig_caption\\nFor the simulated data set (\\nsim2), where signals have been exaggerated to pinpoint underlying causes of various observations, genes with induced DTU (whose true overall log-fold change is 0) show a clear overestimation of log-fold changes when using\\nsimplesum counts. However, none of the real data sets contain a similar population of genes, suggesting that for many real data sets, simple gene counting leads to overall similar conclusions as accounting for underlying changes in transcript usage.\\nDISCUSS\\ntitle_1\\nDiscussion\\nDISCUSS\\nparagraph\\nIn this article, we have contrasted transcript- and gene-resolution analyses in terms of both abundance estimation and statistical inference, and illustrated that gene-level results are often more accurate, powerful and interpretable than transcript-level results. Not surprisingly, however, accurate transcript-level estimation and inference play an important role in deriving appropriate gene-level results, and it is therefore imperative to continue improving abundance estimation and inference methods applicable to individual transcripts, since misestimation can propagate to the gene level. We have shown that when testing for changes in overall gene expression (DGE), traditional gene counting approaches may lead to an inflated false discovery rate compared to methods aggregating transcript-level TPM values or incorporating correction factors derived from these, for genes where the relative isoform usage differs between the compared conditions. These correction factors can be calculated from the output of transcript abundance programs, using e.g., the provided R package ( tximport). It is important to note that the average transcript length offsets must account for the differences in transcript usage between the samples and thus using (sample-independent) exon-union gene lengths will not improve performance.\\nDISCUSS\\nparagraph\\nOn the six data sets studied here, simple counting with featureCounts led to very similar conclusions as estimated gene counts from Salmon, when combined with count-based statistical inference tools such as edgeR and DESeq2. Moreover, p-value distributions and mean-variance relationships were similar for actual and estimated counts. Taken together, this suggests that the negative binomial assumption made by the count-based tools is flexible enough to accommodate also estimated counts. All evaluated counting approaches, with and without the inclusion of average transcript length offsets, gave comparable DGE results for genes where DTU was not present. Thus, the extent of the FDR inflation in experimental data depends on the extent of DTU between the compared conditions; notably, our simulation introduced rather extreme levels of DTU, hence the inflated FDR, and the difference between the approaches was considerably smaller in real data sets. Recent studies have also shown that many genes express mainly one, dominant isoform  and for such genes, we expect that simple gene counting will work well.\\nDISCUSS\\nparagraph\\nAll evaluations in this study were performed using well-established count-based differential analysis tools. These methods take as input a matrix of counts, which is assumed to correctly represent the origin of each read in a particular set of libraries. However, due to sequence similarities among transcripts or genes, there is often a hidden uncertainty in the feature abundance estimates, even when the set of input reads is fixed. With the development of fast, alignment-free abundance estimation methods, this uncertainty can now be estimated rapidly using bootstrap approaches (see e.g. Figure 1b). Method development is currently underway in the field to account for this uncertainty in the differential expression analysis (e.g., MetaDiff , sleuth ), which has the potential to improve performance of both DTE and DGE analyses. If such methods are based on (potentially transformed) aggregated transcript counts as gene-level abundance measures, DGE analysis will still be affected by the presence of DTU, and thus could benefit from the inclusion of average transcript length offsets, or by instead using the sum of transcript TPMs as gene abundance measures.\\nDISCUSS\\nparagraph\\nOur results highlight the importance of carefully specifying the question of interest before selecting a statistical approach. Summarization of abundance estimates at the gene level before performing the statistical testing should be the method of choice if the interest is in finding changes in the overall transcriptional output of a gene. However, it is suboptimal if the goal is to identify genes for which at least one of the transcripts show differences in transcriptional output, since it may miss genes where two transcripts change in opposite directions, or where a lowly expressed transcript changes. For gene-level detection of DTE (that is, whether any transcript showed a change in expression between the conditions), statistical testing applied to aggregated gene counts led to reduced power and slightly inflated FDR compared to performing the statistical test on the transcript level and aggregating results within genes ( Supplementary Figure 9). Statistical inference on aggregated transcript TPMs (scaledTPM) showed low power for detecting changes that did not affect the overall transcriptional output of the gene, as expected. An alternative to DTE analysis, for potential improved interpretability, is to perform a combination of DGE and DTU analyses, both resulting in gene-level inferences. Table 2 summarizes our results and give suggested workflows for the different types of analyses we have considered.\\nT2.xml\\nT2\\nTABLE\\ntable_title_caption\\nSummary of suitable analysis approaches for the three types of comparative analyses discussed in the manuscript (DGE, DTE and DTU).\\nT2.xml\\nT2\\nTABLE\\ntable\\n<?xml version=\"1.0\" encoding=\"UTF-8\"?>\\n<table frame=\"hsides\" rules=\"groups\" content-type=\"article-table\"><thead><tr><th align=\"left\" rowspan=\"1\" colspan=\"1\">Task</th><th align=\"left\" rowspan=\"1\" colspan=\"1\">Input data</th><th align=\"left\" rowspan=\"1\" colspan=\"1\">Software (examples)</th><th align=\"left\" rowspan=\"1\" colspan=\"1\">Post-processing</th></tr></thead><tbody><tr><td rowspan=\"3\" valign=\"top\" colspan=\"1\">DGE</td><td rowspan=\"3\" valign=\"top\" colspan=\"1\">Aggregated transcript counts +\\n<break/>average transcript length offsets, or\\n<break/>simple counts + average transcript\\n<break/>length offsets</td><td rowspan=\"1\" colspan=\"1\">Salmon, kallisto, BitSeq, RSEM</td><td rowspan=\"3\" valign=\"top\" colspan=\"1\"/></tr><tr><td rowspan=\"1\" colspan=\"1\">tximport</td></tr><tr><td rowspan=\"1\" colspan=\"1\">DESeq2, edgeR, voom/limma</td></tr><tr><td rowspan=\"3\" valign=\"top\" colspan=\"1\">DTE</td><td rowspan=\"3\" valign=\"top\" colspan=\"1\">Transcript counts</td><td rowspan=\"1\" colspan=\"1\">Salmon, kallisto, BitSeq, RSEM</td><td rowspan=\"3\" valign=\"top\" colspan=\"1\">Optional gene-level\\n<break/>aggregation</td></tr><tr><td rowspan=\"1\" colspan=\"1\">tximport</td></tr><tr><td rowspan=\"1\" colspan=\"1\">DESeq2, edgeR, sleuth, voom/limma</td></tr><tr><td rowspan=\"2\" valign=\"top\" colspan=\"1\">DTU/DEU</td><td rowspan=\"2\" valign=\"top\" colspan=\"1\">Transcript counts or bin counts,\\n<break/>depending on interpretation potential\\n<sup><xref rid=\"ref-18\" ref-type=\"bibr\">18</xref></sup>\\n</td><td rowspan=\"1\" colspan=\"1\">Salmon, kallisto, BitSeq, RSEM</td><td rowspan=\"2\" valign=\"top\" colspan=\"1\">Optional gene-level\\n<break/>aggregation</td></tr><tr><td rowspan=\"1\" colspan=\"1\">DEXSeq</td></tr></tbody></table>\\n\\nTask\\tInput data\\tSoftware (examples)\\tPost-processing\\t \\tDGE\\tAggregated transcript counts +\\t \\taverage transcript length offsets, or\\t \\tsimple counts + average transcript\\t \\tlength offsets\\tSalmon, kallisto, BitSeq, RSEM\\t\\t \\ttximport\\t \\tDESeq2, edgeR, voom/limma\\t \\tDTE\\tTranscript counts\\tSalmon, kallisto, BitSeq, RSEM\\tOptional gene-level\\t \\taggregation\\t \\ttximport\\t \\tDESeq2, edgeR, sleuth, voom/limma\\t \\tDTU/DEU\\tTranscript counts or bin counts,\\t \\tdepending on interpretation potential\\t \\t\\tSalmon, kallisto, BitSeq, RSEM\\tOptional gene-level\\t \\taggregation\\t \\tDEXSeq\\t \\t\\nDISCUSS\\nparagraph\\nFinally, we note that abundance estimation at the gene level can reduce the impact of technical biases on expression levels, which have been shown to lead to estimation errors, such as expression being attributed to the wrong isoform . Non-uniform coverage from amplification bias or from position bias (3\\' coverage bias from poly-(A) selection) can result in unidentifiable transcript-level estimation. While correction of technical artifacts in coverage can be attempted computationally, through estimation of sequence- and position-specific biases , we note that such errors and estimation problems are also minimized when summarizing expression to the gene level. This being said, there may of course be situations where a direct transcript-level analysis is appropriate. For example, in a cancer setting where a specific deleterious splice variant is of interest (e.g., AR-V7 in prostate cancer ), inferences directly at the transcript level may be preferred. However, while this may be preferred for individual known transcripts, transcriptome-wide differential expression analyses may not be warranted, given the associated multiple testing cost.\\nSUPPL\\ntitle_1\\nData availability\\nSUPPL\\nparagraph\\nThe data referenced by this article are under copyright with the following copyright statement: Copyright:   2016 Soneson C et al.\\nSUPPL\\nparagraph\\nData associated with the article are available under the terms of the Creative Commons Zero \"No rights reserved\" data waiver (CC0 1.0 Public domain dedication).  \\nSUPPL\\nparagraph\\n F1000Research: Dataset 1. Data set 1, 10.5256/f1000research.7563.d114722 \\nSUPPL\\nparagraph\\n F1000Research: Dataset 2. Data set 2, 10.5256/f1000research.7563.d114723 \\nSUPPL\\nparagraph\\n F1000Research: Dataset 3. Data set 3, 10.5256/f1000research.7563.d114724 \\nSUPPL\\nparagraph\\n F1000Research: Dataset 4. Data set 4, 10.5256/f1000research.7563.d114725 \\nSUPPL\\nparagraph\\n F1000Research: Dataset 5. Data set 5, 10.5256/f1000research.7563.d114726 \\nSUPPL\\nparagraph\\n F1000Research: Dataset 6. Data set 6, 10.5256/f1000research.7563.d114730 \\nSUPPL\\ntitle_1\\nSoftware availability\\nSUPPL\\ntitle_2\\nSoftware access\\nSUPPL\\nparagraph\\n http://bioconductor.org/packages/tximport \\nSUPPL\\ntitle_2\\nSource code as at the time of publication\\nSUPPL\\nparagraph\\n https://github.com/F1000Research/tximport \\nSUPPL\\ntitle_2\\nArchived source code as at the time of publication\\nSUPPL\\nparagraph\\nSUPPL\\ntitle_2\\nSoftware license\\nSUPPL\\nparagraph\\n tximport is released under a GNU Public License (GPL).\\nSUPPL\\ntitle_1\\nSupplementary material\\nSUPPL\\nparagraph\\n Supplementary file 1. \\nSUPPL\\nparagraph\\nSupplementary File 1 (pdf) contains more detailed information about the data sets, supplementary methods and supplementary figures referred to in the text.\\nSUPPL\\nparagraph\\nsurname:Liao;given-names:Y\\nsurname:Smyth;given-names:GK\\nsurname:Shi;given-names:W\\n10.1093/bioinformatics/btt656\\nREF\\nBioinformatics.\\nref\\nfeatureCounts: an efficient general purpose program for assigning sequence reads to genomic features.\\nsurname:Anders;given-names:S\\nsurname:Pyl;given-names:PT\\nsurname:Huber;given-names:W\\n10.1093/bioinformatics/btu638\\nREF\\nBioinformatics.\\nref\\nHTSeq--a Python framework to work with high-throughput sequencing data.\\nsurname:Trapnell;given-names:C\\nsurname:Roberts;given-names:A\\nsurname:Goff;given-names:L\\nREF\\nNat Protoc.\\nref\\nsurname:Li;given-names:B\\nsurname:Dewey;given-names:CN\\nREF\\nBMC Bioinformatics.\\nref\\nsurname:Glaus;given-names:P\\nsurname:Honkela;given-names:A\\nsurname:Rattray;given-names:M\\n10.1093/bioinformatics/bts260\\nREF\\nBioinformatics.\\nref\\nIdentifying differentially expressed transcripts from RNA-seq data with biological variation.\\nsurname:Bray;given-names:N\\nsurname:Pimentel;given-names:H\\nsurname:Melsted;given-names:P\\nREF\\nref\\nNear-optimal RNA-Seq quantification.\\nsurname:Patro;given-names:R\\nsurname:Duggal;given-names:G\\nsurname:Kingsford;given-names:C\\nREF\\nbioRxiv.\\nref\\nAccurate, fast, and model-aware transcript expression quantification with Salmon.\\nsurname:Mortazavi;given-names:A\\nsurname:Williams;given-names:BA\\nsurname:McCue;given-names:K\\nREF\\nNat Methods.\\nref\\nMapping and quantifying mammalian transcriptomes by RNA-Seq.\\nsurname:Trapnell;given-names:C\\nsurname:Williams;given-names:BA\\nsurname:Pertea;given-names:G\\nREF\\nNat Biotechnol.\\nref\\nTranscript assembly and quantification by RNA-Seq reveals unannotated transcripts and isoform switching during cell differentiation.\\nsurname:Wagner;given-names:GP\\nsurname:Kin;given-names:K\\nsurname:Lynch;given-names:VJ\\nREF\\nTheory Biosci.\\nref\\nMeasurement of mRNA abundance using RNA-seq data: RPKM measure is inconsistent among samples.\\nsurname:Love;given-names:MI\\nsurname:Huber;given-names:W\\nsurname:Anders;given-names:S\\nREF\\nGenome Biol.\\nref\\nModerated estimation of fold change and dispersion for RNA-seq data with DESeq2.\\nsurname:Robinson;given-names:MD\\nsurname:McCarthy;given-names:DJ\\nsurname:Smyth;given-names:GK\\n10.1093/bioinformatics/btp616\\nREF\\nBioinformatics.\\nref\\nsurname:Ritchie;given-names:ME\\nsurname:Phipson;given-names:B\\nsurname:Wu;given-names:D\\nREF\\nNucleic Acids Res.\\nref\\n limma powers differential expression analyses for RNA-sequencing and microarray studies.\\nsurname:Bottomly;given-names:D\\nsurname:Walter;given-names:NA\\nsurname:Hunter;given-names:JE\\nREF\\nPLoS One.\\nref\\nEvaluating gene expression in C57BL/6J and DBA/2J mouse striatum using RNA-Seq and microarrays.\\nsurname:Yang;given-names:S\\nsurname:Marin-Juez;given-names:R\\nsurname:Meijer;given-names:AH\\nREF\\nBMC Genomics.\\nref\\nCommon and specific downstream signaling targets controlled by Tlr2 and Tlr5 innate immune signaling in zebrafish.\\nsurname:Currais;given-names:A\\nsurname:Goldberg;given-names:J\\nsurname:Farrokhi;given-names:C\\nREF\\nAging (Albany NY).\\nref\\nA comprehensive multiomics approach toward understanding the relationship between aging and dementia.\\nsurname:Chang;given-names:AJ\\nsurname:Ortega;given-names:FE\\nsurname:Riegler;given-names:J\\nREF\\nNature.\\nref\\nOxygen regulation of breathing through an olfactory receptor activated by lactate.\\nsurname:Soneson;given-names:C\\nsurname:Matthes;given-names:KL\\nsurname:Nowicka;given-names:M\\nREF\\nbioRxiv.\\nref\\nDifferential transcript usage from RNA-seq data: isoform pre-filtering improves performance of count-based methods.\\nsurname:Kanitz;given-names:A\\nsurname:Gypas;given-names:F\\nsurname:Gruber;given-names:AJ\\nREF\\nGenome Biol.\\nref\\nsurname:Roberts;given-names:A\\nsurname:Trapnell;given-names:C\\nsurname:Donaghey;given-names:J\\nREF\\nGenome Biol.\\nref\\nImproving RNA-Seq expression estimates by correcting for fragment bias.\\nsurname:Robert;given-names:C\\nsurname:Watson;given-names:M\\nREF\\nGenome Biol.\\nref\\nErrors in RNA-Seq quantification affect genes of relevance to human disease.\\nsurname:Anders;given-names:S\\nsurname:Reyes;given-names:A\\nsurname:Huber;given-names:W\\nREF\\nGenome Res.\\nref\\nDetecting differential usage of exons from RNA-seq data.\\nsurname:Lawrence;given-names:M\\nsurname:Huber;given-names:W\\nsurname:Pages;given-names:H\\nREF\\nPLoS Comput Biol.\\nref\\nSoftware for computing and annotating genomic ranges.\\nsurname:Trapnell;given-names:C\\nsurname:Hendrickson;given-names:DG\\nsurname:Sauvageau;given-names:M\\nREF\\nNat Biotechnol.\\nref\\nDifferential analysis of gene regulation at transcript resolution with RNA-seq.\\nsurname:Zhao;given-names:S\\nsurname:Xi;given-names:L\\nsurname:Zhang;given-names:B\\nREF\\nPLoS One.\\nref\\nsurname:Gonzalez-Porta;given-names:M\\nsurname:Frankish;given-names:A\\nsurname:Rung;given-names:J\\nREF\\nGenome Biol.\\nref\\nTranscriptome analysis of human tissues and cell lines reveals one dominant transcript per gene.\\nsurname:Jia;given-names:C\\nsurname:Guan;given-names:W\\nsurname:Yang;given-names:A\\nREF\\nBMC Bioinformatics.\\nref\\nMetaDiff: differential isoform expression analysis using random-effects meta-regression.\\nsurname:Love;given-names:MI\\nsurname:Hogenesch;given-names:JB\\nsurname:Irizarry;given-names:RA\\nREF\\nbioRxiv.\\nref\\nModeling of RNA-seq fragment sequence bias reduces systematic errors in transcript abundance estimation.\\nsurname:Antonarakis;given-names:ES\\nsurname:Lu;given-names:C\\nsurname:Wang;given-names:H\\nREF\\nN Engl J Med.\\nref\\nAR-V7 and resistance to enzalutamide and abiraterone in prostate cancer.\\nsurname:Soneson;given-names:C\\nsurname:Love;given-names:MI\\nsurname:Robinson;given-names:MD\\nREF\\nF1000Research.\\nref\\nData set 1 in: Differential analyses for RNA-seq: transcript-level estimates improve gene-level inferences.\\nsurname:Soneson;given-names:C\\nsurname:Love;given-names:MI\\nsurname:Robinson;given-names:MD\\nREF\\nF1000Research.\\nref\\nData set 2 in: Differential analyses for RNA-seq: transcript-level estimates improve gene-level inferences.\\nsurname:Soneson;given-names:C\\nsurname:Love;given-names:MI\\nsurname:Robinson;given-names:MD\\nREF\\nF1000Research.\\nref\\nData set 3 in: Differential analyses for RNA-seq: transcript-level estimates improve gene-level inferences.\\nsurname:Soneson;given-names:C\\nsurname:Love;given-names:MI\\nsurname:Robinson;given-names:MD\\nREF\\nF1000Research.\\nref\\nData set 4 in: Differential analyses for RNA-seq: transcript-level estimates improve gene-level inferences.\\nsurname:Soneson;given-names:C\\nsurname:Love;given-names:MI\\nsurname:Robinson;given-names:MD\\nREF\\nF1000Research.\\nref\\nData set 5 in: Differential analyses for RNA-seq: transcript-level estimates improve gene-level inferences.\\nsurname:Soneson;given-names:C\\nsurname:Love;given-names:MI\\nsurname:Robinson;given-names:MD\\nREF\\nF1000Research.\\nref\\nData set 6 in: Differential analyses for RNA-seq: transcript-level estimates improve gene-level inferences.\\nSUPPL\\nparagraph\\nSUPPL\\ntitle\\nReferee response for version 1\\nSUPPL\\nparagraph\\nPatro\\nSUPPL\\nparagraph\\nRob\\nSUPPL\\nparagraph\\nSUPPL\\nparagraph\\nIn this manuscript, the authors address a few questions of considerable (and perennial) interest in the analysis of RNA-seq data. Specifically, they provide evidence that, using available methods (e.g. DESeq2 / edgeR), assessing differential expression at the gene-level (DGE) is more robust than at the transcript level (DTE). Further, they convincingly argue that estimating abundance at the level of transcripts, and then aggregating these abundances to the gene level leads to improved estimation of differential gene expression. They demonstrate that one of the major factors in this improved estimation is the availability of a sample-specific feature length for each gene (derived from the abundance-weighted length of the expressed transcripts of this gene), which is not possible to obtain with any fixed gene model used by count-based methods. Finally, the authors argue that much of the analysis of interest at the transcript level does not actually require differential transcript expression testing, but rather can be inferred from a combination of DGE and differential transcript usage (DTU); this is an interesting proposition that merits further discussion and analysis. Overall, this is a well-written paper, with extensive and compelling supplementary and supporting data, that addresses a ubiquitous analysis task involving RNA-seq.  It should be of broad interest to the community and makes a valuable contribution. The accompanying software, tximport, is user-friendly and makes it easy to apply the type of analysis recommended herein; it too should be widely useful.\\nSUPPL\\nparagraph\\n Major comments: \\nSUPPL\\nparagraph\\nIt would be very useful to provide the equations used for calculating each of the abundance measured considered directly. Section 4 of the supplementary information is useful to this end, but the reader still has to search a bit to see exactly how each metric is computed (though the fantastic R-Markdown included with the figures means that these computations can be found explicitly).\\nSUPPL\\nparagraph\\nSimilarly, it would be useful to the reader to provide a description, in prose, of how specific experiments were performed (again, the reproducible nature of most of these experiments makes tracking down this information possible, but sometimes time-consuming). For example, how, precisely, was removal of transcripts handled at the level of the genome annotation? If a transcript consists only of constitutive exons, were all of those exons retained in the genome annotation used for STAR + featureCounts, while the transcript was removed in the Salmon index?  \\nSUPPL\\nparagraph\\nThe result that transcript-level abundance estimation is more sensitive to the removal of transcripts than gene-level abundance estimation : this seems intuitive. However, I agree with Dr. Floor\\'s suggestion that:\\nSUPPL\\nparagraph\\n \"The assertion that \"simple counts tended to show a high degree of robustness against incompleteness of the annotation catalog, as evidenced from estimation errors after first removing (at random) 20% of the transcripts\" seems misleading since Salmon-derived gene-level abundances actually show a higher Spearman correlation than count-derived gene-level abundances when subjected to removing a random 20% of transcripts.\" \\nSUPPL\\nparagraph\\nI would suggest rewording this sentence, as the main result seems to be that gene-level analysis is more robust to an incomplete annotation than transcript-level analysis. Transcript-level abundance estimation followed by gene-level analysis seems to perform just as well (actually, better) than gene-level counting in this scenario.\\nSUPPL\\nparagraph\\nThe experiments in the section \"Incorporating transcript-level estimates leads to more accurate DGE results\" suggests the (reasonable) interpretation that the main benefit of incorporating transcript-level abundance estimates when assessing DGE is a more accurate measure of the \"feature\" length of the gene. The authors state \" It is important to note that this improvement is entirely attributable to an improved handling of genes with changes in isoform composition between the conditions.\" This is supported by the fact that using the abundance-weighted average transcript length (i.e. offsets) with counting based approaches improves the results substantially.  However, one other place where transcript-level abundance estimates are useful in the context of DGE is when assessing the expression of paralogous genes. While most multi-mapping reads that derive from different isoforms of the same gene will be uniquely mappable at the level of the genome, and hence will be included in the counts for that gene, reads that map ambiguously among paralogs may not be. In such cases, count-based methods do not have a principled way of apportioning a read between the paralogs involved, and discarding multi-mapping reads may negatively affect estimation of the abundance of the paralogs, even at the gene level. While this case is likely much less common than mis-estimation of DGE as a result of DTU, it is certainly of biological interest. I would suggest adding an analysis, restricted to sets of paralogous genes, comparing how the different approaches perform in this case. This may help to highlight the importance of not only deriving appropriately weighted and sample-dependent lengths for genes, but also on resolving multi-mapping ambiguity that occurs between genomically distinct loci.\\nSUPPL\\nparagraph\\nThe argument that most transcript-level analyses of interest may be addressed by looking at DGE in conjunction with DTU is an interesting one. It is certainly that case that not all tasks for which DTE is used actually require assessing differential expression at the transcript level. One issue with the DGE + DTU-based analysis which warrants further discussion in the manuscript is that I believe that this approach, too, would require correcting for multiple hypothesis testing. Specifically, one is testing both the DGE and the DTU hypotheses for each gene (or for a relevant subset of interest). The correction here is likely to be less harsh than in the case of assessing DTE, but is still worth discussing.\\nSUPPL\\nparagraph\\nMinor comments:\\nSUPPL\\nparagraph\\nAs per Dr. Floor\\'s statement, Salmon (and Sailfish) also incorporate sequence-specific bias correction. Further, RSEM and Salmon (and a few other transcript-level abundance estimation tools) also incorporate the modeling of non-uniform fragment start position distributions. Of course, modeling a non-uniform start position distribution cannot overcome a complete lack of sampling in critical regions that might make determining transcript-level fragment assignment impossible, but it may help in properly apportioning an ambiguously-mapped fragment between transcripts depending on its relative position in each.\\nSUPPL\\nparagraph\\nOne potential added source of variability here is that all Salmon estimates presented in the manuscript make use of Salmon\\'s quasi-mapping of reads, while the STAR + featureCount pipeline makes use of \"traditional\" alignments. This is the primary intended usage mode of Salmon, and absolutely does represent a \"typical\" pipeline for methods that avoid alignment (Salmon, Sailfish, kallisto). However, it would probably be best to mention this as a potential (though likely negligible) additional source of variability.\\nSUPPL\\nparagraph\\nIn the discussion, the authors argue that \"... it is therefore imperative to continue improving abundance estimation and inference methods applicable to individual transcripts, since misestimation can propagate to the gene level.\" This is, of course, an important and valid suggestion. Another direction, on which it would be useful to get the authors\\' thoughts and suggestions, is the development of differential expression tools (at either the transcript or gene level) that can make use of the variance estimates that some tools (like Salmon) can provide. To what extent might incorporating this information help control false positive rates and improve DTE or even DGE estimates?\\nSUPPL\\nparagraph\\nI have read this submission. I believe that I have an appropriate level of expertise to confirm that it is of an acceptable scientific standard.\\nSUPPL\\nparagraph\\nRobinson\\nSUPPL\\nparagraph\\nMark\\nSUPPL\\nparagraph\\nSUPPL\\nparagraph\\nThank you for taking the time to read and review our paper. As per your suggestions, we have expanded the supplementary pdf document with equations defining each of the abundance measures that we used as well as a detailed description of the generation of the incomplete annotation files. We have also reworded the paragraph discussing the influence of an incomplete annotation catalog, and added further discussion points regarding multiple testing correction for DGE+DTU as well as regarding new methods incorporating variance estimates into the differential analysis. However, a deeper discussion of the latter point falls outside the scope of this article.\\nSUPPL\\nparagraph\\nWe found your suggestion to restrict the abundance accuracy comparison to paralogous genes interesting, and the supplement has been expanded with several examples comparing the accuracy of gene-level abundance estimates from Salmon and from STAR+featureCounts, restricted to sets of paralogous genes. While we see a clear advantage of Salmon for many of the paralogous gene groups, the overall difference is only slightly larger for sets of paralogous genes compared to random sets of genes (as can be seen in the revised version of Dataset 1). In addition, we have added the following text to the relevant section in the results:\\nSUPPL\\nparagraph\\n\"...suggesting that the extent of the problem in many real data sets is limited... though some loss of sensitivity for certain genes may be encountered from discarding multi-mapping fragments, which may be recovered through the use of transcript abundance estimators such as Salmon.\"\\nSUPPL\\nparagraph\\nWhile we acknowledge that many of the popular transcript abundance estimation methods incorporate some type of bias estimation/correction (albeit not at the fragment level), and that it is definitely an important (and difficult) research area, a thorough discussion on the relative merits of different bias correction approaches is outside the scope of this paper. We have added citations to relevant literature on coverage bias, have simplified and clarified the last paragraph of the discussion, and have reworded the sentence about unidentifiable estimation due to coverage biases:\\nSUPPL\\nparagraph\\n\"While some extent of coverage variability might be alleviated by corrections for sequence- or position-specific biases, there remain cases where transcript expression cannot be inferred from data ( Figure 1C)\"\\nSUPPL\\nparagraph\\nSUPPL\\ntitle\\nReferee response for version 1\\nSUPPL\\nparagraph\\nFloor\\nSUPPL\\nparagraph\\nStephen N.\\nSUPPL\\nparagraph\\nSUPPL\\nparagraph\\nSoneson, Love and Robinson tackle a crucial question for analysis of RNA deep sequencing data in this manuscript: what is the role of transcript diversity in the accuracy and statistical power associated with measurements of gene expression? The authors make and convincingly show three claims: gene-level estimation and inferences are more robust than those at the transcript-level, and incorporating transcript-level quantification into gene-level abundance leads to improved differential expression testing.  The claims are convincingly proven, the manuscript is well written, and the subject matter is of considerable interest. Furthermore, the described R package tximport should be of broad interest to the RNA deep sequencing community.\\nSUPPL\\nparagraph\\nOverall comments:\\nSUPPL\\nparagraph\\nIt may be useful to indicate explicitly in the text that the methods are contained within the (excellently written and formatted) supplementary material, as this was not apparent. It might be clearest to create a specific methods section that just references supplementary file 1.\\nSUPPL\\nparagraph\\nThe clarity of scatter plots with more than ~hundreds of points (e.g. Figure 1A) could be improved by using partially transparent points to visualize density.\\nSUPPL\\nparagraph\\nIntroduction:\\nSUPPL\\nparagraph\\nParagraph 1: Cufflinks, RSEM and Bitseq are grouped with kallisto and Salmon and it is then stated that some of these methods bypass read alignment. It would be clearer if this were reworded to avoid the ambiguity as to which methods avoid read alignment.\\nSUPPL\\nparagraph\\nParagraph 4: The third claim could be presented more clearly. While it is interesting that simple counting performs similarly to transcript-level quantification procedures, it seems more interesting to this reviewer that incorporating transcript-level information improves the accuracy of differential expression testing at the gene level. Perhaps these two concepts can be combined into one more concise point?\\nSUPPL\\nparagraph\\nResults:\\nSUPPL\\nparagraph\\nThe assertion that \"simple counts tended to show a high degree of robustness against incompleteness of the annotation catalog, as evidenced from estimation errors after first removing (at random) 20% of the transcripts\" seems misleading since Salmon-derived gene-level abundances actually show a higher Spearman correlation than count-derived gene-level abundances when subjected to removing a random 20% of transcripts. Figure 1a bottom left shows that transcript-level abundances are strongly affected by removal of 20% of transcripts, but that gene-level abundances are not strongly changed whether estimated using counts or Salmon. This statement should be reworded.\\nSUPPL\\nparagraph\\nTwo concerns are raised about DTE. It is certainly true that reads are spread across more features when performing DTE as opposed to DGE.However, it is not apparent why analysis of DTE involves grouping of transcripts together for interpretation. DTE implies analysis at the transcript level and therefore no grouping, while DGE could involve some level of grouping of transcripts or quantification at the gene level from the start. The clarity of this could be improved.\\nSUPPL\\nparagraph\\nIt is a very interesting idea to separately frame questions regarding DGE and DTU, which should be adopted widely, as the two are separable questions.\\nSUPPL\\nparagraph\\nThe authors state one possible workflow towards DGE analysis in the section \"Incorporating transcript-level estimates leads to more accurate DGE results.\" Alternative pipelines (e.g. cuffdiff) could be presented in brief.\\nSUPPL\\nparagraph\\nThe observation that simplesum and featureCounts results are highly correlated and therefore that statistical methods based on the Negative Binomial distribution can be used on estimated counts seems of greater importance than is emphasized in the text. This should be elaborated upon in the discussion, since this means that estimated counts from kallisto, express, salmon, etc can be used directly by statistical packages assuming a NB distribution (edgeR, DESeq2, etc). This point is frequently debated in discussions of how to rigorously analyze sequencing data. The conclusion here that NB applies to estimated counts is thus quite important.\\nSUPPL\\nparagraph\\nPlease explain the meaning of the name for each curve in the legend for Figure 3 (i.e. specify that \"avetxl\" means using the offset corresponding to average transcript length.\\nSUPPL\\nparagraph\\nDiscussion:\\nSUPPL\\nparagraph\\nThe assertion that \"gene-level results are more accurate, powerful and interpretable than transcript-level results\" seems an oversimplification given the result that incorporating transcript-level quantification leads to improved DGE detection performance (e.g. Fig 3).\\nSUPPL\\nparagraph\\nPlease cite at minimum Roberts et al., (2011) regarding sequence bias correction as this has been implemented in cufflinks, express and kallisto. Other relevant papers should also be included here, as attempts have been made to address both positional and sequence-specific bias in RNA sequencing data.\\nSUPPL\\nparagraph\\nSupplement:\\nSUPPL\\nparagraph\\nThe usability of the supplemental info could be improved by substituting rasterized for vectorized plots for those with ~hundreds of points.\\nSUPPL\\nparagraph\\nPlease explain the meaning of the name for each curve in the legend for Supplemental Figure 5.\\nSUPPL\\nparagraph\\nI have read this submission. I believe that I have an appropriate level of expertise to confirm that it is of an acceptable scientific standard.\\nREF\\ntitle\\nReferences\\nsurname:Roberts;given-names:A\\nsurname:Trapnell;given-names:C\\nsurname:Donaghey;given-names:J\\nsurname:Rinn;given-names:JL\\nsurname:Pachter;given-names:L\\nREF\\nGenome Biol\\nref\\nImproving RNA-Seq expression estimates by correcting for fragment bias.\\nREF\\nparagraph\\nRobinson\\nREF\\nparagraph\\nMark\\nREF\\nparagraph\\nREF\\nparagraph\\nThank you for taking the time to read and review our paper. In the revised version, we have improved the clarity and usability of the figures and the supplement by using partially transparent points and extending the figure captions. We have also modified the main text to refer more clearly to the method information provided in the supplementary material, and improved the clarity of the text in several places, according to your suggestions.\\nREF\\nparagraph\\nFor additional comments regarding references to bias correction studies, please see responses to Rob Patro.\\n',\n",
       " '25435910': 'BioC-API\\ncollection.key\\nCC BY\\nCellobiose Glycolysis Systems biology Transcription factor Metabolic engineering Biofuels\\nThis is an Open Access article distributed under the terms of the Creative Commons Attribution License (http://creativecommons.org/licenses/by/4.0), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly credited. The Creative Commons Public Domain Dedication waiver (http://creativecommons.org/publicdomain/zero/1.0/) applies to the data made available in this article, unless otherwise stated.\\nsurname:Lin;given-names:Yuping\\nsurname:Chomvong;given-names:Kulika\\nsurname:Acosta-Sampson;given-names:Ligia\\nsurname:Estrela;given-names:Raissa\\nsurname:Galazka;given-names:Jonathan M\\nsurname:Kim;given-names:Soo Rin\\nsurname:Jin;given-names:Yong-Su\\nsurname:Cate;given-names:Jamie HD\\nTITLE\\nKeywords\\nfront\\nLeveraging transcription factors to speed cellobiose fermentation by Saccharomyces cerevisiae\\nABSTRACT\\nabstract_title_1\\nBackground\\nABSTRACT\\nabstract\\nSaccharomyces cerevisiae, a key organism used for the manufacture of renewable fuels and chemicals, has been engineered to utilize non-native sugars derived from plant cell walls, such as cellobiose and xylose. However, the rates and efficiencies of these non-native sugar fermentations pale in comparison with those of glucose. Systems biology methods, used to understand biological networks, hold promise for rational microbial strain development in metabolic engineering. Here, we present a systematic strategy for optimizing non-native sugar fermentation by recombinant S. cerevisiae, using cellobiose as a model.\\nABSTRACT\\nabstract_title_1\\nResults\\nABSTRACT\\nabstract\\nDifferences in gene expression between cellobiose and glucose metabolism revealed by RNA deep sequencing indicated that cellobiose metabolism induces mitochondrial activation and reduces amino acid biosynthesis under fermentation conditions. Furthermore, glucose-sensing and signaling pathways and their target genes, including the cAMP-dependent protein kinase A pathway controlling the majority of glucose-induced changes, the Snf3-Rgt2-Rgt1 pathway regulating hexose transport, and the Snf1-Mig1 glucose repression pathway, were at most only partially activated under cellobiose conditions. To separate correlations from causative effects, the expression levels of 19 transcription factors perturbed under cellobiose conditions were modulated, and the three strongest promoters under cellobiose conditions were applied to fine-tune expression of the heterologous cellobiose-utilizing pathway. Of the changes in these 19 transcription factors, only overexpression of SUT1 or deletion of HAP4 consistently improved cellobiose fermentation. SUT1 overexpression and HAP4 deletion were not synergistic, suggesting that SUT1 and HAP4 may regulate overlapping genes important for improved cellobiose fermentation. Transcription factor modulation coupled with rational tuning of the cellobiose consumption pathway significantly improved cellobiose fermentation.\\nABSTRACT\\nabstract_title_1\\nConclusions\\nABSTRACT\\nabstract\\nWe used systems-level input to reveal the regulatory mechanisms underlying suboptimal metabolism of the non-glucose sugar cellobiose. By identifying key transcription factors that cause suboptimal cellobiose fermentation in engineered S. cerevisiae, and by fine-tuning the expression of a heterologous cellobiose consumption pathway, we were able to greatly improve cellobiose fermentation by engineered S. cerevisiae. Our results demonstrate a powerful strategy for applying systems biology methods to rapidly identify metabolic engineering targets and overcome bottlenecks in performance of engineered strains.\\nABSTRACT\\nabstract_title_1\\nElectronic supplementary material\\nABSTRACT\\nabstract\\nThe online version of this article (doi:10.1186/s13068-014-0126-6) contains supplementary material, which is available to authorized users.\\nINTRO\\ntitle_1\\nBackground\\nFig1\\nFIG\\nfig_caption\\n\\nSuboptimal cellobiose metabolism in engineered\\nSaccharomyces cerevisiae\\n. (A) Fermentation profiles of recombinant cellobiose-utilizing S. cerevisiae with plasmid pRS426-BT on cellobiose or glucose in anaerobic conditions with an initial OD600 of 1. Concentrations: cellobiose (blue circle), glucose (red circle), ethanol from cellobiose (blue triangle), and ethanol from glucose (red triangle). Data represent the mean and standard error of triplicate cultures grown on each source. The arrows indicate the times at which samples were taken for transcription profiling by RNA deep sequencing. (B) Model of the regulation of glucose metabolism and of glucose-sensing and signaling networks in the context of a cellobiose-utilizing pathway. The cellobiose-utilizing pathway was established in S. cerevisiae by introducing a cellodextrin transporter gene (cdt-1) and an intracellular beta-glucosidase gene (gh1-1) from Neurospora crassa. Gpr1 and Gpa2 define a glucose-sensing pathway that works in parallel with Ras2 to activate protein kinase A (PKA), which induces genome-wide regulation. Signals emanating from Snf3 and Rgt2 regulate hexose transporter genes by inactivating the Rgt1 co-repressors Mth1 and Std1. The glucose repression signal that inactivates Snf1 kinase is generated through glucose metabolism, consequently inducing the Mig1/Hxk2-mediated glucose repression pathway. In addition, Snf1 kinase directly mediates phosphorylation of transcription activators of glucose-repressed genes to relieve glucose repression.\\nINTRO\\nparagraph\\nSaccharomyces cerevisiae is a key organism used for the manufacture of renewable fuels and chemicals, but it is not capable of using mixed sugars derived from the plant cell wall. Efforts to broaden the substrate spectrum of S. cerevisiae beyond glucose include the use of an intracellular cellobiose-degrading pathway composed of a cellodextrin transporter and an intracellular beta-glucosidase. Using this approach, S. cerevisiae has been engineered to co-ferment cellobiose and xylose or cellobiose and galactose. However, the rates and efficiencies of cellobiose, xylose, and other non-glucose sugar fermentations pale in comparison with those of glucose, hampering the application of non-glucose fermentation on an industrial scale. For example, suboptimal cellobiose metabolism results in prolonged lag phases and slow rates, albeit with similar ethanol yields, compared with glucose metabolism (Figure 1A).\\nINTRO\\nparagraph\\nS. cerevisiae has evolved hierarchical gene regulatory networks (GRNs) that respond to glucose, and these allow glucose to be consumed rapidly and preferentially even when non-glucose sugars are present (Figure 1B). GRNs controlling the preferential use of glucose are also prevalent in bacteria and other eukaryotes.\\nINTRO\\nparagraph\\nIn S. cerevisiae, three glucose-sensing and signaling pathways have been identified. The first depends on the cAMP-dependent protein kinase A (PKA), which is activated by the G protein-coupled receptor (Gpr1/Gpa2) and Ras2, which in parallel control the majority of glucose-induced changes in gene expression via modulating transcription factors (TFs). In the second, two transmembrane sensors of extracellular glucose, Snf3 and Rgt2, converge on the Rgt1 repressor to regulate the expression of glucose transporters. The third main glucose repression pathway involves the Snf1 kinase complex, which inhibits the Mig1 repressor-containing complex, thereby repressing genes involved in respiration, gluconeogenesis, and the metabolism of alternative carbon sources. These three systems operate as interconnected GRNs. For example, Rgt1 function is mainly altered by the signals generated by Snf3 and Rgt2, but is also regulated by the PKA and Snf1-Mig1 glucose repression pathways.\\nINTRO\\nparagraph\\nThe TFs in the S. cerevisiae glucose-sensing pathways play a central role in layered regulatory networks through complex, combinatorial functions on promoters that are not fully understood, and that may not be conserved across S. cerevisiae species. By contrast, cellobiose is an unusual substrate for S. cerevisiae, and is therefore not recognized as a readily fermentable sugar like glucose. Efforts to optimize cellobiose fermentation in engineered S. cerevisiae through combinatorial transcriptional engineering, experimental evolution, or by exploring and evolving new cellodextrin transporters or an alternative cellobiose phosphorylase pathway have resulted in only limited improvements in cellobiose metabolism.\\nINTRO\\nparagraph\\nTo improve non-native sugar metabolism in S. cerevisiae, systems-level measurements could be exploited to provide important insights into unknown bottlenecks present in engineered strains exhibiting suboptimal performance. Here, we present a strategy for optimizing non-native sugar fermentation by recombinant S. cerevisiae using cellobiose as a model. First, the GRNs perturbed by cellobiose were identified by RNA deep sequencing of the transcriptomes of engineered S. cerevisiae growing on cellobiose or glucose. Second, to identify the underlying causes of suboptimal cellobiose consumption, TFs with significant changes in expression between cellobiose and glucose metabolism in the systems-level analysis were perturbed by deletion or overexpression to examine their effect on cellobiose fermentation. Third, promoters with differing strengths under cellobiose conditions were identified using the transcription profiling data, and were used to fine-tune the expression of the cellobiose-utilizing pathway. Issues of local and global optimizations for strain improvement are longstanding in metabolic engineering, but clear methodologies for these optimizations are not well developed. By harnessing systems-level experiments, we identified key TFs that cause suboptimal cellobiose fermentation, and fine-tuned the expression of the heterologous cellobiose consumption pathway to greatly improve cellobiose fermentation by engineered S. cerevisiae. Our results reveal regulatory mechanisms underlying suboptimal non-glucose sugar fermentation in yeast, and demonstrate a promising paradigm for systems biology-based rational microbial strain engineering.\\nRESULTS\\ntitle_1\\nResults\\nRESULTS\\ntitle_2\\nTranscriptional reprogramming in response to cellobiose\\nFig2\\nFIG\\nfig_caption\\n\\nImpact of cellobiose on central carbon metabolism, amino acid biosynthesis and thiamine biosynthesis of\\nSaccharomyces cerevisiae\\n. (A) Genes involved in mitochondrial function, including the tricarboxylic acid (TCA) cycle, electron transport chain, and oxidative phosphorylation are shown. (B) Genes involved in amino acid and thiamine biosynthesis. (C) Genes involved in sugar transport, glycolysis and fermentation, the pentose phosphate pathway, gluconeogenesis, storage of carbohydrates (trehalose and glycogen), and the glyoxylate cycle. (A-C) Only genes with significantly different expression when comparing cellobiose-grown versus glucose-grown cells are shown (color-coded boxes), including the fold change on cellobiose (C8) versus glucose (G8). Transcription levels that significantly increased or decreased on cellobiose in contrast to glucose (absolute fold changes >=2.0, P <= 0.001) are shown in green and red boxes, respectively.\\nRESULTS\\nparagraph\\nS. cerevisiae cannot metabolize cellobiose naturally. For this study, S. cerevisiae was engineered to achieve cellobiose utilization by introducing both a cellodextrin transporter gene (cdt-1) and an intracellular beta-glucosidase gene (gh1-1) from Neurospora crassa (See Methods). Although cellobiose is a dimer of glucose, metabolism of cellobiose by engineered S. cerevisiae shows substantially prolonged lag phases and slow rates compared with glucose metabolism (Figure 1A). To probe the transcriptional regulatory response of S. cerevisiae to cellobiose, we quantified mRNA abundance during exponential growth on either cellobiose or glucose under anaerobic conditions. Transcription profiling (see Additional file 1: Dataset S1) revealed that 519 (8.2%) of the 6,351 genes annotated in the S. cerevisiae genome had significantly different expression in cellobiose-grown cells compared with glucose-grown cells (absolute fold changes >= 2.0 or more; P <= 0.001) (see Additional file 2: Dataset S2). Based on Gene Ontology (GO) biological process enrichment analysis (see Additional file 3: Dataset S3), genes with significantly increased mRNA levels (fold changes >= 2.0, P <= 0.001) on cellobiose were enriched for mitochondria-associated processes (Figure 2A), such as ATP biosynthesis, mitochondrial electron and proton transport, and the tricarboxylic acid cycle. Genes with significantly decreased mRNA levels (fold changes >= 2.0, P <= 0.001) on cellobiose were enriched for amino acid (mainly methionine, cysteine, arginine, and histidine) and thiamine (vitamin B1) biosynthetic processes (Figure 2B).\\nRESULTS\\nparagraph\\nOnly a few genes in the pathways of central carbon metabolism, which include the pentose phosphate pathway, glyoxylate cycle, gluconeogenesis, and carbohydrate storage (trehalose and glycogen), changed expression levels in the presence of cellobiose compared with glucose (Figure 2C), even though cellobiose fermentation was much slower than glucose fermentation (Figure 1A). The transcript levels of the three glucose-phosphorylating enzymes, encoded by HXK1, HXK2, and GLK1, shifted to the profile similar to that of non-fermentable carbon sources in the cellobiose-grown cells (Figure 2C). When yeast cells are grown on a fermentable carbon source such as glucose, fructose, or mannose, HXK2 is induced. After shifting cells to a non-fermentable carbon source such as ethanol, HXK2 is repressed, and HXK1 and GLK1 are immediately de-repressed. Cells grown on cellobiose induced the expression of HXK1 and GLK1 by 26-fold and 2.8-fold, respectively, over that in glucose-grown cells, although the mRNA levels of HXK2 were unchanged. Hxk1 seemed to be the predominant isoenzyme in cellobiose-grown cells, because HXK1 was highly transcribed in contrast to the other two hexokinase genes (see Additional file 1: Dataset S1). Other genes in central carbon metabolic pathways with large changes in expression included the genes coding for hexose transporters, a key gluconeogenic gene (PCK1), and genes in mitochondrially compartmentalized pathways (Figure 2A, C). S. cerevisiae senses glucose both intracellularly and extracellularly over a wide range of concentrations, which possibly explains the shifts in hexose transporter expression levels observed here, which would be due to the lack of extracellular glucose in the cellobiose cultures.\\nRESULTS\\ntitle_2\\nEffects of TF modulation on cellobiose fermentation\\nFig3\\nFIG\\nfig_caption\\n\\nThe effect of manipulating transcription factors (TFs) on cellobiose fermentation. (A) TFs with significantly different expression levels on cellobiose versus glucose. For all TFs shown, P-values were well below 0.001 (see Additional file 2: Dataset S2). (B) Cellobiose fermentation profiles of TFs overexpressed in strain BY4742. The TFs chosen for overexpression were those downregulated in cellobiose in the wild-type (WT) strain. The relative cellobiose-consumption rate (qsmax) and the relative length of the lag phase were obtained by comparisons WT controls (normalized to 1.0). (C) Cellobiose fermentation profiles of deletion strains in BY4742 background for TFs upregulated in cellobiose versus glucose in (A). (D) Cellobiose fermentation profiles of strains overexpressing TFs upregulated on cellobiose versus glucose in (A). (E) Cellobiose fermentation profiles using TF mutants in strain D452-2. Because D452-2, which is another laboratory strain of S. cerevisiae, seems to have comparable fermentation performance to industrial strains, this strain has attracted attention as a host strain to express foreign sugar-utilizing pathways. In all of the above panels, the cellobiose-consumption pathway was expressed from plasmid pRS316-BT. For panels (B-E), plots of relative qsmax versus relative length of the lag time are shown. Each point represents duplicate anaerobic fermentations using a starting cellobiose concentration of 80 g/l and starting OD600 of 1. The maximum cellobiose-consumption rate and length of the lag phase were 1.22 +- 0.09 g/l/h and 74.28 +- 3.70 h, respectively, for the WT BY4742 strain, and 1.18 +- 0.00 g/l/h and 58.32 +- 2.77 h for the WT D452-2 strain.\\nRESULTS\\nparagraph\\nAs a first step to uncover the regulatory mechanisms underlying suboptimal cellobiose metabolism and thereby to improve cellobiose fermentation by engineered S. cerevisiae, TFs were targeted because of their central position in GRNs. Of the genes with significantly different expression on cellobiose versus glucose, only 19 annotated TFs were identified (Figure 3A; see Additional file 2: Dataset S2). These 19 TFs are widely distributed in the regulatory network of genes for diverse biological processes, many of which with no obvious connection to central carbon metabolism. Of the 11 TFs with decreased expression on cellobiose, Met32 and Met28 are transcription activators of genes in sulfur-containing amino acid metabolism; Thi2 is an activator of thiamine biosynthetic genes; Yap5 is an iron-responsive activator involved in the diauxic shift from glycolysis to aerobic utilization of ethanol; and Uga3, Hms2, and Kar4 are not directly involved in gene regulation of central carbon metabolism. TFs that do have a connection to central carbon metabolism include Mig2 and Mig3, which cooperate with Mig1 to repress many genes in glucose-induced repression; Sip4, an activator of gluconeogenic genes that is regulated by Snf1; and Mal13, an activator for genes in maltose utilization. Of the eight TFs with increased expression on cellobiose, Dal80 is a negative regulator of genes subject to nitrogen catabolite repression (NCR) in nitrogen degradation pathways; Sut1 positively regulates genes in sterol uptake under anaerobic conditions and hypoxic gene expression; Xbp1 is involved in gene regulation during stress or starvation; and Gsm1 is thought to be involved in energy metabolism. TFs related to central carbon metabolism include Usv1, which regulates genes during growth on non-fermentable carbon sources; Adr1, an activator of glucose repressible genes that is positively regulated by Snf1; Cat8, which mediates derepression of many genes during the diauxic shift, and which is upregulated by Snf1 through inactivation of the repressor Mig1 as well as being activated directly by phosphorylation; and Hap4, which is an activator of respiratory genes and mitochondrial function.\\nRESULTS\\nparagraph\\nTo determine the functions of these TFs in GRNs relevant to cellobiose metabolism, the effect of their genetic perturbation on cellobiose fermentation was evaluated. For the 11 TFs with decreased expression on cellobiose, overexpression mutants were tested to determine if suboptimal cellobiose fermentation might be due to their decreased expression on cellobiose (Figure 3B; see Additional file 4: Figure S1A). Most of these mutants showed similar lag phases and fermentation rates on cellobiose. Overexpression of KAR4, MAL13, or YAP5 in strain BY4742 was slightly beneficial to cellobiose fermentation, whereas THI2 overexpression strongly repressed cellobiose fermentation. To probe the connection between the eight TFs induced under cellobiose growth conditions and suboptimal cellobiose fermentation, both deletion and overexpression mutants were tested. As with the 11 TFs above, most deletion or overexpressing strains showed similar lag times and fermentation rates on cellobiose compared with wild-type (WT) yeast (Figure 3C, D; see Additional file 4: Figure S1B, C). The HAP4 deletion strain exhibited shorter lag phases and higher fermentation rates on cellobiose compared with the WT, whereas the ADR1 deletion strain had an increased cellobiose-consumption rate, but had no change in the lag phase and ethanol productivity. The SUT1 deletion strain showed clearly longer lag phases on cellobiose compared with the WT BY4742 strain. When each of the eight TFs was overexpressed individually, SUT1 increased fermentation rates over that of the WT strain, and also reduced the lag phase significantly. Interestingly, overexpression of HAP4, of which deletion led to improved cellobiose fermentation, also showed a shorter lag period and a higher cellobiose fermentation rate compared with the control strain. This result suggests that the functions of Hap4 in GRNs might not be monotonic. DAL80 overexpression reduced the lag phase on cellobiose, but fermentation rates were similar to the WT.\\nRESULTS\\nparagraph\\nS. cerevisiae GRNs have been shown to be strain-dependent, as has the promoter dependence of the cellobiose-consumption pathway. Therefore, TF expression levels that elicited significant differences in cellobiose fermentation compared with WT in the BY4742 background were further evaluated in the D452-2 (MATalpha, leu2, his3, ura3, and can1) strain background (Figure 3E; see Additional file 4: Figure S1D). Testing of the TFs showed that overexpression of KAR4, MAL13, YAP5, or HAP4 had different effects on cellobiose fermentation in the two strain backgrounds. Strikingly, HAP4 overexpression, which greatly enhanced cellobiose utilization in BY4742, had a strongly negative impact in D452-2 (Figure 3D, E). Again, this result suggests that the function of Hap4 in GRNs might not be monotonic, so that subtle expression level changes of Hap4 in the context of global regulation of other TFs in GRNs could lead to inconsistent phenotypes depending on the host strain backgrounds. Taken together, the results showed that only three TF mutations resulted in similar phenotypes in the two tested strains: SUT1 overexpression, HAP4 deletion, and DAL80 overexpression (Figure 3E; see Additional file 4: Figure S1D).\\nRESULTS\\nparagraph\\nIndustrial application of S. cerevisiae to plant biomass conversion will probably require co-consumption of multiple sugars derived from the plant cell wall, such as cellobiose and xylose. To test whether SUT1 or DAL80 overexpression would affect xylose fermentation, a uracil auxotroph of the SR8 strain, which is an engineered and evolved xylose-fermenting strain derived from D452-2, was used. SR8 expresses XYL1 (xylose reductase gene), XYL2 (xylitol dehydrogenase gene), and XKS1 (xylulose kinase gene) from chromosomally integrated copies. Further, ALD6 (aldehyde dehydrogenase gene) was deleted, resulting in strain SR8, which was comparable with the best-performing engineered S. cerevisiae strains reported for xylose fermentation. Overexpression of SUT1 or DAL80 had no impact on xylose fermentation compared with the empty vector control (see Additional file 4: Figure S2). These results suggest that the reprogramming of GRNs through SUT1 and DAL80 overexpression did not affect expression levels of enzymes in the biochemical route for xylose metabolism.\\nRESULTS\\ntitle_2\\nOptimization of the heterologous cellobiose-consumption pathway through promoter engineering\\nFig4\\nFIG\\nfig_caption\\n\\nPromoter engineering to optimize expression of the heterologous cellobiose-utilizing pathway. (A) Identification of strong promoters from transcription profiling of cellobiose-grown (C8) and glucose-grown (G8) cultures. (B) Verification of promoter strengths by measuring green fluorescent protein (GFP) fluorescence using flow cytometry. Anaerobically grown cells on cellobiose were harvested at mid-exponential phase and analyzed. The cell surface density of enhanced green fluorescent protein (eGFP)-tagged CDT-1 is shown. (C) Construction of a promoter library of 4 promoters and 2 gesnes for expression of the cellobiose-utilization pathway. The plasmids pRS316 (CEN URA) and pRS315 (CEN LEU) were used to express cdt-1 and a codon-optimized version of N. crassa gh1-1 (gh1-1a), respectively. (D) Comparison of cellobiose-consumption rates (qsmax) using strains expressing the cellobiose-utilization pathway from the promoter library. The starting OD600 of 1 was used. The promoters for each gene are shown, and the rates are color-coded by relative rates. Fermentation parameters were calculated from the fermentation profiles in Fig. S4D.\\nRESULTS\\nparagraph\\nIn addition to reprogramming of GRNs via perturbations of TFs for improved cellobiose fermentation, expression levels of the heterologous cellobiose-assimilation pathway should be optimized to achieve efficient and rapid fermentation of cellobiose. Promoter engineering has been widely and successfully applied to optimize foreign pathways in host strains. Recently, an approach named \"customized optimization of metabolic pathways by combinatorial transcriptional engineering\" (COMPACTER) was successfully applied to improve recombinant cellobiose-fermenting S. cerevisiae stains by fine-tuning gene expression in a cellobiose-utilizing pathway. Notably, mRNA levels of the foreign genes cdt-1 and gh1-1 from N. crassa seemed to increase in response to cellobiose (see Additional file 4: Figure S3), although these two genes were under the control of the PGK1 promoter, and endogenous PGK1 expression levels were nearly unchanged in the cellobiose and glucose conditions (see Additional file 4: Figure S3). The heterologous cellobiose-utilizing pathway genes gh1-1 and cdt-1 were expressed at moderate levels, which might contribute to slow fermentation rates on cellobiose. To test whether or not increased expression of cdt-1 and/or gh1-1 would improve cellobiose fermentation, mRNA quantification analysis by RNA deep sequencing (see Additional file 1: Dataset S1) was used to determine which genes are highly expressed on cellobiose. The three genes with the highest expression on cellobiose were CCW12, TDH3, and FBA1 (Figure 4A; see Additional file 1: Dataset S1), suggesting that the promoters for these genes might be useful to improve the expression levels of gh1-1 and cdt-1, and further improve cellobiose fermentation. The promoter strengths for CCW12, TDH3, and FBA1 were verified by flow cytometry of cells expressing CDT-1 fused with an enhanced green fluorescent protein (eGFP) from the different promoters (Figure 4B; see Additional file 4: Figure S4A). Additionally, expression of codon-optimized gh1-1 (hereafter named gh1-1a) resulted in about 2-fold enhanced beta-glucosidase activity in cell extracts (see Additional file 4: Figure S4B). To test for the best cellobiose fermentation pathway, 16 combinations of plasmids were created using the promoters PCCW12, PTDH3, and PFBA1, as well as the original promoter PPGK1, to express cdt-1 and gh1-1a (Figure 4C). Comparisons of strains harboring these plasmids indicated that the combination of PTDH3 for cdt-1 expression and PCCW12 for gh1-1a resulted in the best-performing cellobiose-utilizing pathway, with approximately two-fold higher fermentation rates compared with the pathway driven by PGK1 promoters (Figure 4D; see Additional file 4: Figure S4C). Interestingly, the cellobiose-utilizing pathways segregated in performance based on the promoter driving gh1-1a (see Additional file 4: Figure S4D), suggesting that expression levels of gh1-1a may be limiting in cellobiose fermentation.\\nRESULTS\\ntitle_2\\nFig5\\nFIG\\nfig_caption\\n\\nComparisons of cellobiose fermentation using strain D452-2 with the original (P\\nPGK1\\n-driven\\ncdt-1\\nand\\ngh1-1a\\n) and optimized (P\\nTDH3\\n-driven\\ncdt-1\\nand P\\nCCW12\\n-driven\\ngh1-1a\\n) cellobiose-utilizing pathways and transcription factor (TF) mutants. (A) Cellobiose-consumption rates and ethanol productivities with strains expressing the original and optimized cellobiose-utilization pathways, and also either overexpressing SUT1 or harboring a hap4 deletion. For details of the strains, see Additional file 4: Table S1. (B) Fermentation times of the strains in (A). (C) Comparisons of cellobiose-consumption rates and ethanol productivities with WT D452-2 or D452-2 (hap4Delta) expressing the optimized cellobiose-utilization pathway, and additionally overexpressing SUT1 and/or DAL80. (D) Fermentation times of the strains in (C), defined as the time when ethanol reached its maximum titer. In all experiments in (A-D), an initial OD600 of 20 was used. Data represent the mean and standard error of triplicate cultures on each source. Statistical analysis in (A) and (C) was performed using two-way ANOVA (with strains and fermentation rate including qsmax and PEtOH as the factors) followed by Tukey\\'s multiple-comparison posttest (***P < 0.001). Statistical analysis in (B) and (D) was performed using one-way ANOVA followed by Tukey\\'s multiple-comparison posttest (***P < 0.001).\\nRESULTS\\nparagraph\\nTo evaluate the combined effect of TF mutants and optimization of the heterologous cellobiose-utilizing pathway, cellobiose fermentation of strains with the original (PPGK1-driven cdt-1 and gh1-1 or gh1-1a) and optimized (PTDH3-driven cdt-1 and PCCW12-driven gh1-1a) cellobiose-utilizing pathways, as well as with additional SUT1 overexpression or HAP4 deletion, were compared (Figure 5A, B). Cellobiose fermentation experiments were performed using high-cell density inocula to assess the upper limits of cellobiose fermentation performance by the engineered strains. Strains with the original or codon-optimized gh1-1 showed similar fermentation performance. The best cellobiose-utilizing pathway fine-tuned by promoter engineering resulted in an increase of 28% and 24% in cellobiose-consumption rate and ethanol productivity. SUT1 overexpression or HAP4 deletion further increased the cellobiose-consumption rates and ethanol productivities by about 30% and 48%, respectively. Overall, optimization of the cellobiose-utilizing pathway and TF perturbations additively increased cellobiose-consumption rates and ethanol productivities by approximately 66% and 83% (Figure 5A), respectively, and shortened the fermentation times by approximately 45 hours compared with the WT strain expressing the original cellobiose-consumption pathway (Figure 5B).\\nRESULTS\\nparagraph\\nTo test whether the effects of the TFs on cellobiose fermentation are synergistic, D452-2 strains overexpressing DAL80 or SUT1, or with HAP4 deletion, were combined with the best-performing cellobiose-utilizing pathway (Figure 5C, D). When PTDH3-driven cdt-1 and PCCW12-driven gh1-1a were combined with SUT1 overexpression, the cellobiose-consumption rate and ethanol productivity increased by 39% and 40%, respectively (Figure 5C), and the mutant strain finished fermentation approximately 19 hours earlier than the WT strain expressing the optimized cellobiose-consumption pathway (Figure 5D). However, DAL80 overexpression in D452-2 resulted in similar cellobiose fermentation rates to those of the WT strain expressing the optimized cellobiose-consumption pathway, and negatively influenced the effect of SUT1 overexpression when both DAL80 and SUT1 were overexpressed (Figure 5C, D). When D452-2 (hap4 ) was used as a host strain, similar fermentation profiles were observed regardless of whether SUT1 or DAL80 were overexpressed, and the positive effects on cellobiose fermentation were similar to SUT1 overexpression. All the mutants using D452-2 (hap4 ) as the host strain had average increases of 46% and 56% in cellobiose-consumption rates and ethanol productivities, respectively (Figure 5C), and finished fermentation approximately 23 hours earlier than the WT D452-2 expressing the best cellobiose-utilization pathway (Figure 5D).\\nDISCUSS\\ntitle_1\\nDiscussion\\nDISCUSS\\nparagraph\\nProduction of renewable fuels and chemicals from plant cell wall using S. cerevisiae will require substantial engineering to enable this yeast to function on an industrial scale. For example, S. cerevisiae must be adapted to consume abundant non-glucose sugars such as cellobiose and xylose. Evolutionary engineering has been widely adopted to improve S. cerevisiae strains, but these approaches generate strains that are difficult to further engineer, for example, by crossing. Systems biology methods could provide a framework for rational strain engineering, as they report on physiological responses on a genome-wide and organism-wide scale. However, identifying causative effects from systems-level data remains a difficult problem. Here we show that transcription profiling can be used to identify the regulatory mechanisms underlying suboptimal metabolism and swiftly narrow down the targets for engineering S. cerevisiae to rapidly ferment cellobiose, a prototypical non-glucose sugar released from the plant cell wall in industrial processes.\\nDISCUSS\\nparagraph\\nIn this study, transcription profile comparisons between cellobiose-fermenting and glucose-fermenting cultures provided a genome-wide view of differential gene expression, and thus uncovered how the GRNs in yeast are perturbed by the novel non-glucose sugar cellobiose (Figure 1B). The cAMP-dependent PKA pathway, which controls the majority of glucose-induced changes, was activated to an extent under cellobiose conditions, as the overwhelming majority of genes (91.8%) displayed no significant change in transcriptional expression compared with yeast grown on glucose (see Additional file 1: Dataset S1, absolute fold changes >=2.0, P <= 0.001). The decreased expression of RGS2 and PDE1 on cellobiose suggests that cAMP-mediated signaling in cellobiose-grown cells might be similar to that in glucose-grown cells (see Additional file 2: Dataset S2). In the presence of glucose, HXT1, HXT3, STD1, and MIG2 were relieved from the repression complex of Rgt1 (see Additional file 4: Figure S5A). All of these genes were repressed in the presence of cellobiose, and HXT6 and HXT7, which require Snf3 to be repressed by the Snf1-Mig1 glucose repression pathway, were de-repressed in the presence of cellobiose (see Additional file 2: Dataset S2; see Additional file 4: Figure S5A). This observation suggests that, in contrast to extracellular glucose, extracellular cellobiose is not sensed by Snf3 and Rgt2 to regulate glucose transport. Snf3 and Rgt2 were reported to sense extracellular xylose, suggesting that the GRNs involved in sugar transport have varied responses to different non-glucose sugars. Many genes involved in respiration, gluconeogenesis, and the metabolism of alternative carbon sources and their TFs are repressed by the Snf1-Mig1 glucose repression pathway, which allows aerobic fermentation and the preferential use of glucose when cells are grown in mixed sugars. Strikingly, the expression of these genes increased in cellobiose-grown cells (see Additional file 2: Dataset S2, Additional file 4: Figure S5B). This result indicates that the Snf1-Mig1 glucose repression pathway is not activated by cellobiose as it is by glucose. A similar situation may also occur in recombinant xylose-utilizing S. cerevisiae, although additional aspects of xylose also play a role. For instance, Hxk2 has dual functions as a glycolytic enzyme and a regulator of the glucose repression pathway. Xylose can inactivate Hxk2 through an autophosphorylation mechanism because it lacks C6 hydroxyl, which impedes all functions of Hxk2. A Hxk2 variant (Phe159Val) showed increased catalytic activity in the presence of xylose, which improved xylose fermentation by potentially restoring the regulatory function of Hxk2. By contrast, cellobiose is unlikely to inactivate Hxk2, which suggests that Hxk2 might not be a key factor responsible for inactivation of the glucose repression pathway in cellobiose conditions. Taken together, potential limitations for rapid cellobiose metabolism in terms of GRNs were observed in multiple glucose-responsive pathways (Figure 1; see Additional file 4: Figure S5). Future experiments will be required to determine how cellobiose perturbs the GRNs at the molecular level in engineered S. cerevisiae.\\nDISCUSS\\nparagraph\\nUsing transcription profiling, we were able to identify pathways differentially regulated in cellobiose versus glucose fermentations, including mitochondrial function and amino acid biosynthesis. Interestingly, similar patterns of pathway activation and repression were seen with xylose. However, how to apply these new insights to further improve the S. cerevisiae strains is not obvious. By using transcription profiling to identify TFs that are significantly differentially expressed in the two fermentation conditions, we were able to rapidly screen a small number of targets for improved cellobiose utilization. Further, transcription profiling provided quantitative data on transcription promoter strengths under the actual conditions of cellobiose fermentation. These data allowed us to rationally test defined promoter strengths for the heterologous cellobiose-consumption pathway. By combining TF engineering with promoter screening, we improved cellobiose fermentation rates by over 60% (Figure 5A) and fermentations finished around 45 hours earlier than those containing the original engineered yeast strain (Figure 5B), without affecting ethanol yield (see Additional file 4: Figure S6).\\nDISCUSS\\nparagraph\\nOur strategy also identified at least two candidate TFs whose expression levels might be generally engineered in S. cerevisiae strains to improve cellobiose fermentation. SUT1, which improved anaerobic cellobiose fermentation when overexpressed, is an activator of sterol uptake and hypoxic genes, but its regulatory role in central carbon metabolism is not known. Constitutive overexpression of SUT1 has been shown to slow growth on glucose in aerobic cultures by downregulating genes important for respiratory metabolism, and it was also shown to block filamentous growth. HAP4, deletion of which improved anaerobic cellobiose fermentation is a regulatory subunit of the Hap2/3/4/5 complex that positively regulates respiratory gene expression, and is a hotspot of genetic variation in cell physiology between yeast strains. HAP4 overexpression has been reported to positively affect the balance between respiration and fermentation of glucose metabolism in aerobic conditions by inducing mitochondrial function. In contrast to HAP4 deletion, which increased cellobiose fermentation, HAP4 overexpression resulted in contradictory effects in different strain backgrounds. Interestingly, Sut1 and Hap4 are involved in a complex interplay between TFs including Cat8, Sip4, Adr1, and Rds2, which regulate the utilization of non-fermentable carbon sources. DAL80 overexpression shortened the lag phase of cellobiose fermentation (Figure 3D, E), but reduced the positive effect of SUT1 overexpression (Figure 5). Dal80 is one of four regulators in NCR. NCR connects to glucose signaling through the action of the NCR regulator Gln3, but the role of Dal80 in the regulation of sugar metabolism is unknown. Although it is not entirely clear how the three TFs (Sut1, Hap4, and Dal80) regulate cellobiose fermentation, the pathways we identified, including those for mitochondrial function and amino acid metabolism, shed new light on how non-glucose sugars affect S. cerevisiae physiology, and could serve as a starting point for further engineering efforts. Even if these TFs prove to be beneficial only in a narrow host range, the systems-level strategy we employed should be useful in other S. cerevisiae strains to identify a narrow range of targets for strain improvement.\\nDISCUSS\\nparagraph\\nAlthough we focused on transcription profiling as a starting point for rational strain improvement, it is clear that multiple layers of post-transcriptional regulation, including that of TFs, probably also contribute to suboptimal non-glucose sugar fermentation. These additional layers of regulation could be exploited in the future to further improve non-glucose sugar fermentation. For example, central carbon metabolism is thought to be highly regulated at the post-transcriptional level. Additional methods such as ribosome profiling and metabolomics could be used to identify additional bottlenecks in central carbon metabolism that could be addressed in the future. These methods to address post-transcriptional regulation could also be used to identify a small number of targets that could be engineered to improve co-fermentation of plant cell wall-derived sugars (such as cellobiose and xylose, and to improve the production of \"drop-in\" fuels and other renewable chemicals using yeast.\\nCONCL\\ntitle_1\\nConclusions\\nCONCL\\nparagraph\\nOur systems-level study has revealed some of the key regulatory mechanisms underlying suboptimal metabolism of the non-glucose sugar cellobiose, and further optimized cellobiose fermentation by engineered S. cerevisiae. Compared with glucose metabolism, cellobiose metabolism induced mitochondrial activation and reduced amino acid biosynthesis under fermentation conditions. Using systems-level inputs as a starting point, we were able to modulate key TFs that cause suboptimal cellobiose fermentation and fine-tune the expression of the heterologous cellobiose-consumption pathway, greatly improving cellobiose fermentation by engineered S. cerevisiae. Thus, our study demonstrates a promising paradigm for systems biology-based rational microbial strain engineering.\\nMETHODS\\ntitle_1\\nMethods\\nMETHODS\\ntitle_2\\nPlasmids and S. cerevisiae stains\\nMETHODS\\nparagraph\\nFor all plasmids and host strains used in this study, see Additional file 4: Table S1. The RF-cloning method and In-Fusion  HD Cloning Kit (Clontech Laboratories, Inc., Mountain View, CA, USA) were used for constructing plasmids. Genes for cdt-1 and gh1-1 from N. crassa were used to reconstitute an intracellular cellobiose-utilization pathway in S. cerevisiae. eGFP-tagged cdt-1 and gh1-1 expressed from PGK1 promoters were combined into a single plasmid using pRS426 (2 mu URA) or pRS316 (CEN URA) as backbone. The resulting plasmids were named pRS426-BT or pRS316-BT, respectively, and were transformed into S. cerevisiae strains to achieve intracellular utilization of cellobiose. The genes for TFs were inserted into the pRS313 (CEN HIS) plasmid under the control of the TEF1 promoter and CYC1 terminator (for primers, see Additional file 4: Table S2). CEN plasmids were used to minimize variation due to variable plasmid copy number.\\nMETHODS\\nparagraph\\nStrain BY4742 and TF knockout mutants (Open Biosystems Co., Lafayette, CO, USA), D452-2, and a uracil auxotroph of the SR8 strain (which is an engineered xylose-fermenting strain derived from D452-2) were used as host strains as specified in the text. Yeast transformations were performed (Frozen-EZ Yeast Transformation II Kit ; Zymo Research Corp., Irvine, CA, USA) in accordance with the instructions. To select transformants using an amino acid auxotrophic marker, the appropriate complete minimal dropout medium was used, which contains 43.7 g/l drop out base with agar (DOBA), double the recommended amount of the appropriate complete supplement mixture (CSM) dropout mixture (MP Biomedicals, Santa Ana, CA, USA) and 100 mg/l adenine hemisulfate.\\nMETHODS\\nparagraph\\nTo delete TFs in strain D452-2, a PCR-based gene disruption method was used. The KanMX expression cassette was amplified by PCR (for primers, see Additional file 4: Table S2) that target the KanMX expression cassette on the plasmid pUG6 and include 50 bp sequences homologous to upstream and downstream sequences of the TF gene. Positive transformants were selected on YPAD medium (10 g/l yeast extract, 20 g/l Bacto peptone, 100 mg/l adenine hemisulfate and 20 g/l of glucose) with 200 mug/ml of geneticin G418. Diagnostic PCR reactions with primers targeting approximately 200 bp upstream of the TF gene and the KanMX-specific primer (KanB), or specific primers (see Additional file 4: Table S2) targeting the middle of the TF gene, were used to confirm successful deletion.\\nMETHODS\\ntitle_2\\nAnaerobic fermentation\\nMETHODS\\nparagraph\\nFlask fermentations were performed under anaerobic conditions using optimal minimal medium (OMM), which was a modified version of previous defined media compositions, and contained (per liter) 1.7 g yeast nitrogen base (YNB) lacking ammonium sulfate (catalog number Y1251; Sigma-Aldrich, St. Louis, MO, USA), two-fold appropriate CSM dropout mixture (MP Biomedicals, Santa Ana, CA, USA), 10 g (NH4)2SO4, 1 g MgSO4   7H2O, 6 g KH2PO4, 100 mg adenine hemisulfate, 10 mg inositol, as well as an additional 100 mg glutamic acid, 20 mg lysine, and 375 mg serine. In addition, 80 g/l of glucose, 80 g/l cellobiose or 40 g/l xylose were added as carbon sources. The pH of each medium was adjusted to 6.0 and buffered using 100 mM 4-morpholineethanesulfonic acid (MES monohydrate; Sigma-Aldrich, St. Louis, MO, USA). When the initial OD of approximately 1.0 was used for setting up fermentation experiments, the medium was supplemented with 0.42 g/l Tween 80 and 0.01 g/l ergosterol.\\nMETHODS\\nparagraph\\nYeast cells transformed with plasmids expressing the cellobiose-utilization pathway and showing green fluorescent protein (GFP) fluorescence on plates were grown in 50 ml Falcon tubes containing 10 ml OMM media with 20 g/l glucose at 30 C for 24 hours. Cells were then inoculated to prepare starting cultures in 500 ml flasks. Cells were harvested in the early stationary phase and inoculated after washing with sterile double-distilled H2O. The initial OD600 of yeast used for anaerobic fermentations was around 1 or 20, as specified in the text. To achieve anaerobic conditions, cells were grown at 30 C with shaking at 220 rpm in 150 ml sealed serum flasks containing 50 ml media. The flasks were sealed with a rubber stopper clamped with an aluminum seal, then nitrogen gas purging was carried out for 20 minutes. Samples were taken through the stopper via a sterile syringe.\\nMETHODS\\ntitle_2\\nRNA sequencing and data analysis\\nMETHODS\\nparagraph\\nAnaerobic cultures of strain BY4742 expressing the cellobiose-utilizing pathway from plasmid pRS426-BT were carried out in biological triplicate using either 80 g/l glucose or 80 g/l cellobiose. Samples were harvested when approximately 50 g/l of residual sugar was present (Figure 1A). Cells were harvested in Falcon tubes precooled in liquid N2 by centrifuging for 5 minutes at 4000 rpm (3220 x g). The supernatant was discarded, and the pellet was stored at -80 C until further use.\\nMETHODS\\nparagraph\\nTotal RNA was extracted using the RiboPure Yeast kit (Ambion, Austin, TX, USA), in accordance with the manufacturer\\'s instructions, except that cells were disrupted by bead-beating three times for 30 seconds each, with a pause for 30 seconds pbetween runs. Total RNA (4 mug) was used to prepare the multiplexing libraries with barcodes (TruSeq RNA Sample Prep Kit; Illumina) following the manufacturer\\'s instructions. The final cDNA libraries were quantified (Agilent Bioanalyzer 2000; Functional Genomics Laboratory, University of California, Berkeley, CA, USA) and sequenced (Illumina Genome Analyzer-II ; Vincent J. Coates Genomic Sequencing Laboratory, University of California, Berkeley) using standard Illumina operating procedures.\\nMETHODS\\nparagraph\\nSequence reads were assembled and analyzed in CLC Genomics Workbench 6.5 (CLC Bio, Aarhus, Denmark). The S. cerevisiae S288C genome was downloaded from RefSeq at the NCBI (sequence assembly version R64-1-1) including 16 chromosomes and the mitochondrial genome. The genes for N. crassa beta-glucosidase gh1-1 and eGFP-tagged N. crassa cellodextrin transporter cdt-1 as encoded in pRS426-BT were manually annotated and combined with the S. cerevisiae S288C genome as the reference (total size of 12.17 Mb). Expression values were normalized by calculating the reads per kb of mRNA exon per million mapped reads (reads per kb per million; RPKM), and further normalized using the option of \"By totals\". A mean of 38 million 50 bp single reads corresponding to approximately 156-fold coverage of the reference was generated for each library. Following the default parameters in the CLC Genomics Workbench, around 92.3% of reads per library was successfully imported, of which approximately 83.4% was mapped. Of the imported reads, about 76.3% was uniquely mapped. Next, an unpaired two-group comparison of all six libraries using the mapping results was used for quality control analysis. The tools for quality control including box plots and hierarchical clustering showed that the biological triplicate libraries were grouped according to the different sugar conditions (see Additional file 4: Figure S7A-B). Transcription profile data are available in supplementary material (see Additional file 1: Dataset S1) and from the Gene Expression Omnibus (accession no.GSE54825). All annotations were derived from the SGD gene association file.\\nMETHODS\\nparagraph\\nTo identify differential expression in the cellobiose versus glucose fermentations, an unpaired two-group comparison was used to generate fold changes and further analyzed for statistical significance using Baggerley\\'s test. Significantly differentially expressed genes, highlighted in red in volcano plots (see Additional file 4: Figure S7C), were then sorted by applying a stringent false discovery rate (FDR)-corrected cutoff P-value of 0.001 or less and an absolute fold-change threshold of 2.0 or greater. Significantly upregulated and downregulated genes (see Additional file 2: Dataset S2) were further tested for GO biological process enrichment using FunSpec with a P-value cutoff of 0.01 and Bonferroni correction (see Additional file 3: Dataset S3).\\nMETHODS\\ntitle_2\\nOptimization of the cellobiose-utilizing pathway using codon-optimized GH1-1 and promoter engineering\\nMETHODS\\nparagraph\\nTo improve its expression in S. cerevisiae, gh1-1 was codon-optimized by DNA2.0 (USA), and is hereafter named gh1-1a (see Additional file 4: Figure S8). The three genes with the highest expression on cellobiose (CCW12, TDH3, and FBA1), as determined from the RNA deep sequencing data, were identified and further confirmed for promoter strength by overexpressing the eGFP-tagged CDT-1 (see Additional file 1: Dataset S1). A mutant pathway library was created by combining the two cellobiose-utilizing genes cdt-1 and gh1-1a and four promoters (PCCW12, PTDH3, PFBA1 and PPGK1), and screened in strain D452-2 for cellobiose fermentation (Figure 4C). The eGFP fused to CDT-1 was detected by excitation at 488 nm to confirm the promoter strengths. Cells from the mutant pathway library were harvested at mid-exponential phase, washed, and resuspended in 1 x phosphate buffered saline (PBS) pH 7.4. WT D452-2 was used as a negative control. The cells were analyzed and sorted (Cell Lab QUANTA* Flow Cytometer; Beckman Coulter, Brea, CA). Quantitative fluorescence surface density (FSD) was estimated from the raw data of EV (electronic volume) and FL1 (fluorescence, green) values (FSD = FL1/ ((volume channel) ^ (2/3))).\\nMETHODS\\ntitle_2\\nAnalytical methods and calculation of fermentation parameters\\nMETHODS\\nparagraph\\nCell growth was monitored at OD600 using a UV-visible spectrophotometer (8453 UV-vis; Agilent, Santa Clara, CA, USA). Cellobiose, glucose, xylose, and ethanol concentrations were determined by high performance liquid chromatography on a chromatograph (Shimadzu) equipped with a refractive index detector and a fast acid column 100 mm length x 7.8 mm internal diameter (RFQ; Phenomenex Inc., Torrance, CA, USA). The column was eluted with 0.01 N of H2SO4 at a flow rate of 1.0 ml/min at 55 C.\\nMETHODS\\nparagraph\\nFermentation parameters including cellobiose-consumption rate (qsmax) and ethanol productivity (PEtOH) were calculated for the fermentation profiles using Origin 8 (Originlab ). The data for cellobiose-consumption and ethanol production were plotted using Origin 8 (Originlab ), and curves were fitted to a Boltzmann function, which is used to fit a curve with a sigmoidal shape. The equation used was:where A1 is the initial value, A2 is the final value, x0 is the inflection point of the sigmoidal curve, and dx is the time constant. The slope at the inflection point (x0) indicates the maximum rate of fermentation. The value for the slope is:\\nMETHODS\\nparagraph\\nThus, the equation used for calculating cellobiose-consumption rate (g cellobiose/l/h) is:and the equation used for calculating ethanol productivity (g ethanol/l/h) is:\\nMETHODS\\nparagraph\\nTo calculate the lag times for cellobiose consumption and ethanol production, the different values at 12 hour intervals were obtained according to the fitting equation, then log10 transformed for plotting. An intercept between the linear curve and the time axis before x0 was defined as the lag time. An intercept between the linear curve and the time axis after x0 was defined as the fermentation time, at which point cellobiose has been depleted and ethanol reaches maximum titer. The mean rates or oag times for the WT strain in different batches of fermentation were used to normalize those of the TF mutant strains.\\nMETHODS\\ntitle_2\\nStatistical significance tests\\nMETHODS\\nparagraph\\nFor comparison of fermentation between the original recombinant cellobiose-utilizing strain and further engineered strains, ANOVA was used, followed by Tukey\\'s multiple-comparison posttest with a 95% confidence interval. Statistics were performed by using SigmaPlot (version 11.0). The differences were considered significant at P < 0.001.\\nMETHODS\\ntitle_1\\nAdditional files\\nABBR\\ntitle\\nAbbreviations\\nABBR\\nparagraph\\nCSM\\nABBR\\nparagraph\\nComplete supplement mixture\\nABBR\\nparagraph\\nDOBA\\nABBR\\nparagraph\\ndrop out base with agar\\nABBR\\nparagraph\\neGFP\\nABBR\\nparagraph\\nenhanced green fluorescent protein\\nABBR\\nparagraph\\nFDR\\nABBR\\nparagraph\\nfalse discovery rate\\nABBR\\nparagraph\\nFSD\\nABBR\\nparagraph\\nfluorescence surface density, GFP, green fluorescent protein\\nABBR\\nparagraph\\nGO\\nABBR\\nparagraph\\nGene Ontology\\nABBR\\nparagraph\\nGRN\\nABBR\\nparagraph\\nGene regulatory network\\nABBR\\nparagraph\\nNCR\\nABBR\\nparagraph\\nnitrogen catabolite repression\\nABBR\\nparagraph\\nOMM\\nABBR\\nparagraph\\nOptimal minimal medium\\nABBR\\nparagraph\\nPKA\\nABBR\\nparagraph\\nProtein kinase A\\nABBR\\nparagraph\\nTF\\nABBR\\nparagraph\\nTranscription factor\\nABBR\\nparagraph\\nWT\\nABBR\\nparagraph\\nwild type\\nABBR\\nfootnote\\nCompeting interests\\nABBR\\nfootnote\\nThe authors declare that there are no competing interests.\\nABBR\\nfootnote\\nAuthors\\' contributions\\nABBR\\nfootnote\\nYL and JHDC designed the project; YL, KC, LAS, RE, JMG, and SRK performed the experiments; YL, YSJ and JHDC wrote the manuscript; and JHDC supervised the research study and gave final approval of the version to be published. All authors read and approved the final manuscript.\\nREF\\ntitle\\nReferences\\nsurname:Coulier;given-names:L\\nsurname:Zha;given-names:Y\\nsurname:Bas;given-names:R\\nsurname:Punt;given-names:PJ\\nREF\\nBioresour Technol\\nref\\nAnalysis of oligosaccharides in lignocellulosic biomass hydrolysates by high-performance anion-exchange chromatography coupled with mass spectrometry (HPAEC-MS)\\nsurname:Kim;given-names:SR\\nsurname:Ha;given-names:SJ\\nsurname:Wei;given-names:N\\nsurname:Oh;given-names:EJ\\nsurname:Jin;given-names:YS\\nREF\\nTrends Biotechnol\\nref\\nSimultaneous co-fermentation of mixed sugars: a promising strategy for producing cellulosic ethanol\\nsurname:Hong;given-names:KK\\nsurname:Nielsen;given-names:J\\nREF\\nCell Mol Life Sci\\nref\\nsurname:Dellomonaco;given-names:C\\nsurname:Fava;given-names:F\\nsurname:Gonzalez;given-names:R\\nREF\\nMicrob Cell Factories\\nref\\nsurname:Galazka;given-names:JM\\nsurname:Tian;given-names:C\\nsurname:Beeson;given-names:WT\\nsurname:Martinez;given-names:B\\nsurname:Glass;given-names:NL\\nsurname:Cate;given-names:JH\\nREF\\nScience\\nref\\nCellodextrin transport in yeast for improved biofuel production\\nsurname:Ha;given-names:SJ\\nsurname:Galazka;given-names:JM\\nsurname:Kim;given-names:SR\\nsurname:Choi;given-names:JH\\nsurname:Yang;given-names:X\\nsurname:Seo;given-names:JH\\nsurname:Glass;given-names:NL\\nsurname:Cate;given-names:JH\\nsurname:Jin;given-names:YS\\nREF\\nProc Natl Acad Sci U S A\\nref\\nEngineered Saccharomyces cerevisiae capable of simultaneous cellobiose and xylose fermentation\\nsurname:Ha;given-names:SJ\\nsurname:Wei;given-names:Q\\nsurname:Kim;given-names:SR\\nsurname:Galazka;given-names:JM\\nsurname:Cate;given-names:JH\\nsurname:Jin;given-names:YS\\nREF\\nAppl Environ Microbiol\\nref\\nCofermentation of cellobiose and galactose by an engineered Saccharomyces cerevisiae strain\\nsurname:Garcia Sanchez;given-names:R\\nsurname:Hahn-Hagerdal;given-names:B\\nsurname:Gorwa-Grauslund;given-names:MF\\nREF\\nBiotechnol Biofuels\\nref\\nCross-reactions between engineered xylose and galactose pathways in recombinant Saccharomyces cerevisiae\\nsurname:Wisselink;given-names:HW\\nsurname:Toirkens;given-names:MJ\\nsurname:Wu;given-names:Q\\nsurname:Pronk;given-names:JT\\nsurname:Van Maris;given-names:AJ\\nREF\\nAppl Environ Microbiol\\nref\\nNovel evolutionary engineering approach for accelerated utilization of glucose, xylose, and arabinose mixtures by engineered Saccharomyces cerevisiae strains\\nsurname:Hahn-Hagerdal;given-names:B\\nsurname:Karhumaa;given-names:K\\nsurname:Fonseca;given-names:C\\nsurname:Spencer-Martins;given-names:I\\nsurname:Gorwa-Grauslund;given-names:MF\\nREF\\nAppl Microbiol Biotechnol\\nref\\nTowards industrial pentose-fermenting yeast strains\\nsurname:Du;given-names:J\\nsurname:Yuan;given-names:Y\\nsurname:Si;given-names:T\\nsurname:Lian;given-names:J\\nsurname:Zhao;given-names:H\\nREF\\nNucleic Acids Res\\nref\\nCustomized optimization of metabolic pathways by combinatorial transcriptional engineering\\nsurname:Bae;given-names:YH\\nsurname:Kang;given-names:KH\\nsurname:Jin;given-names:YS\\nsurname:Seo;given-names:JH\\nREF\\nJ Biotechnol\\nref\\nMolecular cloning and expression of fungal cellobiose transporters and beta-glucosidases conferring efficient cellobiose fermentation in Saccharomyces cerevisiae\\nsurname:Ha;given-names:SJ\\nsurname:Kim;given-names:SR\\nsurname:Kim;given-names:H\\nsurname:Du;given-names:J\\nsurname:Cate;given-names:JH\\nsurname:Jin;given-names:YS\\nREF\\nBioresour Technol\\nref\\nContinuous co-fermentation of cellobiose and xylose by engineered Saccharomyces cerevisiae\\nsurname:Eriksen;given-names:DT\\nsurname:Hsieh;given-names:PC\\nsurname:Lynn;given-names:P\\nsurname:Zhao;given-names:H\\nREF\\nMicrob Cell Factories\\nref\\nsurname:Zaman;given-names:S\\nsurname:Lippman;given-names:SI\\nsurname:Zhao;given-names:X\\nsurname:Broach;given-names:JR\\nREF\\nAnnu Rev Genet\\nref\\nHow Saccharomyces responds to nutrients\\nsurname:Busti;given-names:S\\nsurname:Coccetti;given-names:P\\nsurname:Alberghina;given-names:L\\nsurname:Vanoni;given-names:M\\nREF\\nSensors (Basel)\\nref\\nGlucose signaling-mediated coordination of cell growth and cell cycle in Saccharomyces cerevisiae\\nsurname:Gelade;given-names:R\\nsurname:Van de Velde;given-names:S\\nsurname:Van Dijck;given-names:P\\nsurname:Thevelein;given-names:JM\\nREF\\nGenome Biol\\nref\\nMulti-level response of the yeast genome to glucose\\nsurname:Rolland;given-names:F\\nsurname:Winderickx;given-names:J\\nsurname:Thevelein;given-names:JM\\nREF\\nFEMS Yeast Res\\nref\\nGlucose-sensing and -signalling mechanisms in yeast\\nsurname:Macneil;given-names:LT\\nsurname:Walhout;given-names:AJ\\nREF\\nGenome Res\\nref\\nsurname:Gorke;given-names:B\\nsurname:Stulke;given-names:J\\nREF\\nNat Rev Microbiol\\nref\\nCarbon catabolite repression in bacteria: many ways to make the most out of nutrients\\nsurname:Zaman;given-names:S\\nsurname:Lippman;given-names:SI\\nsurname:Schneper;given-names:L\\nsurname:Slonim;given-names:N\\nsurname:Broach;given-names:JR\\nREF\\nMol Syst Biol\\nref\\nGlucose regulates transcription in yeast through a network of signaling pathways\\nsurname:Ozcan;given-names:S\\nsurname:Johnston;given-names:M\\nREF\\nMicrobiol Mol Biol Rev\\nref\\nFunction and regulation of yeast hexose transporters\\nsurname:Gancedo;given-names:JM\\nREF\\nMicrobiol Mol Biol Rev\\nref\\nYeast carbon catabolite repression\\nsurname:Palomino;given-names:A\\nsurname:Herrero;given-names:P\\nsurname:Moreno;given-names:F\\nREF\\nNucleic Acids Res\\nref\\nTpk3 and Snf1 protein kinases regulate Rgt1 association with Saccharomyces cerevisiae HXK2 promoter\\nsurname:Yu;given-names:H\\nsurname:Gerstein;given-names:M\\nREF\\nProc Natl Acad Sci U S A\\nref\\nGenomic analysis of the hierarchical structure of regulatory networks\\nsurname:Lelli;given-names:KM\\nsurname:Slattery;given-names:M\\nsurname:Mann;given-names:RS\\nREF\\nAnnu Rev Genet\\nref\\nDisentangling the many layers of eukaryotic transcriptional regulation\\nsurname:Albert;given-names:FW\\nsurname:Treusch;given-names:S\\nsurname:Shockley;given-names:AH\\nsurname:Bloom;given-names:JS\\nsurname:Kruglyak;given-names:L\\nREF\\nNature\\nref\\nGenetics of single-cell protein abundance variation in large yeast populations\\nsurname:Kummel;given-names:A\\nsurname:Ewald;given-names:JC\\nsurname:Fendt;given-names:SM\\nsurname:Jol;given-names:SJ\\nsurname:Picotti;given-names:P\\nsurname:Aebersold;given-names:R\\nsurname:Sauer;given-names:U\\nsurname:Zamboni;given-names:N\\nsurname:Heinemann;given-names:M\\nREF\\nFEMS Yeast Res\\nref\\nDifferential glucose repression in common yeast strains in response to HXK2 deletion\\nsurname:Ha;given-names:SJ\\nsurname:Galazka;given-names:JM\\nsurname:Joong Oh;given-names:E\\nsurname:Kordic;given-names:V\\nsurname:Kim;given-names:H\\nsurname:Jin;given-names:YS\\nsurname:Cate;given-names:JH\\nREF\\nMetab Eng\\nref\\nEnergetic benefits and rapid cellobiose fermentation by Saccharomyces cerevisiae expressing cellobiose phosphorylase and mutant cellodextrin transporters\\nsurname:Ha;given-names:SJ\\nsurname:Kim;given-names:H\\nsurname:Lin;given-names:Y\\nsurname:Jang;given-names:MU\\nsurname:Galazka;given-names:JM\\nsurname:Kim;given-names:TJ\\nsurname:Cate;given-names:JH\\nsurname:Jin;given-names:YS\\nREF\\nAppl Environ Microbiol\\nref\\nSingle amino acid substitutions in HXT2.4 from Scheffersomyces stipitis lead to improved cellobiose fermentation by engineered Saccharomyces cerevisiae\\nsurname:Lian;given-names:J\\nsurname:Li;given-names:Y\\nsurname:HamediRad;given-names:M\\nsurname:Zhao;given-names:H\\nREF\\nBiotechnol Bioeng\\nref\\nsurname:Sadie;given-names:CJ\\nsurname:Rose;given-names:SH\\nsurname:den Haan;given-names:R\\nsurname:van Zyl;given-names:WH\\nREF\\nAppl Microbiol Biotechnol\\nref\\nCo-expression of a cellobiose phosphorylase and lactose permease enables intracellular cellobiose utilisation by Saccharomyces cerevisiae\\nsurname:Aeling;given-names:KA\\nsurname:Salmon;given-names:KA\\nsurname:Laplaza;given-names:JM\\nsurname:Li;given-names:L\\nsurname:Headman;given-names:JR\\nsurname:Hutagalung;given-names:AH\\nsurname:Picataggio;given-names:S\\nREF\\nJ Ind Microbiol Biotechnol\\nref\\nCo-fermentation of xylose and cellobiose by an engineered Saccharomyces cerevisiae\\nsurname:Pitkanen;given-names:J\\nREF\\nImpact of Xylose and Mannose on Central Metabolism of Yeast Saccharomyces cerevisiae\\nref\\nsurname:Rodriguez;given-names:A\\nsurname:De La Cera;given-names:T\\nsurname:Herrero;given-names:P\\nsurname:Moreno;given-names:F\\nREF\\nBiochem J\\nref\\nThe hexokinase 2 protein regulates the expression of the GLK1, HXK1 and HXK2 genes of Saccharomyces cerevisiae\\nsurname:Carrillo;given-names:E\\nsurname:Ben-Ari;given-names:G\\nsurname:Wildenhain;given-names:J\\nsurname:Tyers;given-names:M\\nsurname:Grammentz;given-names:D\\nsurname:Lee;given-names:TA\\nREF\\nMol Biol Cell\\nref\\nCharacterizing the roles of Met31 and Met32 in coordinating Met4-activated transcription in the absence of Met30\\nsurname:Nishimura;given-names:H\\nsurname:Kawasaki;given-names:Y\\nsurname:Kaneko;given-names:Y\\nsurname:Nosaka;given-names:K\\nsurname:Iwashima;given-names:A\\nREF\\nFEBS Lett\\nref\\nCloning and characteristics of a positive regulatory gene, THI2 (PHO6), of thiamin biosynthesis in Saccharomyces cerevisiae\\nsurname:Zampar;given-names:GG\\nsurname:Kummel;given-names:A\\nsurname:Ewald;given-names:J\\nsurname:Jol;given-names:S\\nsurname:Niebel;given-names:B\\nsurname:Picotti;given-names:P\\nsurname:Aebersold;given-names:R\\nsurname:Sauer;given-names:U\\nsurname:Zamboni;given-names:N\\nsurname:Heinemann;given-names:M\\nREF\\nMol Syst Biol\\nref\\nsurname:Li;given-names:L\\nsurname:Bagley;given-names:D\\nsurname:Ward;given-names:DM\\nsurname:Kaplan;given-names:J\\nREF\\nMol Cell Biol\\nref\\nYap5 is an iron-responsive transcriptional activator that regulates vacuolar iron storage in yeast\\nsurname:Andre;given-names:B\\nREF\\nMol Gen Genet\\nref\\nThe UGA3 gene regulating the GABA catabolic pathway in Saccharomyces cerevisiae codes for a putative zinc-finger protein acting on RNA amount\\nsurname:Lorenz;given-names:MC\\nsurname:Heitman;given-names:J\\nREF\\nGenetics\\nref\\nRegulators of pseudohyphal differentiation in Saccharomyces cerevisiae identified through multicopy suppressor analysis in ammonium permease mutant strains\\nsurname:Kurihara;given-names:LJ\\nsurname:Stewart;given-names:BG\\nsurname:Gammie;given-names:AE\\nsurname:Rose;given-names:MD\\nREF\\nMol Cell Biol\\nref\\nKar4p, a karyogamy-specific component of the yeast pheromone response pathway\\nsurname:Westholm;given-names:JO\\nsurname:Nordberg;given-names:N\\nsurname:Muren;given-names:E\\nsurname:Ameur;given-names:A\\nsurname:Komorowski;given-names:J\\nsurname:Ronne;given-names:H\\nREF\\nBMC Genomics\\nref\\nsurname:Vincent;given-names:O\\nsurname:Carlson;given-names:M\\nREF\\nEmbo J\\nref\\nSip4, a Snf1 kinase-dependent transcriptional activator, binds to the carbon source-responsive element of gluconeogenic genes\\nsurname:Charron;given-names:MJ\\nsurname:Dubin;given-names:RA\\nsurname:Michels;given-names:CA\\nREF\\nMol Cell Biol\\nref\\nStructural and functional analysis of the MAL1 locus of Saccharomyces cerevisiae\\nsurname:Magasanik;given-names:B\\nsurname:Kaiser;given-names:CA\\nREF\\nGene\\nref\\nNitrogen regulation in Saccharomyces cerevisiae\\nsurname:Hofman-Bang;given-names:J\\nREF\\nMol Biotechnol\\nref\\nNitrogen catabolite repression in Saccharomyces cerevisiae\\nsurname:Bourot;given-names:S\\nsurname:Karst;given-names:F\\nREF\\nGene\\nref\\nIsolation and characterization of the Saccharomyces cerevisiae SUT1 gene involved in sterol uptake\\nsurname:Regnacq;given-names:M\\nsurname:Alimardani;given-names:P\\nsurname:El Moudni;given-names:B\\nsurname:Berges;given-names:T\\nREF\\nMol Microbiol\\nref\\nSUT1p interaction with Cyc8p(Ssn6p) relieves hypoxic genes from Cyc8p-Tup1p repression in Saccharomyces cerevisiae\\nsurname:Mai;given-names:B\\nsurname:Breeden;given-names:L\\nREF\\nMol Cell Biol\\nref\\nXbp1, a stress-induced transcriptional repressor of the Saccharomyces cerevisiae Swi4/Mbp1 family\\nsurname:Deng;given-names:Y\\nsurname:He;given-names:T\\nsurname:Wu;given-names:Y\\nsurname:Vanka;given-names:P\\nsurname:Yang;given-names:G\\nsurname:Huang;given-names:Y\\nsurname:Yao;given-names:H\\nsurname:Brown;given-names:SJ\\nREF\\nInt J Mol Med\\nref\\nComputationally analyzing the possible biological function of YJL103C-an ORF potentially involved in the regulation of energy process in yeast\\nsurname:Hlynialuk;given-names:C\\nsurname:Schierholtz;given-names:R\\nsurname:Vernooy;given-names:A\\nsurname:van der Merwe;given-names:G\\nREF\\nMicrobiology\\nref\\nNsf1/Ypl230w participates in transcriptional activation during non-fermentative growth and in response to salt stress in Saccharomyces cerevisiae\\nsurname:Ratnakumar;given-names:S\\nsurname:Kacherovsky;given-names:N\\nsurname:Arms;given-names:E\\nsurname:Young;given-names:ET\\nREF\\nGenetics\\nref\\nSnf1 controls the activity of adr1 through dephosphorylation of Ser230\\nsurname:Young;given-names:ET\\nsurname:Dombek;given-names:KM\\nsurname:Tachibana;given-names:C\\nsurname:Ideker;given-names:T\\nREF\\nJ Biol Chem\\nref\\nMultiple pathways are co-regulated by the protein kinase Snf1 and the transcription factors Adr1 and Cat8\\nsurname:Schuller;given-names:HJ\\nREF\\nCurr Genet\\nref\\nTranscriptional control of nonfermentative metabolism in the yeast Saccharomyces cerevisiae\\nsurname:Rahner;given-names:A\\nsurname:Scholer;given-names:A\\nsurname:Martens;given-names:E\\nsurname:Gollwitzer;given-names:B\\nsurname:Schuller;given-names:HJ\\nREF\\nNucleic Acids Res\\nref\\nDual influence of the yeast Cat1p (Snf1p) protein kinase on carbon source-dependent transcriptional activation of gluconeogenic genes by the regulatory gene CAT8\\nsurname:Forsburg;given-names:SL\\nsurname:Guarente;given-names:L\\nREF\\nGenes Dev\\nref\\nsurname:Brons;given-names:JF\\nsurname:De Jong;given-names:M\\nsurname:Valens;given-names:M\\nsurname:Grivell;given-names:LA\\nsurname:Bolotin-Fukuhara;given-names:M\\nsurname:Blom;given-names:J\\nREF\\nYeast\\nref\\nDissection of the promoter of the HAP4 gene in S. cerevisiae unveils a complex regulatory framework of transcriptional regulation\\nsurname:Hosaka;given-names:K\\nsurname:Nikawa;given-names:J\\nsurname:Kodaki;given-names:T\\nsurname:Yamashita;given-names:S\\nREF\\nJ Biochem\\nref\\nA dominant mutation that alters the regulation of INO1 expression in Saccharomyces cerevisiae\\nsurname:Kim;given-names:SR\\nsurname:Skerker;given-names:JM\\nsurname:Kang;given-names:W\\nsurname:Lesmana;given-names:A\\nsurname:Wei;given-names:N\\nsurname:Arkin;given-names:AP\\nsurname:Jin;given-names:YS\\nREF\\nPLoS One\\nref\\nsurname:Wei;given-names:N\\nsurname:Quarterman;given-names:J\\nsurname:Kim;given-names:SR\\nsurname:Cate;given-names:JH\\nsurname:Jin;given-names:YS\\nREF\\nNat Commun\\nref\\nEnhanced biofuel production through coupled acetic acid and xylose consumption by engineered yeast\\nsurname:Fiaux;given-names:J\\nsurname:Cakar;given-names:ZP\\nsurname:Sonderegger;given-names:M\\nsurname:Wuthrich;given-names:K\\nsurname:Szyperski;given-names:T\\nsurname:Sauer;given-names:U\\nREF\\nEukaryot Cell\\nref\\nMetabolic-flux profiling of the yeasts Saccharomyces cerevisiae and Pichia stipitis\\nsurname:Blazeck;given-names:J\\nsurname:Alper;given-names:HS\\nREF\\nBiotechnol J\\nref\\nPromoter engineering: recent advances in controlling transcription at the most fundamental level\\nsurname:Zhou;given-names:H\\nsurname:Cheng;given-names:JS\\nsurname:Wang;given-names:BL\\nsurname:Fink;given-names:GR\\nsurname:Stephanopoulos;given-names:G\\nREF\\nMetab Eng\\nref\\nsurname:Scalcinati;given-names:G\\nsurname:Otero;given-names:JM\\nsurname:Van Vleet;given-names:JR\\nsurname:Jeffries;given-names:TW\\nsurname:Olsson;given-names:L\\nsurname:Nielsen;given-names:J\\nREF\\nFEMS Yeast Res\\nref\\nEvolutionary engineering of Saccharomyces cerevisiae for efficient aerobic xylose consumption\\nsurname:Versele;given-names:M\\nsurname:de Winde;given-names:JH\\nsurname:Thevelein;given-names:JM\\nREF\\nEmbo J\\nref\\nA novel regulator of G protein signalling in yeast, Rgs2, downregulates glucose-activation of the cAMP pathway through direct inhibition of Gpa2\\nsurname:Ma;given-names:P\\nsurname:Wera;given-names:S\\nsurname:Van Dijck;given-names:P\\nsurname:Thevelein;given-names:JM\\nREF\\nMol Biol Cell\\nref\\nThe PDE1-encoded low-affinity phosphodiesterase in the yeast Saccharomyces cerevisiae has a specific function in controlling agonist-induced cAMP signaling\\nsurname:Kaniak;given-names:A\\nsurname:Xue;given-names:Z\\nsurname:Macool;given-names:D\\nsurname:Kim;given-names:JH\\nsurname:Johnston;given-names:M\\nREF\\nEukaryot Cell\\nref\\nRegulatory network connecting two glucose signal transduction pathways in Saccharomyces cerevisiae\\nsurname:Salusjarvi;given-names:L\\nsurname:Kankainen;given-names:M\\nsurname:Soliymani;given-names:R\\nsurname:Pitkanen;given-names:JP\\nsurname:Penttila;given-names:M\\nsurname:Ruohonen;given-names:L\\nREF\\nMicrob Cell Factories\\nref\\nRegulation of xylose metabolism in recombinant Saccharomyces cerevisiae\\nsurname:Jin;given-names:YS\\nsurname:Laplaza;given-names:JM\\nsurname:Jeffries;given-names:TW\\nREF\\nAppl Environ Microbiol\\nref\\nSaccharomyces cerevisiae engineered for xylose metabolism exhibits a respiratory response\\nsurname:Pelaez;given-names:R\\nsurname:Herrero;given-names:P\\nsurname:Moreno;given-names:F\\nREF\\nBiochem J\\nref\\nFunctional domains of yeast hexokinase 2\\nsurname:DelaFuente;given-names:G\\nREF\\nEur J Biochem\\nref\\nSpecific inactivation of yeast hexokinase induced by xylose in the presence of a phosphoryl donor substrate\\nsurname:Fernandez;given-names:R\\nsurname:Herrero;given-names:P\\nsurname:Moreno;given-names:F\\nREF\\nJ Gen Microbiol\\nref\\nInhibition and inactivation of glucose-phosphorylating enzymes from Saccharomyces cerevisiae by D-xylose\\nsurname:Bergdahl;given-names:B\\nsurname:Sandstrom;given-names:AG\\nsurname:Borgstrom;given-names:C\\nsurname:Boonyawan;given-names:T\\nsurname:van Niel;given-names:EW\\nsurname:Gorwa-Grauslund;given-names:MF\\nREF\\nPLoS One\\nref\\nEngineering yeast hexokinase 2 for improved tolerance toward xylose-induced inactivation\\nsurname:Ness;given-names:F\\nsurname:Bourot;given-names:S\\nsurname:Regnacq;given-names:M\\nsurname:Spagnoli;given-names:R\\nsurname:Berges;given-names:T\\nsurname:Karst;given-names:F\\nREF\\nEur J Biochem\\nref\\nSUT1 is a putative Zn[II]2Cys6-transcription factor whose upregulation enhances both sterol uptake and synthesis in aerobically growing Saccharomyces cerevisiae cells\\nsurname:Alimardani;given-names:P\\nsurname:Regnacq;given-names:M\\nsurname:Moreau-Vauzelle;given-names:C\\nsurname:Ferreira;given-names:T\\nsurname:Rossignol;given-names:T\\nsurname:Blondin;given-names:B\\nsurname:Berges;given-names:T\\nREF\\nBiochem J\\nref\\nSUT1-promoted sterol uptake involves the ABC transporter Aus1 and the mannoprotein Dan1 whose synergistic action is sufficient for this process\\nsurname:Foster;given-names:HA\\nsurname:Cui;given-names:M\\nsurname:Naveenathayalan;given-names:A\\nsurname:Unden;given-names:H\\nsurname:Schwanbeck;given-names:R\\nsurname:Hofken;given-names:T\\nREF\\nEukaryot Cell\\nref\\nThe zinc cluster protein Sut1 contributes to filamentation in Saccharomyces cerevisiae\\nsurname:van Maris;given-names:AJ\\nsurname:Bakker;given-names:BM\\nsurname:Brandt;given-names:M\\nsurname:Boorsma;given-names:A\\nsurname:de Mattos;given-names:MJT\\nsurname:Grivell;given-names:LA\\nsurname:Pronk;given-names:JT\\nsurname:Blom;given-names:J\\nREF\\nFEMS Yeast Res\\nref\\nModulating the distribution of fluxes among respiration and fermentation by overexpression of HAP4 in Saccharomyces cerevisiae\\nsurname:Soontorngun;given-names:N\\nsurname:Baramee;given-names:S\\nsurname:Tangsombatvichit;given-names:C\\nsurname:Thepnok;given-names:P\\nsurname:Cheevadhanarak;given-names:S\\nsurname:Robert;given-names:F\\nsurname:Turcotte;given-names:B\\nREF\\nBiochem Biophys Res Commun\\nref\\nGenome-wide location analysis reveals an important overlap between the targets of the yeast transcriptional regulators Rds2 and Adr1\\nsurname:Bertram;given-names:PG\\nsurname:Choi;given-names:JH\\nsurname:Carvalho;given-names:J\\nsurname:Chan;given-names:TF\\nsurname:Ai;given-names:W\\nsurname:Zheng;given-names:XF\\nREF\\nMol Cell Biol\\nref\\nConvergence of TOR-nitrogen and Snf1-glucose signaling pathways onto Gln3\\nsurname:Mittal;given-names:N\\nsurname:Roy;given-names:N\\nsurname:Babu;given-names:MM\\nsurname:Janga;given-names:SC\\nREF\\nProc Natl Acad Sci U S A\\nref\\nDissecting the expression dynamics of RNA-binding proteins in posttranscriptional regulatory networks\\nsurname:Gerashchenko;given-names:MV\\nsurname:Lobanov;given-names:AV\\nsurname:Gladyshev;given-names:VN\\nREF\\nProc Natl Acad Sci U S A\\nref\\nGenome-wide ribosome profiling reveals complex translational regulation in response to oxidative stress\\nsurname:Bergdahl;given-names:B\\nsurname:Heer;given-names:D\\nsurname:Sauer;given-names:U\\nsurname:Hahn-Hagerdal;given-names:B\\nsurname:van Niel;given-names:EW\\nREF\\nBiotechnol Biofuels\\nref\\nDynamic metabolomics differentiates between carbon and energy starvation in recombinant Saccharomyces cerevisiae fermenting xylose\\nsurname:Unger;given-names:T\\nsurname:Jacobovitch;given-names:Y\\nsurname:Dantes;given-names:A\\nsurname:Bernheim;given-names:R\\nsurname:Peleg;given-names:Y\\nREF\\nJ Struct Biol\\nref\\nApplications of the restriction free (RF) cloning procedure for molecular manipulations and protein expression\\nsurname:Tian;given-names:C\\nsurname:Beeson;given-names:WT\\nsurname:Iavarone;given-names:AT\\nsurname:Sun;given-names:J\\nsurname:Marletta;given-names:MA\\nsurname:Cate;given-names:JH\\nsurname:Glass;given-names:NL\\nREF\\nProc Natl Acad Sci U S A\\nref\\nsurname:Christianson;given-names:TW\\nsurname:Sikorski;given-names:RS\\nsurname:Dante;given-names:M\\nsurname:Shero;given-names:JH\\nsurname:Hieter;given-names:P\\nREF\\nGene\\nref\\nMultifunctional yeast high-copy-number shuttle vectors\\nsurname:Sikorski;given-names:RS\\nsurname:Hieter;given-names:P\\nREF\\nGenetics\\nref\\nA system of shuttle vectors and yeast host strains designed for efficient manipulation of DNA in Saccharomyces cerevisiae\\nsurname:Guldener;given-names:U\\nsurname:Heck;given-names:S\\nsurname:Fielder;given-names:T\\nsurname:Beinhauer;given-names:J\\nsurname:Hegemann;given-names:JH\\nREF\\nNucleic Acids Res\\nref\\nA new efficient gene disruption cassette for repeated use in budding yeast\\nsurname:Darby;given-names:RA\\nsurname:Cartwright;given-names:SP\\nsurname:Dilworth;given-names:MV\\nsurname:Bill;given-names:RM\\nREF\\nMethods Mol Biol\\nref\\nWhich yeast species shall I choose? Saccharomyces cerevisiae versus Pichia pastoris (review)\\nsurname:Hanscho;given-names:M\\nsurname:Ruckerbauer;given-names:DE\\nsurname:Chauhan;given-names:N\\nsurname:Hofbauer;given-names:HF\\nsurname:Krahulec;given-names:S\\nsurname:Nidetzky;given-names:B\\nsurname:Kohlwein;given-names:SD\\nsurname:Zanghellini;given-names:J\\nsurname:Natter;given-names:K\\nREF\\nFEMS Yeast Res\\nref\\nREF\\nref\\nRefSeq: NCBI Reference Sequence Database. [http://www.ncbi.nlm.nih.gov/refseq/]\\nsurname:Bolstad;given-names:BM\\nsurname:Irizarry;given-names:RA\\nsurname:Astrand;given-names:M\\nsurname:Speed;given-names:TP\\nREF\\nBioinformatics\\nref\\nA comparison of normalization methods for high density oligonucleotide array data based on variance and bias\\nREF\\nref\\nGene Expression Omnibus. [http://www.ncbi.nlm.nih.gov/geo/]\\nREF\\nref\\nGene Ontology Consortium: Download Annotations. [http://www.geneontology.org/GO.current.annotations.shtml]\\nsurname:Baggerly;given-names:KA\\nsurname:Deng;given-names:L\\nsurname:Morris;given-names:JS\\nsurname:Aldaz;given-names:CM\\n10.1093/bioinformatics/btg173\\nREF\\nBioinformatics\\nref\\nDifferential expression in SAGE: accounting for normal between-library variation\\nsurname:Johnson;given-names:VE\\nREF\\nProc Natl Acad Sci U S A\\nref\\nRevised standards for statistical evidence\\nREF\\nref\\nFunSpec: a web-based cluster interpreter for yeast. [http://funspec.med.utoronto.ca/]\\nsurname:Robinson;given-names:MD\\nsurname:Grigull;given-names:J\\nsurname:Mohammad;given-names:N\\nsurname:Hughes;given-names:TR\\nREF\\nBMC Bioinformatics\\nref\\nFunSpec: a web-based cluster interpreter for yeast\\nsurname:David;given-names:F\\nsurname:Berger;given-names:A\\nsurname:Hansch;given-names:R\\nsurname:Rohde;given-names:M\\nsurname:Franco-Lara;given-names:E\\nREF\\nMicrob Cell Factories\\nref\\nSingle cell analysis applied to antibody fragment production with Bacillus megaterium: development of advanced physiology and bioprocess state estimation tools\\nsurname:Wisselink;given-names:HW\\nsurname:Toirkens;given-names:MJ\\nsurname:del Rosario Franco Berriel;given-names:M\\nsurname:Winkler;given-names:AA\\nsurname:van Dijken;given-names:JP\\nsurname:Pronk;given-names:JT\\nsurname:van Maris;given-names:AJ\\nREF\\nAppl Environ Microbiol\\nref\\nEngineering of Saccharomyces cerevisiae for efficient anaerobic alcoholic fermentation of L-arabinose\\nsurname:Motulsky;given-names:HJ\\nsurname:Christopoulos;given-names:A\\nREF\\nFitting models to biological data using linear and nonlinear regression. A practical guide to curve fitting\\nref\\n',\n",
       " '23671666': 'BioC-API\\ncollection.key\\nCC BY\\nHoxa2 Controls Pcp4 Expression\\nThis is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are properly credited.\\nsurname:Anderson;given-names:Megan\\nsurname:Amin;given-names:Shilu\\nsurname:Luise;given-names:Fabiana\\nsurname:Zeef;given-names:Leo\\nsurname:Bobola;given-names:Nicoletta\\nsurname:Mallo;given-names:Moises\\nTITLE\\nfront\\nABSTRACT\\nabstract\\nBranchial arches are externally visible tissue bands in the head region of all vertebrate embryos. Although initially formed from similar components, each arch will give rise to different head and neck structures. In a screen designed to characterize the molecular control of branchial arch identity in mouse, we identified Pcp4 as a second branchial arch-specific molecular signature. We further show that the transcription factor Hoxa2 binds to Pcp4 chromatin and regulates Pcp4 expression in the second arch. Hoxa2 is also sufficient to induce Pcp4 expression in anterior first arch cells, which are Pcp4-negative.\\nINTRO\\ntitle_1\\nIntroduction\\nINTRO\\nparagraph\\nBranchial arches are transient, repetitive structures of the vertebrate embryo, in which cells of the cranial neural crest and mesoderm are encapsulated by epithelia. There are five to six pairs of arches in amniotes, labeled according to their position along the embryo antero-posterior axis; the first corresponds to the most anterior arch (lying below the forebrain) and the sixth to the most posterior (close to the developing heart). Each of the arches shares a similar architecture and ability to form skeletal elements, their associated muscles, blood supply and nerves, but has its own molecular identity and contributes to diverse head and neck structures. Early development of the branchial arches is instructed by signaling molecules and transcription factors. Dlx transcription factors regulate proximo-distal patterning within each branchial arch. Hox transcription factors control branchial arch identity. A classic example of molecular control of inter-branchial arch identity concerns the first and the second branchial arch. The second arch is the most anterior of the branchial arches to express Hox genes, and mainly expresses Hoxa2 and Hoxb2; the first arch does not express any Hox genes. Hoxa2 specifies second branchial arch (IIBA) identity: the second arch follows a first arch fate in the absence of Hoxa2, and IIBA-skeletal derivatives are replaced by typical first branchial arch (IBA) skeletal elements in a mirror image configuration.\\nINTRO\\nparagraph\\nPcp4 is a small calmodulin (CaM)-binding protein. It affects the rates of association and dissociation of Ca (2+) from CaM and can attenuate the activity of a number of CaM-dependent enzymes, including CaM kinase II. Initially identified as highly abundant in Purkinje cells, it is highly expressed in the developing neural system and has the ability to inhibit cell death and to promote neurite outgrowth in vitro . Changes in the synaptic plasticity of cerebellar Purkinje cells are observed in Pcp4-null mice. Non-neural effects of Pcp4 have also been described, including a positive role in the osteoblastic differentiation of bone marrow stem cells and in the differentiation of B cells.\\nINTRO\\nparagraph\\nTo understand the molecular control of inter-branchial arch identity, we forced expression of Hoxa2, the main determinant of second arch fate, in first arch cells and analyzed changes in global expression. By intersecting the data obtained with related data sets, we uncover novel genes regulated by Hoxa2. In addition, we identify the gene encoding for Pcp4 as highly differentially expressed in first and second arch cells. We show that, while anterior first arch cells are Pcp4-negative, Pcp4 is enriched in second arch cells, and that Pcp4 expression is directly controlled by the transcription factor Hoxa2 in vivo.\\nRESULTS\\ntitle_1\\nResults and Discussion\\nRESULTS\\nparagraph\\nThe transcription factor Hoxa2 specifies second arch identity: the presence of Hoxa2 in IIBA cells modifies the similar ground state shared by first branchial arch (IBA) and IIBA cells and instructs a second arch developmental fate (Fig. 1A). To understand Hoxa2 molecular control of branchial arch development, we forced the expression of Hoxa2 in Hox-negative IBA cells (Fig. 1A). Cells isolated from the branchial arches maintain their molecular identity when grown in vitro . First arch cells were transfected with Hoxa2-IRES-GFP or GFP alone and sorted using FACS. RNA was extracted from GFP-positive cells and the expression profile of Hoxa2-positive IBA cells was compared to control, Hox-negative cells, to identify differentially expressed genes (Fig. 1B). We identified 59 genes highly differentially expressed upon Hoxa2 gain of function in IBA cells (fold change >=2.5; pvalue <0.05), distributed in 36 upregulated and 23 downregulated genes (Table S1). We intersected the genes differentially regulated in Hoxa2 gain of function in IBA cells with two complementary sets of data, the genes bound by Hoxa2 in the IIBA (Hoxa2 ChIP-seq) and the genes differently regulated in wild-type versus Hoxa2 loss of function mutant IIBA; results are summarized in Table 1. We observed the largest overlap between genes upregulated in Hoxa2 gain of function and downregulated in Hoxa2 loss of function. This group, which contains genes activated by Hoxa2, is frequently associated with Hoxa2 bound regions (Fig. 1C, II quadrant; Table S2, segment S1). Conversely, most of the genes upregulated in the Hoxa2 loss of function mutant (repressed by Hoxa2) are not affected by Hoxa2 overexpression and are seldom associated to Hoxa2 binding events, suggesting these changes may be indirect effects of Hoxa2 absence (Fig. 1C, I and IV quadrant; Table S2, segment S5). Interestingly a total of 328 Affymetrix probes (corresponding to 255 unique genes) measured expression changes exclusively in Hoxa2 gain of function, but not in the Hoxa2 mutant IIBA (139 upregulated; 188 probes downregulated; Table S2, segment S2 and S7). They included genes displaying high expression changes upon Hoxa2 gain of function. DAVID analysis clustered the corresponding genes into functional categories closely related to the biological processes regulated by Hox proteins in embryonic development (Fig. 1D). These changes may result from Hoxa2 functioning in a different biological context, despite the common developmental ground state of IBA and IIBA, and correspond therefore to non-physiological Hoxa2 targets. The IIBA expresses Hoxb2 (and to a lesser extent Hoxa1), which may potentially compensate Hoxa2 loss of function and mask changes in the expression of Hoxa2 targets. Gain of function of Hoxa2 in IBA cells provides therefore a complementary system to identify Hoxa2 direct targets in branchial arches mesenchymal cells. Finally, few genes displayed changes in expression with the same sign in Hoxa2 gain and loss of function (Table S2, segments S3 and S6).\\nFIG\\nfig_title_caption\\n\\nHoxa2 gain of function in IBA cells.\\nFIG\\nfig_caption\\nA, In situ hybridization using Hoxa2 probe shows Hoxa2 is mainly expressed in the IIBA (enclosed in the dotted line) and expression is excluded from the IBA (arrow). B, Schematic representation of the experiment: cells isolated from IBA are grown in vitro and transfected with Hoxa2-IRES-GFP (or GFP alone, control). RNA is extracted from GFP-positive cells and analyzed by microarray. C, Pairwise comparison of microarray experiments for Hoxa2 loss of function (x-axis) versus Hoxa2 gain of function (y-axis). Data are plotted as fold change against control in each case (axes in logarithmic scale base 2). Genes in red are nearby a Hoxa2-bound region in ChIP-seq (closest two genes to Hoxa2-bound region were included). D, Functional annotation of genes responsive to Hoxa2 gain of function only. The top over-represented categories are shown; the length of the bars corresponds to the P-values on the x-axis.\\nTABLE\\ntable_title_caption\\nTABLE\\ntable\\n<?xml version=\"1.0\" encoding=\"UTF-8\"?>\\n<table frame=\"hsides\" rules=\"groups\"><colgroup span=\"1\"><col align=\"left\" span=\"1\"/><col align=\"center\" span=\"1\"/><col align=\"center\" span=\"1\"/><col align=\"center\" span=\"1\"/></colgroup><thead><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\"/><td align=\"left\" rowspan=\"1\" colspan=\"1\"/><td align=\"left\" rowspan=\"1\" colspan=\"1\">LOF Down</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">LOF Up</td></tr></thead><tbody><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\"/><td align=\"left\" rowspan=\"1\" colspan=\"1\">\\n<bold>Total</bold>\\n</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">77 (59)</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">30 (19)</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">\\n<bold>GOF Up</bold>\\n</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">151 (118)</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">10 (9)</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">1</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">\\n<bold>GOF Down</bold>\\n</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">191 (150)</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">3 (3)</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">0</td></tr></tbody></table>\\n\\nTABLE\\ntable_footnote\\nThe numbers of Affymetrix probesets with fold change > = 1.75 in either of these experiments are shown; the corresponding unique genes numbers are shown in brackets.\\nRESULTS\\nparagraph\\nThe gene encoding for Pcp4 displayed the highest fold difference in expression in first arch cells transfected with Hoxa2, with low to undetectable level of expression in GFP-positive first arch cells (fold change = 11.5; pvalue = 0.0005). Interestingly, expression of Pcp4 was also highly different when comparing expression profiles of cells isolated from the first and the second arch (NB, data not shown). Since Hoxa2 expression is sufficient to induce Pcp4 expression in IBA cells, we asked whether IIBA cells, which are Hoxa2-positive, endogenously express Pcp4. We analyzed the expression of Pcp4 in branchial arch cells using immunofluorescence: Pcp4 was visualized only in IIBA cells, and was not present in cells derived from the IBA (Fig. 2A-C). We observed a similar distribution in Pcp4 transcripts (Fig. S1).\\nFIG\\nfig_title_caption\\nPcp4 is specifically expressed in IIBA cells.\\nFIG\\nfig_caption\\nA, Head region of a midgestation mouse embryo, with first and second arch highlighted in turquoise and red, respectively. B,C. Immunofluorescence using Pcp4 antibody (green) stains cells isolated from IIBA (B), but not IBA (C). Nuclear staining is blue (DAPI).\\nRESULTS\\nparagraph\\nA survey of Hoxa2 binding events in the IIBA identified binding of Hoxa2 in the first intron of Pcp4 (Fig. 3A). The corresponding peak is included in the top 1% most highly enriched regions after immunoprecipitation with Hoxa2 antibody, which is indicative of strong Hoxa2 binding. The summit region (200-bp window centered on ChIP-seq peak) contains Hox-Pbx and Meis motifs and six TAAT motifs, which correspond to Hoxa2 recognition site (data not shown).\\nFIG\\nfig_title_caption\\nHoxa2 binds Pcp4 in vivo.\\nFIG\\nfig_caption\\nA, High enrichment of the first intron of the Pcp4 gene compared to input (mm9, chr16:96,717,332-96,718,404) in Hoxa2 ChIP-seq. Evolutionary Conserved Regions (ECRs) plot generated by the ECR Browser, comparing the genomic region bound by Hoxa2 between human, chimpanzee, and cow. B, Conventional ChIP on IIBA chromatin confirms enrichment of Hoxa2 to Pcp4. IP10 is a negative control gene. IgG is a non-specific negative control antibody.\\nRESULTS\\nparagraph\\nWe immunoprecipitated IIBA chromatin and found that the region bound by Hoxa2 on Pcp4 intron was highly enriched after immunoprecipitation with a Hoxa2-specific antibody, but not with an unrelated antibody (Fig. 3B). These findings indicate that Hoxa2 is bound to Pcp4 in vivo. Next, we analyzed the expression of Pcp4 in the developing embryo. At E10.5 Pcp4 is expressed in the branchial arch region; this expression corresponds to the developing cranial ganglia and does not appear to be specific for any of the BAs (Fig. 4A, arrow). In addition, Pcp4 is strongly expressed in the dorsal root ganglia (Fig. 4A, arrowhead), and in the developing eye. One day later (E11.5), the expression of Pcp4 in the branchial area extends into the IIBA (Fig. 4B, arrowhead), and is contained within Hoxa2 domain of expression in E11.5 embryos. Strong expression is still observed in the cranial ganglia (Fig. 4B, arrow) and expression intensifies in the brain. We analyzed Pcp4 expression in E11.5 IIBA isolated from wild type (+/+) and Hoxa2 mutant (-/-). We found strong Pcp4 expression in wild type IIBA, but no expression in Hoxa2 mutant IIBA (Fig. 4C). These results indicate that the expression of Pcp4, observed in the IIBA, is dependent on Hoxa2.\\nFIG\\nfig_title_caption\\nHoxa2 regulates Pcp4 in the branchial arches.\\nFIG\\nfig_caption\\nA,B. Whole mount ISH on E10.5 (A) and E11.5 (B) wild-type embryos, using Pcp4 probe. Pcp4 is expressed in the cranial ganglia (arrow, A-B) and dorsal root ganglia (arrowhead in A). One day later Pcp4 expression intensifies in the proximal area of the IIBA (arrowhead in B) in close proximity to the otic vesicle, and in the brain, and caudal expression disappears. C, Semiquantitative RT-PCR in wild type (+/+) and Hoxa2 mutant (-/-) E11.5 IIBA cDNA. Pcp4 is expressed in wild type and not Hoxa2 mutant IIBA cells. Gapdh is a positive control gene and Hoxa2 expression is confirmed in wild type and not Hoxa2 mutant IIBA cells. Ov, otic vesicle.\\nRESULTS\\nparagraph\\nTaken together, the observations that a) Hoxa2 is strongly bound to Pcp4 intron in developing (E11.5) IIBA; b) Hoxa2 is sufficient to induce Pcp4 expression in IBA mesenchymal cells; and c) Hoxa2 is required for expression of Pcp4 in the IIBA, indicate that Hoxa2 is a main regulator of Pcp4 in embryonic development. Additional experiments will be required to understand the function of Pcp4 in the development of the second branchial arch. Local modulation of calcium signal transduction is a key determinant of beak shape, linking control of Ca (2+) signaling to variations in craniofacial morphology. The control of Pcp4 by Hoxa2, leading to its expression in the second arch, and the ability of Pcp4 to modulate Ca (2+) signaling suggest that acquiring IIBA identity may partly rely on a differential ability of IIBA cells to control Ca (2+) levels.\\nMETHODS\\ntitle_1\\nMaterials and Methods\\nMETHODS\\ntitle_2\\nMouse Lines, Cell Culture and Infection\\nMETHODS\\nparagraph\\n Hoxa2 mutant mice were described previously. Branchial arches were isolated from E11.5 mouse embryos, obtained by CD1 mice time-matings. Branchial arches were dissociated into single cells using trypsin and a cell strainer (BD Falcon). A total of 200,000 cells were plated on a 24-well plate and grown in DMEM 10% FCS, at 37 C, 5% CO2. Ecotropic-Phoenix cells were transfected with Fugene and pMYs-IRES-GFP (Cell Biolabs, San Diego, CA) or pMYs-Hoxa2-IRES-GFP (containing Hoxa2 in front of the IRES), and branchial arch cells were infected using supernatants from Ecotropic-Phoenix packaging cells, supplemented with polybrene at 2 microg ml/1 final concentration. After 72 hours cells were sorted by retroviral gene expression of fluorescent proteins using flow cytometry. Animal experiments were carried out under ASPA 1986.\\nMETHODS\\ntitle_2\\nMicroarrays\\nMETHODS\\nparagraph\\nRNA was extracted from sorted cells using Trizol (Life Technologies). Labeled targets were generated from total RNA obtained in two independent experiments, using the 3\\' IVT Express Kit and hybridized to GeneChip Mouse Genome 430 2.0 arrays. Background correction, quantile normalization, and gene expression analysis were performed using RMA in Bioconductor. Principal component analysis (PCA) was performed with Partek Genomics Solution (version 6.5, Copyright 2010, Partek Inc., St. Charles, MO, USA). Differential expression analysis was performed using Limma using the functions lmFit and eBayes. The microarray expression data has been submitted to ArrayExpress (ArrayExpress accession: E-MEXP-3837).\\nMETHODS\\ntitle_2\\nImmunofluorescence\\nMETHODS\\nparagraph\\nBranchial arch cells were isolated as described above and grown on 8-well culture slides (BD Falcon) for 24 hours, fixed and incubated with Pcp4 antibody (Sigma) diluted 1:200 and visualized with AlexaFluor 488 goat anti-rabbit (Invitrogen).\\nMETHODS\\ntitle_2\\nChromatin Immunoprecipitation (ChIP)\\nMETHODS\\nparagraph\\nChIP was performed as described using IIBAs isolated from E11.5 CD1 mouse embryos (24 pairs). The summit of Hoxa2-bound region on Pcp4 gene was amplified using forward 5\\'-ACTCTGTCAGAGCAATTACCA-3 and reverse 5\\'-CTTGTAGGCACTAACTGGGA-3 primers.\\nMETHODS\\ntitle_2\\nExpression Analysis\\nMETHODS\\nparagraph\\nSecond branchial arches of E11.5 embryos from Hoxa2+/- intercrosses were dissected out and snap-frozen in dry ice. After genotyping the embryos, pools were made with the wild type and Hoxa2-/- branchial arches and total RNA was extracted using Trizol. RT-PCR was performed as described, and cDNA was subjected to 24 and 28 cycles of amplification using Hoxa2 primers and Pcp4 F 5\\'-ATGAGTGAGAGACAAGTGCC\\'-3; Pcp4R 5\\'-CTAGGACTGTGATCCTGCCT-3\\', respectively. Whole mount in situ hybridization was performed as described, using Hoxa2 , and Pcp4 probe, amplified from IIBA cDNA using the primers above.\\nSUPPL\\ntitle_1\\nSupporting Information\\nREF\\ntitle\\nReferences\\nREF\\nref\\nDepew MJ, Lufkin T, Rubenstein JL (2002) Specification of jaw subdivisions by Dlx genes. Science, 298, 381-385.\\nREF\\nref\\nGendron-Maguire M, Mallo M, Zhang M, Gridley T (1993) Hoxa-2 mutant mice exhibit homeotic transformation of skeletal elements derived from cranial neural crest. Cell, 75, 1317-1331.\\nREF\\nref\\nRijli FM, Mark M, Lakkaraju S, Dierich A, Dolle P, Chambon P (1993) A homeotic transformation is generated in the rostral branchial region of the head by disruption of Hoxa-2, which acts as a selector gene. Cell, 75, 1333-1349.\\nREF\\nref\\nBarrow JR, Capecchi MR (1999) Compensatory defects associated with mutations in Hoxa1 restore normal palatogenesis to Hoxa2 mutants. Development, 126, 5011-5026.\\nREF\\nref\\nManley NR, Capecchi MR (1997) Hox group 3 paralogous genes act synergistically in the formation of somitic and neural crest-derived structures. Dev Biol, 192, 274-288.\\nREF\\nref\\nKleerekoper QK, Putkey JA (2009) PEP-19, an intrinsically disordered regulator of calmodulin signaling. J Biol Chem, 284, 7455-7464.\\nREF\\nref\\nJohanson RA, Sarau HM, Foley JJ, Slemmon JR (2000) Calmodulin-binding peptide PEP-19 modulates activation of calmodulin kinase II In situ. J Neurosci, 20, 2860-2866.\\nREF\\nref\\nPutkey JA, Waxham MN, Gaertner TR, Brewer KJ, Goldsmith M, Kubota Y, Kleerekoper QK (2008) Acidic/IQ motif regulator of calmodulin. J Biol Chem, 283, 1401-1410.\\nREF\\nref\\nThomas S, Thiery E, Aflalo R, Vayssettes C, Verney C, Berthuy I, Creau N (2003) PCP4 is highly expressed in ectoderm and particularly in neuroectoderm derivatives during mouse embryogenesis. Gene Expr Patterns, 3, 93-97.\\nREF\\nref\\nErhardt JA, Legos JJ, Johanson RA, Slemmon JR, Wang X (2000) Expression of PEP-19 inhibits apoptosis in PC12 cells. Neuroreport, 11, 3719-3723.\\nREF\\nref\\nHarashima S, Wang Y, Horiuchi T, Seino Y, Inagaki N (2011) Purkinje cell protein 4 positively regulates neurite outgrowth and neurotransmitter release. J Neurosci Res, 89, 1519-1530.\\nREF\\nref\\nWei P, Blundon JA, Rong Y, Zakharenko SS, Morgan JI (2011) Impaired locomotor learning and altered cerebellar synaptic plasticity in pep-19/PCP4-null mice. Mol Cell Biol, 31, 2838-2844.\\nREF\\nref\\nXiao J, Wu Y, Chen R, Lin, Wu L, Tian W, Liu L (2008) Expression of Pcp4 gene during osteogenic differentiation of bone marrow mesenchymal stem cells in vitro. Mol Cell Biochem, 309, 143-150.\\nREF\\nref\\nJacobson AC, Weis JJ, Weis JH (2009) CD21 signaling via C3 regulates Purkinje cell protein 4 expression. Mol Immunol, 46, 1488-1493.\\nREF\\nref\\nBaltzinger M, Ori M, Pasqualetti,M, Nardi I, Rijli FM (2005) Hoxa2 knockdown in Xenopus results in hyoid to mandibular homeosis. Dev Dyn, 234, 858-867.\\nREF\\nref\\nKirilenko P, He G, Mankoo BS, Mallo M, Jones R, Bobola N. (2011) Transient activation of meox1 is an early component of the gene regulatory network downstream of hoxa2. Mol Cell Biol, 31, 1301-1308.\\nREF\\nref\\nDonaldson IJ, Amin S, Hensman JJ, Kutejova E, Rattray M, Lawrence N, Hayes A, Ward CM, Bobola N (2012) Genome-wide occupancy links Hoxa2 to Wnt-beta-catenin signaling in mouse embryonic development. Nucleic Acids Res, 40, 3990-4001.\\nREF\\nref\\nDennis G Jr, Sherman BT, Hosack DA, Yang J, Gao W, Lane HC, Lempicki RA (2003) DAVID: Database for Annotation, Visualization, and Integrated Discovery. Genome Biol, 4, P3.\\nREF\\nref\\nPearson JC, Lemons D, McGinnis W (2005) Modulating Hox gene functions during animal body patterning. Nat Rev Genet, 6, 893-904.\\nREF\\nref\\nKameda Y, Nishimaki T, Takeichi M, Chisaka O (2002) Homeobox gene hoxa3 is essential for the formation of the carotid body in the mouse embryos. Dev Biol, 247, 197-209.\\nREF\\nref\\nKameda Y, Watari-Goshima N, Nishimaki T, Chisaka O. (2003) Disruption of the Hoxa3 homeobox gene results in anomalies of the carotid artery system and the arterial baroreceptors. Cell Tissue Res, 311, 343-352.\\nREF\\nref\\nAbzhanov A, Kuo WP, Hartmann C, Grant BR, Grant PR, Tabin CJ (2006) The calmodulin pathway and evolution of elongated beak morphology in Darwin\\'s finches. Nature, 442, 563-567.\\nREF\\nref\\nBolstad BM, Irizarry RA, Astrand M, and Speed TP (2003) A comparison of normalization methods for high density oligonucleotide array data based on variance and bias. Bioinformatics, 19, 185-193.\\nREF\\nref\\nSmyth GK (2004) Linear models and empirical bayes methods for assessing differential expression in microarray experiments. Stat Appl Genet Mol Biol, 3, Article3.\\nREF\\nref\\nKutejova E, Engist B, Self M, Oliver G, Kirilenko P, Bobola N (2008) Six2 functions redundantly immediately downstream of Hoxa2. Development, 135, 1463-1470.\\nREF\\nref\\nKutejova E, Engist B, Mallo M, Kanzler B, Bobola N (2005) Hoxa2 downregulates Six2 in the neural crest-derived mesenchyme. Development, 132, 469-478.\\nREF\\nref\\nKanzler B, Kuschert SJ, Liu YH, Mallo M (1998) Hoxa-2 restricts the chondrogenic domain and inhibits bone formation during development of the branchial area. Development, 125, 2587-2597.\\nREF\\nref\\nMallo M (1997) Retinoic acid disturbs mouse middle ear development in a stage-dependent fashion. Dev Biol, 184, 175-186.\\nREF\\nref\\nOvcharenko I, Nobrega MA, Loots GG, Stubbs L (2004) ECR Browser: a tool for visualizing and accessing data from comparisons of multiple vertebrate genomes. Nucleic Acids Res, 32, W280-286.\\n',\n",
       " '23079210': \"BioC-API\\ncollection.key\\nCC BY\\nThis is an open access article distributed under the terms of the Creative Commons Attribution License (http://creativecommons.org/licenses/by/2.0), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.\\nsurname:Hardy;given-names:Rowan\\nsurname:Juarez;given-names:Maria\\nsurname:Naylor;given-names:Amy\\nsurname:Tu;given-names:Jinwen\\nsurname:Rabbitt;given-names:Elizabeth H\\nsurname:Filer;given-names:Andrew\\nsurname:Stewart;given-names:Paul M\\nsurname:Buckley;given-names:Christopher D\\nsurname:Raza;given-names:Karim\\nsurname:Cooper;given-names:Mark S\\nTITLE\\nfront\\nSynovial DKK1 expression is regulated by local glucocorticoid metabolism in inflammatory arthritis\\nABSTRACT\\nabstract_title_1\\nIntroduction\\nABSTRACT\\nabstract\\nInflammatory arthritis is associated with increased bone resorption and suppressed bone formation. The Wnt antagonist dickkopf-1 (DKK1) is secreted by synovial fibroblasts in response to inflammation and this protein has been proposed to be a master regulator of bone remodelling in inflammatory arthritis. Local glucocorticoid production is also significantly increased during joint inflammation. Therefore, we investigated how locally derived glucocorticoids and inflammatory cytokines regulate DKK1 synthesis in synovial fibroblasts during inflammatory arthritis.\\nABSTRACT\\nabstract_title_1\\nMethods\\nABSTRACT\\nabstract\\nWe examined expression and regulation of DKK1 in primary cultures of human synovial fibroblasts isolated from patients with inflammatory arthritis. The effect of TNFalpha, IL-1beta and glucocorticoids on DKK1 mRNA and protein expression was examined by real-time PCR and ELISA. The ability of inflammatory cytokine-induced expression of the glucocorticoid-activating enzyme 11beta-hydroxysteroid dehydrogenase type 1 (11beta-HSD1) to sensitise fibroblasts to endogenous glucocorticoids was explored. Global expression of Wnt signalling and target genes in response to TNFalpha and glucocorticoids was assessed using a custom array.\\nABSTRACT\\nabstract_title_1\\nResults\\nABSTRACT\\nabstract\\nDKK1 expression in human synovial fibroblasts was directly regulated by glucocorticoids but not proinflammatory cytokines. Glucocorticoids, but not TNFalpha, regulated expression of multiple Wnt agonists and antagonists in favour of inhibition of Wnt signalling. However, TNFalpha and IL-1beta indirectly stimulated DKK1 production through increased expression of 11beta-HSD1.\\nABSTRACT\\nabstract_title_1\\nConclusions\\nABSTRACT\\nabstract\\nThese results demonstrate that in rheumatoid arthritis synovial fibroblasts, DKK1 expression is directly regulated by glucocorticoids rather than TNFalpha. Consequently, the links between synovial inflammation, altered Wnt signalling and bone remodelling are not direct but are dependent on local activation of endogenous glucocorticoids.\\nINTRO\\ntitle_1\\nIntroduction\\nINTRO\\nparagraph\\nDuring adult life, bone formation and resorption are normally tightly coupled such that the amount of bone formed is similar to the amount resorbed. However, in patients with chronic inflammatory arthritis (for example, rheumatoid arthritis (RA)) bone remodelling is abnormal. Bone resorption is increased due to increased activity of osteoclasts whereas bone formation by osteoblasts is suppressed. The uncoupling of formation from resorption results in bone loss and an increased risk of fractures. A similar process is seen in states of systemic glucocorticoid excess such as Cushing's syndrome or during treatment with therapeutic glucocorticoids, but circulating glucocorticoid levels in patients with RA are not elevated. We have previously hypothesised that the bone loss seen in inflammatory arthritis is secondary to local glucocorticoid activation through the 11beta-hydroxysteroid dehydrogenase type 1 (11beta-HSD1) enzyme. This enzyme converts inactive steroids (such as cortisone and prednisone) to their active counterparts (cortisol and prednisolone). Patients lacking this enzyme are unresponsive to cortisone acetate or prednisone therapy due to their inability to activate these steroids in vivo . We have previously demonstrated that this enzyme is highly expressed in human primary synovial fibroblasts and synovial tissue explants. In vitro, the expression and activity of this enzyme increase dramatically in these cells and tissues in response to TNFalpha or IL-1beta. In patients with RA, 11beta-HSD1 activity in synovial tissue and total body measures of 11beta-HSD1 activity are increased and correlate with serum markers of inflammation. In a rodent model of inflammatory arthritis, 11beta-HSD1 activity and expression in the joint are increased, and activity is reduced by anti-TNF therapy. Thus the level of active glucocorticoids within the joint, and specifically within synovial fibroblasts, appears to be high during inflammatory arthritis.\\nINTRO\\nparagraph\\nRecently, secretion of the Wnt antagonist dickkopf-1 (DKK1) has been proposed to be a master regulator of bone remodelling in inflammatory arthritis. DKK1 suppresses osteoblast differentiation but also decreases the expression of osteoprotegerin (OPG) leading to increased osteoclastogenesis. DKK1 is synthesised by murine synovial fibroblasts in response to inflammation through a TNFalpha-dependent mechanism. Neutralisation of DKK1 in mice using anti-DKK1 antibodies reversed the bone loss seen in inflammatory arthritis and resulted in the formation of new bone near the areas of greatest inflammation. In osteoblasts, mesenchymal cells that are developmentally closely related to synovial fibroblasts, glucocorticoids are a very powerful inducer of DKK1 and this effect has been proposed as the mechanism that mediates bone loss due to systemic glucocorticoid excess. Given the increase in 11beta-HSD1 expression that we have observed in synovial fibroblasts, we hypothesised that the regulation of synovial fibroblast DKK1 expression by inflammation was indirect and dependent on the local generation of glucocorticoids within the synovial fibroblasts. Therefore, we assessed the regulation and relative expression of DKK1 following treatments with both glucocorticoids and inflammatory cytokines in primary synovial fibroblasts.\\nMETHODS\\ntitle_1\\nMaterials and methods\\nMETHODS\\ntitle_2\\nPatients\\nMETHODS\\nparagraph\\nBiopsies of matched synovium and skin were obtained during hip, knee or elbow arthroplasty from patients with RA (based on the 1987 American College of Rheumatology (formally the American Rheumatism Association) criteria), osteoarthritis (OA) and ankylosing spondylitis (AS) (based on the modified New York criteria). Tissue was taken on ice from the operating theatre and synovial tissue was prepared within 2 hours by removing any adherent non-synovial tissue. Tissue explant experiments were performed on 20 mg sections prior to enzyme assay or ELISA. Skin tissue was prepared by removing the subcutaneous fat and dividing into 20 mg pieces prior to enzyme assay or ELISA. Primary cultures of synovial and dermal fibroblasts were generated as described previously. Fibroblasts were treated with 0.01 to 10 ng/ml TNFalpha (R&D Systems, Abingdon, UK) or 0.1 to 100 nmol/l of dexamethasone (DEX), cortisol or cortisone with or without 1 mumol/l of the 11beta-HSD inhibitor glycyrrhetinic acid (GE) for 24 hours before harvesting for mRNA analysis or for 48 hours before measuring DKK1 in culture media. All studies had ethical approval from the Local Ethics Committee and informed consent was obtained prior to taking of samples.\\nMETHODS\\ntitle_2\\nRNA extraction and reverse transcription\\nMETHODS\\nparagraph\\nRNA was extracted from cultured fibroblasts using the single-step extraction method (TRI Reagent, Sigma-Aldrich, Poole, UK). Briefly, confluent monolayers of synovial fibroblasts in 6-well plates were lysed in 1 ml of TRI Reagent and RNA isolated as per the manufacturers protocol. RNA were then reverse transcribed using random hexamers in a 20 mul volume, as stated in the manufacturer's protocol (Promega, Madison, WI, USA).\\nMETHODS\\ntitle_2\\nReal-time PCR\\nMETHODS\\nparagraph\\nProbes and primers were based on Assay-on-Demand  sequences (Applied Biosystems, Warrington, UK). mRNA levels for DKK1 (Hs00183740_m1), DKK2 (Hs00997455_m1), WNT2 (Hs00608224_m1) and FRZB (Hs00173503_m1) were assessed using real-time PCR in an ABI 7500 system (Applied Biosystems, Warrington, UK) using a previously reported technique. Reactions contained TaqMan universal PCR master mix (Applied Biosystems, Warrington, UK), 900 nmol primers, 100 to 200 nmol TaqMan probe and 50 ng cDNA. Primers for 18S (Hs03928985_g1) were used as an internal reference. All target gene probes were labelled with the fluorescent label FAM, and the 18S probe with the fluorescent label VIC. Reactions occurred as follows: 50 C for 2 minutes, 95 C for 10 minutes, 40 cycles of 95 C for 15 seconds and 60 C for 1 minute. Data were obtained as Ct values (the cycle number at which logarithmic PCR plots cross a calculated threshold line) and used to determine DeltaCt values (Ct of target gene - Ct of housekeeping gene) as raw data for gene expression (high DeltaCt = low gene expression). The fold change in gene expression was determined by subtracting DeltaCt values for treated cells from their respective control samples. The resulting DeltaDeltaCt values were then used to calculate fold change in gene expression according to the equation 2-DeltaDeltaCt.\\nMETHODS\\ntitle_2\\nReal-time PCR custom TaqMan gene array\\nMETHODS\\nparagraph\\nGlobal expression of Wnt signalling and target genes was assessed using the predefined Wnt signalling gene format with TaqMan Express Plates (Catalogue number 4391524, Applied Biosystems, Warrington, UK). Both 18S and GAPDH were used as an internal reference. FAM-labelled primers and probes were reconstituted to 20 mul in TaqMan universal PCR master mix (Applied Biosystems, Warrington, UK) containing 50 ng of target cDNA. Real-time PCR was performed using an ABI 7500 system (Applied Biosystems, Warrington, UK) using a previously reported technique. Reaction conditions and data analysis were performed as reported for real-time PCR. Array data were submitted to the Gene Expression Omnibus (GEO) repository and given the designation GSE37520.\\nMETHODS\\ntitle_2\\nAnalysis of DKK1 levels by ELISA\\nMETHODS\\nparagraph\\nDKK1 levels in supernatants from cultured cells were measured using a commercially available sandwich ELISA (Catalogue Number: DY1906, R&D systems, Abingdon, UK). Briefly, synovial fibroblasts were seeded at 100,000 cells per well in a 6-well plate. Media was conditioned for 48 hours in the presence or absence of appropriate treatments. Conditioned media was collected and stored at -80 C prior to analysis in triplicate by ELISA. Data were expressed as ng/ml of DKK1 in culture media after 48 hour culture with 100,000 cells from a minimum of three separate fibroblast lines.\\nMETHODS\\ntitle_2\\nStatistics\\nMETHODS\\nparagraph\\nData were reported as the mean +- standard deviation (SD) of synovial explants from separate individuals (unless otherwise stated). Multiple comparisons were assessed using one-way ANOVA with a Dunnett's post hoc analysis. Regression analysis and students paired t tests were performed using SPSS Data Editor (SPSS Inc., Chicago, IL, USA).\\nRESULTS\\ntitle_1\\nResults\\nRESULTS\\ntitle_2\\nInduction of DKK1 expression in human primary synovial fibroblasts is by glucocorticoids and not TNFalpha/IL-1beta\\nRESULTS\\nparagraph\\nThe effects of glucocorticoids and proinflammatory cytokines on DKK1 expression were examined in primary human synovial fibroblasts. Dexamethasone (100 nM) caused a substantial and statistically significant increase in DKK1 mRNA expression (4.9-fold vs control; P < 0.05) (Figure 1A). TNFalpha (10 ng/ml) caused a small and non-significant increase in DKK1 mRNA expression (2.3-fold vs control; NS). In this experimental set-up, levels of endogenous glucocorticoids in the media were below those required to allow an indirect glucocorticoid-mediated effect of TNFalpha expression on DKK1 mRNA expression. TNFalpha did not further augment the effect of glucocorticoid treatment on DKK1 mRNA expression when the two were combined. Similar results were obtained when the effect of dexamethasone on DKK1 mRNA expression was compared with IL-1beta (10 ng/mL) (Figure 1B) with IL-1beta having no direct effect on DKK1 mRNA expression. There was no effect of dexamethasone, TNFalpha or IL-1beta treatment on DKK1 mRNA expression in dermal fibroblasts that were used as non-synovial, fibroblast controls. Dexamethasone also caused a significant increase in DKK1 protein secretion in synovial fibroblast cultures but TNFalpha and IL-1beta had no overall effect (control, 4.4 +- 0.4 vs DEX, 12.3 +- 0.4 ng/ml; P < 0.05) (Figure 1C).\\nF1\\nFIG\\nfig_caption\\nEffects of glucocorticoids/proinflammatory cytokines on DKK1 expression in primary synovial fibroblasts (FLS) or dermal fibroblasts (DF). Results shown are the combined duplicates of four separate FLS or DF cell-lines. (A) Effect of dexamethasone (100 nM), TNFalpha (10 ng/ml), or (B) IL-1beta (10 ng/ml) on DKK1 mRNA expression in FLS. (C) Effect of dexamethasone (100 nM), TNFalpha (10 ng/ml) and IL-1beta (10 ng/ml) on secretion of DKK1 protein in FLS. Dexamethasone but not TNFalpha or IL-1beta, induces significant secretion of DKK1 protein from FLS. *P < 0.05, **P < 0.01.\\nRESULTS\\nparagraph\\nThe time course of DKK1 secretion in response to dexamethasone is shown in Figure 2A. DKK-1 expression increased in a time-dependent manner. There was no increase in DKK1 secretion with TNFalpha above basal expression at any time point, whilst dexamethasone significantly increased expression at all time points examined (48 hour control, 18.6 +- 2.8 ng/ml vs 48 hour DEX, 34.5 +- 1.8 ng/ml; P < 0.05). The expression of DKK1 in response to dexamethasone was dose dependent and reached a peak at 100 nM dexamethasone (control, 1 +- 1.1 ng/ml vs DEX 100 nmol/l, 6.0 +- 1.4 ng/ml; P < 0.05) (Figure 2B). In contrast, TNFalpha and IL-1beta at any dose did not significantly increase secreted DKK1 in dose-response experiments (Figure 2C-D).\\nF2\\nFIG\\nfig_caption\\nRegulation of DKK1 expression by glucocorticoids. (A) Time course for induction of DKK1 protein expression by synovial fibroblasts (FLS) in response to dexamethasone or TNFalpha. (B) Dose response curves for the secretion of DKK1 in response to dexamethasone (expressed relative to baseline expression). *P < 0.05, **P < 0.01 relative to control.\\nRESULTS\\ntitle_2\\nActivation of glucocorticoids within synovial fibroblasts regulates DKK1 synthesis\\nRESULTS\\nparagraph\\nSince dexamethasone is a synthetic glucocorticoid, we further examined whether endogenous glucocorticoids also have the same effect on DKK1 expression (Figure 3). The effect of cortisol was similar to that of dexamethasone in inducing DKK1 mRNA and protein expression (DEX; mRNA 3.1-fold, protein 2.7-fold +- 0.53; cortisol, mRNA 3.2-fold, protein 2.3-fold +- 0.39 vs control; P < 0.05) (Figure 3A and 3B). Cortisone was also found to significantly induce DKK1 mRNA and protein expression (cortisone; mRNA 2.7-fold, protein 1.6-fold +- 1.8 vs control; P < 0.05). To function effectively as a glucocorticoid receptor agonist, cortisone needs to be converted to cortisol by the 11beta-HSD1 enzyme. Inhibition of the 11beta-HSD1 enzyme using glycyrrhetinic acid (GE) blocked the effect of cortisone on DKK1 expression. As observed for dexamethasone, neither cortisol nor cortisone had an impact on DKK1 expression in dermal fibroblasts (data not shown). Given the lack of direct induction of DKK1 expression with TNFalpha, we explored whether TNFalpha treatment could sensitise synovial fibroblasts to cortisone through induction of 11beta-HSD1 activity (Figure 3C). The duration of incubation of synovial fibroblasts with cortisone was reduced to 5 hours, such that conversion of cortisone to cortisol was insufficient to have any effect on DKK1 protein synthesis under basal conditions. Under these conditions, pretreatment with TNFalpha sensitised synovial fibroblasts to the effects of cortisone. This effect was blocked by an inhibitor of 11beta-HSD1 activity, confirming an indirect effect of TNFalpha via upregulation of 11beta-HSD1 activity.\\nF3\\nFIG\\nfig_caption\\nThe effect of endogenous glucocorticoids +/- 11beta-hydroxysteroid dehydrogenase (11beta-HSD) enzyme inhibitor on DKK1 expression. (A) Effect of various glucocorticoids on DKK1 mRNA expression in synovial fibroblasts (FLS). Cortisone caused a significant increase in DKK1 mRNA, an effect blocked by the 11beta-HSD1 inhibitor glycyrrhetinic acid (GE). (B) Effect of glucocorticoids on DKK1 protein secretion in FLS and dermal fibroblasts (DF). (C) Sensitisation of DKK1 secretion by FLS to the effects of cortisone (E) by pretreatment with TNFalpha for 48 hours. *P < 0.05, **P < 0.01 relative to control. Results shown are the combined duplicates of four separate FLS lines.\\nRESULTS\\ntitle_2\\nComparison between DKK1 synthesis in patients with RA, OA and AS\\nRESULTS\\nparagraph\\nTo assess whether the changes observed were specific to synovial fibroblasts of RA origin, the effect of glucocorticoids on DKK1 protein secretion was examined in synovial fibroblasts isolated from patients with different arthritides (OA and AS, n = 5 in total). As with synovial fibroblasts from patients with RA, both cortisol and cortisone were able to induce DKK1 synthesis (data not shown). There was a suggestion that the basal expression level of DKK1 was lower in patients with AS (P < 0.05 when AS cells were compared to other synovial fibroblasts) but the number of patients with AS limited the robustness of this finding.\\nRESULTS\\ntitle_2\\nGlucocorticoids modify Wnt gene expression within synovial fibroblasts in a coordinated manner\\nRESULTS\\nparagraph\\nThe effect of glucocorticoid treatment (100 nM dexamethasone) or TNFalpha (10 ng/ml) on expression of a range of Wnt-related genes (n = 96) was examined using a targeted microarray (a full list of the genes included in the array is given in Additional file 1, Table S1 and data can be found at the GEO repository with reference number GSE37520). Genes with the greatest fold change are shown in Figure 4. In keeping with the results shown earlier, glucocorticoid treatment resulted in a significant increase in DKK1 expression (3.1-fold vs control; P < 0.05). There was, in addition, a substantial induction of the frizzled-related protein-B (FRZB) gene (also known as secreted frizzled related protein-3 (SFRP3)) (10.6-fold vs control; P < 0.05). FRZB, like DKK1, is also a secreted inhibitor of Wnt signalling. In keeping with a coordinated effect to reduce Wnt signalling, the expression of WNT2 was substantially reduced (7.2-fold vs control; P < 0.05). The effect of TNFalpha on expression of Wnt-related genes was less dramatic and did not suggest a coordinated effect on the expression of secreted Wnts and their antagonists. The genes with greatest upregulation in response to TNFalpha treatment were the ubiquitin-like gene UBD and cyclooxygenase-2 (PTGS2) (59.4- and 8.3-fold respectively; P < 0.05). TNFalpha had no significant effect on DKK1 and FRZB expression. Real-time PCR was used to validate the changes in mRNA expression for FRZB and WNT2 in response to dexamethasone and TNFalpha (Figure 4C).\\nF4\\nFIG\\nfig_caption\\nEffects of glucocorticoids or TNFalpha on Wnt-related gene expression. (A and B) Effect of glucocorticoid treatment (100 nM dexamethasone) or TNFalpha (10 ng/ml) on expression of Wnt-related genes (n = 96) was examined using a targeted microarray. Genes with the greatest fold change are shown (the effect of TNFalpha on DKK1 and FRZB expression was not significant). A full list of the genes included in the array and their full gene titles is given in Additional file 1, Table S1. (C) Real-time PCR validation of changes in mRNA expression for FRZB and WNT2 in response to dexamethasone (DEX) or TNFalpha. *P < 0.05, **P < 0.01 relative to control. Results shown are the combined results of three separate FLS lines.\\nDISCUSS\\ntitle_1\\nDiscussion\\nDISCUSS\\nparagraph\\nBone is an important target tissue in patients with RA, with many patients developing local erosions and periarticular and generalised osteoporosis. Previous studies have highlighted the central importance of synovial fibroblasts in the abnormal bone remodelling associated with joint inflammation. These effects were assumed to be due to direct effects of inflammatory mediators, principally on expression of the Wnt signalling antagonist DKK1. Indeed, a growing body of work has demonstrated that this secreted factor is markedly elevated in the serum of patients with rheumatoid arthritis, where its levels positively correlate with multiple markers of disease activity. Most importantly, DKK1 positively correlated with erosive bone loss in patients with rheumatoid arthritis, supporting a central role for this factor in inflammatory bone loss. Our findings demonstrate that the effect of inflammation on DKK1 synthesis is not direct, but instead depends on an increase in local glucocorticoid levels through the induction of the 11beta-HSD1 enzyme (illustrated schematically in Figure 5). High levels of glucocorticoids are linked with a range of conditions that detrimentally affect bone through uncoupling bone formation from resorption, but this is the first study to show a link between effects of glucocorticoids on synovial tissue and uncoupling of bone metabolism through a paracrine effect. Glucocorticoids also cause a coordinated change in production of Wnt signalling modulators by synovial fibroblasts that extends beyond DKK1 with an increase in several antagonists and a suppression of agonists. These findings are in accordance with the effects of glucocorticoids on Wnt production by osteoblasts, a cell type closely related developmentally to the synovial fibroblast.\\nF5\\nFIG\\nfig_caption\\nRole of local glucocorticoid generation in inflammatory changes in bone. Schematic illustration of the mechanism by which synovial inflammation interacts with local generation of active glucocorticoids to modulate Wnt signalling in osteoblasts.\\nDISCUSS\\nparagraph\\nIt is well established that high circulating levels of endogenous or exogenous glucocorticoids suppress bone formation. This is at least partly through a direct mechanism, since mice with osteoblast-specific deletion of the glucocorticoid receptor alpha (GRalpha) or overexpression of 11beta-HSD2 (which inactivates glucocorticoids) within osteoblasts are protected against the effects of high-dose glucocorticoids on bone. These results suggest that the development of synovial hypertrophy and synovial fibroblast hyperplasia and induction of local glucocorticoid generation within synovium are needed for a significant paracrine effect of inflammation on bone. The data also suggest that in clinical situations where glucocorticoids are used to target synovial inflammation the detrimental effects of this therapy on bone could be mediated by direct effects through glucocorticoid receptor signalling in osteoblasts or indirectly through glucocorticoid receptor signalling in synovial fibroblasts.\\nDISCUSS\\nparagraph\\nThe involvement of glucocorticoids in mechanisms to switch off tissue repair is not surprising. Glucocorticoids have anti-anabolic and catabolic effects in a range of tissues. An increase in the levels of glucocorticoids within the tissue through an increase in the activity of the hypothalamo-pituitary-adrenal axis is an important protective response. Adrenal insufficiency in humans or animals leads to a dramatic reduction in ability to withstand inflammatory stress. Within bone, glucocorticoid signalling in synovial fibroblasts may provide a useful way of uncoupling bone resorption from formation during joint disease. Without an ability to temporarily uncouple formation from resorption there is a risk of aberrant, uncoordinated bone deposition that could be detrimental to the function of the joint. Importantly, abnormal osteophyte formation has been recently reported in HSD11B1 knockout mice in response to inflammatory arthritis. At sites of bone remodelling, there was clearly abnormal excessive formation of new bone that was greatest adjacent to the site of synovial tissue inflammation. This is despite the gene for DKK1 being intact, and there being higher levels of circulating TNFalpha and endogenous corticosterone during inflammation, in this model. All these factors would normally be expected to result in a greater impairment of bone formation in knockout animals than wild types. The high corticosterone levels also demonstrate that the phenotype observed is unlikely to be related to an alteration of systemic glucocorticoid levels since excessive bone formation occurred despite the higher circulating glucocorticoid levels.\\nDISCUSS\\nparagraph\\nPrevious studies have linked variation in the expression of DKK1 by synovial fibroblasts to rheumatic diseases associated with excessive bone formation, primarily AS although abnormal expression of the osteocyte-specific protein (and Wnt signalling inhibitor) sclerostin has also been described. We observed no difference in the ability of glucocorticoids to induce DKK1 in a limited number of patients with AS. However, it must be borne in mind that the excessive formation of bone in this condition is normally restricted to the axial spine. The reason for the axial predisposition to AS is unclear but it is possible that this reflects a difference in the regulation or expression of 11beta-HSD1 in the spinal tissues. Synovial tissue is likely to differ between the peripheral and central joints and 11beta-HSD1 expression in some cell types demonstrates regional variation. Polymorphic markers within the HSD11B1 gene have been linked to differences in bone density and fracture risk and could provide tools to examine for differences in bone manifestation of disease in patients with chronic inflammatory conditions.\\nCONCL\\ntitle_1\\nConclusions\\nCONCL\\nparagraph\\nThese data show that local glucocorticoid metabolism has an important role in the regulation of bone remodelling. The 11beta-HSD1 enzyme is thus a potential therapeutic target for treating disorders characterised by uncoupling of bone formation from resorption.\\nABBR\\ntitle_1\\nAbbreviations\\nABBR\\nparagraph\\nCOMP_INT\\ntitle_1\\nCompeting interests\\nCOMP_INT\\nparagraph\\nThe authors have no competing interests.\\nAUTH_CONT\\ntitle_1\\nAuthors' contributions\\nAUTH_CONT\\nparagraph\\nRH, MJ, AN and JT performed DKK1 and molecular biology assays. EH supervised laboratory techniques and secured study funding. AF generated human primary synovial fibroblast cell lines and advised on study design. PMS, CDB, KR and MSC conceived of the study, coordinated its design, interpreted the data and secured study funding. All authors read and approved the final manuscript.\\nSUPPL\\ntitle_1\\nSupplementary Material\\nACK_FUND\\ntitle_1\\nAcknowledgements\\nACK_FUND\\nparagraph\\nsurname:Herman;given-names:S\\nsurname:Kronke;given-names:G\\nsurname:Schett;given-names:G\\nREF\\nTrends in Molecular Medicine\\nref\\nMolecular mechanisms of inflammatory bone damage: emerging targets for therapy\\nsurname:Hardy;given-names:R\\nsurname:Cooper;given-names:MS\\nREF\\nJ Endocrinol\\nref\\nBone loss in inflammatory disorders\\nsurname:Spector;given-names:TD\\nsurname:Hall;given-names:GM\\nsurname:McCloskey;given-names:EV\\nsurname:Kanis;given-names:JA\\nREF\\nBMJ\\nref\\nRisk of vertebral fracture in women with rheumatoid arthritis\\nsurname:Raza;given-names:K\\nsurname:Hardy;given-names:R\\nsurname:Cooper;given-names:MS\\n10.1093/rheumatology/keq212\\nREF\\nRheumatology (Oxford)\\nref\\nsurname:Cooper;given-names:MS\\nsurname:Bujalska;given-names:I\\nsurname:Rabbitt;given-names:E\\nsurname:Walker;given-names:EA\\nsurname:Bland;given-names:R\\nsurname:Sheppard;given-names:MC\\nsurname:Hewison;given-names:M\\nsurname:Stewart;given-names:PM\\nREF\\nJ Bone Miner Res\\nref\\nsurname:Tomlinson;given-names:JW\\nsurname:Walker;given-names:EA\\nsurname:Bujalska;given-names:IJ\\nsurname:Draper;given-names:N\\nsurname:Lavery;given-names:GG\\nsurname:Cooper;given-names:MS\\nsurname:Hewison;given-names:M\\nsurname:Stewart;given-names:PM\\nREF\\nEndocr Rev\\nref\\n11beta-hydroxysteroid dehydrogenase type 1: A tissue-specific regulator of glucocorticoid response\\nsurname:Cooper;given-names:MS\\nsurname:Stewart;given-names:PM\\nREF\\nJ Clin Endocrinol Metab\\nref\\nsurname:Nordenstrom;given-names:A\\nsurname:Marcus;given-names:C\\nsurname:Axelson;given-names:M\\nsurname:Wedell;given-names:A\\nsurname:Ritzen;given-names:EM\\nREF\\nJ Clin Endocrinol Metab\\nref\\nsurname:Hardy;given-names:RS\\nsurname:Filer;given-names:A\\nsurname:Cooper;given-names:MS\\nsurname:Parsonage;given-names:G\\nsurname:Raza;given-names:K\\nsurname:Hardie;given-names:DL\\nsurname:Rabbitt;given-names:EH\\nsurname:Stewart;given-names:PM\\nsurname:Buckley;given-names:CD\\nsurname:Hewison;given-names:M\\nREF\\nArthritis Res Ther\\nref\\nsurname:Hardy;given-names:R\\nsurname:Rabbitt;given-names:EH\\nsurname:Filer;given-names:A\\nsurname:Emery;given-names:P\\nsurname:Hewison;given-names:M\\nsurname:Stewart;given-names:PM\\nsurname:Gittoes;given-names:NJ\\nsurname:Buckley;given-names:CD\\nsurname:Raza;given-names:K\\nsurname:Cooper;given-names:MS\\nREF\\nAnn Rheum Dis\\nref\\nLocal and systemic glucocorticoid metabolism in inflammatory arthritis\\nsurname:Kaur;given-names:K\\nsurname:Hardy;given-names:R\\nsurname:Rabbitt;given-names:EH\\nsurname:Hewison;given-names:M\\nsurname:Cooper;given-names:MS\\nsurname:Ahasan;given-names:MM\\nsurname:Eijken;given-names:M\\nsurname:van Leeuwen;given-names:JP\\nsurname:Filer;given-names:A\\nsurname:Thomas;given-names:AM\\nsurname:Raza;given-names:K\\nsurname:Buckley;given-names:CD\\nsurname:Stewart;given-names:PM\\nREF\\nAnn Rheum Dis\\nref\\nSynergistic induction of local glucocorticoid generation by inflammatory cytokines and glucocorticoids: implications for inflammation associated bone loss\\nsurname:Ergang;given-names:P\\nsurname:Leden;given-names:P\\nsurname:Vagnerova;given-names:K\\nsurname:Klusonova;given-names:P\\nsurname:Miksik;given-names:I\\nsurname:Jurcovicova;given-names:J\\nsurname:Kment;given-names:M\\nsurname:Pacha;given-names:J\\nREF\\nMol Cell Endocrinol\\nref\\nLocal metabolism of glucocorticoids and its role in rat adjuvant arthritis\\nsurname:Diarra;given-names:D\\nsurname:Stolina;given-names:M\\nsurname:van der;given-names:HD\\nsurname:Landewe;given-names:R\\nsurname:Lacey;given-names:D\\nsurname:Richards;given-names:WG\\nsurname:Schett;given-names:G\\nsurname:Polzer;given-names:K\\nsurname:Zwerina;given-names:J\\nsurname:Ominsky;given-names:MS\\nsurname:Dwyer;given-names:D\\nsurname:Korb;given-names:A\\nsurname:Smolen;given-names:J\\nsurname:Hoffmann;given-names:M\\nsurname:Scheinecker;given-names:C\\nREF\\nNature Medicine\\nref\\nDickkopf-1 is a master regulator of joint remodeling\\nsurname:Wang;given-names:FS\\nsurname:Ko;given-names:JY\\nsurname:Yeh;given-names:DW\\nsurname:Ke;given-names:HC\\nsurname:Wu;given-names:HL\\nREF\\nEndocrinology\\nref\\nsurname:Cooper;given-names:MS\\nsurname:Rabbitt;given-names:EH\\nsurname:Goddard;given-names:PE\\nsurname:Bartlett;given-names:WA\\nsurname:Hewison;given-names:M\\nsurname:Stewart;given-names:PM\\nREF\\nJ Bone Miner Res\\nref\\nOsteoblastic 11beta-hydroxysteroid dehydrogenase type 1 activity increases with age and glucocorticoid exposure\\nsurname:Wang;given-names:SY\\nsurname:Liu;given-names:YY\\nsurname:Ye;given-names:H\\nsurname:Guo;given-names:JP\\nsurname:Li;given-names:R\\nsurname:Liu;given-names:X\\nsurname:Li;given-names:ZG\\nREF\\nJ Rheumatol\\nref\\nCirculating Dickkopf-1 is correlated with bone erosion and inflammation in rheumatoid arthritis\\nsurname:Rauch;given-names:A\\nsurname:Seitz;given-names:S\\nsurname:Ostermay;given-names:S\\nsurname:Schinke;given-names:T\\nsurname:Spanbroek;given-names:R\\nsurname:Zaiss;given-names:MM\\nsurname:Angel;given-names:PE\\nsurname:Lerner;given-names:UH\\nsurname:David;given-names:JP\\nsurname:Reichardt;given-names:HM\\nsurname:Amling;given-names:M\\nsurname:Schutz;given-names:G\\nsurname:Baschant;given-names:U\\nsurname:Tuckermann;given-names:JP\\nsurname:Schilling;given-names:AF\\nsurname:Illing;given-names:A\\nsurname:Stride;given-names:B\\nsurname:Kirilov;given-names:M\\nsurname:Mandic;given-names:V\\nsurname:Takacz;given-names:A\\nsurname:Schmidt-Ullrich;given-names:R\\nREF\\nCell Metab\\nref\\nGlucocorticoids suppress bone formation by attenuating osteoblast differentiation via the monomeric glucocorticoid receptor\\nsurname:O'Brien;given-names:CA\\nsurname:Jia;given-names:D\\nsurname:Plotkin;given-names:LI\\nsurname:Bellido;given-names:T\\nsurname:Powers;given-names:CC\\nsurname:Stewart;given-names:SA\\nsurname:Manolagas;given-names:SC\\nsurname:Weinstein;given-names:RS\\nREF\\nEndocrinology\\nref\\nGlucocorticoids act directly on osteoblasts and osteocytes to induce their apoptosis and reduce bone formation and strength\\nsurname:Cooper;given-names:MS\\nsurname:Stewart;given-names:PM\\nREF\\nN Engl J Med\\nref\\nCorticosteroid insufficiency in acutely ill patients\\nsurname:Coutinho;given-names:AE\\nsurname:Gray;given-names:M\\nsurname:Brownstein;given-names:DG\\nsurname:Salter;given-names:DM\\nsurname:Sawatzky;given-names:DA\\nsurname:Clay;given-names:S\\nsurname:Gilmour;given-names:JS\\nsurname:Seckl;given-names:JR\\nsurname:Savill;given-names:JS\\nsurname:Chapman;given-names:KE\\nREF\\nEndocrinology\\nref\\nsurname:Uderhardt;given-names:S\\nsurname:Diarra;given-names:D\\nsurname:Katzenbeisser;given-names:J\\nsurname:David;given-names:JP\\nsurname:Zwerina;given-names:J\\nsurname:Richards;given-names:W\\nsurname:Kronke;given-names:G\\nsurname:Schett;given-names:G\\nREF\\nAnn Rheum Dis\\nref\\nBlockade of Dickkopf (DKK)-1 induces fusion of sacroiliac joints\\nsurname:Appel;given-names:H\\nsurname:Ruiz-Heiland;given-names:G\\nsurname:Sieper;given-names:J\\nsurname:Schett;given-names:G\\nsurname:Listing;given-names:J\\nsurname:Zwerina;given-names:J\\nsurname:Hermann;given-names:M\\nsurname:Mueller;given-names:R\\nsurname:Haibel;given-names:H\\nsurname:Baraliakos;given-names:X\\nsurname:Hempfing;given-names:A\\nsurname:Rudwaleit;given-names:M\\nREF\\nArthritis Rheum\\nref\\nAltered skeletal expression of sclerostin and its link to radiographic progression in ankylosing spondylitis\\nsurname:Hwang;given-names:JY\\nsurname:Lee;given-names:SH\\nsurname:Lee;given-names:JY\\nsurname:Kim;given-names:SY\\nsurname:Kim;given-names:GS\\nsurname:Koh;given-names:JM\\nsurname:Go;given-names:MJ\\nsurname:Kim;given-names:YJ\\nsurname:Kim;given-names:HC\\nsurname:Kim;given-names:TH\\nsurname:Hong;given-names:JM\\nsurname:Park;given-names:EK\\nREF\\nBone\\nref\\nHSD11B1 polymorphisms predicted bone mineral density and fracture risk in postmenopausal women without a clinically apparent hypercortisolemia\\n\",\n",
       " '29980666': 'BioC-API\\ncollection.key\\nCC BY\\nOpen Access This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The images or other third party material in this article are included in the article\\'s Creative Commons license, unless indicated otherwise in a credit line to the material. If material is not included in the article\\'s Creative Commons license and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this license, visit http://creativecommons.org/licenses/by/4.0/.\\nsurname:Weltner;given-names:Jere\\nsurname:Balboa;given-names:Diego\\nsurname:Katayama;given-names:Shintaro\\nsurname:Bespalov;given-names:Maxim\\nsurname:Krjutskov;given-names:Kaarel\\nsurname:Jouhilahti;given-names:Eeva-Mari\\nsurname:Trokovic;given-names:Ras\\nsurname:Kere;given-names:Juha\\nsurname:Otonkoski;given-names:Timo\\nTITLE\\nfront\\nHuman pluripotent reprogramming with CRISPR activators\\nABSTRACT\\nabstract\\nCRISPR-Cas9-based gene activation (CRISPRa) is an attractive tool for cellular reprogramming applications due to its high multiplexing capacity and direct targeting of endogenous loci. Here we present the reprogramming of primary human skin fibroblasts into induced pluripotent stem cells (iPSCs) using CRISPRa, targeting endogenous OCT4, SOX2, KLF4, MYC, and LIN28A promoters. The low basal reprogramming efficiency can be improved by an order of magnitude by additionally targeting a conserved Alu-motif enriched near genes involved in embryo genome activation (EEA-motif). This effect is mediated in part by more efficient activation of NANOG and REX1. These data demonstrate that human somatic cells can be reprogrammed into iPSCs using only CRISPRa. Furthermore, the results unravel the involvement of EEA-motif-associated mechanisms in cellular reprogramming.\\nABSTRACT\\nabstract\\nCRISPRa is an attractive tool for cellular reprogramming due to its multiplexing capacity and direct targeting of genomic loci. Here the authors demonstrate the reprogramming of human fibroblasts into iPSCs, which is enhanced by targeting a conserved Alu-motif.\\nINTRO\\ntitle_1\\nIntroduction\\nINTRO\\nparagraph\\nCRISPRa system relies on sequence specific recruitment of a catalytically inactivated version of Cas9 protein (dCas9) to genomic sequences defined by short guide RNA (gRNA) molecules. The fact that dCas9 effectors can be used to control transcription of targeted endogenous loci makes it useful for mediating cellular reprogramming, which requires silencing and activation of endogenous gene sets for proper cell type conversion. CRISPRa may therefore be beneficial in overcoming reprogramming barriers that limit reprogramming efficiency and contribute to the emergence of partially reprogrammed stable cell populations, often associated with inadequate endogenous gene activation or silencing. Previously, dCas9 effectors have been used to mediate differentiation, transdifferentiation, and reprogramming of various mouse and human cell types, but complete pluripotent reprogramming of human cells using only CRISPRa has not yet been reported.\\nINTRO\\nparagraph\\nIn addition to gene activation, dCas9 effector mediated DNA targeting can be used to decipher the functions of genomic regulatory elements. Combining reprogramming factor promoter targeting gRNAs with targeting of other regulatory elements has high potential in mediating comprehensive resetting of gene regulatory networks. A conserved Alu-motif was recently reported to be enriched in the promoter areas of the first genes expressed during human embryo genome activation (EGA). This sequence is thus likely to be involved in the control of early embryonic transcriptional networks. As human embryos can reprogram somatic cell nuclei, we hypothesized that targeting this EGA-enriched Alu-motif (EEA-motif) could enhance reprogramming of somatic cells to pluripotency.\\nINTRO\\nparagraph\\nDevelopment of reprogramming approaches for faithful recapitulation of cellular phenotypes is an important task, considering the increasing pace with which reprogrammed cells are moving toward clinical trials. Here we describe a method for reprogramming human cells, including primary adult human skin fibroblasts, into induced pluripotent stem cells by CRISPRa. This opens up important possibilities for the development of more extensive CRISPRa reprogramming approaches for human cells. Efficiency of the method depends on the targeting of the EEA-motif, which results in improved activation of a subset of endogenous genes that work as reprogramming factors, including NANOG and ZFP42 (REX1). These results also exemplify the potential in targeting cell type enriched regulatory elements for controlling cell fate.\\nRESULTS\\ntitle_1\\nResults\\nRESULTS\\ntitle_2\\nCRISPRa-mediated reprogramming of NSCs and EEA targeting\\nFig1\\nFIG\\nfig_caption\\nCRISPRa-mediated reprogramming of NSCs and EEA-motif targeting. a Schematic representation of dCas9VPH structure. b Schematic representation of NSC reprogramming into iPSCs with dCas9VPH mediated OCT4 activation. c Immunocytochemical detection of pluripotency markers in NCS-derived iPSCs (top row) and tri-lineage differentiation in plated embryoid bodies (bottom row). Nuclei stained blue. Scale bar = 200 microm. d Targeting of EGA enriched Alu-motif with SpdCas9 gRNAs. e Quantification of iPSC-like alkaline phosphatase positive colonies induced from NSCs. n = 6 independent inductions (P = 0.053, OCT4 targeting with EEA-gRNAs vs. without EEA-gRNAs). Data presented as mean +- s.e.m., two-tailed Student\\'s t-test\\nRESULTS\\nparagraph\\nWe began human cell reprogramming with CRISPRa using a simplified reprogramming scheme. CRISPRa-mediated POU5F1 (OCT4) activation has been used to replace transgenic OCT4 in human fibroblast reprogramming, while the transgenic expression of only OCT4 has been shown to be sufficient for the reprogramming of neuroepithelial stem cells (NSCs) into iPSCs. We therefore combined CRISPRa-mediated OCT4 activation and NSC reprogramming as an initial model using trimethoprim (TMP) stabilized SpdCas9VP192 fused with P65-HSF1 activator domain (DDdCas9VPH) under doxycycline (DOX) inducible promoter (Fig. 1a, b). Expression of DDdCas9VPH and OCT4 targeting guides in iPSC-derived NSCs resulted in the emergence of pluripotent cells in a DOX and TMP dependent manner (Fig. 1c-e). These cells could be expanded into stable dCas9 independent cell lines (Fig. 1c and Supplementary Fig. 1). This demonstrated that CRISPRa mediated activation of endogenous OCT4 alone was sufficient to reprogram NSCs to iPSCs.\\nRESULTS\\nparagraph\\nTo determine if EEA-motif targeting could improve CRISPRa reprogramming of NSCs, we designed a set of five 14 nt gRNAs targeting the 36 bp EEA consensus sequence (Fig. 1d). Addition of the EEA-motif gRNAs in the reprogramming mixture demonstrated a trend in increasing the number of alkaline phosphatase (AP) positive colonies (P = 0.053, Student\\'s t-test) (Fig. 1e). This suggested that EEA-motif targeting could be useful for improving CRISPRa reprogramming efficiency.\\nRESULTS\\ntitle_2\\nPluripotency factor activation with CRISPRa\\nFig2\\nFIG\\nfig_caption\\nOptimization of dCas9 activator and gRNA targeting in HEK293 for reprogramming factor activation. a Locations of promoter targeting gRNAs for reprogramming factors (OCT4, SOX2, KLF4, C-MYC, LIN28A, and NANOG) in relation to transcription start site. b Immunocytochemical staining of reprogramming factors after single gRNA activation and pooled mixture of five guides in HEK293 with dCas9VPH. Pictures are in similar order to guides in Fig. 2a. Best performing guides used for plasmid cloning are marked with dotted lines. Scale bar = 400 microm. c Schematic representation of concatenated reprogramming factor gRNA plasmid construction. d Reprogramming factor activation by qRT-PCR, in HEK293 cells 3 days after transfection and HFFs 4 days after electroporation, using transiently expressed dCas9VPH effector. n = 3, data are from three independently treated samples. Data presented as mean +- s.e.m., two tailed Student\\'s t-test. *P < 0.05, **P < 0.01, ***P < 0.001\\nRESULTS\\nparagraph\\nTo devise a reprogramming system for fibroblasts based solely on CRISPRa, we optimized the promoter targeting of single gRNAs to the canonical reprogramming factors OCT4, MYC, KLF4, SOX2, LIN28A, and NANOG in HEK293 (Fig. 2a, b). Best performing gRNAs targeting OCT4, MYC, KLF4, SOX2, and LIN28A (OMKSL) promoters were concatenated into a single plasmid and tested in transfected HEK293 and human foreskin fibroblasts (HFFs) with dCas9VPH activator (Fig. 2c, d). Robust activation of all targeted genes could be detected in HEK293, whereas HFFs demonstrated robust activation of OCT4 and SOX2 but not the rest of the factors. This suggested that additional guides for KLF4, LIN28A, and MYC targeting would be required for efficient activation of these genes.\\nRESULTS\\ntitle_2\\nFibroblast reprogramming with CRISPRa\\nFig3\\nFIG\\nfig_caption\\nEEA-motif targeting enhances derivation of CRISPRa iPSCs from primary skin fibroblasts. a Schematic representation of skin fibroblast reprogramming with dCas9 activators. b Pluripotency factor expression in CRISPR-iPSC colonies (top row, scale bar = 400 microm) and tri-lineage differentiation markers for ectoderm (TUBB3), mesoderm (Vimentin and alpha-SMA), and endoderm (SOX17 and FOXA2) in embryoid bodies (middle row, scale bar = 200 microm) and teratomas (bottom row, scale bar = 800 microm). c Normal 46, XX karyotype of a CRISPRa iPSC line HEL139.2. d Principal component analysis of CRISPR iPSC lines, control PSC lines and HFFs based on expression of 123 significantly fluctuated genes. e Clustering of iPSC lines and HFFs based on expression of 85 significantly fluctuated and differentially regulated genes. f Clustering of CRISPR iPSC lines and control pluripotent stem cells based on DNA methylation. g Effect of VP192 and VPH domains and EEA-motif targeting on CRISPRa reprogramming efficiency of HFFs. n = 6 from three independent experiments. Data presented as mean +- s.e.m., two tailed Student\\'s t-test. **P < 0.01\\nRESULTS\\nparagraph\\nElectroporation of primary skin fibroblasts with episomally replicating dCas9VPH plasmid, containing TP53 targeting shRNA, EEA-motif targeting gRNA plasmid, reprogramming factor targeting gRNA plasmid (OMKSL), and an additional KLF4 and MYC targeting gRNA plasmid (KM) resulted in the emergence of iPSC-like colonies (Fig. 3a). The resulting colonies could be expanded into iPSC lines demonstrating typical pluripotency markers and differentiation into three germ layer derivatives in vitro and in vivo (Fig. 3b and Supplementary Fig. 2a, b). These CRISPRa-induced iPSC lines presented normal karyotypes (Fig. 3c and Supplementary Fig. 2c), absence of transgenic vectors (Supplementary Fig. 2d), and clustered separately from HFFs, together with control Sendai virus-derived iPSCs (HEL46.11) and H9 embryonic stem cells, by transcriptional (Fig. 3d, e) and DNA methylation profiles (Fig. 3f). Overall, this demonstrated that CRISPRa reprogramming can be used to derive fully reprogrammed iPSCs from human skin fibroblasts.\\nRESULTS\\nparagraph\\nIn order to optimize the CRISPRa reprogramming method, we tested the effect of the activator domains and the EEA-motif gRNAs on reprogramming efficiency using HFFs. CRISPRa reprogramming using dCas9VP192 and EEA-motif gRNAs resulted in the most efficient AP positive colony formation (up to 0.062% of electroporated cells) (Fig. 3g). EEA-motif targeting greatly enhanced the CRISPRa reprogramming efficiency, ranging from 10.5-fold increase with VP192 (P = 0.02, Student\\'s t-test) to 29.2-fold increase with VPH (P = 0.02, Student\\'s t-test) (Fig. 3g). This effect was more prominent in CRISPRa reprogramming than in transgenic reprogramming of HFFs (Supplementary Fig. 3a). The dCas9VP192 activator performed better than dCas9VPH when targeting only OCT4 for activation in an otherwise transgenic reprogramming approach (Supplementary Fig. 3b) or even when no gRNAs were present (Supplementary Fig. 3c). This suggested that the activator itself may interfere with the reprogramming process. We additionally tested P300 core fusions of the two activators, but they did not improve the reprogramming outcome (Supplementary Note 1 and Supplementary Fig. 4). As dCas9VP192 appeared to perform best in the CRISPRa reprogramming of human fibroblasts, it was used in the subsequent experiments.\\nRESULTS\\ntitle_2\\nTranscriptional analysis of CRISPRa reprogramming\\nFig4\\nFIG\\nfig_caption\\nTranscriptional analysis of reprogramming cell populations. a Schematic representation of skin fibroblast reprograming with dCas9VP192 for RNA sequencing samples. b Clustering of all reprogramming samples based on expression of 4972 significantly fluctuated genes. Group 6 represents genes upregulated specifically by EEA-guides at day 12. c Upregulated genes at day 12 of reprogramming in OMKSL + KM + EEA targeted cells (Group 6). d Clustering of samples within day 4, 8, and 12 time points based on differentially regulated genes\\nRESULTS\\nparagraph\\nTo decipher the mechanism behind the increase in reprogramming efficiency mediated by the EEA-motif targeting, we conducted expression profile analysis of HFF cell populations undergoing CRISPRa-induced reprogramming in the presence and absence of the EEA-motif gRNAs (Fig. 4a). Based on fluctuated genes in the full data set, the samples clustered primarily by induction date (Fig. 4b). Additionally, by day 12 the OMKSL + KM + EEA gRNA treated cells clustered separately from the rest of the samples and demonstrated higher expression of 78 genes primarily associated with pluripotency and TGF-beta signalling (Group 6 in Fig. 4b, c).\\nRESULTS\\nparagraph\\nThe bulk RNA samples represent heterogeneous cell populations where the majority of the cells are not undergoing complete reprogramming. The clustering seen by induction day may thus reflect nonspecific responses of the fibroblasts to handling, e.g., electroporation. Therefore, we also compared the samples within each time point. This revealed a common set of higher expressed genes on day 4 in the conditions containing EEA-motif targeting gRNAs (OMKSL + KM + EEA and EEA, Fig. 4d). These EEA-associated genes had a significantly higher number of EEA-gRNA 1 (EEA-g1) binding sites near their upstream regions (-10 kb to +1 kb from TSS) (mean 0.409 per kb, n = 18, P = 5 x 10-5, vs. genomic mean 0.215 per kb for protein coding genes, Monte Carlo sampling). This suggested a preferential initial activation of genes with multiple EEA gRNA target sites. A set of EEA-associated genes was also seen expressed higher in the day 8 samples (Fig. 4d), but these genes did not show enrichment for EEA-g1 sites (mean 0.280 per kb, n = 13, P = 0.068, Monte Carlo sampling). Significant enrichment was also not detected in the genes that were expressed higher by OMKSL + KM only on day 4 (mean 0.224 per kb, n = 15, P = 0.314, Monte Carlo sampling) (Fig. 4d), or in the pluripotency-associated genes that were expressed higher on day 12 (Group 6, mean 0.248, n = 78, P = 0.070, Monte Carlo sampling) (Fig. 4c). However, the day 8 higher expressed EEA-associated genes included factors like TRIM25, linked to LIN28A function, and DROSHA and CCND1 which have been associated with cellular reprogramming (Fig. 4d). EEA-motif targeting at the mid stages of induction may thus contribute to more efficient expression of factors that can promote reprogramming, even if enrichment of EEA-g1 sites was not detected. Unlike day 4 and day 8 higher expressed genes, day 12 genes did not show division between EEA related and OMKSL + KM related sets (Fig. 4d), suggesting that the EEA-motif targeting primarily affects the initial stages of the reprogramming process prior to colony formation.\\nRESULTS\\ntitle_2\\nNANOG and REX1 are EEA-associated reprogramming factors\\nFig5\\nFIG\\nfig_caption\\nEEA-motif targeting improves NANOG and REX1 activation. a Immunostaining of emerging iPSC colonies detects activation of targeted LIN28A before its mRNA reads become detectable in bulk RNA-seq. Scale bar = 400 microm. b HFF reprogramming efficiency after transgenic expression of additional factors belonging to Group 6 (Fig. 4c). n = 5, three independent experiments. c qRT-PCR quantification of NANOG and REX1 activation using dCas9VP192 with and without EEA-motif gRNAs in HEK293. n = 3, three independent experiments. d NANOG activation can be detected by immunostaining in CRISPRa reprogramming colonies 2 days earlier in the presence of EEA-motif targeting guides. Scale bar = 200 microm. Data presented as mean +- s.e.m., two tailed Student\\'s t-test. **P < 0.01, ***P < 0.001\\nRESULTS\\nparagraph\\nDetection of transcriptional changes occurring in small subsets of reprogramming cells can be challenging using RNA-seq of bulk mRNA. This was evident from the absence of detectable LIN28A reads from some of the sequencing samples, although LIN28A protein could clearly be detected by immunostaining in the forming colonies (Fig. 5a). This could also lead to poor detection of reprogramming factors targeted by EEA gRNAs. Assuming the EEA-associated reprogramming factors stay expressed in pluripotent cells, they should be detected more reliably in the day 12 samples, as the fibroblast background is diminished due to expansion of the reprogramming colonies (Fig. 5a). We therefore chose to test a set of seven pluripotency associated factors upregulated by day 12 in the OMKSL + KM + EEA reprogramming data set (Group 6) for their ability to enhance the reprogramming efficiency (Figs 4c and 5b). Transgenic expression of NANOG and REX1 in CRISPRa reprogramming in the absence of EEA-motif gRNAs, using optimized reprogramming factor gRNA plasmid (Supplementary Fig. 5), resulted in improved reprogramming efficiency (Fig. 5b). This indicated that NANOG and REX1 could be mediating the EEA-motif targeting effect. Assuming these factors are downstream effectors of EEA-motif targeting, their direct activation should also be enhanced by EEA-motif gRNAs. Accordingly, dCas9VP192 mediated activation of both NANOG and REX1 promoters in transiently transfected HEK293 resulted in higher expression of these genes in the presence of the EEA-motif gRNAs compared with a TdTomato targeting control gRNA (P = 0.009 for NANOG and P = 0.004 for REX1, Student\\'s t-test) (Fig. 5c). Both NANOG and REX1 loci contain EEA-g1 binding sites near the genes (Supplementary Fig. 6a, b). REX1 expression was improved by EEA-gRNAs even when REX1 gRNAs were replaced with NANOG gRNAs, whereas NANOG expression was not improved by EEA-gRNAs in the presence of REX1 gRNAs (Supplementary Fig. 6c). Therefore, REX1 activation by targeting of the EEA-gRNA site near its promoter may represent a direct activation effect, possibly aided by NANOG mediated targeting of REX1. On the other hand, NANOG activation may be more dependent on additional reprogramming factors or NANOG promoter targeting by dCas9 activators. Both NANOG and REX1 thus appear to be downstream targets of the EEA-motif gRNAs, contributing to its effect on improving reprogramming efficiency. Consistent with this, NANOG protein expression could be detected 2 days earlier in reprogramming colonies in the presence of the EEA-motif targeting gRNAs (Fig. 5d).\\nRESULTS\\nparagraph\\nWe additionally tested the rest of the reprogramming factors (OCT4, SOX2, KLF4, MYC, and LIN28A), and a set of non-pluripotency associated genes to determine if their activation is affected by simultaneous EEA-motif targeting in HEK293. However, this did not result in improved transcriptional activation (Supplementary Fig. 6d, e). The EEA-motif targeting thus appears specific only to a subset of reprogramming factors.\\nRESULTS\\ntitle_2\\nMechanism of EEA-motif targeting\\nFig6\\nFIG\\nfig_caption\\nEGA-enriched Alu-motif targeting is associated with activation and chromatin opening. a Effect of single EEA-motif targeting gRNAs (Fig. 1e) on CRISPRa reprogramming efficiency of HFFs. n = 3, data are from three independent experiments. b Schematic representation of EEA-motif reporter plasmid activation. c qRT-PCR quantification of mCherry reporter activation with EEA-motif gRNAs 1 and 2 in reporter transfected HEK293 relative to five gRNAs. EEA-g1 activates the reporter with higher efficiency than EEA-g2 (P = 0.02). n = 3, three independent experiments. d Expression of Alu derived transcripts in STRT-seq data of pluripotent stem cells and CRISPRa reprogramming cell populations. Centre line, box, and whiskers represent median, quartiles, and 1.5 IQR, respectively. e Effect of dCas9 fusion domain on reprogramming efficiency in transgenic reprogramming with OCT4, SOX2, KLF4, LIN28, L-MYC, and EEA-g1. n = 6, data are from three independent experiments. f Increase in the presence of EEA-g1 sequence in ATAC-seq peaks in TetON-DDdCas9GFP and EEA-g1 expressing HEK293. ctrl n = 3, DOX + TMP n = 5. g Effect of dCas9 fusion domain on reprogramming efficiency in transgenic reprogramming with OCT4, SOX2, LIN28, L-MYC, and EEA-g1. n = 6, data are from three independent experiments. h Schematic model of EEA-motif targeting in gene activation and reprogramming. Data presented as mean +- s.e.m., two tailed Student\\'s t-test. *P < 0.05, **P < 0.01\\nRESULTS\\nparagraph\\nTo further dissect the mechanism behind the EEA-motif targeting in CRISPRa reprogramming, we individually tested all the five guides targeting the EEA-motif. We also included control guides targeting common guide sequences found in human pluripotent stem cell super enhancers, to rule out nonspecific global DNA targeting. Of note, the most common gRNAs in these areas also contained multiple Alu sequences and the control guides 8, 9, and 10 also targeted parts of the EEA-motif. Reprogramming efficiency was mainly dependent on the EEA-motif guide 1 (Fig. 6a), which promoted as efficient reprogramming as the five guides together (Supplementary Fig. 7a). There was a noticeable difference in reprogramming efficiencies between EEA-g1 and EEA-g2, which are located next to each other in the EEA-motif consensus (Fig. 1e). These differences may be explained by guide nucleotide composition, as EEA-g2 contained multiple PAM proximal nucleotides that have been shown to be disfavoured. In accordance with this, EEA-g1 activated a reporter construct with the EEA-motif consensus sequence with higher efficiency than EEA-g2 (P = 0.02, Student\\'s t-test) (Fig. 6b, c and Supplementary Fig. 7b). The gRNA nucleotide sequence affecting its efficiency may thus be a crucial determinant in the EEA-motif targeting effect.\\nRESULTS\\nparagraph\\nAs the EEA-motif is located in the left arm of the Alu consensus sequence, we also assessed the expression of Alu sequences that could be detected in the STRT RNA-seq data in the pluripotent stem cells and the reprogramming samples. Alu expression was higher in pluripotent cells than in fibroblasts (Fig. 6d). In reprogramming samples Alu expression peaked initially in all day 4 samples, possibly as a response to electroporation, and thereafter decreased (Fig. 6d). Alu expression rose again in the day 12 samples of the OMKSL + KM + EEA samples with higher numbers of pluripotent cell colonies. Overall, EEA-motif targeting did not appear to affect Alu expression in the reprogramming cell batches. Therefore, Alu expression itself is unlikely to explain the potentiating effect of EEA-motif targeting on reprogramming.\\nRESULTS\\nparagraph\\nWe next tested the impact of different dCas9-fused effector domains on the EEA-g1 effect in conventional reprogramming with transgenic OCT4, SOX2, KLF4 LIN28, and L-MYC. Interestingly, the effector domain did not have a significant impact on the reprogramming efficiency, whereas absence of the dCas9 protein resulted in reduced reprogramming efficiency compared to the dCas9VP192 control (P = 0.007, Student\\'s t-test) (Fig. 6e). The effect of the EEA-motif targeting therefore appears to be mediated specifically by dCas9. It is possible, that dCas9 binding to high affinity guide sites in the EEA-motif may disrupt the chromatin locally to mediate more efficient activation of adjacent genes. As dCas9 binding to DNA has been demonstrated to open chromatin near its target site, we performed ATAC-seq on samples of HEK293 cells expressing the EEA-g1 and a DOX- and TMP- inducible version of the DDdCas9GFP protein. DOX and TMP treated samples were found to have increased percentage of peaks with overlapping EEA-g1 sites (0.8% in controls, n = 3, vs. 1.9% in treated cells, n = 5, P = 0.011, Student\\'s t-test) (Fig. 6f). This supports a mechanistic model in which dCas9 binding to high efficiency guide sites in the EEA-motif can lead to interference of the local chromatin structure near these elements, which may then contribute to the reprogramming process. As the EEA-gRNA-mediated increase in reprogramming efficiency is weaker in transgenic reprogramming than in CRISPRa reprogramming (Fig. 3g and Supplementary Fig. 3a), we additionally tested the EEA-g1 in suboptimal transgenic reprogramming conditions in the absence of transgenic KLF4. This resulted in complete abrogation of AP positive colony formation except in the presence of dCas9 activators targeting the EEA-motif (Fig. 6g and Supplementary Fig. 7c). Therefore, the activation function of the different dCas9 effectors appears to be important for the CRISPRa reprogramming process, since colonies only formed in dCas9VP192 and dCas9VPH containing conditions. This could be explained by improved activation of pluripotency factors with abundant EEA-motif sequences near them (Fig. 6h). These factors, such as NANOG, may help overcome the absence of KLF4 overexpression, as has been demonstrated by the use of alternative transgenic human reprogramming factor combinations (e.g., OCT4, SOX2, NANOG, and LIN28A).\\nRESULTS\\nparagraph\\nFinally, we tested if the fibroblast CRISPRa reprogramming system could be transferred into inducible transposon-based vectors. To this end, we inserted the DDdCas9VP192 activator, under a DOX-inducible promoter, into a PiggyBac vector and the OSK2M2L1 cassette and five guides for the EEA-motif into another one (Supplementary Fig. 5a and Supplementary Fig. 8a). HFFs electroporated with the PiggyBac vectors formed AP positive colonies, which could be expanded into stable iPSC lines, differentiated into three embryonic germ layer derivatives, and re-induced upon DOX and TMP addition (Supplementary Fig. 8). This demonstrated the applicability of the CRISPRa reprograming system in primary cells using various plasmid vectors and the option for the establishment of secondary reprogrammable systems based on CRISPRa.\\nDISCUSS\\ntitle_1\\nDiscussion\\nDISCUSS\\nparagraph\\nCRISPR activator approaches hold great potential for controlling cellular reprogramming. The high multiplexing capacity of the system allows simultaneous targeting of large numbers of endogenous genes and genomic control elements using only short guide RNA molecules. This type of approach, combined with large scale synthesis of nucleic acids, can enable comprehensive targeting of gene regulatory networks with great precision for controlling cellular fate. However, until now no robust methods have been described for reprogramming human cells into pluripotency by CRISPRa.\\nDISCUSS\\nparagraph\\nWe present a method for the efficient conversion of primary human fibroblasts into bona fide iPSCs based entirely on the transcriptional control of endogenous genes by CRISPRa. Activation of core reprogramming factor promoters alone was sufficient but inefficient, whereas additional targeting of a common Alu element brought the efficiency close to established reprogramming methods (Supplementary Fig. 5c). The more complex activator domains did not improve reprogramming efficiency, which mirrors previously reported results for gene activation, and suggests that the benefit of simple additional fused activation domains may be limited.\\nDISCUSS\\nparagraph\\nIt has been estimated that 13% of human genome consists of Short Interspersed Nuclear Element (SINE) sequences, including Alu elements. Accordingly, EEA-g1 sites can be found in more than 360,000 sites in the human genome. Due to the high abundance of EEA-g1 sites, the motif itself would not be expected to mediate a very strong or specific effect. This is also apparent from the inability of the EEA-gRNAs to reprogram cells by themselves. However, enrichment of the motif sequences near genes may end up enhancing the effect of the motif, as seen in the higher expression of the genes at day 4 that contain multiple EEA-motifs. Although we did not detect significant enrichment of EEA-g1 sites near the 78 pluripotency associated genes upregulated at day 12 (Group 6), Alu family repeats in general have been reported to be enriched near promoters of pluripotent stem cell expressed genes. In the reprogramming context, this may end up biasing the EEA-motif targeting to preferentially affect pluripotency factors. However, in bulk RNA data this effect may be masked by the background of non-reprogramming cells, and therefore more thorough characterization of the EEA-motif targeting effect will require enrichment for the cell populations that undergo successful reprogramming.\\nDISCUSS\\nparagraph\\nThe EEA-motif targeting effect on reprogramming appears to be associated with dCas9 effector-mediated opening of local chromatin, that promotes pluripotency factor activation. Alternatively, it is possible that dCas9 binding to the EEA-motif interferes with other possibly repressive factors targeting the motif. Alu elements have been linked with insulator function, including those near KRT18, which was upregulated at day 8 by EEA-motif targeting (Fig. 4d). Therefore, the opening of these elements, or interference with their function, may contribute to more efficient activation of nearby genes by interfering with chromatin insulation. Further insight into the mechanisms will require characterisation of factors binding to the motif during reprogramming as well as characterization of the function of early embryo factors, which are known to target the motif, e.g., PRD-like totipotent cell homeodomain factors and HNF4alpha.\\nDISCUSS\\nparagraph\\nIn conclusion, CRISPRa reprogramming will provide a powerful tool for inducing pluripotent cells. The core method described here can be further improved by targeting known pluripotency genes and regulatory elements, as well as by screening for novel reprogramming factors and elements. This will pave way for the development of more comprehensive CRISPRa reprogramming strategies, which in combination with transgenic factors, RNAi, and small molecular compounds, will promote more efficient and specific reprogramming of human cells for future applications.\\nMETHODS\\ntitle_1\\nMethods\\nMETHODS\\ntitle_2\\nEthical consent\\nMETHODS\\nparagraph\\nThe generation of the human induced pluripotent stem cell lines used in this study was approved by the Coordinating Ethics Committee of the Helsinki and Uusimaa Hospital District (Nro 423/13/03/00/08) with informed consent of the donors.\\nMETHODS\\ntitle_2\\nCell culture\\nMETHODS\\nparagraph\\nHEK293 cells (ATCC line CRL-1573), human foreskin fibroblasts (HFFs, ATCC line CRL-2429), and adult human dermal fibroblasts (derived in house) were cultured in fibroblast medium (Dulbecco\\'s modified Eagle\\'s medium (DMEM; Life Technologies) containing 10% fetal bovine serum (FBS; Life Technologies), 2 mM GlutaMAX (Life Technologies), and 100 microg/ml penicillin-streptomycin (Life Technologies)). Human induced pluripotent cells and embryonic stem cells were cultured on Matrigel (BD Biosciences) coated plates in E8 medium (Life Technologies) and split using 0.5 mM EDTA. Medium was changed every other day. All cells were kept in an incubator at 37  C and 5% CO2 and tested negative for mycoplasma.\\nMETHODS\\ntitle_2\\nGuide RNA design and production\\nMETHODS\\nparagraph\\nGuide RNAs were designed and assembled as described by Balboa et al.. Briefly, guide RNA expression cassettes, containing U6 promoter, chimeric single guide RNA and a Pol III terminator were assembled by PCR and concatenated into plasmids using Golden Gate assembly. Concatenated guide sets were cloned into episomal OriP-EBNA1 containing plasmids for reprogramming experiments. A list of guide RNA oligonucleotides is provided in the Supplementary Table 1.\\nMETHODS\\ntitle_2\\ndCas9 activator plasmid construction\\nMETHODS\\nparagraph\\ndCas9VPH construct was cloned by adding a P65-HSF1 containing fragment from lenti-MS2-P65-HSF1_Hygro (gift from Feng Zhang, Addgene Plasmid #61426) after the VP192 domain by PCR. dCas9VPP300 was cloned by PCR amplifying the P300 core domain from human cDNA and cloning it after the VP192 domain, as described by Hilton et al.. dCas9VPPH was cloned by adding the P65-HSF1 domain in fusion after the VP192-P300 core domain. Activator plasmids were first cloned into CAG-dCas9VP192-T2A-GFP-IRES-Puro backbone and further cloned into pCXLE-dCas9VP192-T2A-EGFP-shP53 (Addgene plasmid #69535) backbone with XhoI and BsrGI. Plasmids used in this study will be made available on Addgene https://www.addgene.org/Timo_Otonkoski/ see also Supplementary Table 2.\\nMETHODS\\ntitle_2\\nCell transfection\\nMETHODS\\nparagraph\\nHEK293 cells were seeded on tissue culture treated 24 well plates 1 day prior to transfection (105 cells/well). Cells were transfected using 4:1 ratio of FuGENE HD transfection reagent (Promega) in fibroblast culture medium with 500 ng of dCas9 transactivator encoding plasmid and 100-200 ng of guide RNA-PCR or 250 ng of dCas9 transactivator encoding plasmid and 250 ng of concatenated guide RNA encoding plasmid. Cells were cultured for 72 h post-transfection, after which samples were collected for qRT-PCR or immunocytochemical staining. HEK293 cells containing the destabilized dCas9 activators and guides were transfected with dCas9 activator, guide RNA and PiggyBac transposase plasmids, 100 ng of each, and selected with Puromycin (2 mug/ml; Sigma) and G418 (0.5 mg/ml; Life Technologies).\\nMETHODS\\ntitle_2\\nQuantitative reverse transcription PCR\\nMETHODS\\nparagraph\\nTotal RNA was extracted from cells using NucleoSpin Plus RNA kit (Macherey-Nagel). RNA quality and concentration was measured by spectrophotometry using SimpliNano (General Electric). One microgram of total RNA was denatured at 65  C for 1 min and used for reverse transcription (RT) with 0.5 muL Moloney murine leukemia virus (MMLV) reverse transcriptase (M1701, Promega), 0.2 muL Random Primers (C1181, Promega), 1 muL Oligo(dT)18 Primer (SO131, ThermoFIsher) and 0.5 muL Ribolock RNAse inhibitor (EO0382, ThermoFisher) for 90 min at 37  C. For qRT-PCR reactions, 50 ng of retrotranscribed RNA were amplified with 5 muL of forward and reverse primer mix at 2 muM each using 5x HOT FIREPol EvaGreen qPCR Mix Plus (no ROX) in a final volume of 20 muL. QIAgility (Quiagen) liquid handing system was used for pipetting the reactions into 100 well disc that were subsequently sealed and run in Rotor-Gene Q (Qiagen) with a thermal cycle of 95  C for 15 min, followed by 40 cycles of 95  C, 25 s; 57  C, 25 s; 72  C, 25 s, followed by a melting step. Relative quantification of gene expression was analysed using DeltaDeltaCt method, with cyclophilin G (PPIG) as endogenous control and an exogenous positive control used as calibrator. Expression levels are relative to non-treated cells or to hESC as indicated in the figure legends. A list of primers used is provided in the Supplementary Table 3.\\nMETHODS\\ntitle_2\\nNSC differentiation\\nMETHODS\\nparagraph\\nHuman NSCs were derived by differentiating human iPSC HEL24.3, and HEL46.11 lines using small molecule cocktail as described elsewhere, with minor adjustments. Briefly, iPSCs were detached with StemPro Accutase (Thermo Fisher Scientific) and dissociated gently into single cells suspension in hES-medium in the presence of 5 muM ROCK inhibitor (ROCKi; Y-27632, Selleckchem), 10 muM SB431542 (SB; S1067, Selleckchem), 1 muM dorsomorphin (DM; P5499-5MG, Sigma), 3 muM CHIR-99021 (CHIR; Tocris) and 0,5 muM purmorphamine (PMA; 04-0009, Stemgent) After 2 days, medium was changed to N2B27 medium (DMEM/F12:Neurobasal (1:1) supplemented with N2 and B27 without vitamin A, NEAA, PenStrep (all Thermo Fisher Scientific) and heparin (2 microg/ml; H3149-50KU, Sigma)) containing the same small molecule cocktail as above. On day 4, SB and DM were withdrawn and 150 microM ascorbic acid (AA) was added to N2B27. On day 6, the neurospheres were dissociated with 1 ml pipette and plated on Matrigel in N2B27 media containing AA, CHIR and PMA (growth media). First two passages were split at 1:3 ratio and cells were plated into growth media containing 5 muM ROCKi, which was removed next day. Later passages were split with 1:10 and 1:20 ratio using StemPro Accutase. Media were changed every other day.\\nMETHODS\\ntitle_2\\nNSC reprogramming\\nMETHODS\\nparagraph\\nNSCs were grown for at least five passages before electroporation. For electroporation, cells were detached with StemPro Accutase and dissociated into single cells. Cells were washed once with PBS and electroporated with Neon Transfection system (Invitrogen). Two million cells were used per electroporation using 100 microl tips with 1300 V, 30 ms, one pulse settings. A quantity of 2 mug of PB-tight-DDdCas9VPH-GFP-IRES-Neo activator plasmid, 1.5 mug PB-GG-OCT4-1-5-PGK-Puro gRNA plasmid, and 1.5 mug PB-GG-EEA-5g-PGK-Puro gRNA plasmid were used with 0.5 mug PiggyBac rtTA and 0.5 mug of PiggyBac transposase plasmids. One million electroporated cells were plated per 35 mm plate coated with Matrigel in N2B27 media supplemented with 5 muM ROCKi and 10 ng/ml of basic FGF (bFGF, PeproTech). ROCKi was removed the next day. Two days after electroporation cells were treated with Puromycin (0.5 mug/ml; Sigma) and G418 (200 mug/ml; Life Technologies) for 5 days. On day 8 after electroporation reprogramming was initiated by adding doxycycline (DOX, 2 mug/ml; Sigma) and trimethoprim (TMP, 1 muM; Sigma). After 5 days of induction media was changed to hES-medium gradually over a week. During the conversion process cells were split 3 times. On day 18 of induction cells were fixed with 4% paraformaldehyde (PFA) for AP staining or picked for iPSC derivation. Media were changed every other day.\\nMETHODS\\ntitle_2\\nFibroblast reprogramming\\nMETHODS\\nparagraph\\nHuman skin fibroblasts were detached as single cells from the culture plates with TrypLE Select (Gibco) and washed with PBS. Cells were electroporated using the Neon transfection system (Invitrogen). A total of 106 cells and 6 microg of plasmid mixture, containing 2 microg of dCas9 activator plasmid and 4 microg of guide plasmids, were electroporated in a 100 microl tip with 1650 V, 10 ms, and 3x pulse settings. Electroporated fibroblasts were plated on Matrigel coated 100 mm diameter cell culture plates in fibroblast medium. After 4 days cell culture medium was changed to 1:1 mixture of fibroblast medium and hES-medium (KnockOut DMEM (Gibco) supplemented with 20% KO serum replacement (Gibco), 1% GlutaMAX (Gibco), 0.1 mM beta-mercaptoethanol, 1% nonessential amino acids (Gibco), and 6 ng/ml basic fibroblast growth factor (FGF-2; Sigma)) supplemented with sodium butyrate (0.25 mM; Sigma). When first colonies started to emerge, cell culture medium was changed to hES-medium until colonies were picked. For iPSC line derivation, colonies were picked manually and plated on Matrigel coated wells in E8 medium. Media were changed every other day. Reprogramming with transgenic transcription factors was performed as described elsewhere using pCXLE-OCT3/4-shP53, pCXLE-hSK, pCXLE-hUL (Addgene plasmid #27077, #27078, #27080) and pCXLE-OCT4 or pCXLE-SOX2 derived from pCXLE-OCT3/4-shP53 and pCXLE-hSK, respectively. For PiggyBac reprogramming, HFFs were electroporated as described above with PB-tight-DDdCas9VP192-GFP-IRES-Neo, PB-CAG-rtTA-IRES-Neo, PB-GG-EEA-5g-OSK2M2L1-PGK-Puro, and PiggyBac transposase plasmids. Electroporated cells were plated on cell culture dishes in fibroblast medium. Five days after electroporation cells were selected with Puromycin (1 mug/ml; Sigma) and G418 (0.5 mg/ml; Roche) for 2 days after which the selection antibiotic amounts were halved. Selected cells were induced as described above in the presence of TMP (1 microM) and DOX (2 microg/ml). Fresh DOX was supplemented daily. For RNA sequencing samples, passage 10 foreskin fibroblasts were electroporated with pCXLE-dCas9VP192-GFP-shP53 and combinations of GG-EBNA-OSKML-PP, GG-EBNA-KM-PP, and GG-EBNA-EEA-5guides-PP plasmids. A total of 250k (day 4 samples) to 125k cells (day 8 and 12 samples) were plated on Matrigel coated six-well culture plates per well and induced as described above.\\nMETHODS\\ntitle_2\\nPluripotent cell line derivation\\nMETHODS\\nparagraph\\nHEL139 clones were derived from adult female skin fibroblasts (F72) using GG-EBNA-OMKSL-PP, EBNA-EEA-5guides-PGK-Puro, and an additional GG-EBNA-KM-PP plasmid (KLF4 and MYC five guides each). HEL140 was derived from neonatal male skin fibroblasts (HFFs) with GG-EBNA-OMKSL-PP, EBNA-EEA-5guides-PGK-Puro, and GG-EBNA-KM-PP plasmids. HEL141 was derived from neonatal male skin fibroblasts (HFFs) with GG-EBNA-EEA-5guides-PGK-Puro, GG-EBNA-KM-PP, and GG-EBNA-OS-PP plasmids (OCT4 and SOX2 five guides each). The above cell lines were derived using pCXLE-dCas9VPH-T2A-GFP-shP53 activator plasmid. HEL144 was derived from neonatal male skin fibroblasts (HFFs) with inducible PiggyBac vectors using PB-tight-DDdCas9VP192-GFP-IRES-Neo activator and PB-EEA-5g-OSK2M2L1-PGK-Puro guides. Control cell line HEL46.11 was derived from HFFs using CytoTune Sendai Reprogramming Kit (Life Technologies) according to manufacturer\\'s instructions.\\nMETHODS\\ntitle_2\\nAlkaline phosphatase staining\\nMETHODS\\nparagraph\\niPSC colonies were fixed with 4% paraformaldehyde (PFA) solution for 10 min and washed with phosphate buffered saline (PBS). Thereafter cells were stained in NBT/BCIP (Roche) containing buffer (0.1 M Tris HCl pH 9.5, 0.1 M NaCl, 0.05 M MgCl2) until precipitate developed. Reaction was stopped by washing the plates with PBS.\\nMETHODS\\ntitle_2\\nImmunocytochemistry\\nMETHODS\\nparagraph\\nCells were fixed with 4% PFA, permeabilized using 0.2% Triton X-100, and treated with Ultra Vision block (ThermoFisher). Primary antibodies were diluted in 0.1% Tween-20 PBS and incubated either overnight at 6  C with the given dilutions or 2 days in 6  C with halved primary antibody amounts. Secondary antibody incubations were done in room temperature for 30 min in the presence of Hoechst33342 to stain the nuclei. Primary antibodies used were: LIN28A (1:250, D84C11 and D1A1A, Cell Signaling), NANOG (1:250, D73G4, Cell Signaling), OCT4 (1:500, sc-8628, Santa Cruz), SOX2 (1:250, D6D9, Cell Signaling), KLF4 (1:250, HPA002926, Sigma-Aldrich), C-MYC (1:250, D3N8F, Cell Signaling; 1:250, [Y69] ab32072, Abcam), TRA-1-60 (1:50, MA1-023, ThermoFisher), TRA-1-81 (1:100, MA1-024, ThermoFisher) TUBB3 (1:500, MAB1195, R&D Systems), AFP (1:400, A0008, Dako), SMA (1:200, A2547, Sigma), VIMENTIN (1:500, sc-5565, Santa Cruz). SOX17 (1:500, AF1924, R&D Systems), acetyl Histone 3 (1:500, ab47915, Abcam). Secondary antibodies used were: AlexaFluor 488: donkey anti-goat (1:500, A11055 and 11058; Invitrogen), donkey anti-mouse (1:500, A21202 and A21203; Invitrogen) and donkey anti-rabbit (1:500, A21206 and A21207; Invitrogen).\\nMETHODS\\ntitle_2\\nEmbryoid body assay\\nMETHODS\\nparagraph\\niPSCs were split into small clumps and plated on low attachment dishes (Corning) in hESC medium without bFGF to allow embryoid body (EB) formation. The EB culture medium was supplemented overnight with 5 microM ROCK inhibitor (Y-27632, Selleckchem) after the initial cell plating to improve cell viability. Medium was changed every other day. EBs were grown in suspension for 14 days, after which they were plated on gelatin coated cell culture dishes. EBs were allowed to form outgrowths for 7 days after which cells were fixed with 4% PFA for 30 min and permeabilized using 0.2% Triton X100 (Sigma) in PBS for 30 min. Fixed and permeabilized EB outgrowths were stained as described above.\\nMETHODS\\ntitle_2\\nTeratoma assay\\nMETHODS\\nparagraph\\nAbout 200,000 morphologically intact iPSCs at passage 23 were intratesticularly injected into male NMRI nude mice (Scanbur). The resulting tumours were collected 2 months after injection, fixed with 4% PFA, and hematoxylin and eosin stained. Animal care and experiments were approved by the National Animal Experiment Board in Finland (ESAVI/9978/04.10.07/2014).\\nMETHODS\\ntitle_2\\nDNA methylation assay and analysis\\nMETHODS\\nparagraph\\nFor DNA methylation array samples, passage 11-20 CRISPRa iPSCs, passage 35 SeV iPSCs, and passage 50 H9 ESCs were collected, one well of six-well plate each, as well as HFF control fibroblasts. DNA was purified using DNeasy Blood & Tissue Kit (Qiagen) and the concentrations were adjusted to 11 ng/mul using Qubit assay (Thermo Fisther Scientific). PicoGreen Assay (ThermoFisher) was used for subsequent normalization of samples. DNA of the samples and three controls (Zymo_low, Zymo_high and 1331-1 CEPH) was treated with sodium bisulphite using the EZ DNA methylation kit (Zymo Research). DNA methylation was quantified using the Illumina Infinium HumanMethylationEPIC BeadChip on an Illumina iScan System using the manufacturer\\'s standard protocol. Raw IDAT files were processed with Illumina\\'s GenomeStudio v2011.1, and normalized beta-values, except two controls (Zymo_low and Zymo_high), were applied for the clustering of the DNA methylation profile.\\nMETHODS\\ntitle_2\\nSTRT-sequencing\\nMETHODS\\nparagraph\\nRNA samples were collected in Trizol Reagent (Life Technologies), 100 microl of chloroform per 500 mul of sample was added and mixed with Trizol Reagent. After centrifugation (12,000 g for 15 min) the transparent upper phase was collected and RNA was further purified using NucleoSpin RNA kit (Macherey-Nagel). RIN values were measured by Bioanalyzer (Agilent) and total-RNA concentrations scaled to equal (10 ng/mul) using Qubit assay (Thermo Fisher Scientific). Bulk-RNA transcriptome analysis was performed by the STRT RNA-seq method with minor modifications. Briefly, 10 ng of high-quality input RNA was converted to cDNA and amplified to form an Illumina-compatible 46-plex library. In total, 25 PCR cycles were used, but as six base-pair unique molecular identifiers (UMIs) were applied, only the absolute number of unique reads was calculated per analyzed sample. The library was sequenced on three lanes of Illumina HiSeq2000 instrument.\\nMETHODS\\ntitle_2\\nSTRT data analysis\\nMETHODS\\nparagraph\\nThe sequenced raw STRT reads were processed by STRTprep; v3dev branch, d7efcde commit (https://github.com/shka/STRTprep/tree/v3dev). In brief, redundant reads after demultiplexing were excluded according to UMI, and the nonredundant reads were aligned to hg19 human reference genome sequences, ERCC spike-in sequences, and human ribosomal DNA unit (GenBank: U13369). Uniquely mapped reads within (1) the 5\\'-UTR or the proximal upstream (up to 500 bp) of the RefSeq protein coding genes, (2) the 5\\'-UTR or the proximal upstream of some PRD genes, which were not yet defined by RefSeq, and (3) within the first 50 bp of spike-in sequences, were counted. The processed reads were aligned also to Alu canonical sequence (http://www.repeatmasker.org/AluSubfamilies/humanAluSubfamilies.html) using the same methodology with STRTprep, to count Alu transcripts.\\nMETHODS\\nparagraph\\nSignificance of fluctuation on gene expression was tested by comparison with fluctuation of spike-in levels as shown in ref. . Principal component analysis (PCA) was performed on fluctuated genes (corrected p-value <0.05) via PCA plugin of STRTprep. Differential expression between the sample types were tested by SAMstrt. Regulated genes were selected by corrected fluctuation p-value <0.05 (as significant degree of expression change) and differential expression q-value <0.05 (as significant contrast between the types). Hierarchical clustering of normalized expression profiles and the illustration were performed by aheatmap function in NMF package via heatmap_diffexp plugin of STRTprep; color gradient in the heatmap represents Z-score of each gene, and the profile was clustered by Ward\\'s algorithm on Spearman correlation based distance. The fluctuated genes can be independent from the sample types (e.g., batch, circadian, cell-cycle etc.), while the regulated genes are selected by differential expression between the sample types. Therefore, the unsupervised clustering of samples by fluctuated genes (PCA and the Fig. 4b) reflects the major differences among the samples, while clustering using the regulated genes aims for grouping based on these genes.\\nMETHODS\\nparagraph\\nHEL136 cell line was excluded from sequencing analysis due to missing karyotype. One HEL46.11 control sample was excluded from sequencing analysis due to higher expression of differentiation associated genes, indicative of differentiated cells in the sample well.\\nMETHODS\\ntitle_2\\nATAC-sequencing\\nMETHODS\\nparagraph\\nHEK293 cell lines used for ATAC-seq samples were transfected with PiggyBac vectors for TetON-DDdCas9GFP-IRES-Neo, CAG-rtTA-IRES-Neo, and 36bp-guide1-PGK-Puro. Cells were selected with Puromycin and G418, after which cells were sub cloned from single cells and clones were selected based on homogenous GFP expression upon doxycycline addition. For sample preparation, cell clones were treated for 3 days with doxycycline (1 microg/ml) and trimethoprim (1 microM). Fifty thousand cells were collected for ATAC-seq library preparation according to Buenrostro et al.. Briefly, nuclei were extracted by continuous centrifugations and chromatin was exposed for enzymatic tagmentation and 12-plex Illumina-compatible library. All fragments were amplified by Phusion hot-start polymerase using 12 cycles of PCR in total. The ATAC-libary was sequenced on one lane of Illumina HiSeq2000 instrument. Peak calling was done with Homer looking for histone like peaks, using sample 5 (non-treated control) as background.\\nMETHODS\\ntitle_2\\nStatistical analysis\\nMETHODS\\nparagraph\\nAll the reprogramming experiments were replicated in three independent experiments with duplicate samples if enough cells were available. NSC inductions were replicated in six independent experiments with single samples per experiment. All sample sizes are indicated in the corresponding figure legends.\\nMETHODS\\nparagraph\\nStatistical analysis was performed as described in the figure legends. P-values of less than 0.05 were considered significant (*P < 0.05, **P < 0.01, ***P < 0.001). Estimating EEA-g1 enrichment near upstream regions of EEA associated genes was done by Monte Carlo sampling with 105 random permutations of n number of genes (as defined in the text) from a pool of 19,806 protein coding genes.\\nMETHODS\\ntitle_2\\nData availability\\nMETHODS\\nparagraph\\nRNA Sequencing, DNA methylation, and ATAC-seq data related to Figs. 3d-f, 4, and 6d-f are available on Array Express: \"E-MTAB-6185,\" \"E-MTAB-6186,\" \"E-MTAB-6194,\" \"E-MTAB-6195\".\\nSUPPL\\ntitle_1\\nElectronic supplementary material\\nSUPPL\\nfootnote\\nThese authors contributed equally: Juha Kere, Timo Otonkoski.\\nSUPPL\\nfootnote\\nElectronic supplementary material\\nSUPPL\\nfootnote\\nSupplementary Information accompanies this paper at 10.1038/s41467-018-05067-x.\\nSUPPL\\nfootnote\\nPublisher\\'s note: Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.\\nAUTH_CONT\\ntitle\\nAuthor contributions\\nAUTH_CONT\\nparagraph\\nJ.W. and D.B. was responsible for conceptualization, J.W., D.B. and M.B. for methodology, S.K. for software, J.W. and S.K. was involved in formal analysis, J.W., D.B., M.B., K.K., E.M.J. in investigation, J.W. for writing - original draft, all authors were involved in writing - review and editing; R.T., J.K., and T.O. in supervision, and J.K. and T.O. in funding acquisition.\\nCOMP_INT\\ntitle_1\\nCompeting interests\\nCOMP_INT\\nparagraph\\nThe authors declare no competing interests.\\nREF\\ntitle\\nReferences\\nsurname:Jinek;given-names:M\\nREF\\nScience\\nref\\nA programmable dual-RNA-guided DNA endonuclease in adaptive bacterial immunity\\nsurname:Dominguez;given-names:AA\\nsurname:Lim;given-names:WA\\nsurname:Qi;given-names:LS\\nREF\\nNat. Rev. Mol. Cell Biol.\\nref\\nBeyond editing: repurposing CRISPR-Cas9 for precision genome regulation and interrogation\\nsurname:Thakore;given-names:PI\\nsurname:Black;given-names:JB\\nsurname:Hilton;given-names:IB\\nsurname:Gersbach;given-names:CA\\nREF\\nNat. Methods\\nref\\nEditing the epigenome: technologies for programmable transcription and epigenetic modulation\\nsurname:Smith;given-names:ZD\\nsurname:Sindhu;given-names:C\\nsurname:Meissner;given-names:A\\nREF\\nNat. Rev. Mol. Cell Biol.\\nref\\nMolecular features of cellular reprogramming and development\\nsurname:Plath;given-names:K\\nsurname:Lowry;given-names:WE\\nREF\\nNat. Rev. Genet.\\nref\\nProgress in understanding reprogramming to the induced pluripotent state\\nsurname:Mikkelsen;given-names:TS\\nREF\\nNature\\nref\\nDissecting direct reprogramming through integrative genomic analysis\\nsurname:Kearns;given-names:NA\\nREF\\nDevelopment\\nref\\nCas9 effector-mediated regulation of transcription and differentiation in human pluripotent stem cells\\nsurname:Chavez;given-names:A\\nREF\\nNat. Methods\\nref\\nHighly efficient Cas9-mediated transcriptional programming\\nsurname:Chakraborty;given-names:S\\nREF\\nStem Cell Rep.\\nref\\nA CRISPR/Cas9-based system for reprogramming cell lineage specification\\nsurname:Wei;given-names:S\\nREF\\nSci. Rep.\\nref\\nConversion of embryonic stem cells into extraembryonic lineages by CRISPR-mediated activators\\nsurname:Gao;given-names:X\\nREF\\nNucleic Acids Res.\\nref\\nComparison of TALE designer transcription factors and the CRISPR/dCas9 in regulation of gene expression by targeting enhancers\\nsurname:Balboa;given-names:D\\nREF\\nStem Cell Rep.\\nref\\nConditionally stabilized dCas9 activator for controlling gene expression in human cell reprogramming and differentiation\\nsurname:Black;given-names:JB\\nREF\\nCell. Stem. Cell.\\nref\\nTargeted epigenetic remodeling of endogenous loci by CRISPR/Cas9-based transcriptional activators directly converts fibroblasts to neuronal cells\\nsurname:Xiong;given-names:K\\nREF\\nCell Reprogram.\\nref\\nRNA-guided activation of pluripotency genes in human fibroblasts\\nsurname:Liu;given-names:P\\nsurname:Chen;given-names:M\\nsurname:Liu;given-names:Y\\nsurname:Qi;given-names:LS\\nsurname:Ding;given-names:S\\nREF\\nCell. Stem. Cell.\\nref\\nCRISPR-based chromatin remodeling of the endogenous Oct4 or Sox2 locus enables reprogramming to pluripotency\\nsurname:Hilton;given-names:IB\\nREF\\nNat. Biotechnol.\\nref\\nEpigenome editing by a CRISPR-Cas9-based acetyltransferase activates genes from promoters and enhancers\\nsurname:Thakore;given-names:PI\\nREF\\nNat. Methods\\nref\\nsurname:Kearns;given-names:Na\\nREF\\nNat. Methods\\nref\\nFunctional annotation of native enhancers with a Cas9-histone demethylase fusion\\nsurname:Tohonen;given-names:V\\nREF\\nNat. Commun.\\nref\\nNovel PRD-like homeodomain transcription factors and retrotransposon elements in early human development\\nsurname:Tachibana;given-names:M\\nREF\\nCell\\nref\\nHuman embryonic stem cells derived by somatic cell nuclear transfer\\nsurname:Mandai;given-names:M\\nREF\\nN. Engl. J. Med.\\nref\\nAutologous induced stem-cell-derived retinal cells for macular degeneration\\nsurname:Kim;given-names:JB\\nREF\\nCell\\nref\\nOct4-induced pluripotency in adult neural stem cells\\nsurname:Konermann;given-names:S\\nREF\\nNature\\nref\\nGenome-scale transcriptional activation by an engineered CRISPR-Cas9 complex\\nsurname:Takahashi;given-names:K\\nsurname:Yamanaka;given-names:S\\nREF\\nCell\\nref\\nsurname:Yu;given-names:J\\nREF\\nScience\\nref\\nInduced pluripotent stem cell lines derived from human somatic cells\\nsurname:Choudhury;given-names:NR\\nREF\\nCell Rep.\\nref\\nTrim25 is an RNA-specific activator of Lin28a/TuT4-mediated uridylation\\nsurname:Li;given-names:Z\\nsurname:Yang;given-names:CS\\nsurname:Nakashima;given-names:K\\nsurname:Rana;given-names:TM\\nREF\\nref\\nSmall RNA-mediated regulation of iPS cell generation\\nsurname:Ruiz;given-names:S\\nREF\\nCurr. Biol.\\nref\\nA high proliferation rate is required for cell reprogramming and maintenance of human embryonic stem cell identity\\nsurname:Tsankov;given-names:AM\\nREF\\nNature\\nref\\nTranscription factor binding dynamics during human ES cell differentiation\\nsurname:Doench;given-names:JG\\nREF\\nNat. Biotechnol.\\nref\\nRational design of highly active sgRNAs for CRISPR-Cas9-mediated gene inactivation\\nsurname:Jouhilahti;given-names:EM\\nREF\\nDevelopment\\nref\\nThe human PRD-like homeobox gene LEUTX has a central role in embryo genome activation\\nsurname:Barkal;given-names:AA\\nsurname:Srinivasan;given-names:S\\nsurname:Hashimoto;given-names:T\\nsurname:Gifford;given-names:DK\\nsurname:Sherwood;given-names:RI\\nREF\\nPLOS ONE\\nref\\nCas9 functionally opens chromatin\\nsurname:Chavez;given-names:A\\nREF\\nNat. Methods\\nref\\nComparison of Cas9 activators in multiple species\\nsurname:Garcia-Perez;given-names:JL\\nsurname:Widmann;given-names:TJ\\nsurname:Adams;given-names:IR\\nREF\\nDevelopment\\nref\\nThe impact of transposable elements on mammalian development\\nsurname:Ge;given-names:SX\\nREF\\nBMC Genom.\\nref\\nExploratory bioinformatics investigation reveals importance of \\'junk\\' DNA in early embryo development\\nsurname:Takahashi;given-names:K\\nREF\\nNat. Commun.\\nref\\nInduction of pluripotency in human somatic cells via a transient state resembling primitive streak-like mesendoderm\\nsurname:Cacchiarelli;given-names:D\\nREF\\nCell\\nref\\nIntegrative analyses of human reprogramming reveal dynamic nature of induced pluripotency\\nsurname:Willoughby;given-names:DA\\nsurname:Vilalta;given-names:A\\nsurname:Oshima;given-names:RG\\nREF\\nJ. Biol. Chem.\\nref\\nsurname:Maeso;given-names:I\\nREF\\nBMC Biol.\\nref\\nsurname:Madissoon;given-names:E\\nREF\\nSci. Rep.\\nref\\nCharacterization and target genes of nine human PRD-like homeobox domain genes expressed exclusively in early embryos\\nsurname:Petropoulos;given-names:S\\nREF\\nCell\\nref\\nSingle-cell RNA-seq reveals lineage and X chromosome dynamics in human preimplantation embryos\\nsurname:Bolotin;given-names:E\\nREF\\nBMC Genom.\\nref\\nNuclear receptor HNF4alpha binding sequences are widespread in Alu repeats\\nsurname:Horlbeck;given-names:MA\\nREF\\neLife\\nref\\nCompact and highly active next-generation libraries for CRISPR-mediated gene repression and activation\\nsurname:Klann;given-names:TS\\nREF\\nNat. Biotechnol.\\nref\\nCRISPR-Cas9 epigenome editing enables high-throughput screening for functional regulatory elements in the human genome\\nsurname:Trokovic;given-names:R\\nsurname:Weltner;given-names:J\\nsurname:Otonkoski;given-names:T\\nREF\\nStem Cell Res.\\nref\\nGeneration of iPSC line HEL24.3 from human neonatal foreskin fibroblasts\\nsurname:Reinhardt;given-names:P\\nREF\\nref\\nDerivation and expansion using only small molecules of human neural progenitors for neurodegenerative disease modeling\\nsurname:Okita;given-names:K\\nREF\\nNat. Methods\\nref\\nA more efficient method to generate integration-free human iPS cells\\nsurname:Islam;given-names:S\\nREF\\nNat. Protoc.\\nref\\nHighly multiplexed and strand-specific single-cell RNA 5\\' end sequencing\\nsurname:Krjutskov;given-names:K\\nREF\\nHum. Reprod.\\nref\\nSingle-cell transcriptome analysis of endometrial tissue\\nsurname:Katayama;given-names:S\\nsurname:Tohonen;given-names:V\\nsurname:Linnarsson;given-names:S\\nsurname:Kere;given-names:J\\n10.1093/bioinformatics/btt511\\nREF\\nBioinformatics\\nref\\nSAMstrt: statistical test for differential expression in single-cell transcriptome with spike-in normalization\\nsurname:Gaujoux;given-names:R\\nsurname:Seoighe;given-names:C\\nREF\\nBMC Bioinforma.\\nref\\nA flexible R package for nonnegative matrix factorization\\nsurname:Buenrostro;given-names:JD\\nsurname:Wu;given-names:B\\nsurname:Chang;given-names:HY\\nsurname:Greenleaf;given-names:WJ\\nREF\\nCurr. Protoc. Mol. Biol.\\nref\\nATAC-seq: a method for assaying chromatin accessibility genome-wide\\n',\n",
       " '21299862': 'BioC-API\\ncollection.key\\nCC BY\\nThis is an open access article distributed under the terms of the Creative Commons Attribution License (http://creativecommons.org/licenses/by/2.0), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.\\nsurname:Salazar;given-names:Marcela D\\nsurname:Ratnam;given-names:Maya\\nsurname:Patki;given-names:Mugdha\\nsurname:Kisovic;given-names:Ivana\\nsurname:Trumbly;given-names:Robert\\nsurname:Iman;given-names:Mohamed\\nsurname:Ratnam;given-names:Manohar\\nTITLE\\nfront\\nDuring hormone depletion or tamoxifen treatment of breast cancer cells the estrogen receptor apoprotein supports cell cycling through the retinoic acid receptor alpha1 apoprotein\\nABSTRACT\\nabstract_title_1\\nIntroduction\\nABSTRACT\\nabstract\\nCurrent hormonal adjuvant therapies for breast cancer including tamoxifen treatment and estrogen depletion are overall tumoristatic and are severely limited by the frequent recurrence of the tumors. Regardless of the resistance mechanism, development and progression of the resistant tumors requires the persistence of a basal level of cycling cells during the treatment for which the underlying causes are unclear.\\nABSTRACT\\nabstract_title_1\\nMethods\\nABSTRACT\\nabstract\\nIn estrogen-sensitive breast cancer cells the effects of hormone depletion and treatment with estrogen, tamoxifen, all-trans retinoic acid (ATRA), fulvestrant, estrogen receptor alpha (ER) siRNA or retinoic acid receptor alpha (RARalpha) siRNA were studied by examining cell growth and cycling, apoptosis, various mRNA and protein expression levels, mRNA profiles and known chromatin associations of RAR. RARalpha subtype expression was also examined in breast cancer cell lines and tumors by competitive PCR.\\nABSTRACT\\nabstract_title_1\\nResults\\nABSTRACT\\nabstract\\nBasal proliferation persisted in estrogen-sensitive breast cancer cells grown in hormone depleted conditioned media without or with 4-hydroxytamoxifen (OH-Tam). Downregulating ER using either siRNA or fulvestrant inhibited basal proliferation by promoting cell cycle arrest, without enrichment for ErbB2/3+ overexpressing cells. The basal expression of RARalpha1, the only RARalpha isoform that was expressed in breast cancer cell lines and in most breast tumors, was supported by apo-ER but was unaffected by OH-Tam; RAR-beta and -gamma were not regulated by apo-ER. Depleting basal RARalpha1 reproduced the antiproliferative effect of depleting ER whereas its restoration in the ER depleted cells partially rescued the basal cycling. The overlapping tamoxifen-insensitive gene regulation by apo-ER and apo-RARalpha1 comprised activation of mainly genes promoting cell cycle and mitosis and suppression of genes involved in growth inhibition; these target genes were generally insensitive to ATRA but were enriched in RAR binding sites in associated chromatin regions.\\nABSTRACT\\nabstract_title_1\\nConclusions\\nABSTRACT\\nabstract\\nIn hormone-sensitive breast cancer, ER can support a basal fraction of S-phase cells (i) without obvious association with ErbB2/3 expression, (ii) by mechanisms unaffected by hormone depletion or OH-Tam and (iii) through maintenance of the basal expression of apo-RARalpha1 to regulate a set of ATRA-insensitive genes. Since isoform 1 of RARalpha is genetically redundant, its targeted inactivation or downregulation should be further investigated as a potential means of enhancing hormonal adjuvant therapy.\\nINTRO\\ntitle_1\\nIntroduction\\nINTRO\\nparagraph\\nMost breast tumors in both premenopausal and postmenopausal women express estrogen receptor type alpha (ER). Tamoxifen is a Selective Estrogen Receptor Modulator (SERM) widely used for adjuvant therapy in the treatment of ER+ breast cancer. In the hormone-sensitive tumors, tamoxifen acts as a partial antagonist, impairing ER function by competing with estrogen for binding to the receptor; however, more than three years of tamoxifen treatment only results in approximately 50% reduction in the incidence of invasive breast cancer in women at high risk, whereas about a third of ER+ breast tumors are intrinsically resistant to tamoxifen.\\nINTRO\\nparagraph\\nThird generation aromatase inhibitors (AI) present a valuable alternative to tamoxifen adjuvant therapy in postmenopausal women with ER+ breast cancer. Aromatase activity is essential for catalyzing the conversion to estrogen of steroid precursors in peripheral tissues, the major source of estrogen production in postmenopausal women. Upon treatment with AI, aromatase activity is reduced by at least 96% and circulating estrogen is virtually absent, inhibiting hormone-dependent tumor growth. In spite of the sensitivity of tamoxifen-resistant tumors to AI, breast tumors also acquire resistance to AI after long term treatment, resulting in disease recurrence and aggressive tumor growth. Clinical trials are underway to assess the possibility of delaying the onset of resistance by administering AI for two to three years following two to three years of tamoxifen treatment. The mechanistic basis underlying breast tumor resistance to either hormone depletion or to tamoxifen is still inadequately understood. In the vast majority of cases, resistance must occur through hormone-independent ER signaling events. Accordingly, Selective Estrogen Receptor Downregulators (SERDs, for example, Faslodex) have been found to be effective inhibitors of ER+ breast tumor growth but their utility is limited to their use as second or third line therapeutics in postmenopausal women with metastatic disease due to their broader impact on physiological ER signaling pathways in normal tissues. Therefore, it is imperative to continue to identify critical downstream events of ER signaling in breast cancer.\\nINTRO\\nparagraph\\nBreast cancer therapy trials have also been designed to explore the effect of retinoid compounds either alone or in combination with tamoxifen. In in vitro and pre-clinical models of breast cancer using MCF-7 cell xenografts, all-trans- retinoic acid (ATRA) alone or in combination with tamoxifen induced cell cycle arrest and apoptosis, leading to tumor regression through activation of multiple signal transduction pathways. Synergistic anti-tumor effects have been noted in vitro for the combination of retinoid and tamoxifen and multiple molecular mechanisms for the ligand effects have been reported. However, toxicity issues due to ATRA treatment was a challenge in patients with advanced breast cancer during phaseI/II clinical trials. Fenretinide, a synthetic amide of retinoic acid, has a better toxicological profile acting on both ER+ and ER- breast tumors principally by inducing apoptosis by both retinoic acid receptor (RAR) -dependent and -independent mechanisms; this drug showed a modest chemopreventive effect only in younger premenopausal women.\\nINTRO\\nparagraph\\nHormonal adjuvant therapy of breast cancer is overall tumoristatic with cell death balancing a basal level of cell proliferation. From a fundamental mechanistic standpoint, for resistance to develop in the long term during either hormone depletion or tamoxifen adjuvant therapy, the latent tumors must sustain a basal level of cell cycling to enable the generation and/or progression of genetic or epigenetic changes leading to resistance. It is the premise of this study that understanding the mechanisms that support the persistence of a small fraction of cells in S-phase throughout the course of hormonal adjuvant therapy in breast cancer will shed light on this critical precondition for the eventual development of resistance to the treatments. Since estrogen-independent ER signaling has been implicated in the development of resistance to adjuvant therapy, it was the goal of this study to examine the relationship between hormone-independent actions of ER and the basal cycling state of estrogen deprived breast cancer cells. Further, the ER-RAR axis has only been investigated in the context of ligand-dependent effects; it was therefore of additional interest to explore a possible interplay between the apo- forms of ER and RAR and its impact on basal proliferation, that is, under conditions of hormone depletion or tamoxifen antagonism.\\nINTRO\\nparagraph\\nEstrogen-sensitive breast cancer cell lines (MCF-7, T47 D and ZR-75-1) have proven to be exceptionally reliable predictive models both in vitro and in vivo for clinical drug response and the development of clinical drug resistance in breast cancer. We have observed that the expected basal proliferating state of hormone-depleted or tamoxifen treated breast cancer cells may be reproduced in vitro in established cell lines for an indefinite period by avoiding the common practice of intermittently replenishing the culture media, thus avoiding depletion of autocrine growth factors. We, therefore, used in vitro models to investigate the potential impact of hormone-independent actions of ER on the survival or proliferation of hormone-sensitive breast cancer cells and the related mechanisms under conditions that mimic hormonal adjuvant therapy, that is, estrogen-depletion and tamoxifen treatment.\\nMETHODS\\ntitle_1\\nMaterials and methods\\nMETHODS\\ntitle_2\\nChemicals and reagents\\nMETHODS\\nparagraph\\nDulbecco\\'s minimum essential medium (DMEM), glutamine and penicillin/streptomycin/glutamine stock mix were purchased from Life Technologies, Inc. (Carlsbad, CA, USA). Fetal bovine serum (FBS) and charcoal-stripped FBS were from Invitrogen (Carlsbad, CA, USA). Fugene 6 and Dharmafect 1 were from Roche Diagnostics (Indianapolis, IN, USA) and Dharmacon (Thermo Scientific Dharmacon, Inc., Lafayette, CO, USA), respectively. ERalpha (J-003401-12), RARalpha (J-003437-07) and control (D-001810-02) small interfering RNA (siRNA) were purchased from Dharmacon (Thermo Scientific Dharmacon, Inc.). Affinity purified rabbit and mouse antibodies to human ERalpha (sc-543), RARalpha (sc-551), RARbeta (sc-552), RARgamma (sc-550) and glyceraldehyde-3-phosphate dehydrogenase (GAPDH; sc-47724) were from Santa Cruz Biotechnologies (Santa Cruz, CA, USA). Peroxidase-conjugated secondary antibody was from Vector Laboratories (Burlingame, CA, USA). For standard PCR, HotStart Taq Plus DNA Polymerase was used (Qiagen, Germantown, MD, USA). Reagents for real time PCR, primers and TaqMan probes were purchased from Applied Biosystems (Branchburg, NJ, USA). PI/RNase staining buffer was from BD Pharmigen (San Diego, CA, USA). The Guava Nexin Reagent was purchased from Guava Technologies (Guava Technologies, Inc., Hayward, CA, USA). The protease inhibitor cocktail kit was obtained from Pierce Biotechnology (Thermo Scientific, Rockford, IL, USA). 17beta-estradiol (E2), 4-hydroxytamoxifen (OH-Tam), all-trans-Retinoic acid (ATRA) and fulvestrant were purchased from Sigma Aldrich (Saint Louis, MO USA). ATRA stock solution (5 mmol/L) was made in a mixture of 50% ethanol and 50% DMSO (Fisher Chemical, Fair Lawn, NJ, USA). First strand cDNA from human peripheral blood leukocytes (PBL), thymus and spleen were obtained from Biochain Institute (Biochain Institute Inc., Hayward, CA, USA). Total RNA from normal human breast and human breast tumors were obtained from Biochain Institute Institute and Clonetech (Clonetech Laboratories Inc., Mountain View, CA, USA).\\nMETHODS\\ntitle_2\\nCell culture and treatment with fulvestrant or ATRA\\nMETHODS\\nparagraph\\nMCF-7 and T47 D (American Type Culture Collection) cells were cultured in DMEM supplemented with FBS (10%), penicillin (100 unit/ml), streptomycin (100 mug/ml) and L-glutamine (2 mM). ZR-75-1 (American Type Culture Collection) cells were cultured in RPMI 1640 supplemented with FBS (10%), penicillin (100 unit/ml), streptomycin (100 mug/ml) and L-glutamine (2 mM). Hormone depleted cells were grown in low glucose phenol-red free media supplemented with 5% charcoal-stripped FBS (v/v) and L-glutamine (2 mM) for 48 hours before the experiments. Hormone-depleted MCF-7 cells were grown to 60 to 70% confluency in six-well plates and treated with vehicle or ATRA (1 muM) for 24 hours. After 24 hours, the cells were harvested for total RNA isolation and mRNA profiling. Hormone-depleted MCF-7 cells were grown to 30 to 40% confluency in six-well plates and treated with vehicle or fulvestrant (100 nM) for up to 4 days. After 72 hours to 96 hours, the cells were harvested for isolation of total RNA and protein.\\nMETHODS\\ntitle_2\\nTransfection and gene silencing\\nMETHODS\\nparagraph\\nCells were plated at 20% confluence in low glucose phenol red free medium supplemented with 5% charcoal stripped FBS and glutamine 24 hours to 48 hours prior to transfection. Treatment with vehicle (ethanol), E2 (1 nM) or OH-Tam (100 nM) was begun an additional 24 hours later. Cells were transfected with control siRNA, ERalpha siRNA or RARalpha siRNA (100 pmol/mL) in 24-well microplates or 25 cm2 flasks using 2 mul and 12.5 mul of Dharmafect 1 (Thermo Scientific Dharmacon Inc.), respectively, according to the vendor\\'s protocol. The cell culture medium was not replenished for the duration of the experiment. In the RARalpha1 rescue experiments, 2 x 106 cells were co-transfected with 2 mug of either the vector plasmid or RARalpha1 expression plasmid and with control siRNA or ERalpha siRNA by nucleofection using the Kit V Amaxa Nucleofection System (Amaxa Biosystems, Allendale, NJ, USA) according to the vendor\\'s instructions. In the RARalpha1 rescue of fulvestrant- treated cells and RARalpha1 overexpression experiments, 2 mug RARalpha1 expression plasmid was introduced at a cell density of 2.5 x 105 cells per well in six-well plates using Fugene 6 according to the manufacturer\\'s protocol.\\nMETHODS\\ntitle_2\\nCell growth assay\\nMETHODS\\nparagraph\\nCells were seeded in 24-well microplates at 20% confluence in phenol-red free media supplemented with 5% charcoal-stripped FBS (v/v) and incubated at 37 C with 5% CO2 for 24 h. Cells were transfected with either control siRNA or siRNA targeting ERalpha using Dharmafect 1. Twenty-four hours after transfection the media was replaced with fresh phenol-red free media supplemented with 5% charcoal-stripped FBS (v/v) and the cells were treated with vehicle (ethanol), E2 (1 nM) or OH-Tam (100 nM) for the following five days; the culture media was not changed during this period but E2 and OH-Tam were replenished every 48 h. Viable cell counts were monitored using the trypan blue dye exclusion assay at intervals of 24 h.\\nMETHODS\\ntitle_2\\nCell cycle analysis\\nMETHODS\\nparagraph\\nCells were trypsinized and harvested in phenol red free medium supplemented with charcoal stripped FBS. Cells (1 x 106) were washed and resuspended in 500 mul PBS. The cells were fixed by adding 500 mul 100% ice cold ethanol, drop-wise with agitation and incubated on ice for 20 minutes. The cells were sedimented by brief centrifugation at 200 xg for five minutes and the excess ethanol decanted. After the remaining ethanol was dried off, the cells were resuspended in 500 mul of PI/RNase solution. The cells were incubated in the dark at room temperature for 20 minutes and the cell cycle distribution determined by flow cytometric analysis using a FACSCalibur cell analyzer (BD Biosciences, San Jose, CA, USA). The data were acquired with BD CellQuest Pro software and analyzed using ModFit LT software.\\nMETHODS\\ntitle_2\\nApoptosis assay\\nMETHODS\\nparagraph\\nEarly stage apoptosis of cells was measured by Guava Nexin analysis using the Guava Nexin Reagent staining kit according to the manufacturer\\'s instructions. Briefly, 8 x 104 cells were incubated for 20 minutes at room temperature with the Guava Nexin Reagent and 2,000 cells per sample were analyzed using the Guava System.\\nMETHODS\\ntitle_2\\nWestern blot\\nMETHODS\\nparagraph\\nCells were harvested by trypsinization, lysed in a high salt-detergent buffer (400 nM NaCl; 10 nM Tris, pH 8.0; 1 mM EDTA; 1 mM EGTA; beta-mercaptoethanol; and 0.1% Triton x-100) containing a protease inhibitor cocktail kit and incubated on ice for 30 minutes. Cell lysates were heated to 95 C for five minutes. Protein samples (10 to 20 mug) were resolved by electrophoresis on 8% sodium dodecylsulfate-polyacrylamide gels and electrophoretically transferred to PVDF membranes (Millipore Corporation, Bedford, MA, USA). The blots were probed with the appropriate primary antibody and the appropriate horseradish peroxidase conjugated secondary antibody and the protein bands visualized using enhanced chemiluminescence as described. The chemiluminescent signals were quantified using the FluorChem HD2 imaging system (Alpha Innotech/Cell Biosciences, Inc., Santa Clara, CA, USA) and normalized to GAPDH.\\nMETHODS\\ntitle_2\\nRNA isolation, reverse transcription PCR and Real time PCR\\nMETHODS\\nparagraph\\nTotal RNA from cells was isolated using the RNeasy mini kit (Qiagen, Georgetown, MD, USA). Reverse transcription PCR reactions were performed using 500 ng of total RNA and the high capacity complementary DNA Archive kit (Applied Biosystems) according to the vendor\\'s protocol. cDNAs of RARalpha1 and RARalpha2 were amplified by competitive PCR. The upstream and downstream primers used for amplification of RARalpha1 and RARalpha2 were as follows: RARalpha1, 5\\'-GCCAGGCGCTCTGACCACTC-3\\' and 5\\'-AGCCCTTGCAGCCCTCACAG-3\\'; RARalpha2, 5\\'-ACTCCGCTTTGGAATGGCTCAAAC-3\\' and 5\\'-AGCCCTTGCAGCCCTCACAG-3\\'. The cDNA for the house keeping gene glyceraldehyde-3- phosphate dehydrogenase (GAPDH) was amplified and the primer sequences used were as follows: 5\\'-TGGTCACCAGGGCTGCTTTT-3\\' and 5\\'-GGTGAAGACGCCAGTGGACT-3\\'. The cycling parameters were: 95 C for 15 minutes; 94 C for 30 sec; 60 C for 30 sec; 72 C for 30 sec and 72 C for 10 minutes. RARalpha1 and RARalpha2 cDNAs were amplified in the same reaction, yielding products of 222 bp and 182 bp, respectively. PCR products were separated in ethidium bromide-stained 2% agarose gels by electrophoresis. cDNA was also measured by quantitative real time PCR in the 7500 StepOne Plus Real time PCR System (Applied Biosystems). Primers and TaqMan probes for the human ERalpha, CCNA, CDKN1, ERBB2, ERBB3, MUC20, LYPD1, RARalpha and GAPDH genes were obtained from the Applied Biosystems inventory. All samples were measured in triplicate and normalized to GAPDH values.\\nMETHODS\\ntitle_2\\nmRNA profiling\\nMETHODS\\nparagraph\\nThe Affymetrix chips were purchased from Affymetrix (Santa Clara, CA, USA) DNA microarray analysis using Affymetrix was performed as a full service global gene expression study at the transcriptional profiling core facility of the Cancer Institute of New Jersey. Total RNA samples were used to generate labeled cRNAs, which were hybridized to human U133 Plus2.0 Affymetrix microarrays. The expression data were analysed initially using Affymetrix GeneChip Operating Software to create CEL files. The CEL files were imported into the Bioconductor program affylmGUI. The probe set level intensities were quantified and normalized using robust multiarray averaging and quantile normalization. Differential expression between treatments was determined using the limma linear modeling method, and the significance of differences was ranked by the moderated t-statistic. The values for signal intensities were corrected for siRNA transfection efficiencies determined using a Green Fluorescent Protein (GFP) reporter expression plasmid. To identify genes differentially expressed under the different treatments, the fold-changes were calculated by dividing the average signal of the treatment by the control, and genes with a fold-change greater or lesser than a given threshold were chosen. The advantage of this approach is that rejection of many false negatives is avoided, compared to requiring a statistically significant difference in expression, but has the potential drawback of including false positives. When we limited the genes in Tables 1 and 2 to those showing significant differential expression at the P = 0.05 level by the linear modeling method, 25/54 genes in Table 1 and 24/68 genes in Table 2 were retained. In the reduced sets of genes, similar percentages of genes showed RARalpha peaks as in the larger gene set, confirming the generality of our result. The Affymetrix data are deposited in GEO (Accession number: [GEO:GSE26298]).\\nT1.xml\\nT1\\nTABLE\\ntable_caption\\nTamoxifen Insensitive genes supported by the Apo-ER -> Apo-RARalpha Axis\\nT1.xml\\nT1\\nTABLE\\ntable\\n<?xml version=\"1.0\" encoding=\"UTF-8\"?>\\n<table frame=\"hsides\" rules=\"groups\"><thead><tr><th align=\"left\">Gene</th><th align=\"center\">ATRA effect</th><th align=\"center\">RAR<sup>a </sup>binding &lt;10 kb</th><th align=\"center\">Fold change siER</th><th align=\"center\">Fold change siRARalpha1</th><th align=\"left\">Gene</th><th align=\"center\">ATRA effect</th><th align=\"center\">RAR<sup>a </sup>binding &lt;10 kb</th><th align=\"center\">Fold change siER</th><th align=\"center\">Fold change siRARalpha1</th></tr></thead><tbody><tr><td align=\"left\"><italic>LGALS1</italic></td><td align=\"center\"> </td><td align=\"center\">-</td><td align=\"center\">0.31</td><td align=\"center\">0.46</td><td align=\"left\">CASC5</td><td align=\"center\">-</td><td align=\"center\">-</td><td align=\"center\">0.47</td><td align=\"center\">0.51</td></tr><tr><td align=\"left\"><italic>METTL7A</italic></td><td align=\"center\"> </td><td align=\"center\">-</td><td align=\"center\">0.37</td><td align=\"center\">0.53</td><td align=\"left\">BIRC5</td><td align=\"center\">-</td><td align=\"center\">-</td><td align=\"center\">0.53</td><td align=\"center\">0.48</td></tr><tr><td align=\"left\"><italic>CDKN3</italic></td><td align=\"center\"> </td><td align=\"center\">-</td><td align=\"center\">0.44</td><td align=\"center\">0.60</td><td align=\"left\">PTTG1</td><td align=\"center\">-</td><td align=\"center\">-</td><td align=\"center\">0.57</td><td align=\"center\">0.54</td></tr><tr><td align=\"left\"><italic>XK</italic></td><td align=\"center\"> </td><td align=\"center\">-</td><td align=\"center\">0.46</td><td align=\"center\">0.54</td><td align=\"left\">PLK4</td><td align=\"center\">-</td><td align=\"center\">-</td><td align=\"center\">0.56</td><td align=\"center\">0.49</td></tr><tr><td align=\"left\"><italic>GHR</italic></td><td align=\"center\"> </td><td align=\"center\">-</td><td align=\"center\">0.41</td><td align=\"center\">0.57</td><td align=\"left\">CENPA</td><td align=\"center\">-</td><td align=\"center\">-</td><td align=\"center\">0.58</td><td align=\"center\">0.56</td></tr><tr><td align=\"left\"><italic>YPEL1</italic></td><td align=\"center\"> </td><td align=\"center\">-</td><td align=\"center\">0.58</td><td align=\"center\">0.54</td><td align=\"left\">TMSB15A</td><td align=\"center\">-</td><td align=\"center\">-</td><td align=\"center\">0.42</td><td align=\"center\">0.41</td></tr><tr><td align=\"left\"><italic>SHANK2</italic></td><td align=\"center\"> </td><td align=\"center\"><bold>++</bold></td><td align=\"center\">0.58</td><td align=\"center\">0.57</td><td align=\"left\">C5</td><td align=\"center\">-</td><td align=\"center\">-</td><td align=\"center\">0.38</td><td align=\"center\">0.48</td></tr><tr><td align=\"left\"><italic>ENY2</italic></td><td align=\"center\">-</td><td align=\"center\"><bold>++</bold></td><td align=\"center\">0.54</td><td align=\"center\">0.45</td><td align=\"left\">CDC2</td><td align=\"center\">-</td><td align=\"center\">-</td><td align=\"center\">0.56</td><td align=\"center\">0.59</td></tr><tr><td align=\"left\"><italic>ONECUT2</italic></td><td align=\"center\">-</td><td align=\"center\"><bold>++</bold></td><td align=\"center\">0.50</td><td align=\"center\">0.58</td><td align=\"left\">CCNA2</td><td align=\"center\">-</td><td align=\"center\">-</td><td align=\"center\">0.59</td><td align=\"center\">0.52</td></tr><tr><td align=\"left\"><italic>HIST1H4C</italic></td><td align=\"center\">-</td><td align=\"center\"><bold>++</bold></td><td align=\"center\">0.21</td><td align=\"center\">0.31</td><td align=\"left\">LOC150759</td><td align=\"center\">-</td><td align=\"center\">-</td><td align=\"center\">0.49</td><td align=\"center\">0.59</td></tr><tr><td align=\"left\"><italic>NCAPH</italic></td><td align=\"center\">-</td><td align=\"center\"><bold>++</bold></td><td align=\"center\">0.47</td><td align=\"center\">0.53</td><td align=\"left\">NCAPG</td><td align=\"center\">-</td><td align=\"center\">-</td><td align=\"center\">0.50</td><td align=\"center\">0.51</td></tr><tr><td align=\"left\"><italic>CENPN</italic></td><td align=\"center\">-</td><td align=\"center\"><bold>++</bold></td><td align=\"center\">0.56</td><td align=\"center\">0.55</td><td align=\"left\">CENPM</td><td align=\"center\">-</td><td align=\"center\">-</td><td align=\"center\">0.57</td><td align=\"center\">0.50</td></tr><tr><td align=\"left\"><italic>UBE2T</italic></td><td align=\"center\">-</td><td align=\"center\"><bold>++</bold></td><td align=\"center\">0.53</td><td align=\"center\">0.60</td><td align=\"left\">FAM64A</td><td align=\"center\">-</td><td align=\"center\">-</td><td align=\"center\">0.55</td><td align=\"center\">0.58</td></tr><tr><td align=\"left\"><italic>PHF19</italic></td><td align=\"center\">-</td><td align=\"center\"><bold>++</bold></td><td align=\"center\">0.50</td><td align=\"center\">0.50</td><td align=\"left\">MND1</td><td align=\"center\">-</td><td align=\"center\">-</td><td align=\"center\">0.51</td><td align=\"center\">0.55</td></tr><tr><td align=\"left\"><italic>ZNF367</italic></td><td align=\"center\">-</td><td align=\"center\"><bold>++</bold></td><td align=\"center\">0.52</td><td align=\"center\">0.51</td><td align=\"left\">FGFR1</td><td align=\"center\">-</td><td align=\"center\">-</td><td align=\"center\">0.56</td><td align=\"center\">0.53</td></tr><tr><td align=\"left\"><italic>SNORA72</italic></td><td align=\"center\">-</td><td align=\"center\"><bold>++</bold></td><td align=\"center\">0.43</td><td align=\"center\">0.58</td><td align=\"left\">HELLS</td><td align=\"center\">-</td><td align=\"center\">-</td><td align=\"center\">0.58</td><td align=\"center\">0.55</td></tr><tr><td align=\"left\"><italic>KIF23</italic></td><td align=\"center\">-</td><td align=\"center\"><bold>+</bold></td><td align=\"center\">0.58</td><td align=\"center\">0.56</td><td align=\"left\">TNFAIP8L1</td><td align=\"center\">-</td><td align=\"center\">-</td><td align=\"center\">0.47</td><td align=\"center\">0.55</td></tr><tr><td align=\"left\"><italic>ENAH</italic></td><td align=\"center\">-</td><td align=\"center\"><bold>+</bold></td><td align=\"center\">0.54</td><td align=\"center\">0.50</td><td align=\"left\">OVOS2</td><td align=\"center\">-</td><td align=\"center\">-</td><td align=\"center\">0.40</td><td align=\"center\">0.58</td></tr><tr><td align=\"left\"><italic>MAD2L1</italic></td><td align=\"center\">-</td><td align=\"center\"><bold>+</bold></td><td align=\"center\">0.40</td><td align=\"center\">0.55</td><td align=\"left\">ZNF141</td><td align=\"center\">-</td><td align=\"center\">-</td><td align=\"center\">0.55</td><td align=\"center\">0.52</td></tr><tr><td align=\"left\"><italic>SMC4</italic></td><td align=\"center\">-</td><td align=\"center\"><bold>+</bold></td><td align=\"center\">0.54</td><td align=\"center\">0.60</td><td align=\"left\">LOC100129673</td><td align=\"center\">-</td><td align=\"center\">-</td><td align=\"center\">0.39</td><td align=\"center\">0.48</td></tr><tr><td align=\"left\"><italic>SEC31A</italic></td><td align=\"center\">-</td><td align=\"center\"><bold>+</bold></td><td align=\"center\">0.55</td><td align=\"center\">0.57</td><td align=\"left\">ASPM</td><td align=\"center\">-</td><td align=\"center\">-</td><td align=\"center\">0.45</td><td align=\"center\">0.45</td></tr><tr><td align=\"left\"><italic>SFPQ</italic></td><td align=\"center\">-</td><td align=\"center\"><bold>+</bold></td><td align=\"center\">0.60</td><td align=\"center\">0.49</td><td align=\"left\">EPHX4</td><td align=\"center\">-</td><td align=\"center\">-</td><td align=\"center\">0.51</td><td align=\"center\">0.55</td></tr><tr><td align=\"left\"><italic>CENPF</italic></td><td align=\"center\">-</td><td align=\"center\"><bold>+</bold></td><td align=\"center\">0.51</td><td align=\"center\">0.58</td><td align=\"left\">HNRPD</td><td align=\"center\">-</td><td align=\"center\">-</td><td align=\"center\">0.36</td><td align=\"center\">0.55</td></tr><tr><td align=\"left\"><italic>COBL</italic></td><td align=\"center\">-</td><td align=\"center\">-</td><td align=\"center\">0.54</td><td align=\"center\">0.52</td><td align=\"left\">ARPC5L</td><td align=\"center\">-</td><td align=\"center\">-</td><td align=\"center\">0.59</td><td align=\"center\">0.52</td></tr><tr><td align=\"left\"><italic>PBK</italic></td><td align=\"center\">-</td><td align=\"center\">-</td><td align=\"center\">0.56</td><td align=\"center\">0.56</td><td align=\"left\">RGS3</td><td align=\"center\">-</td><td align=\"center\">-</td><td align=\"center\">0.57</td><td align=\"center\">0.45</td></tr><tr><td align=\"left\"><italic>MLF1IP</italic></td><td align=\"center\">-</td><td align=\"center\">-</td><td align=\"center\">0.44</td><td align=\"center\">0.58</td><td align=\"left\">SIPA1L1</td><td align=\"center\">-</td><td align=\"center\">-</td><td align=\"center\">0.55</td><td align=\"center\">0.58</td></tr><tr><td align=\"left\"><italic>SAMHD1</italic></td><td align=\"center\">-</td><td align=\"center\">-</td><td align=\"center\">0.60</td><td align=\"center\">0.53</td><td/><td/><td/><td/><td/></tr></tbody></table>\\n\\nT1.xml\\nT1\\nTABLE\\ntable_foot\\na RAR binding site identified within 10 kb of the transcription start site (Hua et al., 2009).\\nT1.xml\\nT1\\nTABLE\\ntable_foot\\n++ RAR binding site detected under high stringency (Hua et al., 2009).\\nT1.xml\\nT1\\nTABLE\\ntable_foot\\n+ RAR binding site detected under low stringency (Hua et al., 2009).\\nT1.xml\\nT1\\nTABLE\\ntable_foot\\n  Up-regulated by ATRA (1 muM, 24 h) >=2-fold.\\nT1.xml\\nT1\\nTABLE\\ntable_foot\\n  Down-regulated by ATRA (1 muM, 24 h) <=50%.\\nT1.xml\\nT1\\nTABLE\\ntable_foot\\nT2.xml\\nT2\\nTABLE\\ntable_caption\\nTamoxifen Insensitive genes Repressed by the Apo-ER -> Apo-RARalpha Axis\\nT2.xml\\nT2\\nTABLE\\ntable\\n<?xml version=\"1.0\" encoding=\"UTF-8\"?>\\n<table frame=\"hsides\" rules=\"groups\"><thead><tr><th align=\"left\">Gene</th><th align=\"center\">ATRA<break/>effect</th><th align=\"center\">RAR <sup>a </sup>binding &lt;10 kb</th><th align=\"center\">Fold change siER</th><th align=\"center\">Fold change siRARalpha1</th><th align=\"left\">Gene</th><th align=\"center\">ATRA effect</th><th align=\"center\">RAR <sup>a </sup>binding &lt;10 kb</th><th align=\"center\">Fold change siER</th><th align=\"center\">Fold change siRARalpha1</th></tr></thead><tbody><tr><td align=\"left\"><italic>IFI44L</italic></td><td align=\"center\"> </td><td align=\"center\"><bold>+</bold></td><td align=\"center\">1.69</td><td align=\"center\">2.22</td><td align=\"left\"><italic>UBL3</italic></td><td align=\"center\">-</td><td align=\"center\"><bold>++</bold></td><td align=\"center\">1.61</td><td align=\"center\">1.58</td></tr><tr><td align=\"left\"><italic>IFI44</italic></td><td align=\"center\"> </td><td align=\"center\">-</td><td align=\"center\">2.09</td><td align=\"center\">2.28</td><td align=\"left\"><italic>SLC7A11</italic></td><td align=\"center\">-</td><td align=\"center\"><bold>++</bold></td><td align=\"center\">1.71</td><td align=\"center\">2.03</td></tr><tr><td align=\"left\"><italic>CCPG1</italic></td><td align=\"center\"> </td><td align=\"center\">-</td><td align=\"center\">1.67</td><td align=\"center\">1.56</td><td align=\"left\"><italic>KIAA0652</italic></td><td align=\"center\">-</td><td align=\"center\"><bold>++</bold></td><td align=\"center\">1.57</td><td align=\"center\">1.63</td></tr><tr><td align=\"left\"><italic>SLC25A36</italic></td><td align=\"center\"> </td><td align=\"center\">-</td><td align=\"center\">1.66</td><td align=\"center\">1.73</td><td align=\"left\"><italic>PHLDB1</italic></td><td align=\"center\">-</td><td align=\"center\"><bold>++</bold></td><td align=\"center\">1.64</td><td align=\"center\">1.55</td></tr><tr><td align=\"left\"><italic>SETD5</italic></td><td align=\"center\"> </td><td align=\"center\"><bold>++</bold></td><td align=\"center\">1.52</td><td align=\"center\">2.11</td><td align=\"left\"><italic>CCNT2</italic></td><td align=\"center\">-</td><td align=\"center\">-</td><td align=\"center\">1.53</td><td align=\"center\">1.63</td></tr><tr><td align=\"left\"><italic>MALL</italic></td><td align=\"center\"> </td><td align=\"center\">-</td><td align=\"center\">1.67</td><td align=\"center\">1.81</td><td align=\"left\"><italic>STX3</italic></td><td align=\"center\">-</td><td align=\"center\">-</td><td align=\"center\">1.52</td><td align=\"center\">1.59</td></tr><tr><td align=\"left\"><italic>LYPD1</italic></td><td align=\"center\"> </td><td align=\"center\">-</td><td align=\"center\">1.66</td><td align=\"center\">1.59</td><td align=\"left\"><italic>CLN8</italic></td><td align=\"center\">-</td><td align=\"center\">-</td><td align=\"center\">1.71</td><td align=\"center\">1.72</td></tr><tr><td align=\"left\"><italic>FAM186A</italic></td><td align=\"center\"> </td><td align=\"center\">-</td><td align=\"center\">1.64</td><td align=\"center\">1.75</td><td align=\"left\"><italic>UBXN10</italic></td><td align=\"center\">-</td><td align=\"center\">-</td><td align=\"center\">1.73</td><td align=\"center\">1.64</td></tr><tr><td align=\"left\"><italic>GPR158</italic></td><td align=\"center\"> </td><td align=\"center\">-</td><td align=\"center\">1.81</td><td align=\"center\">1.54</td><td align=\"left\"><italic>PVR</italic></td><td align=\"center\">-</td><td align=\"center\">-</td><td align=\"center\">1.63</td><td align=\"center\">2.39</td></tr><tr><td align=\"left\"><italic>CDKN1A</italic></td><td align=\"center\"> </td><td align=\"center\">-</td><td align=\"center\">1.70</td><td align=\"center\">1.51</td><td align=\"left\"><italic>TNFRSF10A</italic></td><td align=\"center\">-</td><td align=\"center\">-</td><td align=\"center\">1.54</td><td align=\"center\">1.62</td></tr><tr><td align=\"left\"><italic>CEACAM6</italic></td><td align=\"center\"> </td><td align=\"center\"><bold>+</bold></td><td align=\"center\">2.35</td><td align=\"center\">1.63</td><td align=\"left\"><italic>FLJ31958</italic></td><td align=\"center\">-</td><td align=\"center\">-</td><td align=\"center\">1.71</td><td align=\"center\">1.52</td></tr><tr><td align=\"left\"><italic>LGALS3BP</italic></td><td align=\"center\"> </td><td align=\"center\">-</td><td align=\"center\">1.98</td><td align=\"center\">1.92</td><td align=\"left\"><italic>C18orf25</italic></td><td align=\"center\">-</td><td align=\"center\">-</td><td align=\"center\">1.50</td><td align=\"center\">1.61</td></tr><tr><td align=\"left\"><italic>CTSS</italic></td><td align=\"center\"> </td><td align=\"center\">-</td><td align=\"center\">1.50</td><td align=\"center\">1.86</td><td align=\"left\"><italic>TAPBP</italic></td><td align=\"center\">-</td><td align=\"center\">-</td><td align=\"center\">1.62</td><td align=\"center\">1.51</td></tr><tr><td align=\"left\"><italic>SELL</italic></td><td align=\"center\"> </td><td align=\"center\">-</td><td align=\"center\">1.75</td><td align=\"center\">1.71</td><td align=\"left\"><italic>ADAM17</italic></td><td align=\"center\">-</td><td align=\"center\">-</td><td align=\"center\">1.58</td><td align=\"center\">1.63</td></tr><tr><td align=\"left\"><italic>SHROOM1</italic></td><td align=\"center\"> </td><td align=\"center\">-</td><td align=\"center\">1.68</td><td align=\"center\">1.59</td><td align=\"left\"><italic>HCP5</italic></td><td align=\"center\">-</td><td align=\"center\">-</td><td align=\"center\">1.59</td><td align=\"center\">2.52</td></tr><tr><td align=\"left\"><italic>CP</italic></td><td align=\"center\"> </td><td align=\"center\"><bold>++</bold></td><td align=\"center\">1.71</td><td align=\"center\">1.75</td><td align=\"left\"><italic>DISC1</italic></td><td align=\"center\">-</td><td align=\"center\">-</td><td align=\"center\">1.80</td><td align=\"center\">1.74</td></tr><tr><td align=\"left\"><italic>ABHD2</italic></td><td align=\"center\"> </td><td align=\"center\"><bold>++</bold></td><td align=\"center\">1.56</td><td align=\"center\">1.57</td><td align=\"left\"><italic>SP100</italic></td><td align=\"center\">-</td><td align=\"center\">-</td><td align=\"center\">1.77</td><td align=\"center\">1.61</td></tr><tr><td align=\"left\"><italic>ABLIM1</italic></td><td align=\"center\"> </td><td align=\"center\"><bold>++</bold></td><td align=\"center\">1.56</td><td align=\"center\">1.55</td><td align=\"left\"><italic>AASS</italic></td><td align=\"center\">-</td><td align=\"center\">-</td><td align=\"center\">1.53</td><td align=\"center\">2.17</td></tr><tr><td align=\"left\"><italic>ALOX5</italic></td><td align=\"center\"> </td><td align=\"center\">-</td><td align=\"center\">1.72</td><td align=\"center\">2.09</td><td align=\"left\"><italic>HLA-G</italic></td><td align=\"center\">-</td><td align=\"center\">-</td><td align=\"center\">1.50</td><td align=\"center\">2.33</td></tr><tr><td align=\"left\"><italic>HLA-C</italic></td><td align=\"center\"> </td><td align=\"center\">-</td><td align=\"center\">1.51</td><td align=\"center\">2.51</td><td align=\"left\"><italic>HLA-B</italic></td><td align=\"center\">-</td><td align=\"center\">-</td><td align=\"center\">1.61</td><td align=\"center\">2.08</td></tr><tr><td align=\"left\"><italic>MAP1B</italic></td><td align=\"center\"> </td><td align=\"center\">-</td><td align=\"center\">2.12</td><td align=\"center\">1.51</td><td align=\"left\"><italic>RSAD2</italic></td><td align=\"center\">-</td><td align=\"center\">-</td><td align=\"center\">1.84</td><td align=\"center\">3.19</td></tr><tr><td align=\"left\"><italic>ARHGAP1</italic></td><td align=\"center\"> </td><td align=\"center\">-</td><td align=\"center\">1.58</td><td align=\"center\">1.55</td><td align=\"left\"><italic>SGSM3</italic></td><td align=\"center\">-</td><td align=\"center\">-</td><td align=\"center\">1.72</td><td align=\"center\">1.50</td></tr><tr><td align=\"left\"><italic>SP110</italic></td><td align=\"center\"> </td><td align=\"center\">-</td><td align=\"center\">1.58</td><td align=\"center\">1.63</td><td align=\"left\"><italic>HLA-J</italic></td><td align=\"center\">-</td><td align=\"center\">-</td><td align=\"center\">1.52</td><td align=\"center\">2.04</td></tr><tr><td align=\"left\"><italic>RTP4</italic></td><td align=\"center\"> </td><td align=\"center\">-</td><td align=\"center\">1.60</td><td align=\"center\">1.81</td><td align=\"left\"><italic>SESN1</italic></td><td align=\"center\">-</td><td align=\"center\">-</td><td align=\"center\">1.53</td><td align=\"center\">1.60</td></tr><tr><td align=\"left\"><italic>MUC20</italic></td><td align=\"center\"> </td><td align=\"center\">-</td><td align=\"center\">1.75</td><td align=\"center\">1.91</td><td align=\"left\"><italic>ANO10</italic></td><td align=\"center\">-</td><td align=\"center\">-</td><td align=\"center\">1.51</td><td align=\"center\">1.64</td></tr><tr><td align=\"left\"><italic>RNF38</italic></td><td align=\"center\">-</td><td align=\"center\"><bold>+</bold></td><td align=\"center\">1.80</td><td align=\"center\">1.69</td><td align=\"left\"><italic>SAMD9</italic></td><td align=\"center\">-</td><td align=\"center\">-</td><td align=\"center\">1.68</td><td align=\"center\">1.56</td></tr><tr><td align=\"left\"><italic>C9orf80</italic></td><td align=\"center\">-</td><td align=\"center\"><bold>+</bold></td><td align=\"center\">1.64</td><td align=\"center\">1.53</td><td align=\"left\"><italic>FLJ13197</italic></td><td align=\"center\">-</td><td align=\"center\">-</td><td align=\"center\">1.68</td><td align=\"center\">1.57</td></tr><tr><td align=\"left\"><italic>CEACAM5</italic></td><td align=\"center\">-</td><td align=\"center\">-</td><td align=\"center\">2.35</td><td align=\"center\">1.70</td><td align=\"left\"><italic>PGLS</italic></td><td align=\"center\">-</td><td align=\"center\">-</td><td align=\"center\">1.54</td><td align=\"center\">1.81</td></tr><tr><td align=\"left\"><italic>CNOT4</italic></td><td align=\"center\">-</td><td align=\"center\">-</td><td align=\"center\">1.79</td><td align=\"center\">1.92</td><td align=\"left\"><italic>LMO3</italic></td><td align=\"center\">-</td><td align=\"center\">-</td><td align=\"center\">1.61</td><td align=\"center\">1.58</td></tr><tr><td align=\"left\"><italic>PLXNA2</italic></td><td align=\"center\">-</td><td align=\"center\">-</td><td align=\"center\">1.65</td><td align=\"center\">1.86</td><td align=\"left\"><italic>DNAJC21</italic></td><td align=\"center\">-</td><td align=\"center\">-</td><td align=\"center\">1.68</td><td align=\"center\">1.61</td></tr><tr><td align=\"left\"><italic>TTC9</italic></td><td align=\"center\">-</td><td align=\"center\">-</td><td align=\"center\">1.76</td><td align=\"center\">1.51</td><td align=\"left\"><italic>SETD2</italic></td><td align=\"center\">-</td><td align=\"center\">-</td><td align=\"center\">1.52</td><td align=\"center\">2.11</td></tr><tr><td align=\"left\"><italic>EIF5A2</italic></td><td align=\"center\">-</td><td align=\"center\">-</td><td align=\"center\">1.90</td><td align=\"center\">1.67</td><td align=\"left\"><italic>ZNF24</italic></td><td align=\"center\">-</td><td align=\"center\">-</td><td align=\"center\">2.10</td><td align=\"center\">1.68</td></tr><tr><td align=\"left\"><italic>FLCN</italic></td><td align=\"center\">-</td><td align=\"center\"><bold>++</bold></td><td align=\"center\">1.51</td><td align=\"center\">1.53</td><td align=\"left\"><italic>TTLL11</italic></td><td align=\"center\">-</td><td align=\"center\">-</td><td align=\"center\">1.72</td><td align=\"center\">1.51</td></tr><tr><td align=\"left\"><italic>BAZ2A</italic></td><td align=\"center\">-</td><td align=\"center\"><bold>++</bold></td><td align=\"center\">1.53</td><td align=\"center\">1.58</td><td align=\"left\"><italic>ZNF544</italic></td><td align=\"center\">-</td><td align=\"center\">-</td><td align=\"center\">1.58</td><td align=\"center\">1.71</td></tr></tbody></table>\\n\\nT2.xml\\nT2\\nTABLE\\ntable_foot\\na RAR binding site identified within 10 kb of the transcription start site (Hua et al., 2009).\\nT2.xml\\nT2\\nTABLE\\ntable_foot\\n++ RAR binding site detected under high stringency (Hua et al., 2009).\\nT2.xml\\nT2\\nTABLE\\ntable_foot\\n+ RAR binding site detected under low stringency (Hua et al., 2009).\\nT2.xml\\nT2\\nTABLE\\ntable_foot\\n  Up-regulated by ATRA (1 muM, 24 h) >=2-fold.\\nT2.xml\\nT2\\nTABLE\\ntable_foot\\n  Down-regulated by ATRA (1 muM, 24 h) <=50%.\\nT2.xml\\nT2\\nTABLE\\ntable_foot\\nMETHODS\\ntitle_2\\nStatistical analyses\\nMETHODS\\nparagraph\\nExperimental values are presented as mean +- standard deviation (s.d.). The statistical significance of differences (P-value) between values being compared was determined using analysis of variance. In all cases, the differences noted in the text are reflected by a P-value of <0.001.\\nRESULTS\\ntitle_1\\nResults\\nRESULTS\\ntitle_2\\nIn estrogen-sensitive MCF-7 cells basal proliferation is supported by ER in the absence of hormone\\nRESULTS\\nparagraph\\nIn the following studies, in experiments in which MCF-7 cells were depleted of hormone, the virtual absence of hormone was confirmed in two ways. First, the effect of OH-Tam on the expression of the classical E2 target gene, EGR3 (Early Growth Response 3 gene) was examined as a functional test. The expression of EGR3 mRNA is exquisitely sensitive to upregulation by E2, in a manner that is completely antagonized by OH-Tam. In the hormone depleted cells, the inability of OH-Tam to further decrease EGR3 mRNA indicated the virtual absence of hormone (Figure 1A). Second, the effect of E2 on the relative phosphorylation level at ser118 of ER was examined. The binding of E2 strongly induces phosphorylation of ER at this site. The absence of hormone was further confirmed by the observation of much lower phosphorylation at ser118 of ER in the hormone depleted cells, when compared with control cells treated with E2 (Figure 1B).\\nF1\\nFIG\\nfig_caption\\nHormone-independent effects of ER on the proliferation of hormone-sensitive MCF-7 cells. In all the experiments, MCF-7 cells were first cultivated in phenol red free DMEM containing 5% charcoal-stripped FBS for 48 hours to deplete hormone. EGR3 mRNA levels in the cells were measured by real time RT-PCR after a brief (8 h) treatment with vehicle (No ligand), E2 (1 nM) or OH-Tam (100 nM) (A). Cells were also treated with vehicle (No ligand) or E2 (10 nM) for 30 minutes and ligand-dependent activation of ER was analyzed by western blot using a specific antibody to detect phosphorylation at ser118 of ER; GAPDH was probed as a loading control (B). Hormone-depleted MCF-7 cells were transfected with either ER siRNA or control siRNA and maintained in hormone-depleted conditioned media without further replenishment of the media. Twenty-four hours after transfection, cells were treated with vehicle (No ligand), E2 (1 nM) or OH-Tam (100 nM); the treatments were repeated every 48 hours without changing the media. Following the treatment, viable cells were counted daily for six days by the Trypan Blue dye exclusion assay (C). In parallel, cells were harvested on each day of the treatments and total RNA extracted from them; the mRNA for ER was measured by real time RT-PCR and the values were normalized to those for GAPDH (D). In addition, cells were harvested in parallel on each day of the treatments for western blot analysis using antibody to ER; GAPDH was probed in each blot as a loading control (E). P-values for the differences noted in the text were <=0.001.\\nRESULTS\\nparagraph\\nWhen hormone depleted MCF-7 cells were seeded at a low confluency (<20 percent) and grown, in the absence of hormone and without replenishing the medium, they continued to proliferate and the proliferation was not inhibited by OH-Tam; we will call this \\'basal\\' proliferation. Under these conditions, we have monitored basal proliferation in MCF-7 cells for more than three months of continuous culture. Treatment with E2 stimulated the cell growth demonstrating that the cells were hormone-sensitive (Figure 1C). The basal proliferation was diminished by knocking down ER (Figure 1C). The expression level of ER mRNA (Figure 1D) progressively decreased with ligand (E2 or OH-Tam) treatment but the ER protein level (Figure 1E) was stabilized by the ligands during the treatment period; however, ER siRNA specifically and substantially down-regulated ER mRNA (within 24 h) and ER protein (within 48 h) (Figure 1D and 1E). The results demonstrate a profound role for apo-ER in supporting basal proliferation in hormone-sensitive MCF-7 cells.\\nRESULTS\\ntitle_2\\nApo-ER supports MCF-7 cell proliferation by promoting cell cycle progression\\nRESULTS\\nparagraph\\nThe net proliferation rate of MCF-7 cells under various conditions indicated in Figure 1 may be determined by changes in the rates of cell cycling as well as the rates of cell death. Figure 2A illustrates that during basal proliferation approximately 20% of the cells were in S-phase. Whereas E2 roughly doubled the proportion of S-phase cells, OH-Tam (100 nM) did not appreciably alter the basal cell cycle distribution (Figure 2A). However, knocking down ER in the hormone-depleted cells decreased the S-phase cells by approximately 50% (Figure 2A); the S-phase inhibition was accompanied by an increase in the proportion of cells in G1 phase indicating that the ER knockdown inhibited cell proliferation by inducing cell cycle arrest.\\nF2\\nFIG\\nfig_caption\\nHormone-independent effects of ER on cell cycle phase distribution in hormone-sensitive MCF-7 cells. Hormone-depleted MCF-7 cells were transfected with either ER siRNA or control siRNA and maintained in hormone-depleted conditioned media without further replenishment of the media. Twenty-four hours after transfection, cells were treated with vehicle (No ligand), E2 (1 nM) or OH-Tam (100 nM); the treatments were repeated every 48 hours without changing the media. (A) On the indicated days, the cells as treated above were harvested for flow cytometry analysis to determine their cell cycle phase distribution. (B) On Day 4 of the above treatment, cells were harvested to measure the proportion of apoptotic cells by Annexin V staining. P-values for the differences noted in the text were <=0.001.\\nRESULTS\\nparagraph\\nAs an alternative method of depleting ER, the hormone depleted MCF-7 cells were treated with fulvestrant, a well established SERD, which causes proteolytic degradation of ER. As expected, fulvestrant treatment resulted in a substantial decrease in ER protein (Figure 3B), without affecting the level of ER mRNA (Figure 3A). Similar to knocking down ER with siRNA, treatment with fulvestrant caused cell cycle arrest (Figure 3C), providing complementary evidence for the role of apo-ER in supporting cell cycling in hormone depleted MCF-7 cells.\\nF3\\nFIG\\nfig_caption\\nEffect of fulvestrant on cell cycle phase distribution in hormone-depleted MCF-7 cells. Hormone-depleted MCF-7 cells were treated with vehicle control or fulvestrant (100 nM) and maintained in hormone-depleted conditioned media without further replenishment of the media. The treatments were repeated every 48 hours for up to five days. (A) Cells were harvested after three and four days of treatment as described above and total RNA extracted from them. The mRNA for ER was measured by real time RT-PCR and the values were normalized to those for GAPDH. (B) Cells were harvested after three and four days of treatment as described above for western blot analysis using antibody to ER. GAPDH was probed as a loading control. (C) The cells treated as above were harvested on days 3 and 4 for flow cytometry analysis to determine their cell cycle phase distribution. (D) On Day 4 of the above treatment, cells were harvested to measure the proportion of apoptotic cells by Annexin V staining. P-values for the differences noted in the text were <=0.001.\\nRESULTS\\nparagraph\\nIn contrast to cell cycle distribution the rate of apoptosis, measured by staining the cells for Annexin V did not show a significant change due to ligand treatment or knocking down ER compared to hormone depletion (Figure 2B). Fulvestrant treatment modestly increased the proportion of Annexin V positive cells (from 4.6% to 8%) (Figure 3D). Therefore, the principal mechanism by which ER supports basal proliferation of hormone-sensitive MCF-7 cells is by promoting cell cycle progression.\\nRESULTS\\ntitle_2\\nApo-ER supports basal cycling of MCF-7 cells through regulation of apo-RARalpha\\nRESULTS\\nparagraph\\nAmong the list of genes whose expression decreased significantly upon knocking down ER in MCF-7 cells depleted of hormone (Additional files 1, 2, 3, 4 and 5), we observed a decrease in RARalpha. Since the actions of antiestrogens and retinoids on breast cancer cells are profoundly regulated by an ER-RAR axis, it was of interest to examine a possible functional relationship between the two receptors in the absence of ligand in the context of basal cycling of ER+ breast cancer cells. In hormone depleted MCF-7 cells, knocking down ER decreased the level of RARalpha mRNA by about 40% and to a greater extent the RARalpha protein (Figure 4A). However, the basal ER level was unaltered by knocking down RARalpha (Figure 4A). Knocking down either ER or RARalpha did not significantly alter the expression of RARs -beta and -gamma (Figure 4B). The results indicate that apo-RARalpha is specifically regulated by apo-ER but not vice versa.\\nF4\\nFIG\\nfig_caption\\nRole of RARalpha in mediating the hormone-independent effect of ER on basal level cell cycling. (A) Effect of knocking down either ER or RARalpha on the mRNA levels (left panel) and protein levels (right panel) of ER and RARalpha. Cells were transfected with control siRNA, ER siRNA or RARalpha siRNA and four days later the cells were harvested to extract total RNA for the measurement of ER and RARalpha mRNA by real time RT-PCR; the values were normalized those for GAPDH (control). The cells were also harvested four days after transfection for western blot analysis using antibody to either ER or RARalpha; the blots were probed for GAPDH as a loading control. (B) The cells were transfected as described for Panel A and the cell lysates were probed by western blot using antibodies specific for RARbeta and RARgamma; GAPDH was probed as a loading control. (C) Cells transfected as described in Panel A with control siRNA, ER siRNA and RARalpha siRNA were analysed by flow cytometry for cell cycle phase distribution. (D) RARalpha1 expression plasmid and siRNA against ER were co-transfected into hormone-depleted MCF-7 cells by nucleofection. As controls, cells were co-transfected with either control siRNA or ER siRNA and the vector plasmid. Cells were harvested three days after transfection and the cell cycle phase distribution determined by flow cytometry (left panel). The cells were also harvested at the same time for western blot analysis of the lysates using antibody to ER and RARalpha (right panel); GAPDH was probed as a loading control. (E) RARalpha1 expression plasmid or control vector plasmid was transfected into hormone-depleted MCF-7 cells using Fugene 6. The cells were treated with fulvestrant (100 nM) or vehicle and harvested after 72 h. The cell cycle phase distribution was determined by flow cytometry. (F) The cells were transfected as described in Panel E and harvested 96 hours after transfection. The cell cycle phase distribution was determined by flow cytometry (left panel) and the cell lysates were probed by Western blot using antibody specific for RARalpha (right panel); GAPDH was probed as a loading control.\\nRESULTS\\nparagraph\\nSimilar to knocking down ER, knocking down RARalpha in hormone-depleted MCF-7 cells decreased the fraction of S-phase cells (Figure 4C). To test whether the effect of knocking down ER on the basal cycling of MCF-7 cells could be mediated by apo-RARalpha, the latter protein was introduced ectopically at the time of knocking down ER by co-transfecting an RARalpha isoform1 expression plasmid (Figure 4D). Restoring RARalpha at a level approaching the original basal level of the protein partially rescued cell cycling (Figure 4D). Similarly, RARalpha also partially rescued basal cell cycling in MCF-7 cells treated with fulvestrant (Figure 4E). On the other hand, overexpression of RARalpha1 had no impact on the cell cycle distribution, indicating that the endogenous level of RARalpha1 is optimal for supporting the basal level of cell cycling (Figure 4F). Taken together, the above results demonstrate that the ability of apo-ER to support basal cell proliferation is mediated to a large extent by its ability to support the expression of the basal level of apo-RARalpha.\\nRESULTS\\ntitle_2\\nRESULTS\\nparagraph\\nSince the MCF-7 cell line comprises a heterogeneous population of cells, and since ErbB2 and ErbB3 overexpression is associated with resistance to hormonal adjuvants it was of interest to test whether the ErbB2 and ErbB3 status of the cells was related to the dependence of basal cell cycling of MCF-7 cells on apo-ER or apo-RARalpha. As expected, treating hormone depleted MCF-7 cells with E2 inhibited ErbB2 and ErbB3 mRNA expression (Figure 5). Following knockdown of either ER or RARalpha, there was not a significant change in the mRNA levels of ErbB2 and ErbB3 compared to the hormone depleted control cells (Figure 5). Thus, the regulation of cell cycling of MCF-7 cells by apo-ER or apo-RARalpha did not show an obvious selectivity for a subpopulation of cells distinguishable by their ErbB2 or ErbB3 status.\\nF5\\nFIG\\nfig_caption\\nERBB2 and ERBB3 mRNA levels following depletion of apo-ER or apo-RARalpha1 in MCF-7 cells. Cells were treated with vehicle, E2 (1 nM) or transfected with control siRNA, ER siRNA or RARalpha siRNA. Ninety-six days later the cells were harvested to extract total RNA for the measurement of ERBB2 and ERBB3 by real time RT-PCR. Values are normalized to the GAPDH values. P-values for the differences noted in the text were <=0.001.\\nRESULTS\\ntitle_2\\nMCF-7 cells and several clinical breast tumors express isoform 1 of RARalpha\\nRESULTS\\nparagraph\\nGiven the significant differences in structure and regulation between the two RARalpha isoforms it was of interest to determine whether only one or both isoforms were relevant in the regulation of RARalpha by apo-ER in breast cancer cells. cDNA prepared from the total RNA of MCF-7 cells as well as five breast tumors and normal human tissue controls (breast, spleen, thymus and peripheral blood leukocytes) were analyzed for RARalpha isoform expression by PCR. As seen in Figure 6, whereas both RARalpha1 and RARalpha2 were expressed in spleen, thymus and peripheral blood leukocytes, normal breast and MCF-7 cells as well as four of the five breast tumors expressed virtually exclusively RARalpha1 (Figure 6). The observations made in the limited number of clinical tumors need to be extended to a larger number of samples; nevertheless, the observation is consistent with previous evidence for epigenetic silencing of RARalpha2 expression in MCF-7 cells and, together with the preceding observation that RARalpha1 rescued basal cell cycling in the ER knockdown cells, indicates that RARalpha1 is the relevant receptor isoform in the current study of breast tumor cells.\\nF6\\nFIG\\nfig_caption\\nIdentification of major RARalpha isoforms in various normal tissues and in breast cancer. Total RNA was extracted from MCF-7 cells, five human breast tumors and normal tissue controls (peripheral blood leukocytes, thymus, spleen and breast) and reverse transcribed into cDNA by RT-PCR. The cDNA fragments were amplified by competitive PCR using forward primers specifically against RARalpha1 and RARalpha2 and a common reverse primer. The PCR products were identified by electrophoresis on a 2% agarose gel by ethedium bromide staining. The cDNA for GAPDH was amplified in each sample as an internal control.\\nRESULTS\\ntitle_2\\nApo-RARalpha1 mediates regulation by apo-ER of tamoxifen-insensitive gene complements principally engaged in the cell division cycle\\nRESULTS\\nparagraph\\nIn contrast to the effect of knocking down ER, the basal level of RARalpha1 did not appreciably decrease due to tamoxifen treatment (Figure 7A); the basal levels of RARs -beta and -gamma also only decreased by a relatively small extent due to tamoxifen treatment (Figure 7A). The gene targets downstream of the apo-ER   apo-RARalpha1 pathway were identified by mRNA profiling using Affymetrix microarray analysis. Accordingly, data from separate ER knockdown and RARalpha1 knockdown experiments in hormone depleted MCF-7 cells were used to identify overlapping sets of genes that were either up or down regulated by both ER and RARalpha1. The Affymetrix microarray analysis was validated for a few target genes known to regulate cell proliferation by real time RT-PCR; they include CDKN1A, MUC20 and LYPD1, which are negative regulators of cell proliferation whose basal expression was repressed by apo-ER and apo-RAR as well as CCNA2, a positive regulator of proliferation whose basal expression was supported by apo-ER and RARalpha1 (Figure 7B).\\nF7\\nFIG\\nfig_caption\\nTamoxifen-insensitivity of RAR expression and the functional categories of target genes of the apo-ER apo-RARalpha1 axis. (A) Hormone-depleted MCF-7 cells were treated with OH-Tam (100 nM) or vehicle control. Seventy-two hours later, the cell lysates were prepared and analysed by western blot using antibodies specific for RARalpha, RARbeta or RARgamma; GAPDH was probed as a loading control. The band intensities are normalized to GAPDH and equalized to a value of 1 for the untreated sample. (B) Real time RT-PCR analysis to confirm the effect of knocking down either ER or RARalpha on the mRNA levels of representative target genes found by Affymetrix DNA microarray in this study to be regulated by apo-ER and apo-RAR in MCF-7 cells: Cells were transfected with control siRNA, ER siRNA or RARalpha siRNA and four days later the cells were harvested to extract total RNA for the measurement of the relevant mRNAs; the values were normalized those for GAPDH (control). The P-values for the differences noted in the text were <0.001. (C) and (D) mRNA profiling was performed to identify common target genes of apo-ER and apo-RARalpha1 in MCF-7 cells. Apo-ER and apo-RARalpha1 were knocked down separately in hormone-depleted MCF-7 cells. Seventy-two hours after transfection with the appropriate siRNA, total mRNA was extracted and mRNA profiling was carried out using Affymetrixs microarray analysis. P-values for the differences noted in the text were <=0.001.\\nRESULTS\\nparagraph\\nTable 1 lists the common target genes of apo-ER and apo-RARalpha1 whose basal expression was supported by both apo-ER and apo-RARalpha1 in a tamoxifen-insensitive manner; these genes were identified based on a decrease in basal mRNA level by at least 40% due to transfection of cells with siRNA specific for either ER or RARalpha1. Of the 53 annotated genes that were identified in this manner, gene ontology analysis (DAVID Bioinformatics Resources 2008) revealed genes known to support the cell division cycle and mitosis as the predominant functional category (Figure 7C); the gene sets included additional functional categories that support proliferation (Figure 7C).\\nRESULTS\\nparagraph\\nTable 2 lists the common target genes of apo-ER and apo-RARalpha1 whose basal expression was decreased by apo-ER or apo-RARalpha1 in a tamoxifen-insensitive manner; these genes were identified based on an increase in basal mRNA level by at least 50 percent due to transfection of cells with siRNA specific for either ER or RARalpha1. Of the 68 annotated genes that were identified in this manner, gene ontology analysis (DAVID Bioinformatics Resources 2008) revealed several categories of genes (Figure 7D); however, among genes functionally related to proliferation, there was enrichment for those that are known to negatively regulate cell proliferation (Figure 7D) in contrast to the genes activated by apo-ER/apo-RARalpha1 noted above.\\nRESULTS\\ntitle_2\\nThe common gene targets of apo-ER and apo-RARalpha1 are generally insensitive to ATRA but are enriched in RAR binding sites in associated chromatin regions\\nRESULTS\\nparagraph\\nIn the classical mechanism of the transcriptional activity of class II nuclear receptors, including RAR, the apo-protein is in a repressive association with the target gene and the binding of agonist to the receptor results in gene activation due to a co-regulator switch. However, Affymetrix microarray analysis of MCF-7 cells following treatment with ATRA (1 muM, 24 h) indicated that only a fraction of the genes regulated by apo-ER/apo-RARalpha1 were regulated by ATRA (Tables 1 and 2). Specifically, among the 53 genes whose basal expression was supported by apo-ER/apo-RARalpha1, 6 genes were inhibited by ATRA to <50 percent and 1 was activated >2-fold (Table 1). On the other hand, among the 68 genes whose basal expression was repressed by apo-ER/apo-RARalpha1, 15 genes were activated by ATRA >2-fold and 10 genes were inhibited to <50 percent. Notably, none of the ATRA regulated genes in Tables 1 and 2 are functionally associated with the cell division cycle.\\nRESULTS\\nparagraph\\nSince putative RAR binding sites have been globally mapped in the chromatin of MCF-7 cells, we used this information to identify the presence of RAR binding sites associated with the apo-ER/apo-RARalpha1 target genes listed in Tables 1 and 2. As indicated in the Tables, 18 of the 53 genes in Table 1 and 14 of the 68 genes in Table 2 were associated with RAR binding sites within a distance of 10 kb of the transcription start sites. The genes in Table 1 had significant enrichment (Hypergeometric test) for RAR binding sites within 10 kb with P = 0.03. The enrichment for RAR binding sites for the genes in Table 2 had a P-value of 0.19. Thus, despite the low frequency with which the apo-ER/apo-RARalpha1 target genes are regulated by ATRA, there is a significant enrichment among them for associated RAR binding sites, largely among gene targets that are insensitive to ATRA.\\nRESULTS\\ntitle_2\\nDown-regulation of RARalpha1 inhibits hormone-independent cell cycle progression in other breast cancer cell lines\\nRESULTS\\nparagraph\\nSince down-regulating, rather than activating RARalpha1 may be an attractive therapeutic strategy in concert with estrogen ablation in breast cancer, it was of interest to test the effect of depleting RARalpha1 in other model cell lines. Similar to MCF7 cells, RARalpha1 but not RARalpha2 was expressed in the estrogen-sensitive T47 D and ZR-75-1 cell lines as determined by competitive RT-PCR (Figure 8A). Interestingly, in contrast to MCF-7 cells, RARalpha1 was not regulated by apo-ER and further knocking down apo-ER did not affect cell cycling (data not shown). Thus, the regulation of RARalpha1 by apo-ER is somehow disrupted in these cells; this finding further supports the conclusion that regulation of RARalpha1 underlies the effect of apo-ER on cell cycling. Nevertheless, in either cell line, knocking down RARalpha1 (Figure 8B showing RARalpha1 mRNA and 8C showing RARalpha1 protein) inhibited cell cycle progression in the absence of hormone (Figure 8D), similar to MCF7 cells. The results demonstrate a consistent role for RARalpha1 in supporting a basal level cycling in hormone depleted breast cancer cells.\\nF8\\nFIG\\nfig_caption\\nEffect of depleting RARalpha1 on cell cycle phase distribution of hormone-depleted T47 D and ZR-75-1 cells. (A) Total RNA was extracted from T47 D and ZR-75-1 cells and normal tissue controls (peripheral blood leukocytes, thymus and spleen) and reverse transcribed into cDNA by RT-PCR. The cDNA fragments were amplified by competitive PCR using forward primers specifically against RARalpha1 and RARalpha2 and a common reverse primer. The PCR products were identified as previously described (Figure 5). The cDNA for GAPDH was amplified in each sample as an internal control. In B and C, cells were transfected with control siRNA or RARalpha siRNA and 72 hours later the cells were harvested to extract total RNA or to prepare cell lysates. In B, the mRNA for RARalpha1 was measured by real time RT-PCR and the values were normalized those for GAPDH (control). In C, the cells lysates were analyzed by western blot using antibody to RARalpha; the blots were probed for GAPDH as a loading control. (D) Seventy-two hours after transfection with control siRNA or RARalpha siRNA, T47 D and ZR-75-1 cells were harvested for flow cytometry analysis to determine their cell cycle phase distribution. P-values for the differences noted in the text were <=0.001.\\nDISCUSS\\ntitle_1\\nDiscussion\\nDISCUSS\\nparagraph\\nER is known to regulate genes in a ligand-independent manner. Hormone-independent actions of ER play an important role in supporting the growth of hormone-refractory breast tumors. On the other hand, studies of gene regulation by ER in estrogen-sensitive breast cancer cells have mostly focused on estrogen-responsive genes that have profound roles in tumor growth and development and the effects of tamoxifen on gene regulation by estrogen. The findings of this study, however, highlight a potentially significant mechanism of hormone-independent transcriptional action of ER in hormone-sensitive breast cancer cells. This action of ER is clearly a major contributor to the ability of hormone-sensitive breast cancer cells to maintain a basal level of proliferation under conditions of hormone-depletion. This effect of apo-ER occurred primarily through supporting the cell division cycle. Remarkably, the action of apo-ER was also rather insensitive to tamoxifen at a dose that is clinically relevant to circulating concentrations of the drug that induce all of the surrogate biomarkers of clinical response. Similar to clinical breast tumors, breast cancer cell lines are heterogeneous and can yield clonal populations of inherently tamoxifen resistant cells that are variably ER-dependent; nevertheless, the fraction of S-phase cells in hormone-depleted or tamoxifen-treated cells under the in vitro conditions in this study was much higher than the frequency of emergence of aggressively growing colonies in tamoxifen-treated cultures. Therefore, it is likely that the basal level of S-phase cells observed in hormone-depleted or tamoxifen treated cultures represent a substantial proportion of cells in which the cell cycle progression is slowed. In a tumor environment, however, this slow proliferation must be offset by cell death, resulting in an overall tumoristatic effect. Since a basal level of cell division is an essential pre-condition for progressive events leading to the eventual development of resistance of breast tumors to hormonal adjuvant therapy, understanding the mechanism of the hormone-independent effects of ER in hormone-sensitive cells is important.\\nDISCUSS\\nparagraph\\nIn hormone-sensitive breast cancer cells, the well-established ER-RAR axis has been best characterized in the context of ligand effects (estrogen, retinoids, tamoxifen + retinoids). The results of this study however establish that in hormone-sensitive cells that are depleted of hormone or treated with tamoxifen, a major mechanism by which ER supports the cell cycle is by supporting the basal expression of RARalpha1. The role of RARalpha1 in mediating the action of apo-ER is strongly evident from the following observations: (i) In hormone-depleted cells, apo-ER maintained the basal expression level of RARalpha1 but was not itself regulated by RARalpha1; (ii) The regulation of RARalpha1 by apo-ER was insensitive to tamoxifen; (iii) Knocking down RARalpha1 negatively impacted the basal cell cycle progression and restoring basal apo-RARalpha1 levels rescued basal level cell division following depletion of ER; (iv) Apo-RARalpha1 independently regulated a complement of genes in a manner that strongly favored cell division similar to their regulation by apo-ER. This mechanism was remarkable for the following reasons. First, apo-ER regulated the alpha1 subtype of RAR but not RARs -beta or - gamma. Second, most of the common target genes of apo-ER and apo-RARalpha1 including all of the genes involved in the cell division cycle were insensitive to ATRA. These findings suggest that a major molecular mechanism by which apo-ER supports basal cell division in hormone-sensitive breast cancer cells may not be sensitive to conventional RAR ligands (agonists), but would be predictably opposed by specific inactivators or down-regulators of RARalpha1.\\nDISCUSS\\nparagraph\\nThe cell cycle regulation, which occurs through the apo-ER/apo-RARalpha1 axis, could theoretically exclude a subpopulation(s) of cells; such a subpopulation(s) could represent tumor cells that are either inherently resistant to hormonal adjuvants or that undergo progressive changes leading to resistance. Whereas the findings in this study do not preclude this possibility, we found no evidence for residual cycling cells following depletion of apo-ER or apo-RARalpha1 that were characterized by ErbB2 or ErbB3 overexpression, common features associated with a resistant phenotype.\\nDISCUSS\\nparagraph\\nIt is well established that the RARalpha gene is activated by estrogen; however, there is evidence in the literature that ER associates at a basal level with the core promoter of the RARalpha gene by tethering to DNA bound Sp1. Apo-ER may thus directly regulate the RARalpha gene to maintain the basal expression level of RARalpha1. The observation that the RARalpha1 protein level decreased more dramatically than its mRNA upon knocking down ER suggests that apo-ER also regulates RARalpha1 by additional posttranscriptional mechanisms.\\nDISCUSS\\nparagraph\\nThe results of this study further indicate that multiple molecular mechanisms must underlie the downstream action of apo-RARalpha1 on target genes in the context of mediating the effects of apo-ER. The apo-ER/apo-RARalpha1 axis regulates genes in both a positive and a negative manner to support cell division; both sets of target genes were enriched for associated chromatin sites of RAR binding, suggesting that RARalpha1 must act on these target genes by direct as well as indirect mechanisms. RAR belongs to the Class II subfamily of nuclear receptors, which typically, in their ligand-free (apoprotein) form, maintain a transcriptionally repressed state of target genes activated by the corresponding agonists. However, only a small fraction of genes regulated by the apo-ER-RARalpha1 axis appeared to be regulated by this classical mechanism of action of RARalpha1, since (i) the genes repressed by apo-RARalpha1 were largely insensitive to ATRA and (ii) most genes activated by apo-RARalpha1 were ATRA-insensitive. Therefore, apo-RARalpha1 must act by non-classical mechanisms on most of the target genes, including those with associated RAR binding sites.\\nDISCUSS\\nparagraph\\nRARalpha is consistently present in the nucleus in breast tumors and its expression levels correlate with that of the proliferation marker, ki-67. The functional RARalpha isoform in different hormone-sensitive breast cancer cell lines and that identified in a limited number of breast tumors was almost exclusively of type 1, an isoform that is believed to be genetically redundant. The findings reported here would predict that agents that selectively target the alpha1 subtype of RAR for functional inhibition or degradation would significantly enhance hormone ablation therapy in breast cancer since this would further decrease cycling cells. Since the effect of depleting RARalpha1 occurs through a different gene regulatory program compared with retinoid agonists, and since RARalpha1 is genetically redundant and could be the major or only RARalpha subtype in breast cancer cells, this new approach (rather than the use of RAR agonists) to enhancing hormonal adjuvant therapy in breast cancer may be clinically more acceptable. The structural divergence of the two RARalpha isoforms arising from alternative promoter usage and alternative splicing includes differences in functional sub-domains which may enable their differential targeting with pharmacological agents. This approach may have fewer side effects than SERDs due to a redundancy of RAR subtypes in other tissues. Studies are underway to test this concept in pre-clinical models of hormone-sensitive breast cancer.\\nCONCL\\ntitle_1\\nConclusions\\nCONCL\\nparagraph\\nWe have observed that in hormone-sensitive breast cancer cells, there is a hormone-independent component through which ER supports proliferation and that apo-RARalpha1 is a major mediator of this effect. The data also show that the majority of genes regulated by apo-RARalpha1 do not confirm to the classical model of gene regulation by Class II nuclear receptors since they are not regulated by ATRA. Further, selectively down-regulating RARalpha1 might significantly enhance hormone ablation therapy in breast cancer since this would further decrease cycling cells. Since the effect of depleting RARalpha1 occurs through a different gene regulatory program compared with retinoid agonists, and since RARalpha1 is genetically redundant and appears to be the only RARalpha subtype in breast tumor cell lines and possibly the major subtype in clinical tumors, this new approach (rather than the use of RAR agonists) to enhancing hormonal adjuvant therapy in breast cancer may be clinically more acceptable. RARalpha1 is also structurally divergent from RARalpha2, which is expressed in normal tissues, so that it should be possible to develop selective down-regulators of RARalpha1.\\nABBR\\ntitle_1\\nAbbreviations\\nABBR\\nparagraph\\nCOMP_INT\\ntitle_1\\nCompeting interests\\nCOMP_INT\\nparagraph\\nThe authors declare that they have no competing interests.\\nAUTH_CONT\\ntitle_1\\nAuthors\\' contributions\\nAUTH_CONT\\nparagraph\\nMDS conducted most of the experiments and contributed to the planning and analysis. MR, MP and IK contributed to the execution of some of the experiments. RT contributed to the bioinformatics analysis. IM contributed with an intellectual clinical perspective to the studies. MR was responsible for the overall direction of the project.\\nSUPPL\\ntitle_1\\nSupplementary Material\\nACK_FUND\\ntitle_1\\nAcknowledgements\\nACK_FUND\\nparagraph\\nThis work was supported by NIH R01 grant CA 140690 and the Harold & Helen McMaster Endowment to M.R. The authors acknowledge Sean Linkes for assistance with flow cytometry and the use of the institutional Flow Cytometry core lab.\\nsurname:Ali;given-names:S\\nsurname:Coombes;given-names:RC\\nREF\\nNat Rev Cancer\\nref\\nEndocrine-responsive breast cancer and strategies for combating resistance\\nsurname:Cuzick;given-names:J\\nsurname:Powles;given-names:T\\nsurname:Veronesi;given-names:U\\nsurname:Forbes;given-names:J\\nsurname:Edwards;given-names:R\\nsurname:Ashley;given-names:S\\nsurname:Boyle;given-names:P\\nREF\\nLancet\\nref\\nOverview of the main outcomes in breast-cancer prevention trials\\nsurname:Dunn;given-names:BK\\nsurname:Ford;given-names:LG\\nREF\\nBreast J\\nref\\nFrom adjuvant therapy to breast cancer prevention: BCPT and STAR\\nsurname:Mauri;given-names:D\\nsurname:Pavlidis;given-names:N\\nsurname:Polyzos;given-names:NP\\nsurname:Ioannidis;given-names:JP\\nREF\\nJ Natl Cancer Inst\\nref\\nSurvival with aromatase inhibitors and inactivators versus standard hormonal therapy in advanced breast cancer: meta-analysis\\nsurname:Mouridsen;given-names:H\\nsurname:Giobbie-Hurder;given-names:A\\nsurname:Gelber;given-names:RD\\nsurname:Coates;given-names:AS\\nsurname:Goldhirsch;given-names:A\\nsurname:Thurlimann;given-names:B\\nsurname:Paridaens;given-names:R\\nsurname:Smith;given-names:I\\nsurname:Mauriac;given-names:L\\nsurname:Forbes;given-names:JF\\nsurname:Price;given-names:KN\\nsurname:Regan;given-names:MM\\nREF\\nN Engl J Med\\nref\\nsurname:Howell;given-names:A\\nsurname:Cuzick;given-names:J\\nsurname:Baum;given-names:M\\nsurname:Buzdar;given-names:A\\nsurname:Dowsett;given-names:M\\nsurname:Forbes;given-names:JF\\nsurname:Hoctin-Boes;given-names:G\\nsurname:Houghton;given-names:J\\nsurname:Locker;given-names:GY\\nsurname:Tobias;given-names:JS\\nREF\\nLancet\\nref\\nsurname:Brodie;given-names:A\\nREF\\nTrends Endocrinol Metab\\nref\\nAromatase inhibitors in breast cancer\\nsurname:Howell;given-names:A\\nsurname:Dowsett;given-names:M\\nREF\\nBreast Cancer Res\\nref\\nEndocrinology and hormone therapy in breast cancer: aromatase inhibitors versus antioestrogens\\nsurname:Masri;given-names:S\\nsurname:Phung;given-names:S\\nsurname:Wang;given-names:X\\nsurname:Wu;given-names:X\\nsurname:Yuan;given-names:YC\\nsurname:Wagman;given-names:L\\nsurname:Chen;given-names:S\\nREF\\nCancer Res\\nref\\nGenome-wide analysis of aromatase inhibitor-resistant, tamoxifen-resistant, and long-term estrogen-deprived cells reveals a role for estrogen receptor\\nsurname:Dowsett;given-names:M\\nsurname:Cuzick;given-names:J\\nsurname:Coombes;given-names:C\\nsurname:Snowdon;given-names:C\\nsurname:Gnant;given-names:M\\nsurname:Jakesz;given-names:R\\nsurname:Kaufmann;given-names:M\\nsurname:Boccardo;given-names:F\\nsurname:Godwin;given-names:J\\nsurname:Davies;given-names:C\\nsurname:Peto;given-names:R\\nsurname:Ingle;given-names:J\\nsurname:Coates;given-names:A\\nsurname:Forbes;given-names:J\\nsurname:Bliss;given-names:J\\nsurname:Buyse;given-names:M\\nsurname:Baum;given-names:M\\nsurname:Buzdar;given-names:A\\nsurname:Colleoni;given-names:M\\nREF\\nJ Clin Oncol\\nref\\nMeta-analysis of breast cancer outcomes in adjuvant trials of aromatase inhibitors versus tamoxifen\\nsurname:Winer;given-names:EP\\nsurname:Hudis;given-names:C\\nsurname:Cobleigh;given-names:MA\\nsurname:Mamounas;given-names:EP\\nsurname:Goldstein;given-names:LJ\\nsurname:Whelan;given-names:TJ\\nsurname:Powles;given-names:TJ\\nsurname:Bryant;given-names:J\\nsurname:Perkins;given-names:C\\nsurname:Perotti;given-names:J\\nsurname:Braun;given-names:S\\nsurname:Langer;given-names:AS\\nsurname:Burstein;given-names:HJ\\nsurname:Browman;given-names:GP\\nsurname:Somerfield;given-names:MR\\nsurname:Wolff;given-names:AC\\nsurname:Pritchard;given-names:KI\\nsurname:Ingle;given-names:JN\\nsurname:Chlebowski;given-names:RT\\nsurname:Gelber;given-names:R\\nsurname:Edge;given-names:SB\\nsurname:Gralow;given-names:J\\nREF\\nJ Clin Oncol\\nref\\nsurname:Musgrove;given-names:EA\\nsurname:Sutherland;given-names:RL\\nREF\\nNat Rev Cancer\\nref\\nBiological determinants of endocrine resistance in breast cancer\\nsurname:Kuske;given-names:B\\nsurname:Naughton;given-names:C\\nsurname:Moore;given-names:K\\nsurname:Macleod;given-names:KG\\nsurname:Miller;given-names:WR\\nsurname:Clarke;given-names:R\\nsurname:Langdon;given-names:SP\\nsurname:Cameron;given-names:DA\\nREF\\nEndocr Relat Cancer\\nref\\nEndocrine therapy resistance can be associated with high estrogen receptor alpha (ERalpha) expression and reduced ERalpha phosphorylation in breast cancer models\\nS2\\nSuppl 1\\nsurname:Osborne;given-names:CK\\nsurname:Wakeling;given-names:A\\nsurname:Nicholson;given-names:RI\\nREF\\nBr J Cancer\\nref\\nFulvestrant: an oestrogen receptor antagonist with a novel mechanism of action\\nsurname:Bross;given-names:PF\\nsurname:Baird;given-names:A\\nsurname:Williams;given-names:G\\nsurname:Pazdur;given-names:R\\nsurname:Chen;given-names:G\\nsurname:Jee;given-names:JM\\nsurname:Lostritto;given-names:RT\\nsurname:Morse;given-names:DE\\nsurname:Rosario;given-names:LA\\nsurname:Williams;given-names:GM\\nsurname:Yang;given-names:P\\nsurname:Rahman;given-names:A\\nREF\\nClin Cancer Res\\nref\\nFulvestrant in postmenopausal women with advanced breast cancer\\nsurname:Zanardi;given-names:S\\nsurname:Serrano;given-names:D\\nsurname:Argusti;given-names:A\\nsurname:Barile;given-names:M\\nsurname:Puntoni;given-names:M\\nsurname:Decensi;given-names:A\\nREF\\nEndocr Relat Cancer\\nref\\nClinical trials with retinoids for breast cancer chemoprevention\\nsurname:Zhou;given-names:Q\\nsurname:Stetler-Stevenson;given-names:M\\nsurname:Steeg;given-names:PS\\nREF\\nOncogene\\nref\\nsurname:Toma;given-names:S\\nsurname:Isnardi;given-names:L\\nsurname:Raffo;given-names:P\\nsurname:Dastoli;given-names:G\\nsurname:De Francisci;given-names:E\\nsurname:Riccardi;given-names:L\\nsurname:Palumbo;given-names:R\\nsurname:Bollag;given-names:W\\nREF\\nInt J Cancer\\nref\\nsurname:Yang;given-names:LM\\nsurname:Tin;given-names:UC\\nsurname:Wu;given-names:K\\nsurname:Brown;given-names:P\\nREF\\nJ Mammary Gland Biol Neoplasia\\nref\\nRole of retinoid receptors in the prevention and treatment of breast cancer\\nsurname:Wang;given-names:Y\\nsurname:He;given-names:QY\\nsurname:Chen;given-names:H\\nsurname:Chiu;given-names:JF\\nREF\\nExp Cell Res\\nref\\nsurname:Danforth;given-names:DN;suffix:Jr\\nREF\\nJ Endocrinol\\nref\\nAll trans-retinoic acid acts synergistically with hydroxytamoxifen and transforming-growth factor beta to stimulate apoptosis in MCF-7 breast cancer cells\\nsurname:Budd;given-names:GT\\nsurname:Adamson;given-names:PC\\nsurname:Ganapathi;given-names:R\\nsurname:Gupta;given-names:M\\nsurname:Homayoun;given-names:P\\nsurname:Sandstrom;given-names:SK\\nsurname:Murphy;given-names:RF\\nsurname:McLain;given-names:D\\nsurname:Tuason;given-names:L\\nsurname:Peereboom;given-names:D\\nsurname:Bukowski;given-names:RM\\nREF\\nClin Cancer Res\\nref\\nsurname:Decensi;given-names:A\\nsurname:Zanardi;given-names:S\\nsurname:Argusti;given-names:A\\nsurname:Bonanni;given-names:B\\nsurname:Costa;given-names:A\\nsurname:Veronesi;given-names:U\\nREF\\nNat Clin Pract Oncol\\nref\\nFenretinide and risk reduction of second breast cancer\\nsurname:Millar;given-names:AW\\nsurname:Lynch;given-names:KP\\nREF\\nNat Rev Cancer\\nref\\nRethinking clinical trials for cytostatic drugs\\nsurname:Badia;given-names:E\\nsurname:Oliva;given-names:J\\nsurname:Balaguer;given-names:P\\nsurname:Cavailles;given-names:V\\nREF\\nCurr Med Chem\\nref\\nTamoxifen resistance and epigenetic modifications in breast cancer cell lines\\nsurname:Conzen;given-names:SD\\nREF\\nMol Endocrinol\\nref\\nMinireview: nuclear receptors and breast cancer\\nsurname:O\\'Regan;given-names:RM\\nsurname:Gajdos;given-names:C\\nsurname:Dardes;given-names:RC\\nsurname:De Los Reyes;given-names:A\\nsurname:Park;given-names:W\\nsurname:Rademaker;given-names:AW\\nsurname:Jordan;given-names:VC\\nREF\\nJ Natl Cancer Inst\\nref\\nsurname:Simstein;given-names:R\\nsurname:Burow;given-names:M\\nsurname:Parker;given-names:A\\nsurname:Weldon;given-names:C\\nsurname:Beckman;given-names:B\\nREF\\nExp Biol Med (Maywood)\\nref\\nApoptosis, chemoresistance, and breast cancer: insights from the MCF-7 cell model system\\nsurname:Zhu;given-names:Y\\nsurname:Wang;given-names:A\\nsurname:Liu;given-names:MC\\nsurname:Zwart;given-names:A\\nsurname:Lee;given-names:RY\\nsurname:Gallagher;given-names:A\\nsurname:Wang;given-names:Y\\nsurname:Miller;given-names:WR\\nsurname:Dixon;given-names:JM\\nsurname:Clarke;given-names:R\\nREF\\nInt J Oncol\\nref\\nsurname:Creighton;given-names:CJ\\nsurname:Massarweh;given-names:S\\nsurname:Huang;given-names:S\\nsurname:Tsimelzon;given-names:A\\nsurname:Hilsenbeck;given-names:SG\\nsurname:Osborne;given-names:CK\\nsurname:Shou;given-names:J\\nsurname:Malorni;given-names:L\\nsurname:Schiff;given-names:R\\nREF\\nCancer Res\\nref\\nDevelopment of resistance to targeted therapies transforms the clinically associated molecular profile subtype of breast tumor xenografts\\nsurname:Hao;given-names:H\\nsurname:d\\'Alincourt-Salazar;given-names:M\\nsurname:Kelley;given-names:KM\\nsurname:Shatnawi;given-names:A\\nsurname:Mukherjee;given-names:S\\nsurname:Shah;given-names:YM\\nsurname:Ratnam;given-names:M\\nREF\\nOncogene\\nref\\nsurname:Wettenhall;given-names:JM\\nsurname:Simpson;given-names:KM\\nsurname:Satterley;given-names:K\\nsurname:Smyth;given-names:GK\\n10.1093/bioinformatics/btl025\\nREF\\nBioinformatics\\nref\\nsurname:Ali;given-names:S\\nsurname:Metzger;given-names:D\\nsurname:Bornert;given-names:JM\\nsurname:Chambon;given-names:P\\nREF\\nEMBO J\\nref\\nsurname:Leroy;given-names:P\\nsurname:Krust;given-names:A\\nsurname:Zelent;given-names:A\\nsurname:Mendelsohn;given-names:C\\nsurname:Garnier;given-names:JM\\nsurname:Kastner;given-names:P\\nsurname:Dierich;given-names:A\\nsurname:Chambon;given-names:P\\nREF\\nEMBO J\\nref\\nMultiple isoforms of the mouse retinoic acid receptor alpha are generated by alternative splicing and differential induction by retinoic acid\\nsurname:Farias;given-names:EF\\nsurname:Arapshian;given-names:A\\nsurname:Bleiweiss;given-names:IJ\\nsurname:Waxman;given-names:S\\nsurname:Zelent;given-names:A\\nsurname:Mira;given-names:YLR\\nREF\\nCell Growth Differ\\nref\\nRetinoic acid receptor alpha2 is a growth suppressor epigenetically silenced in MCF-7 human breast cancer cells\\nsurname:Huang da;given-names:W\\nsurname:Sherman;given-names:BT\\nsurname:Lempicki;given-names:RA\\nREF\\nNat Protoc\\nref\\nSystematic and integrative analysis of large gene lists using DAVID bioinformatics resources\\nP3\\nsurname:Dennis;given-names:G;suffix:Jr\\nsurname:Sherman;given-names:BT\\nsurname:Hosack;given-names:DA\\nsurname:Yang;given-names:J\\nsurname:Gao;given-names:W\\nsurname:Lane;given-names:HC\\nsurname:Lempicki;given-names:RA\\nREF\\nGenome Biol\\nref\\nDAVID: Database for Annotation, Visualization, and Integrated Discovery\\nsurname:Altucci;given-names:L\\nsurname:Gronemeyer;given-names:H\\nREF\\nNat Rev Cancer\\nref\\nThe promise of retinoids to fight against cancer\\nsurname:Hua;given-names:S\\nsurname:Kittler;given-names:R\\nsurname:White;given-names:KP\\nREF\\nCell\\nref\\nGenomic antagonism between retinoic acid and estrogen signaling in breast cancer\\nsurname:Cvoro;given-names:A\\nsurname:Tzagarakis-Foster;given-names:C\\nsurname:Tatomer;given-names:D\\nsurname:Paruthiyil;given-names:S\\nsurname:Fox;given-names:MS\\nsurname:Leitman;given-names:DC\\nREF\\nMol Cell\\nref\\nDistinct roles of unliganded and liganded estrogen receptors in transcriptional repression\\nsurname:Cardamone;given-names:MD\\nsurname:Bardella;given-names:C\\nsurname:Gutierrez;given-names:A\\nsurname:Di Croce;given-names:L\\nsurname:Rosenfeld;given-names:MG\\nsurname:Di Renzo;given-names:MF\\nsurname:De Bortoli;given-names:M\\nREF\\nProc Natl Acad Sci USA\\nref\\nERalpha as ligand-independent activator of CDH-1 regulates determination and maintenance of epithelial morphology in breast cancer cells\\nsurname:Welboren;given-names:WJ\\nsurname:Sweep;given-names:FC\\nsurname:Span;given-names:PN\\nsurname:Stunnenberg;given-names:HG\\nREF\\nEndocr Relat Cancer\\nref\\nGenomic actions of estrogen receptor alpha: what are the targets and how are they regulated?\\nsurname:Decensi;given-names:A\\nsurname:Gandini;given-names:S\\nsurname:Lien;given-names:EA\\nsurname:Guerrieri-Gonzaga;given-names:A\\nsurname:Johansson;given-names:H\\nsurname:Manetti;given-names:L\\nsurname:Bonanni;given-names:B\\nsurname:Sandri;given-names:MT\\nsurname:Barreca;given-names:A\\nsurname:Costa;given-names:A\\nsurname:Robertson;given-names:C\\nREF\\nJ Clin Oncol\\nref\\nsurname:Decensi;given-names:A\\nsurname:Robertson;given-names:C\\nsurname:Sandri;given-names:MT\\nsurname:Pelosi;given-names:G\\nsurname:Luini;given-names:A\\nsurname:Goldhirsch;given-names:A\\nsurname:Lien;given-names:EA\\nsurname:Veronesi;given-names:U\\nsurname:Viale;given-names:G\\nsurname:Pigatto;given-names:F\\nsurname:Johansson;given-names:H\\nsurname:Kisanga;given-names:ER\\nsurname:Veronesi;given-names:P\\nsurname:Torrisi;given-names:R\\nsurname:Cazzaniga;given-names:M\\nsurname:Mora;given-names:S\\nREF\\nJ Natl Cancer Inst\\nref\\nsurname:Coser;given-names:KR\\nsurname:Wittner;given-names:BS\\nsurname:Shioda;given-names:T\\nsurname:Rosenthal;given-names:NF\\nsurname:Collins;given-names:SC\\nsurname:Melas;given-names:A\\nsurname:Smith;given-names:SL\\nsurname:Mahoney;given-names:CJ\\nsurname:Shioda;given-names:K\\nsurname:Isselbacher;given-names:KJ\\nsurname:Ramaswamy;given-names:S\\nREF\\nProc Natl Acad Sci USA\\nref\\nAntiestrogen-resistant subclones of MCF-7 human breast cancer cells are derived from a common monoclonal drug-resistant progenitor\\nsurname:Ross-Innes;given-names:CS\\nsurname:Stark;given-names:R\\nsurname:Holmes;given-names:KA\\nsurname:Schmidt;given-names:D\\nsurname:Spyrou;given-names:C\\nsurname:Russell;given-names:R\\nsurname:Massie;given-names:CE\\nsurname:Vowler;given-names:SL\\nsurname:Eldridge;given-names:M\\nsurname:Carroll;given-names:JS\\nREF\\nGenes Dev\\nref\\nCooperative interaction between retinoic acid receptor-alpha and estrogen receptor in breast cancer\\nsurname:Stern;given-names:DF\\nREF\\nJ Mammary Gland Biol Neoplasia\\nref\\nERBB3/HER3 and ERBB2/HER2 duet in mammary development and breast cancer\\nsurname:Liu;given-names:B\\nsurname:Ordonez-Ercan;given-names:D\\nsurname:Fan;given-names:Z\\nsurname:Edgerton;given-names:SM\\nsurname:Yang;given-names:X\\nsurname:Thor;given-names:AD\\nREF\\nInt J Cancer\\nref\\nDownregulation of erbB3 abrogates erbB2-mediated tamoxifen resistance in breast cancer cells\\nsurname:Sun;given-names:G\\nsurname:Porter;given-names:W\\nsurname:Safe;given-names:S\\nREF\\nMol Endocrinol\\nref\\nEstrogen-induced retinoic acid receptor alpha 1 gene expression: role of estrogen receptor-Sp1 complex\\nsurname:van der Leede;given-names:BM\\nsurname:Geertzema;given-names:J\\nsurname:Vroom;given-names:TM\\nsurname:Decimo;given-names:D\\nsurname:Lutz;given-names:Y\\nsurname:van der Saag;given-names:PT\\nsurname:van der Burg;given-names:B\\nREF\\nAm J Pathol\\nref\\nsurname:Lufkin;given-names:T\\nsurname:Lohnes;given-names:D\\nsurname:Mark;given-names:M\\nsurname:Dierich;given-names:A\\nsurname:Gorry;given-names:P\\nsurname:Gaub;given-names:MP\\nsurname:LeMeur;given-names:M\\nsurname:Chambon;given-names:P\\nREF\\nProc Natl Acad Sci USA\\nref\\n',\n",
       " '21535883': 'BioC-API\\ncollection.key\\nCC BY\\nThis is an Open Access article distributed under the terms of the Creative Commons Attribution License (http://creativecommons.org/licenses/by/2.0), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.\\nsurname:Alberts;given-names:Rudi\\nsurname:Lu;given-names:Lu\\nsurname:Williams;given-names:Robert W\\nsurname:Schughart;given-names:Klaus\\nTITLE\\nfront\\nGenome-wide analysis of the mouse lung transcriptome reveals novel molecular gene interaction networks and cell-specific expression signatures\\nABSTRACT\\nabstract_title_1\\nBackground\\nABSTRACT\\nabstract\\nThe lung is critical in surveillance and initial defense against pathogens. In humans, as in mice, individual genetic differences strongly modulate pulmonary responses to infectious agents, severity of lung disease, and potential allergic reactions. In a first step towards understanding genetic predisposition and pulmonary molecular networks that underlie individual differences in disease vulnerability, we performed a global analysis of normative lung gene expression levels in inbred mouse strains and a large family of BXD strains that are widely used for systems genetics. Our goal is to provide a key community resource on the genetics of the normative lung transcriptome that can serve as a foundation for experimental analysis and allow predicting genetic predisposition and response to pathogens, allergens, and xenobiotics.\\nABSTRACT\\nabstract_title_1\\nMethods\\nABSTRACT\\nabstract\\nSteady-state polyA+ mRNA levels were assayed across a diverse and fully genotyped panel of 57 isogenic strains using the Affymetrix M430 2.0 array. Correlations of expression levels between genes were determined. Global expression QTL (eQTL) analysis and network covariance analysis was performed using tools and resources in GeneNetwork http://www.genenetwork.org.\\nABSTRACT\\nabstract_title_1\\nResults\\nABSTRACT\\nabstract\\nExpression values were highly variable across strains and in many cases exhibited a high heri-tability factor. Several genes which showed a restricted expression to lung tissue were identified. Using correlations between gene expression values across all strains, we defined and extended memberships of several important molecular networks in the lung. Furthermore, we were able to extract signatures of immune cell subpopulations and characterize co-variation and shared genetic modulation. Known QTL regions for respiratory infection susceptibility were investigated and several cis-eQTL genes were identified. Numerous cis- and trans-regulated transcripts and chromosomal intervals with strong regulatory activity were mapped. The Cyp1a1 P450 transcript had a strong trans-acting eQTL (LOD 11.8) on Chr 12 at 36 +- 1 Mb. This interval contains the transcription factor Ahr that has a critical mis-sense allele in the DBA/2J haplotype and evidently modulates transcriptional activation by AhR.\\nABSTRACT\\nabstract_title_1\\nConclusions\\nABSTRACT\\nabstract\\nLarge-scale gene expression analyses in genetic reference populations revealed lung-specific and immune-cell gene expression profiles and suggested specific gene regulatory interactions.\\nINTRO\\ntitle_1\\nBackground\\nINTRO\\nparagraph\\nThe lung is the first line of defense against many pathogens and inhaled xenobiotics and is therefore a key part of the immune system. Host defense is strongly influenced by genetic differences and several studies have shown that the genetic background and sequence difference among humans and other host species modulate susceptibility and resistance to infectious diseases, allergens, and xenobiotics. Systems genetics is a modern extension of complex trait analysis that jointly analyzes and integrates large sets of genotypes and phenotypes to explain and predict variation in outcome measures and disease severity (for review see). A typical systems genetics study relies on extensive single nucleotide polymorphism (SNP) data sets, matched data on RNA expression in key cells, tissues, or organs and a core set of key dependent measures such as disease susceptibility. These data are collected across a panel or population of genetically diverse individuals or strains. This group of individuals represents a natural genetic perturbation, with well defined genotype and haplotype differences comprising the \"treatment.\" The independent measurements in this case can consist either of the genotype or of crucial intervening variables such as the expression of genes and proteins.\\nINTRO\\nparagraph\\nIn this study, we exploited a very well characterized panel of inbred strains of mice (a mouse genetic reference panel) that consists of two parts--a small set of standard inbred strains and a larger family of BXD type recombinant inbred strains. The genome of each BXD strain represents a mixture of the C57BL/6J and DBA/2J parental background and is homozygous at almost every genomic location. The genomic make-up of each BXD line has been determined by extensive mapping with molecular markers. After performing microarray expression analysis for each of the BXD mice, the expression level of each gene can be used as a quantitative trait (e.g. ). By comparing these expression values for all BXD mice with their molecular markers data along the genome, genomic expression quantitative trait loci (eQTL) can be identified that are likely to regulate the expression of one or several genes. When an eQTL is located at the same genomic position as the gene itself (within a 10Mb interval of the gene) it is considered as a cis-eQTL. In this case, variations in the promoter sequence or in elements that determine the stability of the mRNA of the gene are the most likely causes for the observed differences in expression levels. If the eQTL is at a distant location from the regulated gene, the eQTL is referred to as a trans-eQTL.\\nINTRO\\nparagraph\\nHere, we performed a global gene expression analysis from the lungs of 47 BXD and eight widely used inbred strains. The aim of our study was to reveal genes and gene networks in mouse lung in steady state condition. We found that many genes had high variation in expression and that often this variation was highly heritable. This allowed us to identify many cis- and trans-eQTLs. In addition, we used the correlation structure in the data to obtain expression signatures for specific cell types within the lung.\\nMETHODS\\ntitle_1\\nMethods\\nMETHODS\\ntitle_2\\nMouse strains and sample preparation\\nMETHODS\\nparagraph\\nC57BL/6J, BALB/cByJ, FVB/NJ, and WSB/EiJ, as well as B6D2F1 and D2B6F1 lines were obtained from the University of Tennessee Health Science Center (UTHSC). DBA/2J, 129X1/SvJ, LP/J and SJL/J were obtained from The Jackson Laboratory. Mice from 38 BXD recombinant inbred strains were obtained from UTHSC and mice from nine BXD strains were obtained from The Jackson Laboratory. All animals were housed at UTHSC before sacrifice. Mice were killed by cervical dislocation and whole lungs including blood were removed and placed in RNAlater. Total RNA was extracted from the lungs using RNA STAT-60 (Tel-Test Inc.). RNA from two to five animals per strain were pooled and used for gene expression analysis. Animals used in this study were between 49 and 93 days of age. All inbred strains were profiled for both sexes, and for a given BXD strain either males or females were used. Mice were maintained under specific pathogen free conditions. All protocols involving mice were approved by the UTHSC Animal Care and Use Committee.\\nMETHODS\\ntitle_2\\nMicroarray analysis\\nMETHODS\\nparagraph\\nGene expression profiling was performed using Affymetrix GeneChip Mouse Genome 430 2.0 Arrays at UTHSC. Samples were amplified according to the recommended protocols by the manufacturer (Affymetrix, Santa Clara, CA, USA). In all cases, 4-5 mug of each biotinylated cRNA preparation was fragmented and included in a hybridization cocktail containing four biotinylated hybridization controls (BioB, BioC, BioD, and Cre), as recommended by the manufacturer. Samples were hybridized for 16 hours. After hybridization, GeneChips were washed, stained with SAPE, and read using an Affymetrix GeneChip fluidic station and scanner.\\nMETHODS\\ntitle_2\\nData preprocessing and analysis\\nMETHODS\\nparagraph\\nData analysis was performed using the GeneNetwork web service, a large resource with phenotypes and mRNA expression data for several genetic reference populations and multiple organisms. The expression data were preprocessed like all other datasets in GeneNetwork: adding an offset of 1 unit to each signal intensity value to ensure that the logarithm of all values were positive, computing the log2 value, performing a quantile normalization of the log2 values for the total set of arrays using the same initial steps used by the RMA transform, computing the Z scores for each cell value, multiplying all Z scores by 2 and adding 8 to the value of all Z scores. The advantage of this variant of a Z transformation is that all values are positive and that 1 unit represents approximately a 2-fold difference in expression as determined using the spike-in control probe sets (see for details). For correlation analyses we used Pearson\\'s correlation unless otherwise stated. Heritability was determined using ANOVA with one factor mouse strain, and by dividing the mean between-mouse-strain variance by the sum of the mean between-mouse strain variance plus the mean within-strain variance.\\nMETHODS\\ntitle_2\\nQTL Mapping and expression analyses\\nMETHODS\\nparagraph\\nAll probe sets were mapped using standard interval mapping methods at 1 cM intervals (~2 Mb) along all autosomes and the X chromosome. This procedure generates estimates of linkage between variation in transcript expression levels and chromosomal location. The entire set of values can be used to construct a set of QTL maps for all chromosomes (except Chr Y and the mitochondrial genome) in which position is plotted on the x-axis and the strength of linkage--the likelihood ratio statistic (LRS) or log of the odds ratio (LOD)-is plotted on the y-axis. An LRS of 18 or higher is significant at a genome-wide p value of < 0.5. To compute LRS values we exploited the computationally efficient Haley-Knott regression equations and a set of 3796 SNPs and microsatellite markers that we and others have genotyped over the past decade. In order to rapidly map all 45,101 probe sets we used our customized QTL Reaper code http://qtlreaper.sourceforge.net/. QTL Reaper performs up to a million permutations of an expression trait to calculate the genome-wide empirical p value and the LRS scores associated with each interval or marker. The peak linkage value and position was databased in GeneNetwork and users can rapidly retrieve and view these mapping results for any probe set. Any of the QTL maps can also be rapidly regenerated using the same Haley-Knott methods, again using functions imbedded in GeneNetwork. GeneNetwork also enable a search for epistatic interactions (pair scanning function) and composite interval mapping with control for a single marker.\\nMETHODS\\ntitle_2\\nData quality control\\nMETHODS\\nparagraph\\nWe used two simple but effective methods to confirm correct sample identification of all data entered into GeneNetwork. Expression of the Xist transcript (probe set 1427262_at) was used to validate the sex of the sample. Xist is involved in the inactivation of one X chromosome in females and is only expressed at high levels in females. Other genes that show strong sex-specific expression are Eif2s3y, Jarid1d and Ddx3y. In addition, we investigated several genes that exhibit a strongly bimodal Mendelian expression pattern, meaning that one parental allele exhibits a high expression level whereas the other allele exhibits a low expression and only the F1 hybrids are intermediate. The expression level of such transcripts is directly correlated with the genotype at this locus and they can collectively be used to confirm sample genotype and hence strain. For example, expression of the Rpgrip1 transcript (probe set 1421144_at) has a distinctly bimodal distribution, intermediate values for F1 animals, and is associated with a LOD score peak of 50 that corresponds precisely to the location of the cognate gene on Chr 14 at 52.5 Mb.\\nRESULTS\\ntitle_1\\nResults\\nRESULTS\\ntitle_2\\nVariation in gene expression\\nRESULTS\\nparagraph\\nThe Affymetrix M430 2.0 array that we used includes 45,101 probe sets and provides consensus estimates of expression for the vast majority of all protein coding genes. Table 1 gives an overview of the range of variation across strains in each of the probe sets used. Strikingly, more than 2,000 genes showed a range of expression that was larger than four-fold different between the strain with the lowest and the highest expression. Among the genes with the most extreme range in expression levels were Krt4 (keratin 4), Krt13 (keratin 13) and Krtdap (keratinocyte differentiation associated protein). Another gene with highly variable expression was Cftr (cystic fibrosis transmembrane conductance regulator homolog). This important lung disease-causing gene showed a four-fold variation in expression levels between strains. Several other genes with high variation were sex-specifically expressed genes, like Xist (inactive X specific transcripts), Ddx3y (DEAD (Asp-Glu-Ala-Asp) box polypeptide 3, Y-linked) and Serpina1b (serine (or cysteine) preptidase inhibitor, clade A, member 1B).\\nT1.xml\\nT1\\nTABLE\\ntable_caption\\nVariation in gene expression for 45,101 probe sets.\\nT1.xml\\nT1\\nTABLE\\ntable\\n<?xml version=\"1.0\" encoding=\"UTF-8\"?>\\n<table frame=\"hsides\" rules=\"groups\"><thead><tr><th align=\"center\">Fold change range</th><th align=\"center\">Log<sub>2 </sub>range</th><th align=\"right\">No. of genes</th></tr></thead><tbody><tr><td align=\"center\">1-2</td><td align=\"center\">0-1</td><td align=\"right\">30,392</td></tr><tr><td align=\"center\">2-4</td><td align=\"center\">1-2</td><td align=\"right\">11,965</td></tr><tr><td align=\"center\">4-8</td><td align=\"center\">2-3</td><td align=\"right\">1,980</td></tr><tr><td align=\"center\">8-16</td><td align=\"center\">3-4</td><td align=\"right\">498</td></tr><tr><td align=\"center\">16-32</td><td align=\"center\">4-5</td><td align=\"right\">132</td></tr><tr><td align=\"center\">32-64</td><td align=\"center\">5-6</td><td align=\"right\">60</td></tr><tr><td align=\"center\">64-inf</td><td align=\"center\">6-inf</td><td align=\"right\">42</td></tr></tbody></table>\\n\\nT1.xml\\nT1\\nTABLE\\ntable_foot\\nFold changes between the lowest and highest expressed mouse stains per probe sets were cal-culated and divided in seven bins. The corresponding range on log2 scale and the amount of genes in each range are given.\\nRESULTS\\ntitle_2\\nHeritability of variation in gene expression\\nRESULTS\\nparagraph\\nTo investigate to which extent the variation in expression was due to genetic effects, we calculated the heritability for each of the genes, which is the fraction of variation in expression caused by genetics. The heritability values ranged from as high as 0.96 (most of the variance was associated with between-strain differences) until as low as 0.01. Genes with the largest heritability were Cdk17/Pctk2 (cyclin-dependent kinase 17, probe set 1446130_at), Gm1337 (predicted gene 1337, 1443287_at) and Pdxdc1/KIAA0251 (pyridoxal-dependent decarboxylase domain containing 1, 1452705_at), all having a value above 0.99. High heritability values indicate that it is likely to successfully map QTLs that influence gene expression values.\\nRESULTS\\ntitle_2\\nLung-specific genes\\nRESULTS\\nparagraph\\nThe large dataset in GeneNetwork and its built-in features allowed us to compare the gene ex-pression patterns in the lung with data from 25 other tissues. First, we identified the most highly expressed genes in lung (Table 2 lists the 15 highest expression signals). Two of these genes were highly restricted to the lung and trachea: Sftpc (surfactant associated protein C) and Ager (advanced glycosylation end product-specific receptor) (Figure 1A, B) whereas Scgb1a1 (secretoglobin, family 1A, member 1 (uteroglobin)) was highly expressed in lung but also showed expression in some other tissues (Figure 1C). On the other hand, Hba-a1 (hemoglobin alpha, adult chain 1) was expressed at high levels in most tissues (Figure 1D). We then used Stfpc in a tissue correlation analysis to identify other genes that may not be as highly expressed but still be restricted to lung tissue. The first 70 probe sets found were then analyzed as above for lung-specific expression, and 15 genes were identified (Table 3). A comparison to the expression patterns described in the BioGPS database confirmed that the majority was only expressed in lung, most of them at high level. Two genes were not restricted to the lung according to BioGPS, and five genes were also found at lower levels in one other tissue (Table 3).\\nT2.xml\\nT2\\nTABLE\\ntable_caption\\nList of 15 probe sets with highest expression signals in the lung.\\nT2.xml\\nT2\\nTABLE\\ntable\\n<?xml version=\"1.0\" encoding=\"UTF-8\"?>\\n<table frame=\"hsides\" rules=\"groups\"><thead><tr><th align=\"left\">Probe set</th><th align=\"left\">Symbol</th><th align=\"left\">Description</th><th align=\"left\">Location (Chr, Mb)</th><th align=\"left\">Mean Expr</th><th align=\"center\">Tissue-specific expression</th></tr></thead><tbody><tr><td align=\"left\">1428361_x_at</td><td align=\"left\"><italic>Hba-a1</italic></td><td align=\"left\">hemoglobin alpha, adult chain 1</td><td align=\"left\">Chr11: 32.184441</td><td align=\"left\">15,10</td><td align=\"center\">MT</td></tr><tr><td align=\"left\">1418639_at</td><td align=\"left\"><italic>Sftpc</italic></td><td align=\"left\">surfactant associated protein C</td><td align=\"left\">Chr14: 70.920826</td><td align=\"left\">14,92</td><td align=\"center\">LS</td></tr><tr><td align=\"left\">1452543_a_at</td><td align=\"left\"><italic>Scgb1a1</italic></td><td align=\"left\">secretoglobin, family 1A, member 1 (uteroglobin)</td><td align=\"left\">Chr19: 9.158206</td><td align=\"left\">14,78</td><td align=\"center\">LHOT</td></tr><tr><td align=\"left\">1417184_s_at</td><td align=\"left\"><italic>Hbb-b2</italic></td><td align=\"left\">hemoglobin, beta adult minor chain</td><td align=\"left\">Chr7: 110.976103</td><td align=\"left\">14,74</td><td align=\"center\">MT</td></tr><tr><td align=\"left\">1441958_s_at</td><td align=\"left\"><italic>Ager</italic></td><td align=\"left\">advanced glycosylation end product-specific receptor</td><td align=\"left\">Chr17: 34.737745</td><td align=\"left\">14,69</td><td align=\"center\">LS</td></tr><tr><td align=\"left\">AFFX-b-ActinMur/M12481<break/>_3_at</td><td align=\"left\"><italic>Actb</italic></td><td align=\"left\">actin beta, cytoplasmic</td><td align=\"left\">Chr5: 143.665528</td><td align=\"left\">14,67</td><td align=\"center\">MT</td></tr><tr><td align=\"left\">1452757_s_at</td><td align=\"left\"><italic>Hba-a1</italic></td><td align=\"left\">hemoglobin alpha, adult chain 1</td><td align=\"left\">Chr11: 32.196742</td><td align=\"left\">14,66</td><td align=\"center\">MT</td></tr><tr><td align=\"left\">1416642_a_at</td><td align=\"left\"><italic>Tpt1</italic></td><td align=\"left\">tumor protein, translationally-controlled 1</td><td align=\"left\">Chr14: 76.246098</td><td align=\"left\">14,62</td><td align=\"center\">MT</td></tr><tr><td align=\"left\">1418509_at</td><td align=\"left\"><italic>Cbr2</italic></td><td align=\"left\">carbonyl reductase 2</td><td align=\"left\">Chr11: 120.628111</td><td align=\"left\">14,62</td><td align=\"center\">LHOT</td></tr><tr><td align=\"left\">1436996_x_at</td><td align=\"left\"><italic>Lzp-s</italic></td><td align=\"left\">P lysozyme structural and lysozyme</td><td align=\"left\">Chr10: 116.724902</td><td align=\"left\">14,62</td><td align=\"center\">ND</td></tr><tr><td align=\"left\">1416624_a_at</td><td align=\"left\"><italic>Uba52</italic></td><td align=\"left\">ubiquitin A-52 residue ribosomal protein fusion product 1</td><td align=\"left\">Chr8: 73.032191</td><td align=\"left\">14,58</td><td align=\"center\">MT</td></tr><tr><td align=\"left\">1427021_s_at</td><td align=\"left\"><italic>Fth1</italic></td><td align=\"left\">ferritin heavy chain 1</td><td align=\"left\">Chr19: 10.057382</td><td align=\"left\">14,57</td><td align=\"center\">ND</td></tr><tr><td align=\"left\">AFFX-MURINE_B2_at</td><td align=\"left\"><italic>B2</italic></td><td align=\"left\">AFFX-MURINE_B2_at short interspersed nuclear element (SINE) class of repeat (probes target Chr 1 and Chr 2 most heavily)</td><td align=\"left\">N/A</td><td align=\"left\">14,52</td><td align=\"center\">ND</td></tr><tr><td align=\"left\">1415906_at</td><td align=\"left\"><italic>Tmsb4x</italic></td><td align=\"left\">thymosin, beta 4, X chromosome</td><td align=\"left\">ChrX: 163.645132</td><td align=\"left\">14,51</td><td align=\"center\">MT</td></tr><tr><td align=\"left\">1449436_s_at</td><td align=\"left\"><italic>Ubb</italic></td><td align=\"left\">ubiquitin B</td><td align=\"left\">Chr11: 62.366564</td><td align=\"left\">14,50</td><td align=\"center\">MT</td></tr></tbody></table>\\n\\nProbe set\\tSymbol\\tDescription\\tLocation (Chr, Mb)\\tMean Expr\\tTissue-specific expression\\t \\t1428361_x_at\\tHba-a1\\themoglobin alpha, adult chain 1\\tChr11: 32.184441\\t15,10\\tMT\\t \\t1418639_at\\tSftpc\\tsurfactant associated protein C\\tChr14: 70.920826\\t14,92\\tLS\\t \\t1452543_a_at\\tScgb1a1\\tsecretoglobin, family 1A, member 1 (uteroglobin)\\tChr19: 9.158206\\t14,78\\tLHOT\\t \\t1417184_s_at\\tHbb-b2\\themoglobin, beta adult minor chain\\tChr7: 110.976103\\t14,74\\tMT\\t \\t1441958_s_at\\tAger\\tadvanced glycosylation end product-specific receptor\\tChr17: 34.737745\\t14,69\\tLS\\t \\tAFFX-b-ActinMur/M12481_3_at\\tActb\\tactin beta, cytoplasmic\\tChr5: 143.665528\\t14,67\\tMT\\t \\t1452757_s_at\\tHba-a1\\themoglobin alpha, adult chain 1\\tChr11: 32.196742\\t14,66\\tMT\\t \\t1416642_a_at\\tTpt1\\ttumor protein, translationally-controlled 1\\tChr14: 76.246098\\t14,62\\tMT\\t \\t1418509_at\\tCbr2\\tcarbonyl reductase 2\\tChr11: 120.628111\\t14,62\\tLHOT\\t \\t1436996_x_at\\tLzp-s\\tP lysozyme structural and lysozyme\\tChr10: 116.724902\\t14,62\\tND\\t \\t1416624_a_at\\tUba52\\tubiquitin A-52 residue ribosomal protein fusion product 1\\tChr8: 73.032191\\t14,58\\tMT\\t \\t1427021_s_at\\tFth1\\tferritin heavy chain 1\\tChr19: 10.057382\\t14,57\\tND\\t \\tAFFX-MURINE_B2_at\\tB2\\tAFFX-MURINE_B2_at short interspersed nuclear element (SINE) class of repeat (probes target Chr 1 and Chr 2 most heavily)\\tN/A\\t14,52\\tND\\t \\t1415906_at\\tTmsb4x\\tthymosin, beta 4, X chromosome\\tChrX: 163.645132\\t14,51\\tMT\\t \\t1449436_s_at\\tUbb\\tubiquitin B\\tChr11: 62.366564\\t14,50\\tMT\\t \\t\\nT2.xml\\nT2\\nTABLE\\ntable_foot\\nMean Expr: mean expression in lung for BXD strains. LS: lung specific expression, LHOT: highly expressed in lung but also in other tissues, MT: expressed in many tissues or mainly in non-lung tissues, ND: no data for other tissues than lung available.\\nF1\\nFIG\\nfig_caption\\nTissue distribution in 25 other tissues of some of the most highly expressed genes in the lung. The expression levels for (A) Stpc, (B) Ager, (C) Scgb1a1 and (D) Hba-a1 in different tissues are shown. Please note that in this representation the gene is correlated with itself to illustrate only its issue distribution.\\nT3.xml\\nT3\\nTABLE\\ntable_caption\\nList of genes with lung-restricted expression found by tissue correlation analysis with Sftpc.\\nT3.xml\\nT3\\nTABLE\\ntable\\n<?xml version=\"1.0\" encoding=\"UTF-8\"?>\\n<table frame=\"hsides\" rules=\"groups\"><thead><tr><th align=\"left\">Probe set</th><th align=\"left\">Symbol</th><th align=\"left\">Description</th><th align=\"left\">Location (Chr, Mb)</th><th align=\"left\">Mean Expr</th><th align=\"left\">BioGPS expression</th></tr></thead><tbody><tr><td align=\"left\">1418639_at</td><td align=\"left\"><italic>Sftpc</italic></td><td align=\"left\">surfactant associated protein C</td><td align=\"left\">Chr14: 70.920826</td><td align=\"left\">14,92</td><td align=\"left\">high in lung, low in nucleus accumbens</td></tr><tr><td align=\"left\">1437028_at</td><td align=\"left\"><italic>Sftpb</italic></td><td align=\"left\">surfactant associated protein B (nonciliated bronchiolar and alveolar type 2 cell signature)</td><td align=\"left\">Chr6: 72.260763</td><td align=\"left\">13,68</td><td align=\"left\">high in lung only</td></tr><tr><td align=\"left\">1422334_a_at</td><td align=\"left\"><italic>Sftpa1</italic></td><td align=\"left\">surfactant associated protein A1</td><td align=\"left\">Chr14: 41.946994</td><td align=\"left\">14,24</td><td align=\"left\">high in lung only</td></tr><tr><td align=\"left\">1422346_at</td><td align=\"left\"><italic>Nkx2-1 (Titf1)</italic></td><td align=\"left\">thyroid <italic>trans</italic>cription factor 1</td><td align=\"left\">Chr12: 57.634187</td><td align=\"left\">8,07</td><td align=\"left\">lung only</td></tr><tr><td align=\"left\">1417057_a_at</td><td align=\"left\"><italic>Lamp3</italic></td><td align=\"left\">lysosomal-associated membrane protein 3</td><td align=\"left\">Chr16: 19.653875</td><td align=\"left\">12,09</td><td align=\"left\">high in lung, low in ES cells and some cell lines</td></tr><tr><td align=\"left\">1421404_at</td><td align=\"left\"><italic>Cxcl15</italic></td><td align=\"left\">chemokine (C-X-C motif) ligand 15</td><td align=\"left\">Chr5: 91.230349</td><td align=\"left\">13,87</td><td align=\"left\">high in lung only</td></tr><tr><td align=\"left\">1441958_s_at</td><td align=\"left\"><italic>Ager</italic></td><td align=\"left\">advanced glycosylation end product-specific receptor</td><td align=\"left\">Chr17: 34.737745</td><td align=\"left\">14,69</td><td align=\"left\">high in lung only</td></tr><tr><td align=\"left\">1436787_x_at</td><td align=\"left\"><italic>Sec14l3</italic></td><td align=\"left\">SEC14-like protein 3</td><td align=\"left\">Chr11: 3.978573</td><td align=\"left\">13,21</td><td align=\"left\">only data for human available - not lung specific</td></tr><tr><td align=\"left\">1425218_a_at</td><td align=\"left\"><italic>Scgb3a2</italic></td><td align=\"left\">secretoglobin, family 3A, member 2</td><td align=\"left\">Chr18: 43.924081</td><td align=\"left\">14,17</td><td align=\"left\">high in lung only</td></tr><tr><td align=\"left\">1449428_at</td><td align=\"left\"><italic>Cldn18</italic></td><td align=\"left\">claudin 18</td><td align=\"left\">Chr9: 99.591247</td><td align=\"left\">12,70</td><td align=\"left\">highest in lung, lower in stomach</td></tr><tr><td align=\"left\">1449525_at</td><td align=\"left\"><italic>Fmo3</italic></td><td align=\"left\">flavin containing monooxygenase 3</td><td align=\"left\">Chr1: 164.884088</td><td align=\"left\">10,90</td><td align=\"left\">high in lung, maybe weak in some other tissues</td></tr><tr><td align=\"left\">1425814_a_at</td><td align=\"left\"><italic>Calcrl</italic></td><td align=\"left\">calcitonin receptor-like</td><td align=\"left\">Chr2: 84.170818</td><td align=\"left\">12,91</td><td align=\"left\">high in lung, weak in macrophages</td></tr><tr><td align=\"left\">1421373_at</td><td align=\"left\"><italic>Cox4i2</italic></td><td align=\"left\">cytochrome c oxidase subunit IV isoform 2</td><td align=\"left\">Chr2: 152.582819</td><td align=\"left\">9,24</td><td align=\"left\">not specific for lung</td></tr><tr><td align=\"left\">1419699_at</td><td align=\"left\"><italic>Scgb3a1</italic></td><td align=\"left\">secretoglobin, family 3A, member 1</td><td align=\"left\">Chr11: 49.477871</td><td align=\"left\">13,68</td><td align=\"left\">high in lung only</td></tr><tr><td align=\"left\">1451604_a_at</td><td align=\"left\"><italic>Acvrl1</italic></td><td align=\"left\">activin A receptor, type II-like 1</td><td align=\"left\">Chr15: 100.968668</td><td align=\"left\">11,86</td><td align=\"left\">high in lung only</td></tr><tr><td align=\"left\">1420347_at</td><td align=\"left\"><italic>Plunc</italic></td><td align=\"left\">palate, lung, and nasal epithelium carcinoma associated</td><td align=\"left\">Chr2: 153.973359</td><td align=\"left\">13,42</td><td align=\"left\">high in lung, low in heart</td></tr></tbody></table>\\n\\nProbe set\\tSymbol\\tDescription\\tLocation (Chr, Mb)\\tMean Expr\\tBioGPS expression\\t \\t1418639_at\\tSftpc\\tsurfactant associated protein C\\tChr14: 70.920826\\t14,92\\thigh in lung, low in nucleus accumbens\\t \\t1437028_at\\tSftpb\\tsurfactant associated protein B (nonciliated bronchiolar and alveolar type 2 cell signature)\\tChr6: 72.260763\\t13,68\\thigh in lung only\\t \\t1422334_a_at\\tSftpa1\\tsurfactant associated protein A1\\tChr14: 41.946994\\t14,24\\thigh in lung only\\t \\t1422346_at\\tNkx2-1 (Titf1)\\tthyroid transcription factor 1\\tChr12: 57.634187\\t8,07\\tlung only\\t \\t1417057_a_at\\tLamp3\\tlysosomal-associated membrane protein 3\\tChr16: 19.653875\\t12,09\\thigh in lung, low in ES cells and some cell lines\\t \\t1421404_at\\tCxcl15\\tchemokine (C-X-C motif) ligand 15\\tChr5: 91.230349\\t13,87\\thigh in lung only\\t \\t1441958_s_at\\tAger\\tadvanced glycosylation end product-specific receptor\\tChr17: 34.737745\\t14,69\\thigh in lung only\\t \\t1436787_x_at\\tSec14l3\\tSEC14-like protein 3\\tChr11: 3.978573\\t13,21\\tonly data for human available - not lung specific\\t \\t1425218_a_at\\tScgb3a2\\tsecretoglobin, family 3A, member 2\\tChr18: 43.924081\\t14,17\\thigh in lung only\\t \\t1449428_at\\tCldn18\\tclaudin 18\\tChr9: 99.591247\\t12,70\\thighest in lung, lower in stomach\\t \\t1449525_at\\tFmo3\\tflavin containing monooxygenase 3\\tChr1: 164.884088\\t10,90\\thigh in lung, maybe weak in some other tissues\\t \\t1425814_a_at\\tCalcrl\\tcalcitonin receptor-like\\tChr2: 84.170818\\t12,91\\thigh in lung, weak in macrophages\\t \\t1421373_at\\tCox4i2\\tcytochrome c oxidase subunit IV isoform 2\\tChr2: 152.582819\\t9,24\\tnot specific for lung\\t \\t1419699_at\\tScgb3a1\\tsecretoglobin, family 3A, member 1\\tChr11: 49.477871\\t13,68\\thigh in lung only\\t \\t1451604_a_at\\tAcvrl1\\tactivin A receptor, type II-like 1\\tChr15: 100.968668\\t11,86\\thigh in lung only\\t \\t1420347_at\\tPlunc\\tpalate, lung, and nasal epithelium carcinoma associated\\tChr2: 153.973359\\t13,42\\thigh in lung, low in heart\\t \\t\\nT3.xml\\nT3\\nTABLE\\ntable_foot\\nMean Expr: mean expression in lung for BXD strains. BioGPS: evaluation of expression pattern as described in BioGPS.\\nRESULTS\\ntitle_2\\nIdentification of gene networks using correlations\\nRESULTS\\nparagraph\\nThe large data set for expression values for ~39,000 transcripts in 57 mouse strains allowed us to calculate correlations between any pair of genes. A Spearman rank correlation analysis identified 12,985 pairs of genes with a correlation value above 0.8, and 604 pairs showed a correlation value of 0.9 or higher. For example, the expression of Klra3 (killer cell lectin-like receptor subfamilily A, member 3) was strongly correlated with the expression of Gzma (granzyme A) (Figure 2A). Klra3 also appeared to be strongly correlated with Il18rap (interleukin 18 receptor accessory protein, Figure 2B). We then calculated the first principal component of the Klra3, Gzma, Il18rap and Klrg1 (killer cell lectin-like receptor subfamily G, member 1) genes and used it to determine the correlations with all other genes in the lung data set. In this way, we could identify a network of nine genes exhibiting a correlation of >= 0.8 with this principal component (Figure 2C). One of the newly identified genes was Prf1 (perforin 1) which was correlated with a p-value of < 10-16 with the principal component (Figure 2D). If genes exhibit a strong correlation of their expression values, one may hypothesize that they are involved in the same biological process or pathway, or they may be expressed in the same cell type.\\nF2\\nFIG\\nfig_caption\\nExpression signals for strongly correlated genes in the lung. Numbers indicate BXD strains, and the parental C57BL/6J and DBA/2J strains as well as F1 individuals are presented. Expression signals of (A) Klra3 and Gzma (p < 10-16) and (B) Klra3 versus Il18rap (p < 10-11) were strongly correlated. (C) List of nine genes highly correlated with the first principal component of the expression of Gzma, Klrg1, Klra3 and Il18rap. (D) Strong correlation between the first principal component and Prf1 (p < 10-16). X- and Y-axis of the plots show the names of genes used for the analysis.\\nRESULTS\\nparagraph\\nIn a similar way, we identified another gene network of 20 genes that exhibited very high correlations of their expression levels across all mouse strains. All possible pairs of genes in this network showed a correlation above 0.95 (Figure 3). The network contained two keratin genes, Krt4 (keratin 4) and Krt13 (keratin 13) and genes involved in cytoskeleton functions, again pointing to a possible interaction of these genes in the same pathway or biological process. Further gene networks found by correlation studies were related to B and T cells (see below).\\nF3\\nFIG\\nfig_caption\\nThe cytokeratin network. Pearson correlations (listed below the diagonal) showed very high correlations between all pairs of the 20 selected genes. Spearman Rank correlations are shown above the diagonal.\\nRESULTS\\ntitle_2\\nCorrelation analysis identified gene expression signatures for T and B cells\\nRESULTS\\nparagraph\\nThe hemoglobin genes Hba-a1 (hemoglobin alpha, adult chain 1) and Hbb-b2 (hemoglobin beta, adult minor chain) were among the top 10 genes with highest expression values in our lung data set. The high levels of hemoglobin transcripts suggested that circulating blood cells, including immune cells, may also be analyzed in our data set. Therefore, we investigated the gene expression networks of known immune cell markers, e.g. Cd3 genes as specific markers for T cells. We calculated the correlations of Cd3d (Cd3 antigen, delta polypeptide) expression levels over all BXD lines with all other genes. This analysis revealed 20 genes with a very highly correlated expression value (p-value below 10-14, Figure 4). Most of these genes were known T cell markers or involved in T cell regulation. Eight out of the 12 genes with the strongest correlations were also exclusively expressed in T cells according to the BioGPS database (Wu et al., 2009): Cd3d, Itk, Tcrb-13V Cd3e, Cd3g, Scap1, Cd6 and Cd5 (see Figure 4 for full gene names). Similarly, we searched for B cell-specific signatures starting with the B cell marker gene Cd19 (CD19 antigen). The probe set \"1450570_a_at\" detected Cd19 mRNA levels and showed a mean expression level of 9.3. We found 14 probe sets with a correlation above 0.80 (p-value < 10-14, Figure 5). A comparison with the BioGPS database revealed that eight of them, Cd19, Cd79b, Faim3, Cd79a, Blk, B3gnt5, Cd22 and Blr1 (see Figure 5 for full gene names) were also exclusively expressed in B cells. Therefore, these genes can be considered as T and B cell signature genes which may be used to follow the presence and infiltration of T and B cells in the lung under normal and pathological conditions.\\nF4\\nFIG\\nfig_caption\\nGene signatures for T-cells. List of the strongest correlates for Cd3d (probe set 1422828_at), all correlated at p < 10-13.\\nF5\\nFIG\\nfig_caption\\nGene signatures for B-cells. List of the strongest correlates for Cd19 (probe set 1450570_at), all correlated at p < 10-12.\\nRESULTS\\ntitle_2\\nIdentification of candidate genes regulating phenotypic traits in the lung\\nRESULTS\\nparagraph\\nOnce a QTL for a phenotypic trait has been found, it will be important to identify the underlying quantitative gene (QTG) which is causing the variation. Searching cis-eQTLs in the QTL interval represents one suitable approach. As a prototype for this approach in our lung data set, we examined two traits for which lung phenotypes were studied in the BXD population and which were available in GeneNetwork. Boon et al. described several QTLs for the susceptibility of BXD mice to influenza A infections. We analyzed one significant QTL peak on chromosome 2 and two suggestive peaks on chromosomes 7 and 17. Seven cis-eQTL regulated genes were found in the chromosome 2 QTL interval (Table 4), including the Hc (hemolytic complement) gene which was shown to contribute to influenza susceptibility. The analysis of the QTL region on chromosome 7 revealed 12 cis-regulated genes in the lung, including Trim12 (tripartite motif protein 12) and Trim34 (tripartite motif protein 34) which were also described as potential candidate QTGs by. In the chromosome 17 QTL region, we found 17 cis-eQTL genes, of which Prkcn (protein kinase C, nu), Qpct (glutaminyl-peptide cyclotransferase (glutaminyl cyclase) and Mta3 (metastasis associated 3) were suggested as potential QTGs by. Another lung-specific phenotype in the GeneNetwork database is \"Mycoplasmosis susceptibility, alveolar exudate\" (GeneNetwork ID 10692, and Cartner et al. unpublished). This trait showed a significant QTL on chromosome 10, between 105 and 130 Mb. The analysis of our lung expression data revealed 16 cis-eQTLs in the genomic interval (Table 5). Three of the cis-QTL genes have been associated previously with immune functions and thus represent suitable candidates to regulate this trait: Chst (carbohydrate (keratan sulfate Gal-6) sulfotransferase 1) was found to exhibit a critical role in lymphocyte trafficking during chronic inflammation. The transcription factor Maf (avian musculoaponeurotic fibrosarcoma (v-maf) AS42 oncogene homolog) was shown to play a role in the transcriptional regulation of cytokine expression and immune cell markers, e.g.. Nrp1 (neuropilin 1) has been primarily described as neuronal receptor but appears also to play a role in the primary immune response and formation of the immunological synapse.\\nT4.xml\\nT4\\nTABLE\\ntable_caption\\nCis-eQTLs identified in QTL inteval on chromosome 2 for influenza susceptibility.\\nT4.xml\\nT4\\nTABLE\\ntable\\n<?xml version=\"1.0\" encoding=\"UTF-8\"?>\\n<table frame=\"hsides\" rules=\"groups\"><thead><tr><th align=\"left\">Probe set</th><th align=\"left\">Symbol</th><th align=\"left\">Description</th><th align=\"left\">Location (Chr, Mb)</th><th align=\"left\">Mean Expr</th><th align=\"left\">Max LRS</th></tr></thead><tbody><tr><td align=\"left\">1423602_at</td><td align=\"left\"><italic>Traf1</italic></td><td align=\"left\">Tnf receptor-associated factor 1</td><td align=\"left\">Chr2: 34.798805</td><td align=\"left\">9,28</td><td align=\"left\">21,1</td></tr><tr><td align=\"left\">1419407_at</td><td align=\"left\"><italic>Hc</italic></td><td align=\"left\">hemolytic complement</td><td align=\"left\">Chr2: 34.838908</td><td align=\"left\">12,00</td><td align=\"left\">82,7</td></tr><tr><td align=\"left\">1441635_at</td><td align=\"left\"><italic>Nr6a1</italic></td><td align=\"left\">nuclear receptor subfamily 6, group A, member 1</td><td align=\"left\">Chr2: 38.736451</td><td align=\"left\">7,51</td><td align=\"left\">20,2</td></tr><tr><td align=\"left\">1455743_at</td><td align=\"left\"><italic>Olfml2a</italic></td><td align=\"left\">olfactomedin-like 2A</td><td align=\"left\">Chr2: 38.816929</td><td align=\"left\">8,28</td><td align=\"left\">63,2</td></tr><tr><td align=\"left\">1430379_at</td><td align=\"left\"><italic>Zfhx1b</italic></td><td align=\"left\">zinc finger homeobox 1b</td><td align=\"left\">Chr2: 44.931019</td><td align=\"left\">9,08</td><td align=\"left\">82,4</td></tr><tr><td align=\"left\">1438516_at</td><td align=\"left\"><italic>Rif1</italic></td><td align=\"left\">Rap1 interacting factor 1</td><td align=\"left\">Chr2: 51.975068</td><td align=\"left\">8,03</td><td align=\"left\">38,8</td></tr><tr><td align=\"left\">1444530_at</td><td align=\"left\"><italic>Neb</italic></td><td align=\"left\">nebulin</td><td align=\"left\">Chr2: 51.991339</td><td align=\"left\">8,14</td><td align=\"left\">86,9</td></tr></tbody></table>\\n\\nProbe set\\tSymbol\\tDescription\\tLocation (Chr, Mb)\\tMean Expr\\tMax LRS\\t \\t1423602_at\\tTraf1\\tTnf receptor-associated factor 1\\tChr2: 34.798805\\t9,28\\t21,1\\t \\t1419407_at\\tHc\\themolytic complement\\tChr2: 34.838908\\t12,00\\t82,7\\t \\t1441635_at\\tNr6a1\\tnuclear receptor subfamily 6, group A, member 1\\tChr2: 38.736451\\t7,51\\t20,2\\t \\t1455743_at\\tOlfml2a\\tolfactomedin-like 2A\\tChr2: 38.816929\\t8,28\\t63,2\\t \\t1430379_at\\tZfhx1b\\tzinc finger homeobox 1b\\tChr2: 44.931019\\t9,08\\t82,4\\t \\t1438516_at\\tRif1\\tRap1 interacting factor 1\\tChr2: 51.975068\\t8,03\\t38,8\\t \\t1444530_at\\tNeb\\tnebulin\\tChr2: 51.991339\\t8,14\\t86,9\\t \\t\\nT4.xml\\nT4\\nTABLE\\ntable_foot\\nFor each gene, only the highest LRS is shown. Mean Expr: mean expression in lung for BXD strains.\\nT5.xml\\nT5\\nTABLE\\ntable_caption\\nCis-eQTLs identified in QTL on chromosome 8 for Mycoplasmosis susceptibility trait.\\nT5.xml\\nT5\\nTABLE\\ntable\\n<?xml version=\"1.0\" encoding=\"UTF-8\"?>\\n<table frame=\"hsides\" rules=\"groups\"><thead><tr><th align=\"left\">Probe set</th><th align=\"left\">Symbol</th><th align=\"left\">Description</th><th align=\"left\">Location (Chr, Mb)</th><th align=\"left\">Mean Expr</th><th align=\"left\">Max LRS</th></tr></thead><tbody><tr><td align=\"left\">1435883_at</td><td align=\"left\"><italic>AW413431</italic></td><td align=\"left\">expressed sequence AW413431</td><td align=\"left\">Chr8: 109.374192</td><td align=\"left\">8,27</td><td align=\"left\">37</td></tr><tr><td align=\"left\">1436986_at</td><td align=\"left\"><italic>Sntb2</italic></td><td align=\"left\">syntrophin, basic 2</td><td align=\"left\">Chr8: 109.537595</td><td align=\"left\">6,94</td><td align=\"left\">23,6</td></tr><tr><td align=\"left\">1437003_at</td><td align=\"left\"><italic>5730419I09Rik</italic></td><td align=\"left\">RIKEN cDNA 5730419I09 gene</td><td align=\"left\">Chr8: 109.543026</td><td align=\"left\">9,79</td><td align=\"left\">24,6</td></tr><tr><td align=\"left\">1451052_at</td><td align=\"left\"><italic>Cog8</italic></td><td align=\"left\">component of oligomeric golgi complex 8</td><td align=\"left\">Chr8: 109.570082</td><td align=\"left\">10,35</td><td align=\"left\">23,7</td></tr><tr><td align=\"left\">1417766_at</td><td align=\"left\"><italic>1810044O22Rik</italic></td><td align=\"left\">RIKEN cDNA 1810044O22 gene</td><td align=\"left\">Chr8: 109.710789</td><td align=\"left\">11,80</td><td align=\"left\">38</td></tr><tr><td align=\"left\">1429725_at</td><td align=\"left\"><italic>Atbf1</italic></td><td align=\"left\">AT motif binding factor 1</td><td align=\"left\">Chr8: 111.481987</td><td align=\"left\">8,84</td><td align=\"left\">70,1</td></tr><tr><td align=\"left\">1453393_a_at</td><td align=\"left\"><italic>Chst4</italic></td><td align=\"left\">carbohydrate (chondroitin 6/keratan) sulfo<italic>trans</italic>ferase 4</td><td align=\"left\">Chr8: 112.553165</td><td align=\"left\">7,33</td><td align=\"left\">71,9</td></tr><tr><td align=\"left\">1427513_at</td><td align=\"left\"><italic>Nudt7</italic></td><td align=\"left\">nudix (nucleoside diphosphate linked moiety X)-type motif 7</td><td align=\"left\">Chr8: 116.678269</td><td align=\"left\">6,95</td><td align=\"left\">22,7</td></tr><tr><td align=\"left\">1446412_at</td><td align=\"left\"><italic>Wwox</italic></td><td align=\"left\">WW domain-containing oxidoreductase</td><td align=\"left\">Chr8: 117.339587</td><td align=\"left\">7,46</td><td align=\"left\">94,7</td></tr><tr><td align=\"left\">1444073_at</td><td align=\"left\"><italic>Maf</italic></td><td align=\"left\">avian musculoaponeurotic fibrosarcoma (v-maf) AS42 oncogene homolog</td><td align=\"left\">Chr8: 118.225461</td><td align=\"left\">7,93</td><td align=\"left\">121</td></tr><tr><td align=\"left\">1449964_a_at</td><td align=\"left\"><italic>Mlycd</italic></td><td align=\"left\">malonyl-CoA decarboxylase (test Mendelian in BXDs with high DBA/2J allele)</td><td align=\"left\">Chr8: 121.934407</td><td align=\"left\">9,63</td><td align=\"left\">34,8</td></tr><tr><td align=\"left\">1418856_a_at</td><td align=\"left\"><italic>Fanca</italic></td><td align=\"left\">Fanconi anemia, complementation group A</td><td align=\"left\">Chr8: 125.792224</td><td align=\"left\">7,98</td><td align=\"left\">78,5</td></tr><tr><td align=\"left\">1460109_at</td><td align=\"left\"><italic>D8Ertd325e</italic></td><td align=\"left\">DNA segment, Chr 8, ERATO Doi 325, expressed</td><td align=\"left\">Chr8: 125.915951</td><td align=\"left\">7,60</td><td align=\"left\">89,8</td></tr><tr><td align=\"left\">1449307_at</td><td align=\"left\"><italic>Dbndd1</italic></td><td align=\"left\">dysbindin (dystrobrevin binding protein 1)</td><td align=\"left\">Chr8: 126.029666</td><td align=\"left\">7,14</td><td align=\"left\">24,1</td></tr><tr><td align=\"left\">1446982_at</td><td align=\"left\"><italic>Pard3</italic></td><td align=\"left\">par-3 (partitioning defective 3) homolog (C. elegans)</td><td align=\"left\">Chr8: 130.036847</td><td align=\"left\">8,02</td><td align=\"left\">87,9</td></tr><tr><td align=\"left\">1448944_at</td><td align=\"left\"><italic>Nrp1</italic></td><td align=\"left\">neuropilin 1</td><td align=\"left\">Chr8: 131.027919</td><td align=\"left\">11,95</td><td align=\"left\">42,4</td></tr></tbody></table>\\n\\nProbe set\\tSymbol\\tDescription\\tLocation (Chr, Mb)\\tMean Expr\\tMax LRS\\t \\t1435883_at\\tAW413431\\texpressed sequence AW413431\\tChr8: 109.374192\\t8,27\\t37\\t \\t1436986_at\\tSntb2\\tsyntrophin, basic 2\\tChr8: 109.537595\\t6,94\\t23,6\\t \\t1437003_at\\t5730419I09Rik\\tRIKEN cDNA 5730419I09 gene\\tChr8: 109.543026\\t9,79\\t24,6\\t \\t1451052_at\\tCog8\\tcomponent of oligomeric golgi complex 8\\tChr8: 109.570082\\t10,35\\t23,7\\t \\t1417766_at\\t1810044O22Rik\\tRIKEN cDNA 1810044O22 gene\\tChr8: 109.710789\\t11,80\\t38\\t \\t1429725_at\\tAtbf1\\tAT motif binding factor 1\\tChr8: 111.481987\\t8,84\\t70,1\\t \\t1453393_a_at\\tChst4\\tcarbohydrate (chondroitin 6/keratan) sulfotransferase 4\\tChr8: 112.553165\\t7,33\\t71,9\\t \\t1427513_at\\tNudt7\\tnudix (nucleoside diphosphate linked moiety X)-type motif 7\\tChr8: 116.678269\\t6,95\\t22,7\\t \\t1446412_at\\tWwox\\tWW domain-containing oxidoreductase\\tChr8: 117.339587\\t7,46\\t94,7\\t \\t1444073_at\\tMaf\\tavian musculoaponeurotic fibrosarcoma (v-maf) AS42 oncogene homolog\\tChr8: 118.225461\\t7,93\\t121\\t \\t1449964_a_at\\tMlycd\\tmalonyl-CoA decarboxylase (test Mendelian in BXDs with high DBA/2J allele)\\tChr8: 121.934407\\t9,63\\t34,8\\t \\t1418856_a_at\\tFanca\\tFanconi anemia, complementation group A\\tChr8: 125.792224\\t7,98\\t78,5\\t \\t1460109_at\\tD8Ertd325e\\tDNA segment, Chr 8, ERATO Doi 325, expressed\\tChr8: 125.915951\\t7,60\\t89,8\\t \\t1449307_at\\tDbndd1\\tdysbindin (dystrobrevin binding protein 1)\\tChr8: 126.029666\\t7,14\\t24,1\\t \\t1446982_at\\tPard3\\tpar-3 (partitioning defective 3) homolog (C. elegans)\\tChr8: 130.036847\\t8,02\\t87,9\\t \\t1448944_at\\tNrp1\\tneuropilin 1\\tChr8: 131.027919\\t11,95\\t42,4\\t \\t\\nT5.xml\\nT5\\nTABLE\\ntable_foot\\nFor each gene, only the highest LRS is shown. Mean Expr: mean expression in lung for BXD strains.\\nRESULTS\\ntitle_2\\nCis- and trans-eQTLs\\nRESULTS\\nparagraph\\nWe then performed a search for eQTLs on a global level, for all probe sets. In this analysis 5,214 cis- and 15,485 trans-regulated genes were identified at an LRS threshold of 12 (Table 6 and Figure 6). When the LRS threshold was increased to 50, 1,332 cis-regulated genes were found, whereas the number of trans-regulated genes was reduced to 15. This observation indicates that many of the trans-eQTL showed a much lower significance value than the cis-eQTL. Next, we present examples for one cis- and one trans-eQTL. A strong eQTL was detected on chromosome 14, at 52 megabases (Mb; Figure 7B) regulating the expression levels of Ang (angiogenin, ribonuclease, RNase A family, 5) (Figure 7A). Since Ang is located at the same position as the eQTL (51.7 Mb on chromosome 14) it represents a cis-eQTL. Furthermore, a strong eQTL was found on chromosome 12 regulating the expression levels of the Cyp1a1 gene (cytochrome P450, family 1, subfamily a, polypeptide 1) (Figure 7C,D). Cyp1a1 is located on chromosome 9 and the corresponding eQTL was found on chromosome 12 (trans-eQTL). The eQTL significance interval contained nine genes, four of which were expressed in lung at a level above 10. Ahr (aryl-hydrocarbon receptor) was one of the four genes and was at the top of the QTL peak (Figure 8). It is the most likely candidate for Cyp1a1 regulation. In conclusion, our data set contained a large number of genes whose expression levels are likely to be influenced by allelic variations in the genomes of C57BL/6J and DBA/2J. Therefore, the presence of pairs of regulated genes and their corresponding eQTLs predicts possible regulatory interactions and will allow searching for yet unknown regulatory networks.\\nT6.xml\\nT6\\nTABLE\\ntable_caption\\nAmount of cis- and trans-regulated transcripts for different significance thresholds\\nT6.xml\\nT6\\nTABLE\\ntable\\n<?xml version=\"1.0\" encoding=\"UTF-8\"?>\\n<table frame=\"hsides\" rules=\"groups\"><thead><tr><th align=\"center\">Threshold (LRS)</th><th align=\"center\">No. of <italic>cis </italic>eQTLs</th><th align=\"right\">No. of <italic>trans </italic>eQTLs</th></tr></thead><tbody><tr><td align=\"center\">12</td><td align=\"center\">5,214</td><td align=\"right\">15,485</td></tr><tr><td align=\"center\">16</td><td align=\"center\">4,391</td><td align=\"right\">3,149</td></tr><tr><td align=\"center\">20</td><td align=\"center\">3,666</td><td align=\"right\">536</td></tr><tr><td align=\"center\">30</td><td align=\"center\">2,500</td><td align=\"right\">48</td></tr><tr><td align=\"center\">50</td><td align=\"center\">1,332</td><td align=\"right\">15</td></tr></tbody></table>\\n\\nF6\\nFIG\\nfig_caption\\nGenome-wide graph of cis- and trans-eQTLs. The positions of the eQTLs are plotted against the locations of the corresponding transcript along the genome. Cis-regulated genes are located at the diagonal, all other dots represent trans-regulated genes. The significance level of each QTL is indicated by the color.\\nF7\\nFIG\\nfig_caption\\nExamples for variations of expression levels in different BXD and inbred mouse strains and ge-nome-wide analysis of cis- and trans-eQTLs. (A) Log2 expression levels for Ang1 and (B) corresponding cis-eQTL signal. The numbers at the top are chromosomes. The blue line represents the significance level of the QTL expressed as LRS score (likelihood ratio statistic). A positive additive coefficient (green line) indicates that DBA/2J alleles increased trait values. A negative additive coefficient (red line) indicates that C57BL/6J alleles increased trait values. The two horizontal lines mark the genome-wide significance levels at p < 0.05 (red line) and p < 0.63 (gray line). Ang1 is located on Chr 14 (triangle) and the QTL peak is at the same location as the gene. (C) Log2 expression levels for Cyp1a1 and (D) corresponding trans eQTL peak. Cyp1a1 is on chromosome 9 (triangle) and the QTL was found on chromosome 12.\\nF8\\nFIG\\nfig_caption\\nThe transcription factor Ahr is located within the trans-eQTL region for Cyp1a1. The strongest eQTL for Cyp1a1 maps to Chromosome 12 and the QTL peaks between 36 and 37 megabases. The gene Ahr, indicated by an arrow, is located exactly at the top of the QTL peak.\\nDISCUSS\\ntitle_1\\nDiscussion\\nDISCUSS\\nparagraph\\nHere, we performed global gene expression profiling in eight inbred mouse strains and a cohort of BXD recombinant inbred strains from whole lung tissues. Our studies identified several lung-specific genes, large variations in gene expression levels, and a strong heritability in many gene expression traits. Correlation analysis of gene expression and genotypes identified potential gene interaction networks, pairs of trans- and cis-eQTLs, and genes with cis-eQTLs that may represent candidate genes involved in susceptibility to respiratory infections. In addition, one specific gene interaction pathway was identified in which Ahr regulates the Cyp1a1 gene.\\nDISCUSS\\nparagraph\\nUsing tissue correlations of gene expression patterns across the BXD strains, we identified 16 genes with a highly restricted expression in the lung of which 14 could be validated by comparison to the BioGPS database. The second most strongly expressed gene in the lung tissues was Sftpc which has been shown to play a role in lung development and the prevention of pneumonitis and emphysema. Also, Sftpc deficiency increases the severity of respiratory syncytial virus-induced pulmonary inflammation. Furthermore, Scgb1a1 and Ager were amongst the five most strongly expressed genes. Scgb1a1 is expressed in lung clara cells and its deficiency results in enhanced susceptibility to environmental agents. Scgb3a1 (secretoglobin, family 3A, member 1) and Scgb3a2 (secretoglobin, family 3A, member 2) were shown by others to be highly expressed in the lung and lower levels in other organs. Scgb3a2 is down-regulated in inflamed airways and plays an important role in lung development. Sftpb (surfactant associated protein B (non-ciliated bronchiolar and alveolar type 2 cell signature) is a hydrophobic peptide which enhances the surface properties of pulmonary surfactant and is expressed in non-ciliated bronchiolar and aleveolar type 2 cells. Maintenance of Sftpb expression is critical for survival during acute lung injury and reduction of alveolar expression causes surfactant dysfunction and respiratory failure. Plunc (palate, lung, and nasal epithelium carcinoma associated) is expressed in the oral, lingual, pharyngeal and respiratory epithelia and members of the Plunc gene family are thought to pay a role in the innate immune response. The presence of Plunc protein in the lung decreases the levels of Mycoplasma pneumoniae and its levels are reduced in allergic inflammatory conditions. Thus, the lung data set allowed us to find important genes that are expressed primarily in the lung and are important for lung homeostasis and prevention of disease.\\nDISCUSS\\nparagraph\\nIt should be noted that our analysis of genes with \"restricted expression to the lung\" is not ex-clusive; it only refers to the tissues that are represented in GeneNetwork and BioGPS. Also, the analysis performed here should not be considered to be comprehensive. More sophisticated approaches may be employed to identify additional genes which also fulfill the criterion of \"lung-restricted\" expression.\\nDISCUSS\\nparagraph\\nFurthermore, genes may not be apparent in the lung transcriptome because they are expressed only in a small fraction of cells within the lung. This issue of dilution of expression signals is an important one and we have studied it in several tissues with considerable care (eye, retina, and numerous brain regions) using the same genetic methods and the same array platform. We were consistently able to detect expression of genes that are only expressed in very small cell subpopulations (<0.1%) such as rare amacrine cell subclasses in the retina or very rare oxytocin-expressing neurons (<2000) in whole brain samples. The reason for the increased sensitivity is that with such large sample sizes (~70 lung arrays) the signal-to-noise ratios are much better than standard studies using Affymetrix arrays. These stuides typically use far fewer arrays and do not use genetic methods to \"validate\" the source of signal.\\nDISCUSS\\nparagraph\\nThe strong signal for hemoglobin and lymphocyte-specific genes clearly showed that gene ex-pression patterns of circulating blood cells are readily detectable in the lung transcriptome. This raises the question if an organ should be studied with or without containing blood. The correct answer to this question depends of course on the particular circumstances. However, we feel strongly that a global systems and genetic approach requires the analysis of the entire organ. The expression of genes is not cell-autonomous and depends on cellular micro envi-ronment, physical factors (gas pressure and gradients, etc), pathogen exposure, and many types of interactions. These factors also influence the expression of genes in blood cells. Therefore, we think that it is imperative to look simultaneously at all cells in a function unit: in this case the whole lung plus its containing blood.\\nDISCUSS\\nparagraph\\nIn conclusion, the combined analysis of expression levels and correlations in a variety of tissues tissue allowed us to determine genes with restricted or preferential expression in the lung. For several of these genes, an important function in the lung has been described and the same may be assumed for the others. This information will also contribute to a better understanding of the biological function of these genes.\\nDISCUSS\\nparagraph\\nMany phenotypic traits have been studied for the BXD mouse populations and several QTLs were identified which influence diseases or vulnerability in the lung. The detection of cis eQTLs in the very same tissue is one method to identify potential candidate genes under the QTL which may causally influence the trait. Here, we investigated two traits in more detail, susceptibility to influenza virus and susceptibility to mycoplasmosis. Several cis-eQTLs were found in the corresponding QTL regions and in each case, genes could be identified with a presumed role in the host immune defense (discussed already in the results section). Thus, the study of cis-eQTLs in our data set may provide valuable candidates for other quantitative trait genes that influence important lung phenotypes. Furthermore, we found 13 BXD lines with low expression signals for Krt4, Krt13 and Krtdap. Krt4 and Krt13 have been shown to be responsible for White sponge nevus (WSN), also known as Cannon\\'s disease, which is an autosomal dominant skin condition in humans. We propose that the 13 mouse strains have genetic alterations which result in low transcript levels of these genes and they may represent a good model for Cannon\\'s disease. It should be noted, however, that no cis-eQTLs found were found for any of the Krt genes.\\nDISCUSS\\nparagraph\\nWe also identified a set of genes for which the expression levels correlated highly with members of the Klr gene family. Klra3 and Klrg1 are killer cell lectin-like receptors that are exclusively expressed on natural killer cells (NK cells). NK cells form a major component of the innate immune system and kill cells by releasing small cytoplasmic granules of proteins called perforins and granzymes. Both Gzma and Prf1 were in the gene network that we identified. In addition, correlations can also be used to expand already known gene networks in specific cell populations. When starting with the Cd3 T cell marker and calculating correlations with all other transcripts measured, we identified a strongly correlated network of genes, in which most of the genes were known as T cell markers or to be involved in T cell activation or homeostasis. In a similar way, when starting with the Cd19 B cell marker, we could identify a strongly correlated network of B cell signature genes. The analysis of these T cell and B cell expression signatures in the Bi-oGPS data base with expression profiles in mouse tissues revealed that indeed >90% of the T and B cell markers were specifically expressed in either T or B cells. Furthermore, most of the T and B cell signature genes represented genes with known function in B and T cell differentiation, activation and homeostasis. For example, the T cell signature included genes encoding subunits of the T cell receptor: Cd3d (CD3 antigen, delta po-lypeptide), Cd3g (CD3 antigen, gamma polypeptide), Tcra (T-cell receptor alpha chain) and Tcrb-V13 (T-cell receptor beta, variable 13) and Lat (linker for activation of T cells) which are involved in T cell activation. The B cell signature contained components of the B cell antigen receptor complex, Cd19 (CD19 antigen) and Cd79a (CD79A antigen (immunoglobulin-associated alpha)), as well as Blk (B lymphoid kinase) tyrosine kinase which is associated with the receptors. Also, the correlations for both signatures in the spleen expression data set in GeneNetwork could indeed confirm that the signatures were strongly correlated (data not shown). In summary, these studies demonstrate that correlation analyses are able to identify genes which very likely interact in a common network or biological process. The approach used here may thus have a great potential to identify new networks and biological processes in the lung. In addition, starting with a known bona-fide cell-specific gene and then analyzing gene expression values across strains, it is possible to identify a set of highly correlated genes. These gene sets genes can now be used as cell-specific signature genes in complex transcriptome studies, e.g. to detect infiltrating immune cells in the lungs after infection.\\nDISCUSS\\nparagraph\\nThe genetic mapping of lung expression profiles revealed many cis- and trans-eQTLs, indicating that many gene expression patterns in lung have a strong genetic component. Trans-eQTLs allow the identification of gene-gene regulatory networks. As an example, we found that the transcription factor Ahr was present in a trans-eQTL region detected for the Cyp1a1 gene. Ahr is a transcription factor known to induce Cyp1a1 transcription levels after ligand binding. Six binding sites for the Ahr receptor ligand have been revealed in the 700-basepair DNA domain upstream of Cyp1a1 . However, a critical leucine-to-proline substitution in Ahr results in a 15 to 20-fold reduction in the binding affinity of the proline variant found in DBA/2J compared to the leucine variant found in C57BL/6J. Indeed, in our data set, expression values for Cyp1a1were low for BXD strains carrying the DBA/2J allele at the Ahr locus and high for the strains carrying the C57BL/6J allele. Since Ahr is not cis-regulated in lung, the downstream effects appear to be only caused by changes in Ahr protein binding affinity. Although the interaction between Cyp1a1 and Ahr as such is not a new finding, it is quite remarkable that the interaction becomes apparent in lungs which were not exposed to an inducing xenobiotic. Furthermore, we do not see this relationship in several other tissues, such as liver. Therefore, our observation suggests that in the lung, which is potentially exposed to many xenobiotics, the Ahr receptor may always be activated at a low level. Alternatively, Ahr expression may be stimulated by yet unknown ligands that are also present under normal environmental conditions.\\nCONCL\\ntitle_1\\nConclusions\\nCONCL\\nparagraph\\nHere, we showed that whole genome expression analysis of the lungs from a large set of strains of the BXD mouse population can be exploited to identify important gene regulatory networks. We found a large number of expression correlations and QTLs which can be further investigated to better understand molecular interaction networks in the lung. The search for cis-eQTLs in genomic intervals that were identified previously as QTLs for infectious diseases revealed several quantitative trait candidate genes. In addition, we demonstrated that the analysis of gene expression correlations, starting with a few cell-specific genes, could identify a larger set of genes which allows detecting the presence of B and T cells within the transcriptome of the whole lung. Such expression signatures will be very important to follow normal and abnormal host responses during infections and other diseases of the lung.\\nCOMP_INT\\ntitle_1\\nCompeting interests\\nCOMP_INT\\nparagraph\\nThe authors declare that they have no competing interests.\\nAUTH_CONT\\ntitle_1\\nAuthors\\' contributions\\nAUTH_CONT\\nparagraph\\nRA performed the bioinformatics analysis and wrote the manuscript. KS designed the experiments, performed the bioinformatics analysis and wrote the manuscript. LL and RWW prepared the study material and supervised the expression array studies. RW contributed to writing of the manuscript. All authors have read and approved the final manuscript.\\nACK_FUND\\ntitle_1\\nAcknowledgements\\nACK_FUND\\nparagraph\\nThis work was supported by intra-mural grants from the Helmholtz-Association (Program \"Infection and Immunity\") and a research grant \"FluResearchNet\" (No. 01KI07137) from the German Ministry of Education and Research to KS. RWW acknowledges the support of the UTHSC Center for Integrative and Translational Genomics and NIH grant P20DA21131 and U01AA13499, and LL was supported by the NIH grant U01AA014425. We thank Dr. Yan Jiao and Weikuan Gu at UTHSC and the VA Medical Center, Memphis, for running the Affymetrix arrays in their core facility. We also thank the GeneNetwork development team, including Arthur Centeno, Xiaodong Zhou, Ning Liu, Zachary Sloan, and Lei Yan, for their help in integrating and error-checking the lung transcriptome data. We also thank Samuel C. Cartner for making the data on the Mycoplasmosis susceptibility phenotype available to us prior to publication.\\nsurname:Schughart;given-names:K\\nREF\\nMamm Genome\\nref\\nSYSGENET: a meeting report from a new European network for systems genetics\\nsurname:Morahan;given-names:G\\nsurname:Peeva;given-names:V\\nsurname:Mehta;given-names:M\\nsurname:Williams;given-names:R\\nREF\\nJ Autoimmun\\nref\\nSystems genetics can provide new insights in to immune regulation and autoimmunity\\nsurname:Threadgill;given-names:DW\\nsurname:Hunter;given-names:KW\\nsurname:Williams;given-names:RW\\nREF\\nMamm Genome\\nref\\nsurname:Alberts;given-names:R\\nsurname:Fu;given-names:J\\nsurname:Swertz;given-names:MA\\nsurname:Lubbers;given-names:LA\\nsurname:Albers;given-names:CJ\\nsurname:Jansen;given-names:RC\\nREF\\nBrief Bioinform\\nref\\nCombining microarrays and genetic analysis\\nsurname:Chesler;given-names:EJ\\nsurname:Lu;given-names:L\\nsurname:Shou;given-names:S\\nsurname:Qu;given-names:Y\\nsurname:Gu;given-names:J\\nsurname:Wang;given-names:J\\nsurname:Hsu;given-names:HC\\nsurname:Mountz;given-names:JD\\nsurname:Baldwin;given-names:NE\\nsurname:Langston;given-names:MA\\nREF\\nNat Genet\\nref\\nComplex trait analysis of gene expression uncovers polygenic and pleiotropic networks that modulate nervous system function\\nsurname:Jansen;given-names:RC\\nsurname:Nap;given-names:JP\\nREF\\nTrends Genet\\nref\\nGenetical genomics: the added value from segregation\\nsurname:Rosen;given-names:GD\\nsurname:Pung;given-names:CJ\\nsurname:Owens;given-names:CB\\nsurname:Caplow;given-names:J\\nsurname:Kim;given-names:H\\nsurname:Mozhui;given-names:K\\nsurname:Lu;given-names:L\\nsurname:Williams;given-names:RW\\nREF\\nGenes Brain Behav\\nref\\nGenetic modulation of striatal volume by loci on Chrs 6 and 17 in BXD recombinant inbred mice\\nsurname:Geisert;given-names:EE\\nsurname:Lu;given-names:L\\nsurname:Freeman-Anderson;given-names:NE\\nsurname:Templeton;given-names:JP\\nsurname:Nassr;given-names:M\\nsurname:Wang;given-names:X\\nsurname:Gu;given-names:W\\nsurname:Jiao;given-names:Y\\nsurname:Williams;given-names:RW\\nREF\\nMol Vis\\nref\\nGene expression in the mouse eye: an online resource for genetics using 103 strains of mice\\nsurname:Gaglani;given-names:SM\\nsurname:Lu;given-names:L\\nsurname:Williams;given-names:RW\\nsurname:Rosen;given-names:GD\\nREF\\nBMC Neurosci\\nref\\nsurname:Gatti;given-names:D\\nsurname:Maki;given-names:A\\nsurname:Chesler;given-names:EJ\\nsurname:Kirova;given-names:R\\nsurname:Kosyk;given-names:O\\nsurname:Lu;given-names:L\\nsurname:Manly;given-names:KF\\nsurname:Williams;given-names:RW\\nsurname:Perkins;given-names:A\\nsurname:Langston;given-names:MA\\nREF\\nHepatology\\nref\\nGenome-level analysis of genetic regulation of liver gene expression networks\\nsurname:Bystrykh;given-names:L\\nsurname:Weersing;given-names:E\\nsurname:Dontje;given-names:B\\nsurname:Sutton;given-names:S\\nsurname:Pletcher;given-names:MT\\nsurname:Wiltshire;given-names:T\\nsurname:Su;given-names:AI\\nsurname:Vellenga;given-names:E\\nsurname:Wang;given-names:J\\nsurname:Manly;given-names:KF\\nREF\\nNat Genet\\nref\\nUncovering regulatory pathways that affect hematopoietic stem cell function using \\'genetical genomics\\'\\nsurname:Bao;given-names:L\\nsurname:Wei;given-names:L\\nsurname:Peirce;given-names:JL\\nsurname:Homayouni;given-names:R\\nsurname:Li;given-names:H\\nsurname:Zhou;given-names:M\\nsurname:Chen;given-names:H\\nsurname:Lu;given-names:L\\nsurname:Williams;given-names:RW\\nsurname:Pfeffer;given-names:LM\\nREF\\nMamm Genome\\nref\\nCombining gene expression QTL mapping and phenotypic spectrum analysis to uncover gene regulatory relationships\\nsurname:Chesler;given-names:EJ\\nsurname:Lu;given-names:L\\nsurname:Wang;given-names:J\\nsurname:Williams;given-names:RW\\nsurname:Manly;given-names:KF\\nREF\\nNat Neurosci\\nref\\nsurname:Irizarry;given-names:RA\\nsurname:Hobbs;given-names:B\\nsurname:Collin;given-names:F\\nsurname:Beazer-Barclay;given-names:YD\\nsurname:Antonellis;given-names:KJ\\nsurname:Scherf;given-names:U\\nsurname:Speed;given-names:TP\\nREF\\nBiostatistics\\nref\\nExploration, normalization, and summaries of high density oligonucleotide array probe level data\\nsurname:Haley;given-names:CS\\nsurname:Knott;given-names:SA\\nREF\\nHeredity\\nref\\nA simple regression method for mapping quantitative trait loci in line crosses using flanking markers\\nsurname:Shifman;given-names:S\\nsurname:Bell;given-names:JT\\nsurname:Copley;given-names:RR\\nsurname:Taylor;given-names:MS\\nsurname:Williams;given-names:RW\\nsurname:Mott;given-names:R\\nsurname:Flint;given-names:J\\nREF\\nPLoS Biol\\nref\\nA high-resolution single nucleotide polymorphism genetic map of the mouse genome\\nRESEARCH0046\\nsurname:Williams;given-names:RW\\nsurname:Gu;given-names:J\\nsurname:Qi;given-names:S\\nsurname:Lu;given-names:L\\nREF\\nGenome Biol\\nref\\nsurname:Brockdorff;given-names:N\\nsurname:Ashworth;given-names:A\\nsurname:Kay;given-names:GF\\nsurname:Cooper;given-names:P\\nsurname:Smith;given-names:S\\nsurname:McCabe;given-names:VM\\nsurname:Norris;given-names:DP\\nsurname:Penny;given-names:GD\\nsurname:Patel;given-names:D\\nsurname:Rastan;given-names:S\\nREF\\nNature\\nref\\nsurname:Wu;given-names:C\\nsurname:Orozco;given-names:C\\nsurname:Boyer;given-names:J\\nsurname:Leglise;given-names:M\\nsurname:Goodale;given-names:J\\nsurname:Batalov;given-names:S\\nsurname:Hodge;given-names:CL\\nsurname:Haase;given-names:J\\nsurname:Janes;given-names:J\\nsurname:Huss;given-names:JW\\nREF\\nGenome Biol\\nref\\nBioGPS: an extensible and customizable portal for querying and organizing gene annotation resources\\nsurname:Boon;given-names:AC\\nsurname:deBeauchamp;given-names:J\\nsurname:Hollmann;given-names:A\\nsurname:Luke;given-names:J\\nsurname:Kotb;given-names:M\\nsurname:Rowe;given-names:S\\nsurname:Finkelstein;given-names:D\\nsurname:Neale;given-names:G\\nsurname:Lu;given-names:L\\nsurname:Williams;given-names:RW\\nREF\\nJ Virol\\nref\\nsurname:Cartner;given-names:SC\\nsurname:Simecka;given-names:JW\\nsurname:Briles;given-names:DE\\nsurname:Cassell;given-names:GH\\nsurname:Lindsey;given-names:JR\\nREF\\nInfect Immun\\nref\\nResistance to mycoplasmal lung disease in mice is a complex genetic trait\\nsurname:Hiraoka;given-names:N\\nsurname:Kawashima;given-names:H\\nsurname:Petryniak;given-names:B\\nsurname:Nakayama;given-names:J\\nsurname:Mitoma;given-names:J\\nsurname:Marth;given-names:JD\\nsurname:Lowe;given-names:JB\\nsurname:Fukuda;given-names:M\\nREF\\nJ Biol Chem\\nref\\nCore 2 branching beta1,6-N-acetylglucosaminyltransferase and high endothelial venule-restricted sul-fotransferase collaboratively control lymphocyte homing\\nsurname:Cao;given-names:S\\nsurname:Liu;given-names:J\\nsurname:Chesi;given-names:M\\nsurname:Bergsagel;given-names:PL\\nsurname:Ho;given-names:IC\\nsurname:Donnelly;given-names:RP\\nsurname:Ma;given-names:X\\nREF\\nJ Immunol\\nref\\nsurname:Cao;given-names:S\\nsurname:Liu;given-names:J\\nsurname:Song;given-names:L\\nsurname:Ma;given-names:X\\nREF\\nJ Immunol\\nref\\nThe protooncogene c-Maf is an essential transcription factor for IL-10 gene expression in macrophages\\nsurname:Hiramatsu;given-names:Y\\nsurname:Suto;given-names:A\\nsurname:Kashiwakuma;given-names:D\\nsurname:Kanari;given-names:H\\nsurname:Kagami;given-names:S\\nsurname:Ikeda;given-names:K\\nsurname:Hirose;given-names:K\\nsurname:Watanabe;given-names:N\\nsurname:Grusby;given-names:MJ\\nsurname:Iwamoto;given-names:I\\nREF\\nJ Leukoc Biol\\nref\\nc-Maf activates the promoter and enhancer of the IL-21 gene, and TGF-beta inhibits c-Maf-induced IL-21 production in CD4+ T cells\\nsurname:Nakamura;given-names:M\\nsurname:Hamada;given-names:M\\nsurname:Hasegawa;given-names:K\\nsurname:Kusakabe;given-names:M\\nsurname:Suzuki;given-names:H\\nsurname:Greaves;given-names:DR\\nsurname:Moriguchi;given-names:T\\nsurname:Kudo;given-names:T\\nsurname:Takahashi;given-names:S\\nREF\\nGene\\nref\\nc-Maf is essential for the F4/80 expression in macrophages in vivo\\nsurname:Pot;given-names:C\\nsurname:Jin;given-names:H\\nsurname:Awasthi;given-names:A\\nsurname:Liu;given-names:SM\\nsurname:Lai;given-names:CY\\nsurname:Madan;given-names:R\\nsurname:Sharpe;given-names:AH\\nsurname:Karp;given-names:CL\\nsurname:Miaw;given-names:SC\\nsurname:Ho;given-names:IC\\nREF\\nJ Immunol\\nref\\nCutting edge: IL-27 induces the transcription factor c-Maf, cytokine IL-21, and the costimulatory recep-tor ICOS that coordinately act together to promote differentiation of IL-10-producing Tr1 cells\\nsurname:Voice;given-names:J\\nsurname:Donnelly;given-names:S\\nsurname:Dorsam;given-names:G\\nsurname:Dolganov;given-names:G\\nsurname:Paul;given-names:S\\nsurname:Goetzl;given-names:EJ\\nREF\\nJ Immunol\\nref\\nc-Maf and JunB mediation of Th2 differentiation induced by the type 2 G protein-coupled receptor (VPAC2) for vasoactive intestinal peptide\\nsurname:Xu;given-names:J\\nsurname:Yang;given-names:Y\\nsurname:Qiu;given-names:G\\nsurname:Lal;given-names:G\\nsurname:Wu;given-names:Z\\nsurname:Levy;given-names:DE\\nsurname:Ochando;given-names:JC\\nsurname:Bromberg;given-names:JS\\nsurname:Ding;given-names:Y\\nREF\\nJ Immunol\\nref\\nc-Maf regulates IL-10 expression during Th17 polarization\\nsurname:Tordjman;given-names:R\\nsurname:Lepelletier;given-names:Y\\nsurname:Lemarchandel;given-names:V\\nsurname:Cambot;given-names:M\\nsurname:Gaulard;given-names:P\\nsurname:Hermine;given-names:O\\nsurname:Romeo;given-names:PH\\nREF\\nNat Immunol\\nref\\nA neuronal receptor, neuropilin-1, is essential for the initiation of the primary immune response\\nsurname:Wulfing;given-names:C\\nsurname:Rupp;given-names:F\\nREF\\nNat Immunol\\nref\\nNeuropilin-1: another neuronal molecule in the \"immunological synapse\"\\nsurname:Bridges;given-names:JP\\nsurname:Wert;given-names:SE\\nsurname:Nogee;given-names:LM\\nsurname:Weaver;given-names:TE\\nREF\\nJ Biol Chem\\nref\\nExpression of a human surfactant protein C mutation associated with interstitial lung disease disrupts lung development in transgenic mice\\nsurname:Glasser;given-names:SW\\nsurname:Detmer;given-names:EA\\nsurname:Ikegami;given-names:M\\nsurname:Na;given-names:CL\\nsurname:Stahlman;given-names:MT\\nsurname:Whitsett;given-names:JA\\nREF\\nJ Biol Chem\\nref\\nPneumonitis and emphysema in sp-C gene targeted mice\\nsurname:Glasser;given-names:SW\\nsurname:Witt;given-names:TL\\nsurname:Senft;given-names:AP\\nsurname:Baatz;given-names:JE\\nsurname:Folger;given-names:D\\nsurname:Maxfield;given-names:MD\\nsurname:Akinbi;given-names:HT\\nsurname:Newton;given-names:DA\\nsurname:Prows;given-names:DR\\nsurname:Korfhagen;given-names:TR\\nREF\\nAm J Physiol Lung Cell Mol Physiol\\nref\\nSurfactant protein C-deficient mice are susceptible to respiratory syncytial virus infection\\nsurname:Stripp;given-names:BR\\nsurname:Reynolds;given-names:SD\\nsurname:Boe;given-names:IM\\nsurname:Lund;given-names:J\\nsurname:Power;given-names:JH\\nsurname:Coppens;given-names:JT\\nsurname:Wong;given-names:V\\nsurname:Reynolds;given-names:PR\\nsurname:Plopper;given-names:CG\\nREF\\nAm J Respir Cell Mol Biol\\nref\\nClara cell secretory protein deficiency alters clara cell secretory apparatus and the protein composition of airway lining fluid\\nsurname:Porter;given-names:D\\nsurname:Lahti-Domenici;given-names:J\\nsurname:Torres-Arzayus;given-names:M\\nsurname:Chin;given-names:L\\nsurname:Polyak;given-names:K\\nREF\\nMech Dev\\nref\\nExpression of high in normal-1 (HIN-1) and uteroglobin related protein-1 (UGRP-1) in adult and developing tissues\\nsurname:Chiba;given-names:Y\\nsurname:Kusakabe;given-names:T\\nsurname:Kimura;given-names:S\\nREF\\nAm J Physiol Lung Cell Mol Physiol\\nref\\nDecreased expression of uteroglobin-related protein 1 in inflamed mouse airways is mediated by IL-9\\nsurname:Kurotani;given-names:R\\nsurname:Tomita;given-names:T\\nsurname:Yang;given-names:Q\\nsurname:Carlson;given-names:BA\\nsurname:Chen;given-names:C\\nsurname:Kimura;given-names:S\\nREF\\nAm J Respir Crit Care Med\\nref\\nRole of secretoglobin 3A2 in lung development\\nsurname:Bohinski;given-names:RJ\\nsurname:Huffman;given-names:JA\\nsurname:Whitsett;given-names:JA\\nsurname:Lattier;given-names:DL\\nREF\\nJ Biol Chem\\nref\\nCis-active elements controlling lung cell-specific expression of human pulmonary surfactant protein B gene\\nsurname:Bein;given-names:K\\nsurname:Wesselkamper;given-names:SC\\nsurname:Liu;given-names:X\\nsurname:Dietsch;given-names:M\\nsurname:Majumder;given-names:N\\nsurname:Concel;given-names:VJ\\nsurname:Medvedovic;given-names:M\\nsurname:Sartor;given-names:MA\\nsurname:Henning;given-names:LN\\nsurname:Venditto;given-names:C\\nREF\\nAm J Respir Cell Mol Biol\\nref\\nSurfactant-associated protein B is critical to survival in nickel-induced injury in mice\\nsurname:Melton;given-names:KR\\nsurname:Nesslein;given-names:LL\\nsurname:Ikegami;given-names:M\\nsurname:Tichelaar;given-names:JW\\nsurname:Clark;given-names:JC\\nsurname:Whitsett;given-names:JA\\nsurname:Weaver;given-names:TE\\nREF\\nAm J Physiol Lung Cell Mol Physiol\\nref\\nSP-B deficiency causes respiratory failure in adult mice\\nsurname:LeClair;given-names:EE\\nsurname:Nomellini;given-names:V\\nsurname:Bahena;given-names:M\\nsurname:Singleton;given-names:V\\nsurname:Bingle;given-names:L\\nsurname:Craven;given-names:CJ\\nsurname:Bingle;given-names:CD\\nREF\\nGenomics\\nref\\nCloning and expression of a mouse member of the PLUNC protein family exclusively expressed in tongue epithe-lium\\nsurname:Bingle;given-names:CD\\nsurname:Craven;given-names:CJ\\nREF\\nHum Mol Genet\\nref\\nPLUNC: a novel family of candidate host defence proteins expressed in the upper airways and nasopharynx\\nsurname:Chu;given-names:HW\\nsurname:Thaikoottathil;given-names:J\\nsurname:Rino;given-names:JG\\nsurname:Zhang;given-names:G\\nsurname:Wu;given-names:Q\\nsurname:Moss;given-names:T\\nsurname:Refaeli;given-names:Y\\nsurname:Bowler;given-names:R\\nsurname:Wenzel;given-names:SE\\nsurname:Chen;given-names:Z\\nREF\\nJ Immunol\\nref\\nFunction and regulation of SPLUNC1 protein in Mycoplasma infection and allergic inflam-mation\\nsurname:Canon;given-names:AB\\nREF\\nArch Dermatol Syphilol\\nref\\nWhite sponge nevus of the mucosa (nevus spongiosus albus mucosa)\\nsurname:Terrinoni;given-names:A\\nsurname:Rugg;given-names:EL\\nsurname:Lane;given-names:EB\\nsurname:Melino;given-names:G\\nsurname:Felix;given-names:DH\\nsurname:Munro;given-names:CS\\nsurname:McLean;given-names:WH\\nREF\\nJ Dent Res\\nref\\nA novel mutation in the keratin 13 gene causing oral white sponge nevus\\nsurname:Rugg;given-names:EL\\nsurname:McLean;given-names:WH\\nsurname:Allison;given-names:WE\\nsurname:Lunny;given-names:DP\\nsurname:Macleod;given-names:RI\\nsurname:Felix;given-names:DH\\nsurname:Lane;given-names:EB\\nsurname:Munro;given-names:CS\\nREF\\nNat Genet\\nref\\nA mutation in the mucosal keratin K4 is associated with oral white sponge nevus\\nsurname:Trapani;given-names:JA\\nsurname:Smyth;given-names:MJ\\nREF\\nNat Rev Immunol\\nref\\nFunctional significance of the perforin/granzyme cell death pathway\\nsurname:Nebert;given-names:DW\\nsurname:Dalton;given-names:TP\\nsurname:Okey;given-names:AB\\nsurname:Gonzalez;given-names:FJ\\nREF\\nJ Biol Chem\\nref\\nsurname:Okey;given-names:AB\\nsurname:Riddick;given-names:DS\\nsurname:Harper;given-names:PA\\nREF\\nToxicol Lett\\nref\\nsurname:Sutter;given-names:CH\\nsurname:Yin;given-names:H\\nsurname:Li;given-names:Y\\nsurname:Mammen;given-names:JS\\nsurname:Bodreddigari;given-names:S\\nsurname:Stevens;given-names:G\\nsurname:Cole;given-names:JA\\nsurname:Sutter;given-names:TR\\nREF\\nProc Natl Acad Sci USA\\nref\\nEGF receptor signaling blocks aryl hydrocarbon receptor-mediated transcription and cell differentiation in human epidermal keratinocytes\\nsurname:Denison;given-names:MS\\nsurname:Fisher;given-names:JM\\nsurname:Whitlock;given-names:JP;suffix:Jr\\nREF\\nJ Biol Chem\\nref\\nProtein-DNA interactions at recognition sites for the dioxin-Ah receptor complex\\nsurname:Chang;given-names:C\\nsurname:Smith;given-names:DR\\nsurname:Prasad;given-names:VS\\nsurname:Sidman;given-names:CL\\nsurname:Nebert;given-names:DW\\nsurname:Puga;given-names:A\\nREF\\nPharmacogenetics\\nref\\nTen nucleotide differences, five of which cause amino acid changes, are associated with the Ah receptor locus polymorphism of C57BL/6 and DBA/2 mice\\n',\n",
       " '19709424': \"BioC-API\\ncollection.key\\nCC BY\\nGenome Biology\\nThis is an open access article distributed under the terms of the Creative Commons Attribution License (), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.\\nsurname:Reyes;given-names:Raquel\\nsurname:Alcalde;given-names:Jose\\nsurname:Izquierdo;given-names:Jose M\\nTITLE\\nfront\\nDepletion of T-cell intracellular antigen proteins promotes cell proliferation\\nABSTRACT\\nabstract\\nThe transcriptome of TIA-1/TIAR-depleted cells indicates roles in inflammation, cell-cell signaling, immune suppression, angiogenesis, metabolism and cell proliferation.\\nABSTRACT\\nabstract_title_1\\nBackground\\nABSTRACT\\nabstract\\nT-cell intracellular antigen-1 (TIA-1) and TIA-1 related/like protein (TIAR/TIAL1), two DNA/RNA binding proteins broadly expressed in eukaryotic cells, participate in the regulation of gene expression through RNA metabolism. Despite the biological relevance of these regulators, there are no genome-wide studies assessing global transcriptomic and phenotypic impacts after changes in the expression and/or function of these proteins.\\nABSTRACT\\nabstract_title_1\\nResults\\nABSTRACT\\nabstract\\nUsing high-throughput gene expression profiling, we found that the TIA-1/TIAR-depleted cell phenotype is linked to a transcriptome involved in the control of inflammation, cell-cell signaling, immune-suppression, angiogenesis, metabolism and cell proliferation. Induced genes included pro-inflammatory cytokines, inflammatory chemokines, growth-stimulating factors and pro-angiogenic inducers. Repressed genes involved the RAS oncogene family member RAB40B, regulators of cytoskeleton organization and biogenesis and a mitochondrial modulator. Consistent with these observations, depletion of TIA proteins in HeLa cells results in increased cell proliferation, altered cell-cycle and anchorage-independent growth. Mechanistically, the changes associated with the steady-state target mRNA levels regulated by TIA proteins are consistent with overlapping effects on gene basal transcription rate and mRNA turnover.\\nABSTRACT\\nabstract_title_1\\nConclusions\\nABSTRACT\\nabstract\\nCollectively, our findings suggest a role for TIA proteins as cellular sensors that modulate gene expression control at the transcriptional and post-transcriptional levels, coupling cell proliferation responses and metabolic homeostasis to cell survival and growth.\\nINTRO\\ntitle_1\\nBackground\\nINTRO\\nparagraph\\nOne of the most important challenges resulting from our knowledge of the human genome - and, in general, of all genomes - is to understand how the collection of RNAs and proteins that define us as an organism are generated. Nowadays, we accept that the origin of our transcriptomic and proteomic diversity is due not just to the number of genes present in our genome but also to the dynamic and differential regulation of their expression. The characterization of transcriptional and post-transcriptional events leading to the generation of multiple RNAs, proteins and functions from a single or distinct RNA precursors reveals the existence of multiple layers and networks involved in regulating the diverse biological functions controlled by the transcriptome.\\nINTRO\\nparagraph\\nT-cell intracellular antigen (TIA)-1 and TIA-1 related/like factor (TIAR/TIAL1) are two proteins that play important roles in many aspects of RNA biology, from RNA transcription to splicing, stability and translation. Indeed, TIA proteins have been shown to interact with RNA polymerase II, DNA and RNA and to participate in the control of alternative pre-mRNA splicing. TIA proteins bind to uridine-rich sequences, which are mostly located in the introns, and seem to facilitate the recruitment of U1 small nuclear ribonucleoprotein, thus promoting the recognition and processing of atypical 5' splice sites. The cytoplasmic pool of these proteins has been linked to the control of translation and/or stability of some mRNAs through binding to adenine- and uridine-rich sequences located in 3' untranslated regions. Additionally, these proteins are involved in cell responses to metabolic and genotoxic stresses as well as the formation of stress granules. It has also been reported that these proteins play a relevant role in the control of cell death and viral replication. In fact, mice that lack either TIA-1 or TIAR show high rates of embryonic lethality, indicating a relevant role for these proteins during embryonic development. Furthermore, recent studies have identified a set of mRNAs that interact and are regulated by these proteins.\\nINTRO\\nparagraph\\nAltogether, these findings strongly support a key role for TIA-1 and TIAR proteins in the control of different aspects of RNA metabolism and function. However, little is known about how these regulators control the transcriptome and about the specific and/or overlapping functions associated with TIA proteins at this level. Here, we use a loss-of-function approach combined with global mRNA expression pattern and phenotypic profiling analyses to identify targets and pathways regulated by TIA proteins. Our data suggest that these proteins are involved in the transcriptional and post-transcriptional regulation of specific transcripts associated with the control of key cellular functions, such as inflammation, cell proliferation and anchorage-independent growth responses.\\nRESULTS\\ntitle_1\\nResults\\nRESULTS\\ntitle_2\\nGene expression profiling of TIA-1 and TIAR-depleted HeLa cells\\nRESULTS\\nparagraph\\nTo study the role of TIA-1 and TIAR in the global control of gene expression, we transfected HeLa cells with double-stranded small interfering RNAs (siRNAs) targeting TIA-1 and/or TIAR mRNAs or with control siRNA (c) as previously described. The effect of the siRNAs on TIA-1 and TIAR expression was assessed by western blotting (Figure 1a) and by semiquantitative and quantitative RT-PCR analyses (Figure 1b). Under these experimental conditions, 80 to 90% depletion of both TIA-1 and TIAR proteins (Figure 1a) and mRNAs (Figure 1b) was achieved 72 h after transfection, in agreement with previous findings. In contrast, U2AF65 and alpha-tubulin proteins (Figure 1a) and beta-actin mRNA (Figure 1b), used as controls for siRNA specificity, were not significantly affected by the treatment.\\nF1\\nFIG\\nfig_caption\\nRNA interference-mediated depletion of TIA proteins in HeLa cells. (a) Immunoblot analysis of HeLa cell lysates (5 mug (lane 1; C(1/5)) or 25 mug (lanes 2 to 5)) prepared 72 h after transfection with siRNAs against control (C; lanes 1 and 2), TIA-1 (lane 3), TIAR (lane 4), and TIA-1 plus TIAR (lane 5; 1+R). TIA-1(a/b) and TIAR(a/b) refer to the major isoforms of TIA-1 and TIAR, respectively. The blot was probed with antibodies against TIA-1, TIAR, U2AF65 and alpha-tubulin proteins, as indicated. Molecular weight markers and the identities of protein bands are shown. The intensities of the different protein bands from immunoblots were quantified by densitometry. The represented values were normalized and are expressed relative to alpha-tubulin (alpha-Tub), whose value is fixed arbitrarily to 1, and are means +- standard error of the mean (SEM; n = 10; *P < 0.05). (b) Cytoplasmic mRNAs from post-transfected HeLa cells in (a) were analyzed by RT-PCR. Positions of size markers and the predicted alternatively spliced products are indicated. Quantification of relative levels of TIA-1, TIAR and beta-actin mRNAs in the above post-tranfected HeLa cells by real time RT-PCR. The represented values were normalized and are expressed relative to beta-actin mRNA, whose value is fixed arbitrarily to 1, and are means +- SEM (n = 3; *P < 0.01).\\nRESULTS\\nparagraph\\nIn order to define gene expression profiles resulting from the suppression of TIA-1 and TIAR expression, we assessed the differences in global gene expression patterns in HeLa cells transfected with siRNAs targeting both TIA-1 and TIAR compared to HeLa cells transfected with control siRNA. Comparisons were made using hierarchical clustering analysis of gene expression patterns from three array independent experiments performed in three different biological samples for each experimental condition tested (Additional data file 1). As shown in Figure 2a, depletion of TIA-1 and TIAR resulted in a marked alteration in the usual expression patterns of genes. To identify cohorts of genes regulated by TIA proteins, the gene expression array dataset was analyzed using Robust Multichip Average normalization followed by identification of differentially expressed genes using the Rank Products method. In these analyses, only genes showing at least a 1.5-fold difference in expression compared to the control were considered. A total of 472 genes were differentially expressed (P < 0.05), of which 73 and 399 were up- and down-regulated, respectively (Figure 2b; Additional data file 2).\\nF2\\nFIG\\nfig_caption\\nKnockdown of TIA proteins activates an acute inflammation-promoting transcriptome. (a) Heat map representation depicting microarray data for genes that are differentially regulated by siRNA-mediated reduction of TIA-1 and TIAR. Mean fold-change values from three independent replicates for each biological condition tested are given for 538 probe sets (horizontal lanes; out of a total of 47,400 sets) that were detectable above background showing at least a 1.5-fold change in expression and were statistically different (P < 0.05) in the two experimental conditions tested. Red and green indicate up- and down-regulation, respectively, relative to control siRNA-transfected HeLa cells. The color scale to the right indicates the magnitude of the fold change (base 2 logarithm) for a particular transcript. (b) Total number of genes whose expression was up- or down-regulated in TIA-1 and TIAR-depleted cells. (c, d) Graphic pie representations of the distribution of up- (c) and down-regulated (d) genes (P < 0.01) using the GO biological process category. (e, f) Graphic pie representations of the distribution of up- (e) and down-regulated (f) genes (P < 0.05) using the KEGG pathway database. In all cases (c-f), percentages shown reflect the portion of total genes that are associated with the biological functions and pathways indicated.\\nRESULTS\\nparagraph\\nIn an attempt to address the functional relevance of the observed changes in high-density DNA microarrays, Gene Ontology (GO) analyses were performed for the up- and down-regulated genes. GO analysis was able to identify the main categories that had significant representation (P < 0.01) of differentially expressed genes controlled by TIA proteins (Figure 2c, d; Additional data file 3). GO categories related to inflammation and immune responses, cell proliferation, cell-cell signaling, chemotaxis, cell motility, angiogenesis and anagen were among the enriched categories in up-regulated genes (Figure 2c). On the other hand, GO categories associated with apoptosis induction, humoral immune response, cell differentiation and chromatin modifications were particularly prevalent in down-regulated genes (Figure 2d).\\nRESULTS\\nparagraph\\nIn the same vein, Kyoto Encyclopedia of Genes and Genomes (KEGG) database analysis integrating individual components into unified pathways was used to identify the enrichment of specific pathway components in functionally regulated gene groups (Figure 2e, f; Additional data file 3). The results indicate that six KEGG pathways were significantly enriched (P < 0.05) in up-regulated genes, including cytokine-cytokine receptor interaction, apoptosis inhibition, epithelial cell signaling in Helicobacter pylori infection, hematopoietic cell lineage, Toll-like receptor signaling pathway and gamma-hexachlorocyclohexane degradation (Figure 2e; Additional data file 3), and five KEGG pathways were significantly enriched (P < 0.05) in down-regulated genes, involving several metabolic pathways such as tryptophan metabolism, amino acid degradation, propionate metabolism, limonene and pinene degradation and vitamin B6 metabolism (Figure 2f; Additional data file 3).\\nRESULTS\\nparagraph\\nTaken together, these results suggest the activation of inflammation and proliferation-promoting pathways followed by alterations in apoptotic and immune responses as well as in cell signaling and metabolism that can mediate or modulate their own pathway or the cross-talk between pathways that account for the functional reprogramming of TIA-1/TIAR-depleted HeLa cells.\\nRESULTS\\ntitle_2\\nDistinct and overlapping functions of TIA-1 and TIAR in the control of the transcriptome\\nRESULTS\\nparagraph\\nTo determine whether TIA-1 and TIAR have selective and specific effects on the control of the gene expression pathways described above, we next performed gene expression profiling analyses in HeLa cells in which either TIA-1 or TIAR was inactivated (Additional data files 4 and Additional data files 5). Inactivation of TIA-1 revealed 124 differentially expressed genes (P < 0.05), of which 36 and 88 were up- and down-regulated, respectively (Additional data file 4). On the other hand, knockdown of TIAR resulted in the differential expression (P < 0.05) of 94 genes, of which 23 and 71 were up- and down-regulated, respectively (Additional data file 5).\\nRESULTS\\nparagraph\\nNext, GO biological process categories and KEGG pathway analyses were performed as before. A summary of GO annotations by biological process categories is provided in Additional data file 6a-d. GO categories (P < 0.01) associated with both TIA-1 (Additional data files 6a and Additional data files 7) and TIAR (Additional data files 6b and Additional data files 8) knockdown are related to inflammation and cell proliferation processes, including up-regulated genes in the following categories: cell-cell signaling, inflammatory response, chemotaxis, negative and positive regulation of cell proliferation, immune response, G-protein coupled receptor protein signaling pathway, cell adhesion, angiogenesis, cell motility, transduction and cell surface receptor mediated signaling. These results agree with the observations previously found in HeLa cells under TIA-1 and TIAR double silencing (Figure 2c), indicating that TIA-1 and TIAR are redundant in the control of these pathways. However, GO categories (P < 0.01) of down-regulated genes associated with the depletion of either TIA-1 (Additional data files 6c and Additional data files 7) or TIAR (Additional data files 6d and Additional data files 8) showed striking differences in the gene clusters. Indeed, whereas GO categories of down-regulated genes in TIA-1 knockdown cells are linked to genes involved in apoptosis induction, negative regulation of cytokine biosynthesis and metabolic responses (Additional data files 6c and Additional data files 7, GO categories of down-regulated genes in TIAR knockdown cells were limited to genes implicated in RNA processing (Additional data files 6d and Additional data files 8).\\nRESULTS\\nparagraph\\nResults from KEGG pathway analysis (P < 0.05) are provided in Additional data file 6e-h. KEGG data for up-regulated genes in TIA-1 (Additional data files 6e and Additional data files 7) and TIAR (Additional data files 6f and Additional data files 8) knockdown cells are associated with genes related to cytokine-cytokine receptor interaction, apoptosis inhibition, and immune, chemotaxis, inflammatory, proliferative and metabolic responses, in agreement with the results described above (Additional data file 6a, b). On the other hand, KEGG data for down-regulated genes in TIA-1 knockdown cells are linked to vascular epidermal growth factor signaling pathway and citrate and reductive carboxylase cycles (Additional data files 6g and Additional data files 7), and KEGG data for TIAR knockdown cells are associated with several pathways related to the metabolism of amino acids, vitamins, fatty acids, glycerophospholipids and energy availability (Additional data files 6h and Additional data files 8), suggesting that both TIA-1 and TIAR can regulate specific aspects of cell metabolism.\\nRESULTS\\nparagraph\\nCollectively, these results indicate that TIA-1 and TIAR regulate specific and overlapping aspects of the transcriptome, suggesting that their functional effects can be redundant, additive and/or independent, in agreement with previous findings.\\nRESULTS\\ntitle_2\\nA molecular hallmark involving cytokines, chemokines and growth-stimulating factors is associated with the depletion of TIA proteins\\nRESULTS\\nparagraph\\nTo test whether reduced expression of TIA proteins is associated with the regulation of characteristic gene clusters, up- and down-regulated genes were distributed using Venn diagrams (Figure 3a). In these analyses, we identified a gene signature involving nine up- and six down-regulated genes (Figure 3a). This gene signature is composed of up-regulated genes, including IL-8, AREG, GDF15, IL-6, EREG, KYNU, PTGS2, CXCL1 and CXCL2 (Figure 3b, indicated in red) which are mediators of inflammation, cell-cell signaling, cell proliferation, tryptophan metabolism, immune and angiogenesis responses, and down-regulated genes such as RAB40B, TMSL8, UCP2, PAK3, TIA-1 and TIAR (Figure 3b, indicated in green), which represent a set of heterogeneous genes implicated in protein transport and phosphorylation, cytoskeleton organization and biogenesis, control of the production of reactive oxygen species/oxidative metabolism and induction of apoptosis.\\nF3\\nFIG\\nfig_caption\\nIdentification of a gene cluster associated with the silencing of TIA proteins. (a) Venn diagrams depicting the numbers of genes that were up-regulated (left) or down-regulated (right) by TIA-1 (green), TIAR (red) or TIA-1 plus TIAR (blue) silencing. All data for all genes are presented in Additional data files 2 to 5 and 7 and 8. (b) Gene cluster defining a molecular signature of up- (highlighted in red) and down-regulated (highlighted in green) genes in TIA-1 and TIAR-depleted HeLa cells. The microarray data were analyzed by the Rank Products method. Fold is an average measure of the fold change in differential expression and the false discovery rate (FDR) indicates the expected percentage of false positives. The Probe-set ID is the Unigene probe-set identifier.\\nRESULTS\\ntitle_2\\nValidation of microarray-predicted changes in gene expression patterns\\nRESULTS\\nparagraph\\nThe effects on steady-state mRNA levels detected by the microarray were independently validated using quantitative RT-PCR assays for 26 different genes. Figure 4a shows validation of predicted up-regulated (EREG, PTGS2, IL-1A, IL-6, IL-8, AREG, GDF15, CXCL1, CXCL2 and KYNU) and down-regulated (TIA-1, TIAR, PAK3, TMSL8, UCP2, CD24, TNFSF10, RAB40B, MMP2, TIMP2, FASTK and TFDP2) genes that were increased or decreased, respectively, by depletion of TIA-1 and TIAR in HeLa cells. As expected, the results confirmed predicted up- and down-regulation for all genes analyzed (Figure 4a). Furthermore, APPBP2, MTFR1, beta-actin and OPN1 genes were used as controls (Figure 4a). Additionally, to know whether TIA-1 and TIAR have additive, overlapping or synergistic effects on steady-state target mRNA levels, some up-regulated (IL-6, IL-8, AREG, PTGS2 and EREG) and down-regulated (TNFSF10 and CD24) genes, as well as the beta-actin gene as a control, were also validated by quantitative RT-PCR with independent silencing of either TIA-1 or TIAR. As shown in Figure 4b, depletion of TIA-1 or/and TIAR did affect partially the steady-state levels of analyzed mRNAs, showing that the actions of TIA proteins are additive and overlapping rather than synergistic. Taken together, these results are fully consistent with the observations performed by microarray analyses, suggesting that some changes observed in our microarray experiments could represent putative novel target genes regulated by TIA-1 and/or TIAR proteins.\\nF4\\nFIG\\nfig_caption\\nValidation of microarray-predicted changes by quantitative RT-PCR (qRT-PCR). (a) Differential expression of 26 genes, 10 up-regulated (bottom bars), 12 down-regulated (top bars) and 4 unchanged (middle bars), was verified by real time RT-PCR using specific primer pairs (Additional data file 11). The represented values were normalized and are expressed relative to beta-actin, whose value is fixed arbitrarily to 1, and are means +- standard error of the mean (SEM; n = 3; *P < 0.001; **P < 0.01; ***P < 0.05). (b) Differential expression of eight genes, five up-regulated (IL-6, IL-8, AREG, PTGS2 and EREG), two down-regulated (TNFSF10 and CD24) and one unaffected (beta-actin), from cytoplasmic RNA isolated from HeLa cells transfected with either control (white bars), TIA-1 (grey bars), TIAR (dark grey bars) or TIA-1 plus TIAR (black bars) siRNAs was verified as above. The represented values were normalized and are expressed relative to beta-actin, whose value is fixed arbitrarily to 1, and are means +- SEM (n = 3; *P < 0.001; **P < 0.01).\\nRESULTS\\ntitle_2\\nDepletion of TIA proteins alters both mRNA stability and basal gene transcription\\nRESULTS\\nparagraph\\nTo determine the mechanism underlying the alterations in gene expression observed through inactivation of TIA proteins, HeLa cells transfected with control siRNA or with TIA-1 and TIAR siRNAs were treated with actinomycin D, a selective inhibitor of RNA polymerase II (Figure 5a, b). The results suggest an effect on mRNA stability, as indicated by the steady-state mRNA levels of PTGS2, GDF15, IL-8, TIMP2 and TNFSF10 in TIA-1/TIAR-depleted HeLa cells (Figure 5a). However, the specific contribution of mRNA turnover can not be measured accurately in control HeLa cells because steady-state levels of some target mRNAs are extremely low. Perhaps a stressing stimulus that enhances mRNA decay is needed in order to establish properly whether TIA proteins affect mRNA stability. Additionally, some representative genes were also quantified at the protein level by western blotting. Figure 5b shows the results of TIA-1, TIAR and Prostaglandin-endoperoxide synthase 2 (PTGS2) protein expression levels, which are consistent with the real-time PCR results. As controls, Mcl-1 protein expression was specifically affected by the treatment with actinomycin D, but HuR and alpha-tubulin protein expression was not modified by either TIA-1 and TIAR depletion or by the treatment with actinomycin D (Figure 5b).\\nF5\\nFIG\\nfig_caption\\nSilencing of TIA proteins alters both mRNA stability and gene transcription. (a) DNA transcription was inhibited in control and TIA-1/TIAR-depleted HeLa cells by the addition of actinomycin D (Act D; 5 mug/ml) to the culture medium. Cytoplasmic RNA was isolated at various times (0, 3 and 6 h) after the addition of the inhibitor Act D and analyzed by real time RT-PCR. Relative levels of beta-actin, PTGS2, GDF15, IL-8, TIMP2 and TNFSF10 mRNAs were determined with the specific primer pairs (Additional data file 11). A representative experiment is shown. Steady-state RNA levels are represented at various times after the addition of Act D. Open and closed circles illustrate relative RNA levels in control (C) and TIA-1/TIAR (1+R)-depleted samples, respectively. (b) Validation of differential expression for selected genes by western blot analysis. Protein extracts (5 mug (lane 1; C(1/5)) or 25 mug (lanes 2 to 8)) derived from post-transfected HeLa cells with control (lanes 1 to 5) and TIA-1 and TIAR (lanes 6 to 8) siRNAs and treated with Act D as in (a) were analyzed with the indicated antibodies. A representative blot is shown. Molecular weight markers and the identities of protein bands are indicated. TIA-1(a/b) and TIAR(a/b) refer to the major isoforms of TIA-1 and TIAR, respectively. (c, d) Effect of the TIA-1/TIAR silencing on transcriptional activation of the PTGS2 (c) and IL-6 (d) gene promoters. A schematic representation of PTGS2 (c) and IL-6 (d) human gene promoters is shown. In both cases, cis-acting consensus sequences are represented by boxes. Control and TIA-1/TIAR-depleted HeLa cells were transiently cotransfected with PTGS2 or IL-6 promoter-driven firefly luciferase constructs together with a beta-galactosidase-expressing plasmid (used as a transfection control) as described in Materials and methods. Sixteen hours after cotransfection, cells were cultured in the presence of DMSO or 20 ng/ml PMA for 6 h and assayed for luciferase and beta-galactosidase activities. The represented values in (c, d) - the ratio between luciferase relative light units (RLU)/beta-galactosidase activity - were normalized and are expressed relative to the control sample treated with DMSO, whose value is fixed arbitrarily to 1, and are means +- standard error of the mean (n = 3; *P < 0.001 in (c); *P < 0.01 in (d)).\\nRESULTS\\nparagraph\\nGiven that the results on mRNA turnover do not conclusively explain the steady-state target mRNA levels in TIA-1/TIAR-depleted HeLa cells, we decided to estimate the contribution of the TIA-1/TIAR-induced silencing on transcription activation of two target genes. For this purpose, control and TIA-1/TIAR-depleted HeLa cells were cotransfected transiently with PTGS2 and IL-6 promoter-driven firefly luciferase constructs and a beta-galactosidase-expressing plasmid used as a transfection control. Sixteen hours after transfection, cells were stimulated with dimethyl sulfoxide (DMSO) or phorbol 12-myristate 13-acetate (PMA), as described in the Materials and methods section, and 6 hours after stimulation both luciferase and beta-galactosidase activities were measured in whole-cell extracts. As shown in Figure 5c, d, the depletion of TIA proteins was able to promote a significant and reproducible induction (2- to 2.5-fold) of both luciferase constructs under the control of PTGS2 (Figure 5c) and IL-6 (Figure 5d) human promoter sequences, suggesting that reduced expression of TIA proteins deregulates the basal transcriptional activity of both promoters. Taken together, these results could suggest that the gene expression patterns detected in TIA-1/TIAR-depleted HeLa cells might be the result of an overlapping regulation, implying the involvement of several molecular events at the post-transcriptional and transcriptional levels.\\nRESULTS\\ntitle_2\\nSilencing of TIA proteins results in increased cell proliferation and anchorage-independent growth\\nRESULTS\\nparagraph\\nAs shown in Figure 2 and in Additional data file 6, GO analyses suggested a role for TIA-1 and TIAR proteins in the control of cell proliferation. Thus, to validate at the functional level the results obtained with the GO analyses and to determine the putative role of TIA-1 and TIAR on cell proliferation, we examined the proliferative potential of HeLa cells with diminished expression of TIA-1 and/or TIAR. As shown in Figure 6, reduction of TIA-1 and/or TIAR expression in HeLa cells resulted in increased cell proliferation compared to control cells. Indeed, total cell numbers (Figure 6a) as well as measurement of metabolic activity by methyl thiazolyl tetrazolium assay (Figure 6b) support an individual and collective role for TIA proteins in the control of cell proliferation. Furthermore, TIA-1 and/or TIAR silencing promoted a significant increase in the proportion of cells in S and G2/M phases concurrent with a reduction in the proportion of cells in G0/G1 (Figure 6c). Most interestingly, the colony-forming capacity in plaque (Figure 6d) and soft agar (Figure 6e) was substantially increased in TIA-1/TIAR-depleted cells compared with the corresponding control cells. In addition, the number of colonies as well as the number of cells per colony were significantly increased in HeLa cells lacking TIA proteins (Figure 6d, e, left and right panels, respectively). Altogether, these results indicate that the knockdown of TIA-1 and TIAR proteins in HeLa cells triggers cell proliferation and anchorage-independent growth.\\nF6\\nFIG\\nfig_caption\\nDepletion of TIA proteins activates cell proliferation and anchorage-independent growth. (a) Post-transfected HeLa cells with siRNAs against control (C; white diamond), TIA-1 (1; grey square), TIAR (R; grey circle) or TIA-1 plus TIAR (1+R; black circle) were seeded in six-well plates and the number was counted on the days indicated for 6 days. Each time point represents the means +- standard error of the mean (SEM; n = 3 to 6; *P < 0.01). (b) Cells grown for 6 days as in (a) were monitored by methyl thiazolyl tetrazolium (MTT) assays. The represented values were normalized and are expressed relative to control (C), whose value is fixed arbitrarily to 1, and are means +- SEM (n = 3 to 5; *P < 0.001). (c) Analysis of cell-cycle phases by flow cytometry after propidium iodide staining. The data are means +- SEM (n = 3; *P < 0.01; **P < 0.05). (d) Representative photographs of colonies generated from transfected HeLa cells. One week after seeding 2,000 cells in a 6-well plate, the cells were fixed in paraformaldehyde (5%), stained with crystal violet (0.01%) and quantified by light microscopy. The represented values are the means +- SEM from three independent experiments performed in duplicate for at least eleven independent colonies (*P < 0.001). A colony was defined as a cell cluster when containing at least 86 cells. Scale bars represent 200 mum. (e) Soft-agar colony formation. In each 6-well plate, 5,000 post-transfected HeLa cells as above were seeded into soft-agar matrix and incubated for 14 days. The cell number was counted under a light microscope after staining with crystal violet as before. The represented values are means +- SEM for three independent experiments performed in duplicate for at least 36 counted fields (*P < 0.001; **P < 0.01). In this case, a colony was defined as a cell cluster containing at least 4 cells. Scale bars represent 100 mum.\\nDISCUSS\\ntitle_1\\nDiscussion\\nDISCUSS\\nparagraph\\nGene expression profiling has revolutionized our ability to understand the concerted and global response of the human transcriptome at the molecular level. Collectively, our data suggest that either expression or suppression of TIA-1 and/or TIAR proteins triggers pathway-specific and biologically coherent regulatory programs linked to an acute inflammation and cell proliferation-promoting transcriptome. They also reveal that TIA proteins can provide a novel regulatory layer of mechanisms for cross-talk between different pathways that coordinate cellular phenotypes such as metabolic homeostasis, cell survival and growth. Most importantly, we would like to indicate that our findings agree with and can explain at the molecular level previous observations based on mice models where suppression of TIA-1 expression promotes chronic inflammatory responses as well.\\nDISCUSS\\nparagraph\\nA molecular hallmark associated with the depletion of TIA proteins comprises a gene signature including those encoding pro-inflammatory cytokines (IL-1A and IL-6), inflammatory chemokines (CXCL1, CXCL2 and IL-8), growth-stimulating factors (AREG, EREG and GDF15) and pro-angiogenic inducers (PTGS2 and IL-8) as well as the RAS oncogene family member RAB40B, regulators of cytoskeleton organization and biogenesis (TMSL8 and PAK3) and a mitochondrial modulator (UCP2). This gene cluster contains the ensemble of gene functions that may coordinately work to activate the acquisition of cell proliferation and anchorage-independent cell growth phenotypes observed in TIA-1/TIAR-depleted cells. Using a bioinformatic tool (the Pathway Studio software (version 6.0) and the 'ResNet Mammalian' database developed by Ariadne Genomics), a gene network based on biochemical data was created for experimentally determined interactions between clusters of up- and down-regulated genes associated with the depletion of TIA-1 and TIAR in HeLa cells (Figure 7; Additional data file 9). This network suggests a cellular scenario where silencing of these proteins leads to increased expression of a gene cluster encoding components of inflammation, angiogenesis and cell growth signaling, with a predicted enhancer effect on these pathways (Figure 7; Additional data file 9). This in silico simulation could integrate our results at a molecular level as well as previous observations on the importance of the cell-cell signaling, chemotaxis, adhesion, angiogenesis and immune responses mediated by interleukins, chemokines and growth-stimulating factors in molecularly connecting chronic inflammation and compensatory proliferation.\\nF7\\nFIG\\nfig_caption\\nInteraction network associated with the gene cluster induced upon suppression of TIA proteins. Visualization of biochemical relationships between gene clusters up-regulated (red) and down-regulated (green) identified by microarray analyses were computed using the Pathway Studio (version 6.0) software and the 'ResNet Mammalian' database developed by Ariadne Genomics, Inc.. The network topology illustrates positive (red boxes) and negative (green boxes)-regulated cell processes based on interactions after inactivation of TIA proteins. The nature of the regulators is depicted schematically as shown in the legend to the right. Continuous and dotted lines illustrate direct and indirect protein-protein interactions, respectively. The thickness of blue lines is proportional to the number of interactions reported in the literature.\\nDISCUSS\\nparagraph\\nWhy should inflammation and cell proliferation signaling result in a coherent transcriptional and post-transcriptional program of gene regulation targeting similar classes of genes? Promotion of cell proliferation and growth can be achieved through either abnormally activated or deregulated signaling pathways involved in cell cycle regulation or, quite often, abnormal growth signals outside the transformed cell, which are usually not considered. In this regard, TIA-1/TIAR target genes regulating proliferation include those encoding growth- and survival-stimulating factors, such as the epidermal growth factor receptor (EGFR)/pan-HER ligand epiregulin (EREG), the epidermal growth factor-like growth factor amphiregulin (AREG), and a member of the transforming growth factor-beta (TGF-beta) superfamily known as growth differentiation factor-15 (GDF15) (Figure 7). Moreover, the cellular processes and pathways promoting these growth-stimulating factors can be reinforced by IL-6 and IL-8, the chemokines CXCL1 and 2 and cyclooxygenase-2, which contribute to cell growth through self-sufficiency in growth signals, evasion of apoptosis, insensitivity to growth inhibitors as well as inflammatory and angiogenic potential responses.\\nDISCUSS\\nparagraph\\nThese findings provide novel insights into the critical role of individual regulators such as TIA proteins to control, indirectly rather than directly, protein output from hundreds of target genes in a given cellular context. These regulators may impact physiological processes by regulating the concentrations of a few key cellular proteins that may be components of a single or functionally interrelated pathways. This is an interesting observation because it suggests that the molecular links that lead to gene expression control are intimately interconnected, pointing to a widespread role for TIA proteins in the negative regulation of transcriptional rates of the human genome. Thus, these proteins may function as adaptors or coupling factors to facilitate cross-talk between different layers that define the complex interactions involved in the regulation of the human transcriptome.\\nDISCUSS\\nparagraph\\nA close inspection of differentially expressed genes between control versus TIA-1/TIAR-depleted HeLa cells reveals the up-regulation of cell-cycle activators such as CDC23 and the down-regulation of cell-cycle suppressors such as CHES1, MPHOSPH9, CCNB1 (encoding cyclin B1), ANAPC7, NASP and CDC7, which could also account for the enhanced proliferation. Additionally, genes for cell-cycle controlling transcription factors - for example, TFDP1, TFDP2 and E2F8 - as well as other cell-cycle progression related factors - UHRF1, BARD1, RABGAP1, PSRC1, PCGF6 and SEPT6 - are also down-regulated in HeLa cells with TIA-1/TIAR-induced silencing (Additional data files 9 and Additional data files 10). To our knowledge, this is the first study to analyze, in parallel, global transcriptomic and phenotypic impacts after changes in the expression and/or function of TIA proteins.\\nDISCUSS\\nparagraph\\nWhat could be the molecular mechanism that mediates the changes in steady-state mRNA levels triggered by TIA proteins? We propose that, in addition to mediating some of the signaling effects of the cytokines, chemokines and growth-stimulating factors, the putative molecular mechanism may implicate the activation of basal transcriptional machinery through up-regulation of the gene TAF5L as well genes for nuclear transcription factors - for example, the CREM and MAFF families - together with the down-regulation of genes for transcription repressors, such as PCGF2, NR2F2, SUDS3 and PCGF6 (Additional data files 9 and Additional data files 10). Constitutive activation/repression of basal transcriptional machinery and/or selective transcription factors is an emerging hallmark of various transformed cell types. In addition, they have specifically been implicated in the development of proliferative phenotypes through autocrine and paracrine mechanisms. Our TIA-1/TIAR silencing signature is associated with the up-regulation of critical NF-kappaB and CREM effectors, including CXCL1, CXCL2, PTGS2, IL-6 or IL-8, which have been implicated in cell proliferation and growth. In summary, TIA-1/TIAR-depleted HeLa cells are characterized by a molecular signature indicative of global and constitutive activation of NF-kappaB and CREM downstream effector genes associated with cell proliferation. Presumably, the up-regulation of genes required for proliferation, survival and inhibition of apoptosis might be essential for anchorage-independent growth. These conjectures, however, will have to be proven by additional studies in order to determine the precise role of these genes in the phenotypes observed in this study.\\nDISCUSS\\nparagraph\\nNevertheless, the coordinate expression of inflammation and/or growth-stimulating genes requires both transcriptional and post-transcriptional regulation. Indeed, this gene coordination is also able to exert a positive effect on inflammation and cell proliferation or a positive feedback control on these pathways by regulating the expression of some of its components in TIA-1/TIAR-dependent regulatory events. These multifunctional proteins have been implicated in several steps of RNA dynamics and metabolism, including transcription, RNA splicing, RNA stability and translation, but little is known about the putative mechanisms and selectivity of transcriptional repression exerted by these proteins. It is important to note that these regulators have been localized on the carboxy-terminal domain of RNA polymerase II; therefore, they could interact with basal transcription machinery and influence its activity. In addition, both proteins contain three RNA-binding motifs and are structurally close to heterogeneous ribonucleoproteins. Consistently, both TIA proteins directly bind to RNA as well as to single- and double-stranded DNA. It is tempting to speculate that TIA proteins could also interact with specific promoter sequences and/or with gene-specific/general transcription factors. In this regard, it is more than reasonable to suggest that these proteins can carry out functions during transcription. Further, an increasing body of evidence indicates that transcription and RNA splicing processes are coupled.\\nDISCUSS\\nparagraph\\nCollectively, we postulate that, during transcription, TIA-1 and/or TIAR proteins can be recruited from the carboxy-terminal domain of RNA polymerase II to interact with genomic DNA sequences and/or basal transcription machinery and subsequently slow down RNA polymerase II and, therefore, the rate of transcription. Moreover, these proteins can perform triple functions during chromatin structure organization, transcription and splicing because they can potentially shuttle between DNA and RNA substrates and it has been demonstrated that factors affecting the rate of RNA polymerase II-mediated transcriptional elongation/initiation could influence alternative splicing patterns.\\nDISCUSS\\nparagraph\\nOn the other hand, the prevalent role of these proteins as gene expression post-transcriptional modulators in the regulation of processes of alternative pre-mRNA splicing as well as their role in the stability and translation of specific cellular mRNAs is well-documented. TIA proteins regulate the alternative pre-mRNA splicing of approximately 15% of alternative cassette human exons through binding to U-rich intronic elements, facilitating atypical 5' splice site recognition by U1 small nuclear ribonucleoproteins. These proteins have also been well characterized as regulators of the stability and translation of cellular mRNAs in the cytosol. The proteins have been implicated in stress-induced translational arrest, colocalizing after stress with poly(A)+ RNA in the cytoplasmic foci known as stress granules. TIA-1 and TIAR are both able to bind to the 3'-untranslated regions of the stability/translation regulatory U- and AU-rich elements. This aspect of TIA-1/TIAR function has recently been the focus of a broad and systematic analysis to identify mRNAs associated with both RNA-binding proteins. However, it is interesting to note that no overlap is observed between gene targets reported to be repressed by TIA-1 at the translation level and genes regulated by TIA1/TIAR at the alternative splicing level. This suggests that TIA proteins regulate distinct subsets of genes at the splicing and translation levels. Indeed, TIA-1 and/or TIAR proteins might independently switch on/off molecular plasticity in gene regulation machineries such as transcription, alternative splicing, stability and/or translation to coordinate the establishment of the diversity of specific transcriptomes and proteomes that are required in complex cellular responses such as cell proliferation and growth. Thus, the roles of these regulators and further newly identified targets in these physiological processes deserve further investigation.\\nCONCL\\ntitle_1\\nConclusions\\nCONCL\\nparagraph\\nThe data presented in this manuscript illustrate specific changes in transcriptomic dynamics associated with cell-cell signaling, immuno-suppression, inflammation, angiogenesis, metabolism and cell proliferation-related mRNAs upon silencing of TIA-1 and/or TIAR proteins in HeLa cells. A gene cluster including cytokines, chemokines and growth-stimulating factors was identified as a molecular hallmark/signature associated with the depletion of TIA proteins. From a mechanistic viewpoint, TIA proteins could modulate transcriptional and post-transcriptional events associated with the control of target gene promoter basal activity and specific mRNA turnover. Additionally, the results also show that the knockdown of TIA-1 and/or TIAR proteins in HeLa cells triggers cell proliferation and anchorage-independent growth. In silico simulation of a biochemical interaction network incorporating up- and down-regulated target genes is consistent with a set of functions coherent with the different roles of several in vivo pathways to activate the acquisition of cell proliferation and anchorage-independent growth phenotypes detected after TIA-1/TIAR silencing. Collectively, our results argue that silencing of TIA proteins can trigger pathway-specific changes and affect genes with coherent functions that are important for cell proliferation and growth responses.\\nMETHODS\\ntitle_1\\nMaterials and methods\\nMETHODS\\ntitle_2\\nCell culture and RNA interference analysis\\nMETHODS\\nparagraph\\nHeLa S2 cells were processed and RNA interference experiments were carried out as described previously.\\nMETHODS\\ntitle_2\\nTransfections, luciferase and beta-galatosidase assays\\nMETHODS\\nparagraph\\nThe human PTGS2 promoter [GenBank:AF276953.2] construct containing the full-length promoter sequence fused to a firefly luciferase reporter gene, named p2-1900 (-1,796, +104), was generated as described previously. The human IL-6 gene promoter [GenBank:AF372214.2] construct containing a 651-bp fragment of the human promoter sequence was fused to a firefly luciferase reporter gene, named pIL-6-luc651, as described previously. Both constructs were kindly provided by Dr Y Revilla. The beta-galactosidase plasmid pcDNA3.1/myc-His/lacZ (Invitrogen, Carlsbad, CA, USA) was kindly provided by Dr J Berlanga. Control and TIA-1/TIAR-depleted HeLa cells, as described above, were co-transfected transiently with 50 and 500 ng of specific reporter lacZ and firefly luciferase plasmids, respectively, in six-well plates. The cells were incubated at 37 C for 16 h and then post-transfected cells were stimulated with DMSO or 20 ng/ml of PMA in DMSO. Six hours after stimulation, cells were lysed with 200 mul of cell culture lysis reagent (Promega, Madison, WI, USA) and microcentrifuged at 14,000 rpm for 5 minutes at 4 C, and 20 and 40 mul of each supernatant was used to determine firefly luciferase activity in a Monolight 2010 luminometer (Analytical Luminescence Laboratory, San Diego, CA, USA). beta-Galactosidase activity was measured as described previously. Luciferase activity was expressed as relative light units (RLU) per milligram of protein determined by the bicinchoninic acid method (Pierce, Rockford, IL, USA) and normalized to beta-galactosidase-specific activity expressed as nmoles of hydrolyzed o-nitrophenyl-beta-D-galactopyranoside (Sigma, St. Louis, MO, USA) per minute and milligram of protein at 37 C. Co-transfection experiments were performed in triplicate and the data are presented as the means of the ratio RLU/beta-galactosidase, expressed as fold induction relative to the corresponding control values (means +- standard error of the mean).\\nMETHODS\\ntitle_2\\nProliferation and cell-cycle analyses\\nMETHODS\\nparagraph\\nFor cell growth analysis, post-transfected HeLa cells (2 x 104) were seeded and collected for counting from triplicate at the indicated time points for 6 days. Cell proliferation was also quantified by measuring the conversion of methyl thiazolyl tetrazolium into DMSO-soluble formazan by living cells, with absorbance measured at 560 nm using a spectrophotometer. Cell-cycle analysis was carried out by flow cytometry after propidium iodide staining.\\nMETHODS\\ntitle_2\\nAnchorage-independent growth assays\\nMETHODS\\nparagraph\\nSoft agar colony formation was assayed by plating in duplicate 1 x 103 cells in 0.35% agarose in 10% fetal calf serum containing media on a 0.8% agarose base in six-well plates. After 14 days, the resulting colonies were stained with 0.01% crystal violet solution, counted and photographed under a light microscope at 100x magnification.\\nMETHODS\\ntitle_2\\nPreparation of cell extracts and western blot analysis\\nMETHODS\\nparagraph\\nWhole-cell extracts were performed and processed as described previously. Immunoblots were carried out using the following antibodies: anti-U2AF65 (MC3; provided by J Valcarcel); anti-PTGS2 (provided by MA Iniguez), anti-TIA-1, anti-TIAR and anti-HuR (Santa Cruz Biotechnology, Santa Cruz, CA, USA), anti-Mcl-1 (Biomol, Farmingdale, NY, USA) and anti-alpha-tubulin (Sigma, St. Louis, MO, USA).\\nMETHODS\\ntitle_2\\nMETHODS\\nparagraph\\nCytoplasmic RNA isolations and semiquantitative RT-PCR analysis were carried out as described previously. The expression of selected genes was assessed independently by quantitative RT-PCR as described in. A list of primer pairs is given in Additional data file 11. RNA turnover experiments were performed by adding actinomycin D (5 mug/ml) to the growth medium.\\nMETHODS\\ntitle_2\\nTranscriptome analysis\\nMETHODS\\nparagraph\\nRNA quality checks, amplification, labeling, hybridization with Human Genome U133 Plus 2.0 Array Chips (approximately 55,000 transcripts; Affymetrix Inc., Santa Clara, CA, USA) and initial data extraction were performed at the Genomic Array Core Facility in Scientific Park, Madrid at the Universidad Complutense de Madrid. Comparison of multiple cDNA array images (three independent experiments per biological condition tested) was carried out by using an average of all of the gene signals on the array (global normalization) to normalize the signal between arrays (Additional data file 1). Heat map and sample clustering plots as well as GO and KEGG database analyses were performed using software programs provided by Integromics . Networks of biochemical relationships between gene clusters identified as up-regulated and down-regulated by microarray analysis were created using the Pathway Studio software (version 6.0) and the 'ResNet Mammalian' database developed by Ariadne Genomics, Inc. (Rockville, MD, USA). Microarray data have been deposited in the EMBL-EBI ARRAY EXPRESS database and are accessible through the ArrayExpress accession number [ArrayExpress:E-MTAB-93].\\nMETHODS\\ntitle_2\\nStatistical analysis\\nMETHODS\\nparagraph\\nRepresented values are shown as means +- standard error of the mean. Differences were tested for significance by means of the Student's t-test. TIA-1/TIAR siRNA effects on HeLa cells were compared with respect to the control siRNA transfection. A probability level P < 0.05 was considered significant.\\nABBR\\ntitle_1\\nAbbreviations\\nABBR\\nparagraph\\nAUTH_CONT\\ntitle_1\\nAuthors' contributions\\nAUTH_CONT\\nparagraph\\nJMI conceived the research and designed all the experiments. JMI, RR and JA carried out the experiments presented in this paper. RR, JA and JMI wrote the paper. All authors provided feedback and approved the final manuscript.\\nSUPPL\\ntitle_1\\nAdditional data files\\nSUPPL\\nparagraph\\nThe following additional data are available with the online version of this paper: variability of data as function of the mean (MvA), RNA digestion, raw and normalized data box plots (Additional data file 1); summary of differentially expressed genes in TIA-1 and TIAR-depleted HeLa cells (Additional data file 2); summary of GO and KEGG database analyses of microarray-predicted genes from TIA-1 and TIAR-depleted HeLa cells (Additional data file 3); summary of differentially expressed genes in TIA-1-depleted HeLa cells (Additional data file 4); summary of differentially expressed genes in TIAR-depleted HeLa cells (Additional data file 5); global mRNA expression patterns in TIA-1 or TIAR-depleted HeLa cells (Additional data file 6); summary of GO and KEGG database analyses of microarray-predicted genes from TIA-1-depleted HeLa cells (Additional data file 7); summary of GO and KEGG database analyses of microarray-predicted genes from TIAR-depleted HeLa cells (Additional data file 8); global network of genes modulated after TIA-1 and TIAR silencing (Additional data file 9); list of genes up- and down-regulated after TIA-1/TIAR silencing that function as transcription and/or cell-cycle modulators (Additional data file 10); list of primer pair sequences used to validate microarray-predicted genes by real-time PCR analysis (Additional data file 11).\\nSUPPL\\ntitle_1\\nSupplementary Material\\nsurname:Gingeras;given-names:TR\\nREF\\nGenome Res\\nref\\nOrigin of phenotypes: genes and transcripts.\\nsurname:Das;given-names:R\\nsurname:Yu;given-names:J\\nsurname:Zhang;given-names:Z\\nsurname:Gygi;given-names:MP\\nsurname:Krainer;given-names:AR\\nsurname:Gygi;given-names:SP\\nsurname:Reed;given-names:R\\nREF\\nMol Cell\\nref\\nSR proteins function in coupling RNAP II transcription to pre-mRNA splicing.\\nsurname:Suswam;given-names:EA\\nsurname:Li;given-names:YY\\nsurname:Mahtani;given-names:H\\nsurname:King;given-names:PH\\nREF\\nNucleic Acids Res\\nref\\nNovel DNA-binding properties of the RNA-binding protein TIAR.\\nsurname:McAlinden;given-names:A\\nsurname:Liang;given-names:L\\nsurname:Mukudai;given-names:Y\\nsurname:Imamura;given-names:T\\nsurname:Sandell;given-names:LJ\\nREF\\nJ Biol Chem\\nref\\nNuclear protein TIA-1 regulates COL2A1 alternative splicing and interacts with precursor mRNA and genomic DNA.\\nsurname:Dember;given-names:LM\\nsurname:Kim;given-names:ND\\nsurname:Liu;given-names:KQ\\nsurname:Anderson;given-names:P\\nREF\\nJ Biol Chem\\nref\\nIndividual RNA recognition motifs of TIA-1 and TIAR have different RNA binding specificities.\\nsurname:Del Gatto-Konczak;given-names:F\\nsurname:Bourgeois;given-names:CF\\nsurname:Le Guiner;given-names:C\\nsurname:Kister;given-names:L\\nsurname:Gesnel;given-names:MC\\nsurname:Stevenin;given-names:J\\nsurname:Breathnach;given-names:R\\nREF\\nMol Cell Biol\\nref\\nThe RNA-binding protein TIA-1 is a novel mammalian splicing regulator acting through intron sequences adjacent to a 5' splice site.\\nsurname:Forch;given-names:P\\nsurname:Puig;given-names:O\\nsurname:Kedersha;given-names:N\\nsurname:Martinez;given-names:C\\nsurname:Granneman;given-names:S\\nsurname:Seraphin;given-names:B\\nsurname:Anderson;given-names:P\\nsurname:Valcarcel;given-names:J\\nREF\\nMol Cell\\nref\\nThe apoptosis-promoting factor TIA-1 is a regulator of alternative pre-mRNA splicing.\\nsurname:Aznarez;given-names:I\\nsurname:Barash;given-names:Y\\nsurname:Shai;given-names:O\\nsurname:He;given-names:D\\nsurname:Zielenski;given-names:J\\nsurname:Tsui;given-names:LC\\nsurname:Parkinson;given-names:J\\nsurname:Frey;given-names:BJ\\nsurname:Rommens;given-names:JM\\nsurname:Blencowe;given-names:BJ\\nREF\\nGenome Res\\nref\\nA systematic analysis of intronic sequences downstream of 5' splice sites reveals a widespread role for U-rich motifs and TIA1/TIAL1 proteins alternative splicing regulation.\\nsurname:Forch;given-names:P\\nsurname:Puig;given-names:O\\nsurname:Martinez;given-names:C\\nsurname:Seraphin;given-names:B\\nsurname:Valcarcel;given-names:J\\nREF\\nEMBO J\\nref\\nThe splicing regulator TIA-1 interacts with U1-C to promote U1 snRNP recruitment to 5' splice sites.\\nsurname:Izquierdo;given-names:JM\\nsurname:Majos;given-names:N\\nsurname:Bonnal;given-names:S\\nsurname:Martinez;given-names:C\\nsurname:Castelo;given-names:R\\nsurname:Guigo;given-names:R\\nsurname:Bilbao;given-names:D\\nsurname:Valcarcel;given-names:J\\nREF\\nMol Cell\\nref\\nsurname:Gueydan;given-names:C\\nsurname:Droogmans;given-names:L\\nsurname:Chalon;given-names:P\\nsurname:Huez;given-names:G\\nsurname:Caput;given-names:D\\nsurname:Kruys;given-names:V\\nREF\\nJ Biol Chem\\nref\\nIdentification of TIAR as a protein binding to the translational regulatory AU-rich element of tumor necrosis factor alpha mRNA.\\nsurname:Piecyk;given-names:M\\nsurname:Wax;given-names:S\\nsurname:Anderson;given-names:P\\nsurname:Beck;given-names:AR\\nsurname:Kedersha;given-names:N\\nsurname:Gupta;given-names:M\\nsurname:Maritim;given-names:B\\nsurname:Chen;given-names:S\\nsurname:Gueydan;given-names:C\\nsurname:Kruys;given-names:V\\nsurname:Streuli;given-names:M\\nREF\\nEMBO J\\nref\\nTIA-1 is a translational silencer that selectively regulates the expression of TNF-alpha.\\nsurname:Pullmann;given-names:R;suffix:Jr\\nsurname:Kim;given-names:HH\\nsurname:Abdelmohsen;given-names:K\\nsurname:Lal;given-names:A\\nsurname:Martindale;given-names:JL\\nsurname:Yang;given-names:X\\nsurname:Gorospe;given-names:M\\nREF\\nMol Cell Biol\\nref\\nAnalysis of turnover and translation regulatory RNA-binding protein expression through binding to cognate mRNAs.\\nsurname:Yamasaki;given-names:S\\nsurname:Stoecklin;given-names:G\\nsurname:Kedersha;given-names:N\\nsurname:Simarro;given-names:M\\nsurname:Anderson;given-names:P\\nREF\\nJ Biol Chem\\nref\\nT-cell intracellular antigen-1 (TIA-1)-induced translational silencing promotes the decay of selected mRNAs.\\nsurname:Kedersha;given-names:N\\nsurname:Anderson;given-names:P\\nREF\\nBiochem Soc Trans\\nref\\nStress granules: sites of mRNA triage that regulate mRNA stability and translatability.\\nsurname:Lal;given-names:A\\nsurname:Abdelmohsen;given-names:K\\nsurname:Pullman;given-names:R\\nsurname:Kawai;given-names:T\\nsurname:Galban;given-names:S\\nsurname:Yang;given-names:X\\nsurname:Brewer;given-names:G\\nsurname:Gorospe;given-names:M\\nREF\\nMol Cell\\nref\\nPosttranscriptional derepression of GADD45alpha by genotoxis stress.\\nsurname:Tian;given-names:Q\\nsurname:Streuli;given-names:M\\nsurname:Saito;given-names:H\\nsurname:Schlossman;given-names:SF\\nsurname:Anderson;given-names:P\\nREF\\nCell\\nref\\nA polyadenylate binding protein localized to the granules of cytolytic lymphocytes induces DNA fragmentation in target cells.\\nsurname:Kawakami;given-names:A\\nsurname:Tian;given-names:Q\\nsurname:Duan;given-names:X\\nsurname:Streuli;given-names:M\\nsurname:Schlossman;given-names:SF\\nsurname:Anderson;given-names:P\\nREF\\nProc Natl Acad Sci\\nref\\nIdentification and functional characterization of a TIA-1-related nucleolysin.\\nsurname:Iseni;given-names:F\\nsurname:Garcin;given-names:D\\nsurname:Nishio;given-names:M\\nsurname:Kedersha;given-names:N\\nsurname:Anderson;given-names:P\\nsurname:Kolakofsky;given-names:D\\nREF\\nEMBO J\\nref\\nSendai virus trailer RNA binds TIAR, a cellular protein involved in virus-induced apoptosis.\\nsurname:Li;given-names:W\\nsurname:Li;given-names:Y\\nsurname:Kedersha;given-names:N\\nsurname:Anderson;given-names:P\\nsurname:Emara;given-names:M\\nsurname:Swiderek;given-names:KM\\nsurname:Moreno;given-names:GT\\nsurname:Brinton;given-names:MA\\nREF\\nJ Virol\\nref\\nCell proteins TIA-1 and TIAR interact with the 3' stem-loop of the West Nile virus complementary minus-strand RNA and facilitate virus replication.\\nsurname:Beck;given-names:AR\\nsurname:Miller;given-names:JJ\\nsurname:Anderson;given-names:P\\nsurname:Streuli;given-names:M\\nREF\\nProc Natl Acad Sci\\nref\\nRNA-binding protein TIAR is essential for primordial germ cell development.\\nsurname:Lopez de Silanes;given-names:I\\nsurname:Galban;given-names:S\\nsurname:Martindale;given-names:JL\\nsurname:Yang;given-names:X\\nsurname:Mazan-Mamczarz;given-names:K\\nsurname:Indig;given-names:FE\\nsurname:Falco;given-names:G\\nsurname:Zhan;given-names:M\\nsurname:Gorospe;given-names:M\\nREF\\nMol Cell Biol\\nref\\nIdentification and functional outcome of mRNAs associated with RNA-binding protein TIA-1.\\nsurname:Mazan-Mamczarz;given-names:K\\nsurname:Lal;given-names:A\\nsurname:Martindale;given-names:JL\\nsurname:Kawai;given-names:T\\nsurname:Gorospe;given-names:M\\nREF\\nMol Cell Biol\\nref\\nTranslational repression by RNA-binding protein TIAR.\\nsurname:Kim;given-names:HS\\nsurname:Kuwano;given-names:Y\\nsurname:Wilce;given-names:JA\\nsurname:Zhan;given-names:M\\nsurname:Pullmann;given-names:R;suffix:Jr\\nsurname:Mazan-Mamczarz;given-names:K\\nsurname:Li;given-names:H\\nsurname:Kedersha;given-names:N\\nsurname:Anderson;given-names:P\\nsurname:Wilce;given-names:MC\\nsurname:Gorospe;given-names:M\\nREF\\nMol Cell Biol\\nref\\nsurname:Izquierdo;given-names:JM\\nREF\\nJ Biol Chem\\nref\\nHu antigen R (HuR) functions as an alternative pre-mRNA splicing regulator of Fas apoptosis-promoting receptor on exon definition.\\nsurname:Izquierdo;given-names:JM\\nREF\\nMol Biol Cell\\nref\\nFas splicing regulation during early apoptosis is linked to caspase-mediated cleavage of U2AF65.\\nsurname:Izquierdo;given-names:JM\\nsurname:Valcarcel;given-names:J\\nREF\\nJ Biol Chem\\nref\\nTwo isoforms of the T-cell intracellular antigen 1 (TIA-1) splicing factors display distinct splicing regulation activities. Control of TIA-1 isoform ratio by TIA-1 related protein.\\nsurname:Irizarry;given-names:RA\\nsurname:Bolstad;given-names:BM\\nsurname:Collin;given-names:F\\nsurname:Cope;given-names:LM\\nsurname:Hobbs;given-names:B\\nsurname:Speed;given-names:TP\\nREF\\nNucleic Acids Res\\nref\\nSummaries of Affymetrix GeneChip probe level data.\\nsurname:Breitling;given-names:R\\nsurname:Armengaud;given-names:P\\nsurname:Amtmann;given-names:A\\nsurname:Herzyk;given-names:P\\nREF\\nFEBS Lett\\nref\\nRank products: a simple, yet powerful, new method to detect differentially regulated genes in replicated microarray experiments.\\nsurname:Iniguez;given-names:MA\\nsurname:Martinez-Martinez;given-names:S\\nsurname:Punzon;given-names:C\\nsurname:Redondo;given-names:JM\\nsurname:Fresno;given-names:M\\nREF\\nJ Biol Chem\\nref\\nsurname:Eickelberg;given-names:O\\nsurname:Roth;given-names:M\\nsurname:Mussmann;given-names:R\\nsurname:Rudiger;given-names:JJ\\nsurname:Tamm;given-names:M\\nsurname:Perruchoud;given-names:AP\\nsurname:Block;given-names:L-H\\nREF\\nCirculation\\nref\\nCalcium channel blockers activate the interleukin 6 gene via the transcription factors NF-IL-6 and NF-kappaB in primary human vascular smooth muscle cells.\\nsurname:Phillips;given-names:K\\nsurname:Kedersha;given-names:N\\nsurname:Shen;given-names:L\\nsurname:Blackshear;given-names:PJ\\nsurname:Anderson;given-names:P\\nREF\\nProc Natl Acad Sci\\nref\\nArthritis suppressor genes TIA-1 and TTP dampen the expression of tumor necrosis factor alpha, cyclooxygenase 2 and inflammatory arthritis.\\nsurname:Nguyen;given-names:DX\\nsurname:Massague;given-names:J\\nREF\\nNat Rev Genet\\nref\\nGenetic determinants of cancer metastasis.\\nsurname:Naugler;given-names:WE\\nsurname:Karin;given-names:M\\nREF\\nCurr Opin Genet Dev\\nref\\nNF-kappaB and cancer-identifying targets and mechanisms.\\nsurname:Hanahan;given-names:D\\nsurname:Weinberg;given-names:RA\\nREF\\nCell\\nref\\nThe hallmarks of cancer.\\nsurname:Kroemer;given-names:G\\nsurname:Pouyssegur;given-names:J\\nREF\\nCancer Cell\\nref\\nTumor cell metabolism: cancer's achilles' heel.\\nsurname:Reyes;given-names:R\\nsurname:Izquierdo;given-names:JM\\nREF\\nBiochem Biophys Res Commun\\nref\\nHalf pint couples transcription and splicing of eIF4E-1,2 gene during fly development.\\nREF\\nref\\nGenomics Unit, Scientific Park, Madrid\\nREF\\nref\\nIntegromics \\nREF\\nref\\nAriadne Genomics\\nREF\\nref\\nEMBL-EBI ARRAY EXPRESS Database\\nREF\\nref\\nAffymetrix - NetAffx  Analysis Center\\nREF\\nref\\nUniGene Database\\nREF\\nref\\nEntrez Gene Database\\nREF\\nref\\nSwissProt Database\\nREF\\nref\\nOnline Mendelian Inheritance in Man database\\nREF\\nref\\nGene Ontology Database\\nREF\\nref\\nGenMAPP Database\\nREF\\nref\\nInterPro Database\\nREF\\nref\\nUniversal ProbeLibrary Assay Design Center Software from Roche\\nREF\\nref\\nNCBI BLAST\\n\"}"
      ]
     },
     "execution_count": 12,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "documents"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'26925227': {'hardware_4': ['illumina hiseq 2000'],\n",
       "  'organism_part_5': [],\n",
       "  'experimental_designs_10': ['case control design'],\n",
       "  'assay_by_molecule_14': ['rna assay'],\n",
       "  'study_type_18': ['rna-seq of coding rna']},\n",
       " '25435910': {'hardware_4': [],\n",
       "  'organism_part_5': [],\n",
       "  'experimental_designs_10': [],\n",
       "  'assay_by_molecule_14': ['rna assay'],\n",
       "  'study_type_18': ['rna-seq of coding rna']},\n",
       " '23671666': {'hardware_4': [],\n",
       "  'organism_part_5': [],\n",
       "  'experimental_designs_10': [],\n",
       "  'assay_by_molecule_14': ['rna assay'],\n",
       "  'study_type_18': ['transcription profiling by array']},\n",
       " '23079210': {'hardware_4': [],\n",
       "  'organism_part_5': [],\n",
       "  'experimental_designs_10': [],\n",
       "  'assay_by_molecule_14': [],\n",
       "  'study_type_18': ['transcription profiling by rt-pcr']},\n",
       " '29980666': {'hardware_4': ['illumina hiseq 2000'],\n",
       "  'organism_part_5': [],\n",
       "  'experimental_designs_10': [],\n",
       "  'assay_by_molecule_14': [],\n",
       "  'study_type_18': []},\n",
       " '21299862': {'hardware_4': [],\n",
       "  'organism_part_5': [],\n",
       "  'experimental_designs_10': [],\n",
       "  'assay_by_molecule_14': ['rna assay'],\n",
       "  'study_type_18': ['transcription profiling by array']},\n",
       " '21535883': {'hardware_4': [],\n",
       "  'organism_part_5': [],\n",
       "  'experimental_designs_10': ['in vivo design'],\n",
       "  'assay_by_molecule_14': ['rna assay'],\n",
       "  'study_type_18': ['transcription profiling by array']},\n",
       " '19709424': {'hardware_4': [],\n",
       "  'organism_part_5': [],\n",
       "  'experimental_designs_10': [],\n",
       "  'assay_by_molecule_14': ['rna assay'],\n",
       "  'study_type_18': ['transcription profiling by array']}}"
      ]
     },
     "execution_count": 13,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "labels"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## load_modules and run_modules\n",
    "\n",
    "\n",
    "For these, just take a look at/run ´main.py´.\n",
    "load_modules loads a set of documents/labels, as well as a context_shortener and form_filler, as specified in the arguments.\n",
    "run_modules iterates through these and evaluate each time, and stores the scores in wandb/weave (requires login details - create your own wandb project if you want to use this)"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "upcast",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.10.14"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
