{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [],
   "source": [
    "from dataset_loader import load_arxpr_data\n",
    "\n",
    "from llama_index.core import Document\n",
    "from llama_index.core.node_parser import get_leaf_nodes, get_root_nodes\n",
    "from llama_index.core.node_parser import (SentenceSplitter, MarkdownNodeParser, MarkdownElementNodeParser)\n",
    "\n",
    "from llama_index.core import VectorStoreIndex\n",
    "from llama_index.core import Settings\n",
    "from llama_index.core.schema import TextNode\n",
    "from llama_index.llms.ollama import Ollama\n",
    "from llama_index.embeddings.ollama import OllamaEmbedding\n",
    "from llama_index.core.ingestion import IngestionPipeline\n",
    "\n",
    "from llama_index.core.extractors import (\n",
    "    # SummaryExtractor,\n",
    "    QuestionsAnsweredExtractor,\n",
    "    KeywordExtractor,\n",
    "    # BaseExtractor,\n",
    ")\n",
    "\n",
    "\n",
    "import nest_asyncio\n",
    "nest_asyncio.apply()\n",
    "\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Markdown Parser"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 223,
   "metadata": {},
   "outputs": [],
   "source": [
    "\n",
    "def _pseudo_markdown_splitter(text: str, chunk_size=2000, chunk_overlap=125, markdown_headers=[], exclude_headers=[], verbose=False):\n",
    "    \"\"\"\n",
    "    Borrowing the LangChain markdown splitter to spot header str in lieu of #, ##, ###, etc. \n",
    "    Keeps the md-formated header in medatdata dict if found, else nothing.\n",
    "    Then recursively splits the text without breaking paragraphs.\n",
    "    \n",
    "    Args:\n",
    "        text: str\n",
    "            The text from UnstructuredXMLLoader in line-separated format header, subheader, pargaraphs.\n",
    "        ...\n",
    "        headers_to_split_on: list\n",
    "            A list of tuples of the form (header, metadata_key) where header is a str that\n",
    "            will be used to split the text and metadata_key is the key in the metadata dict\n",
    "            that will be used to store ONLY markdown-formatted header.\n",
    "\n",
    "    Returns:\n",
    "        splits: list\n",
    "            A list of Langchain doc objects, each containing paragraphs and artifact sub-headers according to \n",
    "            chunk size; and 'metadata' key would contain real markdown headers IF any.\n",
    "    \n",
    "    \"\"\"\n",
    "\n",
    "    from langchain_text_splitters import MarkdownHeaderTextSplitter\n",
    "    from langchain_text_splitters import RecursiveCharacterTextSplitter\n",
    "\n",
    "    # split by headers\n",
    "    markdown_splitter = MarkdownHeaderTextSplitter(markdown_headers)\n",
    "    md_header_splits = markdown_splitter.split_text(text)\n",
    "\n",
    "    # filter by exclusion headers\n",
    "    toss = [doc for doc in md_header_splits for exclusion in exclude_headers if exclusion in doc.metadata] \n",
    "    md_filtered_splits = [doc for doc in md_header_splits if doc not in toss]    \n",
    "\n",
    "    if verbose:\n",
    "        print(f\"md_header_splits: {len(md_header_splits)}\")\n",
    "        print(f\"toss: {len(toss)} \\n{toss}\")\n",
    "        print(f\"md_filtered_splits: {len(md_filtered_splits)} \\n{md_filtered_splits}\")\n",
    "\n",
    "    # chunk\n",
    "    text_splitter = RecursiveCharacterTextSplitter(chunk_size=chunk_size, chunk_overlap=chunk_overlap)\n",
    "    splits = text_splitter.split_documents(md_filtered_splits)\n",
    "    \n",
    "    return splits\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 225,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "md_header_splits: 16\n",
      "toss: 0 \n",
      "[]\n",
      "md_filtered_splits: 16 \n",
      "[Document(page_content='BioC-API\\ncollection.key\\nCC BY-NC-SA\\nSplicing role of mitotic regulator kills tumor cells\\nThis article is distributed under the terms of an Attribution-Noncommercial-Share Alike-No Mirror Sites license for the first six months after the publication date (see http://www.rupress.org/terms). After six months it is available under a Creative Commons License (Attribution-Noncommercial-Share Alike 3.0 Unported license, as described at http://creativecommons.org/licenses/by-nc-sa/3.0/).\\nsurname:Wan;given-names:Yihan\\nsurname:Zheng;given-names:Xiaobin\\nsurname:Chen;given-names:Haiyang\\nsurname:Guo;given-names:Yuxuan\\nsurname:Jiang;given-names:Hao\\nsurname:He;given-names:Xiaonan\\nsurname:Zhu;given-names:Xueliang\\nsurname:Zheng;given-names:Yixian\\nTITLE\\nfront\\nSplicing function of mitotic regulators links R-loop-mediated DNA damage to tumor cell killing\\nABSTRACT\\nabstract\\nMitotic regulators BuGZ and Bub3 play a critical role in RNA splicing during interphase, and disruption of this function leads to R-loop formation, DNA damage, and p53 activation.\\nABSTRACT\\nabstract\\nAlthough studies suggest that perturbing mitotic progression leads to DNA damage and p53 activation, which in turn lead to either cell apoptosis or senescence, it remains unclear how mitotic defects trigger p53 activation. We show that BuGZ and Bub3, which are two mitotic regulators localized in the interphase nucleus, interact with the splicing machinery and are required for pre-mRNA splicing. Similar to inhibition of RNA splicing by pladienolide B, depletion of either BuGZ or Bub3 led to increased formation of RNA-DNA hybrids (R-loops), which led to DNA damage and p53 activation in both human tumor cells and primary cells. Thus, R-loop-mediated DNA damage and p53 activation offer a mechanistic explanation for apoptosis of cancer cells and senescence of primary cells upon disruption of the dual-function mitotic regulators. This demonstrates the importance of understanding the full range of functions of mitotic regulators to develop antitumor drugs.'), Document(page_content='title_1\\nIntroduction  \\nparagraph\\nExtensive studies have shown that prolonged mitotic arrest can lead to DNA damage and p53 activation. Although p53 activation in these cells would explain why targeting mitotic regulators could be effective for cancer therapy, how mitotic arrest leads to DNA damage and p53 activation is not fully understood in some contexts. For example, prolonged mitosis is proposed to cause DNA or cellular damage that would in turn activate p53. Supporting this idea, prolonged mitotic arrest has been shown to cause Caspase activation, which could activate CAD (Caspase-activated DNase). Although CAD could lead to DNA damage and p53 activation, how prolonged mitosis activates Caspases is not clear in this context. Additionally, mitotic timer has been suggested to sense the prolonged mitotic arrest in either a p53-dependent or independent manner. While a p53-dependent timer could link prolonged mitotic block to p53 activation, neither the nature of this timer nor the signal that activates p53 has been defined in these settings.  \\nparagraph\\nThe difficulty in defining the mitotic trigger for DNA damage and p53 activation could be because we have not looked at the right stage of the cell cycle. Indeed, several mitotic regulators are found in the interphase nucleus. Therefore, p53 activation could be caused by the disruption of the interphase nuclear functions of these mitotic regulators. Recently, a nuclear zinc finger protein BuGZ has been shown to regulate mitosis by directly binding to the spindle assembly checkpoint protein Bub3 to promote its loading to kinetochores and chromosome alignment. Interestingly, Bub3 is also localized to the interphase nucleus, and the interaction between BuGZ and Bub3 can be detected throughout the cell cycle. As expected, BuGZ depletion in various cancer cell lines resulted in a great reduction in the kinetochore Bub3 levels, chromosome misalignment, and mitotic block. Curiously, upon a prolonged mitotic block, most of the BuGZ-depleted tumor cells undergo mitotic death (mitotic catastrophe). By investigating this mitotic catastrophe phenomenon, we have uncovered an unrecognized interphase nuclear function of BuGZ and Bub3. This interphase function helps to explain why the disruption of the two mitotic regulators could lead to p53 activation.', metadata={'introduction': ''}), Document(page_content='title_1\\nResults and discussion  \\ntitle_2\\nDepletion of BuGZ causes apoptosis in cancer cells and senescence in primary fibroblasts  \\nparagraph\\nPrevious studies have shown that BuGZ depletion in cancer cells destabilizes Bub3 and causes chromosome misalignment and mitotic arrest followed by massive cell death. To further study the function of BuGZ, we used siRNA to deplete the protein in three cancer cell lines (HeLa, HT29, or TOV21G) and the primary human foreskin fibroblasts (HFFs). Consistent with the role of BuGZ in maintaining Bub3 protein level, BuGZ depletion in these cells by 60 h of siRNA treatment led to Bub3 reduction (Fig. 1 A) and an elevation of mitotic index (Fig. S1 A). This shows that BuGZ is required for efficient chromosome alignment in both cancer cells and HFFs, as would be expected based on the Bub3 reduction upon BuGZ depletion.\\nfig1', metadata={'result': ''}), Document(page_content=\"fig_caption\\nReduction of BuGZ causes apoptosis in human cancer cells but senescence in primary foreskin fibroblasts (HFFs). (A) Depletion of BuGZ by siRNA treatment in the indicated cells. Two different amounts of control lysates (1/2 or 1) were loaded. Cells were analyzed 60 h after siRNAs transfection. alpha-tubulin, loading control. The numbers at the bottom of each lane indicate relative levels of BuGZ or Bub3 compared with controls, which were set to 1. Note that >90% of BuGZ was depleted by RNAi. (B) FACS assays for the quantification of apoptotic cells at 72 h after RNAi. Early apoptotic cells were Annexin V positive (see the bottom right quadrants), whereas late apoptotic ones were Annexin V and 7-aminoactinomycin D (7-AAD) double positive (the top right quadrants). All values are percentages of total cells, presented as mean +- SD from three independent experiments. (C) Representative microscopic fields of cells at 72 h after RNAi. Bar, 100 microm. (D) Growth curves of cells treated with control or BuGZ siRNA. Error bars indicate SD. Student's t test: n.s., not significant; *, P < 0.05; **, P < 0.01; ***, P < 0.001 from three independent experiments. (E) Stable knockdown of BuGZ with lentivirus-expressed shRNA in HFFs. The virus-infected cells were enriched by puromycin selection. The cell lysates were collected at day 5. (F) Growth curves of HFFs at the indicated days after shRNA transfection. Error bars indicate SD. Student's t test: **, P < 0.01; ***, P < 0.001 from three independent experiments. (G) Increased senescence in BuGZ shRNA-transfected HFFs as judged by SA-beta-gal staining (arrowheads). Bar, 50 microm. (H) Percentage of SA-beta-gal-positive cells at the indicated days after shRNA transfection. Error bars indicate SEM. Student's t test: ***, P < 0.001 from three independent experiments.\", metadata={'figure': ''}), Document(page_content='paragraph\\nBy 5 d of the siRNA treatment, all three cancer cell lines exhibited a large increase in apoptosis as judged by the early and late apoptotic markers Annexin V and 7-AAD, respectively (Fig. 1 B), which correlated with a striking reduction in cell numbers (Fig. 1, C and D). In contrast, BuGZ siRNA treatment of HFFs did not affect cell survival (Fig. S1 B), which is similar to the observation in primary neural stem cells, despite a moderate elevation in mitotic index (Fig. S1 A). Nevertheless, because primary somatic cells usually grow more slowly than tumor cells, we reasoned that a time >5 d might be required for properly assessing the effect of BuGZ depletion on HFFs. We thus used lentivirus-mediated RNAi to stably knock down BuGZ in HFFs (Fig. 1 E). We found that, from day 8 after infection, the proliferation rate of the BuGZ-RNAi HFFs began to significantly slow down (Fig. 1 F). Moreover, a threefold increase of senescent cells was observed at 30 d compared with controls (Fig. 1, G and H), which indicates accelerated senescence of BuGZ-depleted HFFs. Thus, BuGZ depletion causes apoptosis in cancer cells and senescence in primary HFFs.  \\ntitle_2\\nDepletion of BuGZ leads to p53 activation  \\nparagraph\\nSince p53 accumulation is known to cause either apoptosis or senescence depending on the cellular context, we asked whether BuGZ depletion led to p53 induction in cancer cells and HFFs. HeLa and TOV21G cells have wild-type p53. Although in HeLa cells p53 protein is targeted for degradation by the human papillomavirus E6 proteins under normal conditions, its accumulation has been reported upon DNA damage or oxidation. HT29 cells have a point mutation in the transactivation domain of p53 (R273H), but studies have shown that this mutant p53 induces apoptosis or cell cycle arrest upon various challenges. Activation of p53 in primary cells with intact cell cycle checkpoints has been shown to cause senescence. Consistently, we found that the levels of total p53 and activated p53 (phospho-p53 at serine 15) were increased upon BuGZ depletion in both the cancer cells tested and HFFs (Fig. 2, A and B).\\nfig2', metadata={'result': ''}), Document(page_content=\"fig_caption\\nActivation of p53 upon BuGZ depletion leads to apoptosis or senescence. (A) Up-regulation of p53 after 48 h of BuGZ siRNA treatment. Loading control, alpha-tubulin. (B) Increased phospho-p53 (Ser15) in the nuclei of BuGZ-depleted cells (arrowheads). BuGZ-positive (arrows) and phospho-p53 negative cells (arrows and white broken circles) are indicated. Cells were assayed 48 h after siRNA treatment. Bars, 10 microm. (C) Inhibition of p53 by PFT-alpha increased the numbers of viable BuGZ-depleted cells. Error bars indicate SD. Student's t test: **, P < 0.01 from three independent experiments. (D-F) p53 RNAi in HFFs repressed senescence induced by BuGZ knockdown. HFFs were infected with lentivirus to express the indicated shRNAs. Western blotting (D) and quantification (E) of SA-beta-gal staining (F) were performed at 30 d after infection. Arrowheads in F represent the SA-beta-gal-positive staining cells. Bar, 100 microm. Error bars indicate SEM. Student's t test: *, P < 0.05; **, P < 0.01 from three independent experiments.\", metadata={'figure': ''}), Document(page_content='paragraph\\nOur previous studies indicated that many BuGZ-depleted HeLa cells initially arrested in mitosis and then underwent cell death. To test whether p53 activation could be the cause of cancer cell death, we treated cells with p53 inhibitor PFT-alpha 12 h after transfection of control or BuGZ siRNA and found that the inhibition of p53 significantly rescued the cell number reduction of the BuGZ-depleted cancer cells (Fig. 2 C). When HFFs were coinfected with lentivirus to express p53 shRNA, senescence induced by BuGZ depletion upon prolonged culturing was also significantly inhibited (Fig. 2, D-F). Because depletion of BuGZ down-regulated Bub3 (Fig. 1 A), we investigated whether Bub3 reduction could also lead to p53 activation. We found that knockdown of Bub3 by RNAi caused prominent p53 accumulation and activation in both cancer cells and HFFs (Fig. S1, C-F). These findings show that p53 mediates both apoptosis in cancer cells and senescence in HFFs upon BuGZ depletion.  \\ntitle_2\\nBuGZ and Bub3 are required for proper pre-mRNA splicing  \\nparagraph\\nNext we explored how the depletion of BuGZ or Bub3 could activate p53. Because BuGZ and Bub3 bind to one another and are both present in the interphase nucleus, their depletion could disrupt an interphase nuclear function, which in turn might trigger p53 activation. To investigate this, we examined our previous proteome analyses of BuGZ-interacting proteins. Besides Bub3, we noticed several spliceosome components among the top candidates of BuGZ-interacting proteins (Fig. 3 A). Among a number of available antibodies to spliceosome components, we found that the antibodies to two:U2AF65 and SF3a3:worked in both immunoprecipitation and Western blotting, which allowed us to use reciprocal immunoprecipitation to provide further support for an interaction between BuGZ/Bub3 and the spliceosome (Fig. 3, B and C). Consistently, both BuGZ and Bub3 have also been found in previous proteomic analyses of spliceosomes.\\nfig3', metadata={'result': ''}), Document(page_content='fig_caption\\nA requirement of BuGZ and Bub3 for RNA splicing. (A) Proteins with known functions in RNA splicing (highlighted in orange) were found as candidate BuGZ-interacting proteins. (B) FLAG-tagged BuGZ or Bub3, but not FLAG-luciferase (Luci.), immunoprecipitated the spliceosome components SF3a3 and U2AF65 in HEK293T cells. (C) BuGZ and Bub3 interacted with SF3a3 and U2AF65 in vivo. Reciprocal immunoprecipitations of endogenous proteins were performed using HT29 cell lysates with the indicated mouse monoclonal (mAb) or rabbit antibodies (rAb). h.c., antibody heavy chain. Experiments were performed three times. (D) Analyses of splicing defects by RNA-sequencing of HFFs and TOV21G cells treated by the splicing inhibitor pladienolide B or RNAi of BuGZ or Bub3. The numbers of genes exhibiting the depicted splicing defects are shown as pie charts. RNA-Seq was from two biological repeats. (E) Two representative genes, BRD2 and WAC, exhibiting intron retention upon treatment with pladienolide B or RNAi of BuGZ or Bub3. The red boxes highlight examples of intron retention. The fold increases of the intron retention are plotted to the right. The p-values were generated by simulation (MATS: a Bayesian framework for flexible detection of differential alternative splicing from RNA-Seq data).', metadata={'figure': ''}), Document(page_content='paragraph\\nTo test whether BuGZ and Bub3 play roles in splicing, we depleted each protein in cancer cells (TOV21G) or HFFs and performed RNA sequencing (accession number: GSE67770). Cells treated with pladienolide B, a potent splicing inhibitor, were used as positive controls. Among the splicing defects depicted in Fig. 3 D, we found that the depletion of either Bub3 or BuGZ mainly caused exon skipping, whereas pladienolide B treatment mainly resulted in intron retention (Fig. 3, D and E; and Table S1). The difference in the resultant splicing defects between Pladienolide B inhibition (which targets SF3b3) and BuGZ or Bub3 depletion suggests that BuGZ and Bub3 function differentially from SF3b3. Whereas further studies are required to understand how BuGZ and Bub3 regulate splicing, these analyses show that the well-established SAC component Bub3 and its regulator BuGZ also play roles in interphase RNA splicing. Depletion of BuGZ in transformed cells or treatment of tumor cells with pladienolide B have been shown to kill these cells in vitro. Interestingly, using the Drosophila melanogaster eye tumor model, we found that the eye hyperproliferation induced by the activation of either the Jak-Stat or the Hippo (unpublished data) pathway was inhibited by the eye-specific BuGZ RNAi or by feeding the flies with pladienolide B (Fig. S2 A).  \\ntitle_2\\nDepletion of BuGZ or Bub3 causes R-loop formation and DNA damage  \\nparagraph\\nDeficiency in RNA splicing has been shown to cause an increased formation of RNA-DNA hybrids (R-loops) in the nucleus, and subsequent DNA breaks and p53 activation. Therefore, p53-mediated killing of transformed cells or senescence of primary cells as we observed in the above studies could be caused by defects in splicing and the subsequent R-loop formation. To determine whether cells depleted of BuGZ or Bub3 had an increased R-loop formation, we used the monoclonal antibody S9.6, which recognizes RNA-DNA hybrids. Similar to the pladienolide B treatment (Fig. 4 A), depletion of BuGZ or Bub3 markedly stimulated R-loop formation in both cancer cells and HFFs (Fig. 4 A and Fig. S2, B and C). These cells also exhibited increased DNA damage, as judged by the appearance of prominent nuclear foci of gammaH2AX (; Fig. 4, B-F; Fig. S2, D-H; and Fig. S3 A). These results suggest that defects in the splicing function of BuGZ and Bub3 lead to R-loop-induced DNA damage, which in turn contributes to p53 activation.\\nfig4', metadata={'result': ''}), Document(page_content=\"fig_caption\\nDepletion of BuGZ or Bub3 leads to increased R-loop formation and DNA damage. (A) Depletion of BuGZ or Bub3 resulted in increased R-loop formation in HT29 cells. Pladienolide B treatment was used as a positive control. White broken circles outline the nuclei with no or low levels of R-loops. Bar, 5 microm. The quantification of mean fluorescence intensity of the R-loop is shown on the right. Student's t test: ***, P < 0.001 from three independent experiments. (B and C) Depletion of BuGZ (B) or Bub3 (C) by two different siRNAs in HT29 cells caused increased DNA damage, as judged by the presence of prominent nuclear foci of gamma-H2Ax. BuGZ- or Bub3-depleted cells (arrowheads) exhibit more gamma-H2Ax foci than cells with insufficient or no depletion (arrows). White broken circles outline the nuclei. Bars, 10 microm. (D-F) Quantification of gamma-H2Ax intensities in individual cells (D) and distributions of gamma-H2Ax intensities (E and F). Error bars indicate SEM. Student's t test: ***, P < 0.001. Cell numbers (n) analyzed were from three independent experiments.\", metadata={'figure': ''}), Document(page_content='title_2\\nInhibition of R-loop formation attenuates p53 activation and DNA damage caused by depletion of BuGZ or Bub3  \\nparagraph\\nTo confirm that the R-loops formed in cells depleted of BuGZ or Bub3 indeed contributed to DNA damage and p53 activation, we used a modified form of RNase H1 lacking the mitochondria targeting sequence. This form of RNase H1 has been shown to localize only in the nucleus and to resolve R-loops. By analyzing the nuclei that were positive for the GFP-tagged RNase H1 or GFP control, we found that expression of the enzyme resulted in significant reduction of both gammaH2AX foci (Fig. 5, A-C) and p53 activation (Fig. 5, D-F) in BuGZ-depleted HT29 cells. A similar effect of RNase H1 was also observed in pladienolide B-treated (Fig. S3 B) or Bub3-depleted (Fig. 5, G-J) HT29 cells. The inability of RNase H1 to fully rescue DNA damage and p53 activation could be caused by the heterogeneous level of GFP-RNase H1 expression, the inefficiency of GFP-RNase H1 to resolve certain R-loops, or other uncharacterized defects. Nonetheless, these findings show that R-loop formation due to the disruption of the splicing function of Bub3 and BuGZ (Fig. S3 C) contributes to DNA damage and p53 activation.\\nfig5', metadata={'result': ''}), Document(page_content=\"fig_caption\\nExpression of RNase H1 in BuGZ- or Bub3-depleted HT29 cells reduces gamma-H2Ax and phospho-p53 signals. (A) GFP-RNase H1 (RNase), but not GFP, repressed BuGZ depletion-induced DNA damage in HT29 cells. Cells were first transfected by siRNAs for 12 h and then by GFP-RNase H1 or GFP plasmid for 48 h followed by immunofluorescence microscopy analyses. BuGZ-depleted cells with or without GFP fluorescence are indicated by arrowheads or arrows, respectively. The broken lines indicate nuclear boundaries. Bar, 10 microm. (B and C) Quantifications for the mean fluorescence intensity of gamma-H2Ax (B) and the intensity distributions (C) of the experiments shown in A. (D) GFP-RNase H1 repressed BuGZ depletion-induced p53 phosphorylation in HT29 cells. Arrowheads and arrows indicate cells with or without GFP-RNase H1, respectively. White broken circles outline the nuclei. (E and F) Quantifications for the mean fluorescence intensity of phospho-p53 (E) and the intensity distributions (F) of the experiments shown in D. (G and H) Quantifications for the mean fluorescence intensity of gamma-H2Ax (G) and the intensity distributions (H) in Bub3-depleted and GFP- or GFP-RNase H1-expressing HT29 cells. (I and J) Quantifications for the mean fluorescence intensity of phospho-p53 (I) and the intensity distributions (J) in Bub3-depleted and GFP- or GFP-RNase H1-expressing HT29 cells. Error bars indicate SEM. Student's t test: *, P < 0.05; **, P < 0.01; ***, P < 0.001. Cell numbers (n) analyzed were from three independent experiments.\", metadata={'figure': ''}), Document(page_content='paragraph\\nExtensive efforts have been devoted to understanding how mitotic arrest as a result of disrupting mitotic regulators can lead to p53-dependent cell death or cell cycle arrest in cancer cells. By exploring the mitosis-independent functions, we have uncovered a critical role of the mitotic regulators BuGZ and Bub3 in interphase RNA splicing. Importantly, we demonstrated that disruption of this interphase function by depleting BuGZ or Bub3 leads to R-loop formation, DNA damage, and p53 activation. Furthermore, our results suggest that the splicing inhibitor pladienolide B, a potent anticancer reagent, kills cancer cells through a similar mechanism. Our findings demonstrate that by looking beyond the mitosis functions of the established mitotic regulators, we could gain significant insights into how aberrant mitosis is connected to the suppression of cell proliferation and the safeguard of genome stability. Therefore, a broadened understanding of the function of mitotic regulators beyond cell division should facilitate the selection of therapeutic targets in the effort of developing new antitumor drugs for cancer chemotherapy.', metadata={'result': ''}), Document(page_content=\"title_1\\nMaterials and methods  \\nparagraph\\nFor the sources and sequences of mammalian expression plasmids, antibodies (with dilutions used), siRNA, shRNA, small molecules (with concentrations used), and cell lines, please refer to Table S2. The plasmid for expressing GFP-RNase H1 was provided by R.J. Crouch (National Institute of Child Health and Human Development, Bethesda, MD).  \\ntitle_2\\nCell culture, plasmids, and siRNA transfection  \\nparagraph\\nFor TOV21G cell (ATCC) culture, the base medium was a 1:1 mixture of MCDB 105 medium (Sigma-Aldrich) containing a final concentration of 1.5 g/liter sodium bicarbonate and Medium 199 (Sigma-Aldrich) containing a final concentration of 2.2 g/liter sodium bicarbonate. To make the complete growth medium, fetal bovine serum (Gibco) was added to a final concentration of 15%. HT29 (HTB-38; ATCC) and HeLa (CCL-2; ATCC) cells were cultured in Dulbecco's modified Eagle's medium (Gibco) containing 10% fetal bovine serum (Gibco) in 5% CO2 at 37 C. The HFF cells (Millipore) were cultured in FibroGRO LS (low serum) Complete Medium (Millipore). HFFs were changed into fresh medium every other day and passaged using Accutase (Millipore) at a 1:3 dilution when reaching ~80% of confluency.  \\nparagraph\\nThe plasmids or siRNA transfections were performed using Lipofectamine 2000 (Life Technologies) or Lipofectamine RNAiMAX (Life Technologies), respectively. Cells were fixed for immunofluorescence analyses 48 h after siRNA transfection unless otherwise indicated. For cotransfection of siRNA and plasmids, siRNA was transfected first using the reverse transfection protocol. In brief, 10 pmol siRNA oligo was diluted in 50 microl Opti-MEM I Medium (Gibco) per well of a 24-well tissue culture plate. 1.5 microl of Lipofectamine RNAiMAX was added to each well. After mixing, the plate was incubated for 20 min at room temperature. Meanwhile, cells to be transfected were trypsinized and resuspended in complete growth medium without antibiotics. Cell suspension containing 20,000 cells was added into each well for transfection. After 24 h of transfection, cells were transfected with the indicated plasmids. For immunofluorescence analyses in Fig. 5, cells were fixed 48 h after plasmid transfection. For mitotic index calculation in Fig. S1, both tumor cells and primary cells were fixed at 60 h after RNAi followed by immunostaining. For FACS analysis of apoptotic cells in Fig. 1 and Fig. S1, both tumor cells and primary cells were analyzed 72 h after siRNA transfection.  \\ntitle_2\\nFACS analyses of apoptotic cells  \\nparagraph\\nFITC Annexin V Apoptosis Detection kit with 7-AAD (640922; BioLegend) was used to detect apoptosis. 5 x 105 cells that were treated by control or BuGZ siRNAs (Fig. 1 B) were trypsinized and resuspended as individual cells in 100 microl of Annexin V-binding buffer (140 mM NaCl, 4 mM KCl, 0.75 mM MgCl2, and 10 mM Hepes, pH 7.4). 5 microl FITC-Annexin V and 5 microl 7-AAD were added to each cell suspension. After incubation for 15 min at room temperature, cells were analyzed with FACS Aria III. For all samples, 100,000 events were recorded. FACSDIVA software (BD) was used for all flow cytometry analyses. Samples with buffer alone (background autofluorescence control), FITC-Annexin V, or 7-AAD staining alone (compensation controls for each fluorochrome) were used for compensation and gating for the apoptotic-positive cells. PE-Cy5 channel was used to detect 7-AAD fluorescence.  \\ntitle_2\\nImmunofluorescence microscopy and quantifications  \\nparagraph\\nTo detect DNA-RNA hybrids (R-loops) using the monoclonal antibody S9.6 (provided by L. Wahba and D. Koshland, University of California, Berkeley, Berkeley, CA), we found that the treatment used for mitotic chromosome spread improves the detection of R-loops in the nuclei of HT29 cells, whereas standard immunostaining was sufficient for HFFs. We modified the chromosome-spread method to treat the interphase HT29 cells. Control or siRNA-treated HT29 cells were trypsinized and resuspended as single cell suspensions and then treated with hypotonic solution (75 mM KCl) prewarmed to 37 C. The cells were then fixed with freshly made fixative solution (methanol/glacial acetic acid 3:1) overnight at 4 C. To spread the cells on a coverslip, the fixed cell suspension was placed on the coverslip exposed to the steam of hot water (90 C) for 30 s, which caused the cells to spread their chromatin on the coverslip. These cells were blocked by blocking buffer (5% BSA and 0.5% Triton X-100 in PBS) for 1 h and then incubated with 1:1,000 dilution of the S9.6 antibody overnight in blocking buffer. After three washes with blocking buffer, the cells were incubated with 1:1,000 dilution of secondary antibody for 1 h. For detection of R-loops in HFFs, cells were fixed using ice-cold methanol for 15 min followed by treatment with 70% ethanol for 15 min. Cells were washed three times with 70% ethanol for 5 min each followed by blocking and antibody staining as above.  \\nparagraph\\nFor immunodetection of alpha-tubulin, gamma-H2Ax, phospho-p53, BuGZ, and Bub3, cells grown on coverslips were fixed in 4% paraformaldehyde for 15 min and then permeabilized with 0.5% Triton X-100 in PBS for 5 min. The source and dilutions of the primary and secondary antibodies can be found in the Table S2.  \\nparagraph\\nTo co-stain phospho-p53 and Bub3 with two mouse monoclonal antibodies (Table S2), the antibodies were directly coupled with Alexa Fluor 546 or Alexa Fluor 647, respectively, by using an APEX Alexa Fluor antibody labeling kit (A20183 and A20186; Life Technologies).  \\nparagraph\\nTo detect GFP or GFP fusion proteins, cells were fixed in the same manner and imaged directly. After antibody staining and washes, the cells were stained with DAPI at 0.25 microg/ml in PBS for 5 min and mounted in Fluorescence Mounting medium (Dako).  \\nparagraph\\nFluorescence images were captured using a microscope (ECLIPSE TE2000-U; Nikon) driven by MetaMorph software (Molecular Devices) or a laser confocal microscope (TCS SP5; Leica) at the same setting throughout each given experiment. For confocal microscopy, 63x 1.4 NA oil and 40x 1.25 NA oil objective lenses (Leica) were used and the cells were imaged by scanning optical sections at ~0.5-microm intervals.  \\nparagraph\\nTo quantify the mitotic index, the number of cells in prophase through telophase (judged by alpha-tubulin and DAPI staining) was divided by the total number of cells analyzed.  \\nparagraph\\nTo quantify nuclear immunofluorescence intensity from confocal images, maximal projection was created from the z stacks for each nucleus. We used a method based on a previous study to determine the nuclear fluorescence signal from both confocal images and images from epifluorescence microscopes. In brief, the nuclear regions (X) were defined by DAPI staining. A larger area surrounding the nucleus was then drawn (Y). Areas of X and Y were determined as Xa and Ya. The mean fluorescence intensity for X and Y were measured as Xf and Yf. Signal intensity for each nucleus (Xi) was determined as Xi = Xf - (Yf x Ya - Xf x Xa)/(Ya - Xa).  \\nparagraph\\nAll experiments were performed at least three times. A total of 17-159 nuclei were quantified from these experiments and plotted.  \\ntitle_2\\nSenescence-associated (SA)-beta-gal assay  \\nparagraph\\nWe used the Senescence beta-Galactosidase Staining kit (9860S; Cell Signaling) to detect beta-galactosidase activity at pH 6.0. In brief, cells were fixed in the fixative solution for 10-15 min at room temperature. The fixed cells were then incubated with beta-galactosidase staining solution containing X-gal at 37 C overnight. As the blue color developed, bright-field cell images were taken using an Axiovert 25 microscope (Carl Zeiss) connected to a Canon camera.  \\ntitle_2\\nLive cell number count  \\nparagraph\\nA 0.4% Trypan blue stock solution (15250061; Life Technologies) was used to stain dead cells. 10 microl of Trypan blue stock solution was added to 100 microl of cell suspension. Cells negative for Trypan blue staining were counted as live cells using hemocytometer.  \\ntitle_2\\nImmunoprecipitation and Western blotting analyses  \\nparagraph\\nTo confirm the interaction between BuGZ or Bub3 and spliceosome components, we used two methods. In the first method, we transfected HEK293T cells with Flag-BuGZ, Flag-Bub3, or Flag-luciferase (negative control). After transfection, ~1 x 107 cells were lysed in 1 ml lysis buffer (20 mM Tris-Cl, pH 7.4, 150 mM KCl, 1 mM EDTA, 1 mM NaF, 1 mM DTT, 1% NP-40, 10% glycerol, and 1 mM sodium pyrophosphate) containing protease inhibitors (at 1 tablet/10 ml; cOmplete, Mini, EDTA-free, Protease Inhibitor Cocktail Tablets, 11836170001; Roche) at 4 C. After 10 min of centrifugation at 13,000 rpm in an Eppendorf tabletop centrifuge, the supernatant containing soluble proteins was incubated with a 30-microl slurry of anti-Flag M2 resin (A2220; Sigma-Aldrich) at 4 C for 4 h. After washing the beads with wash buffer (50 mM Tris-Cl, pH 7.4, 150 mM NaCl, 1 mM EDTA, 50 mM NaF, 1 mM DTT, and 1% NP-40) three times, the bound proteins were eluted with 100 microl Flag-peptide (F4799; Sigma-Aldrich) dissolved in the wash buffer at a concentration of 0.5 mg/ml.  \\nparagraph\\nIn the second method, we used antibodies to BuGZ (HPA017013; Sigma-Aldrich), Bub3 (611731; BD), U2AF65 (ab37530; Abcam), Sf3a3 (ab156873; Abcam), or control IgG (12-370, 12-371; Millipore) to immunoprecipitate the respective endogenous proteins from HT29 cell lysates made from ~1 x 107 cells for each immunoprecipitation. Each antibody was preincubated with a 50-microl slurry of protein G Dynabeads (10007D; Life Technologies) for 30 min followed by washes with lysis buffer. The beads were added to the cell lysate and incubated overnight. The beads were then washed as described in the previous paragraph, and bound proteins were eluted with 100 microl of elution buffer (50 mM glycine at pH 2.5) and dissolved in the SDS sample buffer.  \\nparagraph\\nFor Western blotting analyses of the Flag antibody immunoprecipitated proteins, 1% of input lysate and 10% of resuspended immunoprecipitates were loaded per lane for SDS-PAGE. Because the molecular weights of the proteins of BuGZ, U2AF65, and Sf3a3 are close to the heavy chain of IgG, we reduced the signal from the antibody heavy chain by using the secondary antibody specific to the rabbit light chain (211-032-171; Jackson ImmunoResearch Laboratories, Inc.). Flag (F3165; Sigma-Aldrich), BuGZ (HPA017013; Sigma-Aldrich), Bub3 (611731; BD), U2AF65 (ab37530; Abcam), or Sf3a3 (ab156873; Abcam) antibodies were used at the indicated dilutions (Table S2).  \\ntitle_2\\nWhole-transcriptome shotgun sequencing (RNA-Seq) and data analyses  \\nparagraph\\nArcturus PicoPure RNA isolation kit (KIT0204; Life Technologies) was used to prepare total RNA from cells. Ribo-Zero Gold kits (human/mouse/rat; MRZG126; Epicentre) were used to remove both cytoplasmic (nuclear-encoded) rRNA and mitochondrial rRNA from total RNA preparations, and libraries were built using the Illumina TruSeq RNA sample prep kit V2 (RS-122-2001; Illumina). Libraries were sequenced using an Illumina HiSeq 2000. Single-end 100-bp reads were obtained.  \\nparagraph\\nRNA-Seq reads were first mapped to the human genome (hg19) using Tophat2. Transcript annotation from Ensembl was used as reference for exon junctions in read mapping. We then used Cufflinks to assemble unannotated transcripts. Assembled transcripts from the same cell type were further merged by Cuffmerge. Finally, we used MATS to call differential alternative splicing, and the merged transcripts from the previous step were used as transcript annotation in MATS.  \\ntitle_2\\nDrosophila stocks  \\nparagraph\\nThe w1118 allele (Bloomington Drosophila Stock Center no. 3605) was used as the wild-type control. The RNAi strains for BuGZ were obtained from the Vienna Drosophila RNAi library: w1118; P{GD9485}v45449 (Vienna Drosophila Resource Center no. v45449). dsRNA-GD9485 (2L:16,744,705 ... 16,745,060) was inserted in the third chromosome.  \\nparagraph\\nThe UAS strains used were: UAS-upd transgenic fly generated through transformation of UAS-upd construct into W1118 flies by standard methods (provided by X. Chen, Johns Hopkins University, Baltimore, MD) and UAS-ykiS111A.S168A.S250A.V5 (Bloomington Drosophila Stock Center no. 28817). The Gal4 allele used to drive the UAS lines is ey-Gal4 (Bloomington Drosophila Stock Center no. 5534), which is expressed in the Drosophila eyes.  \\ntitle_2\\nDrosophila genetics and culture  \\nparagraph\\nIf not specifically mentioned, all the flies were maintained on standard cornmeal/molasses/yeast fly food (the recipe for 1 liter of food: 50 g cornmeal, 18.75 g yeast, 5 g agar, 62.5 ml molasses, 11.5 ml 5% anti-fungal agent Tegosept, and 35 ml propionic acid). Flies were cultured at 25 C on a normal light/dark cycle condition, unless otherwise indicated.  \\nparagraph\\nTo induce the tumor formation in the Drosophila eyes, ey-Gal4 was used to drive oncogene expression specifically in the eye. UAS-upd and UAS-ykiS111A.S168A.S250A.V5 transgenic flies were used to generate eye tumors.  \\nparagraph\\nTo investigate the tumor repression effect of BuGZ RNAi, we used ey-Gal4 to drive both UAS-upd (or UAS-ykiS111A.S168A.S250A.V5) and UAS-BuGZ RNAi. The crosses were set up at 25 C, and eye tumor formation was examined after eclosion.  \\nparagraph\\nTo evaluate the tumor repression effect of the splicing inhibitor pladienolide B, ey-Gal4>UAS-upd (or UAS-ykiS111A.S168A.S250A.V5) flies were fed with DMSO/pladienolide B (5 microM) from the first instar larval stage. Eye tumor formation was examined after eclosion.  \\nparagraph\\nTo quantify the area of each Drosophila eye, bright-field images of the eyes were taken by placing the fly's head dorsal-side-up under a stereomicroscope (SMZ1500; Nikon) with a camera (AxioCam HRC; Carl Zeiss). This allows us to visualize the contour and half of the total area of the fly eyes. The rough eyes have a rough contour and a larger area than normal eyes, which are quantified by measuring the pixel number in each eye area using Photoshop CS5 (Adobe).  \\nparagraph\\nTo investigate the RNAi knockdown efficiency of the UAS-BuGZ RNAi allele, tub-Gal4 (Bloomington Drosophila Stock Center no. 5138) was used to drive the ubiquitous expression of UAS-BuGZ RNAi, with the UAS-LacZ RNAi allele as the control. Total RNAs were collected at either the third instar larval stage or pupal stage, and qRT-PCR was performed to examine the mRNA level of BuGZ.  \\ntitle_2\\nScanning EM  \\nparagraph\\nDrosophila heads were excised and placed in McDowell's and Trump's 4F:1G fixative for 1 h to several weeks at 4 C. After two rinses in 0.1 M sodium phosphate buffer (pH 7.2), samples were dehydrated in a graded ethanol series to 100% ethanol, at which time they were critical point-dried with liquid CO2. Finally, they were mounted on specimen stubs with colloidal silver, sputter-coated with gold-palladium, and examined with a scanning electron microscope (Quanta 200 Environmental Scanning Electron Microscope; FEI).  \\ntitle_2\\nQuantitative real-time PCR  \\nparagraph\\nTotal RNA was isolated from Drosophila larvae or pupae using an RNA isolation kit (RNeasy Plus Mini kit; QIAGEN) and the manufacturer's protocol. Quantitative RT-PCR was performed using the iScript One Step RT-PCR kit (170-8892; Bio-Rad Laboratories) on a real-time PCR detection system (CFX96; Bio-Rad Laboratories). 50 ng of total RNA was reverse transcribed and amplified as follows: 50 C for 10 min, 95 C for 5 min, 95 C for 10 s, 60 C for 30 s, 72 C for 1 min. Steps 2-4 were repeated for 40 cycles. Each reaction was performed in triplicate, and the results of three independent experiments were used for statistical analysis. Relative mRNA expression levels were quantified using the DeltaDeltaC(t) method. Results were normalized to those for RP49, and primer sequences were as follows. Rp49 (Drosophila): forward primer, TACAGGCCCAAGATCGTGAA; and reverse primer, TCTCCTTGCGCTTCTTGGA. BuGZ (Drosophila): forward primer, AAGCCACCTTAGGATCATGG; and reverse primer, GGGTCCGGTGTACAACTTCT.  \\ntitle_2\\nOnline supplemental material  \\nparagraph\\nFig. S1 shows the effect of BuGZ reduction on the mitotic index and cell proliferation. Fig. S2 shows the tumor repression effect of BuGZ RNAi or pladienolide B treatment. Fig. S3 shows that DNA damage caused by splicing inhibitor treatment can be rescued by forced expression of RNase H1. Table S1 gives a list of differential alternative splicing events caused by BuGZ RNAi, Bub3 RNAi, or pladienolide B treatment. Table S2 gives the sources and sequences of mammalian expression plasmids, antibodies (with dilutions used), siRNA, shRNA, small molecules (with concentrations used), and cell lines. Online supplemental material is available at http://www.jcb.org/cgi/content/full/jcb.201409073/DC1. Additional data are available in the JCB DataViewer at http://dx.doi.org/10.1083/jcb.201409073.dv.\", metadata={'methods': ''}), Document(page_content='title_1\\nSupplementary Material  \\nfootnote_title\\nAbbreviations used in this paper:  \\nfootnote\\nHFF  \\nfootnote\\nhuman foreskin fibroblast  \\nfootnote\\nSA-beta-gal  \\nfootnote\\nsenescence-associated beta-gal\\nsurname:Aguilera;given-names:A.\\nsurname:Garcia-Muse;given-names:T.', metadata={'supplement': ''}), Document(page_content=\"Mol. Cell.\\nref\\nR loops: from transcription byproducts to threats to genome stability\\nsurname:Blagosklonny;given-names:M.V.  \\nCell Cycle.\\nref\\nProlonged mitosis versus tetraploid checkpoint: how p53 measures the duration of mitosis\\nsurname:Campisi;given-names:J.\\nsurname:d'Adda di Fagagna;given-names:F.  \\nNat. Rev. Mol. Cell Biol.\\nref\\nCellular senescence: when bad things happen to good cells\\nsurname:Campos;given-names:P.B.\\nsurname:Sartore;given-names:R.C.\\nsurname:Abdalla;given-names:S.N.\\nsurname:Rehen;given-names:S.K.  \\nJ. Vis. Exp.\\nref\\nChromosomal spread preparation of human embryonic stem cells for karyotyping\\nsurname:Cerritelli;given-names:S.M.\\nsurname:Frolova;given-names:E.G.\\nsurname:Feng;given-names:C.\\nsurname:Grinberg;given-names:A.\\nsurname:Love;given-names:P.E.\\nsurname:Crouch;given-names:R.J.  \\nMol. Cell.\\nref\\nFailure to produce mitochondrial DNA results in embryonic lethality in Rnaseh1 null mice\\nsurname:Chen;given-names:Q.M.\\nsurname:Merrett;given-names:J.B.\\nsurname:Dilley;given-names:T.\\nsurname:Purdom;given-names:S.  \\nOncogene.\\nref\\nDown regulation of p53 with HPV E6 delays and modifies cell death in oxidant response of human diploid fibroblasts: an apoptosis-like cell death associated with mitosis\\nsurname:Drolet;given-names:M.\\nsurname:Phoenix;given-names:P.\\nsurname:Menzel;given-names:R.\\nsurname:Masse;given-names:E.\\nsurname:Liu;given-names:L.F.\\nsurname:Crouch;given-names:R.J.  \\nProc. Natl. Acad. Sci. USA.\\nref\\nOverexpression of RNase H partially complements the growth defect of an Escherichia coli delta topA mutant: R-loop formation is a major problem in the absence of DNA topoisomerase I\\nsurname:Effenberger;given-names:K.A.\\nsurname:Perriman;given-names:R.J.\\nsurname:Bray;given-names:W.M.\\nsurname:Lokey;given-names:R.S.\\nsurname:Ares;given-names:M.;suffix:Jr\\nsurname:Jurica;given-names:M.S.  \\nJ. Biomol. Screen.\\nref\\nA high-throughput splicing assay identifies new classes of inhibitors of human and yeast spliceosomes\\nsurname:Eskens;given-names:F.A.\\nsurname:Ramos;given-names:F.J.\\nsurname:Tabernero;given-names:J.\\nsurname:Burger;given-names:H.\\nsurname:O'Brien;given-names:J.P.\\nsurname:Piera;given-names:A.\\nsurname:de Jonge;given-names:M.J.\\nsurname:Mizui;given-names:Y.\\nsurname:Wiemer;given-names:E.A.\\nsurname:Carreras;given-names:M.J.\\nsurname:Baselga;given-names:J.  \\nClin. Cancer Res.\\nref\\nDatabase\\nsurname:Forbes;given-names:S.A.\\nsurname:Bindal;given-names:N.\\nsurname:Bamford;given-names:S.\\nsurname:Cole;given-names:C.\\nsurname:Kok;given-names:C.Y.\\nsurname:Beare;given-names:D.\\nsurname:Jia;given-names:M.\\nsurname:Shepherd;given-names:R.\\nsurname:Leung;given-names:K.\\nsurname:Menzies;given-names:A.  \\nNucleic Acids Res.\\nref\\nsurname:Gan;given-names:W.\\nsurname:Guan;given-names:Z.\\nsurname:Liu;given-names:J.\\nsurname:Gui;given-names:T.\\nsurname:Shen;given-names:K.\\nsurname:Manley;given-names:J.L.\\nsurname:Li;given-names:X.  \\nGenes Dev.\\nref\\nR-loop-mediated genomic instability is caused by impairment of replication fork progression\\nsurname:Ganem;given-names:N.J.\\nsurname:Pellman;given-names:D.  \\nJ. Cell Biol.\\nref\\nLinking abnormal mitosis to the acquisition of DNA damage\\nsurname:Gascoigne;given-names:K.E.\\nsurname:Taylor;given-names:S.S.  \\nCancer Cell.\\nref\\nCancer cells display profound intra- and interline variation following prolonged exposure to antimitotic drugs\\nsurname:Gonzalez;given-names:C.  \\nNat. Rev. Cancer.\\nref\\nDrosophila melanogaster: a model and a tool to investigate malignancy and identify new therapeutics\\nsurname:Groh;given-names:M.\\nsurname:Lufino;given-names:M.M.\\nsurname:Wade-Martins;given-names:R.\\nsurname:Gromak;given-names:N.  \\nPLoS Genet.\\nref\\nR-loops associated with triplet repeat expansions promote gene silencing in Friedreich ataxia and fragile X syndrome\\nsurname:Hayashi;given-names:M.T.\\nsurname:Cesare;given-names:A.J.\\nsurname:Fitzpatrick;given-names:J.A.\\nsurname:Lazzerini-Denchi;given-names:E.\\nsurname:Karlseder;given-names:J.  \\nNat. Struct. Mol. Biol.\\nref\\nA telomere-dependent DNA damage checkpoint induced by prolonged mitotic arrest\\nsurname:Hegele;given-names:A.\\nsurname:Kamburov;given-names:A.\\nsurname:Grossmann;given-names:A.\\nsurname:Sourlis;given-names:C.\\nsurname:Wowro;given-names:S.\\nsurname:Weimann;given-names:M.\\nsurname:Will;given-names:C.L.\\nsurname:Pena;given-names:V.\\nsurname:Luhrmann;given-names:R.\\nsurname:Stelzl;given-names:U.  \\nMol. Cell.\\nref\\nDynamic protein-protein interaction wiring of the human spliceosome\\nsurname:Huang;given-names:H.C.\\nsurname:Shi;given-names:J.\\nsurname:Orth;given-names:J.D.\\nsurname:Mitchison;given-names:T.J.  \\nCell Cycle.\\nref\\nCell death when the SAC is out of commission\\nsurname:Inuzuka;given-names:H.\\nsurname:Shaik;given-names:S.\\nsurname:Onoyama;given-names:I.\\nsurname:Gao;given-names:D.\\nsurname:Tseng;given-names:A.\\nsurname:Maser;given-names:R.S.\\nsurname:Zhai;given-names:B.\\nsurname:Wan;given-names:L.\\nsurname:Gutierrez;given-names:A.\\nsurname:Lau;given-names:A.W.  \\nNature.\\nref\\nSCF(FBW7) regulates cellular apoptosis by targeting MCL1 for ubiquitylation and destruction\\n3B\\nsurname:Ismail;given-names:N.\\nsurname:Pihie;given-names:A.H.\\nsurname:Nallapan;given-names:M.  \\nAnticancer Res.\\nref\\nsurname:Jiang;given-names:H.\\nsurname:He;given-names:X.\\nsurname:Wang;given-names:S.\\nsurname:Jia;given-names:J.\\nsurname:Wan;given-names:Y.\\nsurname:Wang;given-names:Y.\\nsurname:Zeng;given-names:R.\\nsurname:Yates;given-names:J.;suffix:III\\nsurname:Zhu;given-names:X.\\nsurname:Zheng;given-names:Y.  \\nDev. Cell.\\nref\\nA microtubule-associated zinc finger protein, BuGZ, regulates mitotic chromosome alignment by ensuring Bub3 stability and kinetochore targeting\\nsurname:Khoo;given-names:K.H.\\nsurname:Verma;given-names:C.S.\\nsurname:Lane;given-names:D.P.  \\nNat. Rev. Drug Discov.\\nref\\nDrugging the p53 pathway: understanding the route to clinical efficacy\\nsurname:Kiger;given-names:A.A.\\nsurname:Jones;given-names:D.L.\\nsurname:Schulz;given-names:C.\\nsurname:Rogers;given-names:M.B.\\nsurname:Fuller;given-names:M.T.  \\nScience.\\nref\\nStem cell self-renewal specified by JAK-STAT activation in response to a support cell cue\\nsurname:Kim;given-names:D.\\nsurname:Pertea;given-names:G.\\nsurname:Trapnell;given-names:C.\\nsurname:Pimentel;given-names:H.\\nsurname:Kelley;given-names:R.\\nsurname:Salzberg;given-names:S.L.  \\nGenome Biol.\\nref\\nsurname:King;given-names:J.M.\\nsurname:Hays;given-names:T.S.\\nsurname:Nicklas;given-names:R.B.  \\nJ. Cell Biol.\\nref\\nDynein is a transient kinetochore component whose binding is regulated by microtubule attachment, not tension\\nsurname:Komarov;given-names:P.G.\\nsurname:Komarova;given-names:E.A.\\nsurname:Kondratov;given-names:R.V.\\nsurname:Christov-Tselkov;given-names:K.\\nsurname:Coon;given-names:J.S.\\nsurname:Chernov;given-names:M.V.\\nsurname:Gudkov;given-names:A.V.  \\nScience.\\nref\\nA chemical inhibitor of p53 that protects mice from the side effects of cancer therapy\\nsurname:Kotake;given-names:Y.\\nsurname:Sagane;given-names:K.\\nsurname:Owa;given-names:T.\\nsurname:Mimori-Kiyosue;given-names:Y.\\nsurname:Shimizu;given-names:H.\\nsurname:Uesugi;given-names:M.\\nsurname:Ishihama;given-names:Y.\\nsurname:Iwata;given-names:M.\\nsurname:Mizui;given-names:Y.  \\nNat. Chem. Biol.\\nref\\nSplicing factor SF3b as a target of the antitumor natural product pladienolide\\nsurname:Lanni;given-names:J.S.\\nsurname:Jacks;given-names:T.  \\nMol. Cell. Biol.\\nref\\nCharacterization of the p53-dependent postmitotic checkpoint following spindle disruption\\nsurname:Li;given-names:X.\\nsurname:Manley;given-names:J.L.  \\nCell.\\nref\\nInactivation of the SR protein splicing factor ASF/SF2 results in genomic instability\\nsurname:Lin;given-names:Y.\\nsurname:Dent;given-names:S.Y.\\nsurname:Wilson;given-names:J.H.\\nsurname:Wells;given-names:R.D.\\nsurname:Napierala;given-names:M.  \\nProc. Natl. Acad. Sci. USA.\\nref\\nR loops stimulate genetic instability of CTG CAG repeats\\nsurname:McDowell;given-names:E.M.\\nsurname:Trump;given-names:B.F.  \\nArch. Pathol. Lab. Med.\\nref\\nHistologic fixatives suitable for diagnostic light and electron microscopy\\nsurname:Orth;given-names:J.D.\\nsurname:Loewer;given-names:A.\\nsurname:Lahav;given-names:G.\\nsurname:Mitchison;given-names:T.J.  \\nMol. Biol. Cell.\\nref\\nProlonged mitotic arrest triggers partial activation of apoptosis, resulting in DNA damage and p53 induction\\nsurname:Park;given-names:J.W.\\nsurname:Tokheim;given-names:C.\\nsurname:Shen;given-names:S.\\nsurname:Xing;given-names:Y.  \\nMethods Mol. Biol.\\nref\\nIdentifying differential alternative splicing events from RNA sequencing data using RNASeq-MATS\\nsurname:Paulsen;given-names:R.D.\\nsurname:Soni;given-names:D.V.\\nsurname:Wollman;given-names:R.\\nsurname:Hahn;given-names:A.T.\\nsurname:Yee;given-names:M.C.\\nsurname:Guan;given-names:A.\\nsurname:Hesley;given-names:J.A.\\nsurname:Miller;given-names:S.C.\\nsurname:Cromwell;given-names:E.F.\\nsurname:Solow-Cordero;given-names:D.E.  \\nMol. Cell.\\nref\\nA genome-wide siRNA screen reveals diverse cellular processes and pathways that mediate genome stability\\nsurname:Pfaffl;given-names:M.W.  \\nNucleic Acids Res.\\nref\\nA new mathematical model for relative quantification in real-time RT-PCR\\nsurname:Phillips;given-names:D.D.\\nsurname:Garboczi;given-names:D.N.\\nsurname:Singh;given-names:K.\\nsurname:Hu;given-names:Z.\\nsurname:Leppla;given-names:S.H.\\nsurname:Leysath;given-names:C.E.  \\nJ. Mol. Recognit.\\nref\\nsurname:Pilch;given-names:D.R.\\nsurname:Sedelnikova;given-names:O.A.\\nsurname:Redon;given-names:C.\\nsurname:Celeste;given-names:A.\\nsurname:Nussenzweig;given-names:A.\\nsurname:Bonner;given-names:W.M.  \\nBiochem. Cell Biol.\\nref\\nCharacteristics of gamma-H2AX foci at DNA double-strand breaks sites\\nsurname:Pollier;given-names:J.\\nsurname:Rombauts;given-names:S.\\nsurname:Goossens;given-names:A.  \\nMethods Mol. Biol.\\nref\\nsurname:Porameesanaporn;given-names:Y.\\nsurname:Uthaisang-Tanechpongtamb;given-names:W.\\nsurname:Jarintanan;given-names:F.\\nsurname:Jongrungruangchok;given-names:S.\\nsurname:Thanomsub Wongsatayanon;given-names:B.  \\nOncol. Rep.\\nref\\nTerrein induces apoptosis in HeLa human cervical carcinoma cells through p53 and ERK regulation\\nsurname:Quignon;given-names:F.\\nsurname:Rozier;given-names:L.\\nsurname:Lachages;given-names:A.M.\\nsurname:Bieth;given-names:A.\\nsurname:Simili;given-names:M.\\nsurname:Debatisse;given-names:M.  \\nOncogene.\\nref\\nSustained mitotic block elicits DNA breaks: one-step alteration of ploidy and chromosome integrity in mammalian cells\\nsurname:Sato;given-names:M.\\nsurname:Muguruma;given-names:N.\\nsurname:Nakagawa;given-names:T.\\nsurname:Okamoto;given-names:K.\\nsurname:Kimura;given-names:T.\\nsurname:Kitamura;given-names:S.\\nsurname:Yano;given-names:H.\\nsurname:Sannomiya;given-names:K.\\nsurname:Goji;given-names:T.\\nsurname:Miyamoto;given-names:H.  \\nCancer Sci.\\nref\\nHigh antitumor activity of pladienolide B and its derivative in gastric cancer\\nsurname:Scheffner;given-names:M.\\nsurname:Werness;given-names:B.A.\\nsurname:Huibregtse;given-names:J.M.\\nsurname:Levine;given-names:A.J.\\nsurname:Howley;given-names:P.M.  \\nCell.\\nref\\nThe E6 oncoprotein encoded by human papillomavirus types 16 and 18 promotes the degradation of p53\\nsurname:Shen;given-names:S.\\nsurname:Park;given-names:J.W.\\nsurname:Huang;given-names:J.\\nsurname:Dittmar;given-names:K.A.\\nsurname:Lu;given-names:Z.X.\\nsurname:Zhou;given-names:Q.\\nsurname:Carstens;given-names:R.P.\\nsurname:Xing;given-names:Y.  \\nNucleic Acids Res.\\nref\\nMATS: a Bayesian framework for flexible detection of differential alternative splicing from RNA-Seq data\\nsurname:Song;given-names:G.\\nsurname:Mao;given-names:Y.B.\\nsurname:Cai;given-names:Q.F.\\nsurname:Yao;given-names:L.M.\\nsurname:Ouyang;given-names:G.L.\\nsurname:Bao;given-names:S.D.  \\nBraz. J. Med. Biol. Res.\\nref\\nCurcumin induces human HT-29 colon adenocarcinoma cell apoptosis by activating p53 and regulating apoptosis-related protein expression\\nsurname:Stahler;given-names:F.\\nsurname:Roemer;given-names:K.  \\nOncogene.\\nref\\nMutant p53 can provoke apoptosis in p53-deficient Hep3B cells with delayed kinetics relative to wild-type p53\\nsurname:Surova;given-names:O.\\nsurname:Zhivotovsky;given-names:B.  \\nOncogene.\\nref\\nVarious modes of cell death induced by DNA damage\\nsurname:Toledo;given-names:C.M.\\nsurname:Herman;given-names:J.A.\\nsurname:Olsen;given-names:J.B.\\nsurname:Ding;given-names:Y.\\nsurname:Corrin;given-names:P.\\nsurname:Girard;given-names:E.J.\\nsurname:Olson;given-names:J.M.\\nsurname:Emili;given-names:A.\\nsurname:DeLuca;given-names:J.G.\\nsurname:Paddison;given-names:P.J.  \\nDev. Cell.\\nref\\nBuGZ is required for Bub3 stability, Bub1 kinetochore function, and chromosome alignment\\nsurname:Trapnell;given-names:C.\\nsurname:Williams;given-names:B.A.\\nsurname:Pertea;given-names:G.\\nsurname:Mortazavi;given-names:A.\\nsurname:Kwan;given-names:G.\\nsurname:van Baren;given-names:M.J.\\nsurname:Salzberg;given-names:S.L.\\nsurname:Wold;given-names:B.J.\\nsurname:Pachter;given-names:L.  \\nNat. Biotechnol.\\nref\\nTranscript assembly and quantification by RNA-Seq reveals unannotated transcripts and isoform switching during cell differentiation\\nsurname:Trapnell;given-names:C.\\nsurname:Roberts;given-names:A.\\nsurname:Goff;given-names:L.\\nsurname:Pertea;given-names:G.\\nsurname:Kim;given-names:D.\\nsurname:Kelley;given-names:D.R.\\nsurname:Pimentel;given-names:H.\\nsurname:Salzberg;given-names:S.L.\\nsurname:Rinn;given-names:J.L.\\nsurname:Pachter;given-names:L.  \\nNat. Protoc.\\nref\\nsurname:Uetake;given-names:Y.\\nsurname:Sluder;given-names:G.  \\nCurr. Biol.\\nref\\nProlonged prometaphase blocks daughter cell proliferation despite normal completion of mitosis\\nsurname:Wahba;given-names:L.\\nsurname:Amon;given-names:J.D.\\nsurname:Koshland;given-names:D.\\nsurname:Vuica-Ross;given-names:M.  \\nMol. Cell.\\nref\\nRNase H and multiple RNA biogenesis factors cooperate to prevent RNA:DNA hybrids from generating genome instability\\nsurname:Wahl;given-names:M.C.\\nsurname:Will;given-names:C.L.\\nsurname:Luhrmann;given-names:R.  \\nCell.\\nref\\nThe spliceosome: design principles of a dynamic RNP machine\\nsurname:Wertz;given-names:I.E.\\nsurname:Kusam;given-names:S.\\nsurname:Lam;given-names:C.\\nsurname:Okamoto;given-names:T.\\nsurname:Sandoval;given-names:W.\\nsurname:Anderson;given-names:D.J.\\nsurname:Helgason;given-names:E.\\nsurname:Ernst;given-names:J.A.\\nsurname:Eby;given-names:M.\\nsurname:Liu;given-names:J.  \\nNature.\\nref\\nSensitivity to antitubulin chemotherapeutics is regulated by MCL1 and FBW7\\nsurname:Wimberly;given-names:H.\\nsurname:Shee;given-names:C.\\nsurname:Thornton;given-names:P.C.\\nsurname:Sivaramakrishnan;given-names:P.\\nsurname:Rosenberg;given-names:S.M.\\nsurname:Hastings;given-names:P.J.  \\nNat. Commun.\\nref\\nsurname:Wongsurawat;given-names:T.\\nsurname:Jenjaroenpun;given-names:P.\\nsurname:Kwoh;given-names:C.K.\\nsurname:Kuznetsov;given-names:V.  \\nNucleic Acids Res.\\nref\\nQuantitative model of R-loop forming structures reveals a novel level of RNA-DNA interactome complexity\", metadata={'reference': ''})]\n"
     ]
    }
   ],
   "source": [
    "exclude_headers = []\n",
    "splits = _pseudo_markdown_splitter(text, chunk_size=2000, chunk_overlap=125, markdown_headers=markdown_headers, \n",
    "                                   exclude_headers=exclude_headers, verbose=True)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 108,
   "metadata": {},
   "outputs": [],
   "source": [
    "\n",
    "def _match_sequence(seq1: str, seq2_list: list, threshold=0.7):\n",
    "    \"\"\"\n",
    "    Gives a sequence similarity match between seq1 and seq2 from seq2_list.\n",
    "\n",
    "    Args: \n",
    "        seq1: str\n",
    "            The sequence to match\n",
    "        seq_list: list\n",
    "            A list of sequences to match against seq1\n",
    "        threshold: float\n",
    "            The minimum similarity ratio; <.8 for short sequences seems to work well.\n",
    "    Return:\n",
    "        seq2: str\n",
    "            The sequence that passes the matching threshold or None if insufficient match.\n",
    "            \n",
    "    \"\"\"\n",
    "\n",
    "    from difflib import SequenceMatcher\n",
    "\n",
    "    for seq2 in seq2_list:\n",
    "        if SequenceMatcher(None, seq1, seq2).ratio() > threshold:\n",
    "            return seq2\n",
    "    return None\n",
    "        \n",
    "\n",
    "\n",
    "def _split_non_md_headers(doc_lc, headers=[]):\n",
    "    \"\"\" A secondery cleaner to extract leftover non-markdown headers from the text. \n",
    "    \n",
    "    Args:\n",
    "        doc_lc: LangChain Document \n",
    "            A list of Langchain doc objects, each containing paragraphs and artifact sub-headers according to \n",
    "            chunk size; and 'metadata' key would contain real markdown headers IF any.\n",
    "        headers: list    \n",
    "            A list of possible headers (e.g. headers = [\"introduction\", \"paragraph\", \"title_1\", \"title_2\", \"fig_caption\"])\n",
    "    Return:\n",
    "        cleaned_docs: dict\n",
    "            Contains core paragraphs under dict['text'] and associated headers under dict['headers']\n",
    "\n",
    "    \"\"\"\n",
    "\n",
    "    cleaned_docs = {}\n",
    "    cleaned_docs['text'] = \"\"\n",
    "    cleaned_docs['headers'] = []\n",
    "    text = doc_lc.page_content\n",
    "\n",
    "    for line in text.split(\"\\n\"):\n",
    "        match = _match_sequence(line.lower(), headers)\n",
    "        if match:\n",
    "            cleaned_docs['headers'].append(match)\n",
    "        else:\n",
    "            cleaned_docs['text'] += line\n",
    "    return cleaned_docs"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 227,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "N datasets with exactly one label, for each field:\n",
      "{'assay_by_molecule_14': 3,\n",
      " 'assay_count_7': 4,\n",
      " 'experimental_design_10': 0,\n",
      " 'experimental_factors_20': 2,\n",
      " 'hardware_4': 1,\n",
      " 'name_19': 4,\n",
      " 'no_of_samples_22': 0,\n",
      " 'no_of_samples_23': 0,\n",
      " 'organism_16': 3,\n",
      " 'releasedate_12': 4,\n",
      " 'sample_count_13': 4,\n",
      " 'sex_2': 0,\n",
      " 'study_type_18': 3,\n",
      " 'technology_15': 4,\n",
      " 'type_21': 0,\n",
      " 'type_9': 3}\n",
      "N datasets with at least one label, for each field\n",
      "{'assay_by_molecule_14': 5,\n",
      " 'assay_count_7': 4,\n",
      " 'experimental_design_10': 0,\n",
      " 'experimental_factors_20': 4,\n",
      " 'hardware_4': 1,\n",
      " 'name_19': 5,\n",
      " 'no_of_samples_22': 0,\n",
      " 'no_of_samples_23': 0,\n",
      " 'organism_16': 5,\n",
      " 'releasedate_12': 5,\n",
      " 'sample_count_13': 5,\n",
      " 'sex_2': 1,\n",
      " 'study_type_18': 5,\n",
      " 'technology_15': 5,\n",
      " 'type_21': 5,\n",
      " 'type_9': 5}\n"
     ]
    }
   ],
   "source": [
    "t,l = load_arxpr_data(5)\n",
    "\n",
    "# keys for manual check on papers\n",
    "paper_keys = ['25918225', '18618715', '28845460', '25977295', '18631455']\n",
    "text = t['25918225']\n",
    "\n",
    "markdown_headers = [\n",
    "    (\"METHODS\", \"methods\"),\n",
    "    (\"METHODOLOGY\", \"methods\"),   \n",
    "    (\"RESULT\", \"result\"),\n",
    "    (\"RESULTS\", \"result\"),\n",
    "    (\"FIG\", \"figure\"),\n",
    "    (\"FIGURE\", \"figure\"),\n",
    "    (\"INTRO\", \"introduction\"),\n",
    "    (\"INTRODUCTION\", \"introduction\"),\n",
    "    (\"REF\", \"reference\"),\n",
    "    (\"REFERENCES\", \"reference\"),\n",
    "    (\"DISCUSS\", \"discussion\"),\n",
    "    (\"DISCUSSION\", \"discussion\"),\n",
    "    (\"SUPPL\", \"supplement\"),\n",
    "    (\"SUPPLEMENT\", \"supplement\"),\n",
    "    (\"abstract_title_1\", \"abstract\"),\n",
    "]\n",
    "\n",
    "exclude_headers = ['reference', 'supplement']\n",
    "\n",
    "\n",
    "# TODO: need a script to generate a list of headers from a large set or sampled papers.\n",
    "other_headers = [\"introduction\",\n",
    "                \"paragraph\",\n",
    "                \"title_1\", \"title_2\",\n",
    "                \"fig_caption\",\n",
    "                \"abstract\",\n",
    "                \"supplementary material\",\n",
    "                \"materials and methods\",\n",
    "                \"results\",\n",
    "                \"discussion\",\n",
    "                \"results and discussion\",\n",
    "                \"footnote_title\",\n",
    "                \"figures and tables\",\n",
    "                \"summary\", \n",
    "                \"ref\",\n",
    "                \"lancetref\",\n",
    "                \"experimental procedures\"\n",
    "                \"fig1\", \"fig2\", \"fig3\", \"fig4\", \"fig5\", \"fig6\", \"fig7\", \"fig8\", \"fig9\", \"fig10\",\n",
    "                ]\n",
    "\n",
    "Settings.embed_model = OllamaEmbedding(model_name=\"llama3.1:8b\")\n",
    "Settings.llm = Ollama(model=\"llama3.1:8b\") \n",
    "llm = Ollama(model=\"llama3.1:8b\") \n",
    "embed_model = OllamaEmbedding(model_name=\"llama3.1:8b\")\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Pipeline: Markdown + Recursive \n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 228,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "MD Headers: {} \n",
      "Other Headers: ['title_1', 'abstract', 'abstract', 'abstract', 'abstract'] \n",
      "Size change: 1061 ? 1004\n",
      "BioC-APIcollection.keyCC BY-NC-SASplicing role of mitotic regulator kills tumor cellsThis article is distributed under the terms of an Attribution-Noncommercial-Share Alike-No Mirror Sites license for the first six months after the publication date (see http://www.rupress.org/terms). After six months it is available under a Creative Commons License (Attribution-Noncommercial-Share Alike 3.0 Unported license, as described at http://creativecommons.org/licenses/by-nc-sa/3.0/).surname:Wan;given-names:Yihansurname:Zheng;given-names:Xiaobinsurname:Chen;given-names:Haiyangsurname:Guo;given-names:Yuxuansurname:Jiang;given-names:Haosurname:He;given-names:Xiaonansurname:Zhu;given-names:Xueliangsurname:Zheng;given-names:YixianfrontSplicing function of mitotic regulators links R-loop-mediated DNA damage to tumor cell killingMitotic regulators BuGZ and Bub3 play a critical role in RNA splicing during interphase, and disruption of this function leads to R-loop formation, DNA damage, and p53 activation.\n",
      "\n",
      "MD Headers: {} \n",
      "Other Headers: ['abstract', 'abstract'] \n",
      "Size change: 981 ? 963\n",
      "Although studies suggest that perturbing mitotic progression leads to DNA damage and p53 activation, which in turn lead to either cell apoptosis or senescence, it remains unclear how mitotic defects trigger p53 activation. We show that BuGZ and Bub3, which are two mitotic regulators localized in the interphase nucleus, interact with the splicing machinery and are required for pre-mRNA splicing. Similar to inhibition of RNA splicing by pladienolide B, depletion of either BuGZ or Bub3 led to increased formation of RNA-DNA hybrids (R-loops), which led to DNA damage and p53 activation in both human tumor cells and primary cells. Thus, R-loop-mediated DNA damage and p53 activation offer a mechanistic explanation for apoptosis of cancer cells and senescence of primary cells upon disruption of the dual-function mitotic regulators. This demonstrates the importance of understanding the full range of functions of mitotic regulators to develop antitumor drugs.\n",
      "\n",
      "MD Headers: {'introduction': ''} \n",
      "Other Headers: ['title_1', 'introduction', 'paragraph', 'paragraph'] \n",
      "Size change: 1070 ? 1027\n",
      "Extensive studies have shown that prolonged mitotic arrest can lead to DNA damage and p53 activation. Although p53 activation in these cells would explain why targeting mitotic regulators could be effective for cancer therapy, how mitotic arrest leads to DNA damage and p53 activation is not fully understood in some contexts. For example, prolonged mitosis is proposed to cause DNA or cellular damage that would in turn activate p53. Supporting this idea, prolonged mitotic arrest has been shown to cause Caspase activation, which could activate CAD (Caspase-activated DNase). Although CAD could lead to DNA damage and p53 activation, how prolonged mitosis activates Caspases is not clear in this context. Additionally, mitotic timer has been suggested to sense the prolonged mitotic arrest in either a p53-dependent or independent manner. While a p53-dependent timer could link prolonged mitotic block to p53 activation, neither the nature of this timer nor the signal that activates p53 has been defined in these settings.  \n",
      "\n",
      "MD Headers: {'introduction': ''} \n",
      "Other Headers: ['paragraph'] \n",
      "Size change: 1268 ? 1258\n",
      "The difficulty in defining the mitotic trigger for DNA damage and p53 activation could be because we have not looked at the right stage of the cell cycle. Indeed, several mitotic regulators are found in the interphase nucleus. Therefore, p53 activation could be caused by the disruption of the interphase nuclear functions of these mitotic regulators. Recently, a nuclear zinc finger protein BuGZ has been shown to regulate mitosis by directly binding to the spindle assembly checkpoint protein Bub3 to promote its loading to kinetochores and chromosome alignment. Interestingly, Bub3 is also localized to the interphase nucleus, and the interaction between BuGZ and Bub3 can be detected throughout the cell cycle. As expected, BuGZ depletion in various cancer cell lines resulted in a great reduction in the kinetochore Bub3 levels, chromosome misalignment, and mitotic block. Curiously, upon a prolonged mitotic block, most of the BuGZ-depleted tumor cells undergo mitotic death (mitotic catastrophe). By investigating this mitotic catastrophe phenomenon, we have uncovered an unrecognized interphase nuclear function of BuGZ and Bub3. This interphase function helps to explain why the disruption of the two mitotic regulators could lead to p53 activation.\n",
      "\n",
      "MD Headers: {'result': ''} \n",
      "Other Headers: ['title_1', 'results and discussion', 'title_1', 'paragraph', 'fig2'] \n",
      "Size change: 866 ? 809\n",
      "Depletion of BuGZ causes apoptosis in cancer cells and senescence in primary fibroblasts  Previous studies have shown that BuGZ depletion in cancer cells destabilizes Bub3 and causes chromosome misalignment and mitotic arrest followed by massive cell death. To further study the function of BuGZ, we used siRNA to deplete the protein in three cancer cell lines (HeLa, HT29, or TOV21G) and the primary human foreskin fibroblasts (HFFs). Consistent with the role of BuGZ in maintaining Bub3 protein level, BuGZ depletion in these cells by 60 h of siRNA treatment led to Bub3 reduction (Fig. 1 A) and an elevation of mitotic index (Fig. S1 A). This shows that BuGZ is required for efficient chromosome alignment in both cancer cells and HFFs, as would be expected based on the Bub3 reduction upon BuGZ depletion.\n",
      "\n",
      "MD Headers: {'figure': ''} \n",
      "Other Headers: ['fig_caption'] \n",
      "Size change: 1840 ? 1828\n",
      "Reduction of BuGZ causes apoptosis in human cancer cells but senescence in primary foreskin fibroblasts (HFFs). (A) Depletion of BuGZ by siRNA treatment in the indicated cells. Two different amounts of control lysates (1/2 or 1) were loaded. Cells were analyzed 60 h after siRNAs transfection. alpha-tubulin, loading control. The numbers at the bottom of each lane indicate relative levels of BuGZ or Bub3 compared with controls, which were set to 1. Note that >90% of BuGZ was depleted by RNAi. (B) FACS assays for the quantification of apoptotic cells at 72 h after RNAi. Early apoptotic cells were Annexin V positive (see the bottom right quadrants), whereas late apoptotic ones were Annexin V and 7-aminoactinomycin D (7-AAD) double positive (the top right quadrants). All values are percentages of total cells, presented as mean +- SD from three independent experiments. (C) Representative microscopic fields of cells at 72 h after RNAi. Bar, 100 microm. (D) Growth curves of cells treated with control or BuGZ siRNA. Error bars indicate SD. Student's t test: n.s., not significant; *, P < 0.05; **, P < 0.01; ***, P < 0.001 from three independent experiments. (E) Stable knockdown of BuGZ with lentivirus-expressed shRNA in HFFs. The virus-infected cells were enriched by puromycin selection. The cell lysates were collected at day 5. (F) Growth curves of HFFs at the indicated days after shRNA transfection. Error bars indicate SD. Student's t test: **, P < 0.01; ***, P < 0.001 from three independent experiments. (G) Increased senescence in BuGZ shRNA-transfected HFFs as judged by SA-beta-gal staining (arrowheads). Bar, 50 microm. (H) Percentage of SA-beta-gal-positive cells at the indicated days after shRNA transfection. Error bars indicate SEM. Student's t test: ***, P < 0.001 from three independent experiments.\n",
      "\n",
      "MD Headers: {'result': ''} \n",
      "Other Headers: ['paragraph', 'title_1', 'paragraph'] \n",
      "Size change: 1228 ? 1199\n",
      "By 5 d of the siRNA treatment, all three cancer cell lines exhibited a large increase in apoptosis as judged by the early and late apoptotic markers Annexin V and 7-AAD, respectively (Fig. 1 B), which correlated with a striking reduction in cell numbers (Fig. 1, C and D). In contrast, BuGZ siRNA treatment of HFFs did not affect cell survival (Fig. S1 B), which is similar to the observation in primary neural stem cells, despite a moderate elevation in mitotic index (Fig. S1 A). Nevertheless, because primary somatic cells usually grow more slowly than tumor cells, we reasoned that a time >5 d might be required for properly assessing the effect of BuGZ depletion on HFFs. We thus used lentivirus-mediated RNAi to stably knock down BuGZ in HFFs (Fig. 1 E). We found that, from day 8 after infection, the proliferation rate of the BuGZ-RNAi HFFs began to significantly slow down (Fig. 1 F). Moreover, a threefold increase of senescent cells was observed at 30 d compared with controls (Fig. 1, G and H), which indicates accelerated senescence of BuGZ-depleted HFFs. Thus, BuGZ depletion causes apoptosis in cancer cells and senescence in primary HFFs.  Depletion of BuGZ leads to p53 activation  \n",
      "\n",
      "MD Headers: {'result': ''} \n",
      "Other Headers: ['title_1', 'paragraph', 'fig2'] \n",
      "Size change: 972 ? 948\n",
      "Depletion of BuGZ leads to p53 activation  Since p53 accumulation is known to cause either apoptosis or senescence depending on the cellular context, we asked whether BuGZ depletion led to p53 induction in cancer cells and HFFs. HeLa and TOV21G cells have wild-type p53. Although in HeLa cells p53 protein is targeted for degradation by the human papillomavirus E6 proteins under normal conditions, its accumulation has been reported upon DNA damage or oxidation. HT29 cells have a point mutation in the transactivation domain of p53 (R273H), but studies have shown that this mutant p53 induces apoptosis or cell cycle arrest upon various challenges. Activation of p53 in primary cells with intact cell cycle checkpoints has been shown to cause senescence. Consistently, we found that the levels of total p53 and activated p53 (phospho-p53 at serine 15) were increased upon BuGZ depletion in both the cancer cells tested and HFFs (Fig. 2, A and B).\n",
      "\n",
      "MD Headers: {'figure': ''} \n",
      "Other Headers: ['fig_caption'] \n",
      "Size change: 1047 ? 1035\n",
      "Activation of p53 upon BuGZ depletion leads to apoptosis or senescence. (A) Up-regulation of p53 after 48 h of BuGZ siRNA treatment. Loading control, alpha-tubulin. (B) Increased phospho-p53 (Ser15) in the nuclei of BuGZ-depleted cells (arrowheads). BuGZ-positive (arrows) and phospho-p53 negative cells (arrows and white broken circles) are indicated. Cells were assayed 48 h after siRNA treatment. Bars, 10 microm. (C) Inhibition of p53 by PFT-alpha increased the numbers of viable BuGZ-depleted cells. Error bars indicate SD. Student's t test: **, P < 0.01 from three independent experiments. (D-F) p53 RNAi in HFFs repressed senescence induced by BuGZ knockdown. HFFs were infected with lentivirus to express the indicated shRNAs. Western blotting (D) and quantification (E) of SA-beta-gal staining (F) were performed at 30 d after infection. Arrowheads in F represent the SA-beta-gal-positive staining cells. Bar, 100 microm. Error bars indicate SEM. Student's t test: *, P < 0.05; **, P < 0.01 from three independent experiments.\n",
      "\n",
      "MD Headers: {'result': ''} \n",
      "Other Headers: ['paragraph', 'title_1', 'paragraph', 'fig2'] \n",
      "Size change: 1990 ? 1955\n",
      "Our previous studies indicated that many BuGZ-depleted HeLa cells initially arrested in mitosis and then underwent cell death. To test whether p53 activation could be the cause of cancer cell death, we treated cells with p53 inhibitor PFT-alpha 12 h after transfection of control or BuGZ siRNA and found that the inhibition of p53 significantly rescued the cell number reduction of the BuGZ-depleted cancer cells (Fig. 2 C). When HFFs were coinfected with lentivirus to express p53 shRNA, senescence induced by BuGZ depletion upon prolonged culturing was also significantly inhibited (Fig. 2, D-F). Because depletion of BuGZ down-regulated Bub3 (Fig. 1 A), we investigated whether Bub3 reduction could also lead to p53 activation. We found that knockdown of Bub3 by RNAi caused prominent p53 accumulation and activation in both cancer cells and HFFs (Fig. S1, C-F). These findings show that p53 mediates both apoptosis in cancer cells and senescence in HFFs upon BuGZ depletion.  BuGZ and Bub3 are required for proper pre-mRNA splicing  Next we explored how the depletion of BuGZ or Bub3 could activate p53. Because BuGZ and Bub3 bind to one another and are both present in the interphase nucleus, their depletion could disrupt an interphase nuclear function, which in turn might trigger p53 activation. To investigate this, we examined our previous proteome analyses of BuGZ-interacting proteins. Besides Bub3, we noticed several spliceosome components among the top candidates of BuGZ-interacting proteins (Fig. 3 A). Among a number of available antibodies to spliceosome components, we found that the antibodies to two:U2AF65 and SF3a3:worked in both immunoprecipitation and Western blotting, which allowed us to use reciprocal immunoprecipitation to provide further support for an interaction between BuGZ/Bub3 and the spliceosome (Fig. 3, B and C). Consistently, both BuGZ and Bub3 have also been found in previous proteomic analyses of spliceosomes.\n",
      "\n",
      "MD Headers: {'figure': ''} \n",
      "Other Headers: ['fig_caption'] \n",
      "Size change: 1304 ? 1292\n",
      "A requirement of BuGZ and Bub3 for RNA splicing. (A) Proteins with known functions in RNA splicing (highlighted in orange) were found as candidate BuGZ-interacting proteins. (B) FLAG-tagged BuGZ or Bub3, but not FLAG-luciferase (Luci.), immunoprecipitated the spliceosome components SF3a3 and U2AF65 in HEK293T cells. (C) BuGZ and Bub3 interacted with SF3a3 and U2AF65 in vivo. Reciprocal immunoprecipitations of endogenous proteins were performed using HT29 cell lysates with the indicated mouse monoclonal (mAb) or rabbit antibodies (rAb). h.c., antibody heavy chain. Experiments were performed three times. (D) Analyses of splicing defects by RNA-sequencing of HFFs and TOV21G cells treated by the splicing inhibitor pladienolide B or RNAi of BuGZ or Bub3. The numbers of genes exhibiting the depicted splicing defects are shown as pie charts. RNA-Seq was from two biological repeats. (E) Two representative genes, BRD2 and WAC, exhibiting intron retention upon treatment with pladienolide B or RNAi of BuGZ or Bub3. The red boxes highlight examples of intron retention. The fold increases of the intron retention are plotted to the right. The p-values were generated by simulation (MATS: a Bayesian framework for flexible detection of differential alternative splicing from RNA-Seq data).\n",
      "\n",
      "MD Headers: {'result': ''} \n",
      "Other Headers: ['paragraph', 'title_1', 'paragraph'] \n",
      "Size change: 1440 ? 1411\n",
      "To test whether BuGZ and Bub3 play roles in splicing, we depleted each protein in cancer cells (TOV21G) or HFFs and performed RNA sequencing (accession number: GSE67770). Cells treated with pladienolide B, a potent splicing inhibitor, were used as positive controls. Among the splicing defects depicted in Fig. 3 D, we found that the depletion of either Bub3 or BuGZ mainly caused exon skipping, whereas pladienolide B treatment mainly resulted in intron retention (Fig. 3, D and E; and Table S1). The difference in the resultant splicing defects between Pladienolide B inhibition (which targets SF3b3) and BuGZ or Bub3 depletion suggests that BuGZ and Bub3 function differentially from SF3b3. Whereas further studies are required to understand how BuGZ and Bub3 regulate splicing, these analyses show that the well-established SAC component Bub3 and its regulator BuGZ also play roles in interphase RNA splicing. Depletion of BuGZ in transformed cells or treatment of tumor cells with pladienolide B have been shown to kill these cells in vitro. Interestingly, using the Drosophila melanogaster eye tumor model, we found that the eye hyperproliferation induced by the activation of either the Jak-Stat or the Hippo (unpublished data) pathway was inhibited by the eye-specific BuGZ RNAi or by feeding the flies with pladienolide B (Fig. S2 A).  Depletion of BuGZ or Bub3 causes R-loop formation and DNA damage  \n",
      "\n",
      "MD Headers: {'result': ''} \n",
      "Other Headers: ['title_1', 'paragraph', 'fig2'] \n",
      "Size change: 1106 ? 1082\n",
      "Depletion of BuGZ or Bub3 causes R-loop formation and DNA damage  Deficiency in RNA splicing has been shown to cause an increased formation of RNA-DNA hybrids (R-loops) in the nucleus, and subsequent DNA breaks and p53 activation. Therefore, p53-mediated killing of transformed cells or senescence of primary cells as we observed in the above studies could be caused by defects in splicing and the subsequent R-loop formation. To determine whether cells depleted of BuGZ or Bub3 had an increased R-loop formation, we used the monoclonal antibody S9.6, which recognizes RNA-DNA hybrids. Similar to the pladienolide B treatment (Fig. 4 A), depletion of BuGZ or Bub3 markedly stimulated R-loop formation in both cancer cells and HFFs (Fig. 4 A and Fig. S2, B and C). These cells also exhibited increased DNA damage, as judged by the appearance of prominent nuclear foci of gammaH2AX (; Fig. 4, B-F; Fig. S2, D-H; and Fig. S3 A). These results suggest that defects in the splicing function of BuGZ and Bub3 lead to R-loop-induced DNA damage, which in turn contributes to p53 activation.\n",
      "\n",
      "MD Headers: {'figure': ''} \n",
      "Other Headers: ['fig_caption'] \n",
      "Size change: 1089 ? 1077\n",
      "Depletion of BuGZ or Bub3 leads to increased R-loop formation and DNA damage. (A) Depletion of BuGZ or Bub3 resulted in increased R-loop formation in HT29 cells. Pladienolide B treatment was used as a positive control. White broken circles outline the nuclei with no or low levels of R-loops. Bar, 5 microm. The quantification of mean fluorescence intensity of the R-loop is shown on the right. Student's t test: ***, P < 0.001 from three independent experiments. (B and C) Depletion of BuGZ (B) or Bub3 (C) by two different siRNAs in HT29 cells caused increased DNA damage, as judged by the presence of prominent nuclear foci of gamma-H2Ax. BuGZ- or Bub3-depleted cells (arrowheads) exhibit more gamma-H2Ax foci than cells with insufficient or no depletion (arrows). White broken circles outline the nuclei. Bars, 10 microm. (D-F) Quantification of gamma-H2Ax intensities in individual cells (D) and distributions of gamma-H2Ax intensities (E and F). Error bars indicate SEM. Student's t test: ***, P < 0.001. Cell numbers (n) analyzed were from three independent experiments.\n",
      "\n",
      "MD Headers: {'result': ''} \n",
      "Other Headers: ['title_1', 'paragraph', 'fig2'] \n",
      "Size change: 1230 ? 1206\n",
      "Inhibition of R-loop formation attenuates p53 activation and DNA damage caused by depletion of BuGZ or Bub3  To confirm that the R-loops formed in cells depleted of BuGZ or Bub3 indeed contributed to DNA damage and p53 activation, we used a modified form of RNase H1 lacking the mitochondria targeting sequence. This form of RNase H1 has been shown to localize only in the nucleus and to resolve R-loops. By analyzing the nuclei that were positive for the GFP-tagged RNase H1 or GFP control, we found that expression of the enzyme resulted in significant reduction of both gammaH2AX foci (Fig. 5, A-C) and p53 activation (Fig. 5, D-F) in BuGZ-depleted HT29 cells. A similar effect of RNase H1 was also observed in pladienolide B-treated (Fig. S3 B) or Bub3-depleted (Fig. 5, G-J) HT29 cells. The inability of RNase H1 to fully rescue DNA damage and p53 activation could be caused by the heterogeneous level of GFP-RNase H1 expression, the inefficiency of GFP-RNase H1 to resolve certain R-loops, or other uncharacterized defects. Nonetheless, these findings show that R-loop formation due to the disruption of the splicing function of Bub3 and BuGZ (Fig. S3 C) contributes to DNA damage and p53 activation.\n",
      "\n",
      "MD Headers: {'figure': ''} \n",
      "Other Headers: ['fig_caption'] \n",
      "Size change: 1548 ? 1536\n",
      "Expression of RNase H1 in BuGZ- or Bub3-depleted HT29 cells reduces gamma-H2Ax and phospho-p53 signals. (A) GFP-RNase H1 (RNase), but not GFP, repressed BuGZ depletion-induced DNA damage in HT29 cells. Cells were first transfected by siRNAs for 12 h and then by GFP-RNase H1 or GFP plasmid for 48 h followed by immunofluorescence microscopy analyses. BuGZ-depleted cells with or without GFP fluorescence are indicated by arrowheads or arrows, respectively. The broken lines indicate nuclear boundaries. Bar, 10 microm. (B and C) Quantifications for the mean fluorescence intensity of gamma-H2Ax (B) and the intensity distributions (C) of the experiments shown in A. (D) GFP-RNase H1 repressed BuGZ depletion-induced p53 phosphorylation in HT29 cells. Arrowheads and arrows indicate cells with or without GFP-RNase H1, respectively. White broken circles outline the nuclei. (E and F) Quantifications for the mean fluorescence intensity of phospho-p53 (E) and the intensity distributions (F) of the experiments shown in D. (G and H) Quantifications for the mean fluorescence intensity of gamma-H2Ax (G) and the intensity distributions (H) in Bub3-depleted and GFP- or GFP-RNase H1-expressing HT29 cells. (I and J) Quantifications for the mean fluorescence intensity of phospho-p53 (I) and the intensity distributions (J) in Bub3-depleted and GFP- or GFP-RNase H1-expressing HT29 cells. Error bars indicate SEM. Student's t test: *, P < 0.05; **, P < 0.01; ***, P < 0.001. Cell numbers (n) analyzed were from three independent experiments.\n",
      "\n",
      "MD Headers: {'result': ''} \n",
      "Other Headers: ['paragraph'] \n",
      "Size change: 1150 ? 1140\n",
      "Extensive efforts have been devoted to understanding how mitotic arrest as a result of disrupting mitotic regulators can lead to p53-dependent cell death or cell cycle arrest in cancer cells. By exploring the mitosis-independent functions, we have uncovered a critical role of the mitotic regulators BuGZ and Bub3 in interphase RNA splicing. Importantly, we demonstrated that disruption of this interphase function by depleting BuGZ or Bub3 leads to R-loop formation, DNA damage, and p53 activation. Furthermore, our results suggest that the splicing inhibitor pladienolide B, a potent anticancer reagent, kills cancer cells through a similar mechanism. Our findings demonstrate that by looking beyond the mitosis functions of the established mitotic regulators, we could gain significant insights into how aberrant mitosis is connected to the suppression of cell proliferation and the safeguard of genome stability. Therefore, a broadened understanding of the function of mitotic regulators beyond cell division should facilitate the selection of therapeutic targets in the effort of developing new antitumor drugs for cancer chemotherapy.\n",
      "\n",
      "MD Headers: {'methods': ''} \n",
      "Other Headers: ['title_1', 'materials and methods', 'paragraph', 'title_1', 'paragraph', 'paragraph'] \n",
      "Size change: 1245 ? 1173\n",
      "For the sources and sequences of mammalian expression plasmids, antibodies (with dilutions used), siRNA, shRNA, small molecules (with concentrations used), and cell lines, please refer to Table S2. The plasmid for expressing GFP-RNase H1 was provided by R.J. Crouch (National Institute of Child Health and Human Development, Bethesda, MD).  Cell culture, plasmids, and siRNA transfection  For TOV21G cell (ATCC) culture, the base medium was a 1:1 mixture of MCDB 105 medium (Sigma-Aldrich) containing a final concentration of 1.5 g/liter sodium bicarbonate and Medium 199 (Sigma-Aldrich) containing a final concentration of 2.2 g/liter sodium bicarbonate. To make the complete growth medium, fetal bovine serum (Gibco) was added to a final concentration of 15%. HT29 (HTB-38; ATCC) and HeLa (CCL-2; ATCC) cells were cultured in Dulbecco's modified Eagle's medium (Gibco) containing 10% fetal bovine serum (Gibco) in 5% CO2 at 37 C. The HFF cells (Millipore) were cultured in FibroGRO LS (low serum) Complete Medium (Millipore). HFFs were changed into fresh medium every other day and passaged using Accutase (Millipore) at a 1:3 dilution when reaching ~80% of confluency.  \n",
      "\n",
      "MD Headers: {'methods': ''} \n",
      "Other Headers: ['paragraph', 'title_1', 'paragraph'] \n",
      "Size change: 1339 ? 1310\n",
      "The plasmids or siRNA transfections were performed using Lipofectamine 2000 (Life Technologies) or Lipofectamine RNAiMAX (Life Technologies), respectively. Cells were fixed for immunofluorescence analyses 48 h after siRNA transfection unless otherwise indicated. For cotransfection of siRNA and plasmids, siRNA was transfected first using the reverse transfection protocol. In brief, 10 pmol siRNA oligo was diluted in 50 microl Opti-MEM I Medium (Gibco) per well of a 24-well tissue culture plate. 1.5 microl of Lipofectamine RNAiMAX was added to each well. After mixing, the plate was incubated for 20 min at room temperature. Meanwhile, cells to be transfected were trypsinized and resuspended in complete growth medium without antibiotics. Cell suspension containing 20,000 cells was added into each well for transfection. After 24 h of transfection, cells were transfected with the indicated plasmids. For immunofluorescence analyses in Fig. 5, cells were fixed 48 h after plasmid transfection. For mitotic index calculation in Fig. S1, both tumor cells and primary cells were fixed at 60 h after RNAi followed by immunostaining. For FACS analysis of apoptotic cells in Fig. 1 and Fig. S1, both tumor cells and primary cells were analyzed 72 h after siRNA transfection.  FACS analyses of apoptotic cells  \n",
      "\n",
      "MD Headers: {'methods': ''} \n",
      "Other Headers: ['title_1', 'paragraph', 'title_1', 'paragraph'] \n",
      "Size change: 1010 ? 972\n",
      "FACS analyses of apoptotic cells  FITC Annexin V Apoptosis Detection kit with 7-AAD (640922; BioLegend) was used to detect apoptosis. 5 x 105 cells that were treated by control or BuGZ siRNAs (Fig. 1 B) were trypsinized and resuspended as individual cells in 100 microl of Annexin V-binding buffer (140 mM NaCl, 4 mM KCl, 0.75 mM MgCl2, and 10 mM Hepes, pH 7.4). 5 microl FITC-Annexin V and 5 microl 7-AAD were added to each cell suspension. After incubation for 15 min at room temperature, cells were analyzed with FACS Aria III. For all samples, 100,000 events were recorded. FACSDIVA software (BD) was used for all flow cytometry analyses. Samples with buffer alone (background autofluorescence control), FITC-Annexin V, or 7-AAD staining alone (compensation controls for each fluorochrome) were used for compensation and gating for the apoptotic-positive cells. PE-Cy5 channel was used to detect 7-AAD fluorescence.  Immunofluorescence microscopy and quantifications  \n",
      "\n",
      "MD Headers: {'methods': ''} \n",
      "Other Headers: ['title_1', 'paragraph', 'paragraph', 'paragraph'] \n",
      "Size change: 1867 ? 1827\n",
      "Immunofluorescence microscopy and quantifications  To detect DNA-RNA hybrids (R-loops) using the monoclonal antibody S9.6 (provided by L. Wahba and D. Koshland, University of California, Berkeley, Berkeley, CA), we found that the treatment used for mitotic chromosome spread improves the detection of R-loops in the nuclei of HT29 cells, whereas standard immunostaining was sufficient for HFFs. We modified the chromosome-spread method to treat the interphase HT29 cells. Control or siRNA-treated HT29 cells were trypsinized and resuspended as single cell suspensions and then treated with hypotonic solution (75 mM KCl) prewarmed to 37 C. The cells were then fixed with freshly made fixative solution (methanol/glacial acetic acid 3:1) overnight at 4 C. To spread the cells on a coverslip, the fixed cell suspension was placed on the coverslip exposed to the steam of hot water (90 C) for 30 s, which caused the cells to spread their chromatin on the coverslip. These cells were blocked by blocking buffer (5% BSA and 0.5% Triton X-100 in PBS) for 1 h and then incubated with 1:1,000 dilution of the S9.6 antibody overnight in blocking buffer. After three washes with blocking buffer, the cells were incubated with 1:1,000 dilution of secondary antibody for 1 h. For detection of R-loops in HFFs, cells were fixed using ice-cold methanol for 15 min followed by treatment with 70% ethanol for 15 min. Cells were washed three times with 70% ethanol for 5 min each followed by blocking and antibody staining as above.  For immunodetection of alpha-tubulin, gamma-H2Ax, phospho-p53, BuGZ, and Bub3, cells grown on coverslips were fixed in 4% paraformaldehyde for 15 min and then permeabilized with 0.5% Triton X-100 in PBS for 5 min. The source and dilutions of the primary and secondary antibodies can be found in the Table S2.  \n",
      "\n",
      "MD Headers: {'methods': ''} \n",
      "Other Headers: ['paragraph', 'paragraph', 'paragraph', 'paragraph', 'paragraph', 'paragraph', 'title_1', 'paragraph'] \n",
      "Size change: 1998 ? 1914\n",
      "To co-stain phospho-p53 and Bub3 with two mouse monoclonal antibodies (Table S2), the antibodies were directly coupled with Alexa Fluor 546 or Alexa Fluor 647, respectively, by using an APEX Alexa Fluor antibody labeling kit (A20183 and A20186; Life Technologies).  To detect GFP or GFP fusion proteins, cells were fixed in the same manner and imaged directly. After antibody staining and washes, the cells were stained with DAPI at 0.25 microg/ml in PBS for 5 min and mounted in Fluorescence Mounting medium (Dako).  Fluorescence images were captured using a microscope (ECLIPSE TE2000-U; Nikon) driven by MetaMorph software (Molecular Devices) or a laser confocal microscope (TCS SP5; Leica) at the same setting throughout each given experiment. For confocal microscopy, 63x 1.4 NA oil and 40x 1.25 NA oil objective lenses (Leica) were used and the cells were imaged by scanning optical sections at ~0.5-microm intervals.  To quantify the mitotic index, the number of cells in prophase through telophase (judged by alpha-tubulin and DAPI staining) was divided by the total number of cells analyzed.  To quantify nuclear immunofluorescence intensity from confocal images, maximal projection was created from the z stacks for each nucleus. We used a method based on a previous study to determine the nuclear fluorescence signal from both confocal images and images from epifluorescence microscopes. In brief, the nuclear regions (X) were defined by DAPI staining. A larger area surrounding the nucleus was then drawn (Y). Areas of X and Y were determined as Xa and Ya. The mean fluorescence intensity for X and Y were measured as Xf and Yf. Signal intensity for each nucleus (Xi) was determined as Xi = Xf - (Yf x Ya - Xf x Xa)/(Ya - Xa).  All experiments were performed at least three times. A total of 17-159 nuclei were quantified from these experiments and plotted.  Senescence-associated (SA)-beta-gal assay  \n",
      "\n",
      "MD Headers: {'methods': ''} \n",
      "Other Headers: ['title_1', 'paragraph', 'title_1', 'paragraph', 'title_1', 'paragraph', 'paragraph'] \n",
      "Size change: 1976 ? 1907\n",
      "Senescence-associated (SA)-beta-gal assay  We used the Senescence beta-Galactosidase Staining kit (9860S; Cell Signaling) to detect beta-galactosidase activity at pH 6.0. In brief, cells were fixed in the fixative solution for 10-15 min at room temperature. The fixed cells were then incubated with beta-galactosidase staining solution containing X-gal at 37 C overnight. As the blue color developed, bright-field cell images were taken using an Axiovert 25 microscope (Carl Zeiss) connected to a Canon camera.  Live cell number count  A 0.4% Trypan blue stock solution (15250061; Life Technologies) was used to stain dead cells. 10 microl of Trypan blue stock solution was added to 100 microl of cell suspension. Cells negative for Trypan blue staining were counted as live cells using hemocytometer.  Immunoprecipitation and Western blotting analyses  To confirm the interaction between BuGZ or Bub3 and spliceosome components, we used two methods. In the first method, we transfected HEK293T cells with Flag-BuGZ, Flag-Bub3, or Flag-luciferase (negative control). After transfection, ~1 x 107 cells were lysed in 1 ml lysis buffer (20 mM Tris-Cl, pH 7.4, 150 mM KCl, 1 mM EDTA, 1 mM NaF, 1 mM DTT, 1% NP-40, 10% glycerol, and 1 mM sodium pyrophosphate) containing protease inhibitors (at 1 tablet/10 ml; cOmplete, Mini, EDTA-free, Protease Inhibitor Cocktail Tablets, 11836170001; Roche) at 4 C. After 10 min of centrifugation at 13,000 rpm in an Eppendorf tabletop centrifuge, the supernatant containing soluble proteins was incubated with a 30-microl slurry of anti-Flag M2 resin (A2220; Sigma-Aldrich) at 4 C for 4 h. After washing the beads with wash buffer (50 mM Tris-Cl, pH 7.4, 150 mM NaCl, 1 mM EDTA, 50 mM NaF, 1 mM DTT, and 1% NP-40) three times, the bound proteins were eluted with 100 microl Flag-peptide (F4799; Sigma-Aldrich) dissolved in the wash buffer at a concentration of 0.5 mg/ml.  \n",
      "\n",
      "MD Headers: {'methods': ''} \n",
      "Other Headers: ['paragraph', 'paragraph', 'title_1', 'paragraph', 'paragraph'] \n",
      "Size change: 1989 ? 1938\n",
      "In the second method, we used antibodies to BuGZ (HPA017013; Sigma-Aldrich), Bub3 (611731; BD), U2AF65 (ab37530; Abcam), Sf3a3 (ab156873; Abcam), or control IgG (12-370, 12-371; Millipore) to immunoprecipitate the respective endogenous proteins from HT29 cell lysates made from ~1 x 107 cells for each immunoprecipitation. Each antibody was preincubated with a 50-microl slurry of protein G Dynabeads (10007D; Life Technologies) for 30 min followed by washes with lysis buffer. The beads were added to the cell lysate and incubated overnight. The beads were then washed as described in the previous paragraph, and bound proteins were eluted with 100 microl of elution buffer (50 mM glycine at pH 2.5) and dissolved in the SDS sample buffer.  For Western blotting analyses of the Flag antibody immunoprecipitated proteins, 1% of input lysate and 10% of resuspended immunoprecipitates were loaded per lane for SDS-PAGE. Because the molecular weights of the proteins of BuGZ, U2AF65, and Sf3a3 are close to the heavy chain of IgG, we reduced the signal from the antibody heavy chain by using the secondary antibody specific to the rabbit light chain (211-032-171; Jackson ImmunoResearch Laboratories, Inc.). Flag (F3165; Sigma-Aldrich), BuGZ (HPA017013; Sigma-Aldrich), Bub3 (611731; BD), U2AF65 (ab37530; Abcam), or Sf3a3 (ab156873; Abcam) antibodies were used at the indicated dilutions (Table S2).  Whole-transcriptome shotgun sequencing (RNA-Seq) and data analyses  Arcturus PicoPure RNA isolation kit (KIT0204; Life Technologies) was used to prepare total RNA from cells. Ribo-Zero Gold kits (human/mouse/rat; MRZG126; Epicentre) were used to remove both cytoplasmic (nuclear-encoded) rRNA and mitochondrial rRNA from total RNA preparations, and libraries were built using the Illumina TruSeq RNA sample prep kit V2 (RS-122-2001; Illumina). Libraries were sequenced using an Illumina HiSeq 2000. Single-end 100-bp reads were obtained.  \n",
      "\n",
      "MD Headers: {'methods': ''} \n",
      "Other Headers: ['paragraph', 'title_1', 'paragraph', 'paragraph', 'title_1', 'paragraph', 'paragraph', 'paragraph'] \n",
      "Size change: 1944 ? 1862\n",
      "RNA-Seq reads were first mapped to the human genome (hg19) using Tophat2. Transcript annotation from Ensembl was used as reference for exon junctions in read mapping. We then used Cufflinks to assemble unannotated transcripts. Assembled transcripts from the same cell type were further merged by Cuffmerge. Finally, we used MATS to call differential alternative splicing, and the merged transcripts from the previous step were used as transcript annotation in MATS.  Drosophila stocks  The w1118 allele (Bloomington Drosophila Stock Center no. 3605) was used as the wild-type control. The RNAi strains for BuGZ were obtained from the Vienna Drosophila RNAi library: w1118; P{GD9485}v45449 (Vienna Drosophila Resource Center no. v45449). dsRNA-GD9485 (2L:16,744,705 ... 16,745,060) was inserted in the third chromosome.  The UAS strains used were: UAS-upd transgenic fly generated through transformation of UAS-upd construct into W1118 flies by standard methods (provided by X. Chen, Johns Hopkins University, Baltimore, MD) and UAS-ykiS111A.S168A.S250A.V5 (Bloomington Drosophila Stock Center no. 28817). The Gal4 allele used to drive the UAS lines is ey-Gal4 (Bloomington Drosophila Stock Center no. 5534), which is expressed in the Drosophila eyes.  Drosophila genetics and culture  If not specifically mentioned, all the flies were maintained on standard cornmeal/molasses/yeast fly food (the recipe for 1 liter of food: 50 g cornmeal, 18.75 g yeast, 5 g agar, 62.5 ml molasses, 11.5 ml 5% anti-fungal agent Tegosept, and 35 ml propionic acid). Flies were cultured at 25 C on a normal light/dark cycle condition, unless otherwise indicated.  To induce the tumor formation in the Drosophila eyes, ey-Gal4 was used to drive oncogene expression specifically in the eye. UAS-upd and UAS-ykiS111A.S168A.S250A.V5 transgenic flies were used to generate eye tumors.  \n",
      "\n",
      "MD Headers: {'methods': ''} \n",
      "Other Headers: ['paragraph', 'paragraph', 'paragraph', 'paragraph', 'title_1', 'paragraph', 'title_1', 'paragraph'] \n",
      "Size change: 1994 ? 1912\n",
      "To investigate the tumor repression effect of BuGZ RNAi, we used ey-Gal4 to drive both UAS-upd (or UAS-ykiS111A.S168A.S250A.V5) and UAS-BuGZ RNAi. The crosses were set up at 25 C, and eye tumor formation was examined after eclosion.  To evaluate the tumor repression effect of the splicing inhibitor pladienolide B, ey-Gal4>UAS-upd (or UAS-ykiS111A.S168A.S250A.V5) flies were fed with DMSO/pladienolide B (5 microM) from the first instar larval stage. Eye tumor formation was examined after eclosion.  To quantify the area of each Drosophila eye, bright-field images of the eyes were taken by placing the fly's head dorsal-side-up under a stereomicroscope (SMZ1500; Nikon) with a camera (AxioCam HRC; Carl Zeiss). This allows us to visualize the contour and half of the total area of the fly eyes. The rough eyes have a rough contour and a larger area than normal eyes, which are quantified by measuring the pixel number in each eye area using Photoshop CS5 (Adobe).  To investigate the RNAi knockdown efficiency of the UAS-BuGZ RNAi allele, tub-Gal4 (Bloomington Drosophila Stock Center no. 5138) was used to drive the ubiquitous expression of UAS-BuGZ RNAi, with the UAS-LacZ RNAi allele as the control. Total RNAs were collected at either the third instar larval stage or pupal stage, and qRT-PCR was performed to examine the mRNA level of BuGZ.  Scanning EM  Drosophila heads were excised and placed in McDowell's and Trump's 4F:1G fixative for 1 h to several weeks at 4 C. After two rinses in 0.1 M sodium phosphate buffer (pH 7.2), samples were dehydrated in a graded ethanol series to 100% ethanol, at which time they were critical point-dried with liquid CO2. Finally, they were mounted on specimen stubs with colloidal silver, sputter-coated with gold-palladium, and examined with a scanning electron microscope (Quanta 200 Environmental Scanning Electron Microscope; FEI).  Quantitative real-time PCR  \n",
      "\n",
      "MD Headers: {'methods': ''} \n",
      "Other Headers: ['title_1', 'paragraph', 'title_1', 'supplementary material', 'paragraph'] \n",
      "Size change: 1888 ? 1819\n",
      "Quantitative real-time PCR  Total RNA was isolated from Drosophila larvae or pupae using an RNA isolation kit (RNeasy Plus Mini kit; QIAGEN) and the manufacturer's protocol. Quantitative RT-PCR was performed using the iScript One Step RT-PCR kit (170-8892; Bio-Rad Laboratories) on a real-time PCR detection system (CFX96; Bio-Rad Laboratories). 50 ng of total RNA was reverse transcribed and amplified as follows: 50 C for 10 min, 95 C for 5 min, 95 C for 10 s, 60 C for 30 s, 72 C for 1 min. Steps 2-4 were repeated for 40 cycles. Each reaction was performed in triplicate, and the results of three independent experiments were used for statistical analysis. Relative mRNA expression levels were quantified using the DeltaDeltaC(t) method. Results were normalized to those for RP49, and primer sequences were as follows. Rp49 (Drosophila): forward primer, TACAGGCCCAAGATCGTGAA; and reverse primer, TCTCCTTGCGCTTCTTGGA. BuGZ (Drosophila): forward primer, AAGCCACCTTAGGATCATGG; and reverse primer, GGGTCCGGTGTACAACTTCT.  Fig. S1 shows the effect of BuGZ reduction on the mitotic index and cell proliferation. Fig. S2 shows the tumor repression effect of BuGZ RNAi or pladienolide B treatment. Fig. S3 shows that DNA damage caused by splicing inhibitor treatment can be rescued by forced expression of RNase H1. Table S1 gives a list of differential alternative splicing events caused by BuGZ RNAi, Bub3 RNAi, or pladienolide B treatment. Table S2 gives the sources and sequences of mammalian expression plasmids, antibodies (with dilutions used), siRNA, shRNA, small molecules (with concentrations used), and cell lines. Online supplemental material is available at http://www.jcb.org/cgi/content/full/jcb.201409073/DC1. Additional data are available in the JCB DataViewer at http://dx.doi.org/10.1083/jcb.201409073.dv.\n"
     ]
    }
   ],
   "source": [
    "# Pipeline: Markdown + Recursive \n",
    "\n",
    "metadata_extractors = [\n",
    "    # QuestionsAnsweredExtractor(questions=3),\n",
    "    # KeywordExtractor(keywords=5)\n",
    "    ]\n",
    "\n",
    "pipeline = IngestionPipeline(transformations=metadata_extractors)\n",
    "\n",
    "nodes = []\n",
    "splits = _pseudo_markdown_splitter(text, chunk_size=2000, chunk_overlap=125, markdown_headers=markdown_headers, exclude_headers=exclude_headers)\n",
    "\n",
    "for doc in splits:\n",
    "    split = _split_non_md_headers(doc, headers=other_headers)\n",
    "\n",
    "    print(f'\\nMD Headers: {doc.metadata} \\nOther Headers: {split[\"headers\"]} \\nSize change: {len(doc.page_content)} ? {len(split[\"text\"])}')\n",
    "    print(f'{split[\"text\"]}')\n",
    "\n",
    "    # Avoid creating a node with empty text\n",
    "    if len(split[\"text\"]) == 0: continue\n",
    "\n",
    "    nodes.append(TextNode(text=split['text']))\n",
    "\n",
    "docs = pipeline.run(documents=nodes)\n",
    "index = VectorStoreIndex(nodes=docs)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 112,
   "metadata": {},
   "outputs": [],
   "source": [
    "from llama_index.core.response.notebook_utils import display_source_node\n",
    "\n",
    "base_retriever = index.as_retriever(similarity_top_k=3)\n",
    "\n",
    "query_str = \"What is the use of RT-PCR kit?\"\n",
    "base_nodes = base_retriever.retrieve(query_str)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "for node in base_nodes: \n",
    "    print('')\n",
    "    print(node.get_content(metadata_mode='all'))\n",
    "    # print(node.metadata)\n",
    "    # print(node.embedding)\n",
    "    # display_source_node(node, source_length=500)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Pipeline: Topic Node Parsing"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 163,
   "metadata": {},
   "outputs": [],
   "source": [
    "from llama_index.node_parser.topic import TopicNodeParser\n",
    "\n",
    "def topic_parse(documents, chunk_size=1000, similarity_threshold=0.8, window_size=2, similarity_method=\"embedding\"):\n",
    "    \"\"\" A function to parse the text into topic nodes. \"\"\"\n",
    "    \n",
    "    node_parser = TopicNodeParser.from_defaults(\n",
    "        embed_model=embed_model,\n",
    "        llm=llm,\n",
    "        max_chunk_size=chunk_size,\n",
    "        similarity_method=similarity_method,  # can be \"llm\" or \"embedding\"\n",
    "        similarity_threshold=similarity_threshold,\n",
    "        window_size=window_size,  # paper suggests window_size=5\n",
    "    )\n",
    "    return {\"nodes\": node_parser.get_nodes_from_documents(documents, show_progress=True), \n",
    "            \"params\": {\"chunk_size\": chunk_size, \"threshold\": similarity_threshold, \"window\": window_size}}\n",
    "\n",
    "    # return {\"nodes\": node_parser(documents), \n",
    "    #         \"params\": {\"chunk_size\": chunk_size, \"threshold\": similarity_threshold, \"window\": window_size}}\n",
    "\n",
    "\n",
    "def print_topic_nodes(parsed):\n",
    "    print(parsed['params'])\n",
    "    for i in range(len(parsed['nodes'])):\n",
    "        print(f'node {i}:\\n{parsed[\"nodes\"][i].get_content()}')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "<bound method TextNode.get_content of Document(id_='c1ab428e-a608-484d-bb6e-5a5b3fda3b35', embedding=None, metadata={}, excluded_embed_metadata_keys=[], excluded_llm_metadata_keys=[], relationships={}, text='Mitotic regulators BuGZ and Bub3 play a critical role in RNA splicing during interphase, and disruption of this function leads to R-loop formation, DNA damage, and p53 activation.', mimetype='text/plain', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n')>"
      ]
     },
     "execution_count": 229,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "whole_text = \"\"\n",
    "nodes = []\n",
    "splits = _pseudo_markdown_splitter(text, chunk_size=1000, chunk_overlap=0, markdown_headers=markdown_headers, exclude_headers=exclude_headers)\n",
    "\n",
    "for doc in splits:\n",
    "    split = _split_non_md_headers(doc, headers=other_headers)\n",
    "\n",
    "    # print(f'\\nMD Headers: {doc.metadata} \\nOther Headers: {split[\"headers\"]} \\nSize change: {len(doc.page_content)} ? {len(split[\"text\"])}')\n",
    "    # print(f'{split[\"text\"]}')\n",
    "\n",
    "    # Avoid creating a node with empty text\n",
    "    if len(split[\"text\"]) == 0: continue\n",
    "\n",
    "    nodes.append(Document(text=split['text']))\n",
    "\n",
    "# print(whole_text)\n",
    "nodes[1].get_content"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "nodes "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 231,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "application/vnd.jupyter.widget-view+json": {
       "model_id": "f93d7cc1b46a4ee7b2087676554f4051",
       "version_major": 2,
       "version_minor": 0
      },
      "text/plain": [
       "Parsing nodes:   0%|          | 0/56 [00:00<?, ?it/s]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "No valid JSON found in the response: assistant: It seems there is no content provided for me to decompose. Please provide the content, and I will follow the rules to create a list of simple propositions.\n",
      "\n",
      "If you're ready to proceed, please go ahead and share the content, and I'll get started!\n",
      "Failed to parse JSON: [ \n",
      "\"The solution contained the wash buffer.\"\n",
      "\"The concentration of the wash buffer was 0.5 milligrams per milliliter.\" ]\n",
      "{'chunk_size': 2000, 'threshold': 0.8, 'window': 5}\n",
      "node 0:\n",
      "The article is distributed under a specific license for the first six months after publication. The license is Attribution-Noncommercial-Share Alike-No Mirror Sites. After six months, the article will be available under another Creative Commons License. This other license is Attribution-Noncommercial-Share Alike 3.0 Unported.\n",
      "node 1:\n",
      "Wan Yihang has a surname and given name. Yihang Wan's surname is Wan and his given name is Yihang. Zheng Xiaobin has a surname and given name. Xiaobin Zheng's surname is Zheng and her given name is Xiaobin. Chen Haiyang has a surname and given name. Haiyang Chen's surname is Chen and his given name is Haiyang. Guo Yuxuan has a surname and given name. Yuxuan Guo's surname is Guo and his given name is Yuxuan. Jiang Hao has a surname and given name. Hao Jiang's surname is Jiang and his given name is Hao. He Xiaonan has a surname and given name. Xiaonan He's surname is He and her given name is Xiaonan. Zhu Xueliang has a surname and given name. Xueliang Zhu's surname is Zhu and his given name is Xueliang. Zheng Yixian has a surname and given name. Yixian Zheng's surname is Zheng and her given name is Yixian.\n",
      "node 2:\n",
      "The role of mitotic regulator kills tumor cells. Mitotic regulators have a specific splicing function. This splicing function links R-loop-mediated DNA damage to tumor cell killing.\n",
      "node 3:\n",
      "BuGZ and Bub3 are mitotic regulators.\n",
      "node 4:\n",
      "BuGZ and Bub3 play a critical role in RNA splicing during interphase. RNA splicing during interphase is performed by BuGZ and Bub3. The function of BuGZ and Bub3 leads to R-loop formation, DNA damage, and p53 activation. Disruption of the function of BuGZ and Bub3 results in R-loop formation. Disruption of the function of BuGZ and Bub3 causes DNA damage. Disruption of the function of BuGZ and Bub3 triggers p53 activation.\n",
      "node 5:\n",
      "Studies suggest that perturbing mitotic progression leads to DNA damage and p53 activation, which in turn lead to either cell apoptosis or senescence. It remains unclear how mitotic defects trigger p53 activation.\n",
      "node 6:\n",
      "BuGZ interacts with the splicing machinery Bub3 interacts with the splicing machinery BuGZ is a mitotic regulator localized in the interphase nucleus. Bub3 is a mitotic regulator localized in the interphase nucleus. The depletion of BuGZ led to increased formation of RNA-DNA hybrids (R-loops). The depletion of Bub3 led to increased formation of RNA-DNA hybrids (R-loops). The inhibition of RNA splicing by pladienolide B led to increased formation of RNA-DNA hybrids (R-loops). Increased formation of RNA-DNA hybrids (R-loops) leads to DNA damage. DNA damage leads to p53 activation. p53 activation leads to either cell apoptosis or senescence. The depletion of BuGZ led to DNA damage and p53 activation in human tumor cells. The depletion of Bub3 led to DNA damage and p53 activation in human tumor cells. The depletion of BuGZ led to DNA damage and p53 activation in primary cells. The depletion of Bub3 led to DNA damage and p53 activation in primary cells. R-loop-mediated DNA damage and p53 activation offer a mechanistic explanation for apoptosis of cancer cells. R-loop-mediated DNA damage and p53 activation offer a mechanistic explanation for senescence of primary cells. The disruption of the dual-function mitotic regulators leads to apoptosis of cancer cells. The disruption of the dual-function mitotic regulators leads to senescence of primary cells. Understanding the full range of functions of mitotic regulators is important for developing antitumor drugs.\n",
      "node 7:\n",
      "Extensive studies have shown that prolonged mitotic arrest can lead to DNA damage. Prolonged mitotic arrest can lead to p53 activation. P53 activation would explain why targeting mitotic regulators could be effective for cancer therapy. The process by which mitotic arrest leads to DNA damage and p53 activation is not fully understood in some contexts. Prolonged mitosis is proposed to cause DNA or cellular damage that would in turn activate p53. Prolonged mitotic arrest has been shown to cause Caspase activation. Caspase activation could lead to CAD (Caspase-activated DNase) activation. CAD could lead to DNA damage and p53 activation. How prolonged mitosis activates Caspases is not clear in this context. The mitotic timer has been suggested to sense the prolonged mitotic arrest. The mitotic timer senses the prolonged mitotic arrest either in a p53-dependent manner or independently of p53. A p53-dependent timer could link prolonged mitotic block to p53 activation. Neither the nature of this timer nor the signal that activates p53 has been understood.\n",
      "node 8:\n",
      "The difficulty in defining the mitotic trigger for DNA damage and p53 activation could be because we have not looked at the right stage of the cell cycle. Several mitotic regulators are found in the interphase nucleus. P53 activation could be caused by the disruption of the interphase nuclear functions of these mitotic regulators. A nuclear zinc finger protein called BuGZ has been shown to regulate mitosis. BuGZ directly binds to the spindle assembly checkpoint protein Bub3 to promote its loading to kinetochores and chromosome alignment. Bub3 is also localized to the interphase nucleus. The interaction between BuGZ and Bub3 can be detected throughout the cell cycle. Depleting BuGZ in various cancer cell lines resulted in a reduction of kinetochore Bub3 levels, chromosome misalignment, and mitotic block. BuGZ depletion caused most of the tumor cells to undergo mitotic death when experiencing a prolonged mitotic block.\n",
      "node 9:\n",
      "We investigated the mitotic catastrophe phenomenon. By investigating this mitotic catastrophe phenomenon, we uncovered an unrecognized interphase nuclear function of BuGZ. BuGZ has an unrecognized interphase nuclear function. We also uncovered an unrecognized interphase nuclear function of Bub3. Bub3 has an unrecognized interphase nuclear function. The interphase function of BuGZ and Bub3 helps to explain why the disruption of these two mitotic regulators could lead to p53 activation. The disruption of BuGZ and Bub3 can lead to p53 activation.\n",
      "node 10:\n",
      "Depletion of BuGZ causes apoptosis in cancer cells. Apoptosis in cancer cells caused by Depletion of BuGZ. Senescence was observed in primary fibroblasts due to Depletion of BuGZ. Previous studies have demonstrated that BuGZ depletion destabilizes Bub3. BuGZ depletion causes chromosome misalignment and mitotic arrest followed by massive cell death. To further study the function of BuGZ, siRNA was used to deplete the protein in three cancer cell lines: HeLa, HT29, or TOV21G. Primary human foreskin fibroblasts (HFFs) were also treated with siRNA to deplete BuGZ. Consistent with the role of BuGZ in maintaining Bub3 protein level, BuGZ depletion led to a reduction of Bub3 protein by 60 hours after siRNA treatment. A rise in mitotic index was observed (Fig. S1 A). BuGZ is required for efficient chromosome alignment in both cancer cells and HFFs. The reduction of Bub3 upon BuGZ depletion would be expected to result in inefficient chromosome alignment.\n",
      "node 11:\n",
      "Reduction of BuGZ causes apoptosis in human cancer cells Apoptosis in human cancer cells occurs due to reduction of BuGZ BuGZ depletion by siRNA treatment in cells caused a significant decrease in its levels Cells were analyzed 60 hours after siRNAs transfection for the effect of BuGZ reduction\n",
      "node 12:\n",
      "Alpha-tubulin was used as a loading control in Western blot analysis\n",
      "node 13:\n",
      "Relative levels of BuGZ or Bub3 compared to controls were determined and set at 1\n",
      "node 14:\n",
      "More than 90% of BuGZ was depleted by RNAi treatment BuGZ depletion caused the induction of apoptosis in human cancer cells Apoptotic cells were quantified using FACS assays for Annexin V positivity Early apoptotic cells were positive for Annexin V alone, while late apoptotic cells were positive for both Annexin V and 7-aminoactinomycin D (7-AAD) Results from three independent experiments showed consistent percentages of total cells undergoing apoptosis The percentage of early apoptotic cells was presented as mean +- SD Late apoptotic cells were also quantified using FACS assays for double positivity with Annexin V and 7-AAD BuGZ reduction caused the induction of senescence in primary foreskin fibroblasts (HFFs) Representative microscopic fields of HFFs showed the effect of BuGZ reduction on cell morphology at 72 hours after RNAi treatment A bar of 100 microm was used to measure the size of cells in microscopic fields Growth curves of human cancer cells and primary foreskin fibroblasts treated with siRNAs were analyzed for the effect of BuGZ reduction\n",
      "node 15:\n",
      "control or BuGZ siRNA was used.\n",
      "node 16:\n",
      "Error bars in Figure E indicated the standard deviation (SD). Student's t test results were as follows: n.s., not significant; *, P < 0.05; **, P < 0.01; ***, P < 0.001 from three independent experiments.\n",
      "node 17:\n",
      "Stable knockdown of BuGZ was achieved with lentivirus-expressed shRNA in HFFs. The virus-infected cells were enriched by puromycin selection. Cell lysates were collected at day 5 after transfection. Growth curves of HFFs at the indicated days after shRNA transfection showed significant differences.\n",
      "node 18:\n",
      "Error bars in Figure F indicated SD. Student's t test results for growth curves: **, P < 0.01; ***, P < 0.001 from three independent experiments. Increased senescence was observed in BuGZ shRNA-transfected HFFs as judged by SA-beta-gal staining (arrowheads). Bar length in Figure G was 50 microm. Percentage of SA-beta-gal-positive cells at the indicated days after shRNA transfection showed significant differences. Error bars in Figure H indicated SEM. Student's t test results for senescence: ***, P < 0.001 from three independent experiments.\n",
      "node 19:\n",
      "By five days of siRNA treatment, all three cancer cell lines exhibited a large increase in apoptosis. Apoptosis in the cancer cell lines was judged by early and late apoptotic markers Annexin V and 7-AAD. The increase in apoptosis correlated with a striking reduction in cell numbers in the cancer cell lines.\n",
      "node 20:\n",
      "BuGZ siRNA treatment of HFFs did not affect cell survival.\n",
      "node 21:\n",
      "HFFs exhibited a moderate elevation in mitotic index after BuGZ siRNA treatment.\n",
      "node 22:\n",
      "Primary somatic cells usually grow more slowly than tumor cells.\n",
      "node 23:\n",
      "A time greater than five days might be required for properly assessing the effect of BuGZ depletion on HFFs. We used lentivirus-mediated RNAi to stably knock down BuGZ in HFFs. The proliferation rate of the BuGZ-RNAi HFFs began to significantly slow down from day eight after infection. The BuGZ-RNAi HFFs showed a threefold increase of senescent cells at thirty days compared with controls.\n",
      "node 24:\n",
      "BuGZ depletion was indicated by indicator H. Indicator H signified an accelerated senescence of BuGZ-depleted HFFs. The accelerated senescence was a result of BuGZ depletion in HFFs. BuGZ depletion caused apoptosis in cancer cells. Apoptosis in cancer cells was a consequence of BuGZ depletion. BuGZ depletion led to senescence in primary HFFs. Senescence in primary HFFs resulted from BuGZ depletion.\n",
      "node 25:\n",
      "Depletion of BuGZ leads to p53 activation p53 accumulation causes apoptosis or senescence depending on cellular context\n",
      "node 26:\n",
      "HeLa cells have wild-type p53 The human papillomavirus E6 proteins target p53 for degradation in HeLa cells under normal conditions DNA damage or oxidation can induce p53 accumulation in HeLa cells HT29 cells have a point mutation in the transactivation domain of p53 (R273H) Mutant p53 in HT29 cells induces apoptosis or cell cycle arrest p53 activation causes senescence in primary cells with intact cell cycle checkpoints The levels of total p53 are increased upon BuGZ depletion in cancer cells and HFFs The levels of activated p53 (phospho-p53 at serine 15) are increased upon BuGZ depletion in cancer cells and HFFs\n",
      "node 27:\n",
      "Activation of p53 upon BuGZ depletion leads to apoptosis or senescence. The up-regulation of p53 was observed after 48 h of BuGZ siRNA treatment.\n",
      "node 28:\n",
      "The alpha-tubulin was used as a loading control. Phospho-p53 (Ser15) levels were increased in the nuclei of BuGZ-depleted cells. BuGZ-positive cells had no phospho-p53 detected. PFT-alpha inhibited p53 activity, leading to an increase in viable BuGZ-depleted cells. The number of viable BuGZ-depleted cells was significantly higher compared to the control. p53 RNAi repressed senescence induced by BuGZ knockdown in HFFs. SA-beta-gal staining was used to quantify senescence in HFFs. Senescence induction by BuGZ knockdown was significantly reduced with p53 RNAi in HFFs. The effect of p53 RNAi on senescence induction was consistent across three independent experiments.\n",
      "node 29:\n",
      "Results were obtained from three independent experiments. Each experiment was conducted independently of the others.\n",
      "node 30:\n",
      "Our previous studies indicated many HeLa cells depleted of BuGZ initially arrested in mitosis and then underwent cell death. We treated cells with p53 inhibitor PFT-alpha 12 h after transfection of control or BuGZ siRNA to test whether p53 activation could be the cause of cancer cell death. The inhibition of p53 significantly rescued the cell number reduction of the BuGZ-depleted cancer cells. Cells were found to be treated with p53 inhibitor PFT-alpha 12 hours after transfection. PFT-alpha is a p53 inhibitor. We coinfected HFFs with lentivirus to express p53 shRNA and observed its effects. Senescence induced by BuGZ depletion upon prolonged culturing was significantly inhibited in HFFs. Depletion of BuGZ down-regulated Bub3. Bub3 reduction could also lead to p53 activation because it is down-regulated by BuGZ depletion. Knockdown of Bub3 by RNAi caused prominent p53 accumulation and activation in both cancer cells and HFFs. P53 mediates apoptosis in cancer cells upon BuGZ depletion. P53 also mediates senescence in HFFs upon BuGZ depletion.\n",
      "node 31:\n",
      "BuGZ and Bub3 are required for proper pre-mRNA splicing. The depletion of BuGZ or Bub3 could activate p53. BuGZ and Bub3 bind to one another. BuGZ and Bub3 are both present in the interphase nucleus. Their depletion could disrupt an interphase nuclear function. This might trigger p53 activation. We examined our previous proteome analyses of BuGZ-interacting proteins. Besides Bub3, several spliceosome components were noticed among the top candidates of BuGZ-interacting proteins. Among available antibodies to spliceosome components, two: U2AF65 and SF3a3: worked in both immunoprecipitation and Western blotting. Reciprocal immunoprecipitation was used to provide further support for an interaction between BuGZ/Bub3 and the spliceosome. Both BuGZ and Bub3 have been found in previous proteomic analyses of spliceosomes.\n",
      "node 32:\n",
      "A requirement exists for BuGZ and Bub3 in RNA splicing. Proteins with known functions in RNA splicing were found as candidate interacting proteins of BuGZ. BuGZ-interacting proteins include proteins with known functions in RNA splicing. FLAG-tagged BuGZ or Bub3 was immunoprecipitated from HEK293T cells. SF3a3 and U2AF65 components of the spliceosome were immunoprecipitated by FLAG-tagged BuGZ or Bub3. FLAG-luciferase did not immunoprecipitate SF3a3 and U2AF65 in HEK293T cells. FLAG-luciferase is a non-interacting control protein. BuGZ interacted with endogenous SF3a3 and U2AF65 in vivo. Bub3 interacted with endogenous SF3a3 and U2AF65 in vivo. Reciprocal immunoprecipitations of endogenous proteins were performed using HT29 cell lysates. Endogenous proteins in HT29 cells included BuGZ, Bub3, SF3a3, and U2AF65. Immunoprecipitations used mouse monoclonal (mAb) or rabbit antibodies (rAb). Antibody heavy chain (h.c.) was present in immunoprecipitations. Experiments were performed three times to confirm results. Pladienolide B is a splicing inhibitor that affects RNA splicing. RNAi of BuGZ or Bub3 affected RNA splicing in HFFs and TOV21G cells. Splicing defects were analyzed by RNA-sequencing in treated cells. Pie charts showed the numbers of genes exhibiting splicing defects. Genes BRD2 and WAC exhibited intron retention upon treatment with pladienolide B or RNAi of BuGZ or Bub3.\n",
      "node 33:\n",
      "RNAi of BuGZ was performed.\n",
      "node 34:\n",
      "The red boxes highlighted examples of intron retention. Intron retention in BuGZ or Bub3 was shown. Examples of intron retention were identified in BuGZ. Examples of intron retention were also identified in Bub3. The fold increases of the intron retention were plotted to the right. The p-values for differential alternative splicing were generated by simulation. A Bayesian framework called MATS was used for flexible detection of differential alternative splicing from RNA-Seq data.\n",
      "node 35:\n",
      "We tested whether BuGZ and Bub3 play roles in splicing. To test this, we depleted each protein in cancer cells (TOV21G) or human foreskin fibroblasts (HFFs). We performed RNA sequencing with accession number GSE67770. Cells treated with pladienolide B were used as positive controls for splicing inhibition. Depletion of either Bub3 or BuGZ mainly caused exon skipping in cancer cells and HFFs. Pladienolide B treatment, which targets SF3b3, mainly resulted in intron retention (Fig. 3, D and E; and Table S1). The difference in splicing defects between Pladienolide B inhibition and BuGZ or Bub3 depletion suggests that BuGZ and Bub3 function differently from SF3b3. Further studies are required to understand how BuGZ and Bub3 regulate splicing. These analyses show that the well-established SAC component Bub3 and its regulator BuGZ also play roles in interphase RNA splicing. Depletion of BuGZ in transformed cells or treatment of tumor cells with pladienolide B led to changes in RNA splicing.\n",
      "node 36:\n",
      "B have been shown to kill these cells in vitro. Using the Drosophila melanogaster eye tumor model was an interesting approach. The activation of either the Jak-Stat or the Hippo pathway induced eye hyperproliferation. The eye-specific BuGZ RNAi inhibited the eye hyperproliferation. Pladienolide B had the same effect as BuGZ RNAi in inhibiting the eye hyperproliferation when fed to flies. Data on the Hippo pathway were unpublished.\n",
      "node 37:\n",
      "The depletion of BuGZ causes R-loop formation. R-loop formation occurs due to the depletion of BuGZ. Depletion of Bub3 also causes R-loop formation. Bub3 is a protein that, when depleted, leads to R-loop formation and DNA damage. R-loops are structures that form in DNA when certain conditions occur. DNA damage results from the depletion of BuGZ or Bub3. BuGZ and Bub3 have roles in maintaining the integrity of DNA. The absence or reduction of BuGZ or Bub3 leads to a specific type of DNA structure (R-loops). This specific structure (R-loops) is associated with damage to the DNA.\n",
      "node 38:\n",
      "Deficiency in RNA splicing has been shown to cause an increased formation of RNA-DNA hybrids (R-loops) in the nucleus. Subsequent DNA breaks and p53 activation were caused by the deficiency in RNA splicing. p53-mediated killing of transformed cells or senescence of primary cells was observed due to defects in splicing and R-loop formation. Cells depleted of BuGZ had an increased formation of RNA-DNA hybrids (R-loops). The monoclonal antibody S9.6 recognizes RNA-DNA hybrids. Depletion of BuGZ or Bub3 markedly stimulated R-loop formation in cancer cells. Depletion of BuGZ or Bub3 also stimulated R-loop formation in HFFs. These cells exhibited increased DNA damage, as judged by the appearance of prominent nuclear foci of gammaH2AX. Increased DNA damage was observed in cancer cells depleted of BuGZ or Bub3. Cells with defects in splicing function exhibit R-loop-induced DNA damage. R-loop-induced DNA damage contributes to p53-mediated killing of transformed cells or senescence of primary cells.\n",
      "node 39:\n",
      "p53 activation occurred.\n",
      "node 40:\n",
      "Depletion of BuGZ or Bub3 leads to increased R-loop formation. The depletion of BuGZ or Bub3 resulted in increased R-loop formation in HT29 cells. Pladienolide B treatment was used as a positive control. Nuclei with no or low levels of R-loops are outlined by white broken circles.\n",
      "node 41:\n",
      "The size of the bar is 5 micrometers.\n",
      "node 42:\n",
      "The quantification of mean fluorescence intensity of the R-loop is shown on the right. Student's t test was used to compare the results, and the P value was less than 0.001 from three independent experiments.\n",
      "node 43:\n",
      "Depletion of BuGZ or Bub3 caused increased DNA damage in HT29 cells. The depletion of BuGZ or Bub3 by two different siRNAs caused increased DNA damage. Increased DNA damage was judged by the presence of prominent nuclear foci of gamma-H2Ax. BuGZ- or Bub3-depleted cells exhibit more gamma-H2Ax foci than cells with insufficient or no depletion. Cells with insufficient or no depletion are indicated by arrows.\n",
      "node 44:\n",
      "White broken circles outline the nuclei. The size of the bar is 10 micrometers.\n",
      "node 45:\n",
      "Quantification of gamma-H2Ax intensities in individual cells was performed. Distributions of gamma-H2Ax intensities were plotted for comparison. Error bars indicate SEM, which is the standard error of the mean. Student's t test was used to compare the results, and the P value was less than 0.001.\n",
      "node 46:\n",
      "Cell numbers (n) were analyzed from three independent experiments.\n",
      "node 47:\n",
      "The number of cells (n) was P < 0.001.\n",
      "node 48:\n",
      "R-loop formation was inhibited. Attenuation of R-loop formation occurred. P53 activation was attenuated.\n",
      "node 49:\n",
      "Bp53 activation was caused by depletion of BuGZ. DNA damage was caused by depletion of BuGZ.\n",
      "node 50:\n",
      "DNA damage was caused by depletion of Bub3.\n",
      "node 51:\n",
      "BuGZ was depleted. Bub3 was depleted.\n",
      "node 52:\n",
      "We used a modified form of RNase H1 lacking the mitochondria targeting sequence. This form of RNase H1 has been shown to localize only in the nucleus and to resolve R-loops. By analyzing the nuclei that were positive for the GFP-tagged RNase H1 or GFP control, we found Expression of the enzyme resulted in significant reduction of both gammaH2AX foci (Fig. 5, A-C) and p53 activation (Fig. 5, D-F) in BuGZ-depleted HT29 cells. A similar effect of RNase H1 was also observed in pladienolide B-treated (Fig. S3 B) Or Bub3-depleted (Fig. 5, G-J) HT29 cells. The inability of RNase H1 to fully rescue DNA damage and p53 activation could be caused by The heterogeneous level of GFP-RNase H1 expression. The inefficiency of GFP-RNase H1 to resolve certain R-loops, or other uncharacterized defects. Nonetheless, these findings show that R-loop formation due to the disruption\n",
      "node 53:\n",
      "The splicing function of Bub3 contributed to DNA damage. The splicing function of BuGZ contributed to DNA damage. Bub3 and BuGZ contributed to DNA damage DNA damage was related to p53 activation.\n",
      "node 54:\n",
      "The expression of RNase H1 in BuGZ- or Bub3-depleted HT29 cells reduces gamma-H2Ax and phospho-p53 signals. RNase H1 was expressed in cells. RNase H1 is a type of enzyme. BuGZ depletion was induced in HT29 cells. Bub3 depletion was also induced in HT29 cells. HT29 cells are a specific type of cell. RNase H1 expression repressed BuGZ depletion-induced DNA damage in HT29 cells. DNA damage was induced by BuGZ depletion. RNase H1, but not GFP, had this effect. GFP-RNase H1 and GFP were transfected into the cells for 48 hours. Immunofluorescence microscopy analyses were performed on the cells. Cells that were depleted of BuGZ but showed GFP fluorescence were indicated by arrowheads. Cells that were depleted of BuGZ but did not show GFP fluorescence were indicated by arrows.\n",
      "node 55:\n",
      "Nuclear boundaries were indicated by broken lines.\n",
      "node 56:\n",
      "The mean fluorescence intensity of gamma-H2Ax was quantified in HT29 cells. Intensity distributions of gamma-H2Ax were also quantified. Quantifications for the mean fluorescence intensity of gamma-H2Ax are presented in a graph format (B). Intensity distributions of gamma-H2Ax are presented in a graph format (C). GFP-RNase H1 repressed BuGZ depletion-induced p53 phosphorylation in HT29 cells. Arrowheads and arrows indicate cells with or without GFP-RNase H1, respectively. White broken circles outline the nuclei. Quantifications for the mean fluorescence intensity of phospho-p53 are presented in a graph format (E). Intensity distributions of phospho-p53 were also quantified. Intensity distributions of phospho-p53 are presented in a graph format (F).\n",
      "node 57:\n",
      "Experiments were shown in D. The experiments contained G and H quantifications for the mean fluorescence intensity of gamma-H2Ax. Quantifications G were for the mean fluorescence intensity of gamma-H2Ax Gamma-H2Ax's mean fluorescence intensity was measured in Bub3-depleted cells. Gamma-H2Ax's mean fluorescence intensity was also measured in GFP- or GFP-RNase H1-expressing HT29 cells. The experiments contained I and J quantifications for the mean fluorescence intensity of phospho-p53. Quantifications I were for the mean fluorescence intensity of phospho-p53 Phospho-p53's mean fluorescence intensity was measured in Bub3-depleted cells. Phospho-p53's mean fluorescence intensity was also measured in GFP- or GFP-RNase H1-expressing HT29 cells. Error bars indicate the SEM of the experiments. Student's t test results were used to determine statistical significance: P < 0.05 indicates a statistically significant difference with a confidence level of ** P < 0.01 indicates a statistically significant difference with a confidence level of ** P < 0.001 indicates a statistically significant difference with a confidence level of *** Cell numbers (n) analyzed were from three independent experiments.\n",
      "node 58:\n",
      "Extensive efforts have been devoted to understanding how mitotic arrest as a result of disrupting mitotic regulators can lead to p53-dependent cell death or cell cycle arrest in cancer cells. Mitotic arrest was studied as a consequence of disrupting mitotic regulators in cancer cells. The study aimed at understanding the connection between disrupting mitotic regulators and p53-dependent outcomes in cancer cells. Researchers explored the functions of mitotic regulators beyond their role in mitosis to gain insights into cancer cell behavior. Mitotic regulators BuGZ and Bub3 were found to have a critical role in interphase RNA splicing by researchers. The function of BuGZ and Bub3 in RNA splicing was discovered through research on the mitosis-independent functions of these proteins. Disrupting the interphase function of BuGZ or Bub3 leads to R-loop formation, DNA damage, and p53 activation, as shown by experiments. Depleting BuGZ or Bub3 resulted in R-loop formation, DNA damage, and p53 activation, according to research findings. Pladienolide B, a potent anticancer reagent, was found to kill cancer cells through a mechanism similar to disrupting the interphase function of BuGZ or Bub3. Experiments demonstrated that pladienolide B kills cancer cells through a mechanism related to disrupting the interphase function of mitotic regulators. Looking beyond the mitosis functions of established mitotic regulators provided significant insights into how aberrant mitosis affects cell proliferation and genome stability, according to research findings. The study broadened our understanding of the function of mitotic regulators by considering their roles in processes other than mitosis. Mitotic regulators are involved in maintaining genome stability and suppressing cell proliferation through mechanisms beyond their role in mitosis, as suggested by research results.\n",
      "node 59:\n",
      "The process known as cell division should be a factor in facilitating the selection of potential therapeutic targets. Cell division should play a role in enabling researchers to identify potential targets for new antitumor drugs. The purpose of developing new antitumor drugs is for use in cancer chemotherapy. Cancer chemotherapy involves the use of medications intended to treat cancer. Therapeutic targets are factors that, when altered or targeted by therapeutic agents, can help prevent or combat disease.\n",
      "node 60:\n",
      "For sources and sequences of mammalian expression plasmids, please refer to Table S2. Table S2 contains information about antibodies. The dilutions used for antibodies are mentioned in Table S2. Table S2 also includes information about siRNA. The information about shRNA is contained within Table S2. Small molecules and their concentrations are listed in Table S2. Table S2 provides details about cell lines. A plasmid for expressing GFP-RNase H1 was provided by R.J. Crouch. R.J. Crouch works at the National Institute of Child Health and Human Development. The National Institute of Child Health and Human Development is located in Bethesda, MD. Cell culture requires specific procedures which are not specified here. Plasmids must be transfected into cells using specific protocols which are not detailed here. siRNA transfection is a process that involves siRNA and requires specific conditions.\n",
      "node 61:\n",
      "The base medium for TOV21G cell culture was made from a 1:1 mixture of MCDB 105 medium and Medium 199. MCDB 105 medium contained a final concentration of 1.5 g/liter sodium bicarbonate. Medium 199 contained a final concentration of 2.2 g/liter sodium bicarbonate. Fetal bovine serum was added to the base medium at a final concentration of 15% to make the complete growth medium. The complete growth medium was used for HT29 and HeLa cell cultures. HT29 cells were cultured in Dulbecco's modified Eagle's medium containing 10% fetal bovine serum. HeLa cells were also cultured in Dulbecco's modified Eagle's medium containing 10% fetal bovine serum. Both HT29 and HeLa cells were cultured at 5% CO2 and 37 C. HFF cells were cultured in FibroGRO LS Complete Medium. The HFF cells were changed into fresh medium every other day. The HFF cells were passaged using Accutase at a 1:3 dilution when reaching ~80% of confluency.\n",
      "node 62:\n",
      "The plasmids or siRNA transfections were performed using Lipofectamine 2000. Lipofectamine 2000 was provided by Life Technologies. The siRNA transfections were performed using Lipofectamine RNAiMAX. Lipofectamine RNAiMAX was also provided by Life Technologies. Cells were fixed for immunofluorescence analyses after siRNA transfection. 48 hours had passed since the siRNA transfection when cells were fixed for analysis. Alternatively, cells could be fixed at other times if indicated. For cotransfection of siRNA and plasmids, siRNA was transfected first. The reverse transfection protocol was used to transfect siRNA first. 10 pmol of the siRNA oligo was diluted in Opti-MEM I Medium per well. Opti-MEM I Medium was provided by Gibco. 50 microl of Opti-MEM I Medium was used per well of a 24-well tissue culture plate. 1.5 microl of Lipofectamine RNAiMAX was added to each well.\n",
      "node 63:\n",
      "The mixture was incubated for 20 minutes at room temperature after mixing.\n",
      "node 64:\n",
      "Cells were trypsinized and resuspended in complete growth medium without antibiotics. A cell suspension containing 20,000 cells was added into each well for transfection. After 24 hours of transfection, cells were transfected with the indicated plasmids. For immunofluorescence analyses in Fig. 5, cells were fixed after plasmid transfection. 48 hours had passed since plasmid transfection when cells were fixed for analysis.\n",
      "node 65:\n",
      "Both tumor cells and primary cells were fixed at 60 hours after RNAi followed by immunostaining for mitotic index calculation in Fig. S1. Tumor cells and primary cells were fixed together. 60 hours was the time point of fixation after RNAi followed by immunostaining. Immunostaining was performed after RNAi. For FACS analysis of apoptotic cells, both tumor cells and primary cells were analyzed 72 hours after siRNA transfection in Fig. 1 and Fig. S1. Tumor cells and primary cells were analyzed together for FACS. 72 hours was the time point of analysis after siRNA transfection. siRNA transfection preceded FACS analysis.\n",
      "node 66:\n",
      "FACS analyses of apoptotic cells were performed using the FITC Annexin V Apoptosis Detection kit with 7-AAD. The FITC Annexin V Apoptosis Detection kit with 7-AAD was obtained from BioLegend (catalog number: 640922). A total of 5 x 10^5 cells were used for the analysis, which had been treated with either control or BuGZ siRNAs (as shown in Fig. 1 B). The cells were trypsinized and resuspended as individual cells in 100 microl of Annexin V-binding buffer. The Annexin V-binding buffer consisted of 140 mM NaCl, 4 mM KCl, 0.75 mM MgCl2, and 10 mM Hepes, pH 7.4. 5 microl FITC-Annexin V was added to each cell suspension. Additionally, 5 microl 7-AAD was added to each cell suspension. The cells were then incubated for 15 min at room temperature. After incubation, the cells were analyzed using a FACS Aria III. A total of 100,000 events were recorded for all samples. FACSDIVA software (BD) was used to perform all flow cytometry analyses. Compensation and gating controls were used for the apoptotic-positive cells, which included samples with buffer alone (background autofluorescence control), FITC-Annexin V staining alone, or 7-AAD staining alone. The PE-Cy5 channel was used to detect 7-AAD fluorescence.\n",
      "node 67:\n",
      "Immunofluorescence microscopy was used for analysis. The method of immunofluorescence microscopy allowed for quantifications to be performed. Quantifications were obtained using immunofluorescence microscopy.\n",
      "node 68:\n",
      "To detect DNA-RNA hybrids (R-loops), we used the monoclonal antibody S9.6. The monoclonal antibody S9.6 was provided by L. Wahba and D. Koshland of the University of California, Berkeley, Berkeley, CA. Treatment for mitotic chromosome spread improves detection of R-loops in nuclei of HT29 cells. Standard immunostaining was sufficient for HFFs. We modified the chromosome-spread method to treat interphase HT29 cells. Control or siRNA-treated HT29 cells were trypsinized and resuspended as single cell suspensions. The cells were treated with a hypotonic solution (75 mM KCl) prewarmed to 37 C. The cells were fixed with freshly made fixative solution (methanol/glacial acetic acid 3:1) overnight at 4 C. The fixed cell suspension was placed on a coverslip exposed to the steam of hot water (90 C) for 30 s. The heat caused the cells to spread their chromatin on the coverslip. These cells were blocked by blocking buffer (5% BSA and 0.5% Triton X-100 in PBS) for 1 hour.\n",
      "node 69:\n",
      "Cells were then incubated with a 1:1,000 dilution of the S9.6 antibody overnight. The cells were mixed with blocking buffer for this process. After that, the cells underwent three washes with blocking buffer. The cells were then incubated with a 1:1,000 dilution of secondary antibody for one hour. Cells in HFFs were treated with ice-cold methanol for fifteen minutes first. Then they were further treated with 70% ethanol for another fifteen minutes. Three additional washes of the cells with 70% ethanol were done for five minutes each time. Finally, blocking and antibody staining was performed on these cells as before.\n",
      "node 70:\n",
      "Cells grown on coverslips were fixed in 4% paraformaldehyde. The fixation lasted for 15 minutes. Cells were then permeabilized with 0.5% Triton X-100. Triton X-100 was dissolved in PBS. Permeabilization lasted for 5 minutes. Information about primary and secondary antibodies can be found in Table S2. Primary and secondary antibodies underwent dilution according to specifications in Table S2. Mouse monoclonal antibodies were used to co-stain phospho-p53 and Bub3. Phospho-p53 was stained with a mouse monoclonal antibody directly coupled with Alexa Fluor 546. Bub3 was stained with a mouse monoclonal antibody directly coupled with Alexa Fluor 647. Direct coupling of antibodies was performed using the APEX Alexa Fluor antibody labeling kit. The APEX Alexa Fluor antibody labeling kit was acquired from Life Technologies (kit numbers A20183 and A20186). Cells expressing GFP or GFP fusion proteins were fixed in the same manner as others. GFP-expressing cells were imaged directly without further staining. After antibody staining, cells underwent washes. Cells were stained with DAPI at a concentration of 0.25 micrograms per milliliter. DAPI was dissolved in PBS. Staining with DAPI lasted for 5 minutes. The cells were then mounted in Fluorescence Mounting medium (product of Dako).\n",
      "node 71:\n",
      "Fluorescence images were captured using a microscope. The microscope used was an ECLIPSE TE2000-U, made by Nikon. MetaMorph software, made by Molecular Devices, drove the microscope. Alternatively, a laser confocal microscope (TCS SP5) made by Leica was also used. The TCS SP5 confocal microscope had 63x 1.4 NA oil and 40x 1.25 NA oil objective lenses. These objective lenses were provided by Leica. Cells were imaged by scanning optical sections at intervals of approximately ~0.5 micrometers. The cells were analyzed to determine the mitotic index. To calculate the mitotic index, the number of cells in prophase through telophase was counted. The number of cells in prophase through telophase was determined by alpha-tubulin and DAPI staining. The total number of cells analyzed was also counted. The mitotic index was calculated by dividing the number of cells in prophase through telophase by the total number of cells analyzed.\n",
      "node 72:\n",
      "We created a maximal projection from z stacks for each nucleus to quantify nuclear immunofluorescence intensity from confocal images. The method we used was based on a previous study to determine the nuclear fluorescence signal from both confocal images and epifluorescence microscope images. The nuclear regions were defined by DAPI staining as area X. A larger area surrounding the nucleus was drawn as area Y. We determined the areas of X and Y as Xa and Ya, respectively. The mean fluorescence intensity for X and Y were measured as Xf and Yf, respectively. Signal intensity for each nucleus (Xi) was calculated using the formula Xi = Xf - ((Yf x Ya) - (Xf x Xa))/(Ya - Xa).\n",
      "node 73:\n",
      "We performed all experiments at least three times. The number of nuclei quantified from these experiments ranged from 17 to 159 and were plotted for analysis. We also conducted a Senescence-associated beta-gal assay as part of the experiment.\n",
      "node 74:\n",
      "We used the Senescence beta-Galactosidase Staining kit. The kit was purchased from Cell Signaling under catalog number 9860S. Cells were fixed in a fixative solution for 10-15 minutes at room temperature. The cells were then incubated with a beta-galactosidase staining solution containing X-gal. The incubation occurred at 37 degrees Celsius overnight. As the blue color developed, bright-field cell images were taken using an Axiovert 25 microscope. The microscope was connected to a Canon camera.\n",
      "node 75:\n",
      "A 0.4% Trypan blue stock solution was used to stain dead cells. The stock solution was purchased from Life Technologies under catalog number 15250061. 10 microliters of the Trypan blue stock solution were added to 100 microliters of cell suspension. Cells negative for Trypan blue staining were counted as live cells using a hemocytometer. Immunoprecipitation and Western blotting analyses were performed.\n",
      "node 76:\n",
      "We used two methods to confirm the interaction between BuGZ or Bub3 and spliceosome components. In the first method, we transfected HEK293T cells with Flag-BuGZ, Flag-Bub3, or Flag-luciferase (negative control). HEK293T cells are a cell line. We transfected these cells. Flag-BuGZ and Flag-Bub3 are proteins that were introduced into the HEK293T cells. Flag-luciferase is a protein used as a negative control. ~1 x 10^7 cells were lysed in 1 ml lysis buffer containing protease inhibitors. The lysis buffer contained Tris-Cl at pH 7.4, KCl, EDTA, NaF, DTT, NP-40, glycerol, and sodium pyrophosphate. Protease inhibitors were added to the lysis buffer. The protease inhibitor used was cOmplete, Mini, EDTA-free, Protease Inhibitor Cocktail Tablets from Roche. After transfection, the cells were lysed in the lysis buffer at 4 C. The soluble proteins in the supernatant were incubated with a 30-microl slurry of anti-Flag M2 resin at 4 C for 4 h. The anti-Flag M2 resin was from Sigma-Aldrich and had catalog number A2220. After washing the beads with wash buffer three times, the bound proteins were eluted with 100 microl Flag-peptide dissolved in the wash buffer. The Flag-peptide used was from Sigma-Aldrich and had catalog number F4799.\n",
      "node 77:\n",
      "We used antibodies to BuGZ that is known as HPA017013 from Sigma-Aldrich. The antibodies to BuGZ were made by Sigma-Aldrich. We used antibodies to Bub3 that is known as 611731 from BD. The antibodies to Bub3 were made by BD. We used antibodies to U2AF65 that is known as ab37530 from Abcam. The antibodies to U2AF65 were made by Abcam. We used antibodies to Sf3a3 that is known as ab156873 from Abcam. The antibodies to Sf3a3 were made by Abcam. We used control IgG that is known as 12-370 and 12-371 from Millipore. The control IgG was made by Millipore. Each antibody was preincubated with a 50-microl slurry of protein G Dynabeads. The protein G Dynabeads were made by Life Technologies. Each antibody was incubated for 30 min. The antibodies were washed with lysis buffer. We added the beads to cell lysate and incubated overnight. We washed the beads as described in the previous paragraph. Bound proteins were eluted with 100 microl of elution buffer at pH 2.5. The elution buffer was made with glycine. The bound proteins were dissolved in SDS sample buffer.\n",
      "node 78:\n",
      "1% of input lysate was loaded per lane for SDS-PAGE. 10% of resuspended immunoprecipitates were loaded per lane for SDS-PAGE.\n",
      "node 79:\n",
      "The molecular weights of BuGZ, U2AF65, and Sf3a3 are close to the heavy chain of IgG. We reduced the signal from the antibody heavy chain. A secondary antibody specific to the rabbit light chain was used. The name of the secondary antibody is 211-032-171. The secondary antibody is made by Jackson ImmunoResearch Laboratories, Inc. Flag antibodies were used at a certain dilution (Table S2). The Flag antibody is F3165 and it is from Sigma-Aldrich. BuGZ antibodies were used at a certain dilution (Table S2). The BuGZ antibody is HPA017013 and it is from Sigma-Aldrich. Bub3 antibodies were used at a certain dilution (Table S2). The Bub3 antibody is 611731 and it is from BD. U2AF65 antibodies were used at a certain dilution (Table S2). The U2AF65 antibody is ab37530 and it is from Abcam. Sf3a3 antibodies were used at a certain dilution (Table S2). The Sf3a3 antibody is ab156873 and it is from Abcam.\n",
      "node 80:\n",
      "The Arcturus PicoPure RNA isolation kit was used to prepare total RNA from cells. The Arcturus PicoPure RNA isolation kit has a product number of KIT0204 and is manufactured by Life Technologies. Ribo-Zero Gold kits were used to remove cytoplasmic rRNA and mitochondrial rRNA from total RNA preparations. The Ribo-Zero Gold kits are specific for use with human/mouse/rat RNA. The product number of the Ribo-Zero Gold kit is MRZG126, and it is manufactured by Epicentre. Libraries were built using the Illumina TruSeq RNA sample prep kit V2. The product number of the Illumina TruSeq RNA sample prep kit V2 is RS-122-2001, and it is manufactured by Illumina. The libraries were sequenced using an Illumina HiSeq 2000. Single-end 100-bp reads were obtained from the sequencing process. RNA-Seq reads were first mapped to the human genome hg19 using Tophat2. Transcript annotation from Ensembl was used as reference for exon junctions in read mapping. Cufflinks was used to assemble unannotated transcripts. Assembled transcripts from the same cell type were further merged by Cuffmerge. MATS was used to call differential alternative splicing. The merged transcripts from the previous step were used as transcript annotation in MATS. Drosophila stocks are not mentioned in relation to any specific process or kit, so no separate proposition is generated for this sentence.\n",
      "node 81:\n",
      "The w1118 allele was used as the wild-type control. The Bloomington Drosophila Stock Center no. 3605 is an identifier for the w1118 allele. RNAi strains for BuGZ were obtained from a library. The RNAi strains were from the Vienna Drosophila RNAi library. The Vienna Drosophila Resource Center no. v45449 is an identifier for the RNAi strains. dsRNA-GD9485 was inserted in the third chromosome. dsRNA-GD9485 spans base pairs 16,744,705 to 16,745,060 on chromosome 2. The UAS-upd transgenic fly was generated through transformation of a construct into W1118 flies. X. Chen is affiliated with Johns Hopkins University in Baltimore, MD. UAS-ykiS111A.S168A.S250A.V5 is Bloomington Drosophila Stock Center no. 28817. The Gal4 allele ey-Gal4 drives expression of UAS lines. ey-Gal4 is Bloomington Drosophila Stock Center no. 5534. ey-Gal4 is expressed in the Drosophila eyes.\n",
      "node 82:\n",
      "The flies were maintained on standard cornmeal/molasses/yeast fly food. The recipe for 1 liter of the standard cornmeal/molasses/yeast fly food included 50 g cornmeal, 18.75 g yeast, 5 g agar, 62.5 ml molasses, 11.5 ml 5% anti-fungal agent Tegosept, and 35 ml propionic acid. The flies were cultured at a temperature of 25 C on a normal light/dark cycle condition unless otherwise indicated. Unless otherwise indicated, the flies were cultured in standard conditions.\n",
      "node 83:\n",
      "To induce tumor formation in Drosophila eyes, ey-Gal4 was used to drive oncogene expression specifically in the eye. Ey-Gal4 is an element that can drive specific expression of genes in the Drosophila eyes. UAS-upd and UAS-ykiS111A.S168A.S250A.V5 transgenic flies were used to generate eye tumors. The use of UAS-upd and UAS-ykiS111A.S168A.S250A.V5 transgenic flies can induce the formation of eye tumors in Drosophila. To investigate the tumor repression effect of BuGZ RNAi, we used ey-Gal4 to drive both UAS-upd (or UAS-ykiS111A.S168A.S250A.V5) and UAS-BuGZ RNAi.\n",
      "node 84:\n",
      "The crosses were set up at a temperature of 25 C.\n",
      "node 85:\n",
      "Eye tumor formation was examined after eclosion.\n",
      "node 86:\n",
      "Flies with the ey-Gal4>UAS-upd (or UAS-ykiS111A.S168A.S250A.V5) genotype were used to evaluate the tumor repression effect of pladienolide B. The flies were fed with DMSO/pladienolide B (5 microM) from the first instar larval stage. Eye tumor formation was examined after eclosion in these flies. Bright-field images of the eyes were taken by placing the fly's head dorsal-side-up under a stereomicroscope (SMZ1500; Nikon).\n",
      "node 87:\n",
      "The camera used for imaging was an AxioCam HRC (Carl Zeiss). This setup allows us to visualize the contour and half of the total area of the fly eyes. Rough eyes have a rough contour and a larger area than normal eyes. The area of each Drosophila eye is quantified by measuring the pixel number in each eye area using Photoshop CS5 (Adobe). The image analysis software used was Photoshop CS5.\n",
      "node 88:\n",
      "Tub-Gal4 was used to drive the ubiquitous expression of UAS-BuGZ RNAi. The UAS-LacZ RNAi allele was used as the control. Total RNAs were collected at either the third instar larval stage or pupal stage. qRT-PCR was performed to examine the mRNA level of BuGZ. Drosophila heads were excised and placed in McDowell's and Trump's 4F:1G fixative for 1 h to several weeks at 4 C. McDowell's and Trump's 4F:1G is a type of fixative. The Drosophila heads were rinsed twice in 0.1 M sodium phosphate buffer with pH 7.2. Sodium phosphate buffer has a pH of 7.2. The Drosophila heads were dehydrated in a graded ethanol series to 100% ethanol. A graded ethanol series is used for dehydration. Ethanol was used up to 100% concentration. The Drosophila heads were critical point-dried with liquid CO2. Critical point drying uses liquid CO2 as the medium. The Drosophila heads were mounted on specimen stubs with colloidal silver. Colloidal silver is used for mounting specimens. The Drosophila heads were sputter-coated with gold-palladium. Gold-palladium was used as a coating material. A scanning electron microscope (Quanta 200 Environmental Scanning Electron Microscope; FEI) was used to examine the Drosophila heads.\n",
      "node 89:\n",
      "Total RNA was isolated from Drosophila larvae. Total RNA was isolated from Drosophila pupae. The RNA isolation kit used was RNeasy Plus Mini kit. The manufacturer of the RNA isolation kit was QIAGEN. The protocol for RNA isolation followed the manufacturer's instructions. Quantitative RT-PCR was performed using the iScript One Step RT-PCR kit. The product number of the iScript One Step RT-PCR kit used was 170-8892. The manufacturer of the iScript One Step RT-PCR kit was Bio-Rad Laboratories. A real-time PCR detection system (CFX96) was used for the experiments. The manufacturer of the CFX96 real-time PCR detection system was Bio-Rad Laboratories. 50 ng of total RNA was reverse transcribed and amplified. The conditions for reverse transcription were 50 C for 10 min. The first denaturation step for amplification was 95 C for 5 min. Subsequent denaturation steps for amplification were at 95 C for 10 s. The annealing temperature for amplification was 60 C for 30 s. The extension time for amplification was 72 C for 1 min. Steps 2-4 (denaturation, annealing, and extension) were repeated for 40 cycles. Each reaction was performed in triplicate. Results from three independent experiments were used for statistical analysis. Relative mRNA expression levels were quantified using the DeltaDeltaC(t) method. RP49 gene was used as a reference for normalization. The forward primer sequence for RP49 was TACAGGCCCAAGATCGTGAA. The reverse primer sequence for RP49 was TCTCCTTGCGCTTCTTGGA. The forward primer sequence for BuGZ was AAGCCACCTTAGGATCATGG. The reverse primer sequence for BuGZ was GGGTCCGGTGTACAACTTCT.\n",
      "node 90:\n",
      "Fig. S1 shows the effect of BuGZ reduction on the mitotic index. The mitotic index was affected by BuGZ reduction. BuGZ reduction had an effect on cell proliferation. Cell proliferation was influenced by BuGZ reduction. Fig. S2 shows the tumor repression effect of BuGZ RNAi treatment. Tumor repression was achieved with BuGZ RNAi treatment. The tumor repression effect was also shown by pladienolide B treatment. Pladienolide B treatment resulted in tumor repression.\n",
      "node 91:\n",
      "Fig. S3 shows that DNA damage caused by splicing inhibitor treatment can be rescued by forced expression of RNase H1. DNA damage caused by splicing inhibitor treatment occurred. The DNA damage was caused by splicing inhibitor treatment. Forced expression of RNase H1 rescued the DNA damage caused by splicing inhibitor treatment. RNase H1 rescued the DNA damage caused by splicing inhibitor treatment. Table S1 gives a list of differential alternative splicing events caused by BuGZ RNAi treatment. Differential alternative splicing events were caused by BuGZ RNAi treatment. A list of differential alternative splicing events was given in Table S1. The differential alternative splicing events were caused by BuGZ RNAi, Bub3 RNAi, or pladienolide B treatment. Table S2 gives the sources and sequences of mammalian expression plasmids. Mammalian expression plasmids had their sources given in Table S2. The sources and sequences of mammalian expression plasmids were provided in Table S2. Antibodies used for staining had dilutions specified in Table S2. siRNA, shRNA, small molecules, and cell lines had information provided in Table S2.\n"
     ]
    }
   ],
   "source": [
    "parsed = topic_parse(nodes, chunk_size=2000, similarity_threshold=0.8, window_size=5, similarity_method=\"llm\")\n",
    "print_topic_nodes(parsed)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "application/vnd.jupyter.widget-view+json": {
       "model_id": "73c7c2d3d88d4e9799381c872e399fad",
       "version_major": 2,
       "version_minor": 0
      },
      "text/plain": [
       "Parsing nodes:   0%|          | 0/72 [00:00<?, ?it/s]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "No valid JSON found in the response: assistant: It seems you forgot to provide the content for decomposition.\n",
      "\n",
      "To proceed, please provide the text that needs to be broken down into simple propositions. I'll apply the rules to transform it into a list of strings in JSON format.\n",
      "Failed to parse JSON: [ \n",
      "\"The control or BuGZ siRNA was used in an experiment.\" ,\n",
      "\"The error bars indicate the standard deviation (SD) of the results.\" ,\n",
      "\"A Student's t test was performed to determine the significance of the results.\" ,\n",
      "\"In the t test, not significant was determined when P > 0.05.\" ,\n",
      "\"In the t test, a P value less than 0.05 was considered significant (*)\" ,\n",
      "\"In the t test, a P value less than 0.01 was considered highly significant (**)\" ,\n",
      "\"In the t test, a P value less than 0.001 was considered very highly significant (***)\" ,\n",
      "\"The results of three independent experiments were used to determine significance.\" ,\n",
      "\"BuGZ siRNA was used in an experiment involving HFF cells.\" ,\n",
      "\"Lentivirus-expressed shRNA was used to stably knockdown BuGZ in HFFs.\" ,\n",
      "\"HFF cells were infected with lentivirus and enriched by puromycin selection.\" ,\n",
      "\"The cell lysates of the infected HFFs were collected at day 5.\" ,\n",
      "\"Growth curves of HFFs were measured after shRNA transfection.\" ,\n",
      "\"Error bars indicate SD on the growth curves.\" ,\n",
      "\"A Student's t test was performed to determine significance in the growth curve experiments.\" ,\n",
      "\"In the t test, a P value less than 0.01 was considered significant (**)\" ,\n",
      "\"In the t test, a P value less than 0.001 was considered very highly significant (***)\" ,\n",
      "\"BuGZ shRNA-transfected HFFs showed increased senescence.\" ,\n",
      "\"SA-beta-gal staining was used to judge senescence in HFFs.\" ,\n",
      "\"Arrowheads were used to indicate SA-beta-gal-positive cells in the staining results.\" \n",
      "\"The bar was 50 micrometers in length.\" ,\n",
      "\"A Student's t test was performed to determine significance in the percentage of SA-beta-gal-positive cells.\" ,\n",
      "\"In the t test, a P value less than 0.001 was considered very highly significant (***)\" ,\n",
      "\"The error bars indicate SEM on the percentage of SA-beta-gal-positive cells.\" ]\n",
      "Failed to parse JSON: [ \n",
      "\"The w1118 allele was the wild-type control.\"\n",
      "\"The w1118 allele was Bloomington Drosophila Stock Center no. 3605.\"\n",
      "\"The RNAi strains for BuGZ were obtained.\"\n",
      "\"The RNAi strains for BuGZ were from the Vienna Drosophila RNAi library.\"\n",
      "\"The RNAi strain used w1118 and had P{GD9485}v45449 (Vienna Drosophila Resource Center no. v45449).\"\n",
      "\"dsRNA-GD9485 was inserted in the third chromosome.\"\n",
      "\"dsRNA-GD9485 started at 2L:16,744,705 ... 16,745,060.\"\n",
      "\"The UAS-upd transgenic fly was generated through transformation of a construct into W1118 flies by standard methods.\"\n",
      "\"The UAS-upd transgenic fly was provided by X. Chen, Johns Hopkins University, Baltimore, MD.\"\n",
      "\"The UAS-ykiS111A.S168A.S250A.V5 strain was Bloomington Drosophila Stock Center no. 28817.\"\n",
      "\"The Gal4 allele used to drive the UAS lines is ey-Gal4.\"\n",
      "\"The ey-Gal4 allele was Bloomington Drosophila Stock Center no. 5534.\"\n",
      "\"The ey-Gal4 allele is expressed in the Drosophila eyes.\" ]\n",
      "{'chunk_size': 2000, 'threshold': 0.8, 'window': 5}\n",
      "node 0:\n",
      "BioC-API collection key The CC BY-NC-SA license applies.\n",
      "node 1:\n",
      "There is a splicing role of mitotic regulator. The splicing role of mitotic regulator kills tumor cells.\n",
      "node 2:\n",
      "This article has terms that are distributed under an Attribution-Noncommercial-Share Alike-No Mirror Sites license for the first six months after the publication date. After six months, the article is available under a Creative Commons License (Attribution-Noncommercial-Share Alike 3.0 Unported license). The license terms can be found at http://www.rupress.org/terms and http://creativecommons.org/licenses/by-nc-sa/3.0/\n",
      "node 3:\n",
      "Wan's surname is Wan. Yihan Wan's given name is Yihan. Zheng's surname is Zheng. Xiaobin Zheng's given name is Xiaobin. Chen's surname is Chen. Haiyang Chen's given name is Haiyang. Guo's surname is Guo. Yuxuan Guo's given name is Yuxuan. Jiang's surname is Jiang. Haosheng Jiang's given name is Haosheng (Note: I corrected the first name 'Hao' to 'Haosheng' as it seems more accurate) He's surname is He. Xiaonan He's given name is Xiaonan. Zhu's surname is Zhu. Xueliang Zhu's given name is Xueliang. Zheng's surname is Zheng (Note: I assume the same person as mentioned earlier, if not please correct). Yixian Zheng's given name is Yixian.\n",
      "node 4:\n",
      "BuGZ plays a critical role in RNA splicing during interphase. Bub3 plays a critical role in RNA splicing during interphase. Mitotic regulators BuGZ and Bub3 play a critical role in RNA splicing during interphase. Disruption of BuGZ's function leads to R-loop formation. Disruption of BuGZ's function leads to DNA damage. Disruption of BuGZ's function leads to p53 activation. The disruption of BuGZ and Bub3's functions together lead to R-loop formation, DNA damage, and p53 activation.\n",
      "node 5:\n",
      "Studies suggest that perturbing mitotic progression leads to DNA damage and p53 activation. DNA damage and p53 activation lead to either cell apoptosis or senescence. It remains unclear how mitotic defects trigger p53 activation. BuGZ is a mitotic regulator localized in the interphase nucleus. Bub3 is a mitotic regulator localized in the interphase nucleus. BuGZ interacts with the splicing machinery. Bub3 interacts with the splicing machinery. The splicing machinery is required for pre-mRNA splicing. Inhibition of RNA splicing by pladienolide B leads to DNA damage and p53 activation. Depletion of BuGZ leads to increased formation of RNA-DNA hybrids (R-loops). Increased R-loop formation leads to DNA damage. DNA damage and p53 activation occur in human tumor cells upon depletion of BuGZ or Bub3. DNA damage and p53 activation also occur in primary cells upon depletion of BuGZ or Bub3. R-loop-mediated DNA damage and p53 activation explain apoptosis of cancer cells. R-loop-mediated DNA damage and p53 activation also explain senescence of primary cells. Disruption of BuGZ or Bub3 leads to R-loop-mediated DNA damage and p53 activation. Understanding the full range of functions of mitotic regulators is important for developing antitumor drugs.\n",
      "node 6:\n",
      "Extensive studies have shown that prolonged mitotic arrest can lead to DNA damage DNA damage and p53 activation are consequences of prolonged mitotic arrest Although p53 activation in these cells would explain why targeting mitotic regulators could be effective for cancer therapy However, how mitotic arrest leads to DNA damage and p53 activation is not fully understood in some contexts Prolonged mitosis is proposed to cause DNA or cellular damage that would in turn activate p53 Supporting this idea, prolonged mitotic arrest has been shown to cause Caspase activation Caspase activation could lead to CAD (Caspase-activated DNase) activation CAD activation could result in DNA damage and p53 activation However, how prolonged mitosis activates Caspases is not clear in this context A mitotic timer has been suggested to sense the prolonged mitotic arrest The mitotic timer senses prolonged mitotic arrest either in a p53-dependent manner or independently of p53 A p53-dependent timer could link prolonged mitotic block to p53 activation However, the nature of this timer and the signal that activates p53 are unknown\n",
      "node 7:\n",
      "The difficulty in defining the mitotic trigger for DNA damage and p53 activation could be because we have not looked at the right stage of the cell cycle. Several mitotic regulators are found in the interphase nucleus. p53 activation could be caused by the disruption of the interphase nuclear functions of these mitotic regulators. A nuclear zinc finger protein named BuGZ has been shown to regulate mitosis. BuGZ directly binds to the spindle assembly checkpoint protein Bub3 to promote its loading to kinetochores and chromosome alignment. Bub3 is also localized to the interphase nucleus. The interaction between BuGZ and Bub3 can be detected throughout the cell cycle. Depleting BuGZ in various cancer cell lines resulted in a great reduction in the kinetochore Bub3 levels. BuGZ depletion caused chromosome misalignment. BuGZ depletion led to mitotic block. Upon a prolonged mitotic block, most of the BuGZ-depleted tumor cells undergo mitotic death (mitotic catastrophe).\n",
      "node 8:\n",
      "We investigated the phenomenon of mitotic catastrophe.\n",
      "node 9:\n",
      "The investigation helped us discover an unrecognized interphase nuclear function of BuGZ. BuGZ has an interphase nuclear function. We found that Bub3 also possesses this interphase nuclear function. This interphase function helps to explain why disrupting BuGZ and Bub3 could activate p53. The activation of p53 can be caused by disrupting the two mitotic regulators, BuGZ and Bub3.\n",
      "node 10:\n",
      "The depletion of BuGZ causes apoptosis in cancer cells. BuGZ depletion destabilizes Bub3 protein in cancer cells. Chromosome misalignment and mitotic arrest occur in cancer cells due to BuGZ depletion. Massive cell death occurs in cancer cells after chromosome misalignment and mitotic arrest caused by BuGZ depletion. SiRNA was used to deplete BuGZ protein in three cancer cell lines (HeLa, HT29, or TOV21G) and primary human foreskin fibroblasts (HFFs). The use of siRNA to deplete BuGZ led to a reduction of Bub3 protein level after 60 h. Bub3 protein reduction occurred in all cells lines including Hela, HT29, TOV21G, and HFFs due to BuGZ depletion. An elevation of mitotic index was observed in cancer cells (HeLa, HT29, or TOV21G) and primary human foreskin fibroblasts (HFFs) after 60 h of siRNA treatment. BuGZ is required for efficient chromosome alignment in cancer cells and HFFs. Bub3 reduction upon BuGZ depletion indicates that BuGZ maintains Bub3 protein level.\n",
      "node 11:\n",
      "Reduction of BuGZ causes apoptosis in human cancer cells Apoptosis in human cancer cells occurs at 72 hours after siRNA treatment BuGZ was depleted by >90% using RNAi BuGZ depletion was confirmed through alpha-tubulin loading control The numbers at the bottom of each lane indicate relative levels of BuGZ or Bub3 compared with controls The values at the bottom of each lane were set to 1 for comparison Apoptotic cells were quantified using FACS assays after RNAi treatment FACS assays were used to identify early apoptotic cells (Annexin V positive) FACS assays were also used to identify late apoptotic cells (Annexin V and 7-AAD double positive) Growth curves of cells treated with siRNA were generated at 72 hours after RNAi\n",
      "node 12:\n",
      "BuGZ reduction in primary foreskin fibroblasts caused senescence, not apoptosis Primary foreskin fibroblasts underwent senescence instead of apoptosis at 72 hours after siRNA treatment The effect of BuGZ reduction on primary foreskin fibroblasts was different from human cancer cells BuGZ was depleted in both human cancer cells and primary foreskin fibroblasts using RNAi\n",
      "node 13:\n",
      "By 5 days after siRNA treatment, all three cancer cell lines exhibited a large increase in apoptosis. The early and late apoptotic markers Annexin V and 7-AAD correlated with the large increase in apoptosis. The increase in apoptosis was judged by the appearance of Annexin V and 7-AAD, respectively. All three cancer cell lines showed a striking reduction in cell numbers at 5 days after siRNA treatment. Fig. 1 B shows the results for Annexin V and 7-AAD markers. The reduction in cell numbers was observed at day C (Fig. 1) and D (Fig. 1). BuGZ siRNA treatment of HFFs did not affect cell survival. HFFs showed a moderate elevation in mitotic index after BuGZ siRNA treatment. The observation of primary neural stem cells was similar to that of HFFs. Primary somatic cells grow more slowly than tumor cells, so we reasoned that a time >5 d might be required for assessing the effect of BuGZ depletion on HFFs. We used lentivirus-mediated RNAi to stably knock down BuGZ in HFFs (Fig. 1 E). The proliferation rate of the BuGZ-RNAi HFFs began to significantly slow down from day 8 after infection. A threefold increase of senescent cells was observed at 30 d compared with controls (Fig. 1, G).\n",
      "node 14:\n",
      "BuGZ depletion is associated with indicator h) The indicator associated with BuGZ depletion indicates accelerated senescence of BuGZ-depleted human foreskin fibroblasts (HFFs). Accelerated senescence in BuGZ-depleted HFFs occurs. BuGZ depletion causes apoptosis in cancer cells. Apoptosis is a form of programmed cell death. Primary human foreskin fibroblasts are a type of primary cell. BuGZ depletion has an effect on the cellular behavior of primary human foreskin fibroblasts. The cellular behavior of BuGZ-depleted HFFs includes senescence. Senescence is a state in which cells become permanently growth-arrested and usually acquire changes to their cell morphology. BuGZ depletion causes a specific cellular response, namely apoptosis, in cancer cells. Apoptosis in cancer cells means that the cells undergo programmed cell death.\n",
      "node 15:\n",
      "Depletion of BuGZ leads to p53 activation. p53 accumulation causes apoptosis or senescence depending on cellular context. BuGZ depletion was studied in cancer cells and human foreskin fibroblasts (HFFs). HeLa and TOV21G cells have wild-type p53 protein. In HeLa cells, the E6 proteins from human papillomavirus target p53 for degradation under normal conditions. However, HeLa cell's p53 accumulation is reported upon DNA damage or oxidation. HT29 cells have a point mutation in the transactivation domain of p53 (R273H). Mutant p53 in HT29 cells induces apoptosis or cell cycle arrest upon various challenges. p53 activation causes senescence in primary cells with intact cell cycle checkpoints. Total and activated p53 levels were increased in cancer cells and HFFs upon BuGZ depletion.\n",
      "node 16:\n",
      "Activation of p53 upon BuGZ depletion leads to apoptosis or senescence. The up-regulation of p53 occurred after 48 hours of BuGZ siRNA treatment.\n",
      "node 17:\n",
      "Alpha-tubulin was used as a loading control.\n",
      "node 18:\n",
      "Phospho-p53 (Ser15) was increased in the nuclei of cells that had been depleted of BuGZ. Some cells were positive for BuGZ and negative for phospho-p53.\n",
      "node 19:\n",
      "The cells were assayed 48 hours after siRNA treatment.\n",
      "node 20:\n",
      "Bars, measuring 10 micrometers, were used to indicate scale.\n",
      "node 21:\n",
      "Inhibition of p53 by PFT-alpha increased the number of viable cells that had been depleted of BuGZ. PFT-alpha is a substance that inhibits p53. Three independent experiments were performed. The results from these experiments showed a significant difference (P < 0.01). The expression of p53 RNAi in HFFs repressed senescence induced by BuGZ knockdown. Western blotting was used to detect the presence of certain proteins. A Western blot, which shows the presence of SA-beta-gal staining cells, is indicated (D and F). SA-beta-gal staining cells were counted and quantified (E). The error bars in graph E indicate SEM. P < 0.05 indicates a significant difference for the results from graph F. P < 0.01 indicates a more significant difference for the results from graph F.\n",
      "node 22:\n",
      "Three independent experiments were conducted.\n",
      "node 23:\n",
      "The first experiment was completed. The second experiment was finished on time. The third experiment had a unique setup.\n",
      "node 24:\n",
      "Each of these experiments involved multiple steps. Researchers took careful measurements in each trial. Data from all three experiments were analyzed separately. Results from the first two experiments showed promising trends. However, the results of the third experiment produced unexpected outcomes.\n",
      "node 25:\n",
      "Our previous studies indicated that many BuGZ-depleted HeLa cells initially arrested in mitosis. BuGZ-depleted HeLa cells then underwent cell death. We tested whether p53 activation could be the cause of cancer cell death. We treated cells with a p53 inhibitor, PFT-alpha, 12 hours after transfection of control or BuGZ siRNA. The inhibition of p53 significantly rescued the cell number reduction of BuGZ-depleted cancer cells (Fig. 2 C). HeLa cells were coinfected with lentivirus to express p53 shRNA. Senescence induced by BuGZ depletion upon prolonged culturing was also significantly inhibited (Fig. 2, D-F). Because depletion of BuGZ down-regulated Bub3 (Fig. 1 A), we investigated whether Bub3 reduction could also lead to p53 activation. Knockdown of Bub3 by RNAi caused prominent p53 accumulation and activation in both cancer cells and human fetal fibroblasts (HFFs). These findings show that p53 mediates both apoptosis in cancer cells and senescence in HFFs upon BuGZ depletion. BuGZ down-regulated Bub3 (Fig. 1 A).\n",
      "node 26:\n",
      "BuGZ and Bub3 are required for proper pre-mRNA splicing. The depletion of BuGZ or Bub3 could activate p53. BuGZ and Bub3 bind to one another. BuGZ and Bub3 are both present in the interphase nucleus. Their depletion could disrupt an interphase nuclear function. This might trigger p53 activation. We examined our previous proteome analyses of BuGZ-interacting proteins. Besides Bub3, several spliceosome components were noticed among the top candidates of BuGZ-interacting proteins. A number of available antibodies to spliceosome components were used. The antibodies to U2AF65 and SF3a3 worked in both immunoprecipitation and Western blotting. Reciprocal immunoprecipitation provided further support for an interaction between BuGZ/Bub3 and the spliceosome. U2AF65 is a component of the spliceosome. SF3a3 is a component of the spliceosome. The proteomic analyses of spliceosomes previously included BuGZ and Bub3.\n",
      "node 27:\n",
      "Proteins with known functions in RNA splicing were found as candidate BuGZ-interacting proteins. FLAG-tagged BuGZ immunoprecipitated SF3a3 and U2AF65 in HEK293T cells.\n",
      "node 28:\n",
      "FLAG-luciferase did not immunoprecipitate SF3a3 and U2AF65 in HEK293T cells. BuGZ interacted with SF3a3 and U2AF65 in vivo. Bub3 interacted with SF3a3 and U2AF65 in vivo. Reciprocal immunoprecipitations of endogenous proteins were performed using HT29 cell lysates. Mouse monoclonal antibodies (mAb) were used for reciprocal immunoprecipitations. Rabbit antibodies (rAb) were used for reciprocal immunoprecipitations.\n",
      "node 29:\n",
      "The antibody heavy chain (h.c.) was present in the immunoprecipitates.\n",
      "node 30:\n",
      "Experiments were performed three times.\n",
      "node 31:\n",
      "Pladienolide B inhibited RNA splicing in HFFs and TOV21G cells. RNAi of BuGZ resulted in splicing defects in HFFs and TOV21G cells. RNAi of Bub3 resulted in splicing defects in HFFs and TOV21G cells. The numbers of genes exhibiting the depicted splicing defects were shown as pie charts. RNA-Seq was performed on two biological repeats. BRD2 exhibited intron retention upon treatment with pladienolide B or RNAi of BuGZ. WAC exhibited intron retention upon treatment with pladienolide B or RNAi of BuG3.\n",
      "node 32:\n",
      "RNAi of BuGZ was performed. The red boxes highlighted examples of intron retention in the fold changes. Introns were retained in the RNA transcript. Examples of intron retention were highlighted by red boxes. Fold increases of the intron retention were plotted to the right. Plots were generated to show the fold increases of intron retention. p-values for the differential alternative splicing were generated by simulation using MATS. MATS is a Bayesian framework. A Bayesian framework called MATS was used for flexible detection of differential alternative splicing from RNA-Seq data. Differential alternative splicing was detected from RNA-Seq data using MATS.\n",
      "node 33:\n",
      "To test whether BuGZ and Bub3 play roles in splicing, we depleted each protein BuGZ was a protein. Bub3 was a protein. We depleted BuGZ in cancer cells (TOV21G) or HFFs. We performed RNA sequencing (accession number: GSE67770). Cells treated with pladienolide B, a potent splicing inhibitor, were used as positive controls. The depletion of either Bub3 or BuGZ mainly caused exon skipping. Pladienolide B treatment mainly resulted in intron retention (Fig. 3, D and E; and Table S1). The difference in the resultant splicing defects between Pladienolide B inhibition (which targets SF3b3) and BuGZ or Bub3 depletion suggests that BuGZ and Bub3 function differentially from SF3b3. Whereas further studies are required to understand how BuGZ and Bub3 regulate splicing, these analyses show that the well-established SAC component Bub3 and its regulator BuGZ also play roles in interphase RNA splicing. We depleted BuGZ in transformed cells We treated tumor cells with pladienolide\n",
      "node 34:\n",
      "B has been shown to kill these cells in vitro. Using the Drosophila melanogaster eye tumor model, researchers found that The activation of either the Jak-Stat pathway or the Hippo pathway was observed to induce eye hyperproliferation. Researchers found that the eye-specific BuGZ RNAi inhibited the induced eye hyperproliferation. Pladienolide B had an inhibitory effect on the induced eye hyperproliferation when fed to Drosophila melanogaster flies (Fig. S2 A).\n",
      "node 35:\n",
      "The depletion of BuGZ causes R-loop formation. R-loop formation occurs when there is a lack of BuGZ. The depletion of Bub3 also causes R-loop formation. R-loops form in the absence of sufficient Bub3. BuGZ or Bub3 are necessary to prevent the formation of R-loops and associated DNA damage. DNA damage results from the formation of R-loops that occur when there is not enough BuGZ or Bub3.\n",
      "node 36:\n",
      "Deficiency in RNA splicing has been shown to cause an increased formation of RNA-DNA hybrids (R-loops) in the nucleus. Subsequent DNA breaks and p53 activation occur due to RNA splicing deficiency. p53-mediated killing of transformed cells or senescence of primary cells as we observed could be caused by defects in splicing and subsequent R-loop formation. To determine whether cells depleted of BuGZ had an increased R-loop formation, the monoclonal antibody S9.6 was used. The monoclonal antibody S9.6 recognizes RNA-DNA hybrids. Depletion of BuGZ markedly stimulated R-loop formation in cancer cells and HFFs. Similar results were observed for Bub3 depletion, where R-loop formation was also increased in both cancer cells and HFFs. Cells depleted of BuGZ or Bub3 exhibited increased DNA damage, as judged by the appearance of prominent nuclear foci of gammaH2AX. These cells displayed a notable increase in gammaH2AX nuclear foci (Fig. 4, B-F; Fig. S2, D-H; and Fig. S3 A). Results suggest that defects in the splicing function of BuGZ and Bub3 lead to R-loop-induced DNA damage. R-loop-induced DNA damage contributes to cellular effects.\n",
      "node 37:\n",
      "The protein p53 has a role in cell cycle regulation. The protein p53 is activated due to certain cellular stresses. The activation of p53 can lead to cell cycle arrest or apoptosis. Cell cycle arrest and apoptosis can help eliminate damaged cells from the body. There are several pathways that activate p53, including DNA damage response. One pathway leading to p53 activation involves the ATM kinase. ATM is activated in response to DNA double-strand breaks. The activation of ATM leads to increased levels of p53 protein. Increased levels of p53 can lead to cell cycle arrest or apoptosis.\n",
      "node 38:\n",
      "Depletion of BuGZ or Bub3 leads to increased R-loop formation. Depletion of BuGZ or Bub3 resulted in increased R-loop formation in HT29 cells. Pladienolide B treatment was used as a positive control for R-loop formation in HT29 cells. The mean fluorescence intensity of the R-loop is shown on the right. Student's t test results showed that the difference between BuGZ or Bub3 depleted cells and non-depleted cells was statistically significant at P < 0.001 from three independent experiments. Depletion of BuGZ caused increased DNA damage in HT29 cells. Depletion of Bub3 caused increased DNA damage in HT29 cells. The presence of prominent nuclear foci of gamma-H2Ax indicates DNA damage. BuGZ- or Bub3-depleted cells exhibit more gamma-H2Ax foci than cells with insufficient or no depletion. Quantification of gamma-H2Ax intensities in individual cells was performed and the results are shown on the left. The distribution of gamma-H2Ax intensities is shown on the right for BuGZ depleted cells (E) and Bub3 depleted cells (F). Student's t test: ***, P < 0.001 indicates that the difference between BuGZ or Bub3 depleted cells and non-depleted cells was statistically significant at P < 0.001 from three independent experiments.\n",
      "node 39:\n",
      "The probability P is less than 0.001.\n",
      "node 40:\n",
      "Three independent experiments were conducted. Cell numbers (n) were obtained from each of these experiments.\n",
      "node 41:\n",
      "The inhibition of R-loop formation occurred. R-loop formation was attenuated. This attenuation had a consequence on p53 activation. As a result, p53 activation was reduced. The depletion of BuGZ led to DNA damage and p53 activation. Similarly, the depletion of Bub3 caused DNA damage and p53 activation. BuGZ is a protein. Bub3 is also a protein.\n",
      "node 42:\n",
      "To confirm that the R-loops formed in cells depleted of BuGZ or Bub3 indeed contributed to DNA damage and p53 activation A modified form of RNase H1 lacking the mitochondria targeting sequence was used. This form of RNase H1 has been shown to localize only in the nucleus and to resolve R-loops. By analyzing the nuclei that were positive for the GFP-tagged RNase H1 or GFP control Expression of the enzyme resulted in a significant reduction of gammaH2AX foci in BuGZ-depleted HT29 cells. The expression of RNase H1 also led to a significant reduction of p53 activation in BuGZ-depleted HT29 cells. A similar effect of RNase H1 was observed in pladienolide B-treated HT29 cells. A similar effect of RNase H1 was also observed in Bub3-depleted HT29 cells. The inability of RNase H1 to fully rescue DNA damage and p53 activation could be caused by the heterogeneous level of GFP-RNase H1 expression. The inability of RNase H1 to fully rescue DNA damage and p53 activation could also be due to the inefficiency of GFP-RNase H1 to resolve certain R-loops. Other uncharacterized defects may also contribute to the inability of RNase H1 to fully rescue DNA damage and p53 activation.\n",
      "node 43:\n",
      "The splicing function of Bub3 contributed to DNA damage. The splicing function of BuGZ contributed to DNA damage. The combination of the splicing functions of Bub3 and BuGZ (Fig. S3 C) contributed to DNA damage. DNA damage activated p53.\n",
      "node 44:\n",
      "Expression of RNase H1 in BuGZ- or Bub3-depleted HT29 cells reduces gamma-H2Ax and phospho-p53 signals. RNase H1 expression repressed BuGZ depletion-induced DNA damage in HT29 cells. RNase H1, but not GFP, had an effect on BuGZ depletion-induced DNA damage in HT29 cells. Cells were first transfected by siRNAs for 12 hours and then by GFP-RNase H1 or GFP plasmid for 48 hours. HT29 cells were analyzed by immunofluorescence microscopy. BuGZ-depleted cells with or without GFP fluorescence are indicated by arrowheads or arrows, respectively.\n",
      "node 45:\n",
      "Broken lines indicate nuclear boundaries.\n",
      "node 46:\n",
      "The bar is 10 micrometers in length.\n",
      "node 47:\n",
      "RNase H1 expression repressed BuGZ depletion-induced gamma-H2Ax signals in HT29 cells. BuGZ-depleted cells without RNase H1 expression had higher gamma-H2Ax mean fluorescence intensity than BuGZ-depleted cells with RNase H1 expression. The distribution of gamma-H2Ax intensity was different between BuGZ-depleted cells with or without RNase H1 expression. RNase H1 expression repressed BuGZ depletion-induced p53 phosphorylation in HT29 cells. BuGZ-depleted cells without RNase H1 expression had higher phospho-p53 mean fluorescence intensity than BuGZ-depleted cells with RNase H1 expression. The distribution of phospho-p53 intensity was different between BuGZ-depleted cells with or without RNase H1 expression.\n",
      "node 48:\n",
      "Experiments were shown in D. The experiments consisted of G and H.\n",
      "node 49:\n",
      "Quantifications for the mean fluorescence intensity of gamma-H2Ax were measured. Quantifications for the intensity distributions of gamma-H2Ax were measured. Gamma-H2Ax was observed in Bub3-depleted cells. GFP- or GFP-RNase H1-expressing HT29 cells had gamma-H2Ax levels compared to Bub3-depleted cells. Quantifications for the mean fluorescence intensity of phospho-p53 were measured. Quantifications for the intensity distributions of phospho-p53 were measured. Phospho-p53 was observed in Bub3-depleted cells. GFP- or GFP-RNase H1-expressing HT29 cells had phospho-p53 levels compared to Bub3-depleted cells. Error bars indicate SEM. A Student's t test was performed. P-values were calculated: *, P < 0.05; **, P < 0.01; ***, P < 0.001.\n",
      "node 50:\n",
      "Cell numbers (n) were analyzed from three independent experiments.\n",
      "node 51:\n",
      "Extensive efforts have been devoted to understanding how mitotic arrest as a result of disrupting mitotic regulators can lead to p53-dependent cell death or cell cycle arrest in cancer cells. We explored the mitosis-independent functions of the mitotic regulators. A critical role of BuGZ and Bub3 was uncovered in interphase RNA splicing. Disruption of this interphase function by depleting BuGZ or Bub3 leads to R-loop formation, DNA damage, and p53 activation. The pladienolide B kills cancer cells through a similar mechanism. Our findings suggest that looking beyond the mitosis functions of the established mitotic regulators can gain significant insights into how aberrant mitosis is connected to the suppression of cell proliferation and the safeguard of genome stability. A broadened understanding of the function of mitotic regulators beyond their mitosis functions is demonstrated.\n",
      "node 52:\n",
      "Cell division should be facilitated. Facilitating cell division would enable the selection of therapeutic targets. The identification of therapeutic targets would be useful in the development of new antitumor drugs. New antitumor drugs are being developed for cancer chemotherapy.\n",
      "node 53:\n",
      "For sources refer to Table S2. Table S2 contains information about mammalian expression plasmids. Table S2 also contains information about antibodies. The dilutions used for antibodies are included in Table S2. Table S2 has information about siRNA. Table S2 provides details about shRNA. Small molecules and their concentrations are documented in Table S2. Information about various cell lines is present in Table S2. A plasmid for expressing GFP-RNase H1 was provided by R.J. Crouch. R.J. Crouch works at the National Institute of Child Health and Human Development. The National Institute of Child Health and Human Development is located in Bethesda, MD.\n",
      "node 54:\n",
      "The base medium for TOV21G cell culture was a mixture of MCDB 105 medium and Medium 199. MCDB 105 medium contained a final concentration of 1.5 g/liter sodium bicarbonate. Medium 199 contained a final concentration of 2.2 g/liter sodium bicarbonate. To make the complete growth medium for TOV21G cells, fetal bovine serum was added to a final concentration of 15%. HT29 and HeLa cells were cultured in Dulbecco's modified Eagle's medium containing 10% fetal bovine serum. The HT29 and HeLa cells were cultured at 37 C with 5% CO2. HFF cells were cultured in FibroGRO LS Complete Medium. HFF cells had their medium changed every other day. HFF cells were passaged using Accutase at a 1:3 dilution when they reached ~80% confluency.\n",
      "node 55:\n",
      "The plasmids or siRNA transfections were performed using Lipofectamine 2000. Lipofectamine 2000 was manufactured by Life Technologies. Cells were fixed for immunofluorescence analyses 48 hours after siRNA transfection. For cotransfection of siRNA and plasmids, siRNA was transfected first using the reverse transfection protocol. 10 pmol siRNA oligo was diluted in 50 microl Opti-MEM I Medium per well of a 24-well tissue culture plate. Opti-MEM I Medium was manufactured by Gibco. 1.5 microl of Lipofectamine RNAiMAX was added to each well. Lipofectamine RNAiMAX was manufactured by Life Technologies. The plate was incubated for 20 minutes at room temperature.\n",
      "node 56:\n",
      "Cells to be transfected were trypsinized and resuspended in complete growth medium without antibiotics. Cell suspension containing 20,000 cells was added into each well for transfection.\n",
      "node 57:\n",
      "After 24 hours of transfection, cells were transfected with the indicated plasmids. For immunofluorescence analyses in Fig. 5, cells were fixed 48 hours after plasmid transfection.\n",
      "node 58:\n",
      "Tumor cells and primary cells were fixed at 60 hours after RNA interference. The fixation was followed by immunostaining. Both tumor cells and primary cells were analyzed using Flow Cytometry for apoptotic cells in Fig. S1 72 hours after siRNA transfection. Apoptosis FACS analysis was performed for both tumor cells and primary cells in Fig. S1. Primary cells and tumor cells underwent RNAi treatment. The experimental process included immunostaining after fixation of the treated cells.\n",
      "node 59:\n",
      "FACS analyses of apoptotic cells using FITC Annexin V Apoptosis Detection kit with 7-AAD were performed. The FITC Annexin V Apoptosis Detection kit with 7-AAD (640922; BioLegend) was used to detect apoptosis. 5 x 10^5 cells that were treated by control or BuGZ siRNAs were trypsinized and resuspended as individual cells. 100 microl of Annexin V-binding buffer was added to each cell suspension. The Annexin V-binding buffer contained 140 mM NaCl, 4 mM KCl, 0.75 mM MgCl2, and 10 mM Hepes, pH 7.4. 5 microl FITC-Annexin V and 5 microl 7-AAD were added to each cell suspension. The cells were incubated for 15 min at room temperature.\n",
      "node 60:\n",
      "Cells were analyzed with FACS Aria III. 100,000 events were recorded for all samples. FACSDIVA software (BD) was used for all flow cytometry analyses. Samples with buffer alone were used as background autofluorescence control. Samples with FITC-Annexin V staining alone were used as compensation controls for each fluorochrome. Samples with 7-AAD staining alone were used as compensation controls for each fluorochrome. The PE-Cy5 channel was used to detect 7-AAD fluorescence.\n",
      "node 61:\n",
      "Immunofluorescence microscopy was used. Quantifications were performed using immunofluorescence microscopy.\n",
      "node 62:\n",
      "We found that the treatment used for mitotic chromosome spread improves the detection of R-loops in the nuclei of HT29 cells. The treatment used for mitotic chromosome spread was provided by L. Wahba and D. Koshland from the University of California, Berkeley, Berkeley, CA. L. Wahba and D. Koshland are associated with the University of California, Berkeley, Berkeley, CA. S9.6 is a monoclonal antibody provided by L. Wahba and D. Koshland. Standard immunostaining was sufficient for HFFs. We modified the chromosome-spread method to treat interphase HT29 cells. HT29 cells were trypsinized and resuspended as single cell suspensions. The cells were treated with a hypotonic solution prewarmed to 37 C. The hypotonic solution contained 75 mM KCl. The fixed cell suspension was placed on a coverslip exposed to the steam of hot water (90 C) for 30 s. The cells spread their chromatin on the coverslip after being exposed to the steam of hot water (90 C) for 30 s.\n",
      "node 63:\n",
      "These cells were blocked by a blocking buffer. The blocking buffer contained 5% BSA and 0.5% Triton X-100 in PBS. The cells were blocked with the blocking buffer for 1 h.\n",
      "node 64:\n",
      "Cells were treated in a specific way. The cells were incubated with a 1:1000 dilution of the S9.6 antibody overnight in blocking buffer. S9.6 antibody was diluted to 1:1000. Blocking buffer was used for washing the cells. The cells were washed three times with blocking buffer. After washing, the cells were incubated with a 1:1000 dilution of secondary antibody for 1 h. A 1:1000 dilution of secondary antibody was prepared. Secondary antibody was added to the cells for 1 hour. For detection of R-loops in HFFs, a specific procedure was followed. HFFs were fixed using ice-cold methanol for 15 min. Methanol cooled to ice temperature was used for fixing the cells. The time spent on fixing the cells was 15 minutes. After fixation with methanol, HFFs were treated with 70% ethanol for 15 min. Seventy percent ethanol was used for treating the cells. The duration of treatment with 70% ethanol was 15 minutes. HFFs were washed three times with 70% ethanol for 5 min each. Seventy percent ethanol was used for washing the cells. Each wash lasted for 5 minutes. After washing, the HFFs were subjected to blocking and antibody staining as above.\n",
      "node 65:\n",
      "Cells grown on coverslips were used for immunodetection. Cells were fixed in 4% paraformaldehyde for 15 minutes. The purpose of fixing cells was to prepare them for staining. Celss were then permeabilized with 0.5% Triton X-100 in PBS for 5 minutes. Triton X-100 is used for permeabilizing cells. Cells were stained using primary and secondary antibodies (dilutions found in Table S2). The source of the primary and secondary antibodies can be found in Table S2. Phospho-p53 and Bub3 co-staining required two mouse monoclonal antibodies. To label these antibodies, an APEX Alexa Fluor antibody labeling kit was used. This kit allowed for direct coupling with Alexa Fluor 546 or Alexa Fluor 647. The antibodies were labeled using kits from Life Technologies (A20183 and A20186). Cells expressing GFP or GFP fusion proteins were fixed in the same manner. GFP-expressing cells were imaged directly without further staining. After antibody staining, cells were stained with DAPI at 0.25 micrograms per milliliter in PBS for 5 minutes. DAPI is used for nuclear staining. Celss were then mounted in Fluorescence Mounting medium (Dako).\n",
      "node 66:\n",
      "Fluorescence images were captured using a microscope The microscope used was an ECLIPSE TE2000-U, which is a Nikon product. MetaMorph software (which belongs to Molecular Devices) was used to drive the microscope Alternatively, a laser confocal microscope (TCS SP5, which is a Leica product) was also used The microscope settings remained consistent throughout each experiment For confocal microscopy, specific objective lenses were utilized The objective lenses used had magnifications of 63x and 1.4 NA oil for one lens, and 40x with a 1.25 NA oil for the other lens, both belonging to Leica. Both objective lenses were used to image cells at various intervals Cells were imaged by scanning optical sections at approximately 0.5-microm intervals using the objective lenses The mitotic index was quantified by dividing a specific subset of cells by the total number of cells analyzed The subset of cells being referenced had undergone prophase through telophase, which was judged based on alpha-tubulin and DAPI staining\n",
      "node 67:\n",
      "Maximal projection was created from z stacks for each nucleus.\n",
      "node 68:\n",
      "A method based on a previous study was used to determine nuclear fluorescence signal. The nuclear regions were defined by DAPI staining.\n",
      "node 69:\n",
      "A larger area surrounding the nucleus was drawn. Areas of the nuclear region and the larger area were determined as Xa and Ya, respectively. The mean fluorescence intensity for the nuclear region and the larger area were measured as Xf and Yf. Signal intensity for each nucleus (Xi) was calculated using the formula: Xi = Xf - ((Yf * Ya) - (Xf * Xa)) / (Ya - Xa).\n",
      "node 70:\n",
      "All experiments were performed at least three times. A total of 17-159 nuclei were quantified from these experiments and plotted. Senescence-associated beta-gal assay was conducted.\n",
      "node 71:\n",
      "We used the Senescence beta-Galactosidase Staining kit (9860S; Cell Signaling) to detect beta-galactosidase activity. The pH of the solution was 6.0. Cells were fixed in a fixative solution for 10-15 min at room temperature. The cells that were fixed were then incubated with beta-galactosidase staining solution containing X-gal at 37 C overnight. As the blue color developed, bright-field cell images were taken using an Axiovert 25 microscope (Carl Zeiss) connected to a Canon camera. A 0.4% Trypan blue stock solution (15250061; Life Technologies) was used to stain dead cells. 10 microl of Trypan blue stock solution was added to 100 microl of cell suspension. Cells that were negative for Trypan blue staining were counted as live cells using a hemocytometer. Immunoprecipitation and Western blotting analyses were performed.\n",
      "node 72:\n",
      "We used two methods to confirm the interaction between BuGZ or Bub3 and spliceosome components. In the first method, we transfected HEK293T cells with Flag-BuGZ, Flag-Bub3, or Flag-luciferase (negative control). HEK293T cells were transfected with Flag-BuGZ, Flag-Bub3, or Flag-luciferase. Flag-BuGZ and Flag-Bub3 are protein constructs used for transfection. Flag-luciferase is a negative control used in the experiment. After transfection, approximately 1 x 10^7 cells were lysed in 1 ml lysis buffer. The lysis buffer contained Tris-Cl at pH 7.4, KCl, EDTA, NaF, DTT, NP-40, glycerol, and sodium pyrophosphate. The lysis buffer was prepared with protease inhibitors (at 1 tablet/10 ml) from cOmplete, Mini, EDTA-free, Protease Inhibitor Cocktail Tablets (Roche). Cells were lysed for 10 min at 13,000 rpm in an Eppendorf tabletop centrifuge. The supernatant containing soluble proteins was incubated with a 30-microl slurry of anti-Flag M2 resin at 4 C for 4 h. Anti-Flag M2 resin is a reagent used to capture Flag-tagged proteins (A2220; Sigma-Aldrich). The beads were washed with wash buffer three times. Wash buffer contained Tris-Cl at pH 7.4, NaCl, EDTA, NaF, DTT, and NP-40. Bound proteins were eluted with 100 microl Flag-peptide dissolved in the specified context (F4799; Sigma-Aldrich). Flag-peptide is a reagent used to release Flag-tagged proteins from the anti-Flag M2 resin.\n",
      "node 73:\n",
      "The sample was suspended in the wash buffer. The concentration of the sample in the wash buffer was 0.5 milligrams per milliliter. The wash buffer contained specific ingredients.\n",
      "node 74:\n",
      "We used antibodies to BuGZ. The antibodies for BuGZ were from Sigma-Aldrich, product number HPA017013. We used antibodies to Bub3. The antibodies for Bub3 were from BD, product number 611731. We used antibodies to U2AF65. The antibodies for U2AF65 were from Abcam, product number ab37530. We used antibodies to Sf3a3. The antibodies for Sf3a3 were from Abcam, product number ab156873. We used a control IgG as an antibody. The control IgG was from Millipore, product numbers 12-370 and 12-371. Each antibody was preincubated with a slurry of protein G Dynabeads for 30 minutes. The slurry contained 50 microl of protein G Dynabeads from Life Technologies, product number 10007D. We washed the beads with lysis buffer. We added the beads to HT29 cell lysates made from approximately one x 10^7 cells. Each immunoprecipitation used a different antibody, either BuGZ, Bub3, U2AF65, Sf3a3, or control IgG. The antibodies were preincubated for 30 minutes before use in the immunoprecipitations. We incubated the beads with the cell lysates overnight. After incubation, we washed the beads as described previously. We eluted the bound proteins from the beads with 100 microl of elution buffer at pH 2.5. The elution buffer contained 50 mM glycine. We dissolved the eluted proteins in SDS sample buffer.\n",
      "node 75:\n",
      "The amount of input lysate loaded per lane for SDS-PAGE was 1%. The amount of resuspended immunoprecipitates loaded per lane for SDS-PAGE was 10%. The molecular weights of BuGZ, U2AF65, and Sf3a3 are close to the heavy chain of IgG. Using a secondary antibody specific to the rabbit light chain reduced the signal from the antibody heavy chain. The molecular weight of BuGZ is close to the heavy chain of Flag. The molecular weight of U2AF65 is close to the heavy chain of Flag. The molecular weight of Sf3a3 is close to the heavy chain of Flag. Flag antibodies (F3165; Sigma-Aldrich) were used at a specific dilution. BuGZ antibodies (HPA017013; Sigma-Aldrich) were used at a specific dilution. Bub3 antibodies (611731; BD) were used at a specific dilution. U2AF65 antibodies (ab37530; Abcam) were used at a specific dilution. Sf3a3 antibodies (ab156873; Abcam) were used at a specific dilution. The secondary antibody specific to the rabbit light chain was from Jackson ImmunoResearch Laboratories, Inc. and has product number 211-032-171. Whole-transcriptome shotgun sequencing was performed.\n",
      "node 76:\n",
      "Arcturus PicoPure RNA isolation kit was used to prepare total RNA from cells. The Arcturus PicoPure RNA isolation kit is called KIT0204 and made by Life Technologies. Ribo-Zero Gold kits were used to remove both cytoplasmic (nuclear-encoded) rRNA and mitochondrial rRNA from total RNA preparations. The Ribo-Zero Gold kits are human/mouse/rat kits with product code MRZG126 and made by Epicentre. Libraries were built using the Illumina TruSeq RNA sample prep kit V2. The Illumina TruSeq RNA sample prep kit V2 has product code RS-122-2001 and is made by Illumina. Libraries were sequenced using an Illumina HiSeq 2000. Single-end 100-bp reads were obtained from the sequencing process. RNA-Seq reads were first mapped to the human genome hg19 using Tophat2. Transcript annotation from Ensembl was used as reference for exon junctions in read mapping. We then used Cufflinks to assemble unannotated transcripts. Assembled transcripts from the same cell type were further merged by Cuffmerge. Finally, we used MATS to call differential alternative splicing. The merged transcripts from the previous step were used as transcript annotation in MATS.\n",
      "node 77:\n",
      "All the flies were maintained on standard cornmeal/molasses/yeast fly food. The recipe for 1 liter of the standard cornmeal/molasses/yeast fly food included 50 g cornmeal, 18.75 g yeast, 5 g agar, 62.5 ml molasses, 11.5 ml 5% anti-fungal agent Tegosept, and 35 ml propionic acid. Flies were cultured at a temperature of 25 C on a normal light/dark cycle condition, unless otherwise indicated.\n",
      "node 78:\n",
      "The ey-Gal4 was used to drive oncogene expression specifically in the eye to induce tumor formation. UAS-upd transgenic flies were used to generate eye tumors. UAS-ykiS111A.S168A.S250A.V5 transgenic flies were also used to generate eye tumors. To investigate the tumor repression effect of BuGZ RNAi, ey-Gal4 was used to drive both UAS-upd and UAS-BuGZ RNAi. UAS-ykiS111A.S168A.S250A.V5 transgenic flies were also used in combination with UAS-BuGZ RNAi to investigate the tumor repression effect of BuGZ RNAi. The crosses were set up at a temperature of 25 C, and eye tumor formation was examined after eclosion.\n",
      "node 79:\n",
      "Ey-Gal4>UAS-upd flies were used to evaluate the tumor repression effect of pladienolide B. The Ey-Gal4>UAS-upd flies were fed with a solution containing DMSO and pladienolide B from the first instar larval stage. The concentration of pladienolide B in the solution was 5 microM. Pladienolide B is a splicing inhibitor. Eye tumor formation in the Ey-Gal4>UAS-upd flies was examined after eclosion. Bright-field images of the eyes were taken using a stereomicroscope (SMZ1500) made by Nikon. A camera (AxioCam HRC) made by Carl Zeiss was used to take the images. The fly's head was placed dorsal-side-up under the stereomicroscope to take the images. The contour and half of the total area of each Drosophila eye were visualized in the images. Drosophila eyes with tumors have a rough contour and a larger area than normal eyes. The area of each eye was quantified by measuring the pixel number using Photoshop CS5 made by Adobe.\n",
      "node 80:\n",
      "The UAS-BuGZ RNAi allele was used to investigate its RNAi knockdown efficiency. Tub-Gal4 (Bloomington Drosophila Stock Center no. 5138) was employed to drive the ubiquitous expression of UAS-BuGZ RNAi. UAS-LacZ RNAi allele served as a control. Total RNAs were collected from the third instar larval stage or pupal stage. Quantitative real-time PCR (qRT-PCR) was performed to examine the mRNA level of BuGZ.\n",
      "node 81:\n",
      "The McDowell's and Trump's 4F:1G fixative was used for at least one hour to several weeks at a temperature of 4 degrees Celsius. Drosophila heads were excised. The excised Drosophila heads were placed in the McDowell's and Trump's 4F:1G fixative. After two rinses in sodium phosphate buffer (0.1 M, pH 7.2), samples were dehydrated in a graded ethanol series to 100% ethanol. The dehydration process in ethanol was followed by critical point-drying with liquid CO2. The Drosophila heads were then mounted on specimen stubs with colloidal silver. A gold-palladium coating was applied to the specimens. The specimens were examined using a scanning electron microscope (Quanta 200 Environmental Scanning Electron Microscope; FEI).\n",
      "node 82:\n",
      "Total RNA was isolated from Drosophila larvae. Total RNA was isolated from Drosophila pupae. The RNA isolation kit used was RNeasy Plus Mini kit. The manufacturer of the RNA isolation kit was QIAGEN. RNA isolation was performed using the manufacturer's protocol. Quantitative RT-PCR was performed on a real-time PCR detection system. The brand name of the real-time PCR detection system was CFX96. The model number of the real-time PCR detection system was not specified. The brand name of the iScript One Step RT-PCR kit used was Bio-Rad Laboratories. The model number of the iScript One Step RT-PCR kit used was 170-8892. 50 ng of total RNA was reverse transcribed and amplified. The temperature for reverse transcription was 50 C for 10 min. The initial denaturation temperature for PCR was 95 C for 5 min. The denaturation temperature for each PCR cycle was 95 C for 10 s. The annealing temperature for each PCR cycle was 60 C for 30 s. The extension temperature for each PCR cycle was 72 C for 1 min. Steps 2-4 were repeated for 40 cycles. Each reaction was performed in triplicate. The results of three independent experiments were used for statistical analysis. Relative mRNA expression levels were quantified using the DeltaDeltaC(t) method. Results were normalized to those for RP49. The primer sequence for Rp49 was TACAGGCCCAAGATCGTGAA. The primer sequence for Rp49 was reverse, TCTCCTTGCGCTTCTTGGA. The forward primer sequence for BuGZ was AAGCCACCTTAGGATCATGG. The reverse primer sequence for BuGZ was GGGTCCGGTGTACAACTTCT.\n",
      "node 83:\n",
      "Fig. S1 shows an effect of BuGZ reduction on the mitotic index. The effect shown in Fig. S1 is related to cell proliferation. Fig. S2 demonstrates a tumor repression effect caused by treatment with either BuGZ RNAi or pladienolide B. Treatment with splicing inhibitor causes DNA damage. Forced expression of RNase H1 can rescue the DNA damage caused by splicing inhibitor treatment. Table S1 contains information about differential alternative splicing events. These differential alternative splicing events were caused by either BuGZ RNAi, Bub3 RNAi, or pladienolide B treatment.\n",
      "node 84:\n",
      "Table S2 lists sources and sequences of various biological materials and reagents. This includes mammalian expression plasmids, antibodies with dilutions used, siRNA, shRNA, small molecules with concentrations used, and cell lines. Online supplemental material is available at http://www.jcb.org/cgi/content/full/jcb.201409073/DC1. Additional data are also available in the JCB DataViewer at http://dx.doi.org/10.1083/jcb.201409073.dv.\n",
      "node 85:\n",
      "SA-beta-gal is known as senescence-associated beta-gal.\n",
      "node 86:\n",
      "HFF stands for human foreskin fibroblast.\n",
      "node 87:\n",
      "Aguilera was a person with surname Aguilera and given name unspecified. Garcia-Muse was a person with surname Garcia-Muse and given name T.\n",
      "node 88:\n",
      "Blagosklonny, M.V., wrote Mol.Cell. Mol.Cell.R loops were formed from transcription byproducts. The R loops threaten genome stability. Campisi, J., and d'Adda di Fagagna, F., wrote Cell Cycle. Prolonged mitosis occurred versus tetraploid checkpoint in Cell Cycle. p53 measured the duration of mitosis. Campos, P.B., Sartore, R.C., Abdalla, S.N., Rehen, S.K., wrote Nat.Rev.Mol.Cell.Biol. Cellular senescence happened to good cells when bad things occurred in Nat.Rev.Mol.Cell.Biol. Cerritelli, S.M., Frolova, E.G., Feng, C., Grinberg, A., Love, P.E., and Crouch, R.J., wrote J.Vis.Exp. Human embryonic stem cells were prepared for karyotyping through chromosomal spread preparation in J.Vis.Exp.\n",
      "node 89:\n",
      "Chen, Q.M., and Merrett, J.B., wrote Mol.Cell.\n",
      "node 90:\n",
      "Mitochondrial DNA was failed to produce in Rnaseh1 null mice. Rnaseh1 null mice resulted in embryonic lethality.\n",
      "node 91:\n",
      "T Dilley was a person. S Purdom was a person. The surname of T Dilley is Dilley.\n",
      "node 92:\n",
      "The given names of T Dilley are T.\n",
      "node 93:\n",
      "The surname of S Purdom is Purdom. The given names of S Purdom are S.\n",
      "node 94:\n",
      "Oncogene was a thing that something happened to. p53 was down regulated with HPV E6. HPV E6 delayed and modified cell death in oxidant response of human diploid fibroblasts. Cell death was associated with mitosis.\n",
      "node 95:\n",
      "M Drolet was a person. P Phoenix was a person. R Menzel was a person. E Masse was a person. L F Liu was a person. R J Crouch was a person.\n",
      "node 96:\n",
      "Proc Natl Acad Sci USA was a publication.\n",
      "node 97:\n",
      "RNase H was overexpressed. RNase H partially complemented the growth defect of an Escherichia coli delta topA mutant. R-loop formation was a major problem in the absence of DNA topoisomerase I.\n",
      "node 98:\n",
      "K A Effenberger was a person. R J Perriman was a person. W M Bray was a person. R S Lokey was a person. M Ares was a person.\n",
      "node 99:\n",
      "J was a suffix for M Ares.\n",
      "node 100:\n",
      "M Jurica was a person. J Biomol Screen was a publication. A high-throughput splicing assay identified new classes of inhibitors of human and yeast spliceosomes.\n",
      "node 101:\n",
      "F.A. Eskens was a person with surname Eskens and given names F.A.\n",
      "node 102:\n",
      "F.J. Ramos was a person with surname Ramos and given names F.J.\n",
      "node 103:\n",
      "J. Tabernero was a person with surname Tabernero and given names J. H. Burger was a person with surname Burger and given names H. J.P. O'Brien was a person with surname O'Brien and given names J.P. A. Piera was a person with surname Piera and given names A. M.J. de Jonge was a person with surname de Jonge and given names M.J. Y. Mizui was a person with surname Mizui and given names Y. E.A. Wiemer was a person with surname Wiemer and given names E.A. M.J. Carreras was a person with surname Carreras and given names M.J. J. Baselga was a person with surname Baselga and given names J. S.A. Forbes was a person with surname Forbes and given names S.A. N. Bindal was a person with surname Bindal and given names N. S. Bamford was a person with surname Bamford and given names S. C. Cole was a person with surname Cole and given names C. C.Y. Kok was a person with surname Kok and given names C.Y. D. Beare was a person with surname Beare and given names D. M. Jia was a person with surname Jia and given names M. R. Shepherd was a person with surname Shepherd and given names R. K. Leung was a person with surname Leung and given names K. A. Menzies was a person with surname Menzies and given names A. W. Gan was a person with surname Gan and given names W. Z. Guan was a person with surname Guan and given names Z. J. Liu was a person with surname Liu and given names J. T. Gui was a person with surname Gui and given names T. K. Shen was a person with surname Shen and given names K. J.L. Manley was a person with surname Manley and given names J.L. X. Li was a person with surname Li and given names X.\n",
      "node 104:\n",
      "N.J. Ganem and D. J. Pellman caused R-loop-mediated genomic instability. R-loop-mediated genomic instability was caused by impairment of replication fork progression. K.E. Gascoigne and S.S. Taylor linked abnormal mitosis to the acquisition of DNA damage. Abnormal mitosis is linked to the acquisition of DNA damage. Cancer cells display profound intra- and interline variation following prolonged exposure to antimitotic drugs. Drosophila melanogaster is a model and a tool to investigate malignancy and identify new therapeutic strategies. M. Groh, M.M. Lufino, R. Wade-Martins, and N. Gromak used Drosophila melanogaster as a model and a tool to investigate malignancy and identify new therapeutic strategies. R-loops associated with triplet repeat expansions promote gene silencing in Friedreich ataxia and fragile X syndromes. M.T. Hayashi, A.J. Cesare, J.A. Fitzpatrick, and E. Lazzerini-Denchi found that R-loops associated with triplet repeat expansions promote gene silencing in Friedreich ataxia and fragile X syndromes.\n",
      "node 105:\n",
      "J. Karlseder was a researcher. J. Karlseder published an article in Nat. Struct. Mol. Biol. The article by J. Karlseder was titled 'A telomere-dependent DNA damage checkpoint induced by prolonged mitotic arrest' A. Hegele was a researcher. A. Hegele published an article in unknown publication Unknown publication title by A. Hegele A. Kamburov was a researcher. A. Kamburov published an article in unknown publication\n",
      "node 106:\n",
      "Unknown publication title by A. Kamburov\n",
      "node 107:\n",
      "A. Grossmann was a researcher. A. Grossmann published an article in unknown publication Unknown publication title by A. Grossmann C. Sourlis was a researcher. C. Sourlis published an article in unknown publication Unknown publication title by C. Sourlis S. Wowro was a researcher. S. Wowro published an article in unknown publication Unknown publication title by S. Wowro M. Weimann was a researcher. M. Weimann published an article in unknown publication Unknown publication title by M. Weimann\n",
      "node 108:\n",
      "C.L. Will was a researcher. C.L. Will published an article in unknown publication Unknown publication title by C.L. Will V. Pena was a researcher. V. Pena published an article in Mol. Cell The article by V. Pena was titled 'Dynamic protein-protein interaction wiring of the human spliceosome' H.C. Huang was a researcher. H.C. Huang published an article in unknown publication Unknown publication title by H.C. Huang J. Shi was a researcher. J. Shi published an article in unknown publication Unknown publication title by J. Shi J.D. Orth was a researcher. J.D. Orth published an article in Cell Cycle The article by J.D. Orth was titled 'Cell death when the SAC is out of commission' T.J. Mitchison was a researcher. T.J. Mitchison published an article in Cell Cycle The article by T.J. Mitchison was titled 'Cell death when the SAC is out of commission'\n",
      "node 109:\n",
      "H. Inuzuka was a researcher.\n",
      "node 110:\n",
      "H. Inuzuka published an article in unknown publication Unknown publication title by H. Inuzuka S. Shaik was a researcher.\n",
      "node 111:\n",
      "S. Shaik published an article in unknown publication Unknown publication title by S. Shaik I. Onoyama was a researcher.\n",
      "node 112:\n",
      "I. Onoyama published an article in unknown publication Unknown publication title by I. Onoyama\n",
      "node 113:\n",
      "D. Gao was a researcher.\n",
      "node 114:\n",
      "D. Gao published an article in unknown publication Unknown publication title by D. Gao\n",
      "node 115:\n",
      "A. Tseng was a researcher. A. Tseng published an article in unknown publication Unknown publication title by A. Tseng\n",
      "node 116:\n",
      "R.S. Maser was a researcher. R.S. Maser published an article in unknown publication Unknown publication title by R.S. Maser\n",
      "node 117:\n",
      "B. Zhai was a researcher. B. Zhai published an article in unknown publication Unknown publication title by B. Zhai L. Wan was a researcher. L. Wan published an article in unknown publication Unknown publication title by L. Wan A. Gutierrez was a researcher. A. Gutierrez published an article in unknown publication Unknown publication title by A. Gutierrez\n",
      "node 118:\n",
      "A.W. surname:Lau regulates cellular apoptosis. A.W. Lau targets MCL1 for ubiquitylation and destruction by Nature.SCF(FBW7). N. Ismail, A.H. Pihie, and M. Nallapan are researchers of Anticancer Res.\n",
      "node 119:\n",
      "H. Jiang, X. He, S. Wang, J. Jia, Y. Wan, Y. Wang, R. Zeng, J. Yates III, and X. Zhu, and Y. Zheng are authors of various scientific publications. A microtubule-associated zinc finger protein is regulated by BuGZ in Dev. Cell. BuGZ ensures Bub3 stability and kinetochore targeting to regulate mitotic chromosome alignment.\n",
      "node 120:\n",
      "K.H. Khoo and C.S. Verma are authors of Nat. Rev. Drug Discov. D.P. Lane writes about the p53 pathway for clinical efficacious understanding in Nat. Rev. Drug Discov. A.A. Kiger and D.L. Jones are researchers. C. Schulz is a researcher.\n",
      "node 121:\n",
      "M.B. Rogers was a researcher. M.T. Fuller was a researcher. JAK-STAT activation specified stem cell self-renewal by M.B. Rogers and M.T. Fuller. The research of M.B. Rogers and M.T. Fuller involved support cell cues. D. Kim was a researcher. G. Pertea was a researcher. C. Trapnell was a researcher. H. Pimentel was a researcher. R. Kelley was a researcher. S.L. Salzberg was a researcher. Genome Biol published research by S.L. Salzberg et al. J.M. King was a researcher. T.S. Hays was a researcher. R.B. Nicklas was a researcher. J.M. King, T.S. Hays, and R.B. Nicklas published research in J. Cell Biol. Dynein is a transient kinetochore component by J.M. King, T.S. Hays, and R.B. Nicklas. The binding of dynein was regulated by microtubule attachment. Microtubule attachment, not tension, regulates the binding of dynein. P.G. Komarov was a researcher. E.A. Komarova was a researcher. R.V. Kondratov was a researcher. K. Christov-Tselkov was a researcher.\n",
      "node 122:\n",
      "J.S. Coon was a researcher. M.V. Chernov was a researcher. A.V. Gudkov was a researcher. Science published research by A.V. Gudkov et al. The research of A.V. Gudkov et al. involved a chemical inhibitor of p53. The chemical inhibitor of p53 protected mice from cancer therapy side effects. Y. Kotake was a researcher.\n",
      "node 123:\n",
      "K. Sagane was a researcher.\n",
      "node 124:\n",
      "The surname of Owa is Owa. The given names of Owa are T. The surname of Mimori-Kiyosue is Mimori-Kiyosue. The given names of Mimori-Kiyosue are Y. The surname of Shimizu is Shimizu. The given names of Shimizu are H. The surname of Uesugi is Uesugi. The given names of Uesugi are M. The surname of Ishihama is Ishihama. The given names of Ishihama are Y. The surname of Iwata is Iwata. The given names of Iwata are M. The surname of Mizui is Mizui. The given names of Mizui are Y.\n",
      "node 125:\n",
      "Splicing factor SF3b as a target of the antitumor natural product pladienolides.\n",
      "node 126:\n",
      "The surname of Lanni is Lanni. The given names of Lanni are J.S. The surname of Jacks is Jacks. The given names of Jacks are T.\n",
      "node 127:\n",
      "Characterization of the p53-dependent postmitotic checkpoint following spindle disruptions.\n",
      "node 128:\n",
      "The surname of Li is Li.\n",
      "node 129:\n",
      "The given names of Li are X.\n",
      "node 130:\n",
      "The surname of Manley is Manley. The given names of Manley are J.L.\n",
      "node 131:\n",
      "Inactivation of the SR protein splicing factor ASF/SF2 results in genomic instability.\n",
      "node 132:\n",
      "The surname of Lin is Lin. The given names of Lin are Y. The surname of Dent is Dent. The given names of Dent are S.Y. The surname of Wilson is Wilson. The given names of Wilson are J.H. The surname of Wells is Wells. The given names of Wells are R.D.\n",
      "node 133:\n",
      "The surname of Napierala is Napierala. The given names of Napierala are M.\n",
      "node 134:\n",
      "R loops stimulate genetic instability of CTG CAG repeats.\n",
      "node 135:\n",
      "The surname of McDowell is McDowell. The given names of McDowell are E.M. The surname of Trump is Trump. The given names of Trump are B.F.\n",
      "node 136:\n",
      "Histologic fixatives suitable for diagnostic light and electron microscopies were developed by J.D. Orth. J.D. Orth, A. Loewer, G. Lahav, and T.J. Mitchison published a paper on Mol. Biol. Cell. Prolonged mitotic arrest triggers partial activation of apoptosis in cells. Partial activation of apoptosis results in DNA damage and p53 induction. J.W. Park, C. Tokheim, S. Shen, Y. Xing, and others published a paper on Methods Mol. Biol. RNASeq-MATS is a tool for identifying differential alternative splicing events from RNA sequencing data. R.D. Paulsen, D.V. Soni, R. Wollman, A.T. Hahn, M.C. Yee, A. Guan, J.A. Hesley, S.C. Miller, E.F. Cromwell, and D.E. Solow-Cordero were involved in the development of RNASeq-MATS. A paper was published on Mol. Cell.\n",
      "node 137:\n",
      "A genome-wide siRNA screen was performed by Pfaffl, M.W. Pfaffl, M.W., conducted research on diverse cellular processes and pathways that mediate genome stability Nucleic Acids Res. published a study by Pfaffl, M.W., and others\n",
      "node 138:\n",
      "A new mathematical model for relative quantification in real-time RT-PCRs was developed by Phillips, D.D., Garboczi, D.N., Singh, K., Hu, Z., Leppla, S.H., Leysath, C.E., Pilch, D.R., Sedelnikova, O.A., Redon, C., Celeste, A., Nussenzweig, A., and Bonner, W.M. Phillips, D.D., Garboczi, D.N., Singh, K., Hu, Z., Leppla, S.H., Leysath, C.E., Pilch, D.R., Sedelnikova, O.A., Redon, C., Celeste, A., Nussenzweig, A., and Bonner, W.M. published their research in Biochem. Cell Biol. Characteristics of gamma-H2AX foci at DNA double-strand breaks sites were studied by Pollier, J., Rombauts, S., Goossens, A., Porameesanaporn, Y., Uthaisang-Tanechpongtamb, W., and Jarintanan, F. Pollier, J., Rombauts, S., Goossens, A. published their research in Methods Mol. Biol. Uthaisang-Tanechpongtamb, W., and Jarintanan, F. contributed to the study of gamma-H2AX foci at DNA double-strand breaks sites Porameesanaporn, Y., Uthaisang-Tanechpongtamb, W., and Jarintanan, F. were authors of the article published in Methods Mol. Biol.\n",
      "node 139:\n",
      "The surname of one person is Jongrungruangchok.\n",
      "node 140:\n",
      "The given names of that person are Jongrungruangchok. The surname of another person is Thanomsub Wongsatayanon. The given names of that person are Thanomsub Wongsatayanon. That person's occupation is Oncol Rep.\n",
      "node 141:\n",
      "Terrein induces apoptosis in HeLa human cervical carcinoma cells. The process of apoptosis induced by Terrein involves p53 regulations. The process of apoptosis induced by Terrein involves ERK regulations.\n",
      "node 142:\n",
      "The surname of another person is Quignon.\n",
      "node 143:\n",
      "The given names of that person are Quignon.\n",
      "node 144:\n",
      "The surname of another person is Rozier. The given names of that person are Rozier. The surname of another person is Lachages. The given names of that person are A.M. The surname of another person is Bieth. The given names of that person are A. The surname of another person is Simili.\n",
      "node 145:\n",
      "The given names of that person are M.\n",
      "node 146:\n",
      "The surname of another person is Debatisse. The given names of that person are M.\n",
      "node 147:\n",
      "That person's occupation is Oncogene. Sustained mitotic block elicits DNA breaks in mammalian cells. One-step alteration of ploidy and chromosome integrity occurs in mammalian cells.\n",
      "node 148:\n",
      "The surname of another person is Sato. The given names of that person are M. The surname of another person is Muguruma. The given names of that person are N. The surname of another person is Nakagawa. The given names of that person are T. The surname of another person is Okamoto.\n",
      "node 149:\n",
      "The given names of that person are K.\n",
      "node 150:\n",
      "The surname of another person is Kimura. The given names of that person are T.\n",
      "node 151:\n",
      "The surname of another person is Kitamura. The given names of that person are S.\n",
      "node 152:\n",
      "The surname of another person is Yano.\n",
      "node 153:\n",
      "The given names of that person are H.\n",
      "node 154:\n",
      "The surname of another person is Sannomiya. The given names of that person are K.\n",
      "node 155:\n",
      "The surname of another person is Goji. The given names of that person are T. The surname of another person is Miyamoto.\n",
      "node 156:\n",
      "The given names of that person are H.\n",
      "node 157:\n",
      "That person's occupation is Cancer Sci. Pladienolide B and its derivative have high antitumor activity in gastric cancer.\n",
      "node 158:\n",
      "The surname of another person is Scheffner. The given names of that person are M.\n",
      "node 159:\n",
      "The surname of another person is Werness.\n",
      "node 160:\n",
      "The given names of that person are B.A.\n",
      "node 161:\n",
      "J.M Huibregtse was a person with surname Huibregtse. A.J Levine was a person with surname Levine. P.M Howley was a person with surname Howley.\n",
      "node 162:\n",
      "The E6 oncoprotein encoded by human papillomavirus types 16 and 18 promotes the degradation of p53.\n",
      "node 163:\n",
      "S Shen was a person with surname Shen. J.W Park was a person with surname Park. J Huang was a person with surname Huang. K.A Dittmar was a person with surname Dittmar. Z.X Lu was a person with surname Lu. Q Zhou was a person with surname Zhou. R.P Carstens was a person with surname Carstens. Y Xing was a person with surname Xing. G Song was a person with surname Song. Y.B Mao was a person with surname Mao. Q.F Cai was a person with surname Cai. L.M Yao was a person with surname Yao. G.L Ouyang was a person with surname Ouyang. S.D Bao was a person with surname Bao.\n",
      "node 164:\n",
      "Curcumin induces human HT-29 colon adenocarcinoma cell apoptosis by activating p53 and regulating apoptosis-related protein expressions.\n",
      "node 165:\n",
      "F Stahler was a person with surname Stahler. K Roemer was a person with surname Roemer.\n",
      "node 166:\n",
      "Mutant p53 can provoke apoptosis Apoptosis induced by mutant p53 occurs in p53-deficient Hep3B cells The kinetics of apoptosis induced by mutant p53 are delayed relative to wild-type p53 O. Surova and B. Zhivotovsky are authors of an article on mutant p53 and apoptosis The article was published in Oncogene Various modes of cell death can be induced by DNA damage C.M. Toledo, J.A. Herman, J.B. Olsen, Y. Ding, P. Corrin, E.J. Girard, J.M. Olson, A. Emili, J.G. DeLuca, and P.J. Paddison are authors of an article on cell death and DNA damage The article was published in Dev. Cell BuGZ is required for Bub3 stability Bub1 kinetochore function also requires BuGZ Chromosome alignment also requires BuGZ C. Trapnell, B.A. Williams, G. Pertea, A. Mortazavi, G. Kwan, M.J. van Baren, S.L. Salzberg, B.J. Wold, and L. Pachter are authors of an article on BuGZ and chromosome alignment The article was published in Nat. Biotechnol\n",
      "node 167:\n",
      "C Trapnell, A Roberts, L Goff, G Pertea, D Kim, D R Kelley, H Pimentel, S L Salzberg, J L Rinn, and L Pachter conducted a study on transcript assembly and quantification by RNA-Seq. The study aimed to reveal unannotated transcripts and isoform switching during cell differentiation. C Trapnell was the surname of the first author of the study. A Roberts was the given name of one of the authors. L Goff was another author of the study. G Pertea was also an author. D Kim was a participant in the study. D R Kelley was another researcher involved. H Pimentel contributed to the study as well. S L Salzberg was one of the co-authors. J L Rinn participated in the research. L Pachter, along with others, conducted the research as described. Y Uetake and G Sluder conducted another study on prolonged prometaphase blocks daughter cell proliferation. The study was published in Current Biology. Prolonged prometaphase blocks halted daughter cell proliferation despite normal completion of mitosis. L Wahba, J D Amon, D Koshland, and M Vuica-Ross conducted a study on RNase H. The study involved multiple RNA biogenesis factors cooperating to prevent genome instability.\n",
      "node 168:\n",
      "M C Wahl was the surname of an author.\n",
      "node 169:\n",
      "C L Will participated in the research. R Luhrmann contributed to the study as well.\n",
      "node 170:\n",
      "The spliceosome was designed with specific principles by I.E. Wertz. I.E. Wertz, S Kusam, C Lam, T Okamoto, W Sandoval, D.J Anderson, E Helgason, J.A Ernst, M Eby, and J Liu were authors who contributed to the Nature publication. The design principles of a dynamic RNP machine were also investigated by I.E. Wertz and others.\n",
      "node 171:\n",
      "MCL1 and FBW7 are proteins that regulate sensitivity to antitubulin chemotherapeutics.\n",
      "node 172:\n",
      "H.Wimberly, C Shee, P.C Thornton, P Sivaramakrishnan, S.M Rosenberg, and P.J Hastings were authors who contributed to a Nat. Commun publication. Sensitivity to antitubulin chemotherapeutics is regulated by MCL1 and FBW7 according to H Wimberly and others.\n",
      "node 173:\n",
      "T Wongsurawat, P Jenjaroenpun, C.K Kwoh, V Kuznetsov were authors who contributed to a Nucleic Acids Res publication. A novel level of RNA-DNA interactome complexity was revealed by a quantitative model of R-loop forming structures.\n"
     ]
    }
   ],
   "source": [
    "# WITHOUT excluding references & supplements\n",
    "parsed = topic_parse(nodes, chunk_size=2000, similarity_threshold=0.8, window_size=5, similarity_method=\"llm\")\n",
    "print_topic_nodes(parsed)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Failed to parse JSON: \n"
     ]
    },
    {
     "ename": "ReadTimeout",
     "evalue": "timed out",
     "output_type": "error",
     "traceback": [
      "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
      "\u001b[0;31mReadTimeout\u001b[0m                               Traceback (most recent call last)",
      "File \u001b[0;32m~/.local/lib/python3.10/site-packages/httpx/_transports/default.py:69\u001b[0m, in \u001b[0;36mmap_httpcore_exceptions\u001b[0;34m()\u001b[0m\n\u001b[1;32m     68\u001b[0m \u001b[38;5;28;01mtry\u001b[39;00m:\n\u001b[0;32m---> 69\u001b[0m     \u001b[38;5;28;01myield\u001b[39;00m\n\u001b[1;32m     70\u001b[0m \u001b[38;5;28;01mexcept\u001b[39;00m \u001b[38;5;167;01mException\u001b[39;00m \u001b[38;5;28;01mas\u001b[39;00m exc:\n",
      "File \u001b[0;32m~/.local/lib/python3.10/site-packages/httpx/_transports/default.py:233\u001b[0m, in \u001b[0;36mHTTPTransport.handle_request\u001b[0;34m(self, request)\u001b[0m\n\u001b[1;32m    232\u001b[0m \u001b[38;5;28;01mwith\u001b[39;00m map_httpcore_exceptions():\n\u001b[0;32m--> 233\u001b[0m     resp \u001b[38;5;241m=\u001b[39m \u001b[38;5;28;43mself\u001b[39;49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43m_pool\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mhandle_request\u001b[49m\u001b[43m(\u001b[49m\u001b[43mreq\u001b[49m\u001b[43m)\u001b[49m\n\u001b[1;32m    235\u001b[0m \u001b[38;5;28;01massert\u001b[39;00m \u001b[38;5;28misinstance\u001b[39m(resp\u001b[38;5;241m.\u001b[39mstream, typing\u001b[38;5;241m.\u001b[39mIterable)\n",
      "File \u001b[0;32m~/.local/lib/python3.10/site-packages/httpcore/_sync/connection_pool.py:216\u001b[0m, in \u001b[0;36mConnectionPool.handle_request\u001b[0;34m(self, request)\u001b[0m\n\u001b[1;32m    215\u001b[0m     \u001b[38;5;28mself\u001b[39m\u001b[38;5;241m.\u001b[39m_close_connections(closing)\n\u001b[0;32m--> 216\u001b[0m     \u001b[38;5;28;01mraise\u001b[39;00m exc \u001b[38;5;28;01mfrom\u001b[39;00m \u001b[38;5;28;01mNone\u001b[39;00m\n\u001b[1;32m    218\u001b[0m \u001b[38;5;66;03m# Return the response. Note that in this case we still have to manage\u001b[39;00m\n\u001b[1;32m    219\u001b[0m \u001b[38;5;66;03m# the point at which the response is closed.\u001b[39;00m\n",
      "File \u001b[0;32m~/.local/lib/python3.10/site-packages/httpcore/_sync/connection_pool.py:196\u001b[0m, in \u001b[0;36mConnectionPool.handle_request\u001b[0;34m(self, request)\u001b[0m\n\u001b[1;32m    194\u001b[0m \u001b[38;5;28;01mtry\u001b[39;00m:\n\u001b[1;32m    195\u001b[0m     \u001b[38;5;66;03m# Send the request on the assigned connection.\u001b[39;00m\n\u001b[0;32m--> 196\u001b[0m     response \u001b[38;5;241m=\u001b[39m \u001b[43mconnection\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mhandle_request\u001b[49m\u001b[43m(\u001b[49m\n\u001b[1;32m    197\u001b[0m \u001b[43m        \u001b[49m\u001b[43mpool_request\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mrequest\u001b[49m\n\u001b[1;32m    198\u001b[0m \u001b[43m    \u001b[49m\u001b[43m)\u001b[49m\n\u001b[1;32m    199\u001b[0m \u001b[38;5;28;01mexcept\u001b[39;00m ConnectionNotAvailable:\n\u001b[1;32m    200\u001b[0m     \u001b[38;5;66;03m# In some cases a connection may initially be available to\u001b[39;00m\n\u001b[1;32m    201\u001b[0m     \u001b[38;5;66;03m# handle a request, but then become unavailable.\u001b[39;00m\n\u001b[1;32m    202\u001b[0m     \u001b[38;5;66;03m#\u001b[39;00m\n\u001b[1;32m    203\u001b[0m     \u001b[38;5;66;03m# In this case we clear the connection and try again.\u001b[39;00m\n",
      "File \u001b[0;32m~/.local/lib/python3.10/site-packages/httpcore/_sync/connection.py:101\u001b[0m, in \u001b[0;36mHTTPConnection.handle_request\u001b[0;34m(self, request)\u001b[0m\n\u001b[1;32m     99\u001b[0m     \u001b[38;5;28;01mraise\u001b[39;00m exc\n\u001b[0;32m--> 101\u001b[0m \u001b[38;5;28;01mreturn\u001b[39;00m \u001b[38;5;28;43mself\u001b[39;49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43m_connection\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mhandle_request\u001b[49m\u001b[43m(\u001b[49m\u001b[43mrequest\u001b[49m\u001b[43m)\u001b[49m\n",
      "File \u001b[0;32m~/.local/lib/python3.10/site-packages/httpcore/_sync/http11.py:143\u001b[0m, in \u001b[0;36mHTTP11Connection.handle_request\u001b[0;34m(self, request)\u001b[0m\n\u001b[1;32m    142\u001b[0m         \u001b[38;5;28mself\u001b[39m\u001b[38;5;241m.\u001b[39m_response_closed()\n\u001b[0;32m--> 143\u001b[0m \u001b[38;5;28;01mraise\u001b[39;00m exc\n",
      "File \u001b[0;32m~/.local/lib/python3.10/site-packages/httpcore/_sync/http11.py:113\u001b[0m, in \u001b[0;36mHTTP11Connection.handle_request\u001b[0;34m(self, request)\u001b[0m\n\u001b[1;32m    104\u001b[0m \u001b[38;5;28;01mwith\u001b[39;00m Trace(\n\u001b[1;32m    105\u001b[0m     \u001b[38;5;124m\"\u001b[39m\u001b[38;5;124mreceive_response_headers\u001b[39m\u001b[38;5;124m\"\u001b[39m, logger, request, kwargs\n\u001b[1;32m    106\u001b[0m ) \u001b[38;5;28;01mas\u001b[39;00m trace:\n\u001b[1;32m    107\u001b[0m     (\n\u001b[1;32m    108\u001b[0m         http_version,\n\u001b[1;32m    109\u001b[0m         status,\n\u001b[1;32m    110\u001b[0m         reason_phrase,\n\u001b[1;32m    111\u001b[0m         headers,\n\u001b[1;32m    112\u001b[0m         trailing_data,\n\u001b[0;32m--> 113\u001b[0m     ) \u001b[38;5;241m=\u001b[39m \u001b[38;5;28;43mself\u001b[39;49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43m_receive_response_headers\u001b[49m\u001b[43m(\u001b[49m\u001b[38;5;241;43m*\u001b[39;49m\u001b[38;5;241;43m*\u001b[39;49m\u001b[43mkwargs\u001b[49m\u001b[43m)\u001b[49m\n\u001b[1;32m    114\u001b[0m     trace\u001b[38;5;241m.\u001b[39mreturn_value \u001b[38;5;241m=\u001b[39m (\n\u001b[1;32m    115\u001b[0m         http_version,\n\u001b[1;32m    116\u001b[0m         status,\n\u001b[1;32m    117\u001b[0m         reason_phrase,\n\u001b[1;32m    118\u001b[0m         headers,\n\u001b[1;32m    119\u001b[0m     )\n",
      "File \u001b[0;32m~/.local/lib/python3.10/site-packages/httpcore/_sync/http11.py:186\u001b[0m, in \u001b[0;36mHTTP11Connection._receive_response_headers\u001b[0;34m(self, request)\u001b[0m\n\u001b[1;32m    185\u001b[0m \u001b[38;5;28;01mwhile\u001b[39;00m \u001b[38;5;28;01mTrue\u001b[39;00m:\n\u001b[0;32m--> 186\u001b[0m     event \u001b[38;5;241m=\u001b[39m \u001b[38;5;28;43mself\u001b[39;49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43m_receive_event\u001b[49m\u001b[43m(\u001b[49m\u001b[43mtimeout\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43mtimeout\u001b[49m\u001b[43m)\u001b[49m\n\u001b[1;32m    187\u001b[0m     \u001b[38;5;28;01mif\u001b[39;00m \u001b[38;5;28misinstance\u001b[39m(event, h11\u001b[38;5;241m.\u001b[39mResponse):\n",
      "File \u001b[0;32m~/.local/lib/python3.10/site-packages/httpcore/_sync/http11.py:224\u001b[0m, in \u001b[0;36mHTTP11Connection._receive_event\u001b[0;34m(self, timeout)\u001b[0m\n\u001b[1;32m    223\u001b[0m \u001b[38;5;28;01mif\u001b[39;00m event \u001b[38;5;129;01mis\u001b[39;00m h11\u001b[38;5;241m.\u001b[39mNEED_DATA:\n\u001b[0;32m--> 224\u001b[0m     data \u001b[38;5;241m=\u001b[39m \u001b[38;5;28;43mself\u001b[39;49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43m_network_stream\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mread\u001b[49m\u001b[43m(\u001b[49m\n\u001b[1;32m    225\u001b[0m \u001b[43m        \u001b[49m\u001b[38;5;28;43mself\u001b[39;49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mREAD_NUM_BYTES\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43mtimeout\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43mtimeout\u001b[49m\n\u001b[1;32m    226\u001b[0m \u001b[43m    \u001b[49m\u001b[43m)\u001b[49m\n\u001b[1;32m    228\u001b[0m     \u001b[38;5;66;03m# If we feed this case through h11 we'll raise an exception like:\u001b[39;00m\n\u001b[1;32m    229\u001b[0m     \u001b[38;5;66;03m#\u001b[39;00m\n\u001b[1;32m    230\u001b[0m     \u001b[38;5;66;03m#     httpcore.RemoteProtocolError: can't handle event type\u001b[39;00m\n\u001b[0;32m   (...)\u001b[0m\n\u001b[1;32m    234\u001b[0m     \u001b[38;5;66;03m# perspective. Instead we handle this case distinctly and treat\u001b[39;00m\n\u001b[1;32m    235\u001b[0m     \u001b[38;5;66;03m# it as a ConnectError.\u001b[39;00m\n",
      "File \u001b[0;32m~/.local/lib/python3.10/site-packages/httpcore/_backends/sync.py:124\u001b[0m, in \u001b[0;36mSyncStream.read\u001b[0;34m(self, max_bytes, timeout)\u001b[0m\n\u001b[1;32m    123\u001b[0m exc_map: ExceptionMapping \u001b[38;5;241m=\u001b[39m {socket\u001b[38;5;241m.\u001b[39mtimeout: ReadTimeout, \u001b[38;5;167;01mOSError\u001b[39;00m: ReadError}\n\u001b[0;32m--> 124\u001b[0m \u001b[38;5;28;01mwith\u001b[39;00m map_exceptions(exc_map):\n\u001b[1;32m    125\u001b[0m     \u001b[38;5;28mself\u001b[39m\u001b[38;5;241m.\u001b[39m_sock\u001b[38;5;241m.\u001b[39msettimeout(timeout)\n",
      "File \u001b[0;32m/usr/lib/python3.10/contextlib.py:153\u001b[0m, in \u001b[0;36m_GeneratorContextManager.__exit__\u001b[0;34m(self, typ, value, traceback)\u001b[0m\n\u001b[1;32m    152\u001b[0m \u001b[38;5;28;01mtry\u001b[39;00m:\n\u001b[0;32m--> 153\u001b[0m     \u001b[38;5;28;43mself\u001b[39;49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mgen\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mthrow\u001b[49m\u001b[43m(\u001b[49m\u001b[43mtyp\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43mvalue\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43mtraceback\u001b[49m\u001b[43m)\u001b[49m\n\u001b[1;32m    154\u001b[0m \u001b[38;5;28;01mexcept\u001b[39;00m \u001b[38;5;167;01mStopIteration\u001b[39;00m \u001b[38;5;28;01mas\u001b[39;00m exc:\n\u001b[1;32m    155\u001b[0m     \u001b[38;5;66;03m# Suppress StopIteration *unless* it's the same exception that\u001b[39;00m\n\u001b[1;32m    156\u001b[0m     \u001b[38;5;66;03m# was passed to throw().  This prevents a StopIteration\u001b[39;00m\n\u001b[1;32m    157\u001b[0m     \u001b[38;5;66;03m# raised inside the \"with\" statement from being suppressed.\u001b[39;00m\n",
      "File \u001b[0;32m~/.local/lib/python3.10/site-packages/httpcore/_exceptions.py:14\u001b[0m, in \u001b[0;36mmap_exceptions\u001b[0;34m(map)\u001b[0m\n\u001b[1;32m     13\u001b[0m     \u001b[38;5;28;01mif\u001b[39;00m \u001b[38;5;28misinstance\u001b[39m(exc, from_exc):\n\u001b[0;32m---> 14\u001b[0m         \u001b[38;5;28;01mraise\u001b[39;00m to_exc(exc) \u001b[38;5;28;01mfrom\u001b[39;00m \u001b[38;5;21;01mexc\u001b[39;00m\n\u001b[1;32m     15\u001b[0m \u001b[38;5;28;01mraise\u001b[39;00m\n",
      "\u001b[0;31mReadTimeout\u001b[0m: timed out",
      "\nThe above exception was the direct cause of the following exception:\n",
      "\u001b[0;31mReadTimeout\u001b[0m                               Traceback (most recent call last)",
      "Cell \u001b[0;32mIn[158], line 1\u001b[0m\n\u001b[0;32m----> 1\u001b[0m parsed \u001b[38;5;241m=\u001b[39m \u001b[43mtopic_parse\u001b[49m\u001b[43m(\u001b[49m\u001b[43mnodes\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43mchunk_size\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[38;5;241;43m2000\u001b[39;49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43msimilarity_threshold\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[38;5;241;43m0.8\u001b[39;49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43mwindow_size\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[38;5;241;43m5\u001b[39;49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43msimilarity_method\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[38;5;124;43m\"\u001b[39;49m\u001b[38;5;124;43mllm\u001b[39;49m\u001b[38;5;124;43m\"\u001b[39;49m\u001b[43m)\u001b[49m\n\u001b[1;32m      2\u001b[0m print_topic_nodes(parsed)\n",
      "Cell \u001b[0;32mIn[156], line 17\u001b[0m, in \u001b[0;36mtopic_parse\u001b[0;34m(documents, chunk_size, similarity_threshold, window_size, similarity_method)\u001b[0m\n\u001b[1;32m      6\u001b[0m node_parser \u001b[38;5;241m=\u001b[39m TopicNodeParser\u001b[38;5;241m.\u001b[39mfrom_defaults(\n\u001b[1;32m      7\u001b[0m     embed_model\u001b[38;5;241m=\u001b[39membed_model,\n\u001b[1;32m      8\u001b[0m     llm\u001b[38;5;241m=\u001b[39mllm,\n\u001b[0;32m   (...)\u001b[0m\n\u001b[1;32m     12\u001b[0m     window_size\u001b[38;5;241m=\u001b[39mwindow_size,  \u001b[38;5;66;03m# paper suggests window_size=5\u001b[39;00m\n\u001b[1;32m     13\u001b[0m )\n\u001b[1;32m     14\u001b[0m \u001b[38;5;66;03m# return {\"nodes\": node_parser.get_nodes_from_documents(documents, show_progress=True), \u001b[39;00m\n\u001b[1;32m     15\u001b[0m \u001b[38;5;66;03m#         \"params\": {\"chunk_size\": chunk_size, \"threshold\": similarity_threshold, \"window\": window_size}}\u001b[39;00m\n\u001b[0;32m---> 17\u001b[0m \u001b[38;5;28;01mreturn\u001b[39;00m {\u001b[38;5;124m\"\u001b[39m\u001b[38;5;124mnodes\u001b[39m\u001b[38;5;124m\"\u001b[39m: \u001b[43mnode_parser\u001b[49m\u001b[43m(\u001b[49m\u001b[43mdocuments\u001b[49m\u001b[43m)\u001b[49m, \n\u001b[1;32m     18\u001b[0m         \u001b[38;5;124m\"\u001b[39m\u001b[38;5;124mparams\u001b[39m\u001b[38;5;124m\"\u001b[39m: {\u001b[38;5;124m\"\u001b[39m\u001b[38;5;124mchunk_size\u001b[39m\u001b[38;5;124m\"\u001b[39m: chunk_size, \u001b[38;5;124m\"\u001b[39m\u001b[38;5;124mthreshold\u001b[39m\u001b[38;5;124m\"\u001b[39m: similarity_threshold, \u001b[38;5;124m\"\u001b[39m\u001b[38;5;124mwindow\u001b[39m\u001b[38;5;124m\"\u001b[39m: window_size}}\n",
      "File \u001b[0;32m~/.local/lib/python3.10/site-packages/llama_index/core/instrumentation/dispatcher.py:311\u001b[0m, in \u001b[0;36mDispatcher.span.<locals>.wrapper\u001b[0;34m(func, instance, args, kwargs)\u001b[0m\n\u001b[1;32m    308\u001b[0m             _logger\u001b[38;5;241m.\u001b[39mdebug(\u001b[38;5;124mf\u001b[39m\u001b[38;5;124m\"\u001b[39m\u001b[38;5;124mFailed to reset active_span_id: \u001b[39m\u001b[38;5;132;01m{\u001b[39;00me\u001b[38;5;132;01m}\u001b[39;00m\u001b[38;5;124m\"\u001b[39m)\n\u001b[1;32m    310\u001b[0m \u001b[38;5;28;01mtry\u001b[39;00m:\n\u001b[0;32m--> 311\u001b[0m     result \u001b[38;5;241m=\u001b[39m \u001b[43mfunc\u001b[49m\u001b[43m(\u001b[49m\u001b[38;5;241;43m*\u001b[39;49m\u001b[43margs\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[38;5;241;43m*\u001b[39;49m\u001b[38;5;241;43m*\u001b[39;49m\u001b[43mkwargs\u001b[49m\u001b[43m)\u001b[49m\n\u001b[1;32m    312\u001b[0m     \u001b[38;5;28;01mif\u001b[39;00m \u001b[38;5;28misinstance\u001b[39m(result, asyncio\u001b[38;5;241m.\u001b[39mFuture):\n\u001b[1;32m    313\u001b[0m         \u001b[38;5;66;03m# If the result is a Future, wrap it\u001b[39;00m\n\u001b[1;32m    314\u001b[0m         new_future \u001b[38;5;241m=\u001b[39m asyncio\u001b[38;5;241m.\u001b[39mensure_future(result)\n",
      "File \u001b[0;32m~/.local/lib/python3.10/site-packages/llama_index/core/node_parser/interface.py:193\u001b[0m, in \u001b[0;36mNodeParser.__call__\u001b[0;34m(self, nodes, **kwargs)\u001b[0m\n\u001b[1;32m    192\u001b[0m \u001b[38;5;28;01mdef\u001b[39;00m \u001b[38;5;21m__call__\u001b[39m(\u001b[38;5;28mself\u001b[39m, nodes: Sequence[BaseNode], \u001b[38;5;241m*\u001b[39m\u001b[38;5;241m*\u001b[39mkwargs: Any) \u001b[38;5;241m-\u001b[39m\u001b[38;5;241m>\u001b[39m List[BaseNode]:\n\u001b[0;32m--> 193\u001b[0m     \u001b[38;5;28;01mreturn\u001b[39;00m \u001b[38;5;28;43mself\u001b[39;49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mget_nodes_from_documents\u001b[49m\u001b[43m(\u001b[49m\u001b[43mnodes\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[38;5;241;43m*\u001b[39;49m\u001b[38;5;241;43m*\u001b[39;49m\u001b[43mkwargs\u001b[49m\u001b[43m)\u001b[49m\n",
      "File \u001b[0;32m~/.local/lib/python3.10/site-packages/llama_index/core/node_parser/interface.py:165\u001b[0m, in \u001b[0;36mNodeParser.get_nodes_from_documents\u001b[0;34m(self, documents, show_progress, **kwargs)\u001b[0m\n\u001b[1;32m    160\u001b[0m doc_id_to_document \u001b[38;5;241m=\u001b[39m {doc\u001b[38;5;241m.\u001b[39mid_: doc \u001b[38;5;28;01mfor\u001b[39;00m doc \u001b[38;5;129;01min\u001b[39;00m documents}\n\u001b[1;32m    162\u001b[0m \u001b[38;5;28;01mwith\u001b[39;00m \u001b[38;5;28mself\u001b[39m\u001b[38;5;241m.\u001b[39mcallback_manager\u001b[38;5;241m.\u001b[39mevent(\n\u001b[1;32m    163\u001b[0m     CBEventType\u001b[38;5;241m.\u001b[39mNODE_PARSING, payload\u001b[38;5;241m=\u001b[39m{EventPayload\u001b[38;5;241m.\u001b[39mDOCUMENTS: documents}\n\u001b[1;32m    164\u001b[0m ) \u001b[38;5;28;01mas\u001b[39;00m event:\n\u001b[0;32m--> 165\u001b[0m     nodes \u001b[38;5;241m=\u001b[39m \u001b[38;5;28;43mself\u001b[39;49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43m_parse_nodes\u001b[49m\u001b[43m(\u001b[49m\u001b[43mdocuments\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43mshow_progress\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43mshow_progress\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[38;5;241;43m*\u001b[39;49m\u001b[38;5;241;43m*\u001b[39;49m\u001b[43mkwargs\u001b[49m\u001b[43m)\u001b[49m\n\u001b[1;32m    166\u001b[0m     nodes \u001b[38;5;241m=\u001b[39m \u001b[38;5;28mself\u001b[39m\u001b[38;5;241m.\u001b[39m_postprocess_parsed_nodes(nodes, doc_id_to_document)\n\u001b[1;32m    168\u001b[0m     event\u001b[38;5;241m.\u001b[39mon_end({EventPayload\u001b[38;5;241m.\u001b[39mNODES: nodes})\n",
      "File \u001b[0;32m~/.local/lib/python3.10/site-packages/llama_index/core/instrumentation/dispatcher.py:311\u001b[0m, in \u001b[0;36mDispatcher.span.<locals>.wrapper\u001b[0;34m(func, instance, args, kwargs)\u001b[0m\n\u001b[1;32m    308\u001b[0m             _logger\u001b[38;5;241m.\u001b[39mdebug(\u001b[38;5;124mf\u001b[39m\u001b[38;5;124m\"\u001b[39m\u001b[38;5;124mFailed to reset active_span_id: \u001b[39m\u001b[38;5;132;01m{\u001b[39;00me\u001b[38;5;132;01m}\u001b[39;00m\u001b[38;5;124m\"\u001b[39m)\n\u001b[1;32m    310\u001b[0m \u001b[38;5;28;01mtry\u001b[39;00m:\n\u001b[0;32m--> 311\u001b[0m     result \u001b[38;5;241m=\u001b[39m \u001b[43mfunc\u001b[49m\u001b[43m(\u001b[49m\u001b[38;5;241;43m*\u001b[39;49m\u001b[43margs\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[38;5;241;43m*\u001b[39;49m\u001b[38;5;241;43m*\u001b[39;49m\u001b[43mkwargs\u001b[49m\u001b[43m)\u001b[49m\n\u001b[1;32m    312\u001b[0m     \u001b[38;5;28;01mif\u001b[39;00m \u001b[38;5;28misinstance\u001b[39m(result, asyncio\u001b[38;5;241m.\u001b[39mFuture):\n\u001b[1;32m    313\u001b[0m         \u001b[38;5;66;03m# If the result is a Future, wrap it\u001b[39;00m\n\u001b[1;32m    314\u001b[0m         new_future \u001b[38;5;241m=\u001b[39m asyncio\u001b[38;5;241m.\u001b[39mensure_future(result)\n",
      "File \u001b[0;32m~/.local/lib/python3.10/site-packages/llama_index/node_parser/topic/base.py:120\u001b[0m, in \u001b[0;36mTopicNodeParser._parse_nodes\u001b[0;34m(self, nodes, show_progress, **kwargs)\u001b[0m\n\u001b[1;32m    117\u001b[0m nodes_with_progress \u001b[38;5;241m=\u001b[39m get_tqdm_iterable(nodes, show_progress, \u001b[38;5;124m\"\u001b[39m\u001b[38;5;124mParsing nodes\u001b[39m\u001b[38;5;124m\"\u001b[39m)\n\u001b[1;32m    119\u001b[0m \u001b[38;5;28;01mfor\u001b[39;00m node \u001b[38;5;129;01min\u001b[39;00m nodes_with_progress:\n\u001b[0;32m--> 120\u001b[0m     nodes \u001b[38;5;241m=\u001b[39m \u001b[38;5;28;43mself\u001b[39;49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mbuild_topic_based_nodes_from_documents\u001b[49m\u001b[43m(\u001b[49m\u001b[43m[\u001b[49m\u001b[43mnode\u001b[49m\u001b[43m]\u001b[49m\u001b[43m)\u001b[49m\n\u001b[1;32m    121\u001b[0m     all_nodes\u001b[38;5;241m.\u001b[39mextend(nodes)\n\u001b[1;32m    123\u001b[0m \u001b[38;5;28;01mreturn\u001b[39;00m all_nodes\n",
      "File \u001b[0;32m~/.local/lib/python3.10/site-packages/llama_index/node_parser/topic/base.py:255\u001b[0m, in \u001b[0;36mTopicNodeParser.build_topic_based_nodes_from_documents\u001b[0;34m(self, documents)\u001b[0m\n\u001b[1;32m    253\u001b[0m \u001b[38;5;28;01mfor\u001b[39;00m doc \u001b[38;5;129;01min\u001b[39;00m documents:\n\u001b[1;32m    254\u001b[0m     paragraphs \u001b[38;5;241m=\u001b[39m \u001b[38;5;28mself\u001b[39m\u001b[38;5;241m.\u001b[39msplit_into_paragraphs(doc\u001b[38;5;241m.\u001b[39mtext)\n\u001b[0;32m--> 255\u001b[0m     chunks \u001b[38;5;241m=\u001b[39m \u001b[38;5;28;43mself\u001b[39;49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43msemantic_chunking\u001b[49m\u001b[43m(\u001b[49m\u001b[43mparagraphs\u001b[49m\u001b[43m)\u001b[49m\n\u001b[1;32m    256\u001b[0m     nodes \u001b[38;5;241m=\u001b[39m build_nodes_from_splits(\n\u001b[1;32m    257\u001b[0m         chunks,\n\u001b[1;32m    258\u001b[0m         doc,\n\u001b[1;32m    259\u001b[0m         id_func\u001b[38;5;241m=\u001b[39m\u001b[38;5;28mself\u001b[39m\u001b[38;5;241m.\u001b[39mid_func,\n\u001b[1;32m    260\u001b[0m     )\n\u001b[1;32m    261\u001b[0m     all_nodes\u001b[38;5;241m.\u001b[39mextend(nodes)\n",
      "File \u001b[0;32m~/.local/lib/python3.10/site-packages/llama_index/node_parser/topic/base.py:207\u001b[0m, in \u001b[0;36mTopicNodeParser.semantic_chunking\u001b[0;34m(self, paragraphs)\u001b[0m\n\u001b[1;32m    205\u001b[0m \u001b[38;5;28;01mfor\u001b[39;00m j \u001b[38;5;129;01min\u001b[39;00m \u001b[38;5;28mrange\u001b[39m(start_idx, end_idx):\n\u001b[1;32m    206\u001b[0m     \u001b[38;5;28;01mif\u001b[39;00m j \u001b[38;5;129;01mnot\u001b[39;00m \u001b[38;5;129;01min\u001b[39;00m proposition_cache:\n\u001b[0;32m--> 207\u001b[0m         proposition_cache[j] \u001b[38;5;241m=\u001b[39m \u001b[38;5;28;43mself\u001b[39;49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mproposition_transfer\u001b[49m\u001b[43m(\u001b[49m\u001b[43mparagraphs\u001b[49m\u001b[43m[\u001b[49m\u001b[43mj\u001b[49m\u001b[43m]\u001b[49m\u001b[43m)\u001b[49m\n\u001b[1;32m    208\u001b[0m     window_propositions\u001b[38;5;241m.\u001b[39mextend(proposition_cache[j])\n\u001b[1;32m    210\u001b[0m \u001b[38;5;28;01mfor\u001b[39;00m prop \u001b[38;5;129;01min\u001b[39;00m window_propositions:\n",
      "File \u001b[0;32m~/.local/lib/python3.10/site-packages/llama_index/node_parser/topic/base.py:138\u001b[0m, in \u001b[0;36mTopicNodeParser.proposition_transfer\u001b[0;34m(self, paragraph)\u001b[0m\n\u001b[1;32m    130\u001b[0m \u001b[38;5;250m\u001b[39m\u001b[38;5;124;03m\"\"\"\u001b[39;00m\n\u001b[1;32m    131\u001b[0m \u001b[38;5;124;03mConvert a paragraph into a list of self-sustaining statements using LLM.\u001b[39;00m\n\u001b[1;32m    132\u001b[0m \u001b[38;5;124;03m\"\"\"\u001b[39;00m\n\u001b[1;32m    133\u001b[0m messages \u001b[38;5;241m=\u001b[39m [\n\u001b[1;32m    134\u001b[0m     ChatMessage(role\u001b[38;5;241m=\u001b[39m\u001b[38;5;124m\"\u001b[39m\u001b[38;5;124msystem\u001b[39m\u001b[38;5;124m\"\u001b[39m, content\u001b[38;5;241m=\u001b[39mPROPOSITION_SYSTEM_PROMPT),\n\u001b[1;32m    135\u001b[0m     ChatMessage(role\u001b[38;5;241m=\u001b[39m\u001b[38;5;124m\"\u001b[39m\u001b[38;5;124muser\u001b[39m\u001b[38;5;124m\"\u001b[39m, content\u001b[38;5;241m=\u001b[39mparagraph),\n\u001b[1;32m    136\u001b[0m ]\n\u001b[0;32m--> 138\u001b[0m response \u001b[38;5;241m=\u001b[39m \u001b[38;5;28mstr\u001b[39m(\u001b[38;5;28;43mself\u001b[39;49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mllm\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mchat\u001b[49m\u001b[43m(\u001b[49m\u001b[43mmessages\u001b[49m\u001b[43m)\u001b[49m)\n\u001b[1;32m    140\u001b[0m json_start \u001b[38;5;241m=\u001b[39m response\u001b[38;5;241m.\u001b[39mfind(\u001b[38;5;124m\"\u001b[39m\u001b[38;5;124m[\u001b[39m\u001b[38;5;124m\"\u001b[39m)\n\u001b[1;32m    141\u001b[0m json_end \u001b[38;5;241m=\u001b[39m response\u001b[38;5;241m.\u001b[39mrfind(\u001b[38;5;124m\"\u001b[39m\u001b[38;5;124m]\u001b[39m\u001b[38;5;124m\"\u001b[39m) \u001b[38;5;241m+\u001b[39m \u001b[38;5;241m1\u001b[39m\n",
      "File \u001b[0;32m~/.local/lib/python3.10/site-packages/llama_index/core/instrumentation/dispatcher.py:311\u001b[0m, in \u001b[0;36mDispatcher.span.<locals>.wrapper\u001b[0;34m(func, instance, args, kwargs)\u001b[0m\n\u001b[1;32m    308\u001b[0m             _logger\u001b[38;5;241m.\u001b[39mdebug(\u001b[38;5;124mf\u001b[39m\u001b[38;5;124m\"\u001b[39m\u001b[38;5;124mFailed to reset active_span_id: \u001b[39m\u001b[38;5;132;01m{\u001b[39;00me\u001b[38;5;132;01m}\u001b[39;00m\u001b[38;5;124m\"\u001b[39m)\n\u001b[1;32m    310\u001b[0m \u001b[38;5;28;01mtry\u001b[39;00m:\n\u001b[0;32m--> 311\u001b[0m     result \u001b[38;5;241m=\u001b[39m \u001b[43mfunc\u001b[49m\u001b[43m(\u001b[49m\u001b[38;5;241;43m*\u001b[39;49m\u001b[43margs\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[38;5;241;43m*\u001b[39;49m\u001b[38;5;241;43m*\u001b[39;49m\u001b[43mkwargs\u001b[49m\u001b[43m)\u001b[49m\n\u001b[1;32m    312\u001b[0m     \u001b[38;5;28;01mif\u001b[39;00m \u001b[38;5;28misinstance\u001b[39m(result, asyncio\u001b[38;5;241m.\u001b[39mFuture):\n\u001b[1;32m    313\u001b[0m         \u001b[38;5;66;03m# If the result is a Future, wrap it\u001b[39;00m\n\u001b[1;32m    314\u001b[0m         new_future \u001b[38;5;241m=\u001b[39m asyncio\u001b[38;5;241m.\u001b[39mensure_future(result)\n",
      "File \u001b[0;32m~/.local/lib/python3.10/site-packages/llama_index/core/llms/callbacks.py:173\u001b[0m, in \u001b[0;36mllm_chat_callback.<locals>.wrap.<locals>.wrapped_llm_chat\u001b[0;34m(_self, messages, **kwargs)\u001b[0m\n\u001b[1;32m    164\u001b[0m event_id \u001b[38;5;241m=\u001b[39m callback_manager\u001b[38;5;241m.\u001b[39mon_event_start(\n\u001b[1;32m    165\u001b[0m     CBEventType\u001b[38;5;241m.\u001b[39mLLM,\n\u001b[1;32m    166\u001b[0m     payload\u001b[38;5;241m=\u001b[39m{\n\u001b[0;32m   (...)\u001b[0m\n\u001b[1;32m    170\u001b[0m     },\n\u001b[1;32m    171\u001b[0m )\n\u001b[1;32m    172\u001b[0m \u001b[38;5;28;01mtry\u001b[39;00m:\n\u001b[0;32m--> 173\u001b[0m     f_return_val \u001b[38;5;241m=\u001b[39m \u001b[43mf\u001b[49m\u001b[43m(\u001b[49m\u001b[43m_self\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43mmessages\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[38;5;241;43m*\u001b[39;49m\u001b[38;5;241;43m*\u001b[39;49m\u001b[43mkwargs\u001b[49m\u001b[43m)\u001b[49m\n\u001b[1;32m    174\u001b[0m \u001b[38;5;28;01mexcept\u001b[39;00m \u001b[38;5;167;01mBaseException\u001b[39;00m \u001b[38;5;28;01mas\u001b[39;00m e:\n\u001b[1;32m    175\u001b[0m     callback_manager\u001b[38;5;241m.\u001b[39mon_event_end(\n\u001b[1;32m    176\u001b[0m         CBEventType\u001b[38;5;241m.\u001b[39mLLM,\n\u001b[1;32m    177\u001b[0m         payload\u001b[38;5;241m=\u001b[39m{EventPayload\u001b[38;5;241m.\u001b[39mEXCEPTION: e},\n\u001b[1;32m    178\u001b[0m         event_id\u001b[38;5;241m=\u001b[39mevent_id,\n\u001b[1;32m    179\u001b[0m     )\n",
      "File \u001b[0;32m~/.local/lib/python3.10/site-packages/llama_index/llms/ollama/base.py:131\u001b[0m, in \u001b[0;36mOllama.chat\u001b[0;34m(self, messages, **kwargs)\u001b[0m\n\u001b[1;32m    128\u001b[0m     payload[\u001b[38;5;124m\"\u001b[39m\u001b[38;5;124mformat\u001b[39m\u001b[38;5;124m\"\u001b[39m] \u001b[38;5;241m=\u001b[39m \u001b[38;5;124m\"\u001b[39m\u001b[38;5;124mjson\u001b[39m\u001b[38;5;124m\"\u001b[39m\n\u001b[1;32m    130\u001b[0m \u001b[38;5;28;01mwith\u001b[39;00m httpx\u001b[38;5;241m.\u001b[39mClient(timeout\u001b[38;5;241m=\u001b[39mTimeout(\u001b[38;5;28mself\u001b[39m\u001b[38;5;241m.\u001b[39mrequest_timeout)) \u001b[38;5;28;01mas\u001b[39;00m client:\n\u001b[0;32m--> 131\u001b[0m     response \u001b[38;5;241m=\u001b[39m \u001b[43mclient\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mpost\u001b[49m\u001b[43m(\u001b[49m\n\u001b[1;32m    132\u001b[0m \u001b[43m        \u001b[49m\u001b[43murl\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[38;5;124;43mf\u001b[39;49m\u001b[38;5;124;43m\"\u001b[39;49m\u001b[38;5;132;43;01m{\u001b[39;49;00m\u001b[38;5;28;43mself\u001b[39;49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mbase_url\u001b[49m\u001b[38;5;132;43;01m}\u001b[39;49;00m\u001b[38;5;124;43m/api/chat\u001b[39;49m\u001b[38;5;124;43m\"\u001b[39;49m\u001b[43m,\u001b[49m\n\u001b[1;32m    133\u001b[0m \u001b[43m        \u001b[49m\u001b[43mjson\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43mpayload\u001b[49m\u001b[43m,\u001b[49m\n\u001b[1;32m    134\u001b[0m \u001b[43m    \u001b[49m\u001b[43m)\u001b[49m\n\u001b[1;32m    135\u001b[0m     response\u001b[38;5;241m.\u001b[39mraise_for_status()\n\u001b[1;32m    136\u001b[0m     raw \u001b[38;5;241m=\u001b[39m response\u001b[38;5;241m.\u001b[39mjson()\n",
      "File \u001b[0;32m~/.local/lib/python3.10/site-packages/httpx/_client.py:1145\u001b[0m, in \u001b[0;36mClient.post\u001b[0;34m(self, url, content, data, files, json, params, headers, cookies, auth, follow_redirects, timeout, extensions)\u001b[0m\n\u001b[1;32m   1124\u001b[0m \u001b[38;5;28;01mdef\u001b[39;00m \u001b[38;5;21mpost\u001b[39m(\n\u001b[1;32m   1125\u001b[0m     \u001b[38;5;28mself\u001b[39m,\n\u001b[1;32m   1126\u001b[0m     url: URLTypes,\n\u001b[0;32m   (...)\u001b[0m\n\u001b[1;32m   1138\u001b[0m     extensions: RequestExtensions \u001b[38;5;241m|\u001b[39m \u001b[38;5;28;01mNone\u001b[39;00m \u001b[38;5;241m=\u001b[39m \u001b[38;5;28;01mNone\u001b[39;00m,\n\u001b[1;32m   1139\u001b[0m ) \u001b[38;5;241m-\u001b[39m\u001b[38;5;241m>\u001b[39m Response:\n\u001b[1;32m   1140\u001b[0m \u001b[38;5;250m    \u001b[39m\u001b[38;5;124;03m\"\"\"\u001b[39;00m\n\u001b[1;32m   1141\u001b[0m \u001b[38;5;124;03m    Send a `POST` request.\u001b[39;00m\n\u001b[1;32m   1142\u001b[0m \n\u001b[1;32m   1143\u001b[0m \u001b[38;5;124;03m    **Parameters**: See `httpx.request`.\u001b[39;00m\n\u001b[1;32m   1144\u001b[0m \u001b[38;5;124;03m    \"\"\"\u001b[39;00m\n\u001b[0;32m-> 1145\u001b[0m     \u001b[38;5;28;01mreturn\u001b[39;00m \u001b[38;5;28;43mself\u001b[39;49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mrequest\u001b[49m\u001b[43m(\u001b[49m\n\u001b[1;32m   1146\u001b[0m \u001b[43m        \u001b[49m\u001b[38;5;124;43m\"\u001b[39;49m\u001b[38;5;124;43mPOST\u001b[39;49m\u001b[38;5;124;43m\"\u001b[39;49m\u001b[43m,\u001b[49m\n\u001b[1;32m   1147\u001b[0m \u001b[43m        \u001b[49m\u001b[43murl\u001b[49m\u001b[43m,\u001b[49m\n\u001b[1;32m   1148\u001b[0m \u001b[43m        \u001b[49m\u001b[43mcontent\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43mcontent\u001b[49m\u001b[43m,\u001b[49m\n\u001b[1;32m   1149\u001b[0m \u001b[43m        \u001b[49m\u001b[43mdata\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43mdata\u001b[49m\u001b[43m,\u001b[49m\n\u001b[1;32m   1150\u001b[0m \u001b[43m        \u001b[49m\u001b[43mfiles\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43mfiles\u001b[49m\u001b[43m,\u001b[49m\n\u001b[1;32m   1151\u001b[0m \u001b[43m        \u001b[49m\u001b[43mjson\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43mjson\u001b[49m\u001b[43m,\u001b[49m\n\u001b[1;32m   1152\u001b[0m \u001b[43m        \u001b[49m\u001b[43mparams\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43mparams\u001b[49m\u001b[43m,\u001b[49m\n\u001b[1;32m   1153\u001b[0m \u001b[43m        \u001b[49m\u001b[43mheaders\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43mheaders\u001b[49m\u001b[43m,\u001b[49m\n\u001b[1;32m   1154\u001b[0m \u001b[43m        \u001b[49m\u001b[43mcookies\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43mcookies\u001b[49m\u001b[43m,\u001b[49m\n\u001b[1;32m   1155\u001b[0m \u001b[43m        \u001b[49m\u001b[43mauth\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43mauth\u001b[49m\u001b[43m,\u001b[49m\n\u001b[1;32m   1156\u001b[0m \u001b[43m        \u001b[49m\u001b[43mfollow_redirects\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43mfollow_redirects\u001b[49m\u001b[43m,\u001b[49m\n\u001b[1;32m   1157\u001b[0m \u001b[43m        \u001b[49m\u001b[43mtimeout\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43mtimeout\u001b[49m\u001b[43m,\u001b[49m\n\u001b[1;32m   1158\u001b[0m \u001b[43m        \u001b[49m\u001b[43mextensions\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43mextensions\u001b[49m\u001b[43m,\u001b[49m\n\u001b[1;32m   1159\u001b[0m \u001b[43m    \u001b[49m\u001b[43m)\u001b[49m\n",
      "File \u001b[0;32m~/.local/lib/python3.10/site-packages/httpx/_client.py:827\u001b[0m, in \u001b[0;36mClient.request\u001b[0;34m(self, method, url, content, data, files, json, params, headers, cookies, auth, follow_redirects, timeout, extensions)\u001b[0m\n\u001b[1;32m    812\u001b[0m     warnings\u001b[38;5;241m.\u001b[39mwarn(message, \u001b[38;5;167;01mDeprecationWarning\u001b[39;00m)\n\u001b[1;32m    814\u001b[0m request \u001b[38;5;241m=\u001b[39m \u001b[38;5;28mself\u001b[39m\u001b[38;5;241m.\u001b[39mbuild_request(\n\u001b[1;32m    815\u001b[0m     method\u001b[38;5;241m=\u001b[39mmethod,\n\u001b[1;32m    816\u001b[0m     url\u001b[38;5;241m=\u001b[39murl,\n\u001b[0;32m   (...)\u001b[0m\n\u001b[1;32m    825\u001b[0m     extensions\u001b[38;5;241m=\u001b[39mextensions,\n\u001b[1;32m    826\u001b[0m )\n\u001b[0;32m--> 827\u001b[0m \u001b[38;5;28;01mreturn\u001b[39;00m \u001b[38;5;28;43mself\u001b[39;49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43msend\u001b[49m\u001b[43m(\u001b[49m\u001b[43mrequest\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43mauth\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43mauth\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43mfollow_redirects\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43mfollow_redirects\u001b[49m\u001b[43m)\u001b[49m\n",
      "File \u001b[0;32m~/.local/lib/python3.10/site-packages/httpx/_client.py:914\u001b[0m, in \u001b[0;36mClient.send\u001b[0;34m(self, request, stream, auth, follow_redirects)\u001b[0m\n\u001b[1;32m    906\u001b[0m follow_redirects \u001b[38;5;241m=\u001b[39m (\n\u001b[1;32m    907\u001b[0m     \u001b[38;5;28mself\u001b[39m\u001b[38;5;241m.\u001b[39mfollow_redirects\n\u001b[1;32m    908\u001b[0m     \u001b[38;5;28;01mif\u001b[39;00m \u001b[38;5;28misinstance\u001b[39m(follow_redirects, UseClientDefault)\n\u001b[1;32m    909\u001b[0m     \u001b[38;5;28;01melse\u001b[39;00m follow_redirects\n\u001b[1;32m    910\u001b[0m )\n\u001b[1;32m    912\u001b[0m auth \u001b[38;5;241m=\u001b[39m \u001b[38;5;28mself\u001b[39m\u001b[38;5;241m.\u001b[39m_build_request_auth(request, auth)\n\u001b[0;32m--> 914\u001b[0m response \u001b[38;5;241m=\u001b[39m \u001b[38;5;28;43mself\u001b[39;49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43m_send_handling_auth\u001b[49m\u001b[43m(\u001b[49m\n\u001b[1;32m    915\u001b[0m \u001b[43m    \u001b[49m\u001b[43mrequest\u001b[49m\u001b[43m,\u001b[49m\n\u001b[1;32m    916\u001b[0m \u001b[43m    \u001b[49m\u001b[43mauth\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43mauth\u001b[49m\u001b[43m,\u001b[49m\n\u001b[1;32m    917\u001b[0m \u001b[43m    \u001b[49m\u001b[43mfollow_redirects\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43mfollow_redirects\u001b[49m\u001b[43m,\u001b[49m\n\u001b[1;32m    918\u001b[0m \u001b[43m    \u001b[49m\u001b[43mhistory\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43m[\u001b[49m\u001b[43m]\u001b[49m\u001b[43m,\u001b[49m\n\u001b[1;32m    919\u001b[0m \u001b[43m\u001b[49m\u001b[43m)\u001b[49m\n\u001b[1;32m    920\u001b[0m \u001b[38;5;28;01mtry\u001b[39;00m:\n\u001b[1;32m    921\u001b[0m     \u001b[38;5;28;01mif\u001b[39;00m \u001b[38;5;129;01mnot\u001b[39;00m stream:\n",
      "File \u001b[0;32m~/.local/lib/python3.10/site-packages/httpx/_client.py:942\u001b[0m, in \u001b[0;36mClient._send_handling_auth\u001b[0;34m(self, request, auth, follow_redirects, history)\u001b[0m\n\u001b[1;32m    939\u001b[0m request \u001b[38;5;241m=\u001b[39m \u001b[38;5;28mnext\u001b[39m(auth_flow)\n\u001b[1;32m    941\u001b[0m \u001b[38;5;28;01mwhile\u001b[39;00m \u001b[38;5;28;01mTrue\u001b[39;00m:\n\u001b[0;32m--> 942\u001b[0m     response \u001b[38;5;241m=\u001b[39m \u001b[38;5;28;43mself\u001b[39;49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43m_send_handling_redirects\u001b[49m\u001b[43m(\u001b[49m\n\u001b[1;32m    943\u001b[0m \u001b[43m        \u001b[49m\u001b[43mrequest\u001b[49m\u001b[43m,\u001b[49m\n\u001b[1;32m    944\u001b[0m \u001b[43m        \u001b[49m\u001b[43mfollow_redirects\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43mfollow_redirects\u001b[49m\u001b[43m,\u001b[49m\n\u001b[1;32m    945\u001b[0m \u001b[43m        \u001b[49m\u001b[43mhistory\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43mhistory\u001b[49m\u001b[43m,\u001b[49m\n\u001b[1;32m    946\u001b[0m \u001b[43m    \u001b[49m\u001b[43m)\u001b[49m\n\u001b[1;32m    947\u001b[0m     \u001b[38;5;28;01mtry\u001b[39;00m:\n\u001b[1;32m    948\u001b[0m         \u001b[38;5;28;01mtry\u001b[39;00m:\n",
      "File \u001b[0;32m~/.local/lib/python3.10/site-packages/httpx/_client.py:979\u001b[0m, in \u001b[0;36mClient._send_handling_redirects\u001b[0;34m(self, request, follow_redirects, history)\u001b[0m\n\u001b[1;32m    976\u001b[0m \u001b[38;5;28;01mfor\u001b[39;00m hook \u001b[38;5;129;01min\u001b[39;00m \u001b[38;5;28mself\u001b[39m\u001b[38;5;241m.\u001b[39m_event_hooks[\u001b[38;5;124m\"\u001b[39m\u001b[38;5;124mrequest\u001b[39m\u001b[38;5;124m\"\u001b[39m]:\n\u001b[1;32m    977\u001b[0m     hook(request)\n\u001b[0;32m--> 979\u001b[0m response \u001b[38;5;241m=\u001b[39m \u001b[38;5;28;43mself\u001b[39;49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43m_send_single_request\u001b[49m\u001b[43m(\u001b[49m\u001b[43mrequest\u001b[49m\u001b[43m)\u001b[49m\n\u001b[1;32m    980\u001b[0m \u001b[38;5;28;01mtry\u001b[39;00m:\n\u001b[1;32m    981\u001b[0m     \u001b[38;5;28;01mfor\u001b[39;00m hook \u001b[38;5;129;01min\u001b[39;00m \u001b[38;5;28mself\u001b[39m\u001b[38;5;241m.\u001b[39m_event_hooks[\u001b[38;5;124m\"\u001b[39m\u001b[38;5;124mresponse\u001b[39m\u001b[38;5;124m\"\u001b[39m]:\n",
      "File \u001b[0;32m~/.local/lib/python3.10/site-packages/httpx/_client.py:1015\u001b[0m, in \u001b[0;36mClient._send_single_request\u001b[0;34m(self, request)\u001b[0m\n\u001b[1;32m   1010\u001b[0m     \u001b[38;5;28;01mraise\u001b[39;00m \u001b[38;5;167;01mRuntimeError\u001b[39;00m(\n\u001b[1;32m   1011\u001b[0m         \u001b[38;5;124m\"\u001b[39m\u001b[38;5;124mAttempted to send an async request with a sync Client instance.\u001b[39m\u001b[38;5;124m\"\u001b[39m\n\u001b[1;32m   1012\u001b[0m     )\n\u001b[1;32m   1014\u001b[0m \u001b[38;5;28;01mwith\u001b[39;00m request_context(request\u001b[38;5;241m=\u001b[39mrequest):\n\u001b[0;32m-> 1015\u001b[0m     response \u001b[38;5;241m=\u001b[39m \u001b[43mtransport\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mhandle_request\u001b[49m\u001b[43m(\u001b[49m\u001b[43mrequest\u001b[49m\u001b[43m)\u001b[49m\n\u001b[1;32m   1017\u001b[0m \u001b[38;5;28;01massert\u001b[39;00m \u001b[38;5;28misinstance\u001b[39m(response\u001b[38;5;241m.\u001b[39mstream, SyncByteStream)\n\u001b[1;32m   1019\u001b[0m response\u001b[38;5;241m.\u001b[39mrequest \u001b[38;5;241m=\u001b[39m request\n",
      "File \u001b[0;32m~/.local/lib/python3.10/site-packages/httpx/_transports/default.py:232\u001b[0m, in \u001b[0;36mHTTPTransport.handle_request\u001b[0;34m(self, request)\u001b[0m\n\u001b[1;32m    218\u001b[0m \u001b[38;5;28;01massert\u001b[39;00m \u001b[38;5;28misinstance\u001b[39m(request\u001b[38;5;241m.\u001b[39mstream, SyncByteStream)\n\u001b[1;32m    220\u001b[0m req \u001b[38;5;241m=\u001b[39m httpcore\u001b[38;5;241m.\u001b[39mRequest(\n\u001b[1;32m    221\u001b[0m     method\u001b[38;5;241m=\u001b[39mrequest\u001b[38;5;241m.\u001b[39mmethod,\n\u001b[1;32m    222\u001b[0m     url\u001b[38;5;241m=\u001b[39mhttpcore\u001b[38;5;241m.\u001b[39mURL(\n\u001b[0;32m   (...)\u001b[0m\n\u001b[1;32m    230\u001b[0m     extensions\u001b[38;5;241m=\u001b[39mrequest\u001b[38;5;241m.\u001b[39mextensions,\n\u001b[1;32m    231\u001b[0m )\n\u001b[0;32m--> 232\u001b[0m \u001b[38;5;28;01mwith\u001b[39;00m map_httpcore_exceptions():\n\u001b[1;32m    233\u001b[0m     resp \u001b[38;5;241m=\u001b[39m \u001b[38;5;28mself\u001b[39m\u001b[38;5;241m.\u001b[39m_pool\u001b[38;5;241m.\u001b[39mhandle_request(req)\n\u001b[1;32m    235\u001b[0m \u001b[38;5;28;01massert\u001b[39;00m \u001b[38;5;28misinstance\u001b[39m(resp\u001b[38;5;241m.\u001b[39mstream, typing\u001b[38;5;241m.\u001b[39mIterable)\n",
      "File \u001b[0;32m/usr/lib/python3.10/contextlib.py:153\u001b[0m, in \u001b[0;36m_GeneratorContextManager.__exit__\u001b[0;34m(self, typ, value, traceback)\u001b[0m\n\u001b[1;32m    151\u001b[0m     value \u001b[38;5;241m=\u001b[39m typ()\n\u001b[1;32m    152\u001b[0m \u001b[38;5;28;01mtry\u001b[39;00m:\n\u001b[0;32m--> 153\u001b[0m     \u001b[38;5;28;43mself\u001b[39;49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mgen\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mthrow\u001b[49m\u001b[43m(\u001b[49m\u001b[43mtyp\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43mvalue\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43mtraceback\u001b[49m\u001b[43m)\u001b[49m\n\u001b[1;32m    154\u001b[0m \u001b[38;5;28;01mexcept\u001b[39;00m \u001b[38;5;167;01mStopIteration\u001b[39;00m \u001b[38;5;28;01mas\u001b[39;00m exc:\n\u001b[1;32m    155\u001b[0m     \u001b[38;5;66;03m# Suppress StopIteration *unless* it's the same exception that\u001b[39;00m\n\u001b[1;32m    156\u001b[0m     \u001b[38;5;66;03m# was passed to throw().  This prevents a StopIteration\u001b[39;00m\n\u001b[1;32m    157\u001b[0m     \u001b[38;5;66;03m# raised inside the \"with\" statement from being suppressed.\u001b[39;00m\n\u001b[1;32m    158\u001b[0m     \u001b[38;5;28;01mreturn\u001b[39;00m exc \u001b[38;5;129;01mis\u001b[39;00m \u001b[38;5;129;01mnot\u001b[39;00m value\n",
      "File \u001b[0;32m~/.local/lib/python3.10/site-packages/httpx/_transports/default.py:86\u001b[0m, in \u001b[0;36mmap_httpcore_exceptions\u001b[0;34m()\u001b[0m\n\u001b[1;32m     83\u001b[0m     \u001b[38;5;28;01mraise\u001b[39;00m\n\u001b[1;32m     85\u001b[0m message \u001b[38;5;241m=\u001b[39m \u001b[38;5;28mstr\u001b[39m(exc)\n\u001b[0;32m---> 86\u001b[0m \u001b[38;5;28;01mraise\u001b[39;00m mapped_exc(message) \u001b[38;5;28;01mfrom\u001b[39;00m \u001b[38;5;21;01mexc\u001b[39;00m\n",
      "\u001b[0;31mReadTimeout\u001b[0m: timed out"
     ]
    }
   ],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": 155,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Failed to parse JSON: [\"We used the Senescence beta-Galactosidase Staining kit\", \"The kit was purchased from Cell Signaling and had product number 9860S.\", \"We detected beta-galactosidase activity at pH 6.0 using the kit.\", \"Cells were fixed in a fixative solution for 10-15 min at room temperature.\", \"The fixed cells were then incubated with a beta-galactosidase staining solution containing X-gal at 37 C overnight.\", \"As the blue color developed, bright-field cell images were taken using an Axiovert 25 microscope connected to a Canon camera.\", \"We used a 0.4% Trypan blue stock solution from Life Technologies with product number 15250061 to stain dead cells.\", \"The stock solution was added to a cell suspension in the ratio of 10 microl of Trypan blue stock solution per 100 microl of cell suspension.\", \"Cells negative for Trypan blue staining were counted as live cells using hemocytometer.\", \"We used two methods to confirm the interaction between BuGZ or Bub3 and spliceosome components.\", \"In the first method, we transfected HEK293T cells with Flag-BuGZ, Flag-Bub3, or Flag-luciferase.\", \"Flag-luciferase was used as a negative control in the transfection experiment.\", \"The transfected cells were lysed in 1 ml lysis buffer containing protease inhibitors at 4 C.\", \"The lysis buffer contained 20 mM Tris-Cl, pH 7.4, 150 mM KCl, 1 mM EDTA, 1 mM NaF, 1 mM DTT, 1% NP-40, and 10% glycerol.\", \"The lysis buffer also contained 1 mM sodium pyrophosphate.\", \"After centrifugation at 13,000 rpm in an Eppendorf tabletop centrifuge, the supernatant containing soluble proteins was incubated with a 30-microl slurry of anti-Flag M2 resin from Sigma-Aldrich with product number A2220.\", \"The beads were washed with wash buffer three times before eluting the bound proteins.\", \"The bound proteins were eluted with 100 microl Flag-peptide dissolved in the wash buffer at a concentration of 0.5 mg/ml.\"]\n",
      "\n",
      "Here is the output in JSON format:\n",
      "\n",
      "[\n",
      "  \"We used the Senescence beta-Galactosidase Staining kit\",\n",
      "  \"The kit was purchased from Cell Signaling and had product number 9860S.\",\n",
      "  \"We detected beta-galactosidase activity at pH 6.0 using the kit.\",\n",
      "  \"Cells were fixed in a fixative solution for 10-15 min at room temperature.\",\n",
      "  \"The fixed cells were then incubated with a beta-galactosidase staining solution containing X-gal at 37 C overnight.\",\n",
      "  \"As the blue color developed, bright-field cell images were taken using an Axiovert 25 microscope connected to a Canon camera.\",\n",
      "  \"We used a 0.4% Trypan blue stock solution from Life Technologies with product number 15250061 to stain dead cells.\",\n",
      "  \"The stock solution was added to a cell suspension in the ratio of 10 microl of Trypan blue stock solution per 100 microl of cell suspension.\",\n",
      "  \"Cells negative for Trypan blue staining were counted as live cells using hemocytometer.\",\n",
      "  \"We used two methods to confirm the interaction between BuGZ or Bub3 and spliceosome components.\",\n",
      "  \"In the first method, we transfected HEK293T cells with Flag-BuGZ, Flag-Bub3, or Flag-luciferase.\",\n",
      "  \"Flag-luciferase was used as a negative control in the transfection experiment.\",\n",
      "  \"The transfected cells were lysed in 1 ml lysis buffer containing protease inhibitors at 4 C.\",\n",
      "  \"The lysis buffer contained 20 mM Tris-Cl, pH 7.4, 150 mM KCl, 1 mM EDTA, 1 mM NaF, 1 mM DTT, 1% NP-40, and 10% glycerol.\",\n",
      "  \"The lysis buffer also contained 1 mM sodium pyrophosphate.\",\n",
      "  \"After centrifugation at 13,000 rpm in an Eppendorf tabletop centrifuge, the supernatant containing soluble proteins was incubated with a 30-microl slurry of anti-Flag M2 resin from Sigma-Aldrich with product number A2220.\",\n",
      "  \"The beads were washed with wash buffer three times before eluting the bound proteins.\",\n",
      "  \"The bound proteins were eluted with 100 microl Flag-peptide dissolved in the wash buffer at a concentration of 0.5 mg/ml.\"\n",
      "]\n"
     ]
    },
    {
     "ename": "TypeError",
     "evalue": "sequence item 0: expected str instance, dict found",
     "output_type": "error",
     "traceback": [
      "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
      "\u001b[0;31mTypeError\u001b[0m                                 Traceback (most recent call last)",
      "Cell \u001b[0;32mIn[155], line 1\u001b[0m\n\u001b[0;32m----> 1\u001b[0m parsed \u001b[38;5;241m=\u001b[39m \u001b[43mtopic_parse\u001b[49m\u001b[43m(\u001b[49m\u001b[43mnodes\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43mchunk_size\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[38;5;241;43m2000\u001b[39;49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43msimilarity_threshold\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[38;5;241;43m0.8\u001b[39;49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43mwindow_size\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[38;5;241;43m5\u001b[39;49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43msimilarity_method\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[38;5;124;43m\"\u001b[39;49m\u001b[38;5;124;43membedding\u001b[39;49m\u001b[38;5;124;43m\"\u001b[39;49m\u001b[43m)\u001b[49m\n\u001b[1;32m      2\u001b[0m print_topic_nodes(parsed)\n",
      "Cell \u001b[0;32mIn[152], line 17\u001b[0m, in \u001b[0;36mtopic_parse\u001b[0;34m(documents, chunk_size, similarity_threshold, window_size, similarity_method)\u001b[0m\n\u001b[1;32m      6\u001b[0m node_parser \u001b[38;5;241m=\u001b[39m TopicNodeParser\u001b[38;5;241m.\u001b[39mfrom_defaults(\n\u001b[1;32m      7\u001b[0m     embed_model\u001b[38;5;241m=\u001b[39membed_model,\n\u001b[1;32m      8\u001b[0m     llm\u001b[38;5;241m=\u001b[39mllm,\n\u001b[0;32m   (...)\u001b[0m\n\u001b[1;32m     12\u001b[0m     window_size\u001b[38;5;241m=\u001b[39mwindow_size,  \u001b[38;5;66;03m# paper suggests window_size=5\u001b[39;00m\n\u001b[1;32m     13\u001b[0m )\n\u001b[1;32m     14\u001b[0m \u001b[38;5;66;03m# return {\"nodes\": node_parser.get_nodes_from_documents(documents, show_progress=True), \u001b[39;00m\n\u001b[1;32m     15\u001b[0m \u001b[38;5;66;03m#         \"params\": {\"chunk_size\": chunk_size, \"threshold\": similarity_threshold, \"window\": window_size}}\u001b[39;00m\n\u001b[0;32m---> 17\u001b[0m \u001b[38;5;28;01mreturn\u001b[39;00m {\u001b[38;5;124m\"\u001b[39m\u001b[38;5;124mnodes\u001b[39m\u001b[38;5;124m\"\u001b[39m: \u001b[43mnode_parser\u001b[49m\u001b[43m(\u001b[49m\u001b[43mdocuments\u001b[49m\u001b[43m)\u001b[49m, \n\u001b[1;32m     18\u001b[0m         \u001b[38;5;124m\"\u001b[39m\u001b[38;5;124mparams\u001b[39m\u001b[38;5;124m\"\u001b[39m: {\u001b[38;5;124m\"\u001b[39m\u001b[38;5;124mchunk_size\u001b[39m\u001b[38;5;124m\"\u001b[39m: chunk_size, \u001b[38;5;124m\"\u001b[39m\u001b[38;5;124mthreshold\u001b[39m\u001b[38;5;124m\"\u001b[39m: similarity_threshold, \u001b[38;5;124m\"\u001b[39m\u001b[38;5;124mwindow\u001b[39m\u001b[38;5;124m\"\u001b[39m: window_size}}\n",
      "File \u001b[0;32m~/.local/lib/python3.10/site-packages/llama_index/core/instrumentation/dispatcher.py:311\u001b[0m, in \u001b[0;36mDispatcher.span.<locals>.wrapper\u001b[0;34m(func, instance, args, kwargs)\u001b[0m\n\u001b[1;32m    308\u001b[0m             _logger\u001b[38;5;241m.\u001b[39mdebug(\u001b[38;5;124mf\u001b[39m\u001b[38;5;124m\"\u001b[39m\u001b[38;5;124mFailed to reset active_span_id: \u001b[39m\u001b[38;5;132;01m{\u001b[39;00me\u001b[38;5;132;01m}\u001b[39;00m\u001b[38;5;124m\"\u001b[39m)\n\u001b[1;32m    310\u001b[0m \u001b[38;5;28;01mtry\u001b[39;00m:\n\u001b[0;32m--> 311\u001b[0m     result \u001b[38;5;241m=\u001b[39m \u001b[43mfunc\u001b[49m\u001b[43m(\u001b[49m\u001b[38;5;241;43m*\u001b[39;49m\u001b[43margs\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[38;5;241;43m*\u001b[39;49m\u001b[38;5;241;43m*\u001b[39;49m\u001b[43mkwargs\u001b[49m\u001b[43m)\u001b[49m\n\u001b[1;32m    312\u001b[0m     \u001b[38;5;28;01mif\u001b[39;00m \u001b[38;5;28misinstance\u001b[39m(result, asyncio\u001b[38;5;241m.\u001b[39mFuture):\n\u001b[1;32m    313\u001b[0m         \u001b[38;5;66;03m# If the result is a Future, wrap it\u001b[39;00m\n\u001b[1;32m    314\u001b[0m         new_future \u001b[38;5;241m=\u001b[39m asyncio\u001b[38;5;241m.\u001b[39mensure_future(result)\n",
      "File \u001b[0;32m~/.local/lib/python3.10/site-packages/llama_index/core/node_parser/interface.py:193\u001b[0m, in \u001b[0;36mNodeParser.__call__\u001b[0;34m(self, nodes, **kwargs)\u001b[0m\n\u001b[1;32m    192\u001b[0m \u001b[38;5;28;01mdef\u001b[39;00m \u001b[38;5;21m__call__\u001b[39m(\u001b[38;5;28mself\u001b[39m, nodes: Sequence[BaseNode], \u001b[38;5;241m*\u001b[39m\u001b[38;5;241m*\u001b[39mkwargs: Any) \u001b[38;5;241m-\u001b[39m\u001b[38;5;241m>\u001b[39m List[BaseNode]:\n\u001b[0;32m--> 193\u001b[0m     \u001b[38;5;28;01mreturn\u001b[39;00m \u001b[38;5;28;43mself\u001b[39;49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mget_nodes_from_documents\u001b[49m\u001b[43m(\u001b[49m\u001b[43mnodes\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[38;5;241;43m*\u001b[39;49m\u001b[38;5;241;43m*\u001b[39;49m\u001b[43mkwargs\u001b[49m\u001b[43m)\u001b[49m\n",
      "File \u001b[0;32m~/.local/lib/python3.10/site-packages/llama_index/core/node_parser/interface.py:165\u001b[0m, in \u001b[0;36mNodeParser.get_nodes_from_documents\u001b[0;34m(self, documents, show_progress, **kwargs)\u001b[0m\n\u001b[1;32m    160\u001b[0m doc_id_to_document \u001b[38;5;241m=\u001b[39m {doc\u001b[38;5;241m.\u001b[39mid_: doc \u001b[38;5;28;01mfor\u001b[39;00m doc \u001b[38;5;129;01min\u001b[39;00m documents}\n\u001b[1;32m    162\u001b[0m \u001b[38;5;28;01mwith\u001b[39;00m \u001b[38;5;28mself\u001b[39m\u001b[38;5;241m.\u001b[39mcallback_manager\u001b[38;5;241m.\u001b[39mevent(\n\u001b[1;32m    163\u001b[0m     CBEventType\u001b[38;5;241m.\u001b[39mNODE_PARSING, payload\u001b[38;5;241m=\u001b[39m{EventPayload\u001b[38;5;241m.\u001b[39mDOCUMENTS: documents}\n\u001b[1;32m    164\u001b[0m ) \u001b[38;5;28;01mas\u001b[39;00m event:\n\u001b[0;32m--> 165\u001b[0m     nodes \u001b[38;5;241m=\u001b[39m \u001b[38;5;28;43mself\u001b[39;49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43m_parse_nodes\u001b[49m\u001b[43m(\u001b[49m\u001b[43mdocuments\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43mshow_progress\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43mshow_progress\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[38;5;241;43m*\u001b[39;49m\u001b[38;5;241;43m*\u001b[39;49m\u001b[43mkwargs\u001b[49m\u001b[43m)\u001b[49m\n\u001b[1;32m    166\u001b[0m     nodes \u001b[38;5;241m=\u001b[39m \u001b[38;5;28mself\u001b[39m\u001b[38;5;241m.\u001b[39m_postprocess_parsed_nodes(nodes, doc_id_to_document)\n\u001b[1;32m    168\u001b[0m     event\u001b[38;5;241m.\u001b[39mon_end({EventPayload\u001b[38;5;241m.\u001b[39mNODES: nodes})\n",
      "File \u001b[0;32m~/.local/lib/python3.10/site-packages/llama_index/core/instrumentation/dispatcher.py:311\u001b[0m, in \u001b[0;36mDispatcher.span.<locals>.wrapper\u001b[0;34m(func, instance, args, kwargs)\u001b[0m\n\u001b[1;32m    308\u001b[0m             _logger\u001b[38;5;241m.\u001b[39mdebug(\u001b[38;5;124mf\u001b[39m\u001b[38;5;124m\"\u001b[39m\u001b[38;5;124mFailed to reset active_span_id: \u001b[39m\u001b[38;5;132;01m{\u001b[39;00me\u001b[38;5;132;01m}\u001b[39;00m\u001b[38;5;124m\"\u001b[39m)\n\u001b[1;32m    310\u001b[0m \u001b[38;5;28;01mtry\u001b[39;00m:\n\u001b[0;32m--> 311\u001b[0m     result \u001b[38;5;241m=\u001b[39m \u001b[43mfunc\u001b[49m\u001b[43m(\u001b[49m\u001b[38;5;241;43m*\u001b[39;49m\u001b[43margs\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[38;5;241;43m*\u001b[39;49m\u001b[38;5;241;43m*\u001b[39;49m\u001b[43mkwargs\u001b[49m\u001b[43m)\u001b[49m\n\u001b[1;32m    312\u001b[0m     \u001b[38;5;28;01mif\u001b[39;00m \u001b[38;5;28misinstance\u001b[39m(result, asyncio\u001b[38;5;241m.\u001b[39mFuture):\n\u001b[1;32m    313\u001b[0m         \u001b[38;5;66;03m# If the result is a Future, wrap it\u001b[39;00m\n\u001b[1;32m    314\u001b[0m         new_future \u001b[38;5;241m=\u001b[39m asyncio\u001b[38;5;241m.\u001b[39mensure_future(result)\n",
      "File \u001b[0;32m~/.local/lib/python3.10/site-packages/llama_index/node_parser/topic/base.py:120\u001b[0m, in \u001b[0;36mTopicNodeParser._parse_nodes\u001b[0;34m(self, nodes, show_progress, **kwargs)\u001b[0m\n\u001b[1;32m    117\u001b[0m nodes_with_progress \u001b[38;5;241m=\u001b[39m get_tqdm_iterable(nodes, show_progress, \u001b[38;5;124m\"\u001b[39m\u001b[38;5;124mParsing nodes\u001b[39m\u001b[38;5;124m\"\u001b[39m)\n\u001b[1;32m    119\u001b[0m \u001b[38;5;28;01mfor\u001b[39;00m node \u001b[38;5;129;01min\u001b[39;00m nodes_with_progress:\n\u001b[0;32m--> 120\u001b[0m     nodes \u001b[38;5;241m=\u001b[39m \u001b[38;5;28;43mself\u001b[39;49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mbuild_topic_based_nodes_from_documents\u001b[49m\u001b[43m(\u001b[49m\u001b[43m[\u001b[49m\u001b[43mnode\u001b[49m\u001b[43m]\u001b[49m\u001b[43m)\u001b[49m\n\u001b[1;32m    121\u001b[0m     all_nodes\u001b[38;5;241m.\u001b[39mextend(nodes)\n\u001b[1;32m    123\u001b[0m \u001b[38;5;28;01mreturn\u001b[39;00m all_nodes\n",
      "File \u001b[0;32m~/.local/lib/python3.10/site-packages/llama_index/node_parser/topic/base.py:255\u001b[0m, in \u001b[0;36mTopicNodeParser.build_topic_based_nodes_from_documents\u001b[0;34m(self, documents)\u001b[0m\n\u001b[1;32m    253\u001b[0m \u001b[38;5;28;01mfor\u001b[39;00m doc \u001b[38;5;129;01min\u001b[39;00m documents:\n\u001b[1;32m    254\u001b[0m     paragraphs \u001b[38;5;241m=\u001b[39m \u001b[38;5;28mself\u001b[39m\u001b[38;5;241m.\u001b[39msplit_into_paragraphs(doc\u001b[38;5;241m.\u001b[39mtext)\n\u001b[0;32m--> 255\u001b[0m     chunks \u001b[38;5;241m=\u001b[39m \u001b[38;5;28;43mself\u001b[39;49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43msemantic_chunking\u001b[49m\u001b[43m(\u001b[49m\u001b[43mparagraphs\u001b[49m\u001b[43m)\u001b[49m\n\u001b[1;32m    256\u001b[0m     nodes \u001b[38;5;241m=\u001b[39m build_nodes_from_splits(\n\u001b[1;32m    257\u001b[0m         chunks,\n\u001b[1;32m    258\u001b[0m         doc,\n\u001b[1;32m    259\u001b[0m         id_func\u001b[38;5;241m=\u001b[39m\u001b[38;5;28mself\u001b[39m\u001b[38;5;241m.\u001b[39mid_func,\n\u001b[1;32m    260\u001b[0m     )\n\u001b[1;32m    261\u001b[0m     all_nodes\u001b[38;5;241m.\u001b[39mextend(nodes)\n",
      "File \u001b[0;32m~/.local/lib/python3.10/site-packages/llama_index/node_parser/topic/base.py:215\u001b[0m, in \u001b[0;36mTopicNodeParser.semantic_chunking\u001b[0;34m(self, paragraphs)\u001b[0m\n\u001b[1;32m    213\u001b[0m     is_same_topic \u001b[38;5;241m=\u001b[39m \u001b[38;5;28mself\u001b[39m\u001b[38;5;241m.\u001b[39mis_same_topic_llm(current_chunk, prop)\n\u001b[1;32m    214\u001b[0m \u001b[38;5;28;01melif\u001b[39;00m \u001b[38;5;28mself\u001b[39m\u001b[38;5;241m.\u001b[39msimilarity_method \u001b[38;5;241m==\u001b[39m \u001b[38;5;124m\"\u001b[39m\u001b[38;5;124membedding\u001b[39m\u001b[38;5;124m\"\u001b[39m:\n\u001b[0;32m--> 215\u001b[0m     is_same_topic \u001b[38;5;241m=\u001b[39m \u001b[38;5;28;43mself\u001b[39;49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mis_same_topic_embedding\u001b[49m\u001b[43m(\u001b[49m\n\u001b[1;32m    216\u001b[0m \u001b[43m        \u001b[49m\u001b[43mcurrent_chunk\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43mprop\u001b[49m\n\u001b[1;32m    217\u001b[0m \u001b[43m    \u001b[49m\u001b[43m)\u001b[49m\n\u001b[1;32m    218\u001b[0m \u001b[38;5;28;01melse\u001b[39;00m:\n\u001b[1;32m    219\u001b[0m     \u001b[38;5;28;01mraise\u001b[39;00m \u001b[38;5;167;01mValueError\u001b[39;00m(\n\u001b[1;32m    220\u001b[0m         \u001b[38;5;124m\"\u001b[39m\u001b[38;5;124mInvalid similarity method. Choose \u001b[39m\u001b[38;5;124m'\u001b[39m\u001b[38;5;124mllm\u001b[39m\u001b[38;5;124m'\u001b[39m\u001b[38;5;124m or \u001b[39m\u001b[38;5;124m'\u001b[39m\u001b[38;5;124membedding\u001b[39m\u001b[38;5;124m'\u001b[39m\u001b[38;5;124m.\u001b[39m\u001b[38;5;124m\"\u001b[39m\n\u001b[1;32m    221\u001b[0m     )\n",
      "File \u001b[0;32m~/.local/lib/python3.10/site-packages/llama_index/node_parser/topic/base.py:177\u001b[0m, in \u001b[0;36mTopicNodeParser.is_same_topic_embedding\u001b[0;34m(self, current_chunk, new_proposition)\u001b[0m\n\u001b[1;32m    171\u001b[0m \u001b[38;5;28;01mdef\u001b[39;00m \u001b[38;5;21mis_same_topic_embedding\u001b[39m(\n\u001b[1;32m    172\u001b[0m     \u001b[38;5;28mself\u001b[39m, current_chunk: List[\u001b[38;5;28mstr\u001b[39m], new_proposition: \u001b[38;5;28mstr\u001b[39m\n\u001b[1;32m    173\u001b[0m ) \u001b[38;5;241m-\u001b[39m\u001b[38;5;241m>\u001b[39m \u001b[38;5;28mbool\u001b[39m:\n\u001b[1;32m    174\u001b[0m \u001b[38;5;250m    \u001b[39m\u001b[38;5;124;03m\"\"\"\u001b[39;00m\n\u001b[1;32m    175\u001b[0m \u001b[38;5;124;03m    Use embedding-based similarity to determine if the new proposition belongs to the same topic.\u001b[39;00m\n\u001b[1;32m    176\u001b[0m \u001b[38;5;124;03m    \"\"\"\u001b[39;00m\n\u001b[0;32m--> 177\u001b[0m     current_text \u001b[38;5;241m=\u001b[39m \u001b[38;5;124;43m\"\u001b[39;49m\u001b[38;5;124;43m \u001b[39;49m\u001b[38;5;124;43m\"\u001b[39;49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mjoin\u001b[49m\u001b[43m(\u001b[49m\u001b[43mcurrent_chunk\u001b[49m\u001b[43m)\u001b[49m\n\u001b[1;32m    178\u001b[0m     current_text_embedding \u001b[38;5;241m=\u001b[39m \u001b[38;5;28mself\u001b[39m\u001b[38;5;241m.\u001b[39membed_model\u001b[38;5;241m.\u001b[39mget_text_embedding(current_text)\n\u001b[1;32m    179\u001b[0m     new_proposition_embedding \u001b[38;5;241m=\u001b[39m \u001b[38;5;28mself\u001b[39m\u001b[38;5;241m.\u001b[39membed_model\u001b[38;5;241m.\u001b[39mget_text_embedding(new_proposition)\n",
      "\u001b[0;31mTypeError\u001b[0m: sequence item 0: expected str instance, dict found"
     ]
    }
   ],
   "source": [
    "parsed = topic_parse(nodes, chunk_size=2000, similarity_threshold=0.8, window_size=5, similarity_method=\"embedding\")\n",
    "print_topic_nodes(parsed)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": 126,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "In this paper, we introduce a novel graph RAG method for applying LLMs to the medical domain, which we refer to as Medical Graph RAG (MedRAG). This technique improves LLM performance in the medical domain by response queries with grounded source citations and clear interpretations of medical terminology, boosting the transparency and interpretability of the results. This approach involves a three-tier hierarchical graph construction method. Initially, we use documents provided by users as our top-level source to extract entities. These entities are then linked to a second level consisting of more basic entities previously abstracted from credible medical books and papers. Subsequently, these entities are connected to a third level—the fundamental medical dictionary graph—that provides detailed explanations of each medical term and their semantic relationships. We then construct a comprehensive graph at the highest level by linking entities based on their content and hierarchical connections. This method ensures that the knowledge can be traced back to its sources and the results are factually accurate.\n",
      "\n",
      "To respond to user queries, we implement a U-retrieve strategy that combines top-down retrieval with bottom-up response generation. The process begins by structuring the query using predefined medical tags and indexing them through the graphs in a top-down manner. The system then generates responses based on these queries, pulling from meta-graphs—nodes retrieved along with their TopK related nodes and relationships—and summarizing the information into a detailed response. This technique maintains a balance between global context awareness and the contextual limitations inherent in LLMs.\n",
      "\n",
      "Our medical graph RAG provides Intrinsic source citation can enhance LLM transparency, interpretability, and verifiability. The results provides the provenance, or source grounding information, as it generates each response, and demonstrates that an answer is grounded in the dataset. Having the cited source for each assertion readily available also enables a human user to quickly and accurately audit the LLM’s output directly against the original source material. It is super useful in the field of medicine that security is very important, and each of the reasoning should be evidence-based. By using such a method, we construct an evidence-based Medical LLM that the clinician could easiely check the source of the reasoning and calibrate the model response to ensure the safty usage of llm in the clinical senarios.\n",
      "\n",
      "To evaluate our medical graph RAG, we implemented the method on several popular open and closed-source LLMs, including ChatGPT OpenAI (2023a) and LLaMA Touvron et al. (2023), testing them across mainstream medical Q&A benchmarks such as PubMedQA Jin et al. (2019), MedMCQA Pal et al. (2022), and USMLE Kung et al. (2023). For the RAG process, we supplied a comprehensive medical dictionary as the foundational knowledge layer, the UMLS medical knowledge graph Lindberg et al. (1993) as the foundamental layer detailing semantic relationships, and a curated MedC-K dataset Wu et al. (2023) —comprising the latest medical papers and books—as the intermediate level of data to simulate user-provided private data. Our experiments demonstrate that our model significantly enhances the performance of general-purpose LLMs on medical questions. Remarkably, it even surpasses many fine-tuned or specially trained LLMs on medical corpora, solely using the RAG approach without additional training.\n",
      "\n"
     ]
    }
   ],
   "source": [
    "\n",
    "text = \"\"\"In this paper, we introduce a novel graph RAG method for applying LLMs to the medical domain, which we refer to as Medical Graph RAG (MedRAG). This technique improves LLM performance in the medical domain by response queries with grounded source citations and clear interpretations of medical terminology, boosting the transparency and interpretability of the results. This approach involves a three-tier hierarchical graph construction method. Initially, we use documents provided by users as our top-level source to extract entities. These entities are then linked to a second level consisting of more basic entities previously abstracted from credible medical books and papers. Subsequently, these entities are connected to a third level—the fundamental medical dictionary graph—that provides detailed explanations of each medical term and their semantic relationships. We then construct a comprehensive graph at the highest level by linking entities based on their content and hierarchical connections. This method ensures that the knowledge can be traced back to its sources and the results are factually accurate.\n",
    "\n",
    "To respond to user queries, we implement a U-retrieve strategy that combines top-down retrieval with bottom-up response generation. The process begins by structuring the query using predefined medical tags and indexing them through the graphs in a top-down manner. The system then generates responses based on these queries, pulling from meta-graphs—nodes retrieved along with their TopK related nodes and relationships—and summarizing the information into a detailed response. This technique maintains a balance between global context awareness and the contextual limitations inherent in LLMs.\n",
    "\n",
    "Our medical graph RAG provides Intrinsic source citation can enhance LLM transparency, interpretability, and verifiability. The results provides the provenance, or source grounding information, as it generates each response, and demonstrates that an answer is grounded in the dataset. Having the cited source for each assertion readily available also enables a human user to quickly and accurately audit the LLM’s output directly against the original source material. It is super useful in the field of medicine that security is very important, and each of the reasoning should be evidence-based. By using such a method, we construct an evidence-based Medical LLM that the clinician could easiely check the source of the reasoning and calibrate the model response to ensure the safty usage of llm in the clinical senarios.\n",
    "\n",
    "To evaluate our medical graph RAG, we implemented the method on several popular open and closed-source LLMs, including ChatGPT OpenAI (2023a) and LLaMA Touvron et al. (2023), testing them across mainstream medical Q&A benchmarks such as PubMedQA Jin et al. (2019), MedMCQA Pal et al. (2022), and USMLE Kung et al. (2023). For the RAG process, we supplied a comprehensive medical dictionary as the foundational knowledge layer, the UMLS medical knowledge graph Lindberg et al. (1993) as the foundamental layer detailing semantic relationships, and a curated MedC-K dataset Wu et al. (2023) —comprising the latest medical papers and books—as the intermediate level of data to simulate user-provided private data. Our experiments demonstrate that our model significantly enhances the performance of general-purpose LLMs on medical questions. Remarkably, it even surpasses many fine-tuned or specially trained LLMs on medical corpora, solely using the RAG approach without additional training.\n",
    "\"\"\"\n",
    "\n",
    "documents = [Document(text=text)]\n",
    "print(documents[0].get_content())\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 30,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "application/vnd.jupyter.widget-view+json": {
       "model_id": "9710578c87114f128845feb0cb63f798",
       "version_major": 2,
       "version_minor": 0
      },
      "text/plain": [
       "Parsing nodes:   0%|          | 0/1 [00:00<?, ?it/s]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "parsed = topic_parse(documents, chunk_size=2000, similarity_threshold=0.8, window_size=5)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 31,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "42"
      ]
     },
     "execution_count": 31,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "len(parsed['nodes'])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 32,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "{'chunk_size': 2000, 'threshold': 0.8, 'window': 5}\n",
      "node 0:\n",
      "We introduce a novel graph RAG method for applying LLMs to the medical domain, which we refer to as Medical Graph RAG (MedRAG).\n",
      "node 1:\n",
      "This technique improves LLM performance in the medical domain by response queries with grounded source citations and clear interpretations of medical terminology.\n",
      "node 2:\n",
      "This approach involves a three-tier hierarchical graph construction method.\n",
      "node 3:\n",
      "Initially, we use documents provided by users as our top-level source to extract entities.\n",
      "node 4:\n",
      "These entities are then linked to a second level consisting of more basic entities previously abstracted from credible medical books and papers. Subsequently, these entities are connected to a third level—the fundamental medical dictionary graph—that provides detailed explanations of each medical term and their semantic relationships. We then construct a comprehensive graph at the highest level by linking entities based on their content and hierarchical connections.\n",
      "node 5:\n",
      "This method ensures that the knowledge can be traced back to its sources and the results are factually accurate.\n",
      "node 6:\n",
      "We implement a U-retrieve strategy that combines top-down retrieval with bottom-up response generation. The process begins by structuring the query using predefined medical tags and indexing them through the graphs in a top-down manner. The system then generates responses based on these queries, pulling from meta-graphs—nodes retrieved along with their TopK related nodes and relationships—and summarizing the information into a detailed response. This technique maintains a balance between global context awareness and the contextual limitations inherent in LLMs. Our medical graph RAG provides Intrinsic source citation can enhance LLM transparency, interpretability, and verifiability. Intrinsic source citation can enhance LLM transparency, interpretability, and verifiability. The results provide provenance, or source grounding information, as it generates each response, and demonstrates that an answer is grounded in the dataset. Having the cited source for each assertion readily available also enables a human user to quickly and accurately audit the LLM's output directly against the original source material.\n",
      "node 7:\n",
      "It is super useful in the field of medicine that security is very important, and each of the reasoning should be evidence-based.\n",
      "node 8:\n",
      "By using such a method, we construct an evidence-based Medical LLM that the clinician could easily check the source of the reasoning and calibrate the model response to ensure the safety usage of llm in the clinical scenarios.\n",
      "node 9:\n",
      "We introduce a novel graph RAG method for applying LLMs to the medical domain, which we refer to as Medical Graph RAG (MedRAG).\n",
      "node 10:\n",
      "This technique improves LLM performance in the medical domain by response queries with grounded source citations and clear interpretations of medical terminology.\n",
      "node 11:\n",
      "This approach involves a three-tier hierarchical graph construction method.\n",
      "node 12:\n",
      "Initially, we use documents provided by users as our top-level source to extract entities.\n",
      "node 13:\n",
      "These entities are then linked to a second level consisting of more basic entities previously abstracted from credible medical books and papers. Subsequently, these entities are connected to a third level—the fundamental medical dictionary graph—that provides detailed explanations of each medical term and their semantic relationships. We then construct a comprehensive graph at the highest level by linking entities based on their content and hierarchical connections.\n",
      "node 14:\n",
      "This method ensures that the knowledge can be traced back to its sources and the results are factually accurate.\n",
      "node 15:\n",
      "We implement a U-retrieve strategy that combines top-down retrieval with bottom-up response generation. The process begins by structuring the query using predefined medical tags and indexing them through the graphs in a top-down manner. The system then generates responses based on these queries, pulling from meta-graphs—nodes retrieved along with their TopK related nodes and relationships—and summarizing the information into a detailed response. This technique maintains a balance between global context awareness and the contextual limitations inherent in LLMs. Our medical graph RAG provides Intrinsic source citation can enhance LLM transparency, interpretability, and verifiability. Intrinsic source citation can enhance LLM transparency, interpretability, and verifiability. The results provide provenance, or source grounding information, as it generates each response, and demonstrates that an answer is grounded in the dataset. Having the cited source for each assertion readily available also enables a human user to quickly and accurately audit the LLM's output directly against the original source material.\n",
      "node 16:\n",
      "It is super useful in the field of medicine that security is very important, and each of the reasoning should be evidence-based.\n",
      "node 17:\n",
      "By using such a method, we construct an evidence-based Medical LLM that the clinician could easily check the source of the reasoning and calibrate the model response to ensure the safety usage of llm in the clinical scenarios. We implemented the method to evaluate our medical graph RAG.\n",
      "node 18:\n",
      "The method was tested on several popular open and closed-source LLMs, including ChatGPT OpenAI (2023a) and LLaMA Touvron et al. (2023). The tests were conducted across mainstream medical Q&A benchmarks such as PubMedQA Jin et al. (2019), MedMCQA Pal et al. (2022), and USMLE Kung et al. (2023).\n",
      "node 19:\n",
      "For the RAG process, we supplied a comprehensive medical dictionary as the foundational knowledge layer. We also used the UMLS medical knowledge graph Lindberg et al. (1993) as the foundamental layer detailing semantic relationships.\n",
      "node 20:\n",
      "A curated MedC-K dataset Wu et al. (2023) was used as the intermediate level of data to simulate user-provided private data.\n",
      "node 21:\n",
      "Our experiments demonstrate that our model significantly enhances the performance of general-purpose LLMs on medical questions. Remarkably, it even surpasses many fine-tuned or specially trained LLMs on medical corpora, solely using the RAG approach without additional training.\n",
      "node 22:\n",
      "We introduce a novel graph RAG method for applying LLMs to the medical domain, which we refer to as Medical Graph RAG (MedRAG).\n",
      "node 23:\n",
      "This technique improves LLM performance in the medical domain by response queries with grounded source citations and clear interpretations of medical terminology.\n",
      "node 24:\n",
      "This approach involves a three-tier hierarchical graph construction method.\n",
      "node 25:\n",
      "Initially, we use documents provided by users as our top-level source to extract entities.\n",
      "node 26:\n",
      "These entities are then linked to a second level consisting of more basic entities previously abstracted from credible medical books and papers. Subsequently, these entities are connected to a third level—the fundamental medical dictionary graph—that provides detailed explanations of each medical term and their semantic relationships. We then construct a comprehensive graph at the highest level by linking entities based on their content and hierarchical connections.\n",
      "node 27:\n",
      "This method ensures that the knowledge can be traced back to its sources and the results are factually accurate.\n",
      "node 28:\n",
      "We implement a U-retrieve strategy that combines top-down retrieval with bottom-up response generation. The process begins by structuring the query using predefined medical tags and indexing them through the graphs in a top-down manner. The system then generates responses based on these queries, pulling from meta-graphs—nodes retrieved along with their TopK related nodes and relationships—and summarizing the information into a detailed response. This technique maintains a balance between global context awareness and the contextual limitations inherent in LLMs. Our medical graph RAG provides Intrinsic source citation can enhance LLM transparency, interpretability, and verifiability. Intrinsic source citation can enhance LLM transparency, interpretability, and verifiability. The results provide provenance, or source grounding information, as it generates each response, and demonstrates that an answer is grounded in the dataset. Having the cited source for each assertion readily available also enables a human user to quickly and accurately audit the LLM's output directly against the original source material.\n",
      "node 29:\n",
      "It is super useful in the field of medicine that security is very important, and each of the reasoning should be evidence-based.\n",
      "node 30:\n",
      "By using such a method, we construct an evidence-based Medical LLM that the clinician could easily check the source of the reasoning and calibrate the model response to ensure the safety usage of llm in the clinical scenarios. We implemented the method to evaluate our medical graph RAG.\n",
      "node 31:\n",
      "The method was tested on several popular open and closed-source LLMs, including ChatGPT OpenAI (2023a) and LLaMA Touvron et al. (2023). The tests were conducted across mainstream medical Q&A benchmarks such as PubMedQA Jin et al. (2019), MedMCQA Pal et al. (2022), and USMLE Kung et al. (2023).\n",
      "node 32:\n",
      "For the RAG process, we supplied a comprehensive medical dictionary as the foundational knowledge layer. We also used the UMLS medical knowledge graph Lindberg et al. (1993) as the foundamental layer detailing semantic relationships.\n",
      "node 33:\n",
      "A curated MedC-K dataset Wu et al. (2023) was used as the intermediate level of data to simulate user-provided private data.\n",
      "node 34:\n",
      "Our experiments demonstrate that our model significantly enhances the performance of general-purpose LLMs on medical questions. Remarkably, it even surpasses many fine-tuned or specially trained LLMs on medical corpora, solely using the RAG approach without additional training.\n",
      "node 35:\n",
      "We implement a U-retrieve strategy that combines top-down retrieval with bottom-up response generation. The process begins by structuring the query using predefined medical tags and indexing them through the graphs in a top-down manner. The system then generates responses based on these queries, pulling from meta-graphs—nodes retrieved along with their TopK related nodes and relationships—and summarizing the information into a detailed response. This technique maintains a balance between global context awareness and the contextual limitations inherent in LLMs. Our medical graph RAG provides Intrinsic source citation can enhance LLM transparency, interpretability, and verifiability. Intrinsic source citation can enhance LLM transparency, interpretability, and verifiability. The results provide provenance, or source grounding information, as it generates each response, and demonstrates that an answer is grounded in the dataset. Having the cited source for each assertion readily available also enables a human user to quickly and accurately audit the LLM's output directly against the original source material.\n",
      "node 36:\n",
      "It is super useful in the field of medicine that security is very important, and each of the reasoning should be evidence-based.\n",
      "node 37:\n",
      "By using such a method, we construct an evidence-based Medical LLM that the clinician could easily check the source of the reasoning and calibrate the model response to ensure the safety usage of llm in the clinical scenarios. We implemented the method to evaluate our medical graph RAG.\n",
      "node 38:\n",
      "The method was tested on several popular open and closed-source LLMs, including ChatGPT OpenAI (2023a) and LLaMA Touvron et al. (2023). The tests were conducted across mainstream medical Q&A benchmarks such as PubMedQA Jin et al. (2019), MedMCQA Pal et al. (2022), and USMLE Kung et al. (2023).\n",
      "node 39:\n",
      "For the RAG process, we supplied a comprehensive medical dictionary as the foundational knowledge layer. We also used the UMLS medical knowledge graph Lindberg et al. (1993) as the foundamental layer detailing semantic relationships.\n",
      "node 40:\n",
      "A curated MedC-K dataset Wu et al. (2023) was used as the intermediate level of data to simulate user-provided private data.\n",
      "node 41:\n",
      "Our experiments demonstrate that our model significantly enhances the performance of general-purpose LLMs on medical questions. Remarkably, it even surpasses many fine-tuned or specially trained LLMs on medical corpora, solely using the RAG approach without additional training.\n"
     ]
    }
   ],
   "source": [
    "print(parsed['params'])\n",
    "for i in range(len(parsed['nodes'])):\n",
    "    print(f'node {i}:\\n{parsed[\"nodes\"][i].get_content()}')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 127,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "application/vnd.jupyter.widget-view+json": {
       "model_id": "a8d7910c2ade4a9fbd62095ba868938e",
       "version_major": 2,
       "version_minor": 0
      },
      "text/plain": [
       "Parsing nodes:   0%|          | 0/1 [00:00<?, ?it/s]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "\n",
    "node_parser = TopicNodeParser.from_defaults(\n",
    "    llm=llm,\n",
    "    max_chunk_size=1000,\n",
    "    similarity_method=\"llm\",  # can be \"llm\" or \"embedding\"\n",
    "    window_size=5,  # paper suggests window_size=5\n",
    ")\n",
    "\n",
    "nodes = node_parser.get_nodes_from_documents(documents, show_progress=True)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 16,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "node 0:\n",
      "We introduce a novel graph RAG method for applying LLMs to the medical domain, which we refer to as Medical Graph RAG (MedRAG). The technique improves LLM performance in the medical domain by response queries with grounded source citations and clear interpretations of medical terminology. This approach involves a three-tier hierarchical graph construction method.\n",
      "node 1:\n",
      "Initially, we use documents provided by users as our top-level source to extract entities. These entities are then linked to a second level consisting of more basic entities previously abstracted from credible medical books and papers. Subsequently, these entities are connected to a third level—the fundamental medical dictionary graph—that provides detailed explanations of each medical term and their semantic relationships. We then construct a comprehensive graph at the highest level by linking entities based on their content and hierarchical connections. This method ensures that the knowledge can be traced back to its sources and the results are factually accurate.\n",
      "node 2:\n",
      "We implement a U-retrieve strategy that combines top-down retrieval with bottom-up response generation.\n",
      "node 3:\n",
      "The process begins by structuring the query using predefined medical tags.\n",
      "node 4:\n",
      "The query is indexed through the graphs in a top-down manner.\n",
      "node 5:\n",
      "The system then generates responses based on these queries. Responses are pulled from meta-graphs—nodes retrieved along with their TopK related nodes and relationships—\n",
      "node 6:\n",
      "Information is summarized into a detailed response by the system.\n",
      "node 7:\n",
      "This technique maintains a balance between global context awareness and the contextual limitations inherent in LLMs. RAG provides intrinsic source citation can enhance LLM transparency, interpretability, and verifiability. The results provide provenance or source grounding information as it generates each response. The method demonstrates that an answer is grounded in the dataset. Having the cited source for each assertion readily available enables a human user to quickly and accurately audit the LLM's output directly against the original source material.\n",
      "node 8:\n",
      "In the field of medicine, security is very important, and each reasoning should be evidence-based. Using such a method constructs an evidence-based Medical LLM that clinicians can easily check the source of the reasoning and calibrate the model response to ensure safe usage in clinical scenarios.\n",
      "node 9:\n",
      "We introduce a novel graph RAG method for applying LLMs to the medical domain, which we refer to as Medical Graph RAG (MedRAG). The technique improves LLM performance in the medical domain by response queries with grounded source citations and clear interpretations of medical terminology.\n",
      "node 10:\n",
      "This approach involves a three-tier hierarchical graph construction method.\n",
      "node 11:\n",
      "Initially, we use documents provided by users as our top-level source to extract entities. These entities are then linked to a second level consisting of more basic entities previously abstracted from credible medical books and papers. Subsequently, these entities are connected to a third level—the fundamental medical dictionary graph—that provides detailed explanations of each medical term and their semantic relationships. We then construct a comprehensive graph at the highest level by linking entities based on their content and hierarchical connections.\n",
      "node 12:\n",
      "This method ensures that the knowledge can be traced back to its sources and the results are factually accurate.\n",
      "node 13:\n",
      "We implement a U-retrieve strategy that combines top-down retrieval with bottom-up response generation.\n",
      "node 14:\n",
      "The process begins by structuring the query using predefined medical tags.\n",
      "node 15:\n",
      "The query is indexed through the graphs in a top-down manner.\n",
      "node 16:\n",
      "The system then generates responses based on these queries. Responses are pulled from meta-graphs—nodes retrieved along with their TopK related nodes and relationships— Information is summarized into a detailed response by the system.\n",
      "node 17:\n",
      "This technique maintains a balance between global context awareness and the contextual limitations inherent in LLMs. RAG provides intrinsic source citation can enhance LLM transparency, interpretability, and verifiability. The results provide provenance or source grounding information as it generates each response. The method demonstrates that an answer is grounded in the dataset. Having the cited source for each assertion readily available enables a human user to quickly and accurately audit the LLM's output directly against the original source material.\n",
      "node 18:\n",
      "In the field of medicine, security is very important, and each reasoning should be evidence-based. Using such a method constructs an evidence-based Medical LLM that clinicians can easily check the source of the reasoning and calibrate the model response to ensure safe usage in clinical scenarios.\n",
      "node 19:\n",
      "We implemented the method for evaluating our medical graph RAG.\n",
      "node 20:\n",
      "The method was tested on several popular open and closed-source LLMs, including ChatGPT OpenAI (2023a) and LLaMA Touvron et al. (2023).\n",
      "node 21:\n",
      "The LLMs were tested across mainstream medical Q&A benchmarks such as PubMedQA Jin et al. (2019), MedMCQA Pal et al. (2022), and USMLE Kung et al. (2023). For the RAG process, we supplied a comprehensive medical dictionary as the foundational knowledge layer. We also used the UMLS medical knowledge graph Lindberg et al. (1993) as the fundamental layer detailing semantic relationships.\n",
      "node 22:\n",
      "A curated MedC-K dataset Wu et al. (2023) was used as the intermediate level of data to simulate user-provided private data.\n",
      "node 23:\n",
      "Our experiments demonstrated that our model significantly enhances the performance of general-purpose LLMs on medical questions. The model even surpassed many fine-tuned or specially trained LLMs on medical corpora, solely using the RAG approach without additional training. We introduce a novel graph RAG method for applying LLMs to the medical domain, which we refer to as Medical Graph RAG (MedRAG). The technique improves LLM performance in the medical domain by response queries with grounded source citations and clear interpretations of medical terminology. This approach involves a three-tier hierarchical graph construction method.\n",
      "node 24:\n",
      "Initially, we use documents provided by users as our top-level source to extract entities. These entities are then linked to a second level consisting of more basic entities previously abstracted from credible medical books and papers. Subsequently, these entities are connected to a third level—the fundamental medical dictionary graph—that provides detailed explanations of each medical term and their semantic relationships. We then construct a comprehensive graph at the highest level by linking entities based on their content and hierarchical connections. This method ensures that the knowledge can be traced back to its sources and the results are factually accurate.\n",
      "node 25:\n",
      "We implement a U-retrieve strategy that combines top-down retrieval with bottom-up response generation.\n",
      "node 26:\n",
      "The process begins by structuring the query using predefined medical tags.\n",
      "node 27:\n",
      "The query is indexed through the graphs in a top-down manner.\n",
      "node 28:\n",
      "The system then generates responses based on these queries. Responses are pulled from meta-graphs—nodes retrieved along with their TopK related nodes and relationships— Information is summarized into a detailed response by the system.\n",
      "node 29:\n",
      "This technique maintains a balance between global context awareness and the contextual limitations inherent in LLMs. RAG provides intrinsic source citation can enhance LLM transparency, interpretability, and verifiability. The results provide provenance or source grounding information as it generates each response. The method demonstrates that an answer is grounded in the dataset. Having the cited source for each assertion readily available enables a human user to quickly and accurately audit the LLM's output directly against the original source material.\n",
      "node 30:\n",
      "In the field of medicine, security is very important, and each reasoning should be evidence-based. Using such a method constructs an evidence-based Medical LLM that clinicians can easily check the source of the reasoning and calibrate the model response to ensure safe usage in clinical scenarios.\n",
      "node 31:\n",
      "We implemented the method for evaluating our medical graph RAG.\n",
      "node 32:\n",
      "The method was tested on several popular open and closed-source LLMs, including ChatGPT OpenAI (2023a) and LLaMA Touvron et al. (2023).\n",
      "node 33:\n",
      "The LLMs were tested across mainstream medical Q&A benchmarks such as PubMedQA Jin et al. (2019), MedMCQA Pal et al. (2022), and USMLE Kung et al. (2023). For the RAG process, we supplied a comprehensive medical dictionary as the foundational knowledge layer. We also used the UMLS medical knowledge graph Lindberg et al. (1993) as the fundamental layer detailing semantic relationships. A curated MedC-K dataset Wu et al. (2023) was used as the intermediate level of data to simulate user-provided private data. Our experiments demonstrated that our model significantly enhances the performance of general-purpose LLMs on medical questions. The model even surpassed many fine-tuned or specially trained LLMs on medical corpora, solely using the RAG approach without additional training.\n",
      "node 34:\n",
      "We implement a U-retrieve strategy that combines top-down retrieval with bottom-up response generation.\n",
      "node 35:\n",
      "The process begins by structuring the query using predefined medical tags.\n",
      "node 36:\n",
      "The query is indexed through the graphs in a top-down manner.\n",
      "node 37:\n",
      "The system then generates responses based on these queries.\n",
      "node 38:\n",
      "Responses are pulled from meta-graphs—nodes retrieved along with their TopK related nodes and relationships—\n",
      "node 39:\n",
      "Information is summarized into a detailed response by the system.\n",
      "node 40:\n",
      "This technique maintains a balance between global context awareness and the contextual limitations inherent in LLMs. RAG provides intrinsic source citation can enhance LLM transparency, interpretability, and verifiability. The results provide provenance or source grounding information as it generates each response. The method demonstrates that an answer is grounded in the dataset. Having the cited source for each assertion readily available enables a human user to quickly and accurately audit the LLM's output directly against the original source material.\n",
      "node 41:\n",
      "In the field of medicine, security is very important, and each reasoning should be evidence-based. Using such a method constructs an evidence-based Medical LLM that clinicians can easily check the source of the reasoning and calibrate the model response to ensure safe usage in clinical scenarios.\n",
      "node 42:\n",
      "We implemented the method for evaluating our medical graph RAG.\n",
      "node 43:\n",
      "The method was tested on several popular open and closed-source LLMs, including ChatGPT OpenAI (2023a) and LLaMA Touvron et al. (2023). The LLMs were tested across mainstream medical Q&A benchmarks such as PubMedQA Jin et al. (2019), MedMCQA Pal et al. (2022), and USMLE Kung et al. (2023).\n",
      "node 44:\n",
      "For the RAG process, we supplied a comprehensive medical dictionary as the foundational knowledge layer. We also used the UMLS medical knowledge graph Lindberg et al. (1993) as the fundamental layer detailing semantic relationships.\n",
      "node 45:\n",
      "A curated MedC-K dataset Wu et al. (2023) was used as the intermediate level of data to simulate user-provided private data.\n",
      "node 46:\n",
      "Our experiments demonstrated that our model significantly enhances the performance of general-purpose LLMs on medical questions. The model even surpassed many fine-tuned or specially trained LLMs on medical corpora, solely using the RAG approach without additional training.\n"
     ]
    }
   ],
   "source": [
    "# window_size = 5\n",
    "for i in range(len(nodes)):\n",
    "    print(f'node {i}:\\n{nodes[i].get_content()}')\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "node 0:\n",
      "We introduce a novel graph RAG method for applying LLMs to the medical domain, which we refer to as Medical Graph RAG (MedRAG). The MedRAG technique improves LLM performance in the medical domain by response queries with grounded source citations and clear interpretations of medical terminology. This approach involves a three-tier hierarchical graph construction method.\n",
      "node 1:\n",
      "Initially, we use documents provided by users as our top-level source to extract entities. These entities are then linked to a second level consisting of more basic entities previously abstracted from credible medical books and papers. Subsequently, these entities are connected to a third level—the fundamental medical dictionary graph—that provides detailed explanations of each medical term and their semantic relationships. We then construct a comprehensive graph at the highest level by linking entities based on their content and hierarchical connections. This method ensures that the knowledge can be traced back to its sources and the results are factually accurate.\n",
      "node 2:\n",
      "We implement a U-retrieve strategy to respond to user queries. The U-retrieve strategy combines top-down retrieval with bottom-up response generation. The process begins by structuring the query using predefined medical tags. The predefined medical tags are indexed through graphs in a top-down manner. The system generates responses based on these queries. The responses are generated by pulling from meta-graphs. Meta-graphs include nodes retrieved along with their TopK related nodes and relationships.\n",
      "node 3:\n",
      "The information is summarized into a detailed response.\n",
      "node 4:\n",
      "This technique maintains a balance between global context awareness and contextual limitations inherent in LLMs.\n",
      "node 5:\n",
      "We introduce a novel graph RAG method for applying LLMs to the medical domain, which we refer to as Medical Graph RAG (MedRAG). The MedRAG technique improves LLM performance in the medical domain by response queries with grounded source citations and clear interpretations of medical terminology. This approach involves a three-tier hierarchical graph construction method.\n",
      "node 6:\n",
      "Initially, we use documents provided by users as our top-level source to extract entities. These entities are then linked to a second level consisting of more basic entities previously abstracted from credible medical books and papers. Subsequently, these entities are connected to a third level—the fundamental medical dictionary graph—that provides detailed explanations of each medical term and their semantic relationships.\n",
      "node 7:\n",
      "We then construct a comprehensive graph at the highest level by linking entities based on their content and hierarchical connections.\n",
      "node 8:\n",
      "This method ensures that the knowledge can be traced back to its sources and the results are factually accurate.\n",
      "node 9:\n",
      "We implement a U-retrieve strategy to respond to user queries. The U-retrieve strategy combines top-down retrieval with bottom-up response generation. The process begins by structuring the query using predefined medical tags. The predefined medical tags are indexed through graphs in a top-down manner. The system generates responses based on these queries. The responses are generated by pulling from meta-graphs. Meta-graphs include nodes retrieved along with their TopK related nodes and relationships.\n",
      "node 10:\n",
      "The information is summarized into a detailed response.\n",
      "node 11:\n",
      "This technique maintains a balance between global context awareness and contextual limitations inherent in LLMs. RAG provides Intrinsic source citation can enhance LLM transparency, interpretability, and verifiability. The results provide provenance, or source grounding information, as it generates each response. The results demonstrate that an answer is grounded in the dataset. Having the cited source for each assertion readily available enables a human user to quickly and accurately audit the LLM's output directly against the original source material.\n",
      "node 12:\n",
      "It is super useful in the field of medicine that security is very important, and each of the reasoning should be evidence-based. By using such a method, we construct an evidence-based Medical LLM that the clinician could easily check the source of the reasoning and calibrate the model response to ensure the safety usage of llm in the clinical scenarios.\n",
      "node 13:\n",
      "We implement a U-retrieve strategy to respond to user queries. The U-retrieve strategy combines top-down retrieval with bottom-up response generation. The process begins by structuring the query using predefined medical tags. The predefined medical tags are indexed through graphs in a top-down manner. The system generates responses based on these queries. The responses are generated by pulling from meta-graphs. Meta-graphs include nodes retrieved along with their TopK related nodes and relationships.\n",
      "node 14:\n",
      "The information is summarized into a detailed response.\n",
      "node 15:\n",
      "This technique maintains a balance between global context awareness and contextual limitations inherent in LLMs. RAG provides Intrinsic source citation can enhance LLM transparency, interpretability, and verifiability. The results provide provenance, or source grounding information, as it generates each response. The results demonstrate that an answer is grounded in the dataset. Having the cited source for each assertion readily available enables a human user to quickly and accurately audit the LLM's output directly against the original source material.\n",
      "node 16:\n",
      "It is super useful in the field of medicine that security is very important, and each of the reasoning should be evidence-based. By using such a method, we construct an evidence-based Medical LLM that the clinician could easily check the source of the reasoning and calibrate the model response to ensure the safety usage of llm in the clinical scenarios.\n",
      "node 17:\n",
      "We implemented the method for evaluating our medical graph RAG.\n",
      "node 18:\n",
      "The method was tested on several popular open and closed-source LLMs, including ChatGPT OpenAI (2023a) and LLaMA Touvron et al. (2023).\n",
      "node 19:\n",
      "These models were tested across mainstream medical Q&A benchmarks such as PubMedQA Jin et al. (2019), MedMCQA Pal et al. (2022), and USMLE Kung et al. (2023).\n",
      "node 20:\n",
      "For the RAG process, we supplied a comprehensive medical dictionary as the foundational knowledge layer. The UMLS medical knowledge graph Lindberg et al. (1993) was used as the fundamental layer detailing semantic relationships.\n",
      "node 21:\n",
      "A curated MedC-K dataset Wu et al. (2023) —comprising the latest medical papers and books—was used as the intermediate level of data to simulate user-provided private data. Our experiments demonstrate that our model significantly enhances the performance of general-purpose LLMs on medical questions. Remarkably, it even surpasses many fine-tuned or specially trained LLMs on medical corpora, solely using the RAG approach without additional training. RAG provides Intrinsic source citation can enhance LLM transparency, interpretability, and verifiability. The results provide provenance, or source grounding information, as it generates each response. The results demonstrate that an answer is grounded in the dataset. Having the cited source for each assertion readily available enables a human user to quickly and accurately audit the LLM's output directly against the original source material.\n",
      "node 22:\n",
      "It is super useful in the field of medicine that security is very important, and each of the reasoning should be evidence-based. By using such a method, we construct an evidence-based Medical LLM that the clinician could easily check the source of the reasoning and calibrate the model response to ensure the safety usage of llm in the clinical scenarios.\n",
      "node 23:\n",
      "We implemented the method for evaluating our medical graph RAG.\n",
      "node 24:\n",
      "The method was tested on several popular open and closed-source LLMs, including ChatGPT OpenAI (2023a) and LLaMA Touvron et al. (2023). These models were tested across mainstream medical Q&A benchmarks such as PubMedQA Jin et al. (2019), MedMCQA Pal et al. (2022), and USMLE Kung et al. (2023).\n",
      "node 25:\n",
      "For the RAG process, we supplied a comprehensive medical dictionary as the foundational knowledge layer. The UMLS medical knowledge graph Lindberg et al. (1993) was used as the fundamental layer detailing semantic relationships.\n",
      "node 26:\n",
      "A curated MedC-K dataset Wu et al. (2023) —comprising the latest medical papers and books—was used as the intermediate level of data to simulate user-provided private data.\n",
      "node 27:\n",
      "Our experiments demonstrate that our model significantly enhances the performance of general-purpose LLMs on medical questions. Remarkably, it even surpasses many fine-tuned or specially trained LLMs on medical corpora, solely using the RAG approach without additional training.\n"
     ]
    }
   ],
   "source": [
    "# window_size = 2\n",
    "for i in range(len(nodes)):\n",
    "    print(f'node {i}:\\n{nodes[i].get_content()}')\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.10.12"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
